--
-- PostgreSQL database dump
--

-- Dumped from database version 11.13 (Debian 11.13-1.pgdg90+1)
-- Dumped by pg_dump version 11.13 (Debian 11.13-1.pgdg90+1)

SET statement_timeout = 0;
SET lock_timeout = 0;
SET idle_in_transaction_session_timeout = 0;
SET client_encoding = 'UTF8';
SET standard_conforming_strings = on;
SELECT pg_catalog.set_config('search_path', '', false);
SET check_function_bodies = false;
SET xmloption = content;
SET client_min_messages = warning;
SET row_security = off;



COPY public.teams (id, name, created_at, updated_at, created_by_id, last_modified_by_id, description, space_taken) FROM stdin;
1	My projects	2019-05-30 12:40:52.83987	2022-02-09 15:06:12.563535	\N	\N	\N	4701045
\.

--
-- Data for Name: users; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.users (id, full_name, initials, email, encrypted_password, reset_password_token, reset_password_sent_at, remember_created_at, sign_in_count, current_sign_in_at, last_sign_in_at, current_sign_in_ip, last_sign_in_ip, created_at, updated_at, confirmation_token, confirmed_at, confirmation_sent_at, unconfirmed_email, invitation_token, invitation_created_at, invitation_sent_at, invitation_accepted_at, invitation_limit, invited_by_type, invited_by_id, invitations_count, current_team_id, authentication_token, settings, variables, two_factor_auth_enabled, otp_secret, otp_recovery_codes, failed_attempts, locked_at, unlock_token) FROM stdin;
1	Admin	A	young.daniel@epa.gov	$2a$10$AI3j9Y6Wux8xL53K/Uc.WufmW0yVrasJ7TUXRm6aZUJ0/rgj2Qe/i	\N	\N	\N	66	2022-02-15 15:23:04.606563	2022-02-15 11:39:02.513113	161.80.29.93	161.80.29.93	2019-05-30 12:40:52.813253	2022-02-15 15:23:04.60712	\N	2019-05-30 12:40:52.799236	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	1	xHEzdvKY7PLrRhR8QF8Q	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":true,\\"recent\\":true,\\"recent_email\\":true,\\"system_message_email\\":true},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1559174400}}"	f	6GC77RTZHEYIACJ2ETO4U6BV4F25XLE6	\N	0	\N	\N
\.

--
-- Data for Name: teams; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.teams (id, name, created_at, updated_at, created_by_id, last_modified_by_id, description, space_taken) FROM stdin;
2	ECEB SW	2019-07-22 11:19:36.207198	2019-08-28 07:31:09.519806	1	\N	\N	4701045
12	Evaluating de-packaging technologies to remove film plastics from food	2020-09-28 12:17:20.091773	2020-11-18 15:11:37.524353	1	\N	The US Environmental Protection Agency (US EPA) is committed to the establishment of reproducible, representative, accurate, precise methods for micro-plastics (MPs) and nano-plastics (NPs) analysis. These methods also include appropriate quality assurance/quality control (QA/QC) for micro-plastics (MPs) and nano-plastics (NPs) sample collection; extraction from food, food waste, surface and drinking water, dust, sediment and tissue samples. This information will assist in improving micro-plastics (MPs) and nano-plastics (NPs) size, shape and chemical composition characterization, and quantification, particularly for particles in the micron scale size for which information is limited and is relevant to quantifying human and ecological exposure risks.\r\n\r\nThe United States government is committed to reducing food waste by 50% by 2030. To divert food waste from landfills, the US EPA’s Office of Research and Development’s (ORD), Center for Environmental Solutions & Emergency Response (CESER) promotes anaerobic digestion and composting of food waste. However, contamination of food waste with packaging (including film plastics) may complicate composting and anaerobic digestion operations and decrease the market desirability and safety of land application of the compost and digestate resulting from food waste.\r\n\r\nORD in collaboration with our partners within the Agency are developing technologies to test for micro-plastics (MPs) and nano-plastics (NPs) in assessing their impact and the role they have at the interface and within a food waste system. This project, I am leading, involves a team that is developing methods for extracting, characterizing, quantifying and understanding the effects of micro- and nano-plastics (MPs & NPs) in food waste. By studying the potential impacts of these micro- and nano-plastics (MPs & NPs) we can identify how to improve the performance of de-packaging equipment available on the market. We are looking to work with community partners to aid us in the characterization of plastics in food waste streams. This project is designed to have samples collected from 5 - 10 cities across the US, along with 10 food de-packaging facilities, such as restaurants and groceries chosen in each city. The samples will be shipped to the EPA Cincinnati Laboratories, where they will be qualitatively and quantitatively analyzed for micro- and nano-plastics (MPs & NPs). The results from this study will be useful to EPA, restaurant and commercial kitchen operators, food retailers, composters, and wastewater treatment facility staff across the US as they seek to exclude plastics (including micro- and nano-plastics (MPs & NPs)) from the food waste stream.	5231859
10	A-E Team	2020-05-12 16:13:00.284008	2020-05-12 16:34:15.526063	1	\N	\N	5231859
6	R7-Molecular Lab	2019-09-12 18:26:18.054881	2019-09-12 18:26:33.43636	1	\N	\N	4701045
24	FTOH ELN001 (TEST)	2021-08-23 14:58:26.978882	2021-08-26 20:20:07.686571	1	1	\N	5244963
11	CMTB-Mash	2020-06-10 14:43:15.978463	2020-06-10 15:04:44.199439	1	\N	\N	5231859
4	RTEB	2019-08-28 08:34:51.359068	2019-08-28 08:35:56.866066	1	\N	\N	4701045
13	KR	2020-10-07 19:20:23.229065	2020-10-07 19:40:57.552567	1	\N	\N	5231859
23	LC/MS/MS	2021-06-02 16:10:00.462428	2021-06-02 16:31:02.101243	1	\N	\N	5231859
5	R7-Molecular Lab	2019-08-29 12:51:19.486476	2020-06-10 18:25:03.609062	1	\N	\N	4701045
18	Eunice - Legionella	2021-05-20 14:13:33.839527	2021-05-20 14:34:30.57684	1	\N	\N	5231859
9	R7-Microbiology Lab	2019-11-25 18:00:52.098551	2019-11-25 18:21:20.16996	1	\N	\N	5231859
19	MPham	2021-05-20 14:21:20.351753	2021-05-20 15:31:51.303947	1	\N	\N	5231859
21	CCL 5 pesticide DW method	2021-05-20 15:17:11.524468	2021-05-20 15:37:54.570242	1	\N	\N	5231859
16	Alan ' 's lab	2021-05-20 14:12:30.238326	2021-05-26 12:40:04.764169	1	\N	\N	5231859
8	GCRD	2019-11-13 13:57:40.713766	2019-11-13 14:18:38.234422	1	\N	\N	5231859
20	Luxton research group	2021-05-20 14:30:30.144143	2021-05-20 14:51:35.179965	1	\N	\N	5231859
17	Materials Management Lab	2021-05-20 14:12:56.981498	2021-05-20 15:42:01.148981	1	\N	\N	5231859
25	FTOH ELN001	2021-08-23 17:19:39.023542	2021-08-26 20:53:28.260365	1	\N	\N	5231859
14	RonH	2020-11-18 14:53:24.231173	2020-11-18 15:14:41.164133	1	\N	\N	5231859
22	TLee ' 's lab	2021-05-28 15:07:36.615028	2021-07-13 16:25:11.952453	1	\N	\N	9736564
15	LEAF	2021-05-19 20:01:51.360151	2021-05-19 20:22:18.911992	1	\N	\N	5231859
26	PFAS Treatment Team	2021-11-05 12:34:58.662331	2021-11-05 12:34:58.662331	1	\N	\N	1048576
27	Nick Sylvest	2021-11-15 13:50:58.092875	2021-11-15 13:50:58.092875	1	\N	\N	1048576
7	archived	2019-10-21 16:34:58.92351	2021-12-08 18:52:03.348985	1	1		5231859
3	archived 2	2019-08-27 15:14:57.367514	2021-12-08 18:52:25.540081	1	1	LSASD/FSB	4701045
28	TSC Micro Team	2021-12-08 18:53:57.721613	2021-12-08 18:53:57.721613	1	\N	Office of Water-OGWDW-SRMD-TSC\r\nLab Cert Micro Team	1048576
29	Development	2022-02-09 13:04:39.900794	2022-02-09 13:04:39.900794	1	\N	\N	1048576
30	Plastics Projects	2022-02-09 18:56:38.604925	2022-02-09 18:56:38.604925	1	\N	\N	1048576
\.


COPY public.users (id, full_name, initials, email, encrypted_password, reset_password_token, reset_password_sent_at, remember_created_at, sign_in_count, current_sign_in_at, last_sign_in_at, current_sign_in_ip, last_sign_in_ip, created_at, updated_at, confirmation_token, confirmed_at, confirmation_sent_at, unconfirmed_email, invitation_token, invitation_created_at, invitation_sent_at, invitation_accepted_at, invitation_limit, invited_by_type, invited_by_id, invitations_count, current_team_id, authentication_token, settings, variables, two_factor_auth_enabled, otp_secret, otp_recovery_codes, failed_attempts, locked_at, unlock_token) FROM stdin;
7	Steven Baker	SB	baker.steven@epa.gov	$2a$10$ICPlfcM6UFw6tK4V2lxlvuqtyyZh6oOPM.zgTFNj1xMGVjhRuGwdu	7489485018182da8466f40011a62bb60ee4934daea9fd4e589015f729c683181	2020-06-10 18:00:49.039638	\N	4	2020-06-10 18:02:05.605739	2019-09-12 14:32:32.679852	134.67.29.82	172.18.0.1	2019-08-29 12:51:19.049895	2021-04-08 15:54:38.835465	\N	2019-08-29 12:51:19.049895	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	5	f39jNruysQpnqqjxymiu	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1564963200}}"	f	\N	\N	0	\N	\N
8	Michael F Davis	MFD	davis.michael@epa.gov	$2a$10$rJ87MOlGW/k83YJAIBmuEeGI3/ZBS1VtN3cpt7.Bn/SHoc3Ak3E1K	\N	\N	\N	1	2019-09-12 18:26:19.73168	2019-09-12 18:26:19.73168	204.47.3.224	204.47.3.224	2019-09-12 18:26:18.03283	2019-10-21 16:32:02.794186	\N	2019-09-12 18:26:18.03283	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	6	Nya-wA3UuKGdTfqmVdsx	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1568160000}}"	f	\N	\N	0	\N	\N
11	Lynn Tran	LT	tran.lynn@epa.gov	$2a$10$c4SqOT3tq9UcpqnSRVw4MOsEhSPPBEHrwrQ0b8tmE4bPAkxVKTyLC	\N	\N	\N	1	2020-05-12 16:13:07.22217	2020-05-12 16:13:07.22217	134.67.29.93	134.67.29.93	2020-05-12 16:13:00.113092	2020-05-12 16:13:07.223206	\N	2020-05-12 16:13:00.113092	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	10	tveaCuSfP91fbWQ6xxsC	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1588896000}}"	f	\N	\N	0	\N	\N
9	Mustafa Bob	MB	bob.mustafa@epa.gov	$2a$10$g8bkjodGVTMeCGLtrgCT/Ol2vHg.A1QIRwFcC2kko8.rFmE6iL.Ae	\N	\N	2020-09-03 01:48:10.626984	2	2020-09-03 01:48:10.834271	2019-11-13 13:57:43.796409	161.80.29.78	204.46.247.16	2019-11-13 13:57:40.620931	2020-09-03 01:48:10.835428	\N	2019-11-13 13:57:40.620931	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	8	qhR1NL9s9DtXUUszJ3DH	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1573516800}}"	f	\N	\N	0	\N	\N
12	Heath Mash	HM	mash.heath@gmail.com	$2a$10$JYa8qUZ6f0t4mI8D9KMe0ui4KL0AgxGxXPUDKt9SxPLGX2subJYbu	\N	\N	\N	1	2020-06-10 14:43:18.907293	2020-06-10 14:43:18.907293	134.67.29.73	134.67.29.73	2020-06-10 14:43:15.835836	2020-06-10 14:43:18.90789	\N	2020-06-10 14:43:15.835836	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	11	NTcpCpkmzxU6UGDhmBaF	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1590624000}}"	f	\N	\N	0	\N	\N
13	foster.albert@epa.gov	FO	foster.albert@epa.gov	$2a$10$dZ6zFp1jB3LKb7DTe7JVmOkR2TOMUUVVHzhRNLtyV/BVq5eIj4/ue	aee12581ea308dabb2d9f38a0990211c875658e8b42f1d1fc78553c1b3c2d218	2021-04-08 15:15:25.475489	\N	0	\N	\N	\N	\N	2020-09-28 12:22:40.689852	2021-04-08 15:15:25.476092	\N	\N	\N	\N	722027274700af8e9f337e046ef355685fa8c5016e51962a3460ffc9f7ef00a6	2020-09-28 12:22:40.687277	2020-09-28 12:22:54.081772	\N	\N	User	1	0	\N	NitRFXZZNzgaTGo8ui3x	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1598572800}}"	f	\N	\N	0	\N	\N
14	allen.derrick01@epa.gov	AL	allen.derrick01@epa.gov	$2a$10$vzVH3JH/4TMY9j9qQHXPkOyjgPzqASi6cKv08nwRSQSadcODW42SG	\N	\N	\N	0	\N	\N	\N	\N	2020-09-28 12:22:40.847571	2020-09-28 12:22:54.295458	\N	\N	\N	\N	ac9ff132412e6cd0de5ba077fffe02133a85f01924f5de9193fa485f52829fe8	2020-09-28 12:22:40.844769	2020-09-28 12:22:54.294223	\N	\N	User	1	0	\N	2ZcXNzqcLAzQd2wd4i6c	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1598572800}}"	f	\N	\N	0	\N	\N
6	Raghuraman Venkatapathy	RV	raghuraman.venkatapathy@ptsied.com	$2a$10$e6/M68mBJKrIOpHytnacBu0i4RaV7H27vuvpKsHPU9IQfZJZz4Ony	08b387c75ceb1f74e785fa8744ec6cff10d12498394bf9b06c36dcf4e1c47716	2021-09-30 16:28:01.213528	\N	5	2021-04-08 11:22:32.33496	2019-10-21 20:06:05.636033	204.47.160.55	204.47.160.55	2019-08-28 08:34:51.227403	2021-09-30 16:28:01.214296	\N	2019-08-28 08:34:51.227403	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	4	R4E9xZZKKR8VCWjkSXsN	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1564963200}}"	f	\N	\N	0	\N	\N
3	venkatapathy.raghuraman@epa.gov	VE	venkatapathy.raghuraman@epa.gov	$2a$10$/dKTM0FTl/MeAtW4.g990OsHSSVp8G1JvJsnffAXGmLVeNj1gvUKS	d8ea7e30e300a30f593f4910d36421ea898d1154d8f877bdee67ea0af2bad1f1	2021-09-30 19:04:33.545981	\N	0	\N	\N	\N	\N	2019-07-22 11:21:59.866896	2021-09-30 19:04:33.546807	\N	\N	\N	\N	de9ddaacc35d44ac4b3bada836305caea0254770992f584ea17c885b996b0a0c	2019-07-22 11:21:59.865212	2019-07-22 11:22:51.774585	\N	\N	User	1	0	\N	gaEcUK9sQAQegKzk7Y7n	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1563321600}}"	f	\N	\N	0	\N	\N
10	Laura Webb	LW	webb.laura@epa.gov	$2a$10$70w..Tlhq6IHYBkkzdE65udtbBwIa3J2vVWDfc9q0f3qyxyO25T3.	\N	\N	\N	3	2020-05-28 18:43:35.844853	2019-11-25 18:05:01.301753	161.80.29.81	204.47.4.37	2019-11-25 18:00:52.074068	2021-12-08 18:50:02.411233	\N	2019-11-25 18:00:52.074068	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	7	_HH6Em2_VVQijNRiXA5x	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1574380800}}"	f	\N	\N	0	\N	\N
4	Regina Klepikow	RK	klepikow.regina@epa.gov	$2a$10$GptAP1Mk0atDb1kVMVo0/erltYXNalBzhTUcq3jZdKMOTb4CMtGLS	b2c459b046a51a146e98f8d0a624992c9396e805ebaae72429d6e314db73d6e7	2019-09-12 15:31:59.543897	2021-04-08 14:38:25.665847	23	2021-12-08 18:44:51.975804	2021-04-08 14:38:25.675991	134.67.29.87	204.47.4.45	2019-08-27 15:14:57.171544	2021-12-08 18:53:57.743378	\N	2019-08-27 15:14:57.171544	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	7	bRW9P9XYL2RPjfYEkAjn	"{\\"time_zone\\":\\"Central Time (US \\\\u0026 Canada)\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":true,\\"recent\\":true,\\"recent_email\\":true,\\"system_message_email\\":true},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1564963200}}"	f	\N	\N	0	\N	\N
2	gonzalez.michael@epa.gov	GO	gonzalez.michael@epa.gov	$2a$10$73fPcbWH.U0VWkgUXULpUe6Qsa39qz5tQSnJ.I8PleNoB1Feo.dq.	\N	\N	\N	0	\N	\N	\N	\N	2019-07-22 11:21:59.412842	2020-09-28 13:00:11.577763	\N	\N	\N	\N	6caa4949ba6d96013058273d6964df2ddba1393d6ffe4c51577f7cfb6ab0ac9f	2019-07-22 11:21:59.40912	2019-07-22 11:22:51.227472	\N	\N	User	1	0	\N	zT9qv9cMH6zwsZCnB5h3	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1563321600}}"	f	\N	\N	0	\N	\N
15	allen01.derrick@epa.gov	AL	allen01.derrick@epa.gov	$2a$10$byatYzR1JoqUM7by1.aP2uFAyTRzMM.ib9d1zpwaK0xuB9VXxoIUK	\N	\N	\N	0	\N	\N	\N	\N	2020-09-28 13:00:11.955584	2020-09-28 13:00:57.502619	\N	\N	\N	\N	933357f0d6c1f011be24148a84fcf4516313d7f4b1cd943a8460e655b2082df7	2020-09-28 13:00:11.953199	2020-09-28 13:00:57.502027	\N	\N	User	1	0	\N	zVBcagHh5Zz7ErVgTG6c	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1598572800}}"	f	\N	\N	0	\N	\N
16	Kristie Rue	KR	rue.kristie@epa.gov	$2a$10$8wAb0JGT9lNbu0MONsAG9.UAiCewx/q/TyitM6t444ynfWahtB1zC	\N	\N	\N	1	2020-10-07 19:20:27.818384	2020-10-07 19:20:27.818384	204.46.247.45	204.46.247.45	2020-10-07 19:20:22.941078	2020-10-07 19:20:27.819381	\N	2020-10-07 19:20:22.941078	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	13	8RaCM63yHzW-m22H_RT4	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1598572800}}"	f	\N	\N	0	\N	\N
18	shan.yonggui@epa.gov	SH	shan.yonggui@epa.gov	$2a$10$n/R5NmLXZzlVzARMKRjpw.5Wh4fA.OZPdPJ0.BXNEsnTZLUfwqoqO	\N	\N	\N	0	\N	\N	\N	\N	2021-04-08 10:19:49.162409	2021-04-08 10:20:15.149898	\N	\N	\N	\N	75d39bd8e114119d9f22e1102140444a01096d54e54bde2a79f99044125ccc10	2021-04-08 10:19:49.159554	2021-04-08 10:20:15.14879	\N	\N	User	1	0	\N	KLGYaH6Xd8LojH_yig1y	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1617321600}}"	f	\N	\N	0	\N	\N
17	Ronald Hermann	RH	hermann.ronald@epa.gov	$2a$10$jbkgKNy97v0uFl0iSqfb4ObfCFlHqDUZtC8DYDKLkrh8BzpOnYpE6	\N	\N	\N	1	2020-11-18 14:53:26.274456	2020-11-18 14:53:26.274456	161.80.29.64	161.80.29.64	2020-11-18 14:53:24.207357	2020-11-18 14:53:26.275249	\N	2020-11-18 14:53:24.207357	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	14	oEMs7YQ8ss79MpzJsemQ	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1605657600}}"	f	\N	\N	0	\N	\N
22	Max Krause	MK	krause.max@epa.gov	$2a$10$qEcRsvTgu9lsHagJoRvCN.1/guzMY2QvvZsuUOYUZHvib0bLcLw4.	\N	\N	\N	1	2021-05-20 14:12:57.959648	2021-05-20 14:12:57.959648	204.47.179.90	204.47.179.90	2021-05-20 14:12:56.969142	2021-05-20 14:12:57.960071	\N	2021-05-20 14:12:56.969142	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	17	T8ZS1ebnz6rH9zSZ4Dhz	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
20	Susan Thorneloe	ST	thorneloe.susan@epa.gov	$2a$10$6t/z.U0oiED3Wwt0bQYGG.viuMXdR09kwYWZnQLtgLFtx3tPtvDpe	\N	\N	\N	1	2021-05-19 20:01:52.972442	2021-05-19 20:01:52.972442	134.67.29.66	134.67.29.66	2021-05-19 20:01:51.32971	2021-05-19 20:01:52.972877	\N	2021-05-19 20:01:51.32971	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	15	CeWCc3BJWzmBzjqAinqS	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
19	meyer.david@epa.gov	ME	meyer.david@epa.gov	$2a$10$kQM.OX9wnajiEH5FGPRTsOoIaYLhfqevcpSPkiSsVkjB5oIQl1L2a	836ea9abad7672034399508070a6b161e5130a6630a248ff2c6e4f94642869e7	2021-09-20 15:39:48.360367	\N	0	\N	\N	\N	\N	2021-04-08 10:20:34.124229	2021-09-20 15:39:48.36096	\N	\N	\N	\N	1df2189955f7a5e8ab93f4060df48e9f166fd079c45f9838c4163077afc1ccd9	2021-04-08 10:20:34.121408	2021-04-08 10:20:35.774106	\N	\N	User	1	0	\N	tpXjsCwxjNsj9vkYRCh3	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1617321600}}"	f	\N	\N	0	\N	\N
24	Maily Pham	MP	pham.maily@epa.gov	$2a$10$urYHHNT1kKkDkj4SHlVHeeAlf8OUwhDkuhJD50RbO5bhKPZ9fydSC	\N	\N	\N	1	2021-05-20 14:21:21.501064	2021-05-20 14:21:21.501064	134.67.29.84	134.67.29.84	2021-05-20 14:21:20.335829	2021-05-20 14:22:42.023893	\N	2021-05-20 14:21:20.335829	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	19	72R6uVtN6bzskyHMZ_yJ	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%Y-%m-%d\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":true,\\"recent\\":true,\\"recent_email\\":true,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
23	Eunice Varughese	EV	varughese.eunice@epa.gov	$2a$10$MldIMSpZ0lhBVCDlIyAUDO3zwei4YFidDmBWpkWf23/lxvQBh6Fr2	\N	\N	\N	1	2021-05-20 14:13:34.840683	2021-05-20 14:13:34.840683	161.80.29.82	161.80.29.82	2021-05-20 14:13:33.829409	2021-05-20 14:21:31.804195	\N	2021-05-20 14:13:33.829409	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	18	EvaE6Pr_DihAtzKnedJs	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":true,\\"recent\\":true,\\"recent_email\\":true,\\"system_message_email\\":true},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
21	Alan Lindquist	AL	lindquist.alan@epa.gov	$2a$10$PUnwbkrTn2lYMeTLmE4n.eJgbwQFaijem9DWDPk6PmEIfKjHeAFYO	\N	\N	2021-05-27 13:29:14.911352	8	2021-08-03 17:27:05.27712	2021-07-28 19:33:54.815647	134.67.29.72	204.47.164.128	2021-05-20 14:12:30.225562	2021-08-03 17:27:05.278242	\N	2021-05-20 14:12:30.225562	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	16	iETe5zGCjigX1eQQS21X	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":true,\\"recent\\":true,\\"recent_email\\":true,\\"system_message_email\\":true},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
25	Anna Wade	AW	wade.anna@epa.gov	$2a$10$0QC0h53zVcJ8sRzgFkgmzO3jdxBqYYlfnap6xUT5LBi9fmnmjvyim	\N	\N	\N	1	2021-05-20 14:30:31.16005	2021-05-20 14:30:31.16005	161.80.29.92	161.80.29.92	2021-05-20 14:30:30.128607	2021-05-20 14:30:31.160442	\N	2021-05-20 14:30:30.128607	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	20	Un9iyyTP2rdCSN3LHoLA	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
26	Jody Shoemaker	JS	shoemaker.jody@epa.gov	$2a$10$TUUhKT.WKDcJOZ1KjDnIPunWhTBOX3bF.L.vjxVmfnulk3Slx/lOG	\N	\N	\N	1	2021-05-20 15:17:12.523612	2021-05-20 15:17:12.523612	204.47.172.138	204.47.172.138	2021-05-20 15:17:11.508651	2021-05-20 15:17:12.523997	\N	2021-05-20 15:17:11.508651	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	21	SoFDeR-3Rpbnw5zZ4Mt_	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
29	nadagouda.mallikarjuna@epa.gov	NA	nadagouda.mallikarjuna@epa.gov	$2a$10$6SU9oQ1QZqbBFDA8jF9bgukNT/.bhB4ICxcjhSHDQ6VEtUe5iyo2y	\N	\N	\N	0	\N	\N	\N	\N	2021-06-10 16:09:28.042134	2021-06-10 16:10:22.091605	\N	\N	\N	\N	14a432ea8ee32d428c2893e45fdf96949ce9811fe863c161dd50be95911e1d8b	2021-06-10 16:09:28.0154	2021-06-10 16:10:22.089669	\N	\N	User	27	0	\N	FeGXgy6N6nPsM1-xmPa9	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
27	Tae Lee	TL	lee.tae@epa.gov	$2a$10$x78usbFUGXqAOl9QLTOGbefBluAJt6rt9ojB/A1vdE6J7CWXsgRY6	\N	\N	\N	12	2021-07-15 14:29:10.200551	2021-07-13 15:38:18.10693	161.80.29.93	204.47.164.139	2021-05-28 15:07:36.363425	2021-07-15 14:29:10.201947	\N	2021-05-28 15:07:36.363425	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	22	Fax4YjdDZfMfbEsiHsJ6	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	0	\N	\N
30	Aaron Tam	AT	tam.aaron@epa.gov	$2a$10$pepmYUY3QYv/skievSYELOi1cGdgrcokVWoLOQdKSEFn6mH2htVi2	87e34a91c6eff845f1df8997576394085d5f36f546af95756d16b44d10dc0fbe	2021-09-17 13:26:38.638593	2021-08-24 13:38:23.116705	6	2021-09-17 13:36:31.804534	2021-08-27 13:35:23.405832	134.67.29.91	172.19.0.1	2021-08-23 14:58:26.913388	2021-09-17 13:36:31.805933	\N	2021-08-23 14:58:26.913388	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	24	8sZMzxewMijaAbTVciFj	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1628812800}}"	f	\N	\N	0	\N	\N
31	JAMES VOIT	JV	voit.jim@epa.gov	$2a$10$G46TVYVEl1OHhDlPK6JNmeYyoYRYjhxmzmdDO34bDXOLLU.cAEP8O	fedd3c3ad7abf6aaffc91c264a596bc893487b91f702b3da888d73cc0d7a5d91	2021-09-20 15:34:04.232885	2021-08-25 13:44:41.129058	3	2021-08-26 19:42:13.706395	2021-08-25 13:44:41.136026	204.47.165.224	204.47.165.224	2021-08-23 17:19:39.010008	2021-09-20 15:34:04.233974	\N	2021-08-23 17:19:39.010008	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	25	xYqqN3erNzKefPajkr3T	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1628812800}}"	f	\N	\N	0	\N	\N
32	Samantha Smith	SS	smith.samantha@epa.gov	$2a$10$Ty4j2XSIdFw2jqALkLWQee0aoRBqV/GR00cS.CLwOV9XIGaH7kyYe	\N	\N	\N	1	2021-11-05 12:34:58.802778	2021-11-05 12:34:58.802778	204.47.162.64	204.47.162.64	2021-11-05 12:34:58.555595	2021-11-05 12:34:58.803029	\N	2021-11-05 12:34:58.555595	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	26	\N	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false}}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1635120000}}"	f	\N	\N	0	\N	\N
28	Dennis Fine	D	fine.dennis@epa.gov	$2a$10$n4b3Loe4PD.2WZ/zm9Q/CeE1XAa0FQ7PlRFCtGRWLNhrj827WtTRW	f1e14886b35d4e23fbde5ac92bfa70914e3ea63fcb7454003df8d7a454f0bccb	2021-10-05 13:43:59.204446	\N	1	2021-06-02 16:10:07.782103	2021-06-02 16:10:07.782103	204.46.243.38	204.46.243.38	2021-06-02 16:10:00.138764	2021-10-05 13:43:59.204734	\N	2021-06-02 16:10:00.138764	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	23	fqNs-YuTTNEjvpetxCfc	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":true,\\"recent\\":true,\\"recent_email\\":true,\\"system_message_email\\":true},\\"tooltips_enabled\\":true}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1619136000}}"	f	\N	\N	2	\N	\N
34	Jacob Specht	JS	specht.jacob@epa.gov	$2a$10$868rX7wI.FlKXBrm9HCRoOrGEu0CJg06o1sE2ORlW8lApv.R88WvC	\N	\N	\N	4	2022-02-09 13:09:46.903001	2022-02-09 13:09:10.773939	10.140.75.102	10.140.75.102	2022-02-09 13:04:39.882306	2022-02-09 13:34:28.279862	\N	2022-02-09 13:04:39.882306	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	1	\N	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false}}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1642723200}}"	f	\N	\N	0	\N	\N
33	Nicholas Sylvest	NS	sylvest.nicholas01@epa.gov	$2a$10$Cmh/Q6oH4e5GCCEjB4Fh.OCzVdSxWfEvX382n5zof553Vduu8nvIu	\N	\N	\N	7	2022-02-15 12:35:32.931863	2022-02-14 18:35:19.11753	204.47.160.156	204.47.160.156	2021-11-15 13:50:58.002181	2022-02-15 12:35:32.932196	\N	2021-11-15 13:50:58.002181	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	1	\N	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false}}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1635120000}}"	f	\N	\N	0	\N	\N
35	Kameron King	KK	king.kameron@epa.gov	$2a$10$0FQKQCJPHrf0Y3e7Y3v/gOjfGTDQSFaKFQZd2h51vQw0hRXuVnqma	\N	\N	\N	1	2022-02-09 18:56:38.623583	2022-02-09 18:56:38.623583	204.47.160.55	204.47.160.55	2022-02-09 18:56:38.590426	2022-02-09 18:56:38.623715	\N	2022-02-09 18:56:38.590426	\N	\N	\N	\N	\N	\N	\N	\N	\N	0	30	\N	"{\\"time_zone\\":\\"UTC\\",\\"date_format\\":\\"%m/%d/%Y\\",\\"notifications_settings\\":{\\"assignments\\":true,\\"assignments_email\\":false,\\"recent\\":true,\\"recent_email\\":false,\\"system_message_email\\":false}}"	"{\\"export_vars\\":{\\"num_of_export_all_last_24_hours\\":0,\\"last_export_timestamp\\":1644364800}}"	f	\N	\N	0	\N	\N
\.


--
-- Data for Name: experiments; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.experiments (id, name, description, project_id, created_by_id, last_modified_by_id, archived, archived_by_id, archived_on, restored_by_id, restored_on, created_at, updated_at, uuid) FROM stdin;
7	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	6	8	8	f	\N	\N	\N	\N	2019-09-11 21:13:10.47327	2019-09-12 18:26:33.434148	\N
4	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	4	6	6	f	\N	\N	\N	\N	2019-08-25 03:45:05.464976	2019-08-28 08:35:56.863654	\N
2	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	2	4	4	f	\N	\N	\N	\N	2019-08-21 11:56:40.917123	2019-08-27 15:15:47.721524	\N
3	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	3	1	1	f	\N	\N	\N	\N	2019-08-23 15:29:59.608762	2019-08-28 07:31:09.517491	\N
5	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	5	7	7	f	\N	\N	\N	\N	2019-08-22 14:17:08.934627	2019-08-30 16:10:44.500335	\N
6	Mobile Test	Iphone	5	7	7	f	\N	\N	\N	\N	2019-09-12 14:33:50.886125	2019-09-12 14:33:50.886125	\N
8	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	22	9	9	f	\N	\N	\N	\N	2019-11-10 00:16:15.875063	2019-11-13 13:58:19.261473	\N
1	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	1	1	1	f	\N	\N	\N	\N	2019-05-30 04:48:13.863771	2019-10-16 08:08:21.63949	\N
16	90 Day Colilert Media Quality Check	QC - E.coli, E. aerogenes, P. aruginosa, Blank	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:20:50.295736	2019-11-25 14:20:50.295736	\N
17	HPC for Reagent Water		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:25:55.020193	2019-11-25 14:25:55.020193	\N
18	Conductivity for Reagent Water		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:26:13.096919	2019-11-25 14:26:13.096919	\N
19	Chlorine for Reagent Water		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:28:10.554657	2019-11-25 14:28:10.554657	\N
20	Heavy Metals for Reagent Water		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:28:20.873292	2019-11-25 14:28:20.873292	\N
21	Quanti-Tray Sealer Leak Check		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:28:32.559704	2019-11-25 14:28:32.559704	\N
22	Balance Verification		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:28:43.653071	2019-11-25 14:28:43.653071	\N
10	HPC for Reagent Water	Monthly Reagent Water Monitoring - HPC by Idexx QT system	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:11:09.852588	2019-11-25 14:12:46.050167	\N
11	Conductivity for Reagent Water	Monthly Reagent Water Monitoring - Conductivity Meter	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:13:16.785085	2019-11-25 14:13:16.785085	\N
12	Chlorine for Reagent Water	Monthly Reagent Water Monitoring - Chlorine	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:14:34.008707	2019-11-25 14:14:34.008707	\N
13	Heavy Metals for Reagent Water	Annual Reagent Water Monitoring (on all 4 systems)	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:15:39.833671	2019-11-25 14:15:39.833671	\N
14	Quanti-Tray Sealer Leak Check	Monthly Quati-Tray Sealer Monitoring	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:16:46.27006	2019-11-25 14:16:46.27006	\N
15	Balance Verification	Monthly Balance Verification (20mg, 200mg, 2g, 20g and 100g)	24	4	4	f	\N	\N	\N	\N	2019-11-25 14:17:56.234319	2019-11-25 14:17:56.234319	\N
25	Media QA/QC		25	4	4	f	\N	\N	\N	\N	2019-11-25 14:31:00.036023	2019-11-25 14:31:00.036023	\N
26	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	26	10	10	f	\N	\N	\N	\N	2019-11-22 18:51:45.388487	2019-11-25 18:01:05.361296	\N
23	Autoclave Sterilization Check		25	4	4	f	4	2021-04-08 15:37:07	\N	2021-04-08 15:37:22	2019-11-25 14:29:22.052009	2021-04-08 15:46:42.622691	\N
27	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	27	11	11	f	\N	\N	\N	\N	2020-05-12 07:42:53.299935	2020-05-12 16:13:55.089825	\N
32	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	33	16	16	f	\N	\N	\N	\N	2020-10-06 10:07:00.511065	2020-10-07 19:20:39.705081	\N
28	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	28	12	12	f	\N	\N	\N	\N	2020-06-05 14:52:33.358783	2020-06-10 14:44:24.935647	\N
29	1111		29	7	7	f	\N	\N	\N	\N	2020-06-10 18:15:19.809319	2020-06-10 18:16:28.996471	\N
30	TC EC Analysis	Analysis of Total Coliforms and E. coli by ColiLert	30	4	4	f	\N	\N	\N	\N	2020-06-12 11:48:19.677608	2020-06-12 11:48:19.677608	\N
31	FT-IR Analysis	Follow CESER approved SOP 'K-LRTD-SOP-4042-0. '	32	1	1	f	\N	\N	\N	\N	2020-09-28 12:25:52.862621	2021-05-20 10:36:28.301948	\N
24	Autoclave Timing Check	Using a timer, time the 3 stages of the autoclave cycle. Warm Up Stage, Sterilization Stage and Exhaust Stage. The timer and internal data logger must coincide with the correct times.	25	4	4	f	\N	\N	\N	\N	2019-11-25 14:30:18.355678	2021-12-08 18:46:18.510614	\N
34	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	35	17	17	f	\N	\N	\N	\N	2020-11-12 16:23:25.638378	2020-11-18 14:54:18.042757	\N
33	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	34	1	1	f	\N	\N	\N	\N	2020-11-14 11:39:41.864506	2020-11-18 14:51:01.254084	\N
37	Quantifying MPs using Stereomicroscopy	Micro-plastics particle size of less than 5 mm are an emerging contaminant for aquatic environmental, which have attracted increasing attention in worldwide range. Micro-plastics Rapid Screening Approach: A rapid-screening approach to detect and quantify micro-plastics based on fluorescent tagging with Nile Red using a stereomicroscope and camera.	32	1	1	f	\N	\N	\N	\N	2020-12-14 11:29:53.045357	2021-05-20 10:26:27.745201	\N
36	Scanning Electron Microscopy (SEM)	Can identify plastic‐like particles under 0.1 mm.18.	32	1	1	f	\N	\N	\N	\N	2020-12-14 11:28:28.645902	2020-12-14 11:28:28.645902	\N
35	Raman	Raman microscopy can detect the smallest microstructures and particles (>0.5 µm), which is usually a desired feature in microplastic analysis. However, on top of all its good qualities, it does come with certain sample requirements.	32	1	1	f	\N	\N	\N	\N	2020-12-14 11:26:50.758223	2021-05-20 13:08:29.798881	\N
40	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	39	22	22	f	\N	\N	\N	\N	2021-05-16 03:28:40.313284	2021-05-20 14:13:37.776415	\N
38	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	37	20	20	f	\N	\N	\N	\N	2021-05-18 09:14:45.996413	2021-05-19 20:02:04.304944	\N
41	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	40	23	23	f	\N	\N	\N	\N	2021-05-15 23:24:39.024115	2021-05-20 14:13:48.174919	\N
39	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	38	21	21	f	\N	\N	\N	\N	2021-05-15 17:02:29.687747	2021-05-20 14:13:34.017487	\N
44	Sample Analysis	...	32	1	1	f	\N	\N	\N	\N	2021-05-20 14:26:58.303864	2021-05-20 14:26:58.303864	\N
45	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	43	25	25	f	\N	\N	\N	\N	2021-05-20 10:35:48.685042	2021-05-20 14:31:19.360192	\N
43	TOC		42	24	24	f	\N	\N	\N	\N	2021-05-20 14:23:28.636464	2021-05-20 15:31:51.302113	\N
47	Alaska BMP	RARE project 2252 for Region 10	39	22	22	f	\N	\N	\N	\N	2021-05-20 15:42:01.143602	2021-05-20 15:42:01.143602	\N
48	WIZ Sensor at Lake Harsha	Data from the WIZ sensor, Identified as EPA2, installed at the intake of lake Willian H. Harsha.  IT is planned that if the sensor is moved, another experiment will be opened.  This should include notes about the operations, calibration and maintenance of this sensor.	45	21	21	f	\N	\N	\N	\N	2021-05-26 12:40:04.598881	2021-05-26 12:40:04.598881	\N
42	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	41	24	24	f	\N	\N	\N	\N	2021-05-16 23:12:32.329187	2021-05-20 14:46:34.043993	\N
46	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	44	26	26	f	\N	\N	\N	\N	2021-05-15 07:43:14.703189	2021-05-20 15:17:42.773422	\N
53	20210713 Product A,B,C and Alumina 1, 2,3mm	Need to characterize alumina pellet	48	27	27	f	\N	\N	\N	\N	2021-07-12 18:03:35.833531	2021-07-13 16:25:11.950177	\N
49	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	46	27	27	f	\N	\N	\N	\N	2021-05-24 19:11:25.090486	2021-05-28 15:08:48.287706	\N
51	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	54	28	28	f	\N	\N	\N	\N	2021-05-28 08:05:28.065726	2021-06-02 16:10:44.635418	\N
50	20210610 PO4 adsorption test (two samples)	Two new sample adsorption test for PO4 using 1000 ppm solution	48	27	27	f	\N	\N	\N	\N	2021-06-01 13:42:09.273576	2021-06-16 15:59:27.116185	\N
52	20210706 Alumina adsorbent mix for P adsorption	New sample (100% alumina) will be delivered. mix with 5% Biocar  to test for P recovery	48	27	27	f	\N	\N	\N	\N	2021-07-06 15:48:25.913631	2021-07-07 16:08:45.092141	\N
54	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	55	30	30	f	\N	\N	\N	\N	2021-08-16 19:18:00.747016	2021-08-23 14:59:01.086088	\N
58	Moisture Content Determination		59	30	30	f	\N	\N	\N	\N	2021-08-26 20:02:40.596649	2021-08-26 20:20:07.683988	\N
55	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	56	31	31	f	\N	\N	\N	\N	2021-08-17 09:20:33.866158	2021-08-23 17:20:19.943739	\N
56	PFAS Liquid Extraction		57	30	30	f	\N	\N	\N	\N	2021-08-23 18:50:06.211439	2021-08-23 19:47:33.520041	\N
57	TEST EXP 1		58	30	30	f	\N	\N	\N	\N	2021-08-25 16:06:39.685453	2021-08-25 16:08:54.104645	\N
9	Polymerase chain reaction	Polymerase chain reaction (PCR) monitors the amplification of DNA in real time (qPCR cyclers constantly scan qPCR plates). It is, in contrast to the conventional PCR, quantitative, meaning that it enables us to determine the exact concentration (relative or absolute) of the amplified DNA in the sample. Conversely, in conventional PCR we can see the result of amplification only after the PCR is completed (end-point detection).\n\n       Apart from DNA, RNA can also be used as a template (e.g. in case of gene expression studies or detection of RNA viruses). In this case, the RNA needs to be reverse transcribed into DNA (also termed complementary DNA or cDNA) before it is amplified with real-time PCR. There is a term for this combined method: real-time reverse transcription PCR or qRT-PCR (sometimes RT-qPCR) for short.	23	4	4	f	\N	\N	\N	\N	2019-11-23 02:20:37.470969	2021-12-08 18:45:29.081264	\N
59	Test Experiment	Test Experiment	32	1	1	f	\N	\N	\N	\N	2022-02-09 15:06:12.552217	2022-02-09 15:06:12.552217	\N
\.


--
-- Data for Name: my_module_groups; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.my_module_groups (id, created_at, updated_at, created_by_id, experiment_id) FROM stdin;
1	2019-05-30 12:40:53.219019	2019-05-30 12:40:53.219019	\N	1
2	2019-08-27 15:15:00.103911	2019-08-27 15:15:00.103911	\N	2
3	2019-08-28 07:30:58.615745	2019-08-28 07:30:58.615745	\N	3
4	2019-08-28 08:34:53.604088	2019-08-28 08:34:53.604088	\N	4
5	2019-08-29 12:51:23.187426	2019-08-29 12:51:23.187426	\N	5
6	2019-09-12 18:26:18.413201	2019-09-12 18:26:18.413201	\N	7
7	2019-11-13 13:57:41.525534	2019-11-13 13:57:41.525534	\N	8
8	2019-11-25 14:09:22.544972	2019-11-25 14:09:22.544972	\N	9
9	2019-11-25 17:01:05.280245	2019-11-25 17:01:05.280245	4	23
10	2019-11-25 18:00:52.451647	2019-11-25 18:00:52.451647	\N	26
11	2020-05-12 16:13:01.721683	2020-05-12 16:13:01.721683	\N	27
12	2020-06-10 14:43:16.91934	2020-06-10 14:43:16.91934	\N	28
13	2020-10-07 19:20:24.960506	2020-10-07 19:20:24.960506	\N	32
14	2020-11-18 14:50:52.194252	2020-11-18 14:50:52.194252	\N	33
15	2020-11-18 14:53:24.736357	2020-11-18 14:53:24.736357	\N	34
16	2021-05-19 20:01:51.683159	2021-05-19 20:01:51.683159	\N	38
17	2021-05-20 10:26:02.414406	2021-05-20 10:26:02.414406	1	37
19	2021-05-20 10:35:58.798343	2021-05-20 10:35:58.798343	1	31
20	2021-05-20 13:07:49.626333	2021-05-20 13:07:49.626333	1	35
21	2021-05-20 14:12:30.445251	2021-05-20 14:12:30.445251	\N	39
22	2021-05-20 14:12:57.146402	2021-05-20 14:12:57.146402	\N	40
23	2021-05-20 14:13:33.97899	2021-05-20 14:13:33.97899	\N	41
24	2021-05-20 14:21:20.536665	2021-05-20 14:21:20.536665	\N	42
25	2021-05-20 14:30:30.304084	2021-05-20 14:30:30.304084	\N	45
26	2021-05-20 15:17:11.693783	2021-05-20 15:17:11.693783	\N	46
27	2021-05-28 15:07:44.746239	2021-05-28 15:07:44.746239	\N	49
28	2021-06-02 16:10:01.698618	2021-06-02 16:10:01.698618	\N	51
29	2021-08-23 14:58:27.589312	2021-08-23 14:58:27.589312	\N	54
30	2021-08-23 17:19:39.189064	2021-08-23 17:19:39.189064	\N	55
31	2021-08-23 19:46:48.233581	2021-08-23 19:46:48.233581	30	56
32	2021-08-26 20:19:40.529447	2021-08-26 20:19:40.529447	30	58
\.


-- --
-- -- Data for Name: my_module_statuses; Type: TABLE DATA; Schema: public; Owner: postgres
-- --

-- COPY public.my_module_statuses (id, name, description, color, my_module_status_flow_id, previous_status_id, created_by_id, last_modified_by_id, created_at, updated_at) FROM stdin;
-- 1	Not started	\N	#406d86	1	\N	\N	\N	2021-09-30 14:06:04.375996	2021-09-30 14:06:04.375996
-- 2	In progress	\N	#0065ff	1	1	\N	\N	2021-09-30 14:06:04.382981	2021-09-30 14:06:04.382981
-- 3	Completed	\N	#00b900	1	2	\N	\N	2021-09-30 14:06:04.39738	2021-09-30 14:06:04.39738
-- \.


--
-- Data for Name: my_modules; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.my_modules (id, name, due_date, description, x, y, my_module_group_id, created_at, updated_at, archived, archived_on, created_by_id, last_modified_by_id, archived_by_id, restored_by_id, restored_on, workflow_order, experiment_id, state, completed_on, started_on, my_module_status_id, status_changing, changing_from_my_module_status_id) FROM stdin;
64	Experiment design	2019-12-02 14:09:22.549976	\N	0	0	8	2019-11-22 16:16:36.193531	2021-12-08 18:45:29.080009	f	\N	4	\N	\N	\N	\N	0	9	0	\N	\N	1	f	2
133	Quarterly Timing Check	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>QC - Attachment 3</div>\r\n<div>&nbsp;</div>\r\n<div>The purpose of this document is to provide information, requirements and procedures for sterilizing supplies and the handling and disposal of environmental microbiological waste that is produced from the region&rsquo;s microbiology laboratory and mobile laboratory. This standard operating procedure (SOP) is applicable to all Field Services Branch (FSB) personnel working with microbiological waste.</div>\r\n<div>&nbsp;</div>	0	0	\N	2021-04-08 13:20:10.094708	2021-12-08 18:46:18.508214	f	\N	4	4	\N	\N	\N	-1	24	0	\N	\N	2	f	\N
1	Experiment design	2019-06-06 12:40:53.220431	\N	0	0	1	2019-05-26 15:45:33.824176	2019-05-30 12:40:53.282545	f	\N	1	\N	\N	\N	\N	0	1	0	\N	\N	2	f	\N
2	Sampling biological material	2019-06-20 12:40:53.283067	\N	32	0	1	2019-05-29 23:30:38.504863	2019-05-30 12:40:53.429442	f	\N	1	\N	\N	\N	\N	1	1	0	\N	\N	2	f	\N
3	RNA isolation	2019-07-04 12:40:53.299186	\N	64	0	1	2019-05-26 06:56:23.215328	2019-05-30 12:40:53.308013	f	\N	1	\N	\N	\N	\N	2	1	0	\N	\N	2	f	\N
4	RNA quality & quantity - BIOANALYSER	2019-07-18 12:40:53.30854	\N	96	0	1	2019-05-29 20:30:14.028165	2019-05-30 12:40:53.317561	f	\N	1	\N	\N	\N	\N	3	1	0	\N	\N	2	f	\N
5	Reverse transcription	2019-08-01 12:40:53.318081	\N	96	16	1	2019-05-25 03:14:36.556992	2019-05-30 12:40:53.326766	f	\N	1	\N	\N	\N	\N	4	1	0	\N	\N	2	f	\N
6	qPCR	2019-08-15 12:40:53.327251	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	1	2019-05-28 08:41:50.492279	2019-05-30 12:40:53.335989	f	\N	1	\N	\N	\N	\N	5	1	0	\N	\N	2	f	\N
7	Data quality control	2019-08-29 12:40:53.336545	\N	32	16	1	2019-05-27 17:52:09.512858	2019-05-30 12:40:53.344802	f	\N	1	\N	\N	\N	\N	6	1	0	\N	\N	2	f	\N
8	Data analysis - ddCq	2019-09-12 12:40:53.34531	\N	0	16	1	2019-05-27 15:14:25.26159	2019-05-30 12:40:53.353197	f	\N	1	\N	\N	\N	\N	7	1	0	\N	\N	2	f	\N
9	Data analysis - Pfaffl method	2019-06-06 12:40:53.353741	\N	-1	-1	\N	2019-05-27 12:02:53.915739	2019-05-30 12:40:53.361557	t	2019-05-30 12:40:53	1	\N	1	\N	\N	-1	1	0	\N	\N	2	f	\N
154	Prep	\N	\N	47	17	20	2021-05-20 13:07:49.526094	2021-05-20 13:07:49.628307	f	\N	1	1	\N	\N	\N	1	35	0	\N	\N	2	f	\N
153	Sample	\N	\N	0	0	20	2021-05-20 13:07:49.500638	2021-05-20 13:07:49.630408	f	\N	1	1	\N	\N	\N	0	35	0	\N	\N	2	f	\N
87	Experiment design	2020-05-19 16:13:01.727425	\N	0	0	11	2020-05-08 04:52:09.41664	2020-05-12 16:13:02.052697	f	\N	11	\N	\N	\N	\N	0	27	0	\N	\N	2	f	\N
88	Sampling biological material	2020-06-02 16:13:02.05345	\N	32	0	11	2020-05-06 16:25:05.848504	2020-05-12 16:13:02.540466	f	\N	11	\N	\N	\N	\N	1	27	0	\N	\N	2	f	\N
89	RNA isolation	2020-06-16 16:13:02.112145	\N	64	0	11	2020-05-09 01:49:11.151782	2020-05-12 16:13:02.128298	f	\N	11	\N	\N	\N	\N	2	27	0	\N	\N	2	f	\N
90	RNA quality & quantity - BIOANALYSER	2020-06-30 16:13:02.128914	\N	96	0	11	2020-05-09 23:53:09.698891	2020-05-12 16:13:02.151914	f	\N	11	\N	\N	\N	\N	3	27	0	\N	\N	2	f	\N
91	Reverse transcription	2020-07-14 16:13:02.153092	\N	96	16	11	2020-05-06 21:42:52.581572	2020-05-12 16:13:02.18101	f	\N	11	\N	\N	\N	\N	4	27	0	\N	\N	2	f	\N
92	qPCR	2020-07-28 16:13:02.181741	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	11	2020-05-11 19:42:06.087553	2020-05-12 16:13:02.241434	f	\N	11	\N	\N	\N	\N	5	27	0	\N	\N	2	f	\N
93	Data quality control	2020-08-11 16:13:02.242093	\N	32	16	11	2020-05-10 00:38:53.894432	2020-05-12 16:13:02.256034	f	\N	11	\N	\N	\N	\N	6	27	0	\N	\N	2	f	\N
94	Data analysis - ddCq	2020-08-25 16:13:02.256657	\N	0	16	11	2020-05-12 04:28:15.912628	2020-05-12 16:13:02.27052	f	\N	11	\N	\N	\N	\N	7	27	0	\N	\N	2	f	\N
95	Data analysis - Pfaffl method	2020-05-19 16:13:02.271843	\N	-1	-1	\N	2020-05-07 18:11:09.579671	2020-05-12 16:13:02.29156	t	2020-05-12 16:13:02	11	\N	11	\N	\N	-1	27	0	\N	\N	2	f	\N
238	Measure using spectrophotometer	\N	\N	0	48	\N	2021-07-06 15:50:12.408232	2021-07-06 15:50:12.408232	f	\N	27	27	\N	\N	\N	-1	52	0	\N	\N	2	f	\N
96	Experiment design	2020-06-17 14:43:16.922955	\N	0	0	12	2020-06-09 13:52:17.963744	2020-06-10 14:43:17.103143	f	\N	12	\N	\N	\N	\N	0	28	0	\N	\N	2	f	\N
97	Sampling biological material	2020-07-01 14:43:17.103898	\N	32	0	12	2020-06-05 09:26:05.333544	2020-06-10 14:43:17.400664	f	\N	12	\N	\N	\N	\N	1	28	0	\N	\N	2	f	\N
98	RNA isolation	2020-07-15 14:43:17.141421	\N	64	0	12	2020-06-08 10:15:53.302282	2020-06-10 14:43:17.156841	f	\N	12	\N	\N	\N	\N	2	28	0	\N	\N	2	f	\N
99	RNA quality & quantity - BIOANALYSER	2020-07-29 14:43:17.157508	\N	96	0	12	2020-06-06 13:02:43.756946	2020-06-10 14:43:17.171845	f	\N	12	\N	\N	\N	\N	3	28	0	\N	\N	2	f	\N
100	Reverse transcription	2020-08-12 14:43:17.172501	\N	96	16	12	2020-06-05 21:09:30.272857	2020-06-10 14:43:17.187785	f	\N	12	\N	\N	\N	\N	4	28	0	\N	\N	2	f	\N
101	qPCR	2020-08-26 14:43:17.188438	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	12	2020-06-08 09:35:06.203518	2020-06-10 14:43:17.233664	f	\N	12	\N	\N	\N	\N	5	28	0	\N	\N	2	f	\N
102	Data quality control	2020-09-09 14:43:17.234282	\N	32	16	12	2020-06-05 15:40:58.364239	2020-06-10 14:43:17.247675	f	\N	12	\N	\N	\N	\N	6	28	0	\N	\N	2	f	\N
103	Data analysis - ddCq	2020-09-23 14:43:17.248295	\N	0	16	12	2020-06-04 18:42:20.951618	2020-06-10 14:43:17.262058	f	\N	12	\N	\N	\N	\N	7	28	0	\N	\N	2	f	\N
104	Data analysis - Pfaffl method	2020-06-17 14:43:17.262701	\N	-1	-1	\N	2020-06-09 22:51:19.269387	2020-06-10 14:43:17.275364	t	2020-06-10 14:43:17	12	\N	12	\N	\N	-1	28	0	\N	\N	2	f	\N
22	RNA quality & quantity - BIOANALYSER	2019-10-16 07:30:58.756303	\N	96	0	3	2019-08-22 13:41:19.09097	2019-08-28 07:30:59.508303	f	\N	1	\N	\N	\N	\N	3	3	0	\N	\N	2	f	\N
10	Experiment design	2019-09-03 15:15:00.140772	\N	0	0	2	2019-08-27 06:09:10.754213	2019-08-27 15:15:00.633308	f	\N	4	\N	\N	\N	\N	0	2	0	\N	\N	2	f	\N
11	Sampling biological material	2019-09-17 15:15:00.634119	\N	32	0	2	2019-08-26 03:03:12.321432	2019-08-27 15:15:01.40422	f	\N	4	\N	\N	\N	\N	1	2	0	\N	\N	2	f	\N
12	RNA isolation	2019-10-01 15:15:00.731216	\N	64	0	2	2019-08-25 12:28:51.531855	2019-08-27 15:15:00.754615	f	\N	4	\N	\N	\N	\N	2	2	0	\N	\N	2	f	\N
13	RNA quality & quantity - BIOANALYSER	2019-10-15 15:15:00.755755	\N	96	0	2	2019-08-25 19:56:12.821464	2019-08-27 15:15:00.799297	f	\N	4	\N	\N	\N	\N	3	2	0	\N	\N	2	f	\N
14	Reverse transcription	2019-10-29 15:15:00.800081	\N	96	16	2	2019-08-21 16:41:32.346974	2019-08-27 15:15:00.817558	f	\N	4	\N	\N	\N	\N	4	2	0	\N	\N	2	f	\N
174	Sampling biological material	2021-06-10 14:13:33.9888	\N	32	0	23	2021-05-16 05:54:10.795334	2021-05-20 14:13:34.104031	f	\N	23	\N	\N	\N	\N	1	41	0	\N	\N	2	f	\N
15	qPCR	2019-11-12 15:15:00.818361	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	2	2019-08-22 10:03:05.694629	2019-08-27 15:15:00.844456	f	\N	4	\N	\N	\N	\N	5	2	0	\N	\N	2	f	\N
16	Data quality control	2019-11-26 15:15:00.846264	\N	32	16	2	2019-08-21 23:45:38.7469	2019-08-27 15:15:00.868717	f	\N	4	\N	\N	\N	\N	6	2	0	\N	\N	2	f	\N
17	Data analysis - ddCq	2019-12-10 15:15:00.870857	\N	0	16	2	2019-08-23 01:19:28.265534	2019-08-27 15:15:00.900582	f	\N	4	\N	\N	\N	\N	7	2	0	\N	\N	2	f	\N
18	Data analysis - Pfaffl method	2019-09-03 15:15:00.901611	\N	-1	-1	\N	2019-08-24 03:27:54.475847	2019-08-27 15:15:00.977233	t	2019-08-27 15:15:00	4	\N	4	\N	\N	-1	2	0	\N	\N	2	f	\N
19	Experiment design	2019-09-04 07:30:58.620874	\N	0	0	3	2019-08-25 02:29:16.041099	2019-08-28 07:30:58.687067	f	\N	1	\N	\N	\N	\N	0	3	0	\N	\N	2	f	\N
24	qPCR	2019-11-13 07:30:58.835847	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	3	2019-08-23 01:29:55.93468	2019-08-28 07:30:59.669061	f	\N	1	\N	\N	\N	\N	5	3	0	\N	\N	2	f	\N
25	Data quality control	2019-11-27 07:30:58.876679	\N	32	16	3	2019-08-26 09:18:42.766777	2019-08-28 07:30:59.844306	f	\N	1	\N	\N	\N	\N	6	3	0	\N	\N	2	f	\N
106	Experiment design	2020-10-14 19:20:24.966368	\N	0	0	13	2020-10-07 08:59:37.819201	2020-10-07 19:20:25.18097	f	\N	16	\N	\N	\N	\N	0	32	0	\N	\N	2	f	\N
107	Sampling biological material	2020-10-28 19:20:25.181712	\N	32	0	13	2020-10-06 10:25:33.491869	2020-10-07 19:20:25.570387	f	\N	16	\N	\N	\N	\N	1	32	0	\N	\N	2	f	\N
108	RNA isolation	2020-11-11 19:20:25.218883	\N	64	0	13	2020-10-02 08:22:29.710808	2020-10-07 19:20:25.237849	f	\N	16	\N	\N	\N	\N	2	32	0	\N	\N	2	f	\N
109	RNA quality & quantity - BIOANALYSER	2020-11-25 19:20:25.238484	\N	96	0	13	2020-10-06 06:49:51.467566	2020-10-07 19:20:25.254007	f	\N	16	\N	\N	\N	\N	3	32	0	\N	\N	2	f	\N
27	Data analysis - Pfaffl method	2019-09-04 07:30:58.979589	\N	-1	-1	\N	2019-08-26 16:50:52.33039	2019-08-28 07:30:59.029884	t	2019-08-28 07:30:58	1	\N	1	\N	\N	-1	3	0	\N	\N	2	f	\N
20	Sampling biological material	2019-09-18 07:30:58.696027	\N	32	0	3	2019-08-25 04:50:00.284925	2019-08-28 07:30:59.341177	f	\N	1	\N	\N	\N	\N	1	3	0	\N	\N	2	f	\N
110	Reverse transcription	2020-12-09 19:20:25.254656	\N	96	16	13	2020-10-07 10:59:45.374834	2020-10-07 19:20:25.271548	f	\N	16	\N	\N	\N	\N	4	32	0	\N	\N	2	f	\N
23	Reverse transcription	2019-10-30 07:30:58.784873	\N	96	16	3	2019-08-24 04:06:02.588524	2019-08-28 07:30:59.58858	f	\N	1	\N	\N	\N	\N	4	3	0	\N	\N	2	f	\N
111	qPCR	2020-12-23 19:20:25.272501	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	13	2020-10-05 19:49:06.439159	2020-10-07 19:20:25.353409	f	\N	16	\N	\N	\N	\N	5	32	0	\N	\N	2	f	\N
112	Data quality control	2021-01-06 19:20:25.354118	\N	32	16	13	2020-10-06 06:48:56.770031	2020-10-07 19:20:25.369687	f	\N	16	\N	\N	\N	\N	6	32	0	\N	\N	2	f	\N
113	Data analysis - ddCq	2021-01-20 19:20:25.370363	\N	0	16	13	2020-10-02 01:42:36.101561	2020-10-07 19:20:25.387166	f	\N	16	\N	\N	\N	\N	7	32	0	\N	\N	2	f	\N
114	Data analysis - Pfaffl method	2020-10-14 19:20:25.388591	\N	-1	-1	\N	2020-10-03 19:58:35.174116	2020-10-07 19:20:25.404559	t	2020-10-07 19:20:25	16	\N	16	\N	\N	-1	32	0	\N	\N	2	f	\N
26	Data analysis - ddCq	2019-12-11 07:30:58.923403	\N	0	16	3	2019-08-23 12:43:22.09228	2019-08-28 07:30:59.944063	f	\N	1	\N	\N	\N	\N	7	3	0	\N	\N	2	f	\N
21	RNA isolation	2019-10-02 07:30:58.728704	\N	64	0	3	2019-08-24 04:22:51.070785	2019-08-28 07:30:59.424655	f	\N	1	\N	\N	\N	\N	2	3	0	\N	\N	2	f	\N
239	Test Product A, Alumina 2mm	\N	\N	0	0	\N	2021-07-12 18:41:37.048443	2021-07-12 18:41:37.048443	f	\N	27	27	\N	\N	\N	-1	53	0	\N	\N	2	f	\N
126	RNA isolation	2020-12-23 14:53:24.817884	\N	64	0	15	2020-11-16 18:37:02.981229	2020-11-18 14:53:24.836312	f	\N	17	\N	\N	\N	\N	2	34	0	\N	\N	2	f	\N
127	RNA quality & quantity - BIOANALYSER	2021-01-06 14:53:24.837055	\N	96	0	15	2020-11-15 20:49:00.128206	2020-11-18 14:53:24.854249	f	\N	17	\N	\N	\N	\N	3	34	0	\N	\N	2	f	\N
240	Test Product B, Alumina 2mm	\N	\N	32	0	\N	2021-07-12 18:44:37.266273	2021-07-12 18:44:37.266273	f	\N	27	27	\N	\N	\N	-1	53	0	\N	\N	2	f	\N
28	Experiment design	2019-09-04 08:34:53.615587	\N	0	0	4	2019-08-23 00:06:50.1689	2019-08-28 08:34:53.662125	f	\N	6	\N	\N	\N	\N	0	4	0	\N	\N	2	f	\N
29	Sampling biological material	2019-09-18 08:34:53.66299	\N	32	0	4	2019-08-24 22:55:58.779835	2019-08-28 08:34:54.588229	f	\N	6	\N	\N	\N	\N	1	4	0	\N	\N	2	f	\N
30	RNA isolation	2019-10-02 08:34:53.689311	\N	64	0	4	2019-08-25 11:53:38.582426	2019-08-28 08:34:53.711633	f	\N	6	\N	\N	\N	\N	2	4	0	\N	\N	2	f	\N
31	RNA quality & quantity - BIOANALYSER	2019-10-16 08:34:53.712296	\N	96	0	4	2019-08-27 05:49:23.20829	2019-08-28 08:34:53.791181	f	\N	6	\N	\N	\N	\N	3	4	0	\N	\N	2	f	\N
32	Reverse transcription	2019-10-30 08:34:53.791972	\N	96	16	4	2019-08-25 14:38:46.18078	2019-08-28 08:34:53.831403	f	\N	6	\N	\N	\N	\N	4	4	0	\N	\N	2	f	\N
33	qPCR	2019-11-13 08:34:53.991199	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	4	2019-08-23 16:21:33.753782	2019-08-28 08:34:54.154127	f	\N	6	\N	\N	\N	\N	5	4	0	\N	\N	2	f	\N
34	Data quality control	2019-11-27 08:34:54.154901	\N	32	16	4	2019-08-26 04:34:41.892289	2019-08-28 08:34:54.279472	f	\N	6	\N	\N	\N	\N	6	4	0	\N	\N	2	f	\N
35	Data analysis - ddCq	2019-12-11 08:34:54.2803	\N	0	16	4	2019-08-22 22:37:29.757731	2019-08-28 08:34:54.344551	f	\N	6	\N	\N	\N	\N	7	4	0	\N	\N	2	f	\N
36	Data analysis - Pfaffl method	2019-09-04 08:34:54.345258	\N	-1	-1	\N	2019-08-26 01:26:57.931069	2019-08-28 08:34:54.409148	t	2019-08-28 08:34:54	6	\N	6	\N	\N	-1	4	0	\N	\N	2	f	\N
124	Experiment design	2020-11-25 14:53:24.738215	\N	0	0	15	2020-11-13 08:54:37.866581	2020-11-18 14:53:24.791217	f	\N	17	\N	\N	\N	\N	0	34	0	\N	\N	2	f	\N
44	Data analysis - ddCq	2019-12-12 12:51:23.820621	\N	0	16	5	2019-08-28 08:43:36.939563	2019-08-30 16:10:44.495168	f	\N	7	\N	\N	\N	\N	7	5	0	\N	\N	2	f	\N
125	Sampling biological material	2020-12-09 14:53:24.791967	\N	32	0	15	2020-11-18 02:10:45.594427	2020-11-18 14:53:25.010405	f	\N	17	\N	\N	\N	\N	1	34	0	\N	\N	2	f	\N
128	Reverse transcription	2021-01-20 14:53:24.854921	\N	96	16	15	2020-11-15 19:26:24.462197	2020-11-18 14:53:24.870503	f	\N	17	\N	\N	\N	\N	4	34	0	\N	\N	2	f	\N
175	RNA isolation	2021-06-24 14:13:33.998398	\N	64	0	23	2021-05-15 05:34:31.583327	2021-05-20 14:13:34.008395	f	\N	23	\N	\N	\N	\N	2	41	0	\N	\N	2	f	\N
176	RNA quality & quantity - BIOANALYSER	2021-07-08 14:13:34.008775	\N	96	0	23	2021-05-18 13:37:18.916956	2021-05-20 14:13:34.0178	f	\N	23	\N	\N	\N	\N	3	41	0	\N	\N	2	f	\N
274	Task 1	\N	\N	0	0	\N	2021-08-25 16:08:53.513359	2021-08-25 16:08:53.513359	f	\N	30	30	\N	\N	\N	-1	57	0	\N	\N	2	f	\N
129	qPCR	2021-02-03 14:53:24.871239	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	15	2020-11-17 10:39:00.166697	2020-11-18 14:53:24.891006	f	\N	17	\N	\N	\N	\N	5	34	0	\N	\N	2	f	\N
130	Data quality control	2021-02-17 14:53:24.891643	\N	32	16	15	2020-11-14 15:38:48.575066	2020-11-18 14:53:24.905949	f	\N	17	\N	\N	\N	\N	6	34	0	\N	\N	2	f	\N
131	Data analysis - ddCq	2021-03-03 14:53:24.906645	\N	0	16	15	2020-11-13 00:50:27.181451	2020-11-18 14:53:24.921556	f	\N	17	\N	\N	\N	\N	7	34	0	\N	\N	2	f	\N
132	Data analysis - Pfaffl method	2020-11-25 14:53:24.922203	\N	-1	-1	\N	2020-11-15 17:17:06.933888	2020-11-18 14:53:24.933759	t	2020-11-18 14:53:24	17	\N	17	\N	\N	-1	34	0	\N	\N	2	f	\N
241	Experiment design	2021-08-30 14:58:27.593382	\N	0	0	29	2021-08-21 07:49:17.881927	2021-08-23 14:58:27.697209	f	\N	30	\N	\N	\N	\N	0	54	0	\N	\N	2	f	\N
242	Sampling biological material	2021-09-13 14:58:27.697936	\N	32	0	29	2021-08-19 13:25:46.353817	2021-08-23 14:58:27.956096	f	\N	30	\N	\N	\N	\N	1	54	0	\N	\N	2	f	\N
243	RNA isolation	2021-09-27 14:58:27.724427	\N	64	0	29	2021-08-21 13:04:46.404714	2021-08-23 14:58:27.742327	f	\N	30	\N	\N	\N	\N	2	54	0	\N	\N	2	f	\N
244	RNA quality & quantity - BIOANALYSER	2021-10-11 14:58:27.743576	\N	96	0	29	2021-08-19 02:59:10.381141	2021-08-23 14:58:27.760352	f	\N	30	\N	\N	\N	\N	3	54	0	\N	\N	2	f	\N
184	RNA isolation	2021-06-24 14:21:20.563027	\N	64	0	24	2021-05-14 14:21:41.189836	2021-05-20 14:21:20.575224	f	\N	24	\N	\N	\N	\N	2	42	0	\N	\N	2	f	\N
185	RNA quality & quantity - BIOANALYSER	2021-07-08 14:21:20.575673	\N	96	0	24	2021-05-15 18:28:36.664797	2021-05-20 14:21:20.587455	f	\N	24	\N	\N	\N	\N	3	42	0	\N	\N	2	f	\N
186	Reverse transcription	2021-07-22 14:21:20.587958	\N	96	16	24	2021-05-19 16:32:26.250222	2021-05-20 14:21:20.598701	f	\N	24	\N	\N	\N	\N	4	42	0	\N	\N	2	f	\N
37	Experiment design	2019-09-05 12:51:23.194758	\N	0	0	5	2019-08-26 16:50:38.218852	2019-08-29 12:51:23.491471	f	\N	7	\N	\N	\N	\N	0	5	0	\N	\N	2	f	\N
45	Data analysis - Pfaffl method	2019-09-05 12:51:23.884042	\N	-1	-1	\N	2019-08-27 08:39:02.648374	2019-08-29 12:51:24.176689	t	2019-08-29 12:51:23	7	\N	7	\N	\N	-1	5	0	\N	\N	2	f	\N
187	qPCR	2021-08-05 14:21:20.599225	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	24	2021-05-15 15:51:55.029291	2021-05-20 14:21:20.611116	f	\N	24	\N	\N	\N	\N	5	42	0	\N	\N	2	f	\N
38	Sampling biological material	2019-09-19 12:51:23.495402	\N	32	0	5	2019-08-27 02:18:30.211462	2019-08-30 16:10:44.467373	f	\N	7	\N	\N	\N	\N	1	5	0	\N	\N	2	f	\N
39	RNA isolation	2019-10-03 12:51:23.553088	\N	64	0	5	2019-08-28 21:52:07.23236	2019-08-30 16:10:44.472114	f	\N	7	\N	\N	\N	\N	2	5	0	\N	\N	2	f	\N
40	RNA quality & quantity - BIOANALYSER	2019-10-17 12:51:23.586676	\N	96	0	5	2019-08-23 23:10:48.825615	2019-08-30 16:10:44.476731	f	\N	7	\N	\N	\N	\N	3	5	0	\N	\N	2	f	\N
41	Reverse transcription	2019-10-31 12:51:23.696572	\N	96	16	5	2019-08-26 12:00:02.620322	2019-08-30 16:10:44.481344	f	\N	7	\N	\N	\N	\N	4	5	0	\N	\N	2	f	\N
42	qPCR	2019-11-14 12:51:23.736654	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	5	2019-08-24 04:26:59.725494	2019-08-30 16:10:44.485939	f	\N	7	\N	\N	\N	\N	5	5	0	\N	\N	2	f	\N
43	Data quality control	2019-11-28 12:51:23.79225	\N	32	16	5	2019-08-25 14:41:26.270338	2019-08-30 16:10:44.490582	f	\N	7	\N	\N	\N	\N	6	5	0	\N	\N	2	f	\N
136	2022 January - Quarterly Timing Check	2022-01-31 00:00:00	\N	0	0	\N	2021-04-08 13:20:10.130433	2021-04-08 14:20:25.342844	t	2021-04-08 14:20:25	4	4	4	\N	\N	-1	24	0	\N	\N	2	f	\N
135	2021 October - Quarterly Timing Check	2021-10-29 00:00:00	\N	0	0	\N	2021-04-08 13:20:10.123722	2021-04-08 14:20:25.334164	t	2021-04-08 14:20:25	4	4	4	\N	\N	-1	24	0	\N	\N	2	f	\N
134	2021 July - Quarterly Timing Check	2021-07-30 00:00:00	\N	0	0	\N	2021-04-08 13:20:10.116553	2021-04-08 14:20:25.313481	t	2021-04-08 14:20:25	4	4	4	\N	\N	-1	24	0	\N	\N	2	f	\N
155	Experiment design	2021-05-27 14:12:30.446295	\N	0	0	21	2021-05-20 00:52:44.393235	2021-05-20 14:12:30.462206	f	\N	21	\N	\N	\N	\N	0	39	0	\N	\N	2	f	\N
156	Sampling biological material	2021-06-10 14:12:30.462718	\N	32	0	21	2021-05-19 08:01:10.937381	2021-05-20 14:12:30.588792	f	\N	21	\N	\N	\N	\N	1	39	0	\N	\N	2	f	\N
157	RNA isolation	2021-06-24 14:12:30.474989	\N	64	0	21	2021-05-19 12:46:34.281143	2021-05-20 14:12:30.48502	f	\N	21	\N	\N	\N	\N	2	39	0	\N	\N	2	f	\N
158	RNA quality & quantity - BIOANALYSER	2021-07-08 14:12:30.48551	\N	96	0	21	2021-05-15 01:37:58.052195	2021-05-20 14:12:30.495197	f	\N	21	\N	\N	\N	\N	3	39	0	\N	\N	2	f	\N
159	Reverse transcription	2021-07-22 14:12:30.49566	\N	96	16	21	2021-05-14 22:07:04.400627	2021-05-20 14:12:30.506867	f	\N	21	\N	\N	\N	\N	4	39	0	\N	\N	2	f	\N
160	qPCR	2021-08-05 14:12:30.507275	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	21	2021-05-15 01:20:36.830349	2021-05-20 14:12:30.518012	f	\N	21	\N	\N	\N	\N	5	39	0	\N	\N	2	f	\N
161	Data quality control	2021-08-19 14:12:30.518488	\N	32	16	21	2021-05-16 04:22:06.926693	2021-05-20 14:12:30.528204	f	\N	21	\N	\N	\N	\N	6	39	0	\N	\N	2	f	\N
162	Data analysis - ddCq	2021-09-02 14:12:30.528666	\N	0	16	21	2021-05-16 11:32:05.167535	2021-05-20 14:12:30.537737	f	\N	21	\N	\N	\N	\N	7	39	0	\N	\N	2	f	\N
163	Data analysis - Pfaffl method	2021-05-27 14:12:30.53817	\N	-1	-1	\N	2021-05-18 05:48:35.743579	2021-05-20 14:12:30.545957	t	2021-05-20 14:12:30	21	\N	21	\N	\N	-1	39	0	\N	\N	2	f	\N
173	Experiment design	2021-05-27 14:13:33.979753	\N	0	0	23	2021-05-17 08:13:31.930229	2021-05-20 14:13:33.988137	f	\N	23	\N	\N	\N	\N	0	41	0	\N	\N	2	f	\N
177	Reverse transcription	2021-07-22 14:13:34.018196	\N	96	16	23	2021-05-18 09:16:36.410367	2021-05-20 14:13:34.029748	f	\N	23	\N	\N	\N	\N	4	41	0	\N	\N	2	f	\N
178	qPCR	2021-08-05 14:13:34.030106	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	23	2021-05-15 22:51:21.576122	2021-05-20 14:13:34.042161	f	\N	23	\N	\N	\N	\N	5	41	0	\N	\N	2	f	\N
179	Data quality control	2021-08-19 14:13:34.04264	\N	32	16	23	2021-05-19 07:28:19.351926	2021-05-20 14:13:34.05207	f	\N	23	\N	\N	\N	\N	6	41	0	\N	\N	2	f	\N
180	Data analysis - ddCq	2021-09-02 14:13:34.052484	\N	0	16	23	2021-05-19 03:21:27.190915	2021-05-20 14:13:34.061469	f	\N	23	\N	\N	\N	\N	7	41	0	\N	\N	2	f	\N
181	Data analysis - Pfaffl method	2021-05-27 14:13:34.061855	\N	-1	-1	\N	2021-05-14 16:24:12.794033	2021-05-20 14:13:34.069064	t	2021-05-20 14:13:34	23	\N	23	\N	\N	-1	41	0	\N	\N	2	f	\N
188	Data quality control	2021-08-19 14:21:20.611581	\N	32	16	24	2021-05-18 20:12:47.377695	2021-05-20 14:21:20.62392	f	\N	24	\N	\N	\N	\N	6	42	0	\N	\N	2	f	\N
189	Data analysis - ddCq	2021-09-02 14:21:20.624452	\N	0	16	24	2021-05-19 00:52:34.445802	2021-05-20 14:21:20.633926	f	\N	24	\N	\N	\N	\N	7	42	0	\N	\N	2	f	\N
245	Reverse transcription	2021-10-25 14:58:27.761701	\N	96	16	29	2021-08-21 05:11:02.273406	2021-08-23 14:58:27.778453	f	\N	30	\N	\N	\N	\N	4	54	0	\N	\N	2	f	\N
246	qPCR	2021-11-08 14:58:27.779046	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	29	2021-08-20 08:04:20.30144	2021-08-23 14:58:27.795931	f	\N	30	\N	\N	\N	\N	5	54	0	\N	\N	2	f	\N
247	Data quality control	2021-11-22 14:58:27.796326	\N	32	16	29	2021-08-20 21:53:47.829728	2021-08-23 14:58:27.806845	f	\N	30	\N	\N	\N	\N	6	54	0	\N	\N	2	f	\N
46	Experiment design	2019-09-19 18:26:18.415455	\N	0	0	6	2019-09-11 07:10:03.527334	2019-09-12 18:26:18.453321	f	\N	8	\N	\N	\N	\N	0	7	0	\N	\N	2	f	\N
47	Sampling biological material	2019-10-03 18:26:18.454055	\N	32	0	6	2019-09-10 19:50:42.532583	2019-09-12 18:26:18.635917	f	\N	8	\N	\N	\N	\N	1	7	0	\N	\N	2	f	\N
48	RNA isolation	2019-10-17 18:26:18.467395	\N	64	0	6	2019-09-07 10:21:08.668824	2019-09-12 18:26:18.479109	f	\N	8	\N	\N	\N	\N	2	7	0	\N	\N	2	f	\N
49	RNA quality & quantity - BIOANALYSER	2019-10-31 18:26:18.479742	\N	96	0	6	2019-09-10 23:35:56.588842	2019-09-12 18:26:18.490741	f	\N	8	\N	\N	\N	\N	3	7	0	\N	\N	2	f	\N
50	Reverse transcription	2019-11-14 18:26:18.49136	\N	96	16	6	2019-09-11 06:52:13.959539	2019-09-12 18:26:18.501926	f	\N	8	\N	\N	\N	\N	4	7	0	\N	\N	2	f	\N
51	qPCR	2019-11-28 18:26:18.502527	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	6	2019-09-06 21:58:11.074175	2019-09-12 18:26:18.513405	f	\N	8	\N	\N	\N	\N	5	7	0	\N	\N	2	f	\N
52	Data quality control	2019-12-12 18:26:18.514048	\N	32	16	6	2019-09-12 10:50:53.298827	2019-09-12 18:26:18.525367	f	\N	8	\N	\N	\N	\N	6	7	0	\N	\N	2	f	\N
53	Data analysis - ddCq	2019-12-26 18:26:18.525967	\N	0	16	6	2019-09-07 20:19:03.446391	2019-09-12 18:26:18.536297	f	\N	8	\N	\N	\N	\N	7	7	0	\N	\N	2	f	\N
54	Data analysis - Pfaffl method	2019-09-19 18:26:18.536915	\N	-1	-1	\N	2019-09-11 09:59:13.978858	2019-09-12 18:26:18.548079	t	2019-09-12 18:26:18	8	\N	8	\N	\N	-1	7	0	\N	\N	2	f	\N
248	Data analysis - ddCq	2021-12-06 14:58:27.807489	\N	0	16	29	2021-08-21 12:50:40.868649	2021-08-23 14:58:27.823488	f	\N	30	\N	\N	\N	\N	7	54	0	\N	\N	2	f	\N
249	Data analysis - Pfaffl method	2021-08-30 14:58:27.824541	\N	-1	-1	\N	2021-08-20 03:18:29.476359	2021-08-23 14:58:27.834015	t	2021-08-23 14:58:27	30	\N	30	\N	\N	-1	54	0	\N	\N	2	f	\N
137	Experiment design	2021-05-26 20:01:51.684962	\N	0	0	16	2021-05-17 01:16:57.080943	2021-05-19 20:01:51.753474	f	\N	20	\N	\N	\N	\N	0	38	0	\N	\N	2	f	\N
138	Sampling biological material	2021-06-09 20:01:51.754095	\N	32	0	16	2021-05-16 01:42:06.612784	2021-05-19 20:01:51.98315	f	\N	20	\N	\N	\N	\N	1	38	0	\N	\N	2	f	\N
139	RNA isolation	2021-06-23 20:01:51.774688	\N	64	0	16	2021-05-19 06:16:36.809033	2021-05-19 20:01:51.788988	f	\N	20	\N	\N	\N	\N	2	38	0	\N	\N	2	f	\N
140	RNA quality & quantity - BIOANALYSER	2021-07-07 20:01:51.789425	\N	96	0	16	2021-05-19 14:48:03.040742	2021-05-19 20:01:51.805592	f	\N	20	\N	\N	\N	\N	3	38	0	\N	\N	2	f	\N
141	Reverse transcription	2021-07-21 20:01:51.80612	\N	96	16	16	2021-05-16 18:58:02.574886	2021-05-19 20:01:51.821257	f	\N	20	\N	\N	\N	\N	4	38	0	\N	\N	2	f	\N
142	qPCR	2021-08-04 20:01:51.821703	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	16	2021-05-16 08:29:47.236342	2021-05-19 20:01:51.847057	f	\N	20	\N	\N	\N	\N	5	38	0	\N	\N	2	f	\N
143	Data quality control	2021-08-18 20:01:51.847535	\N	32	16	16	2021-05-17 07:01:41.140363	2021-05-19 20:01:51.864059	f	\N	20	\N	\N	\N	\N	6	38	0	\N	\N	2	f	\N
144	Data analysis - ddCq	2021-09-01 20:01:51.864498	\N	0	16	16	2021-05-19 04:35:17.542153	2021-05-19 20:01:51.879249	f	\N	20	\N	\N	\N	\N	7	38	0	\N	\N	2	f	\N
145	Data analysis - Pfaffl method	2021-05-26 20:01:51.880404	\N	-1	-1	\N	2021-05-18 08:01:11.577258	2021-05-19 20:01:51.893501	t	2021-05-19 20:01:51	20	\N	20	\N	\N	-1	38	0	\N	\N	2	f	\N
268	Shake for 2 minutes	\N	\N	305	26	31	2021-08-23 19:46:47.829235	2021-08-23 19:46:48.235743	f	\N	30	30	\N	\N	\N	12	56	0	\N	\N	2	f	\N
269	Add 450 uL of EtOAc extraction layer into mini Uniprep syringeless filters	\N	\N	341	26	31	2021-08-23 19:46:47.833368	2021-08-23 19:46:48.237853	f	\N	30	30	\N	\N	\N	13	56	0	\N	\N	2	f	\N
67	RNA quality & quantity - BIOANALYSER	2020-01-13 14:09:22.853158	\N	96	0	8	2019-11-24 03:57:41.565533	2019-11-25 14:09:23.667769	f	\N	4	\N	\N	\N	\N	3	9	0	\N	\N	2	f	\N
165	Sampling biological material	2021-06-10 14:12:57.159748	\N	32	0	22	2021-05-17 17:57:23.491153	2021-05-20 14:12:57.276684	f	\N	22	\N	\N	\N	\N	1	40	0	\N	\N	2	f	\N
166	RNA isolation	2021-06-24 14:12:57.171379	\N	64	0	22	2021-05-19 14:42:30.699031	2021-05-20 14:12:57.181389	f	\N	22	\N	\N	\N	\N	2	40	0	\N	\N	2	f	\N
167	RNA quality & quantity - BIOANALYSER	2021-07-08 14:12:57.18181	\N	96	0	22	2021-05-15 16:27:24.539533	2021-05-20 14:12:57.192032	f	\N	22	\N	\N	\N	\N	3	40	0	\N	\N	2	f	\N
168	Reverse transcription	2021-07-22 14:12:57.19249	\N	96	16	22	2021-05-20 10:51:12.074036	2021-05-20 14:12:57.203016	f	\N	22	\N	\N	\N	\N	4	40	0	\N	\N	2	f	\N
152	Quantitative analysis of Micro-plastics will be performed using Fourier transformation infrared spectroscopy/microscopy (FTIR/μFTIR).	\N	\N	116	48	19	2021-05-20 10:31:30.181004	2021-05-20 10:35:58.800239	f	\N	1	1	\N	\N	\N	4	31	0	\N	\N	2	f	\N
77	Data Results at 24 hours and 48 hours	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>The Sporetrol contains a thermo-resistant bacteria that at 121&deg;C for 15 minutes under pressure will kill the bacteria. There is a color indicator in the ampule that will change from Purple to Yellow in the presence of enumerated bacteria. The sporetrol amplues are purple and After 15 minute sterilization run, the sporetrol should remain purple. After 24 hours and 48 hours they autoclaved sporetrol should remain purple. The control (non-autoclaved) sporetrol should turn Yellow after 24 and 48 hours. The autoclaved sporetrol if purple at 24 and 48 hours indicated that the decontamination/sterilization was reached.</div>\r\n<div>&nbsp;</div>\r\n<div><strong>Recorded Data:</strong></div>\r\n<ol>\r\n<li>Date Autoclaved</li>\r\n<li>Initial Observed Temperature (55&deg;C-65&deg;C)</li>\r\n<li><span class="">24 hour Observed Temperature</span>\r\n<ol>\r\n<li><span class="">24 hour Observed Color (Yellow=Positive, Purple=Negative)</span></li>\r\n</ol>\r\n</li>\r\n<li><span class="">48 hour Observed Temperature</span>\r\n<ol>\r\n<li><span class="">48 hour Observed Color (Yellow=Positive, Purple=Negative)</span></li>\r\n</ol>\r\n</li>\r\n<li><span class="">Initials</span></li>\r\n</ol>	108	40	9	2019-11-25 17:01:05.112862	2019-11-25 17:32:40.193541	f	\N	4	4	\N	\N	\N	3	23	0	\N	\N	2	f	\N
55	Experiment design	2019-11-20 13:57:41.531174	\N	0	0	7	2019-11-13 04:13:20.413714	2019-11-13 13:57:41.654499	f	\N	9	\N	\N	\N	\N	0	8	0	\N	\N	2	f	\N
56	Sampling biological material	2019-12-04 13:57:41.655329	\N	32	0	7	2019-11-08 08:39:47.992473	2019-11-13 13:57:41.95705	f	\N	9	\N	\N	\N	\N	1	8	0	\N	\N	2	f	\N
57	RNA isolation	2019-12-18 13:57:41.690332	\N	64	0	7	2019-11-10 00:09:04.223309	2019-11-13 13:57:41.708878	f	\N	9	\N	\N	\N	\N	2	8	0	\N	\N	2	f	\N
58	RNA quality & quantity - BIOANALYSER	2020-01-01 13:57:41.709555	\N	96	0	7	2019-11-10 06:06:25.276837	2019-11-13 13:57:41.727588	f	\N	9	\N	\N	\N	\N	3	8	0	\N	\N	2	f	\N
59	Reverse transcription	2020-01-15 13:57:41.728293	\N	96	16	7	2019-11-11 14:58:51.153178	2019-11-13 13:57:41.74825	f	\N	9	\N	\N	\N	\N	4	8	0	\N	\N	2	f	\N
81	RNA quality & quantity - BIOANALYSER	2020-01-13 18:00:52.522548	\N	96	0	10	2019-11-24 13:07:25.943529	2019-11-25 18:00:52.539167	f	\N	10	\N	\N	\N	\N	3	26	0	\N	\N	2	f	\N
82	Reverse transcription	2020-01-27 18:00:52.539918	\N	96	16	10	2019-11-21 10:55:31.936947	2019-11-25 18:00:52.557453	f	\N	10	\N	\N	\N	\N	4	26	0	\N	\N	2	f	\N
84	Data quality control	2020-02-24 18:00:52.576744	\N	32	16	10	2019-11-24 06:44:36.981279	2019-11-25 18:00:52.59147	f	\N	10	\N	\N	\N	\N	6	26	0	\N	\N	2	f	\N
60	qPCR	2020-01-29 13:57:41.748899	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	7	2019-11-12 09:20:52.815439	2019-11-13 13:57:41.779019	f	\N	9	\N	\N	\N	\N	5	8	0	\N	\N	2	f	\N
61	Data quality control	2020-02-12 13:57:41.779704	\N	32	16	7	2019-11-12 06:42:17.44399	2019-11-13 13:57:41.796831	f	\N	9	\N	\N	\N	\N	6	8	0	\N	\N	2	f	\N
62	Data analysis - ddCq	2020-02-26 13:57:41.798031	\N	0	16	7	2019-11-13 09:51:02.902716	2019-11-13 13:57:41.8233	f	\N	9	\N	\N	\N	\N	7	8	0	\N	\N	2	f	\N
63	Data analysis - Pfaffl method	2019-11-20 13:57:41.824025	\N	-1	-1	\N	2019-11-09 10:12:53.678114	2019-11-13 13:57:41.84083	t	2019-11-13 13:57:41	9	\N	9	\N	\N	-1	8	0	\N	\N	2	f	\N
69	qPCR	2020-02-10 14:09:22.924061	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	8	2019-11-24 06:12:46.797912	2019-11-25 14:09:23.865592	f	\N	4	\N	\N	\N	\N	5	9	0	\N	\N	2	f	\N
70	Data quality control	2020-02-24 14:09:23.095018	\N	32	16	8	2019-11-25 05:41:44.186765	2019-11-25 14:09:23.961322	f	\N	4	\N	\N	\N	\N	6	9	0	\N	\N	2	f	\N
74	Prep Autoclave	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>Prior to use: Flush and Drain the Autoclave</strong></div>\r\n<div>&nbsp;</div>\r\n<ol>\r\n<li>Touch Screen to start process</li>\r\n<li>&nbsp;Enter Username : STERIS</li>\r\n<li>&nbsp;Enter Password: 1000</li>\r\n<li>&nbsp;Open front door and open the drain by turning the knob counter clockwise</li>\r\n<li>&nbsp;Close front door</li>\r\n<li>&nbsp;Touch Screen : Start Timer (5 minutes)</li>\r\n<li>&nbsp;Wait for timer to end</li>\r\n<li>&nbsp;Open front door and close drain by turning knob clock wise</li>\r\n<li>&nbsp;Close front door</li>\r\n<li>&nbsp;Ready for use</li>\r\n</ol>	0	0	9	2019-11-25 17:01:05.042107	2021-04-08 15:46:11.264308	f	\N	4	4	\N	\N	\N	0	23	0	\N	\N	2	f	\N
75	Run 15 minute sterilization cycle with Sportrol Ampule	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>Run Media Cycle (15 minutes Sterilization)</div>\r\n<ol>\r\n<li>Slide down the chamber door slowly</li>\r\n<li>Place Sporetrol Ampule in the autoclave</li>\r\n<li>Place the Datalogger in the autoclave (in a beaker of water)</li>\r\n<li>Slide up the chamber door slowly</li>\r\n<li>Touch Screen: Media</li>\r\n<li>Autoclave will start the run</li>\r\n</ol>	21	22	9	2019-11-25 17:01:05.085943	2021-04-08 15:46:42.621471	f	\N	4	4	\N	\N	\N	1	23	0	\N	\N	2	f	\N
72	Data analysis - Pfaffl method	2019-12-02 14:09:23.174896	\N	-1	-1	\N	2019-11-24 12:28:47.883396	2019-11-25 14:09:23.210418	t	2019-11-25 14:09:23	4	\N	4	\N	\N	-1	9	0	\N	\N	2	f	\N
65	Sampling biological material	2019-12-16 14:09:22.744813	\N	32	0	8	2019-11-23 00:31:08.749788	2019-11-25 14:09:23.424581	f	\N	4	\N	\N	\N	\N	1	9	0	\N	\N	2	f	\N
68	Reverse transcription	2020-01-27 14:09:22.889881	\N	96	16	8	2019-11-21 05:21:22.652702	2019-11-25 14:09:23.766545	f	\N	4	\N	\N	\N	\N	4	9	0	\N	\N	2	f	\N
190	Data analysis - Pfaffl method	2021-05-27 14:21:20.634325	\N	0	30	\N	2021-05-17 23:13:44.136979	2021-05-20 14:46:18.787123	f	2021-05-20 14:21:20	24	24	24	24	2021-05-20 14:46:18	-1	42	0	\N	\N	2	f	\N
71	Data analysis - ddCq	2020-03-09 14:09:23.135375	\N	0	16	8	2019-11-21 10:41:03.920793	2019-11-25 14:09:25.035686	f	\N	4	\N	\N	\N	\N	7	9	0	\N	\N	2	f	\N
76	Incubate working ampule and the control ampule at 60°C for 48 hours	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>After sterilization run</div>\r\n<ol>\r\n<li>Open chamber door slowly by sliding down</li>\r\n<li>Using gloves take out sporetrol</li>\r\n<li>Take out datalogger</li>\r\n<li>Place the autoclave sporetrol and a non-autoclaved sporetrol in the small HACH incubator (Set to 60&deg;C)</li>\r\n</ol>	69	22	9	2019-11-25 17:01:05.101269	2019-11-25 18:05:09.389781	f	\N	4	4	\N	\N	\N	2	23	0	\N	\N	2	f	\N
66	RNA isolation	2019-12-30 14:09:22.804904	\N	64	0	8	2019-11-19 19:04:34.436626	2019-11-25 14:09:23.557483	f	\N	4	\N	\N	\N	\N	2	9	0	\N	\N	2	f	\N
73	HPC	\N	\N	0	0	\N	2019-11-25 14:12:02.969975	2019-11-25 14:12:02.969975	f	\N	4	4	\N	\N	\N	-1	10	0	\N	\N	2	f	\N
147	Sample Preparation	\N	\N	38	15	17	2021-05-20 10:26:02.276252	2021-05-20 10:26:02.416702	f	\N	1	1	\N	\N	\N	1	37	0	\N	\N	2	f	\N
146	Screening Microplastics	\N	\N	0	0	17	2021-05-20 10:26:02.263514	2021-05-20 10:26:02.41881	f	\N	1	1	\N	\N	\N	0	37	0	\N	\N	2	f	\N
164	Experiment design	2021-05-27 14:12:57.147692	\N	0	0	22	2021-05-16 15:41:47.055784	2021-05-20 14:12:57.159361	f	\N	22	\N	\N	\N	\N	0	40	0	\N	\N	2	f	\N
151	moisture content of the bulk waste material will be determined by drying in an oven at 105 C.	\N	\N	83	37	19	2021-05-20 10:31:30.174122	2021-05-20 10:35:58.804344	f	\N	1	1	\N	\N	\N	3	31	0	\N	\N	2	f	\N
150	100g of material	\N	\N	51	27	19	2021-05-20 10:31:30.169599	2021-05-20 10:35:58.80645	f	\N	1	1	\N	\N	\N	2	31	0	\N	\N	2	f	\N
149	Food waste from the sources identified	\N	\N	18	16	19	2021-05-20 10:31:30.165229	2021-05-20 10:35:58.8082	f	\N	1	1	\N	\N	\N	1	31	0	\N	\N	2	f	\N
148	Food depackaging facilities sampled	\N	\N	0	0	19	2021-05-20 10:31:30.15722	2021-05-20 10:35:58.809909	f	\N	1	1	\N	\N	\N	0	31	0	\N	\N	2	f	\N
169	qPCR	2021-08-05 14:12:57.203442	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	22	2021-05-16 23:22:31.865343	2021-05-20 14:12:57.213403	f	\N	22	\N	\N	\N	\N	5	40	0	\N	\N	2	f	\N
170	Data quality control	2021-08-19 14:12:57.213772	\N	32	16	22	2021-05-18 22:30:33.776541	2021-05-20 14:12:57.223234	f	\N	22	\N	\N	\N	\N	6	40	0	\N	\N	2	f	\N
78	Experiment design	2019-12-02 18:00:52.456938	\N	0	0	10	2019-11-22 05:02:16.987328	2019-11-25 18:00:52.484249	f	\N	10	\N	\N	\N	\N	0	26	0	\N	\N	2	f	\N
79	Sampling biological material	2019-12-16 18:00:52.484918	\N	32	0	10	2019-11-20 06:10:58.191516	2019-11-25 18:00:52.693183	f	\N	10	\N	\N	\N	\N	1	26	0	\N	\N	2	f	\N
80	RNA isolation	2019-12-30 18:00:52.504531	\N	64	0	10	2019-11-24 11:51:27.918476	2019-11-25 18:00:52.521882	f	\N	10	\N	\N	\N	\N	2	26	0	\N	\N	2	f	\N
85	Data analysis - ddCq	2020-03-09 18:00:52.592078	\N	0	16	10	2019-11-19 21:13:53.745409	2019-11-25 18:00:52.609428	f	\N	10	\N	\N	\N	\N	7	26	0	\N	\N	2	f	\N
86	Data analysis - Pfaffl method	2019-12-02 18:00:52.610027	\N	-1	-1	\N	2019-11-20 10:23:03.201418	2019-11-25 18:00:52.623335	t	2019-11-25 18:00:52	10	\N	10	\N	\N	-1	26	0	\N	\N	2	f	\N
171	Data analysis - ddCq	2021-09-02 14:12:57.223881	\N	0	16	22	2021-05-18 07:05:15.534632	2021-05-20 14:12:57.235069	f	\N	22	\N	\N	\N	\N	7	40	0	\N	\N	2	f	\N
172	Data analysis - Pfaffl method	2021-05-27 14:12:57.235614	\N	-1	-1	\N	2021-05-20 08:59:07.325625	2021-05-20 14:12:57.243082	t	2021-05-20 14:12:57	22	\N	22	\N	\N	-1	40	0	\N	\N	2	f	\N
182	Experiment design	2021-05-27 14:21:20.537708	\N	0	0	24	2021-05-20 01:37:01.582993	2021-05-20 14:21:20.548505	f	\N	24	\N	\N	\N	\N	0	42	0	\N	\N	2	f	\N
183	Sampling biological material	2021-06-10 14:21:20.548894	\N	32	0	24	2021-05-16 18:04:56.878002	2021-05-20 14:21:20.674422	f	\N	24	\N	\N	\N	\N	1	42	0	\N	\N	2	f	\N
83	qPCR	2020-02-10 18:00:52.55812	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	10	2019-11-20 03:18:20.15141	2019-11-25 18:00:52.575045	f	\N	10	\N	\N	\N	\N	5	26	0	\N	\N	2	f	\N
270	Add 1 gram of NaCl to each PP tube	\N	\N	203	26	31	2021-08-23 19:46:47.83828	2021-08-23 19:46:48.239675	f	\N	30	30	\N	\N	\N	9	56	0	\N	\N	2	f	\N
271	Addition of 100 microliters of native precursor spike solution	\N	\N	123	24	31	2021-08-23 19:46:47.842354	2021-08-23 19:46:48.24147	f	\N	30	30	\N	\N	\N	7	56	0	\N	\N	2	f	\N
272	Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)	\N	\N	168	0	31	2021-08-23 19:46:47.846514	2021-08-23 19:46:48.243218	f	\N	30	30	\N	\N	\N	3	56	0	\N	\N	2	f	\N
273	Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)	\N	\N	168	53	31	2021-08-23 19:46:47.850778	2021-08-23 19:46:48.245037	f	\N	30	30	\N	\N	\N	1	56	0	\N	\N	2	f	\N
259	QC Preparation	\N	\N	34	26	31	2021-08-23 19:46:47.769882	2021-08-23 19:46:48.246712	f	\N	30	30	\N	\N	\N	5	56	0	\N	\N	2	f	\N
260	10mL of water into each 15mL Falcon Tube	\N	\N	0	25	31	2021-08-23 19:46:47.79362	2021-08-23 19:46:48.248359	f	\N	30	30	\N	\N	\N	4	56	0	\N	\N	2	f	\N
261	Method Blank Preparation	\N	\N	79	0	31	2021-08-23 19:46:47.798314	2021-08-23 19:46:48.250068	f	\N	30	30	\N	\N	\N	2	56	0	\N	\N	2	f	\N
262	Lab Fortified Blank Preparation	\N	\N	78	52	31	2021-08-23 19:46:47.802697	2021-08-23 19:46:48.251719	f	\N	30	30	\N	\N	\N	14	56	0	\N	\N	2	f	\N
263	Matrix Spike Sample Preparation	\N	\N	79	24	31	2021-08-23 19:46:47.807117	2021-08-23 19:46:48.253416	f	\N	30	30	\N	\N	\N	6	56	0	\N	\N	2	f	\N
250	Experiment design	2021-08-30 17:19:39.190112	\N	0	0	30	2021-08-19 22:52:55.261374	2021-08-23 17:19:39.201725	f	\N	31	\N	\N	\N	\N	0	55	0	\N	\N	2	f	\N
251	Sampling biological material	2021-09-13 17:19:39.202203	\N	32	0	30	2021-08-19 03:21:12.133153	2021-08-23 17:19:39.327777	f	\N	31	\N	\N	\N	\N	1	55	0	\N	\N	2	f	\N
252	RNA isolation	2021-09-27 17:19:39.215367	\N	64	0	30	2021-08-21 18:41:29.847948	2021-08-23 17:19:39.226102	f	\N	31	\N	\N	\N	\N	2	55	0	\N	\N	2	f	\N
253	RNA quality & quantity - BIOANALYSER	2021-10-11 17:19:39.226608	\N	96	0	30	2021-08-20 14:51:42.377647	2021-08-23 17:19:39.23667	f	\N	31	\N	\N	\N	\N	3	55	0	\N	\N	2	f	\N
254	Reverse transcription	2021-10-25 17:19:39.237124	\N	96	16	30	2021-08-19 08:56:11.589041	2021-08-23 17:19:39.247946	f	\N	31	\N	\N	\N	\N	4	55	0	\N	\N	2	f	\N
255	qPCR	2021-11-08 17:19:39.248367	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	30	2021-08-21 14:53:37.039283	2021-08-23 17:19:39.259875	f	\N	31	\N	\N	\N	\N	5	55	0	\N	\N	2	f	\N
105	12	\N	\N	0	0	\N	2020-06-10 18:15:54.786887	2020-06-10 18:15:54.786887	f	\N	7	7	\N	\N	\N	-1	29	0	\N	\N	2	f	\N
256	Data quality control	2021-11-22 17:19:39.260367	\N	32	16	30	2021-08-20 03:10:56.118175	2021-08-23 17:19:39.271442	f	\N	31	\N	\N	\N	\N	6	55	0	\N	\N	2	f	\N
257	Data analysis - ddCq	2021-12-06 17:19:39.271945	\N	0	16	30	2021-08-20 07:30:38.317192	2021-08-23 17:19:39.282186	f	\N	31	\N	\N	\N	\N	7	55	0	\N	\N	2	f	\N
258	Data analysis - Pfaffl method	2021-08-30 17:19:39.282711	\N	-1	-1	\N	2021-08-19 15:27:17.462053	2021-08-23 17:19:39.290622	t	2021-08-23 17:19:39	31	\N	31	\N	\N	-1	55	0	\N	\N	2	f	\N
264	Addition of 100 microliters of native precursor spike solution	\N	\N	122	52	31	2021-08-23 19:46:47.811562	2021-08-23 19:46:48.255134	f	\N	30	30	\N	\N	\N	0	56	0	\N	\N	2	f	\N
265	Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)	\N	\N	168	25	31	2021-08-23 19:46:47.81618	2021-08-23 19:46:48.256813	f	\N	30	30	\N	\N	\N	8	56	0	\N	\N	2	f	\N
266	Shake each aliquot for 2 minutes	\N	\N	237	26	31	2021-08-23 19:46:47.820606	2021-08-23 19:46:48.258478	f	\N	30	30	\N	\N	\N	10	56	0	\N	\N	2	f	\N
267	Add 1mL of Ethyl Acetate	\N	\N	271	26	31	2021-08-23 19:46:47.824811	2021-08-23 19:46:48.260145	f	\N	30	30	\N	\N	\N	11	56	0	\N	\N	2	f	\N
115	Experiment design	2020-11-25 14:50:52.200859	\N	0	0	14	2020-11-15 15:39:18.646591	2020-11-18 14:50:52.298632	f	\N	1	\N	\N	\N	\N	0	33	0	\N	\N	2	f	\N
120	qPCR	2021-02-03 14:50:52.503226	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	14	2020-11-13 20:28:21.326463	2020-11-18 14:50:53.185557	f	\N	1	\N	\N	\N	\N	5	33	0	\N	\N	2	f	\N
123	Data analysis - Pfaffl method	2020-11-25 14:50:52.610163	\N	-1	-1	\N	2020-11-17 06:20:16.508638	2020-11-18 14:50:52.630447	t	2020-11-18 14:50:52	1	\N	1	\N	\N	-1	33	0	\N	\N	2	f	\N
122	Data analysis - ddCq	2021-03-03 14:50:52.575419	\N	0	16	14	2020-11-14 14:39:52.962726	2020-11-18 14:50:53.405271	f	\N	1	\N	\N	\N	\N	7	33	0	\N	\N	2	f	\N
118	RNA quality & quantity - BIOANALYSER	2021-01-06 14:50:52.429472	\N	96	0	14	2020-11-13 22:25:23.892813	2020-11-18 14:50:52.992492	f	\N	1	\N	\N	\N	\N	3	33	0	\N	\N	2	f	\N
275	Analytical balance calibration	\N	\N	0	0	32	2021-08-26 20:19:40.40665	2021-08-26 20:19:40.532208	f	\N	30	30	\N	\N	\N	0	58	0	\N	\N	2	f	\N
116	Sampling biological material	2020-12-09 14:50:52.313366	\N	32	0	14	2020-11-17 09:44:14.378602	2020-11-18 14:50:52.79039	f	\N	1	\N	\N	\N	\N	1	33	0	\N	\N	2	f	\N
121	Data quality control	2021-02-17 14:50:52.540692	\N	32	16	14	2020-11-14 00:12:03.294194	2020-11-18 14:50:53.293472	f	\N	1	\N	\N	\N	\N	6	33	0	\N	\N	2	f	\N
119	Reverse transcription	2021-01-20 14:50:52.466121	\N	96	16	14	2020-11-12 15:31:10.285011	2020-11-18 14:50:53.086687	f	\N	1	\N	\N	\N	\N	4	33	0	\N	\N	2	f	\N
117	RNA isolation	2020-12-23 14:50:52.390071	\N	64	0	14	2020-11-15 23:56:01.342792	2020-11-18 14:50:52.891638	f	\N	1	\N	\N	\N	\N	2	33	0	\N	\N	2	f	\N
221	Experiment design	2021-06-09 16:10:01.707512	\N	0	0	28	2021-06-02 08:33:37.648357	2021-06-02 16:10:01.867244	f	\N	28	\N	\N	\N	\N	0	51	0	\N	\N	2	f	\N
222	Sampling biological material	2021-06-23 16:10:01.868705	\N	32	0	28	2021-05-30 12:03:42.756336	2021-06-02 16:10:02.761717	f	\N	28	\N	\N	\N	\N	1	51	0	\N	\N	2	f	\N
200	Experiment design	2021-05-27 15:17:11.694858	\N	0	0	26	2021-05-16 02:45:16.741456	2021-05-20 15:17:11.708643	f	\N	26	\N	\N	\N	\N	0	46	0	\N	\N	2	f	\N
201	Sampling biological material	2021-06-10 15:17:11.7091	\N	32	0	26	2021-05-15 03:59:26.574752	2021-05-20 15:17:11.839699	f	\N	26	\N	\N	\N	\N	1	46	0	\N	\N	2	f	\N
203	RNA quality & quantity - BIOANALYSER	2021-07-08 15:17:11.73477	\N	96	0	26	2021-05-18 03:19:24.607321	2021-05-20 15:17:11.744999	f	\N	26	\N	\N	\N	\N	3	46	0	\N	\N	2	f	\N
204	Reverse transcription	2021-07-22 15:17:11.745457	\N	96	16	26	2021-05-18 04:02:46.779295	2021-05-20 15:17:11.769129	f	\N	26	\N	\N	\N	\N	4	46	0	\N	\N	2	f	\N
205	qPCR	2021-08-05 15:17:11.769692	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	26	2021-05-17 10:45:35.0848	2021-05-20 15:17:11.780577	f	\N	26	\N	\N	\N	\N	5	46	0	\N	\N	2	f	\N
206	Data quality control	2021-08-19 15:17:11.78105	\N	32	16	26	2021-05-14 19:28:55.800737	2021-05-20 15:17:11.790687	f	\N	26	\N	\N	\N	\N	6	46	0	\N	\N	2	f	\N
207	Data analysis - ddCq	2021-09-02 15:17:11.791057	\N	0	16	26	2021-05-15 05:10:56.677644	2021-05-20 15:17:11.799587	f	\N	26	\N	\N	\N	\N	7	46	0	\N	\N	2	f	\N
208	Data analysis - Pfaffl method	2021-05-27 15:17:11.799994	\N	-1	-1	\N	2021-05-17 03:41:00.267202	2021-05-20 15:17:11.806763	t	2021-05-20 15:17:11	26	\N	26	\N	\N	-1	46	0	\N	\N	2	f	\N
223	RNA isolation	2021-07-07 16:10:01.927669	\N	64	0	28	2021-05-31 16:02:24.282334	2021-06-02 16:10:02.039671	f	\N	28	\N	\N	\N	\N	2	51	0	\N	\N	2	f	\N
224	RNA quality & quantity - BIOANALYSER	2021-07-21 16:10:02.050228	\N	96	0	28	2021-05-31 12:41:39.313349	2021-06-02 16:10:02.13368	f	\N	28	\N	\N	\N	\N	3	51	0	\N	\N	2	f	\N
225	Reverse transcription	2021-08-04 16:10:02.141062	\N	96	16	28	2021-05-31 18:24:00.764111	2021-06-02 16:10:02.272538	f	\N	28	\N	\N	\N	\N	4	51	0	\N	\N	2	f	\N
226	qPCR	2021-08-18 16:10:02.27321	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	28	2021-05-28 10:34:20.767473	2021-06-02 16:10:02.480865	f	\N	28	\N	\N	\N	\N	5	51	0	\N	\N	2	f	\N
227	Data quality control	2021-09-01 16:10:02.481333	\N	32	16	28	2021-06-02 11:56:01.342128	2021-06-02 16:10:02.531272	f	\N	28	\N	\N	\N	\N	6	51	0	\N	\N	2	f	\N
228	Data analysis - ddCq	2021-09-15 16:10:02.531803	\N	0	16	28	2021-05-29 21:08:21.787429	2021-06-02 16:10:02.557712	f	\N	28	\N	\N	\N	\N	7	51	0	\N	\N	2	f	\N
212	Experiment design	2021-06-04 15:07:44.751268	\N	0	0	27	2021-05-25 18:52:48.288501	2021-05-28 15:07:44.95103	f	\N	27	\N	\N	\N	\N	0	49	0	\N	\N	2	f	\N
213	Sampling biological material	2021-06-18 15:07:44.951911	\N	32	0	27	2021-05-22 16:44:05.44071	2021-05-28 15:07:45.927666	f	\N	27	\N	\N	\N	\N	1	49	0	\N	\N	2	f	\N
214	RNA isolation	2021-07-02 15:07:44.995315	\N	64	0	27	2021-05-23 01:16:44.387604	2021-05-28 15:07:45.008743	f	\N	27	\N	\N	\N	\N	2	49	0	\N	\N	2	f	\N
191	Experiment design	2021-05-27 14:30:30.305272	\N	0	0	25	2021-05-19 04:59:24.778543	2021-05-20 14:30:30.313985	f	\N	25	\N	\N	\N	\N	0	45	0	\N	\N	2	f	\N
192	Sampling biological material	2021-06-10 14:30:30.314428	\N	32	0	25	2021-05-19 08:44:03.743611	2021-05-20 14:30:30.426161	f	\N	25	\N	\N	\N	\N	1	45	0	\N	\N	2	f	\N
193	RNA isolation	2021-06-24 14:30:30.324758	\N	64	0	25	2021-05-16 15:49:04.996718	2021-05-20 14:30:30.33458	f	\N	25	\N	\N	\N	\N	2	45	0	\N	\N	2	f	\N
194	RNA quality & quantity - BIOANALYSER	2021-07-08 14:30:30.334997	\N	96	0	25	2021-05-14 21:29:23.556051	2021-05-20 14:30:30.345152	f	\N	25	\N	\N	\N	\N	3	45	0	\N	\N	2	f	\N
195	Reverse transcription	2021-07-22 14:30:30.34565	\N	96	16	25	2021-05-18 11:30:50.251408	2021-05-20 14:30:30.355798	f	\N	25	\N	\N	\N	\N	4	45	0	\N	\N	2	f	\N
197	Data quality control	2021-08-19 14:30:30.366703	\N	32	16	25	2021-05-14 17:19:15.299548	2021-05-20 14:30:30.376824	f	\N	25	\N	\N	\N	\N	6	45	0	\N	\N	2	f	\N
198	Data analysis - ddCq	2021-09-02 14:30:30.377282	\N	0	16	25	2021-05-18 06:59:39.110747	2021-05-20 14:30:30.386671	f	\N	25	\N	\N	\N	\N	7	45	0	\N	\N	2	f	\N
215	RNA quality & quantity - BIOANALYSER	2021-07-16 15:07:45.00921	\N	96	0	27	2021-05-27 21:08:42.355768	2021-05-28 15:07:45.021658	f	\N	27	\N	\N	\N	\N	3	49	0	\N	\N	2	f	\N
216	Reverse transcription	2021-07-30 15:07:45.02208	\N	96	16	27	2021-05-26 12:13:28.841156	2021-05-28 15:07:45.034312	f	\N	27	\N	\N	\N	\N	4	49	0	\N	\N	2	f	\N
217	qPCR	2021-08-13 15:07:45.034759	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	27	2021-05-24 21:43:28.72126	2021-05-28 15:07:45.063272	f	\N	27	\N	\N	\N	\N	5	49	0	\N	\N	2	f	\N
218	Data quality control	2021-08-27 15:07:45.063746	\N	32	16	27	2021-05-22 20:05:33.270171	2021-05-28 15:07:45.075605	f	\N	27	\N	\N	\N	\N	6	49	0	\N	\N	2	f	\N
219	Data analysis - ddCq	2021-09-10 15:07:45.076015	\N	0	16	27	2021-05-24 12:24:09.731718	2021-05-28 15:07:45.088283	f	\N	27	\N	\N	\N	\N	7	49	0	\N	\N	2	f	\N
220	Data analysis - Pfaffl method	2021-06-04 15:07:45.088752	\N	-1	-1	\N	2021-05-22 15:11:01.367459	2021-05-28 15:07:45.112204	t	2021-05-28 15:07:45	27	\N	27	\N	\N	-1	49	0	\N	\N	2	f	\N
229	Data analysis - Pfaffl method	2021-06-09 16:10:02.559393	\N	-1	-1	\N	2021-05-30 13:57:07.390307	2021-06-02 16:10:02.587681	t	2021-06-02 16:10:02	28	\N	28	\N	\N	-1	51	0	\N	\N	2	f	\N
196	qPCR	2021-08-05 14:30:30.356236	PCR is a method where an enzyme\n      (thermostable DNA polymerase, originally isolated in 1960s\n      from bacterium Thermus aquaticus, growing in hot lakes of\n      Yellowstone park, USA) amplifies a short specific part of\n      the template DNA (amplicon) in cycles. In every cycle the\n      number of short specific sections of DNA is doubled, leading\n      to an exponential amplification of targets. More on how\n      conventional PCR works can be found here.	64	16	25	2021-05-17 21:00:10.416783	2021-05-20 14:30:30.366245	f	\N	25	\N	\N	\N	\N	5	45	0	\N	\N	2	f	\N
199	Data analysis - Pfaffl method	2021-05-27 14:30:30.387109	\N	-1	-1	\N	2021-05-18 19:36:51.035706	2021-05-20 14:30:30.394672	t	2021-05-20 14:30:30	25	\N	25	\N	\N	-1	45	0	\N	\N	2	f	\N
211	Instrument Start up	\N	\N	77	0	\N	2021-05-20 15:31:37.563125	2021-05-20 15:31:37.563125	f	\N	24	24	\N	\N	\N	-1	43	0	\N	\N	2	f	\N
210	Sample Run Log	\N	\N	39	0	\N	2021-05-20 15:31:37.558746	2021-05-20 15:31:37.558746	f	\N	24	24	\N	\N	\N	-1	43	0	\N	\N	2	f	\N
209	Prepare Calibration Standards & QCs	\N	\N	0	0	\N	2021-05-20 15:31:37.54966	2021-05-20 15:31:37.54966	f	\N	24	24	\N	\N	\N	-1	43	0	\N	\N	2	f	\N
202	RNA isolation	2021-06-24 15:17:11.721868	\N	64	0	26	2021-05-15 08:57:57.917013	2021-05-20 15:17:11.734299	f	\N	26	\N	\N	\N	\N	2	46	0	\N	\N	2	f	\N
276	Soil Weighing	\N	\N	43	0	32	2021-08-26 20:19:40.430267	2021-08-26 20:19:40.535048	f	\N	30	30	\N	\N	\N	1	58	0	\N	\N	2	f	\N
235	Receive the new sample	2021-07-07 00:00:00	\N	0	0	\N	2021-07-06 15:50:12.345388	2021-07-07 16:07:45.721724	f	\N	27	27	\N	\N	\N	-1	52	1	2021-07-07 16:07:45.720564	\N	3	f	\N
236	Mix with 5% biochar	\N	<div>2mm granular alumina was mixed with 5% Biochar MgCO3 pellet&nbsp;</div>	0	16	\N	2021-07-06 15:50:12.394993	2021-07-07 16:08:21.50664	f	\N	27	27	\N	\N	\N	-1	52	1	2021-07-07 16:08:20.461217	\N	3	f	\N
237	Adsorption test	\N	\N	0	32	\N	2021-07-06 15:50:12.401918	2021-07-07 16:08:45.09065	f	\N	27	27	\N	\N	\N	-1	52	1	2021-07-07 16:08:45.085808	\N	3	f	\N
230	Receive new samples (06/10/2021)	2021-06-10 00:00:00	<div>Two samples are to be delivered to EPA&nbsp;</div>\r\n<div>1. prebaked 250 C Biochar</div>\r\n<div>2. MgCO3 with high surface area.&nbsp;</div>	0	0	\N	2021-06-10 16:07:24.092767	2021-06-10 18:45:11.12213	f	\N	27	27	\N	\N	\N	-1	50	1	2021-06-10 18:45:11.11971	\N	3	f	\N
231	Bake each sample at 250 C	2021-06-14 00:00:00	<div>&nbsp;</div>\r\n<div>Two samples were as shown below:</div>\r\n<div>20210610A (A herein)</div>\r\n<div>20210610B (B herein)&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>A and B will be baked at 250C for 30 min. to check its stability and performance test&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>&nbsp;</div>	0	14	\N	2021-06-10 16:07:24.183893	2021-06-14 19:22:13.908425	f	\N	27	27	\N	\N	\N	-1	50	1	2021-06-14 19:22:13.906075	\N	3	f	\N
233	Collect samples at t = 1,2,3,24 h	\N	\N	0	42	\N	2021-06-10 16:07:24.198035	2021-06-16 15:58:53.607114	f	\N	27	27	\N	\N	\N	-1	50	1	2021-06-16 15:58:53.604512	\N	3	f	\N
232	Run adsorption test	\N	<div>Run PO4 adsorption study with 1000 ppm stock solution&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>collect samples at t= 1,2,3,24 h</div>	0	28	\N	2021-06-10 16:07:24.19286	2021-06-16 15:59:07.711814	f	\N	27	27	\N	\N	\N	-1	50	1	2021-06-16 15:59:07.709925	\N	3	f	\N
234	Analyze samples using Spectraphotometer	\N	\N	0	56	\N	2021-06-10 16:07:24.205766	2021-06-16 15:59:27.114603	f	\N	27	27	\N	\N	\N	-1	50	1	2021-06-16 15:59:27.11289	\N	3	f	\N
\.



--
-- Data for Name: projects; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.projects (id, name, visibility, due_date, team_id, created_at, updated_at, archived, archived_on, created_by_id, last_modified_by_id, archived_by_id, restored_by_id, restored_on, experiments_order, template, demo, project_folder_id) FROM stdin;
1	Demo project by SciNote	0	\N	1	2019-05-30 04:43:13.863771	2021-11-17 12:23:35.71513	f	\N	1	1	\N	\N	\N	\N	f	t	\N
2	[NEW] Demo project by SciNote	0	\N	3	2019-08-21 11:51:40.917123	2019-08-27 15:15:47.722877	f	\N	4	4	\N	\N	\N	\N	f	t	\N
3	[NEW] Demo project by SciNote	0	\N	2	2019-08-23 15:24:59.608762	2019-08-28 07:31:09.518491	f	\N	1	1	\N	\N	\N	\N	f	t	\N
4	[NEW] Demo project by SciNote	0	\N	4	2019-08-25 03:40:05.464976	2019-08-28 08:35:56.864693	f	\N	6	6	\N	\N	\N	\N	f	t	\N
5	[NEW] Demo project by SciNote	0	\N	5	2019-08-22 14:12:08.934627	2020-06-10 18:25:03.604688	f	\N	7	7	\N	\N	\N	\N	f	t	\N
6	[NEW] Demo project by SciNote	0	\N	6	2019-09-11 21:08:10.47327	2019-09-12 18:26:33.435082	f	\N	8	8	\N	\N	\N	\N	f	t	\N
21	AOP	0	\N	1	2019-10-22 15:41:41.702851	2020-09-28 12:27:43.980683	t	2020-09-28 12:27:43	1	1	\N	\N	\N	\N	\N	f	\N
22	[NEW] Demo project by SciNote	0	\N	8	2019-11-10 00:11:15.875063	2019-11-13 13:58:19.26301	f	\N	9	9	\N	\N	\N	\N	f	t	\N
23	[NEW] Demo project by SciNote	0	\N	7	2019-11-23 02:15:37.470969	2021-12-08 18:45:29.083343	f	\N	4	4	\N	\N	\N	\N	f	t	\N
24	Micro QC	0	\N	3	2019-11-25 14:09:53.309553	2019-11-25 14:25:23.460513	t	2019-11-25 14:25:23	4	4	\N	\N	\N	\N	\N	f	\N
25	Micro QC	0	\N	7	2019-11-25 14:25:41.548645	2021-12-08 18:46:18.513594	f	\N	4	4	\N	\N	\N	\N	\N	f	\N
26	[NEW] Demo project by SciNote	0	\N	9	2019-11-22 18:46:45.388487	2019-11-25 18:01:05.362488	f	\N	10	10	\N	\N	\N	\N	f	t	\N
27	[NEW] Demo project by SciNote	0	\N	10	2020-05-12 07:37:53.299935	2020-05-12 16:13:55.091146	f	\N	11	11	\N	\N	\N	\N	f	t	\N
28	[NEW] Demo project by SciNote	0	\N	11	2020-06-05 14:47:33.358783	2020-06-10 14:44:24.937767	f	\N	12	12	\N	\N	\N	\N	f	t	\N
29	test	0	\N	5	2020-06-10 18:15:06.973233	2020-06-10 18:16:28.997828	f	\N	7	7	\N	\N	\N	\N	\N	f	\N
30	Total Coliforms and E. coli in Water	0	\N	7	2020-06-12 11:47:19.254679	2021-04-08 15:53:54.247739	t	2021-04-08 15:52:53	4	4	\N	\N	\N	\N	\N	f	\N
31	Phylochip	0	\N	3	2020-06-12 12:03:51.634987	2021-04-08 15:54:38.834136	f	\N	4	4	\N	\N	\N	\N	\N	f	\N
32	Evaluating de-packaging technologies to remove film plastics from food	1	\N	1	2020-09-28 12:07:46.954924	2022-02-09 15:06:12.560304	f	\N	1	1	\N	\N	\N	\N	\N	f	\N
33	[NEW] Demo project by SciNote	0	\N	13	2020-10-06 10:02:00.511065	2020-10-07 19:20:39.706253	f	\N	16	16	\N	\N	\N	\N	f	t	\N
34	[NEW] Demo project by SciNote	0	\N	12	2020-11-14 11:34:41.864506	2020-11-18 14:51:01.257958	f	\N	1	1	\N	\N	\N	\N	f	t	\N
35	[NEW] Demo project by SciNote	0	\N	14	2020-11-12 16:18:25.638378	2020-11-18 14:54:18.044129	f	\N	17	17	\N	\N	\N	\N	f	t	\N
36	Test	1	\N	1	2021-05-19 14:31:33.17671	2021-05-19 14:32:00.132844	t	2021-05-19 14:32:00	1	1	\N	\N	\N	\N	\N	f	\N
37	[NEW] Demo project by SciNote	0	\N	15	2021-05-18 09:09:45.996413	2021-05-19 20:02:04.306639	f	\N	20	20	\N	\N	\N	\N	f	t	\N
38	[NEW] Demo project by SciNote	0	\N	16	2021-05-15 16:57:29.687747	2021-05-20 14:13:34.018229	f	\N	21	21	\N	\N	\N	\N	f	t	\N
39	[NEW] Demo project by SciNote	0	\N	17	2021-05-16 03:23:40.313284	2021-05-20 15:42:01.145347	f	\N	22	22	\N	\N	\N	\N	f	t	\N
40	[NEW] Demo project by SciNote	0	\N	18	2021-05-15 23:19:39.024115	2021-05-20 14:13:48.175764	f	\N	23	23	\N	\N	\N	\N	f	t	\N
41	[NEW] Demo project by SciNote	0	\N	19	2021-05-16 23:07:32.329187	2021-05-20 14:46:34.044725	f	2021-05-20 14:36:22	24	24	\N	\N	2021-05-20 14:44:13	\N	f	t	\N
42	WID TOC Analyses	1	\N	19	2021-05-20 14:22:42.015889	2021-05-20 15:31:51.302981	f	\N	24	24	\N	\N	\N	\N	\N	f	\N
43	[NEW] Demo project by SciNote	0	\N	20	2021-05-20 10:30:48.685042	2021-05-20 14:31:19.36092	f	\N	25	25	\N	\N	\N	\N	f	t	\N
44	[NEW] Demo project by SciNote	0	\N	21	2021-05-15 07:38:14.703189	2021-05-20 15:17:42.774187	f	\N	26	26	\N	\N	\N	\N	f	t	\N
45	Sensors project	0	\N	16	2021-05-20 15:31:53.771216	2021-05-26 12:40:04.689717	f	\N	21	21	\N	\N	\N	\N	\N	f	\N
47	Membrane distillation	0	\N	22	2021-05-28 15:08:45.989435	2021-05-28 15:08:46.439288	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
46	[NEW] Demo project by SciNote	0	\N	22	2021-05-24 19:06:25.090486	2021-05-28 15:08:48.29073	f	\N	27	27	\N	\N	\N	\N	f	t	\N
48	P Recovery	0	\N	22	2021-05-28 15:09:19.293835	2021-07-13 16:25:11.950953	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
49	NF for PFAS	0	\N	22	2021-05-28 15:09:34.678106	2021-05-28 15:09:34.715009	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
50	GAC for CFF concentrate	0	\N	22	2021-05-28 15:09:46.574297	2021-05-28 15:09:46.650877	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
51	AOX for CFF concentrate discharge	0	\N	22	2021-05-28 15:10:03.253323	2021-05-28 15:10:03.268315	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
52	Pb filter recovery	0	\N	22	2021-05-28 15:10:11.190422	2021-05-28 15:10:11.202863	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
53	CO2 separation using membrane	0	\N	22	2021-05-28 15:10:21.204093	2021-05-28 15:10:21.217913	f	\N	27	27	\N	\N	\N	\N	\N	f	\N
54	[NEW] Demo project by SciNote	0	\N	23	2021-05-28 08:00:28.065726	2021-06-02 16:10:44.638115	f	\N	28	28	\N	\N	\N	\N	f	t	\N
55	[NEW] Demo project by SciNote	0	\N	24	2021-08-16 19:13:00.747016	2021-08-23 14:59:01.086832	f	\N	30	30	\N	\N	\N	\N	f	t	\N
56	[NEW] Demo project by SciNote	0	\N	25	2021-08-17 09:15:33.866158	2021-08-23 17:20:19.945742	f	\N	31	31	\N	\N	\N	\N	f	t	\N
57	PFAS Liquid Extraction 8/23	0	\N	24	2021-08-23 18:49:48.204902	2021-08-23 19:47:33.520932	f	\N	30	30	\N	\N	\N	\N	\N	f	\N
58	TEST	1	\N	24	2021-08-25 16:06:30.29804	2021-08-25 16:08:54.105315	f	\N	30	30	\N	\N	\N	\N	\N	f	\N
59	Moisture Content Determination TEST	0	\N	24	2021-08-26 20:02:24.22913	2021-08-26 20:20:07.685308	f	\N	30	30	\N	\N	\N	\N	\N	f	\N
60	TEST Moisture Determination	0	\N	25	2021-08-26 20:53:28.248447	2021-08-26 20:53:28.258301	f	\N	30	30	\N	\N	\N	\N	\N	f	\N
\.


--
-- Data for Name: activities; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.activities (id, my_module_id, owner_id, type_of, message, created_at, updated_at, project_id, experiment_id, subject_type, subject_id, team_id, group_type, "values") FROM stdin;
224	\N	27	8	\N	2021-06-10 16:07:24.239769	2021-06-10 16:07:24.239769	48	\N	MyModule	231	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Bake each sample at 250 C", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 231, "type": "MyModule", "value": "Bake each sample at 250 C", "value_for": "name"}}}
1	\N	1	92	\N	2019-07-22 11:21:59.679269	2019-07-22 11:21:59.679269	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 2, "type": "User", "value": "gonzalez.michael@epa.gov", "value_for": "name"}}}
2	\N	1	92	\N	2019-07-22 11:21:59.903776	2019-07-22 11:21:59.903776	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 3, "type": "User", "value": "venkatapathy.raghuraman@epa.gov", "value_for": "name"}}}
3	\N	4	92	\N	2019-08-27 15:17:09.473104	2019-08-27 15:17:09.473104	\N	\N	Team	3	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab"}, "message_items": {"role": "Administrator", "team": {"id": 3, "type": "Team", "value": "R7-Molecular Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_invited": {"id": 5, "type": "User", "value": "baker.steven@epa.gov", "value_for": "name"}}}
4	\N	6	92	\N	2019-08-28 08:36:02.657523	2019-08-28 08:36:02.657523	\N	\N	Team	4	4	\N	{"breadcrumbs": {"team": "RTEB"}, "message_items": {"role": "Administrator", "team": {"id": 4, "type": "Team", "value": "RTEB", "value_for": "name"}, "user": {"id": 6, "type": "User", "value": "Raghuraman Venkatapathy", "value_for": "full_name"}, "user_invited": {"id": 3, "type": "User", "value": "venkatapathy.raghuraman@epa.gov", "value_for": "name"}}}
5	\N	1	92	\N	2019-08-28 09:15:40.790255	2019-08-28 09:15:40.790255	\N	\N	Team	2	2	\N	{"breadcrumbs": {"team": "ECEB SW"}, "message_items": {"role": "Administrator", "team": {"id": 2, "type": "Team", "value": "ECEB SW", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 6, "type": "User", "value": "Raghuraman Venkatapathy", "value_for": "name"}}}
6	\N	7	70	\N	2019-08-30 16:08:39.823396	2019-08-30 16:08:39.823396	\N	\N	RepositoryBase	6	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Reagents"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 6, "type": "Repository", "value": "Reagents", "value_for": "name"}}}
7	\N	7	70	\N	2019-08-30 16:10:22.553336	2019-08-30 16:10:22.553336	\N	\N	RepositoryBase	7	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Filters"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 7, "type": "Repository", "value": "Filters", "value_for": "name"}}}
8	\N	7	70	\N	2019-08-30 16:10:29.455541	2019-08-30 16:10:29.455541	\N	\N	RepositoryBase	8	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Extracts"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 8, "type": "Repository", "value": "Extracts", "value_for": "name"}}}
9	\N	7	75	\N	2019-08-30 16:10:44.171522	2019-08-30 16:10:44.171522	\N	\N	RepositoryBase	5	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Samples"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 5, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 21, "type": "RepositoryRow", "value": "ZAT/1", "value_for": "name"}}}
10	\N	7	75	\N	2019-08-30 16:10:44.266051	2019-08-30 16:10:44.266051	\N	\N	RepositoryBase	5	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Samples"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 5, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 22, "type": "RepositoryRow", "value": "ZAT/2", "value_for": "name"}}}
15	\N	7	27	\N	2019-09-12 14:33:50.928733	2019-09-12 14:33:50.928733	5	\N	Experiment	6	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "[NEW] Demo project by SciNote", "experiment": "Mobile Test"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "experiment": {"id": 6, "type": "Experiment", "value": "Mobile Test", "value_for": "name"}}}
16	\N	4	92	\N	2019-09-12 15:39:47.646422	2019-09-12 15:39:47.646422	\N	\N	Team	3	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab"}, "message_items": {"role": "Administrator", "team": {"id": 3, "type": "Team", "value": "R7-Molecular Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_invited": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
17	\N	4	92	\N	2019-10-21 16:32:03.038876	2019-10-21 16:32:03.038876	\N	\N	Team	3	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab"}, "message_items": {"role": "Guest", "team": {"id": 3, "type": "Team", "value": "R7-Molecular Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_invited": {"id": 8, "type": "User", "value": "Michael F Davis", "value_for": "name"}}}
18	\N	1	0	\N	2019-10-22 15:41:41.809164	2019-10-22 15:41:41.809164	21	\N	Project	21	1	\N	{"breadcrumbs": {"team": "My projects", "project": "AOP"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "project": {"id": 21, "type": "Project", "value": "AOP", "value_for": "name"}}}
19	\N	4	0	\N	2019-11-25 14:09:53.360008	2019-11-25 14:09:53.360008	24	\N	Project	24	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 24, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
20	\N	4	27	\N	2019-11-25 14:11:09.865711	2019-11-25 14:11:09.865711	24	\N	Experiment	10	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "HPC for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 10, "type": "Experiment", "value": "HPC for Reagent Water", "value_for": "name"}}}
21	\N	4	8	\N	2019-11-25 14:12:03.016411	2019-11-25 14:12:03.016411	24	\N	MyModule	73	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "my_module": "HPC", "experiment": "HPC for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 73, "type": "MyModule", "value": "HPC", "value_for": "name"}}}
22	\N	4	27	\N	2019-11-25 14:13:16.80009	2019-11-25 14:13:16.80009	24	\N	Experiment	11	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "Conductivity for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 11, "type": "Experiment", "value": "Conductivity for Reagent Water", "value_for": "name"}}}
23	\N	4	27	\N	2019-11-25 14:14:34.021006	2019-11-25 14:14:34.021006	24	\N	Experiment	12	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "Chlorine for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 12, "type": "Experiment", "value": "Chlorine for Reagent Water", "value_for": "name"}}}
24	\N	4	27	\N	2019-11-25 14:15:39.847675	2019-11-25 14:15:39.847675	24	\N	Experiment	13	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "Heavy Metals for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 13, "type": "Experiment", "value": "Heavy Metals for Reagent Water", "value_for": "name"}}}
25	\N	4	27	\N	2019-11-25 14:16:46.284429	2019-11-25 14:16:46.284429	24	\N	Experiment	14	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "Quanti-Tray Sealer Leak Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 14, "type": "Experiment", "value": "Quanti-Tray Sealer Leak Check", "value_for": "name"}}}
26	\N	4	27	\N	2019-11-25 14:17:56.247985	2019-11-25 14:17:56.247985	24	\N	Experiment	15	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "Balance Verification"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 15, "type": "Experiment", "value": "Balance Verification", "value_for": "name"}}}
27	\N	4	27	\N	2019-11-25 14:20:50.310028	2019-11-25 14:20:50.310028	24	\N	Experiment	16	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC", "experiment": "90 Day Colilert Media Quality Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 16, "type": "Experiment", "value": "90 Day Colilert Media Quality Check", "value_for": "name"}}}
28	\N	4	3	\N	2019-11-25 14:25:23.469081	2019-11-25 14:25:23.469081	24	\N	Project	24	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Micro QC"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 24, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
29	\N	4	0	\N	2019-11-25 14:25:41.567642	2019-11-25 14:25:41.567642	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
30	\N	4	27	\N	2019-11-25 14:25:55.032918	2019-11-25 14:25:55.032918	25	\N	Experiment	17	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "HPC for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 17, "type": "Experiment", "value": "HPC for Reagent Water", "value_for": "name"}}}
31	\N	4	27	\N	2019-11-25 14:26:13.107793	2019-11-25 14:26:13.107793	25	\N	Experiment	18	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Conductivity for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 18, "type": "Experiment", "value": "Conductivity for Reagent Water", "value_for": "name"}}}
32	\N	4	27	\N	2019-11-25 14:28:10.56655	2019-11-25 14:28:10.56655	25	\N	Experiment	19	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Chlorine for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 19, "type": "Experiment", "value": "Chlorine for Reagent Water", "value_for": "name"}}}
33	\N	4	27	\N	2019-11-25 14:28:20.883683	2019-11-25 14:28:20.883683	25	\N	Experiment	20	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Heavy Metals for Reagent Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 20, "type": "Experiment", "value": "Heavy Metals for Reagent Water", "value_for": "name"}}}
34	\N	4	27	\N	2019-11-25 14:28:32.571025	2019-11-25 14:28:32.571025	25	\N	Experiment	21	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Quanti-Tray Sealer Leak Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 21, "type": "Experiment", "value": "Quanti-Tray Sealer Leak Check", "value_for": "name"}}}
35	\N	4	27	\N	2019-11-25 14:28:43.663559	2019-11-25 14:28:43.663559	25	\N	Experiment	22	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Balance Verification"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 22, "type": "Experiment", "value": "Balance Verification", "value_for": "name"}}}
36	\N	4	27	\N	2019-11-25 14:29:22.062816	2019-11-25 14:29:22.062816	25	\N	Experiment	23	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 23, "type": "Experiment", "value": "Autoclave Sterilization Check", "value_for": "name"}}}
37	\N	4	27	\N	2019-11-25 14:30:18.369219	2019-11-25 14:30:18.369219	25	\N	Experiment	24	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 24, "type": "Experiment", "value": "Autoclave Timing Check", "value_for": "name"}}}
38	\N	4	27	\N	2019-11-25 14:31:00.050962	2019-11-25 14:31:00.050962	25	\N	Experiment	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Media QA/QC"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 25, "type": "Experiment", "value": "Media QA/QC", "value_for": "name"}}}
39	\N	4	8	\N	2019-11-25 17:01:05.129776	2019-11-25 17:01:05.129776	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
40	\N	4	8	\N	2019-11-25 17:01:05.144331	2019-11-25 17:01:05.144331	25	\N	MyModule	75	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Run 15 minute sterilization cycle with Sportrol Ampule", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 75, "type": "MyModule", "value": "Run 15 minute sterilization cycle with Sportrol Ampule", "value_for": "name"}}}
41	\N	4	8	\N	2019-11-25 17:01:05.152302	2019-11-25 17:01:05.152302	25	\N	MyModule	76	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Incubate working ampule and the control ampule at 60°C for 48 hours", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 76, "type": "MyModule", "value": "Incubate working ampule and the control ampule at 60°C for 48 hours", "value_for": "name"}}}
97	\N	1	92	\N	2020-09-28 13:00:12.063251	2020-09-28 13:00:12.063251	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 13, "type": "User", "value": "foster.albert@epa.gov", "value_for": "name"}}}
42	\N	4	8	\N	2019-11-25 17:01:05.157636	2019-11-25 17:01:05.157636	25	\N	MyModule	77	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Data Results at 24 hours and 48 hours", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 77, "type": "MyModule", "value": "Data Results at 24 hours and 48 hours", "value_for": "name"}}}
43	\N	4	61	\N	2019-11-25 17:01:43.571863	2019-11-25 17:01:43.571863	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1574640000}}}
44	\N	4	108	\N	2019-11-25 17:02:26.955994	2019-11-25 17:02:26.955994	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
45	\N	4	64	\N	2019-11-25 17:02:26.996214	2019-11-25 17:02:26.996214	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
46	\N	4	65	\N	2019-11-25 17:03:12.78487	2019-11-25 17:03:12.78487	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "Drain and Flush Prep procedure (prior to use)", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
47	\N	4	108	\N	2019-11-25 17:03:21.291499	2019-11-25 17:03:21.291499	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
48	\N	4	64	\N	2019-11-25 17:03:21.312117	2019-11-25 17:03:21.312117	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
160	\N	1	0	\N	2021-05-19 14:31:33.25422	2021-05-19 14:31:33.25422	36	\N	Project	36	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Test"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "project": {"id": 36, "type": "Project", "value": "Test", "value_for": "name"}}}
49	\N	4	65	\N	2019-11-25 17:04:02.047769	2019-11-25 17:04:02.047769	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "COMPLETED - Drain and Flush Prep procedure (prior to use)", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}}}
50	\N	4	66	\N	2019-11-25 17:04:04.344327	2019-11-25 17:04:04.344327	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
51	\N	4	64	\N	2019-11-25 17:04:08.241123	2019-11-25 17:04:08.241123	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
52	\N	4	66	\N	2019-11-25 17:04:11.61734	2019-11-25 17:04:11.61734	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
53	\N	4	64	\N	2019-11-25 17:06:19.539963	2019-11-25 17:06:19.539963	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
54	\N	4	66	\N	2019-11-25 17:06:21.222191	2019-11-25 17:06:21.222191	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 74, "type": "Tag", "value": "New tag", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
98	\N	1	27	\N	2020-12-14 11:26:50.8036	2020-12-14 11:26:50.8036	32	\N	Experiment	35	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "experiment": "Raman"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "experiment": {"id": 35, "type": "Experiment", "value": "Raman", "value_for": "name"}}}
55	\N	4	66	\N	2019-11-25 17:06:28.68397	2019-11-25 17:06:28.68397	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "COMPLETED - Drain and Flush Prep procedure (prior to use)", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
56	\N	4	64	\N	2019-11-25 17:06:31.316082	2019-11-25 17:06:31.316082	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "COMPLETED - Drain and Flush Prep procedure (prior to use)", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
57	\N	4	66	\N	2019-11-25 17:06:37.21939	2019-11-25 17:06:37.21939	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "COMPLETED - Drain and Flush Prep procedure (prior to use)", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
58	\N	4	53	\N	2019-11-25 17:07:20.397549	2019-11-25 17:07:20.397549	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
59	\N	4	80	\N	2019-11-25 17:39:31.387015	2019-11-25 17:39:31.387015	\N	\N	Protocol	78	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "b"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 78, "type": "Protocol", "value": "b", "value_for": "name"}}}
63	\N	4	92	\N	2019-11-25 18:03:25.169494	2019-11-25 18:03:25.169494	\N	\N	Team	7	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab"}, "message_items": {"role": "Normal user", "team": {"id": 7, "type": "Team", "value": "R7-Microbiology Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_invited": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
204	\N	27	0	\N	2021-05-28 15:09:34.739173	2021-05-28 15:09:34.739173	49	\N	Project	49	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "NF for PFAS"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 49, "type": "Project", "value": "NF for PFAS", "value_for": "name"}}}
64	\N	4	13	\N	2019-11-25 18:04:51.606519	2019-11-25 18:04:51.606519	25	\N	MyModule	75	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Run 15 minute sterilization cycle with Sportrol Ampule", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 75, "type": "MyModule", "value": "Run 15 minute sterilization cycle with Sportrol Ampule", "value_for": "name"}, "user_target": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "name"}}}
65	\N	4	13	\N	2019-11-25 18:05:09.407087	2019-11-25 18:05:09.407087	25	\N	MyModule	76	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Incubate working ampule and the control ampule at 60°C for 48 hours", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 76, "type": "MyModule", "value": "Incubate working ampule and the control ampule at 60°C for 48 hours", "value_for": "name"}, "user_target": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "name"}}}
66	\N	4	5	\N	2019-11-25 18:05:32.338987	2019-11-25 18:05:32.338987	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"role": "User", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
67	\N	7	80	\N	2020-06-10 18:14:01.104726	2020-06-10 18:14:01.104726	\N	\N	Protocol	106	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "protocol": "Test"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "protocol": {"id": 106, "type": "Protocol", "value": "Test", "value_for": "name"}}}
68	\N	7	81	\N	2020-06-10 18:14:51.050252	2020-06-10 18:14:51.050252	\N	\N	Protocol	106	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "protocol": "Test"}, "message_items": {"step": {"id": 441, "type": "Step", "value": "Sampling", "value_for": "name"}, "user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "protocol": {"id": 106, "type": "Protocol", "value": "Test", "value_for": "name"}, "step_position": {"id": 441, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
69	\N	7	0	\N	2020-06-10 18:15:07.02072	2020-06-10 18:15:07.02072	29	\N	Project	29	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "test"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "project": {"id": 29, "type": "Project", "value": "test", "value_for": "name"}}}
70	\N	7	27	\N	2020-06-10 18:15:19.82421	2020-06-10 18:15:19.82421	29	\N	Experiment	29	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "test", "experiment": "1111"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "experiment": {"id": 29, "type": "Experiment", "value": "1111", "value_for": "name"}}}
71	\N	7	8	\N	2020-06-10 18:15:54.826149	2020-06-10 18:15:54.826149	29	\N	MyModule	105	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "test", "my_module": "12", "experiment": "1111"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "my_module": {"id": 105, "type": "MyModule", "value": "12", "value_for": "name"}}}
72	\N	7	46	\N	2020-06-10 18:18:36.51033	2020-06-10 18:18:36.51033	29	\N	Protocol	107	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "test", "protocol": null, "my_module": "12", "experiment": "1111"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "protocol": {"id": 107, "type": "Protocol", "value": "", "value_for": "name"}, "my_module": {"id": 105, "type": "MyModule", "value": "12", "value_for": "name"}, "protocol_repository": {"id": 106, "type": "Protocol", "value": "Test", "value_for": "name"}}}
73	\N	7	32	\N	2020-06-10 18:25:03.616731	2020-06-10 18:25:03.616731	5	\N	Project	5	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "[NEW] Demo project by SciNote"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "project": {"id": 5, "type": "Project", "value": "[NEW] Demo project by SciNote", "value_for": "name"}}}
74	\N	7	18	\N	2020-06-10 18:30:01.256642	2020-06-10 18:30:01.256642	5	\N	Protocol	42	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "[NEW] Demo project by SciNote", "protocol": null, "my_module": "qPCR", "experiment": "Polymerase chain reaction"}, "message_items": {"step": {"id": 191, "type": "Step", "value": "Master Mix Preparation", "value_for": "name"}, "user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "num_all": "3", "my_module": {"id": 42, "type": "MyModule", "value": "qPCR", "value_for": "name"}, "num_completed": "1", "step_position": {"id": 191, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
75	\N	7	18	\N	2020-06-10 18:30:04.670774	2020-06-10 18:30:04.670774	5	\N	Protocol	42	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "[NEW] Demo project by SciNote", "protocol": null, "my_module": "qPCR", "experiment": "Polymerase chain reaction"}, "message_items": {"step": {"id": 192, "type": "Step", "value": "Setup of the 96-well Plate", "value_for": "name"}, "user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "num_all": "3", "my_module": {"id": 42, "type": "MyModule", "value": "qPCR", "value_for": "name"}, "num_completed": "2", "step_position": {"id": 192, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
76	\N	4	80	\N	2020-06-12 11:32:47.76661	2020-06-12 11:32:47.76661	\N	\N	Protocol	108	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "Colilert 2338.12"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 108, "type": "Protocol", "value": "Colilert 2338.12", "value_for": "name"}}}
77	\N	4	0	\N	2020-06-12 11:47:19.288156	2020-06-12 11:47:19.288156	30	\N	Project	30	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Total Coliforms and E. coli in Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 30, "type": "Project", "value": "Total Coliforms and E. coli in Water", "value_for": "name"}}}
78	\N	4	27	\N	2020-06-12 11:48:19.689471	2020-06-12 11:48:19.689471	30	\N	Experiment	30	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Total Coliforms and E. coli in Water", "experiment": "TC EC Analysis"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 30, "type": "Experiment", "value": "TC EC Analysis", "value_for": "name"}}}
79	\N	4	92	\N	2020-06-12 11:55:49.218042	2020-06-12 11:55:49.218042	\N	\N	Team	7	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab"}, "message_items": {"role": "Normal user", "team": {"id": 7, "type": "Team", "value": "R7-Microbiology Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_invited": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
80	\N	4	0	\N	2020-06-12 12:03:51.656756	2020-06-12 12:03:51.656756	31	\N	Project	31	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Phylochip"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 31, "type": "Project", "value": "Phylochip", "value_for": "name"}}}
81	\N	4	5	\N	2020-06-12 12:04:02.013793	2020-06-12 12:04:02.013793	31	\N	Project	31	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "project": "Phylochip"}, "message_items": {"role": "Owner", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 31, "type": "Project", "value": "Phylochip", "value_for": "name"}, "user_target": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
99	\N	1	28	\N	2020-12-14 11:27:35.872998	2020-12-14 11:27:35.872998	32	\N	Experiment	35	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "experiment": "Raman"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "experiment": {"id": 35, "type": "Experiment", "value": "Raman", "value_for": "name"}}}
82	\N	4	5	\N	2020-06-12 12:04:45.762718	2020-06-12 12:04:45.762718	30	\N	Project	30	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Total Coliforms and E. coli in Water"}, "message_items": {"role": "Owner", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 30, "type": "Project", "value": "Total Coliforms and E. coli in Water", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
83	\N	4	5	\N	2020-06-12 12:04:49.310011	2020-06-12 12:04:49.310011	30	\N	Project	30	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Total Coliforms and E. coli in Water"}, "message_items": {"role": "Owner", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 30, "type": "Project", "value": "Total Coliforms and E. coli in Water", "value_for": "name"}, "user_target": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
84	\N	4	5	\N	2020-06-12 12:04:58.546108	2020-06-12 12:04:58.546108	25	\N	Project	25	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC"}, "message_items": {"role": "Owner", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 25, "type": "Project", "value": "Micro QC", "value_for": "name"}, "user_target": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
85	\N	4	5	\N	2020-06-12 12:05:08.916874	2020-06-12 12:05:08.916874	23	\N	Project	23	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "[NEW] Demo project by SciNote"}, "message_items": {"role": "Owner", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 23, "type": "Project", "value": "[NEW] Demo project by SciNote", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
86	\N	4	5	\N	2020-06-12 12:05:11.95629	2020-06-12 12:05:11.95629	23	\N	Project	23	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "[NEW] Demo project by SciNote"}, "message_items": {"role": "Owner", "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 23, "type": "Project", "value": "[NEW] Demo project by SciNote", "value_for": "name"}, "user_target": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
87	\N	1	0	\N	2020-09-28 12:07:47.154638	2020-09-28 12:07:47.154638	32	\N	Project	32	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "project": {"id": 32, "type": "Project", "value": "Evaluating de-packaging technologies to remove film plastics from food", "value_for": "name"}}}
88	\N	1	92	\N	2020-09-28 12:22:40.324178	2020-09-28 12:22:40.324178	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 2, "type": "User", "value": "gonzalez.michael@epa.gov", "value_for": "name"}}}
89	\N	1	92	\N	2020-09-28 12:22:40.376536	2020-09-28 12:22:40.376536	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 3, "type": "User", "value": "venkatapathy.raghuraman@epa.gov", "value_for": "name"}}}
90	\N	1	92	\N	2020-09-28 12:22:40.762333	2020-09-28 12:22:40.762333	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 13, "type": "User", "value": "foster.albert@epa.gov", "value_for": "name"}}}
91	\N	1	92	\N	2020-09-28 12:22:40.92824	2020-09-28 12:22:40.92824	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 14, "type": "User", "value": "allen.derrick01@epa.gov", "value_for": "name"}}}
92	\N	1	27	\N	2020-09-28 12:25:52.906642	2020-09-28 12:25:52.906642	32	\N	Experiment	31	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "experiment": "FT-IR Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "experiment": {"id": 31, "type": "Experiment", "value": "FT-IR Analysis", "value_for": "name"}}}
93	\N	1	3	\N	2020-09-28 12:27:43.988113	2020-09-28 12:27:43.988113	21	\N	Project	21	1	\N	{"breadcrumbs": {"team": "My projects", "project": "AOP"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "project": {"id": 21, "type": "Project", "value": "AOP", "value_for": "name"}}}
94	\N	1	1	\N	2020-09-28 12:28:01.376751	2020-09-28 12:28:01.376751	1	\N	Project	1	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Demo project by SciNote"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "project": {"id": 1, "type": "Project", "value": "Demo project by SciNote", "value_for": "name"}}}
95	\N	1	92	\N	2020-09-28 13:00:11.641113	2020-09-28 13:00:11.641113	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 2, "type": "User", "value": "gonzalez.michael@epa.gov", "value_for": "name"}}}
96	\N	1	92	\N	2020-09-28 13:00:12.033815	2020-09-28 13:00:12.033815	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 15, "type": "User", "value": "allen01.derrick@epa.gov", "value_for": "name"}}}
100	\N	1	27	\N	2020-12-14 11:28:28.657289	2020-12-14 11:28:28.657289	32	\N	Experiment	36	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "experiment": "Scanning Electron Microscopy (SEM)"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "experiment": {"id": 36, "type": "Experiment", "value": "Scanning Electron Microscopy (SEM)", "value_for": "name"}}}
101	\N	1	27	\N	2020-12-14 11:29:53.057808	2020-12-14 11:29:53.057808	32	\N	Experiment	37	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "experiment": "Quantifying MPs using Stereomicroscopy"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "experiment": {"id": 37, "type": "Experiment", "value": "Quantifying MPs using Stereomicroscopy", "value_for": "name"}}}
102	\N	1	92	\N	2021-04-08 10:19:49.375534	2021-04-08 10:19:49.375534	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 18, "type": "User", "value": "shan.yonggui@epa.gov", "value_for": "name"}}}
103	\N	1	92	\N	2021-04-08 10:20:34.302431	2021-04-08 10:20:34.302431	\N	\N	Team	1	1	\N	{"breadcrumbs": {"team": "My projects"}, "message_items": {"role": "Normal user", "team": {"id": 1, "type": "Team", "value": "My projects", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_invited": {"id": 19, "type": "User", "value": "meyer.david@epa.gov", "value_for": "name"}}}
104	\N	4	28	\N	2021-04-08 13:14:30.97572	2021-04-08 13:14:30.97572	25	\N	Experiment	24	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 24, "type": "Experiment", "value": "Autoclave Timing Check", "value_for": "name"}}}
105	\N	4	8	\N	2021-04-08 13:20:10.139492	2021-04-08 13:20:10.139492	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "March 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "March 2021 - Quarterly Timing Check", "value_for": "name"}}}
106	\N	4	8	\N	2021-04-08 13:20:10.142466	2021-04-08 13:20:10.142466	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "June 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "June 2021 - Quarterly Timing Check", "value_for": "name"}}}
107	\N	4	8	\N	2021-04-08 13:20:10.145092	2021-04-08 13:20:10.145092	25	\N	MyModule	135	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "October 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 135, "type": "MyModule", "value": "October 2021 - Quarterly Timing Check", "value_for": "name"}}}
108	\N	4	8	\N	2021-04-08 13:20:10.14838	2021-04-08 13:20:10.14838	25	\N	MyModule	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "January 2022 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 136, "type": "MyModule", "value": "January 2022 - Quarterly Timing Check", "value_for": "name"}}}
109	\N	4	61	\N	2021-04-08 13:20:22.774034	2021-04-08 13:20:22.774034	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "March 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "March 2021 - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1617840000}}}
142	\N	4	80	\N	2021-04-08 14:50:47.237903	2021-04-08 14:50:47.237903	\N	\N	Protocol	143	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "SOP 2338.15 - Conductivity"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 143, "type": "Protocol", "value": "SOP 2338.15 - Conductivity", "value_for": "name"}}}
110	\N	4	61	\N	2021-04-08 13:20:36.046444	2021-04-08 13:20:36.046444	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "June 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "June 2021 - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1625011200}}}
111	\N	4	61	\N	2021-04-08 13:20:48.363769	2021-04-08 13:20:48.363769	25	\N	MyModule	135	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "October 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 135, "type": "MyModule", "value": "October 2021 - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1635552000}}}
112	\N	4	62	\N	2021-04-08 13:22:22.56644	2021-04-08 13:22:22.56644	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "July 2021 - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "July 2021 - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1627603200}}}
114	\N	4	62	\N	2021-04-08 13:23:32.181731	2021-04-08 13:23:32.181731	25	\N	MyModule	135	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 October - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 135, "type": "MyModule", "value": "2021 October - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1635465600}}}
115	\N	4	62	\N	2021-04-08 13:23:56.750873	2021-04-08 13:23:56.750873	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1619740800}}}
125	\N	4	15	\N	2021-04-08 13:33:57.750398	2021-04-08 13:33:57.750398	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"step": {"id": 563, "type": "Step", "value": "Autoclave Quarterly Timing Check", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "step_position": {"id": 563, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
113	\N	4	61	\N	2021-04-08 13:23:04.859838	2021-04-08 13:23:04.859838	25	\N	MyModule	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2022 January - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 136, "type": "MyModule", "value": "2022 January - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1643587200}}}
116	\N	4	13	\N	2021-04-08 13:24:40.513362	2021-04-08 13:24:40.513362	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "name"}}}
117	\N	4	13	\N	2021-04-08 13:24:44.307034	2021-04-08 13:24:44.307034	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
118	\N	4	13	\N	2021-04-08 13:24:59.322154	2021-04-08 13:24:59.322154	25	\N	MyModule	135	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 October - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 135, "type": "MyModule", "value": "2021 October - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "name"}}}
119	\N	4	13	\N	2021-04-08 13:25:02.471819	2021-04-08 13:25:02.471819	25	\N	MyModule	135	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 October - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 135, "type": "MyModule", "value": "2021 October - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
161	\N	1	3	\N	2021-05-19 14:32:00.154499	2021-05-19 14:32:00.154499	36	\N	Project	36	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Test"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "project": {"id": 36, "type": "Project", "value": "Test", "value_for": "name"}}}
120	\N	4	13	\N	2021-04-08 13:25:13.759512	2021-04-08 13:25:13.759512	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 July - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "2021 July - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "name"}}}
121	\N	4	13	\N	2021-04-08 13:25:16.474333	2021-04-08 13:25:16.474333	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 July - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "2021 July - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
122	\N	4	13	\N	2021-04-08 13:27:29.277109	2021-04-08 13:27:29.277109	25	\N	MyModule	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2022 January - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 136, "type": "MyModule", "value": "2022 January - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "name"}}}
123	\N	4	13	\N	2021-04-08 13:27:32.004784	2021-04-08 13:27:32.004784	25	\N	MyModule	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2022 January - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 136, "type": "MyModule", "value": "2022 January - Quarterly Timing Check", "value_for": "name"}, "user_target": {"id": 10, "type": "User", "value": "Laura Webb", "value_for": "name"}}}
124	\N	4	35	\N	2021-04-08 13:28:21.985329	2021-04-08 13:28:21.985329	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}}}
126	\N	4	18	\N	2021-04-08 14:00:35.666266	2021-04-08 14:00:35.666266	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"step": {"id": 563, "type": "Step", "value": "Autoclave Quarterly Timing Check", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "num_all": "1", "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "num_completed": "1", "step_position": {"id": 563, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
127	\N	4	16	\N	2021-04-08 14:04:56.730062	2021-04-08 14:04:56.730062	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"step": {"id": 563, "type": "Step", "value": "Autoclave Quarterly Timing Check", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "step_position": {"id": 563, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
128	\N	4	15	\N	2021-04-08 14:16:51.266629	2021-04-08 14:16:51.266629	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"step": {"id": 564, "type": "Step", "value": "Autoclave Quarterly Timing Checks - SOP 2338.11", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "step_position": {"id": 564, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
129	\N	4	22	\N	2021-04-08 14:17:11.664042	2021-04-08 14:17:11.664042	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"step": {"id": 564, "type": "Step", "value": "Autoclave Quarterly Timing Checks - SOP 2338.11", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "step_position": {"id": 564, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
130	\N	4	62	\N	2021-04-08 14:18:02.710541	2021-04-08 14:18:02.710541	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 April - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "2021 April - Quarterly Timing Check", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1619654400}}}
131	\N	4	53	\N	2021-04-08 14:19:03.575673	2021-04-08 14:19:03.575673	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 July - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "2021 July - Quarterly Timing Check", "value_for": "name"}}}
176	\N	24	3	\N	2021-05-20 14:36:22.076983	2021-05-20 14:36:22.076983	41	\N	Project	41	19	\N	{"breadcrumbs": {"team": "MPham", "project": "[NEW] Demo project by SciNote"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "project": {"id": 41, "type": "Project", "value": "[NEW] Demo project by SciNote", "value_for": "name"}}}
132	\N	4	54	\N	2021-04-08 14:19:46.783246	2021-04-08 14:19:46.783246	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 July - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "2021 July - Quarterly Timing Check", "value_for": "name"}}}
133	\N	4	10	\N	2021-04-08 14:20:25.331096	2021-04-08 14:20:25.331096	25	\N	MyModule	134	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 July - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 134, "type": "MyModule", "value": "2021 July - Quarterly Timing Check", "value_for": "name"}}}
134	\N	4	10	\N	2021-04-08 14:20:25.340579	2021-04-08 14:20:25.340579	25	\N	MyModule	135	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2021 October - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 135, "type": "MyModule", "value": "2021 October - Quarterly Timing Check", "value_for": "name"}}}
150	\N	4	57	\N	2021-04-08 15:37:22.4778	2021-04-08 15:37:22.4778	25	\N	Experiment	23	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 23, "type": "Experiment", "value": "Autoclave Sterilization Check", "value_for": "name"}}}
135	\N	4	10	\N	2021-04-08 14:20:25.34846	2021-04-08 14:20:25.34846	25	\N	MyModule	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "2022 January - Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 136, "type": "MyModule", "value": "2022 January - Quarterly Timing Check", "value_for": "name"}}}
136	\N	4	18	\N	2021-04-08 14:20:50.098476	2021-04-08 14:20:50.098476	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"step": {"id": 564, "type": "Step", "value": "Autoclave Quarterly Timing Checks - SOP 2338.11", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "num_all": "1", "my_module": {"id": 133, "type": "MyModule", "value": "Quarterly Timing Check", "value_for": "name"}, "num_completed": "1", "step_position": {"id": 564, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
137	\N	4	53	\N	2021-04-08 14:21:00.024926	2021-04-08 14:21:00.024926	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "Quarterly Timing Check", "value_for": "name"}}}
138	\N	4	54	\N	2021-04-08 14:21:13.346685	2021-04-08 14:21:13.346685	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "Quarterly Timing Check", "value_for": "name"}}}
139	\N	4	80	\N	2021-04-08 14:42:18.474532	2021-04-08 14:42:18.474532	\N	\N	Protocol	140	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "Biological Hygiene Plan"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 140, "type": "Protocol", "value": "Biological Hygiene Plan", "value_for": "name"}}}
140	\N	4	80	\N	2021-04-08 14:44:48.028864	2021-04-08 14:44:48.028864	\N	\N	Protocol	141	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "Chlorine in Water by LaMotte TRACER Pocket Tester"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 141, "type": "Protocol", "value": "Chlorine in Water by LaMotte TRACER Pocket Tester", "value_for": "name"}}}
141	\N	4	80	\N	2021-04-08 14:49:12.244184	2021-04-08 14:49:12.244184	\N	\N	Protocol	142	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "SOP 2338.11 - Steam Sterilization (Autoclave) and Decontamination"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 142, "type": "Protocol", "value": "SOP 2338.11 - Steam Sterilization (Autoclave) and Decontamination", "value_for": "name"}}}
143	\N	4	80	\N	2021-04-08 14:52:33.274129	2021-04-08 14:52:33.274129	\N	\N	Protocol	144	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "protocol": "SOP 2338.09 - UV Light Intensity Checks"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 144, "type": "Protocol", "value": "SOP 2338.09 - UV Light Intensity Checks", "value_for": "name"}}}
144	\N	4	46	\N	2021-04-08 15:10:34.11248	2021-04-08 15:10:34.11248	25	\N	Protocol	136	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 136, "type": "Protocol", "value": "", "value_for": "name"}, "my_module": {"id": 133, "type": "MyModule", "value": "Quarterly Timing Check", "value_for": "name"}, "protocol_repository": {"id": 142, "type": "Protocol", "value": "SOP 2338.11 - Steam Sterilization (Autoclave) and Decontamination", "value_for": "name"}}}
145	\N	4	64	\N	2021-04-08 15:12:23.271912	2021-04-08 15:12:23.271912	25	\N	MyModule	133	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"tag": {"id": 73, "type": "Tag", "value": "COMPLETED - Drain and Flush Prep procedure (prior to use)", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 133, "type": "MyModule", "value": "Quarterly Timing Check", "value_for": "name"}}}
146	\N	4	23	\N	2021-04-08 15:16:39.350654	2021-04-08 15:16:39.350654	25	\N	Result	295	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "result": "SOP 2338.11 - Quarterly Timing Checks - Attachment 3", "project": "Micro QC", "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "result": {"id": 295, "type": "Result", "value": "SOP 2338.11 - Quarterly Timing Checks - Attachment 3", "value_for": "name"}, "type_of_result": "file"}}
147	\N	4	25	\N	2021-04-08 15:25:41.697463	2021-04-08 15:25:41.697463	25	\N	Result	295	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "result": "SOP 2338.11 - Quarterly Timing Checks - Attachment 3", "project": "Micro QC", "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "result": {"id": 295, "type": "Result", "value": "SOP 2338.11 - Quarterly Timing Checks - Attachment 3", "value_for": "name"}, "type_of_result": "file"}}
148	\N	4	23	\N	2021-04-08 15:35:45.571587	2021-04-08 15:35:45.571587	25	\N	Result	296	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "result": "SOP 2338.11 - Attachment 3 - Quaterly Timing Checks", "project": "Micro QC", "my_module": "Quarterly Timing Check", "experiment": "Autoclave Timing Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "result": {"id": 296, "type": "Result", "value": "SOP 2338.11 - Attachment 3 - Quaterly Timing Checks", "value_for": "name"}, "type_of_result": "text"}}
149	\N	4	29	\N	2021-04-08 15:37:07.543172	2021-04-08 15:37:07.543172	25	\N	Experiment	23	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "experiment": {"id": 23, "type": "Experiment", "value": "Autoclave Sterilization Check", "value_for": "name"}}}
151	\N	4	54	\N	2021-04-08 15:39:45.565771	2021-04-08 15:39:45.565771	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}}}
152	\N	4	46	\N	2021-04-08 15:40:14.341812	2021-04-08 15:40:14.341812	25	\N	Protocol	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "protocol": null, "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "protocol": {"id": 74, "type": "Protocol", "value": "", "value_for": "name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}, "protocol_repository": {"id": 142, "type": "Protocol", "value": "SOP 2338.11 - Steam Sterilization (Autoclave) and Decontamination", "value_for": "name"}}}
153	\N	4	23	\N	2021-04-08 15:45:39.138917	2021-04-08 15:45:39.138917	25	\N	Result	297	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "result": "SOP 2338.11 - Attachment 4 - Generator Flush Log", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "result": {"id": 297, "type": "Result", "value": "SOP 2338.11 - Attachment 4 - Generator Flush Log", "value_for": "name"}, "type_of_result": "text"}}
154	\N	4	63	\N	2021-04-08 15:46:11.274274	2021-04-08 15:46:11.274274	25	\N	MyModule	74	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Prep Autoclave", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 74, "type": "MyModule", "value": "Prep Autoclave", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1574640000}}}
155	\N	4	61	\N	2021-04-08 15:46:31.348573	2021-04-08 15:46:31.348573	25	\N	MyModule	75	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Run 15 minute sterilization cycle with Sportrol Ampule", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 75, "type": "MyModule", "value": "Run 15 minute sterilization cycle with Sportrol Ampule", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1619740800}}}
156	\N	4	63	\N	2021-04-08 15:46:42.630223	2021-04-08 15:46:42.630223	25	\N	MyModule	75	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Micro QC", "my_module": "Run 15 minute sterilization cycle with Sportrol Ampule", "experiment": "Autoclave Sterilization Check"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "my_module": {"id": 75, "type": "MyModule", "value": "Run 15 minute sterilization cycle with Sportrol Ampule", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1619740800}}}
157	\N	4	3	\N	2021-04-08 15:52:53.98717	2021-04-08 15:52:53.98717	30	\N	Project	30	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "project": "Total Coliforms and E. coli in Water"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "project": {"id": 30, "type": "Project", "value": "Total Coliforms and E. coli in Water", "value_for": "name"}}}
158	\N	4	93	\N	2021-04-08 15:53:54.132901	2021-04-08 15:53:54.132901	\N	\N	Team	7	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab"}, "message_items": {"team": {"id": 7, "type": "Team", "value": "R7-Microbiology Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_removed": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
159	\N	4	93	\N	2021-04-08 15:54:38.803449	2021-04-08 15:54:38.803449	\N	\N	Team	3	3	\N	{"breadcrumbs": {"team": "R7-Molecular Lab"}, "message_items": {"team": {"id": 3, "type": "Team", "value": "R7-Molecular Lab", "value_for": "name"}, "user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "user_removed": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "name"}}}
162	\N	1	8	\N	2021-05-20 10:26:02.284208	2021-05-20 10:26:02.284208	32	\N	MyModule	146	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Screening Microplastics", "experiment": "Quantifying MPs using Stereomicroscopy"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 146, "type": "MyModule", "value": "Screening Microplastics", "value_for": "name"}}}
163	\N	1	8	\N	2021-05-20 10:26:02.287517	2021-05-20 10:26:02.287517	32	\N	MyModule	147	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Sample Preparation", "experiment": "Quantifying MPs using Stereomicroscopy"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 147, "type": "MyModule", "value": "Sample Preparation", "value_for": "name"}}}
164	\N	1	8	\N	2021-05-20 10:31:30.190504	2021-05-20 10:31:30.190504	32	\N	MyModule	148	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Food depackaging facilities sampled", "experiment": "FT-IR Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 148, "type": "MyModule", "value": "Food depackaging facilities sampled", "value_for": "name"}}}
165	\N	1	8	\N	2021-05-20 10:31:30.19313	2021-05-20 10:31:30.19313	32	\N	MyModule	149	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Food waste from the sources identified", "experiment": "FT-IR Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 149, "type": "MyModule", "value": "Food waste from the sources identified", "value_for": "name"}}}
166	\N	1	8	\N	2021-05-20 10:31:30.195649	2021-05-20 10:31:30.195649	32	\N	MyModule	150	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "100g of material", "experiment": "FT-IR Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 150, "type": "MyModule", "value": "100g of material", "value_for": "name"}}}
167	\N	1	8	\N	2021-05-20 10:31:30.198081	2021-05-20 10:31:30.198081	32	\N	MyModule	151	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "moisture content of the bulk waste material will be determined by drying in an oven at 105 C.", "experiment": "FT-IR Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 151, "type": "MyModule", "value": "moisture content of the bulk waste material will be determined by drying in an oven at 105 C.", "value_for": "name"}}}
168	\N	1	8	\N	2021-05-20 10:31:30.201929	2021-05-20 10:31:30.201929	32	\N	MyModule	152	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Quantitative analysis of Micro-plastics will be performed using Fourier transformation infrared spectroscopy/microscopy (FTIR/μFTIR).", "experiment": "FT-IR Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 152, "type": "MyModule", "value": "Quantitative analysis of Micro-plastics will be performed using Fourier transformation infrared spectroscopy/microscopy (FTIR/μFTIR).", "value_for": "name"}}}
169	\N	1	94	\N	2021-05-20 13:02:21.925408	2021-05-20 13:02:21.925408	\N	\N	Team	12	12	\N	{"breadcrumbs": {"team": "Evaluating de-packaging technologies to remove film plastics from food"}, "message_items": {"role": "Administrator", "team": {"id": 12, "type": "Team", "value": "Evaluating de-packaging technologies to remove film plastics from food", "value_for": "name"}, "user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "user_changed": {"id": 3, "type": "User", "value": "venkatapathy.raghuraman@epa.gov", "value_for": "name"}}}
170	\N	1	8	\N	2021-05-20 13:07:49.53406	2021-05-20 13:07:49.53406	32	\N	MyModule	153	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Sample", "experiment": "Raman"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 153, "type": "MyModule", "value": "Sample", "value_for": "name"}}}
171	\N	1	8	\N	2021-05-20 13:07:49.536679	2021-05-20 13:07:49.536679	32	\N	MyModule	154	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "my_module": "Prep", "experiment": "Raman"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "my_module": {"id": 154, "type": "MyModule", "value": "Prep", "value_for": "name"}}}
172	\N	24	0	\N	2021-05-20 14:22:42.029797	2021-05-20 14:22:42.029797	42	\N	Project	42	19	\N	{"breadcrumbs": {"team": "MPham", "project": "WID TOC Analyses"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "project": {"id": 42, "type": "Project", "value": "WID TOC Analyses", "value_for": "name"}}}
173	\N	24	27	\N	2021-05-20 14:23:28.644061	2021-05-20 14:23:28.644061	42	\N	Experiment	43	19	\N	{"breadcrumbs": {"team": "MPham", "project": "WID TOC Analyses", "experiment": "TOC"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "experiment": {"id": 43, "type": "Experiment", "value": "TOC", "value_for": "name"}}}
174	\N	1	27	\N	2021-05-20 14:26:58.313836	2021-05-20 14:26:58.313836	32	\N	Experiment	44	1	\N	{"breadcrumbs": {"team": "My projects", "project": "Evaluating de-packaging technologies to remove film plastics from food", "experiment": "Sample Analysis"}, "message_items": {"user": {"id": 1, "type": "User", "value": "Admin", "value_for": "full_name"}, "experiment": {"id": 44, "type": "Experiment", "value": "Sample Analysis", "value_for": "name"}}}
175	\N	24	80	\N	2021-05-20 14:34:33.218005	2021-05-20 14:34:33.218005	\N	\N	Protocol	208	19	\N	{"breadcrumbs": {"team": "MPham", "protocol": "test"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "protocol": {"id": 208, "type": "Protocol", "value": "test", "value_for": "name"}}}
177	\N	24	28	\N	2021-05-20 14:40:16.13208	2021-05-20 14:40:16.13208	42	\N	Experiment	43	19	\N	{"breadcrumbs": {"team": "MPham", "project": "WID TOC Analyses", "experiment": "TOC"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "experiment": {"id": 43, "type": "Experiment", "value": "TOC", "value_for": "name"}}}
178	\N	24	81	\N	2021-05-20 14:41:37.262358	2021-05-20 14:41:37.262358	\N	\N	Protocol	208	19	\N	{"breadcrumbs": {"team": "MPham", "protocol": "TOC Solutions"}, "message_items": {"step": {"id": 805, "type": "Step", "value": "TOC Stock Standards", "value_for": "name"}, "user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "protocol": {"id": 208, "type": "Protocol", "value": "TOC Solutions", "value_for": "name"}, "step_position": {"id": 805, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
179	\N	24	4	\N	2021-05-20 14:44:13.50207	2021-05-20 14:44:13.50207	41	\N	Project	41	19	\N	{"breadcrumbs": {"team": "MPham", "project": "[NEW] Demo project by SciNote"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "project": {"id": 41, "type": "Project", "value": "[NEW] Demo project by SciNote", "value_for": "name"}}}
180	\N	24	11	\N	2021-05-20 14:46:18.808285	2021-05-20 14:46:18.808285	41	\N	MyModule	190	19	\N	{"breadcrumbs": {"team": "MPham", "project": "[NEW] Demo project by SciNote", "my_module": "Data analysis - Pfaffl method", "experiment": "Polymerase chain reaction"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "my_module": {"id": 190, "type": "MyModule", "value": "Data analysis - Pfaffl method", "value_for": "name"}}}
195	\N	21	80	\N	2021-05-27 17:27:16.701186	2021-05-27 17:27:16.701186	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}}}
181	\N	24	8	\N	2021-05-20 15:31:37.570346	2021-05-20 15:31:37.570346	42	\N	MyModule	209	19	\N	{"breadcrumbs": {"team": "MPham", "project": "WID TOC Analyses", "my_module": "Prepare Calibration Standards & QCs", "experiment": "TOC"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "my_module": {"id": 209, "type": "MyModule", "value": "Prepare Calibration Standards & QCs", "value_for": "name"}}}
182	\N	24	8	\N	2021-05-20 15:31:37.573414	2021-05-20 15:31:37.573414	42	\N	MyModule	210	19	\N	{"breadcrumbs": {"team": "MPham", "project": "WID TOC Analyses", "my_module": "Sample Run Log", "experiment": "TOC"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "my_module": {"id": 210, "type": "MyModule", "value": "Sample Run Log", "value_for": "name"}}}
183	\N	24	8	\N	2021-05-20 15:31:37.576283	2021-05-20 15:31:37.576283	42	\N	MyModule	211	19	\N	{"breadcrumbs": {"team": "MPham", "project": "WID TOC Analyses", "my_module": "Instrument Start up", "experiment": "TOC"}, "message_items": {"user": {"id": 24, "type": "User", "value": "Maily Pham", "value_for": "full_name"}, "my_module": {"id": 211, "type": "MyModule", "value": "Instrument Start up", "value_for": "name"}}}
184	\N	21	0	\N	2021-05-20 15:31:53.785556	2021-05-20 15:31:53.785556	45	\N	Project	45	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "project": "Sensors project"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "project": {"id": 45, "type": "Project", "value": "Sensors project", "value_for": "name"}}}
185	\N	22	27	\N	2021-05-20 15:42:01.158135	2021-05-20 15:42:01.158135	39	\N	Experiment	47	17	\N	{"breadcrumbs": {"team": "Materials Management Lab", "project": "[NEW] Demo project by SciNote", "experiment": "Alaska BMP"}, "message_items": {"user": {"id": 22, "type": "User", "value": "Max Krause", "value_for": "full_name"}, "experiment": {"id": 47, "type": "Experiment", "value": "Alaska BMP", "value_for": "name"}}}
186	\N	21	80	\N	2021-05-20 15:51:51.598131	2021-05-20 15:51:51.598131	\N	\N	Protocol	221	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Calibration"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 221, "type": "Protocol", "value": "WIZ Calibration", "value_for": "name"}}}
187	\N	21	27	\N	2021-05-26 12:40:04.956958	2021-05-26 12:40:04.956958	45	\N	Experiment	48	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "project": "Sensors project", "experiment": "WIZ Sensor at Lake Harsha"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "experiment": {"id": 48, "type": "Experiment", "value": "WIZ Sensor at Lake Harsha", "value_for": "name"}}}
188	\N	21	80	\N	2021-05-26 15:16:55.515911	2021-05-26 15:16:55.515911	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}}}
189	\N	21	81	\N	2021-05-26 16:37:19.615446	2021-05-26 16:37:19.615446	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 846, "type": "Step", "value": "Pre-preparation:  Glassware and ancillary material", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 846, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
203	\N	27	0	\N	2021-05-28 15:09:19.40237	2021-05-28 15:09:19.40237	48	\N	Project	48	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 48, "type": "Project", "value": "P Recovery", "value_for": "name"}}}
205	\N	27	0	\N	2021-05-28 15:09:46.709345	2021-05-28 15:09:46.709345	50	\N	Project	50	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "GAC for CFF concentrate"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 50, "type": "Project", "value": "GAC for CFF concentrate", "value_for": "name"}}}
190	\N	21	82	\N	2021-05-27 13:52:25.255557	2021-05-27 13:52:25.255557	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 846, "type": "Step", "value": "Pre-preparation:  Glassware and ancillary material", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 846, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
191	\N	21	81	\N	2021-05-27 14:50:14.906913	2021-05-27 14:50:14.906913	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 847, "type": "Step", "value": "Chemicals used for reagent preparation", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 847, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
192	\N	21	81	\N	2021-05-27 14:51:11.914398	2021-05-27 14:51:11.914398	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 848, "type": "Step", "value": "Reagent Preparation Day 1", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 848, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
193	\N	21	82	\N	2021-05-27 16:52:20.347376	2021-05-27 16:52:20.347376	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 848, "type": "Step", "value": "Reagent Preparation Day 1", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 848, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
194	\N	21	81	\N	2021-05-27 17:00:12.596854	2021-05-27 17:00:12.596854	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 849, "type": "Step", "value": "Field Day Reagent Preparation", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 849, "type": "Step", "value": "4", "value_for": "position_plus_one"}}}
196	\N	21	81	\N	2021-05-27 17:54:48.075262	2021-05-27 17:54:48.075262	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Field protocol for WIZ re-calibration and reagent replacement", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
197	\N	21	82	\N	2021-05-27 17:55:12.080837	2021-05-27 17:55:12.080837	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Field protocol for WIZ re-calibration and reagent replacement", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
198	\N	21	82	\N	2021-05-27 19:19:27.700175	2021-05-27 19:19:27.700175	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Prepare reagents and equipment", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
199	\N	21	82	\N	2021-05-27 19:36:07.727864	2021-05-27 19:36:07.727864	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 848, "type": "Step", "value": "Reagent Preparation Day 1", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 848, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
200	\N	21	82	\N	2021-05-27 19:55:55.539758	2021-05-27 19:55:55.539758	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 849, "type": "Step", "value": "Field Day Reagent Preparation", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 849, "type": "Step", "value": "4", "value_for": "position_plus_one"}}}
201	\N	21	82	\N	2021-05-27 19:58:35.406619	2021-05-27 19:58:35.406619	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 846, "type": "Step", "value": "Pre-preparation:  Glassware and ancillary material", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 846, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
202	\N	27	0	\N	2021-05-28 15:08:46.677267	2021-05-28 15:08:46.677267	47	\N	Project	47	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "Membrane distillation"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 47, "type": "Project", "value": "Membrane distillation", "value_for": "name"}}}
206	\N	27	0	\N	2021-05-28 15:10:03.297006	2021-05-28 15:10:03.297006	51	\N	Project	51	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "AOX for CFF concentrate discharge"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 51, "type": "Project", "value": "AOX for CFF concentrate discharge", "value_for": "name"}}}
207	\N	27	0	\N	2021-05-28 15:10:11.224864	2021-05-28 15:10:11.224864	52	\N	Project	52	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "Pb filter recovery"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 52, "type": "Project", "value": "Pb filter recovery", "value_for": "name"}}}
208	\N	27	0	\N	2021-05-28 15:10:21.238102	2021-05-28 15:10:21.238102	53	\N	Project	53	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "CO2 separation using membrane"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "project": {"id": 53, "type": "Project", "value": "CO2 separation using membrane", "value_for": "name"}}}
209	\N	27	27	\N	2021-06-01 13:42:09.327065	2021-06-01 13:42:09.327065	48	\N	Experiment	50	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "experiment": "20210528"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "experiment": {"id": 50, "type": "Experiment", "value": "20210528", "value_for": "name"}}}
210	\N	27	28	\N	2021-06-01 13:46:29.159171	2021-06-01 13:46:29.159171	48	\N	Experiment	50	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "experiment": {"id": 50, "type": "Experiment", "value": "20210528 PO4 adsorption test (two samples)", "value_for": "name"}}}
211	\N	21	82	\N	2021-06-01 15:53:37.034255	2021-06-01 15:53:37.034255	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 846, "type": "Step", "value": "Pre-preparation:  Glassware and ancillary material", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 846, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
225	\N	27	8	\N	2021-06-10 16:07:24.243782	2021-06-10 16:07:24.243782	48	\N	MyModule	232	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Run adsorption test", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 232, "type": "MyModule", "value": "Run adsorption test", "value_for": "name"}}}
212	\N	21	82	\N	2021-06-01 16:06:50.979251	2021-06-01 16:06:50.979251	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Prepare reagents and equipment", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
213	\N	21	82	\N	2021-06-01 16:07:40.484037	2021-06-01 16:07:40.484037	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Prepare reagents and equipment", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
214	\N	21	82	\N	2021-06-01 16:11:36.162715	2021-06-01 16:11:36.162715	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Prepare reagents and equipment", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
215	\N	21	81	\N	2021-06-01 16:23:59.382613	2021-06-01 16:23:59.382613	\N	\N	Protocol	221	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Calibration"}, "message_items": {"step": {"id": 891, "type": "Step", "value": "Travel to site.", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 221, "type": "Protocol", "value": "WIZ Calibration", "value_for": "name"}, "step_position": {"id": 891, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
216	\N	21	81	\N	2021-06-01 16:51:11.837313	2021-06-01 16:51:11.837313	\N	\N	Protocol	221	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Calibration"}, "message_items": {"step": {"id": 892, "type": "Step", "value": "Site Entry", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 221, "type": "Protocol", "value": "WIZ Calibration", "value_for": "name"}, "step_position": {"id": 892, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
217	\N	21	82	\N	2021-06-01 16:59:33.365119	2021-06-01 16:59:33.365119	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 850, "type": "Step", "value": "Prepare reagents and equipment", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 850, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
262	\N	27	36	\N	2021-07-13 16:24:14.196092	2021-07-13 16:24:14.196092	48	\N	MyModule	239	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product A, Alumina 2mm", "experiment": "20210713 Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 239, "type": "MyModule", "value": "Test Product A, Alumina 2mm", "value_for": "name"}}}
218	\N	21	81	\N	2021-06-01 17:01:34.144481	2021-06-01 17:01:34.144481	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 893, "type": "Step", "value": "Travel to WIZ location", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 893, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
219	\N	21	82	\N	2021-06-01 17:13:31.940716	2021-06-01 17:13:31.940716	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 893, "type": "Step", "value": "Travel to WIZ location", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 893, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
220	\N	21	81	\N	2021-06-01 17:17:54.285719	2021-06-01 17:17:54.285719	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 894, "type": "Step", "value": "WIZ calibration and reagent replacement", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 894, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
221	\N	21	82	\N	2021-06-01 17:22:34.689603	2021-06-01 17:22:34.689603	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 894, "type": "Step", "value": "WIZ calibration and reagent replacement", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 894, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
222	\N	21	82	\N	2021-06-01 19:55:02.727956	2021-06-01 19:55:02.727956	\N	\N	Protocol	223	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Field Calibration Procedures"}, "message_items": {"step": {"id": 894, "type": "Step", "value": "WIZ calibration and reagent replacement", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 223, "type": "Protocol", "value": "WIZ Field Calibration Procedures", "value_for": "name"}, "step_position": {"id": 894, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
223	\N	27	8	\N	2021-06-10 16:07:24.230628	2021-06-10 16:07:24.230628	48	\N	MyModule	230	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive new samples (06/10/2021)", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 230, "type": "MyModule", "value": "Receive new samples (06/10/2021)", "value_for": "name"}}}
226	\N	27	8	\N	2021-06-10 16:07:24.24676	2021-06-10 16:07:24.24676	48	\N	MyModule	233	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Collect samples at t = 1,2,3,24 h", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 233, "type": "MyModule", "value": "Collect samples at t = 1,2,3,24 h", "value_for": "name"}}}
227	\N	27	8	\N	2021-06-10 16:07:24.249292	2021-06-10 16:07:24.249292	48	\N	MyModule	234	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Analyze samples using Spectraphotometer", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 234, "type": "MyModule", "value": "Analyze samples using Spectraphotometer", "value_for": "name"}}}
228	\N	27	35	\N	2021-06-10 16:08:13.687112	2021-06-10 16:08:13.687112	48	\N	MyModule	230	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive new samples (06/10/2021)", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 230, "type": "MyModule", "value": "Receive new samples (06/10/2021)", "value_for": "name"}}}
229	\N	27	13	\N	2021-06-10 16:08:40.609021	2021-06-10 16:08:40.609021	48	\N	MyModule	230	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive new samples (06/10/2021)", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 230, "type": "MyModule", "value": "Receive new samples (06/10/2021)", "value_for": "name"}, "user_target": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "name"}}}
230	\N	27	92	\N	2021-06-10 16:09:28.312957	2021-06-10 16:09:28.312957	\N	\N	Team	22	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab"}, "message_items": {"role": "Normal user", "team": {"id": 22, "type": "Team", "value": "TLee ' 's lab", "value_for": "name"}, "user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "user_invited": {"id": 29, "type": "User", "value": "nadagouda.mallikarjuna@epa.gov", "value_for": "name"}}}
231	\N	27	61	\N	2021-06-10 16:10:32.520085	2021-06-10 16:10:32.520085	48	\N	MyModule	230	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive new samples (06/10/2021)", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 230, "type": "MyModule", "value": "Receive new samples (06/10/2021)", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1623283200}}}
232	\N	27	61	\N	2021-06-10 16:10:39.095548	2021-06-10 16:10:39.095548	48	\N	MyModule	231	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Bake each sample at 250 C", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 231, "type": "MyModule", "value": "Bake each sample at 250 C", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1623628800}}}
233	\N	27	53	\N	2021-06-10 18:45:11.131652	2021-06-10 18:45:11.131652	48	\N	MyModule	230	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive new samples (06/10/2021)", "experiment": "20210528 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 230, "type": "MyModule", "value": "Receive new samples (06/10/2021)", "value_for": "name"}}}
234	\N	27	28	\N	2021-06-10 18:47:16.215707	2021-06-10 18:47:16.215707	48	\N	Experiment	50	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "experiment": {"id": 50, "type": "Experiment", "value": "20210610 PO4 adsorption test (two samples)", "value_for": "name"}}}
235	\N	27	35	\N	2021-06-14 19:22:09.707851	2021-06-14 19:22:09.707851	48	\N	MyModule	231	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Bake each sample at 250 C", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 231, "type": "MyModule", "value": "Bake each sample at 250 C", "value_for": "name"}}}
236	\N	27	53	\N	2021-06-14 19:22:13.930415	2021-06-14 19:22:13.930415	48	\N	MyModule	231	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Bake each sample at 250 C", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 231, "type": "MyModule", "value": "Bake each sample at 250 C", "value_for": "name"}}}
237	\N	27	35	\N	2021-06-14 19:23:15.850347	2021-06-14 19:23:15.850347	48	\N	MyModule	232	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Run adsorption test", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 232, "type": "MyModule", "value": "Run adsorption test", "value_for": "name"}}}
238	\N	27	53	\N	2021-06-16 15:58:53.641425	2021-06-16 15:58:53.641425	48	\N	MyModule	233	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Collect samples at t = 1,2,3,24 h", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 233, "type": "MyModule", "value": "Collect samples at t = 1,2,3,24 h", "value_for": "name"}}}
239	\N	27	53	\N	2021-06-16 15:59:07.722158	2021-06-16 15:59:07.722158	48	\N	MyModule	232	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Run adsorption test", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 232, "type": "MyModule", "value": "Run adsorption test", "value_for": "name"}}}
240	\N	27	35	\N	2021-06-16 15:59:23.775077	2021-06-16 15:59:23.775077	48	\N	MyModule	234	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Analyze samples using Spectraphotometer", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 234, "type": "MyModule", "value": "Analyze samples using Spectraphotometer", "value_for": "name"}}}
241	\N	27	53	\N	2021-06-16 15:59:27.124212	2021-06-16 15:59:27.124212	48	\N	MyModule	234	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Analyze samples using Spectraphotometer", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 234, "type": "MyModule", "value": "Analyze samples using Spectraphotometer", "value_for": "name"}}}
242	\N	27	23	\N	2021-06-23 15:38:02.34173	2021-06-23 15:38:02.34173	48	\N	Result	487	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "result": "Capacity results", "project": "P Recovery", "my_module": "Analyze samples using Spectraphotometer", "experiment": "20210610 PO4 adsorption test (two samples)"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "result": {"id": 487, "type": "Result", "value": "Capacity results", "value_for": "name"}, "type_of_result": "text"}}
243	\N	21	80	\N	2021-06-24 17:21:36.592661	2021-06-24 17:21:36.592661	\N	\N	Protocol	247	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "WIZ Data collection and storage"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 247, "type": "Protocol", "value": "WIZ Data collection and storage", "value_for": "name"}}}
244	\N	27	27	\N	2021-07-06 15:48:25.938533	2021-07-06 15:48:25.938533	48	\N	Experiment	52	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "experiment": {"id": 52, "type": "Experiment", "value": "20210706 Alumina adsorbent mix for P adsorption", "value_for": "name"}}}
245	\N	27	8	\N	2021-07-06 15:50:12.427804	2021-07-06 15:50:12.427804	48	\N	MyModule	235	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive the new sample", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 235, "type": "MyModule", "value": "Receive the new sample", "value_for": "name"}}}
246	\N	27	8	\N	2021-07-06 15:50:12.432312	2021-07-06 15:50:12.432312	48	\N	MyModule	236	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Mix with 5% biochar", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 236, "type": "MyModule", "value": "Mix with 5% biochar", "value_for": "name"}}}
247	\N	27	8	\N	2021-07-06 15:50:12.435924	2021-07-06 15:50:12.435924	48	\N	MyModule	237	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Adsorption test", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 237, "type": "MyModule", "value": "Adsorption test", "value_for": "name"}}}
248	\N	27	8	\N	2021-07-06 15:50:12.438858	2021-07-06 15:50:12.438858	48	\N	MyModule	238	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Measure using spectrophotometer", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 238, "type": "MyModule", "value": "Measure using spectrophotometer", "value_for": "name"}}}
249	\N	27	35	\N	2021-07-06 15:50:55.512432	2021-07-06 15:50:55.512432	48	\N	MyModule	236	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Mix with 5% biochar", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 236, "type": "MyModule", "value": "Mix with 5% biochar", "value_for": "name"}}}
250	\N	27	13	\N	2021-07-06 15:51:27.751613	2021-07-06 15:51:27.751613	48	\N	MyModule	235	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive the new sample", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 235, "type": "MyModule", "value": "Receive the new sample", "value_for": "name"}, "user_target": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "name"}}}
251	\N	27	61	\N	2021-07-07 16:07:27.367455	2021-07-07 16:07:27.367455	48	\N	MyModule	235	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive the new sample", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 235, "type": "MyModule", "value": "Receive the new sample", "value_for": "name"}, "my_module_duedate": {"type": "Time", "value": 1625616000}}}
252	\N	27	53	\N	2021-07-07 16:07:45.733113	2021-07-07 16:07:45.733113	48	\N	MyModule	235	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Receive the new sample", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 235, "type": "MyModule", "value": "Receive the new sample", "value_for": "name"}}}
253	\N	27	53	\N	2021-07-07 16:08:20.476358	2021-07-07 16:08:20.476358	48	\N	MyModule	236	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Mix with 5% biochar", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 236, "type": "MyModule", "value": "Mix with 5% biochar", "value_for": "name"}}}
254	\N	27	53	\N	2021-07-07 16:08:45.101315	2021-07-07 16:08:45.101315	48	\N	MyModule	237	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Adsorption test", "experiment": "20210706 Alumina adsorbent mix for P adsorption"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 237, "type": "MyModule", "value": "Adsorption test", "value_for": "name"}}}
255	\N	27	27	\N	2021-07-12 18:03:35.86116	2021-07-12 18:03:35.86116	48	\N	Experiment	53	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "experiment": "Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "experiment": {"id": 53, "type": "Experiment", "value": "Product A,B,C and Alumina 1, 2,3mm", "value_for": "name"}}}
256	\N	27	8	\N	2021-07-12 18:41:37.117424	2021-07-12 18:41:37.117424	48	\N	MyModule	239	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product A, Alumina 2mm", "experiment": "Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 239, "type": "MyModule", "value": "Test Product A, Alumina 2mm", "value_for": "name"}}}
257	\N	27	35	\N	2021-07-12 18:44:01.757486	2021-07-12 18:44:01.757486	48	\N	MyModule	239	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product A, Alumina 2mm", "experiment": "Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 239, "type": "MyModule", "value": "Test Product A, Alumina 2mm", "value_for": "name"}}}
258	\N	27	8	\N	2021-07-12 18:44:37.279711	2021-07-12 18:44:37.279711	48	\N	MyModule	240	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product B, Alumina 2mm", "experiment": "Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 240, "type": "MyModule", "value": "Test Product B, Alumina 2mm", "value_for": "name"}}}
259	\N	27	35	\N	2021-07-12 18:44:49.538634	2021-07-12 18:44:49.538634	48	\N	MyModule	240	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product B, Alumina 2mm", "experiment": "Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 240, "type": "MyModule", "value": "Test Product B, Alumina 2mm", "value_for": "name"}}}
260	\N	27	36	\N	2021-07-12 18:44:59.002676	2021-07-12 18:44:59.002676	48	\N	MyModule	240	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product B, Alumina 2mm", "experiment": "Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 240, "type": "MyModule", "value": "Test Product B, Alumina 2mm", "value_for": "name"}}}
261	\N	27	28	\N	2021-07-13 15:38:33.870572	2021-07-13 15:38:33.870572	48	\N	Experiment	53	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "experiment": "20210713 Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "experiment": {"id": 53, "type": "Experiment", "value": "20210713 Product A,B,C and Alumina 1, 2,3mm", "value_for": "name"}}}
263	\N	27	36	\N	2021-07-13 16:25:33.594448	2021-07-13 16:25:33.594448	48	\N	MyModule	239	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product A, Alumina 2mm", "experiment": "20210713 Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 239, "type": "MyModule", "value": "Test Product A, Alumina 2mm", "value_for": "name"}}}
264	\N	27	35	\N	2021-07-13 16:27:14.86303	2021-07-13 16:27:14.86303	48	\N	MyModule	239	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product A, Alumina 2mm", "experiment": "20210713 Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 239, "type": "MyModule", "value": "Test Product A, Alumina 2mm", "value_for": "name"}}}
265	\N	27	36	\N	2021-07-15 14:29:46.395125	2021-07-15 14:29:46.395125	48	\N	MyModule	239	22	\N	{"breadcrumbs": {"team": "TLee ' 's lab", "project": "P Recovery", "my_module": "Test Product A, Alumina 2mm", "experiment": "20210713 Product A,B,C and Alumina 1, 2,3mm"}, "message_items": {"user": {"id": 27, "type": "User", "value": "Tae Lee", "value_for": "full_name"}, "my_module": {"id": 239, "type": "MyModule", "value": "Test Product A, Alumina 2mm", "value_for": "name"}}}
266	\N	21	80	\N	2021-07-21 19:12:50.851556	2021-07-21 19:12:50.851556	\N	\N	Protocol	254	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Field Sample Collection Report"}, "message_items": {"user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 254, "type": "Protocol", "value": "Field Sample Collection Report", "value_for": "name"}}}
267	\N	21	82	\N	2021-07-28 19:42:12.320177	2021-07-28 19:42:12.320177	\N	\N	Protocol	222	16	\N	{"breadcrumbs": {"team": "Alan ' 's lab", "protocol": "Reagent Preparation for WIZ"}, "message_items": {"step": {"id": 847, "type": "Step", "value": "Chemicals used for reagent preparation", "value_for": "name"}, "user": {"id": 21, "type": "User", "value": "Alan Lindquist", "value_for": "full_name"}, "protocol": {"id": 222, "type": "Protocol", "value": "Reagent Preparation for WIZ", "value_for": "name"}, "step_position": {"id": 847, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
268	\N	30	0	\N	2021-08-23 18:49:48.233067	2021-08-23 18:49:48.233067	57	\N	Project	57	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "project": {"id": 57, "type": "Project", "value": "PFAS Liquid Extraction 8/23", "value_for": "name"}}}
269	\N	30	27	\N	2021-08-23 18:50:06.222336	2021-08-23 18:50:06.222336	57	\N	Experiment	56	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "experiment": {"id": 56, "type": "Experiment", "value": "PFAS Liquid Extraction", "value_for": "name"}}}
270	\N	30	8	\N	2021-08-23 19:46:47.858381	2021-08-23 19:46:47.858381	57	\N	MyModule	259	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "QC Preparation", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 259, "type": "MyModule", "value": "QC Preparation", "value_for": "name"}}}
285	\N	30	80	\N	2021-08-23 20:54:28.210957	2021-08-23 20:54:28.210957	\N	\N	Protocol	288	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "Protocol"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 288, "type": "Protocol", "value": "Protocol", "value_for": "name"}}}
271	\N	30	8	\N	2021-08-23 19:46:47.862228	2021-08-23 19:46:47.862228	57	\N	MyModule	260	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "10mL of water into each 15mL Falcon Tube", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 260, "type": "MyModule", "value": "10mL of water into each 15mL Falcon Tube", "value_for": "name"}}}
272	\N	30	8	\N	2021-08-23 19:46:47.864634	2021-08-23 19:46:47.864634	57	\N	MyModule	261	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Method Blank Preparation", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 261, "type": "MyModule", "value": "Method Blank Preparation", "value_for": "name"}}}
273	\N	30	8	\N	2021-08-23 19:46:47.867322	2021-08-23 19:46:47.867322	57	\N	MyModule	262	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Lab Fortified Blank Preparation", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 262, "type": "MyModule", "value": "Lab Fortified Blank Preparation", "value_for": "name"}}}
274	\N	30	8	\N	2021-08-23 19:46:47.869878	2021-08-23 19:46:47.869878	57	\N	MyModule	263	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Matrix Spike Sample Preparation", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 263, "type": "MyModule", "value": "Matrix Spike Sample Preparation", "value_for": "name"}}}
275	\N	30	8	\N	2021-08-23 19:46:47.872515	2021-08-23 19:46:47.872515	57	\N	MyModule	264	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Addition of 100 microliters of native precursor spike solution", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 264, "type": "MyModule", "value": "Addition of 100 microliters of native precursor spike solution", "value_for": "name"}}}
276	\N	30	8	\N	2021-08-23 19:46:47.875091	2021-08-23 19:46:47.875091	57	\N	MyModule	265	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 265, "type": "MyModule", "value": "Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)", "value_for": "name"}}}
277	\N	30	8	\N	2021-08-23 19:46:47.877602	2021-08-23 19:46:47.877602	57	\N	MyModule	266	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Shake each aliquot for 2 minutes", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 266, "type": "MyModule", "value": "Shake each aliquot for 2 minutes", "value_for": "name"}}}
278	\N	30	8	\N	2021-08-23 19:46:47.879907	2021-08-23 19:46:47.879907	57	\N	MyModule	267	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Add 1mL of Ethyl Acetate", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 267, "type": "MyModule", "value": "Add 1mL of Ethyl Acetate", "value_for": "name"}}}
279	\N	30	8	\N	2021-08-23 19:46:47.883095	2021-08-23 19:46:47.883095	57	\N	MyModule	268	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Shake for 2 minutes", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 268, "type": "MyModule", "value": "Shake for 2 minutes", "value_for": "name"}}}
280	\N	30	8	\N	2021-08-23 19:46:47.885471	2021-08-23 19:46:47.885471	57	\N	MyModule	269	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Add 450 uL of EtOAc extraction layer into mini Uniprep syringeless filters", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 269, "type": "MyModule", "value": "Add 450 uL of EtOAc extraction layer into mini Uniprep syringeless filters", "value_for": "name"}}}
281	\N	30	8	\N	2021-08-23 19:46:47.888096	2021-08-23 19:46:47.888096	57	\N	MyModule	270	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Add 1 gram of NaCl to each PP tube", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 270, "type": "MyModule", "value": "Add 1 gram of NaCl to each PP tube", "value_for": "name"}}}
282	\N	30	8	\N	2021-08-23 19:46:47.891089	2021-08-23 19:46:47.891089	57	\N	MyModule	271	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Addition of 100 microliters of native precursor spike solution", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 271, "type": "MyModule", "value": "Addition of 100 microliters of native precursor spike solution", "value_for": "name"}}}
283	\N	30	8	\N	2021-08-23 19:46:47.893355	2021-08-23 19:46:47.893355	57	\N	MyModule	272	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 272, "type": "MyModule", "value": "Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)", "value_for": "name"}}}
284	\N	30	8	\N	2021-08-23 19:46:47.895596	2021-08-23 19:46:47.895596	57	\N	MyModule	273	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "PFAS Liquid Extraction 8/23", "my_module": "Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)", "experiment": "PFAS Liquid Extraction"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 273, "type": "MyModule", "value": "Addition of 100 microliters of surrogate solutions (PreSurS 500pg/uL)", "value_for": "name"}}}
286	\N	30	80	\N	2021-08-24 13:48:38.732545	2021-08-24 13:48:38.732545	\N	\N	Protocol	289	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "QC Creation"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 289, "type": "Protocol", "value": "QC Creation", "value_for": "name"}}}
287	\N	30	81	\N	2021-08-24 14:20:20.679297	2021-08-24 14:20:20.679297	\N	\N	Protocol	289	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "QC Creation"}, "message_items": {"step": {"id": 1015, "type": "Step", "value": "MB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 289, "type": "Protocol", "value": "QC Creation", "value_for": "name"}, "step_position": {"id": 1015, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
288	\N	30	81	\N	2021-08-24 14:21:04.749802	2021-08-24 14:21:04.749802	\N	\N	Protocol	289	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "QC Creation"}, "message_items": {"step": {"id": 1016, "type": "Step", "value": "LFB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 289, "type": "Protocol", "value": "QC Creation", "value_for": "name"}, "step_position": {"id": 1016, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
289	\N	30	81	\N	2021-08-24 14:21:37.625573	2021-08-24 14:21:37.625573	\N	\N	Protocol	289	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "QC Creation"}, "message_items": {"step": {"id": 1017, "type": "Step", "value": "MS Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 289, "type": "Protocol", "value": "QC Creation", "value_for": "name"}, "step_position": {"id": 1017, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
290	\N	30	82	\N	2021-08-24 16:12:02.813211	2021-08-24 16:12:02.813211	\N	\N	Protocol	289	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "QC Creation"}, "message_items": {"step": {"id": 1015, "type": "Step", "value": "MB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 289, "type": "Protocol", "value": "QC Creation", "value_for": "name"}, "step_position": {"id": 1015, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
292	\N	30	82	\N	2021-08-24 17:46:58.453821	2021-08-24 17:46:58.453821	\N	\N	Protocol	289	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "QC Creation"}, "message_items": {"step": {"id": 1015, "type": "Step", "value": "MB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 289, "type": "Protocol", "value": "QC Creation", "value_for": "name"}, "step_position": {"id": 1015, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
293	\N	31	92	\N	2021-08-25 13:47:15.24193	2021-08-25 13:47:15.24193	\N	\N	Team	25	25	\N	{"breadcrumbs": {"team": "FTOH ELN001"}, "message_items": {"role": "Normal user", "team": {"id": 25, "type": "Team", "value": "FTOH ELN001", "value_for": "name"}, "user": {"id": 31, "type": "User", "value": "JAMES VOIT", "value_for": "full_name"}, "user_invited": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "name"}}}
294	\N	30	92	\N	2021-08-25 14:57:58.980177	2021-08-25 14:57:58.980177	\N	\N	Team	24	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)"}, "message_items": {"role": "Normal user", "team": {"id": 24, "type": "Team", "value": "FTOH ELN001 (TEST)", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "user_invited": {"id": 31, "type": "User", "value": "JAMES VOIT", "value_for": "name"}}}
295	\N	30	94	\N	2021-08-25 14:58:54.30407	2021-08-25 14:58:54.30407	\N	\N	Team	24	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)"}, "message_items": {"role": "Administrator", "team": {"id": 24, "type": "Team", "value": "FTOH ELN001 (TEST)", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "user_changed": {"id": 31, "type": "User", "value": "JAMES VOIT", "value_for": "name"}}}
296	\N	30	0	\N	2021-08-25 16:06:30.317071	2021-08-25 16:06:30.317071	58	\N	Project	58	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "TEST"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "project": {"id": 58, "type": "Project", "value": "TEST", "value_for": "name"}}}
297	\N	30	27	\N	2021-08-25 16:06:39.694558	2021-08-25 16:06:39.694558	58	\N	Experiment	57	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "TEST", "experiment": "TEST EXP 1"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "experiment": {"id": 57, "type": "Experiment", "value": "TEST EXP 1", "value_for": "name"}}}
298	\N	30	8	\N	2021-08-25 16:08:53.528016	2021-08-25 16:08:53.528016	58	\N	MyModule	274	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "TEST", "my_module": "Task 1", "experiment": "TEST EXP 1"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 274, "type": "MyModule", "value": "Task 1", "value_for": "name"}}}
299	\N	30	81	\N	2021-08-25 20:48:16.02742	2021-08-25 20:48:16.02742	\N	\N	Protocol	288	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "Protocol"}, "message_items": {"step": {"id": 1018, "type": "Step", "value": "Calculations Word Doc and Sonde SOP Test", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 288, "type": "Protocol", "value": "Protocol", "value_for": "name"}, "step_position": {"id": 1018, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
300	\N	30	0	\N	2021-08-26 20:02:24.247942	2021-08-26 20:02:24.247942	59	\N	Project	59	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "TEST Liquid Content Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "project": {"id": 59, "type": "Project", "value": "TEST Liquid Content Determination", "value_for": "name"}}}
301	\N	30	27	\N	2021-08-26 20:02:40.605538	2021-08-26 20:02:40.605538	59	\N	Experiment	58	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "TEST Liquid Content Determination", "experiment": "Moisture Content Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "experiment": {"id": 58, "type": "Experiment", "value": "Moisture Content Determination", "value_for": "name"}}}
302	\N	30	1	\N	2021-08-26 20:03:07.960372	2021-08-26 20:03:07.960372	59	\N	Project	59	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "Moisture Content Determination TEST"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "project": {"id": 59, "type": "Project", "value": "Moisture Content Determination TEST", "value_for": "name"}}}
304	\N	30	8	\N	2021-08-26 20:19:40.438298	2021-08-26 20:19:40.438298	59	\N	MyModule	275	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "Moisture Content Determination TEST", "my_module": "Analytical balance calibration", "experiment": "Moisture Content Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 275, "type": "MyModule", "value": "Analytical balance calibration", "value_for": "name"}}}
305	\N	30	8	\N	2021-08-26 20:19:40.442086	2021-08-26 20:19:40.442086	59	\N	MyModule	276	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "project": "Moisture Content Determination TEST", "my_module": "Soil Weighing", "experiment": "Moisture Content Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "my_module": {"id": 276, "type": "MyModule", "value": "Soil Weighing", "value_for": "name"}}}
306	\N	30	80	\N	2021-08-26 20:19:59.230825	2021-08-26 20:19:59.230825	\N	\N	Protocol	293	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "Moisture Content Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 293, "type": "Protocol", "value": "Moisture Content Determination", "value_for": "name"}}}
307	\N	30	81	\N	2021-08-26 20:26:39.470763	2021-08-26 20:26:39.470763	\N	\N	Protocol	293	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "Moisture Content Determination"}, "message_items": {"step": {"id": 1019, "type": "Step", "value": "Analytical balance calibration.", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 293, "type": "Protocol", "value": "Moisture Content Determination", "value_for": "name"}, "step_position": {"id": 1019, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
308	\N	30	82	\N	2021-08-26 20:38:33.031454	2021-08-26 20:38:33.031454	\N	\N	Protocol	293	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "Moisture Content Determination"}, "message_items": {"step": {"id": 1019, "type": "Step", "value": "Analytical balance calibration.", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 293, "type": "Protocol", "value": "Moisture Content Determination", "value_for": "name"}, "step_position": {"id": 1019, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
309	\N	30	0	\N	2021-08-26 20:53:28.264646	2021-08-26 20:53:28.264646	60	\N	Project	60	25	\N	{"breadcrumbs": {"team": "FTOH ELN001", "project": "TEST Moisture Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "project": {"id": 60, "type": "Project", "value": "TEST Moisture Determination", "value_for": "name"}}}
310	\N	30	80	\N	2021-08-26 20:54:22.238734	2021-08-26 20:54:22.238734	\N	\N	Protocol	294	25	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "TEST Moisture Determination"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 294, "type": "Protocol", "value": "TEST Moisture Determination", "value_for": "name"}}}
311	\N	30	81	\N	2021-08-26 20:55:14.795527	2021-08-26 20:55:14.795527	\N	\N	Protocol	294	25	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "TEST Moisture Determination"}, "message_items": {"step": {"id": 1020, "type": "Step", "value": "Analytical Balance Calibration", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 294, "type": "Protocol", "value": "TEST Moisture Determination", "value_for": "name"}, "step_position": {"id": 1020, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
312	\N	30	82	\N	2021-08-26 20:55:32.931753	2021-08-26 20:55:32.931753	\N	\N	Protocol	294	25	\N	{"breadcrumbs": {"team": "FTOH ELN001", "protocol": "TEST Moisture Determination"}, "message_items": {"step": {"id": 1020, "type": "Step", "value": "Analytical Balance Calibration", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 294, "type": "Protocol", "value": "TEST Moisture Determination", "value_for": "name"}, "step_position": {"id": 1020, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
313	\N	30	80	\N	2021-09-17 13:37:21.121483	2021-09-17 13:37:21.121483	\N	\N	Protocol	295	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "1214 IDOC"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 295, "type": "Protocol", "value": "1214 IDOC", "value_for": "name"}}}
314	\N	30	81	\N	2021-09-17 14:44:02.40687	2021-09-17 14:44:02.40687	\N	\N	Protocol	295	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "1214 IDOC"}, "message_items": {"step": {"id": 1021, "type": "Step", "value": "MB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 295, "type": "Protocol", "value": "1214 IDOC", "value_for": "name"}, "step_position": {"id": 1021, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
315	\N	30	82	\N	2021-09-17 14:44:20.258095	2021-09-17 14:44:20.258095	\N	\N	Protocol	295	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "1214 IDOC"}, "message_items": {"step": {"id": 1021, "type": "Step", "value": "MB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 295, "type": "Protocol", "value": "1214 IDOC", "value_for": "name"}, "step_position": {"id": 1021, "type": "Step", "value": "1", "value_for": "position_plus_one"}}}
316	\N	30	81	\N	2021-09-17 14:45:10.528179	2021-09-17 14:45:10.528179	\N	\N	Protocol	295	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "1214 IDOC"}, "message_items": {"step": {"id": 1022, "type": "Step", "value": "LFB Creation", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 295, "type": "Protocol", "value": "1214 IDOC", "value_for": "name"}, "step_position": {"id": 1022, "type": "Step", "value": "2", "value_for": "position_plus_one"}}}
317	\N	30	81	\N	2021-09-17 14:45:35.980209	2021-09-17 14:45:35.980209	\N	\N	Protocol	295	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "protocol": "1214 IDOC"}, "message_items": {"step": {"id": 1023, "type": "Step", "value": "Water addition", "value_for": "name"}, "user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "protocol": {"id": 295, "type": "Protocol", "value": "1214 IDOC", "value_for": "name"}, "step_position": {"id": 1023, "type": "Step", "value": "3", "value_for": "position_plus_one"}}}
11	\N	7	75	\N	2019-08-30 16:10:44.330949	2019-08-30 16:10:44.330949	\N	\N	RepositoryBase	5	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Samples"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 5, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 23, "type": "RepositoryRow", "value": "ZAT/3", "value_for": "name"}}}
12	\N	7	75	\N	2019-08-30 16:10:44.393715	2019-08-30 16:10:44.393715	\N	\N	RepositoryBase	5	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Samples"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 5, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 24, "type": "RepositoryRow", "value": "ZAT/4", "value_for": "name"}}}
13	\N	7	75	\N	2019-08-30 16:10:44.455655	2019-08-30 16:10:44.455655	\N	\N	RepositoryBase	5	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Samples"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 5, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 25, "type": "RepositoryRow", "value": "ZAT/5", "value_for": "name"}}}
14	\N	7	73	\N	2019-08-30 16:11:26.209792	2019-08-30 16:11:26.209792	\N	\N	RepositoryBase	6	5	\N	{"breadcrumbs": {"team": "R7-Molecular Lab", "repository": "Reagents"}, "message_items": {"user": {"id": 7, "type": "User", "value": "Steven Baker", "value_for": "full_name"}, "repository": {"id": 6, "type": "Repository", "value": "Reagents", "value_for": "name"}, "repository_row": {"id": 26, "type": "RepositoryRow", "value": "ExTaq", "value_for": "name"}}}
60	\N	4	77	\N	2019-11-25 17:44:01.087713	2019-11-25 17:44:01.087713	\N	\N	RepositoryBase	11	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "repository": "Samples"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "repository": {"id": 11, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_column": {"id": 16, "type": "RepositoryColumn", "value": "project id", "value_for": "name"}}}
61	\N	4	74	\N	2019-11-25 17:45:32.290856	2019-11-25 17:45:32.290856	\N	\N	RepositoryBase	11	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "repository": "Samples"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "repository": {"id": 11, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 37, "type": "RepositoryRow", "value": "KOK/1", "value_for": "name"}}}
62	\N	4	74	\N	2019-11-25 17:45:39.41231	2019-11-25 17:45:39.41231	\N	\N	RepositoryBase	11	7	\N	{"breadcrumbs": {"team": "R7-Microbiology Lab", "repository": "Samples"}, "message_items": {"user": {"id": 4, "type": "User", "value": "Regina Klepikow", "value_for": "full_name"}, "repository": {"id": 11, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 38, "type": "RepositoryRow", "value": "KOK/2", "value_for": "name"}}}
291	\N	30	73	\N	2021-08-24 16:51:32.001932	2021-08-24 16:51:32.001932	\N	\N	RepositoryBase	27	24	\N	{"breadcrumbs": {"team": "FTOH ELN001", "repository": "Samples"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "repository": {"id": 27, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 127, "type": "RepositoryRow", "value": "Surrogate Solution", "value_for": "name"}}}
303	\N	30	73	\N	2021-08-26 20:03:53.17304	2021-08-26 20:03:53.17304	\N	\N	RepositoryBase	27	24	\N	{"breadcrumbs": {"team": "FTOH ELN001 (TEST)", "repository": "Samples"}, "message_items": {"user": {"id": 30, "type": "User", "value": "Aaron Tam", "value_for": "full_name"}, "repository": {"id": 27, "type": "Repository", "value": "Samples", "value_for": "name"}, "repository_row": {"id": 128, "type": "RepositoryRow", "value": "Clean Soil 1/1", "value_for": "name"}}}
\.


--
-- Data for Name: assets; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.assets (id, created_at, updated_at, created_by_id, last_modified_by_id, estimated_size, lock, lock_ttl, version, file_processing, team_id, file_image_quality, view_mode, pdf_preview_processing) FROM stdin;
14	2019-05-30 12:41:23.441937	2019-05-30 12:41:29.521165	1	1	205748	\N	\N	1	f	1	79	0	f
3	2019-05-30 12:41:21.273424	2019-05-30 12:41:27.348128	1	1	598154	\N	\N	1	f	1	97	0	f
15	2019-05-30 12:41:23.721975	2019-05-30 12:41:29.695763	1	1	11317	\N	\N	1	f	1	74	0	f
16	2019-05-30 12:41:23.96384	2019-05-30 12:41:23.988113	1	1	0	\N	\N	1	f	1	\N	0	f
17	2019-05-30 12:41:24.085061	2019-05-30 12:41:30.168052	1	1	133004	\N	\N	1	f	1	88	0	f
4	2019-05-30 12:41:21.547509	2019-05-30 12:41:27.519605	1	1	17875	\N	\N	1	f	1	94	0	f
18	2019-05-30 12:41:24.174577	2019-05-30 12:41:24.195766	1	1	0	\N	\N	1	f	1	\N	0	f
19	2019-05-30 12:41:24.296718	2019-05-30 12:41:24.321155	1	1	0	\N	\N	1	f	1	\N	0	f
20	2019-05-30 12:41:24.430083	2019-05-30 12:41:24.460508	1	1	51251	\N	\N	1	f	1	\N	0	f
5	2019-05-30 12:41:21.717605	2019-05-30 12:41:21.748881	1	1	0	\N	\N	1	f	1	\N	0	f
21	2019-05-30 12:41:24.574267	2019-05-30 12:41:30.702782	1	1	24285	\N	\N	1	f	1	\N	0	f
6	2019-05-30 12:41:21.917783	2019-05-30 12:41:27.891035	1	1	46131	\N	\N	1	f	1	79	0	f
22	2019-05-30 12:41:24.88425	2019-05-30 12:41:24.912969	1	1	62515	\N	\N	1	f	1	\N	0	f
23	2019-05-30 12:41:25.097	2019-05-30 12:41:31.0246	1	1	70324	\N	\N	1	f	1	79	0	f
7	2019-05-30 12:41:22.098573	2019-05-30 12:41:22.121328	1	1	0	\N	\N	1	f	1	\N	0	f
24	2019-05-30 12:41:25.295117	2019-05-30 12:41:31.251635	1	1	82352	\N	\N	1	f	1	79	0	f
8	2019-05-30 12:41:22.220803	2019-05-30 12:41:28.23585	1	1	84197	\N	\N	1	f	1	92	0	f
9	2019-05-30 12:41:22.452614	2019-05-30 12:41:22.483387	1	1	0	\N	\N	1	f	1	\N	0	f
10	2019-05-30 12:41:22.581135	2019-05-30 12:41:28.540861	1	1	57335	\N	\N	1	f	1	79	0	f
11	2019-05-30 12:41:22.709121	2019-05-30 12:41:28.739629	1	1	210623	\N	\N	1	f	1	99	0	f
1	2019-05-30 12:41:20.633487	2019-05-30 12:41:26.577379	1	1	534404	\N	\N	1	f	1	99	0	f
2	2019-05-30 12:41:21.045925	2019-05-30 12:41:26.9211	1	1	596394	\N	\N	1	f	1	99	0	f
12	2019-05-30 12:41:22.93122	2019-05-30 12:41:29.072469	1	1	606138	\N	\N	1	f	1	96	0	f
13	2019-05-30 12:41:23.144004	2019-05-30 12:41:29.282486	1	1	260422	\N	\N	1	f	1	100	0	f
44	2019-08-27 15:15:45.308808	2019-08-27 15:15:45.344652	4	4	51251	\N	\N	1	t	3	\N	0	f
34	2019-08-27 15:15:42.53979	2019-08-27 15:15:42.568967	4	4	57335	\N	\N	1	t	3	\N	0	f
25	2019-08-27 15:15:39.594359	2019-08-27 15:15:39.939361	4	4	534404	\N	\N	1	t	3	\N	0	f
30	2019-08-27 15:15:41.6611	2019-08-27 15:15:41.715085	4	4	46131	\N	\N	1	t	3	\N	0	f
26	2019-08-27 15:15:40.458701	2019-08-27 15:15:40.519413	4	4	596394	\N	\N	1	t	3	\N	0	f
31	2019-08-27 15:15:41.84379	2019-08-27 15:15:41.867244	4	4	0	\N	\N	1	t	3	\N	0	f
42	2019-08-27 15:15:44.842081	2019-08-27 15:15:44.865387	4	4	0	\N	\N	1	t	3	\N	0	f
27	2019-08-27 15:15:40.799667	2019-08-27 15:15:40.830533	4	4	598154	\N	\N	1	t	3	\N	0	f
38	2019-08-27 15:15:43.841146	2019-08-27 15:15:43.871077	4	4	205748	\N	\N	1	t	3	\N	0	f
32	2019-08-27 15:15:42.20422	2019-08-27 15:15:42.234209	4	4	84197	\N	\N	1	t	3	\N	0	f
28	2019-08-27 15:15:41.242458	2019-08-27 15:15:41.306091	4	4	17875	\N	\N	1	t	3	\N	0	f
35	2019-08-27 15:15:42.719908	2019-08-27 15:15:42.749099	4	4	210623	\N	\N	1	t	3	\N	0	f
29	2019-08-27 15:15:41.395459	2019-08-27 15:15:41.41989	4	4	0	\N	\N	1	t	3	\N	0	f
33	2019-08-27 15:15:42.365875	2019-08-27 15:15:42.38862	4	4	0	\N	\N	1	t	3	\N	0	f
37	2019-08-27 15:15:43.545533	2019-08-27 15:15:43.579011	4	4	260422	\N	\N	1	t	3	\N	0	f
40	2019-08-27 15:15:44.558767	2019-08-27 15:15:44.583177	4	4	0	\N	\N	1	t	3	\N	0	f
36	2019-08-27 15:15:43.23399	2019-08-27 15:15:43.263922	4	4	606138	\N	\N	1	t	3	\N	0	f
41	2019-08-27 15:15:44.730908	2019-08-27 15:15:44.761683	4	4	133004	\N	\N	1	t	3	\N	0	f
39	2019-08-27 15:15:44.273735	2019-08-27 15:15:44.307322	4	4	11317	\N	\N	1	t	3	\N	0	f
43	2019-08-27 15:15:45.12733	2019-08-27 15:15:45.161441	4	4	0	\N	\N	1	t	3	\N	0	f
45	2019-08-27 15:15:45.481349	2019-08-27 15:15:45.519476	4	4	24285	\N	\N	1	t	3	\N	0	f
46	2019-08-27 15:15:45.797358	2019-08-27 15:15:45.824671	4	4	62515	\N	\N	1	t	3	\N	0	f
47	2019-08-27 15:15:46.215709	2019-08-27 15:15:46.248221	4	4	70324	\N	\N	1	t	3	\N	0	f
48	2019-08-27 15:15:46.491604	2019-08-27 15:15:46.533296	4	4	82352	\N	\N	1	t	3	\N	0	f
60	2019-08-28 07:31:05.193747	2019-08-28 07:31:05.227206	1	1	606138	\N	\N	1	t	2	\N	0	f
49	2019-08-28 07:31:02.339214	2019-08-28 07:31:02.418061	1	1	534404	\N	\N	1	t	2	\N	0	f
68	2019-08-28 07:31:07.167313	2019-08-28 07:31:07.197174	1	1	51251	\N	\N	1	t	2	\N	0	f
50	2019-08-28 07:31:02.727825	2019-08-28 07:31:02.75923	1	1	596394	\N	\N	1	t	2	\N	0	f
78	2019-08-28 08:35:50.205337	2019-08-28 08:35:50.240676	6	6	46131	\N	\N	1	t	4	\N	0	f
61	2019-08-28 07:31:05.463743	2019-08-28 07:31:05.494335	1	1	260422	\N	\N	1	t	2	\N	0	f
51	2019-08-28 07:31:03.121942	2019-08-28 07:31:03.174337	1	1	598154	\N	\N	1	t	2	\N	0	f
69	2019-08-28 07:31:07.332747	2019-08-28 07:31:07.363364	1	1	24285	\N	\N	1	t	2	\N	0	f
52	2019-08-28 07:31:03.406454	2019-08-28 07:31:03.440317	1	1	17875	\N	\N	1	t	2	\N	0	f
62	2019-08-28 07:31:05.72476	2019-08-28 07:31:05.753036	1	1	205748	\N	\N	1	t	2	\N	0	f
53	2019-08-28 07:31:03.52153	2019-08-28 07:31:03.546697	1	1	0	\N	\N	1	t	2	\N	0	f
90	2019-08-28 08:35:53.878163	2019-08-28 08:35:53.96352	6	6	0	\N	\N	1	t	4	\N	0	f
74	2019-08-28 08:35:48.768796	2019-08-28 08:35:48.814015	6	6	596394	\N	\N	1	t	4	\N	0	f
54	2019-08-28 07:31:03.790926	2019-08-28 07:31:03.821015	1	1	46131	\N	\N	1	t	2	\N	0	f
55	2019-08-28 07:31:04.035004	2019-08-28 07:31:04.081766	1	1	0	\N	\N	1	t	2	\N	0	f
63	2019-08-28 07:31:06.068856	2019-08-28 07:31:06.142847	1	1	11317	\N	\N	1	t	2	\N	0	f
56	2019-08-28 07:31:04.324316	2019-08-28 07:31:04.354715	1	1	84197	\N	\N	1	t	2	\N	0	f
70	2019-08-28 07:31:07.612399	2019-08-28 07:31:07.644386	1	1	62515	\N	\N	1	t	2	\N	0	f
57	2019-08-28 07:31:04.479769	2019-08-28 07:31:04.503934	1	1	0	\N	\N	1	t	2	\N	0	f
64	2019-08-28 07:31:06.397509	2019-08-28 07:31:06.428127	1	1	0	\N	\N	1	t	2	\N	0	f
58	2019-08-28 07:31:04.634158	2019-08-28 07:31:04.668545	1	1	57335	\N	\N	1	t	2	\N	0	f
79	2019-08-28 08:35:50.457475	2019-08-28 08:35:50.484791	6	6	0	\N	\N	1	t	4	\N	0	f
59	2019-08-28 07:31:04.792794	2019-08-28 07:31:04.820674	1	1	210623	\N	\N	1	t	2	\N	0	f
65	2019-08-28 07:31:06.565476	2019-08-28 07:31:06.603701	1	1	133004	\N	\N	1	t	2	\N	0	f
88	2019-08-28 08:35:53.593007	2019-08-28 08:35:53.616632	6	6	0	\N	\N	1	t	4	\N	0	f
71	2019-08-28 07:31:07.871086	2019-08-28 07:31:07.915256	1	1	70324	\N	\N	1	t	2	\N	0	f
66	2019-08-28 07:31:06.705186	2019-08-28 07:31:06.731447	1	1	0	\N	\N	1	t	2	\N	0	f
67	2019-08-28 07:31:06.857156	2019-08-28 07:31:06.881357	1	1	0	\N	\N	1	t	2	\N	0	f
75	2019-08-28 08:35:49.26704	2019-08-28 08:35:49.316759	6	6	598154	\N	\N	1	t	4	\N	0	f
83	2019-08-28 08:35:51.532406	2019-08-28 08:35:51.568296	6	6	210623	\N	\N	1	t	4	\N	0	f
80	2019-08-28 08:35:50.635131	2019-08-28 08:35:50.67224	6	6	84197	\N	\N	1	t	4	\N	0	f
72	2019-08-28 07:31:08.265858	2019-08-28 07:31:08.306792	1	1	82352	\N	\N	1	t	2	\N	0	f
76	2019-08-28 08:35:49.586019	2019-08-28 08:35:49.624749	6	6	17875	\N	\N	1	t	4	\N	0	f
73	2019-08-28 08:35:48.343502	2019-08-28 08:35:48.434021	6	6	534404	\N	\N	1	t	4	\N	0	f
77	2019-08-28 08:35:49.718842	2019-08-28 08:35:49.743228	6	6	0	\N	\N	1	t	4	\N	0	f
81	2019-08-28 08:35:50.872545	2019-08-28 08:35:50.991338	6	6	0	\N	\N	1	t	4	\N	0	f
85	2019-08-28 08:35:52.422963	2019-08-28 08:35:52.458874	6	6	260422	\N	\N	1	t	4	\N	0	f
82	2019-08-28 08:35:51.248625	2019-08-28 08:35:51.309936	6	6	57335	\N	\N	1	t	4	\N	0	f
84	2019-08-28 08:35:51.871945	2019-08-28 08:35:51.979669	6	6	606138	\N	\N	1	t	4	\N	0	f
87	2019-08-28 08:35:53.227687	2019-08-28 08:35:53.267716	6	6	11317	\N	\N	1	t	4	\N	0	f
89	2019-08-28 08:35:53.765391	2019-08-28 08:35:53.794583	6	6	133004	\N	\N	1	t	4	\N	0	f
86	2019-08-28 08:35:52.735907	2019-08-28 08:35:52.767423	6	6	205748	\N	\N	1	t	4	\N	0	f
92	2019-08-28 08:35:54.485611	2019-08-28 08:35:54.536822	6	6	51251	\N	\N	1	t	4	\N	0	f
91	2019-08-28 08:35:54.234416	2019-08-28 08:35:54.271688	6	6	0	\N	\N	1	t	4	\N	0	f
93	2019-08-28 08:35:54.69904	2019-08-28 08:35:54.734662	6	6	24285	\N	\N	1	t	4	\N	0	f
94	2019-08-28 08:35:55.187901	2019-08-28 08:35:55.219072	6	6	62515	\N	\N	1	t	4	\N	0	f
95	2019-08-28 08:35:55.519524	2019-08-28 08:35:55.551572	6	6	70324	\N	\N	1	t	4	\N	0	f
120	2019-08-29 12:52:45.222473	2019-08-29 12:52:45.312858	7	7	82352	\N	\N	1	t	5	\N	0	f
107	2019-08-29 12:52:36.718191	2019-08-29 12:52:36.818604	7	7	210623	\N	\N	1	t	5	\N	0	f
96	2019-08-28 08:35:55.817975	2019-08-28 08:35:55.848044	6	6	82352	\N	\N	1	t	4	\N	0	f
115	2019-08-29 12:52:42.369659	2019-08-29 12:52:42.443722	7	7	0	\N	\N	1	t	5	\N	0	f
97	2019-08-29 12:52:29.967523	2019-08-29 12:52:30.397976	7	7	534404	\N	\N	1	t	5	\N	0	f
108	2019-08-29 12:52:37.819199	2019-08-29 12:52:37.919959	7	7	606138	\N	\N	1	t	5	\N	0	f
98	2019-08-29 12:52:31.485306	2019-08-29 12:52:31.767091	7	7	596394	\N	\N	1	t	5	\N	0	f
125	2019-09-12 18:26:28.306306	2019-09-12 18:26:28.330575	8	8	0	\N	\N	1	t	6	\N	0	f
116	2019-08-29 12:52:42.793514	2019-08-29 12:52:42.868714	7	7	51251	\N	\N	1	t	5	\N	0	f
99	2019-08-29 12:52:32.370157	2019-08-29 12:52:32.427102	7	7	598154	\N	\N	1	t	5	\N	0	f
109	2019-08-29 12:52:38.564569	2019-08-29 12:52:38.712788	7	7	260422	\N	\N	1	t	5	\N	0	f
100	2019-08-29 12:52:32.867545	2019-08-29 12:52:33.110102	7	7	17875	\N	\N	1	t	5	\N	0	f
121	2019-09-12 18:26:27.059801	2019-09-12 18:26:27.282393	8	8	534404	\N	\N	1	t	6	\N	0	f
101	2019-08-29 12:52:33.481296	2019-08-29 12:52:33.625931	7	7	0	\N	\N	1	t	5	\N	0	f
117	2019-08-29 12:52:43.311529	2019-08-29 12:52:43.369323	7	7	24285	\N	\N	1	t	5	\N	0	f
110	2019-08-29 12:52:39.410599	2019-08-29 12:52:39.573575	7	7	205748	\N	\N	1	t	5	\N	0	f
102	2019-08-29 12:52:34.402519	2019-08-29 12:52:34.485807	7	7	46131	\N	\N	1	t	5	\N	0	f
103	2019-08-29 12:52:34.727038	2019-08-29 12:52:34.792084	7	7	0	\N	\N	1	t	5	\N	0	f
129	2019-09-12 18:26:29.122588	2019-09-12 18:26:29.151484	8	8	0	\N	\N	1	t	6	\N	0	f
104	2019-08-29 12:52:35.310027	2019-08-29 12:52:35.374409	7	7	84197	\N	\N	1	t	5	\N	0	f
111	2019-08-29 12:52:40.265367	2019-08-29 12:52:40.348003	7	7	11317	\N	\N	1	t	5	\N	0	f
105	2019-08-29 12:52:35.648302	2019-08-29 12:52:35.726359	7	7	0	\N	\N	1	t	5	\N	0	f
106	2019-08-29 12:52:36.182771	2019-08-29 12:52:36.381377	7	7	57335	\N	\N	1	t	5	\N	0	f
118	2019-08-29 12:52:43.856869	2019-08-29 12:52:43.924931	7	7	62515	\N	\N	1	t	5	\N	0	f
112	2019-08-29 12:52:40.753081	2019-08-29 12:52:40.857992	7	7	0	\N	\N	1	t	5	\N	0	f
136	2019-09-12 18:26:30.770307	2019-09-12 18:26:30.79386	8	8	0	\N	\N	1	t	6	\N	0	f
122	2019-09-12 18:26:27.622903	2019-09-12 18:26:27.662396	8	8	596394	\N	\N	1	t	6	\N	0	f
113	2019-08-29 12:52:41.36553	2019-08-29 12:52:41.488896	7	7	133004	\N	\N	1	t	5	\N	0	f
114	2019-08-29 12:52:41.656482	2019-08-29 12:52:41.749686	7	7	0	\N	\N	1	t	5	\N	0	f
119	2019-08-29 12:52:44.524769	2019-08-29 12:52:44.57701	7	7	70324	\N	\N	1	t	5	\N	0	f
126	2019-09-12 18:26:28.598551	2019-09-12 18:26:28.628663	8	8	46131	\N	\N	1	t	6	\N	0	f
135	2019-09-12 18:26:30.503372	2019-09-12 18:26:30.541001	8	8	11317	\N	\N	1	t	6	\N	0	f
127	2019-09-12 18:26:28.753151	2019-09-12 18:26:28.785892	8	8	0	\N	\N	1	t	6	\N	0	f
123	2019-09-12 18:26:27.907296	2019-09-12 18:26:27.936459	8	8	598154	\N	\N	1	t	6	\N	0	f
130	2019-09-12 18:26:29.278731	2019-09-12 18:26:29.307213	8	8	57335	\N	\N	1	t	6	\N	0	f
124	2019-09-12 18:26:28.164077	2019-09-12 18:26:28.227176	8	8	17875	\N	\N	1	t	6	\N	0	f
132	2019-09-12 18:26:29.716777	2019-09-12 18:26:29.748241	8	8	606138	\N	\N	1	t	6	\N	0	f
128	2019-09-12 18:26:28.95213	2019-09-12 18:26:28.982015	8	8	84197	\N	\N	1	t	6	\N	0	f
131	2019-09-12 18:26:29.453965	2019-09-12 18:26:29.482083	8	8	210623	\N	\N	1	t	6	\N	0	f
134	2019-09-12 18:26:30.24708	2019-09-12 18:26:30.279816	8	8	205748	\N	\N	1	t	6	\N	0	f
133	2019-09-12 18:26:29.97681	2019-09-12 18:26:30.007765	8	8	260422	\N	\N	1	t	6	\N	0	f
140	2019-09-12 18:26:31.397507	2019-09-12 18:26:31.428284	8	8	51251	\N	\N	1	t	6	\N	0	f
138	2019-09-12 18:26:31.06101	2019-09-12 18:26:31.087842	8	8	0	\N	\N	1	t	6	\N	0	f
137	2019-09-12 18:26:30.954702	2019-09-12 18:26:30.983367	8	8	133004	\N	\N	1	t	6	\N	0	f
139	2019-09-12 18:26:31.219772	2019-09-12 18:26:31.243537	8	8	0	\N	\N	1	t	6	\N	0	f
141	2019-09-12 18:26:31.553647	2019-09-12 18:26:31.585333	8	8	24285	\N	\N	1	t	6	\N	0	f
142	2019-09-12 18:26:31.814088	2019-09-12 18:26:31.849335	8	8	62515	\N	\N	1	t	6	\N	0	f
155	2019-11-13 13:58:15.441452	2019-11-13 13:58:23.754577	9	9	210623	\N	\N	1	f	8	99	0	f
162	2019-11-13 13:58:17.389636	2019-11-13 14:18:38.226357	9	9	409332	\N	\N	1	f	8	\N	0	f
143	2019-09-12 18:26:32.085992	2019-09-12 18:26:32.114856	8	8	70324	\N	\N	1	t	6	\N	0	f
156	2019-11-13 13:58:15.758647	2019-11-13 13:58:24.34158	9	9	606138	\N	\N	1	f	8	96	0	f
144	2019-09-12 18:26:32.345416	2019-09-12 18:26:32.376081	8	8	82352	\N	\N	1	t	6	\N	0	f
181	2019-11-25 14:09:32.094887	2019-11-25 14:09:40.307395	4	4	260422	\N	\N	1	f	7	100	0	f
157	2019-11-13 13:58:16.136423	2019-11-13 13:58:24.652568	9	9	260422	\N	\N	1	f	8	100	0	f
163	2019-11-13 13:58:17.551656	2019-11-13 13:58:17.578829	9	9	0	\N	\N	1	t	8	\N	0	f
186	2019-11-25 14:09:33.209803	2019-11-25 14:29:53.512362	4	4	409332	\N	\N	1	f	7	\N	0	f
158	2019-11-13 13:58:16.468496	2019-11-13 13:58:25.05584	9	9	205748	\N	\N	1	f	8	79	0	f
169	2019-11-25 14:09:29.487105	2019-11-25 14:09:35.749737	4	4	534404	\N	\N	1	f	7	99	0	f
164	2019-11-13 13:58:17.713806	2019-11-13 13:58:17.752746	9	9	51251	\N	\N	1	t	8	\N	0	f
177	2019-11-25 14:09:31.21051	2019-11-25 14:09:31.236957	4	4	0	\N	\N	1	t	7	\N	0	f
159	2019-11-13 13:58:16.796636	2019-11-13 13:58:25.307171	9	9	11317	\N	\N	1	f	8	74	0	f
161	2019-11-13 13:58:17.272747	2019-11-13 13:58:25.905807	9	9	133004	\N	\N	1	f	8	88	0	f
145	2019-11-13 13:58:11.677738	2019-11-13 13:58:19.914955	9	9	534404	\N	\N	1	f	8	99	0	f
170	2019-11-25 14:09:29.821104	2019-11-25 14:09:36.331391	4	4	596394	\N	\N	1	f	7	99	0	f
151	2019-11-13 13:58:14.498783	2019-11-13 13:58:14.594548	9	9	0	\N	\N	1	t	8	\N	0	f
174	2019-11-25 14:09:30.732815	2019-11-25 14:09:38.240036	4	4	46131	\N	\N	1	f	7	79	0	f
153	2019-11-13 13:58:15.059646	2019-11-13 13:58:15.102525	9	9	0	\N	\N	1	t	8	\N	0	f
146	2019-11-13 13:58:12.389538	2019-11-13 13:58:20.540798	9	9	596394	\N	\N	1	f	8	99	0	f
184	2019-11-25 14:09:32.942967	2019-11-25 14:09:32.97001	4	4	0	\N	\N	1	t	7	\N	0	f
165	2019-11-13 13:58:17.886638	2019-11-13 13:58:27.285018	9	9	24285	\N	\N	1	f	8	\N	0	f
166	2019-11-13 13:58:18.186973	2019-11-13 13:58:18.222608	9	9	62515	\N	\N	1	t	8	\N	0	f
160	2019-11-13 13:58:17.10959	2019-11-13 13:58:17.138953	9	9	0	\N	\N	1	t	8	\N	0	f
147	2019-11-13 13:58:12.774717	2019-11-13 13:58:21.365639	9	9	598154	\N	\N	1	f	8	97	0	f
171	2019-11-25 14:09:30.090513	2019-11-25 14:09:37.057181	4	4	598154	\N	\N	1	f	7	97	0	f
148	2019-11-13 13:58:13.177202	2019-11-13 13:58:21.624601	9	9	17875	\N	\N	1	f	8	94	0	f
175	2019-11-25 14:09:30.884314	2019-11-25 14:09:30.922764	4	4	0	\N	\N	1	t	7	\N	0	f
167	2019-11-13 13:58:18.45727	2019-11-13 13:58:27.739968	9	9	70324	\N	\N	1	f	8	79	0	f
178	2019-11-25 14:09:31.365994	2019-11-25 14:09:39.080207	4	4	57335	\N	\N	1	f	7	79	0	f
150	2019-11-13 13:58:14.248896	2019-11-13 13:58:22.569465	9	9	46131	\N	\N	1	f	8	79	0	f
168	2019-11-13 13:58:18.729411	2019-11-13 13:58:28.141296	9	9	82352	\N	\N	1	f	8	79	0	f
152	2019-11-13 13:58:14.876137	2019-11-13 13:58:23.030489	9	9	84197	\N	\N	1	f	8	92	0	f
149	2019-11-13 13:58:13.811974	2019-11-13 14:18:36.897278	9	9	121482	\N	\N	1	f	8	\N	0	f
154	2019-11-13 13:58:15.258623	2019-11-13 13:58:23.453485	9	9	57335	\N	\N	1	f	8	79	0	f
172	2019-11-25 14:09:30.35252	2019-11-25 14:09:37.307107	4	4	17875	\N	\N	1	f	7	94	0	f
176	2019-11-25 14:09:31.048748	2019-11-25 14:09:38.670428	4	4	84197	\N	\N	1	f	7	92	0	f
180	2019-11-25 14:09:31.805481	2019-11-25 14:09:39.976818	4	4	606138	\N	\N	1	f	7	96	0	f
179	2019-11-25 14:09:31.524758	2019-11-25 14:09:39.40979	4	4	210623	\N	\N	1	f	7	99	0	f
185	2019-11-25 14:09:33.101897	2019-11-25 14:09:41.56306	4	4	133004	\N	\N	1	f	7	88	0	f
183	2019-11-25 14:09:32.659206	2019-11-25 14:09:40.986051	4	4	11317	\N	\N	1	f	7	74	0	f
182	2019-11-25 14:09:32.384506	2019-11-25 14:09:40.709543	4	4	205748	\N	\N	1	f	7	79	0	f
188	2019-11-25 14:09:33.580891	2019-11-25 14:09:33.614141	4	4	51251	\N	\N	1	t	7	\N	0	f
187	2019-11-25 14:09:33.378323	2019-11-25 14:09:33.429259	4	4	0	\N	\N	1	t	7	\N	0	f
189	2019-11-25 14:09:33.743302	2019-11-25 14:09:42.893382	4	4	24285	\N	\N	1	f	7	\N	0	f
173	2019-11-25 14:09:30.483138	2019-11-25 14:29:52.329856	4	4	121482	\N	\N	1	f	7	\N	0	f
521	2021-05-28 15:08:34.255563	2021-05-28 15:09:03.971744	27	27	205748	\N	\N	1	f	22	79	0	f
199	2019-11-25 18:01:00.748372	2019-11-25 18:01:00.805046	10	10	0	\N	\N	1	t	9	\N	0	f
218	2020-05-12 16:13:49.15136	2020-05-12 16:13:56.452363	11	11	596394	\N	\N	1	f	10	99	0	f
190	2019-11-25 14:09:34.040295	2019-11-25 14:09:34.074591	4	4	62515	\N	\N	1	t	7	\N	0	f
214	2019-11-25 18:01:04.44587	2019-11-25 18:01:04.479322	10	10	62515	\N	\N	1	t	9	\N	0	f
208	2019-11-25 18:01:03.324998	2019-11-25 18:01:03.364856	10	10	0	\N	\N	1	t	9	\N	0	f
201	2019-11-25 18:01:01.213628	2019-11-25 18:01:01.279592	10	10	0	\N	\N	1	t	9	\N	0	f
222	2020-05-12 16:13:50.173494	2020-05-12 16:13:58.512927	11	11	46131	\N	\N	1	f	10	79	0	f
191	2019-11-25 14:09:34.328329	2019-11-25 14:09:43.325728	4	4	70324	\N	\N	1	f	7	79	0	f
192	2019-11-25 14:09:34.63003	2019-11-25 14:09:43.691194	4	4	82352	\N	\N	1	f	7	79	0	f
213	2019-11-25 18:01:04.170865	2019-11-25 18:01:12.767873	10	10	24285	\N	\N	1	f	9	\N	0	f
198	2019-11-25 18:01:00.495405	2019-11-25 18:01:08.339222	10	10	46131	\N	\N	1	f	9	79	0	f
232	2020-05-12 16:13:52.645524	2020-05-12 16:13:52.674604	11	11	0	\N	\N	1	t	10	\N	0	f
215	2019-11-25 18:01:04.707826	2019-11-25 18:01:13.227997	10	10	70324	\N	\N	1	f	9	79	0	f
200	2019-11-25 18:01:00.961537	2019-11-25 18:01:08.764901	10	10	84197	\N	\N	1	f	9	92	0	f
219	2020-05-12 16:13:49.472249	2020-05-12 16:13:57.225393	11	11	598154	\N	\N	1	f	10	97	0	f
216	2019-11-25 18:01:04.974272	2019-11-25 18:01:13.60016	10	10	82352	\N	\N	1	f	9	79	0	f
202	2019-11-25 18:01:01.6076	2019-11-25 18:01:09.165232	10	10	57335	\N	\N	1	f	9	79	0	f
197	2019-11-25 18:01:00.169918	2019-11-25 18:21:19.146378	10	10	121482	\N	\N	1	f	9	\N	0	f
211	2019-11-25 18:01:03.842899	2019-11-25 18:01:03.882382	10	10	0	\N	\N	1	t	9	\N	0	f
203	2019-11-25 18:01:01.841535	2019-11-25 18:01:09.466721	10	10	210623	\N	\N	1	f	9	99	0	f
210	2019-11-25 18:01:03.690839	2019-11-25 18:21:20.16268	10	10	409332	\N	\N	1	f	9	\N	0	f
223	2020-05-12 16:13:50.341629	2020-05-12 16:13:50.371362	11	11	0	\N	\N	1	t	10	\N	0	f
212	2019-11-25 18:01:04.011143	2019-11-25 18:01:04.044893	10	10	51251	\N	\N	1	t	9	\N	0	f
230	2020-05-12 16:13:52.000945	2020-05-12 16:14:01.109373	11	11	205748	\N	\N	1	f	10	79	0	f
204	2019-11-25 18:01:02.202824	2019-11-25 18:01:10.050366	10	10	606138	\N	\N	1	f	9	96	0	f
217	2020-05-12 16:13:47.983075	2020-05-12 16:13:55.885596	11	11	534404	\N	\N	1	f	10	99	0	f
220	2020-05-12 16:13:49.74338	2020-05-12 16:13:57.475792	11	11	17875	\N	\N	1	f	10	94	0	f
205	2019-11-25 18:01:02.503641	2019-11-25 18:01:10.346556	10	10	260422	\N	\N	1	f	9	100	0	f
226	2020-05-12 16:13:50.931122	2020-05-12 16:13:59.489223	11	11	57335	\N	\N	1	f	10	79	0	f
193	2019-11-25 18:00:59.149879	2019-11-25 18:01:05.886838	10	10	534404	\N	\N	1	f	9	99	0	f
206	2019-11-25 18:01:02.761097	2019-11-25 18:01:10.709652	10	10	205748	\N	\N	1	f	9	79	0	f
194	2019-11-25 18:00:59.43726	2019-11-25 18:01:06.502036	10	10	596394	\N	\N	1	f	9	99	0	f
234	2020-05-12 16:13:52.915017	2020-05-12 16:34:15.455064	11	11	409332	\N	\N	1	f	10	\N	0	f
195	2019-11-25 18:00:59.73352	2019-11-25 18:01:07.216074	10	10	598154	\N	\N	1	f	9	97	0	f
207	2019-11-25 18:01:03.053009	2019-11-25 18:01:10.950725	10	10	11317	\N	\N	1	f	9	74	0	f
196	2019-11-25 18:01:00.031103	2019-11-25 18:01:07.464093	10	10	17875	\N	\N	1	f	9	94	0	f
224	2020-05-12 16:13:50.568235	2020-05-12 16:13:59.021886	11	11	84197	\N	\N	1	f	10	92	0	f
209	2019-11-25 18:01:03.521085	2019-11-25 18:01:11.476455	10	10	133004	\N	\N	1	f	9	88	0	f
229	2020-05-12 16:13:51.714516	2020-05-12 16:14:00.661667	11	11	260422	\N	\N	1	f	10	100	0	f
225	2020-05-12 16:13:50.752457	2020-05-12 16:13:50.804867	11	11	0	\N	\N	1	t	10	\N	0	f
228	2020-05-12 16:13:51.386645	2020-05-12 16:14:00.353024	11	11	606138	\N	\N	1	f	10	96	0	f
227	2020-05-12 16:13:51.085501	2020-05-12 16:13:59.789818	11	11	210623	\N	\N	1	f	10	99	0	f
233	2020-05-12 16:13:52.798061	2020-05-12 16:14:02.169226	11	11	133004	\N	\N	1	f	10	88	0	f
231	2020-05-12 16:13:52.351557	2020-05-12 16:14:01.366167	11	11	11317	\N	\N	1	f	10	74	0	f
236	2020-05-12 16:13:53.35006	2020-05-12 16:13:53.415032	11	11	51251	\N	\N	1	t	10	\N	0	f
235	2020-05-12 16:13:53.07641	2020-05-12 16:13:53.121687	11	11	0	\N	\N	1	t	10	\N	0	f
221	2020-05-12 16:13:49.917079	2020-05-12 16:34:13.858145	11	11	121482	\N	\N	1	f	10	\N	0	f
280	2020-10-07 19:20:37.256924	2020-10-07 19:20:37.290785	16	16	0	\N	\N	1	t	13	\N	0	f
265	2020-10-07 19:20:32.968303	2020-10-07 19:20:40.439824	16	16	534404	\N	\N	1	f	13	99	0	f
254	2020-06-10 14:44:22.372773	2020-06-10 14:44:31.870769	12	12	205748	\N	\N	1	f	11	79	0	f
247	2020-06-10 14:44:20.988857	2020-06-10 14:44:21.032132	12	12	0	\N	\N	1	t	11	\N	0	f
238	2020-05-12 16:13:53.792927	2020-05-12 16:13:53.825053	11	11	62515	\N	\N	1	t	10	\N	0	f
282	2020-10-07 19:20:37.559517	2020-10-07 19:40:57.545855	16	16	409332	\N	\N	1	f	13	\N	0	f
243	2020-06-10 14:44:20.106313	2020-06-10 14:44:26.930645	12	12	598154	\N	\N	1	f	11	97	0	f
300	2020-11-18 14:50:57.668824	2020-11-18 14:51:06.38926	1	1	606138	\N	\N	1	f	12	96	0	f
249	2020-06-10 14:44:21.315848	2020-06-10 14:44:21.348482	12	12	0	\N	\N	1	t	11	\N	0	f
255	2020-06-10 14:44:22.638535	2020-06-10 14:44:32.170173	12	12	11317	\N	\N	1	f	11	74	0	f
237	2020-05-12 16:13:53.537283	2020-05-12 16:14:03.677086	11	11	24285	\N	\N	1	f	10	\N	0	f
256	2020-06-10 14:44:22.901693	2020-06-10 14:44:22.931929	12	12	0	\N	\N	1	t	11	\N	0	f
239	2020-05-12 16:13:54.052478	2020-05-12 16:14:04.16452	11	11	70324	\N	\N	1	f	10	79	0	f
240	2020-05-12 16:13:54.335017	2020-05-12 16:14:04.50044	11	11	82352	\N	\N	1	f	10	79	0	f
244	2020-06-10 14:44:20.364049	2020-06-10 14:44:27.152815	12	12	17875	\N	\N	1	f	11	94	0	f
262	2020-06-10 14:44:23.87736	2020-06-10 14:44:23.90836	12	12	62515	\N	\N	1	t	11	\N	0	f
257	2020-06-10 14:44:23.051169	2020-06-10 14:44:32.886971	12	12	133004	\N	\N	1	f	11	88	0	f
272	2020-10-07 19:20:35.274222	2020-10-07 19:20:43.539651	16	16	84197	\N	\N	1	f	13	92	0	f
266	2020-10-07 19:20:33.796675	2020-10-07 19:20:41.010863	16	16	596394	\N	\N	1	f	13	99	0	f
276	2020-10-07 19:20:36.053537	2020-10-07 19:20:44.789275	16	16	606138	\N	\N	1	f	13	96	0	f
261	2020-06-10 14:44:23.633768	2020-06-10 14:44:34.546065	12	12	24285	\N	\N	1	f	11	\N	0	f
270	2020-10-07 19:20:34.931887	2020-10-07 19:20:43.095396	16	16	46131	\N	\N	1	f	13	79	0	f
267	2020-10-07 19:20:34.097252	2020-10-07 19:20:41.85743	16	16	598154	\N	\N	1	f	13	97	0	f
263	2020-06-10 14:44:24.120484	2020-06-10 14:44:34.937337	12	12	70324	\N	\N	1	f	11	79	0	f
259	2020-06-10 14:44:23.292851	2020-06-10 14:44:23.32804	12	12	0	\N	\N	1	t	11	\N	0	f
246	2020-06-10 14:44:20.829905	2020-06-10 14:44:28.071419	12	12	46131	\N	\N	1	f	11	79	0	f
273	2020-10-07 19:20:35.447312	2020-10-07 19:20:35.480899	16	16	0	\N	\N	1	t	13	\N	0	f
260	2020-06-10 14:44:23.47028	2020-06-10 14:44:23.503368	12	12	51251	\N	\N	1	t	11	\N	0	f
264	2020-06-10 14:44:24.362075	2020-06-10 14:44:35.249499	12	12	82352	\N	\N	1	f	11	79	0	f
248	2020-06-10 14:44:21.151929	2020-06-10 14:44:28.460809	12	12	84197	\N	\N	1	f	11	92	0	f
245	2020-06-10 14:44:20.548131	2020-06-10 15:04:42.987459	12	12	121482	\N	\N	1	f	11	\N	0	f
250	2020-06-10 14:44:21.468419	2020-06-10 14:44:28.99738	12	12	57335	\N	\N	1	f	11	79	0	f
271	2020-10-07 19:20:35.119027	2020-10-07 19:20:35.14711	16	16	0	\N	\N	1	t	13	\N	0	f
258	2020-06-10 14:44:23.153067	2020-06-10 15:04:44.190215	12	12	409332	\N	\N	1	f	11	\N	0	f
241	2020-06-10 14:44:18.42216	2020-06-10 14:44:25.669567	12	12	534404	\N	\N	1	f	11	99	0	f
251	2020-06-10 14:44:21.63088	2020-06-10 14:44:29.486601	12	12	210623	\N	\N	1	f	11	99	0	f
242	2020-06-10 14:44:19.818982	2020-06-10 14:44:26.209218	12	12	596394	\N	\N	1	f	11	99	0	f
252	2020-06-10 14:44:21.889922	2020-06-10 14:44:30.6103	12	12	606138	\N	\N	1	f	11	96	0	f
253	2020-06-10 14:44:22.133513	2020-06-10 14:44:31.200559	12	12	260422	\N	\N	1	f	11	100	0	f
268	2020-10-07 19:20:34.370689	2020-10-07 19:20:42.145826	16	16	17875	\N	\N	1	f	13	94	0	f
275	2020-10-07 19:20:35.776434	2020-10-07 19:20:44.254036	16	16	210623	\N	\N	1	f	13	99	0	f
274	2020-10-07 19:20:35.613019	2020-10-07 19:20:43.948748	16	16	57335	\N	\N	1	f	13	79	0	f
277	2020-10-07 19:20:36.362535	2020-10-07 19:20:45.10447	16	16	260422	\N	\N	1	f	13	100	0	f
281	2020-10-07 19:20:37.419314	2020-10-07 19:20:46.335926	16	16	133004	\N	\N	1	f	13	88	0	f
279	2020-10-07 19:20:36.942586	2020-10-07 19:20:45.726881	16	16	11317	\N	\N	1	f	13	74	0	f
278	2020-10-07 19:20:36.676555	2020-10-07 19:20:45.491039	16	16	205748	\N	\N	1	f	13	79	0	f
269	2020-10-07 19:20:34.672052	2020-10-07 19:40:56.495441	16	16	121482	\N	\N	1	f	13	\N	0	f
283	2020-10-07 19:20:37.730033	2020-10-07 19:20:37.76298	16	16	0	\N	\N	1	t	13	\N	0	f
298	2020-11-18 14:50:57.222996	2020-11-18 14:51:05.426604	1	1	57335	\N	\N	1	f	12	79	0	f
284	2020-10-07 19:20:37.899769	2020-10-07 19:20:37.959534	16	16	51251	\N	\N	1	t	13	\N	0	f
309	2020-11-18 14:50:59.651399	2020-11-18 14:51:09.341228	1	1	24285	\N	\N	1	f	12	\N	0	f
299	2020-11-18 14:50:57.392569	2020-11-18 14:51:05.825511	1	1	210623	\N	\N	1	f	12	99	0	f
308	2020-11-18 14:50:59.480932	2020-11-18 14:50:59.519396	1	1	51251	\N	\N	1	t	12	\N	0	f
318	2020-11-18 14:54:13.540591	2020-11-18 14:54:21.18392	17	17	46131	\N	\N	1	f	14	79	0	f
286	2020-10-07 19:20:38.461097	2020-10-07 19:20:38.492545	16	16	62515	\N	\N	1	t	13	\N	0	f
311	2020-11-18 14:51:00.307218	2020-11-18 14:51:09.773022	1	1	70324	\N	\N	1	f	12	79	0	f
295	2020-11-18 14:50:56.721302	2020-11-18 14:50:56.754782	1	1	0	\N	\N	1	t	12	\N	0	f
315	2020-11-18 14:54:12.850973	2020-11-18 14:54:19.947602	17	17	598154	\N	\N	1	f	14	97	0	f
285	2020-10-07 19:20:38.205315	2020-10-07 19:20:47.761627	16	16	24285	\N	\N	1	f	13	\N	0	f
312	2020-11-18 14:51:00.63166	2020-11-18 14:51:10.108415	1	1	82352	\N	\N	1	f	12	79	0	f
287	2020-10-07 19:20:38.728609	2020-10-07 19:20:48.20287	16	16	70324	\N	\N	1	f	13	79	0	f
297	2020-11-18 14:50:57.054772	2020-11-18 14:50:57.082425	1	1	0	\N	\N	1	t	12	\N	0	f
288	2020-10-07 19:20:38.99022	2020-10-07 19:20:48.532453	16	16	82352	\N	\N	1	f	13	79	0	f
301	2020-11-18 14:50:57.957255	2020-11-18 14:51:06.685039	1	1	260422	\N	\N	1	f	12	100	0	f
302	2020-11-18 14:50:58.244911	2020-11-18 14:51:07.147621	1	1	205748	\N	\N	1	f	12	79	0	f
289	2020-11-18 14:50:55.236659	2020-11-18 14:51:02.03051	1	1	534404	\N	\N	1	f	12	99	0	f
321	2020-11-18 14:54:14.092754	2020-11-18 14:54:14.136112	17	17	0	\N	\N	1	t	14	\N	0	f
313	2020-11-18 14:54:12.20889	2020-11-18 14:54:18.65513	17	17	534404	\N	\N	1	f	14	99	0	f
304	2020-11-18 14:50:58.858887	2020-11-18 14:50:58.889244	1	1	0	\N	\N	1	t	12	\N	0	f
303	2020-11-18 14:50:58.517823	2020-11-18 14:51:07.431372	1	1	11317	\N	\N	1	f	12	74	0	f
290	2020-11-18 14:50:55.575895	2020-11-18 14:51:02.636345	1	1	596394	\N	\N	1	f	12	99	0	f
319	2020-11-18 14:54:13.714361	2020-11-18 14:54:13.766514	17	17	0	\N	\N	1	t	14	\N	0	f
310	2020-11-18 14:50:59.93897	2020-11-18 14:50:59.974406	1	1	62515	\N	\N	1	t	12	\N	0	f
316	2020-11-18 14:54:13.12811	2020-11-18 14:54:20.257816	17	17	17875	\N	\N	1	f	14	94	0	f
307	2020-11-18 14:50:59.314723	2020-11-18 14:50:59.34478	1	1	0	\N	\N	1	t	12	\N	0	f
314	2020-11-18 14:54:12.554927	2020-11-18 14:54:19.218575	17	17	596394	\N	\N	1	f	14	99	0	f
291	2020-11-18 14:50:55.85645	2020-11-18 14:51:03.402476	1	1	598154	\N	\N	1	f	12	97	0	f
305	2020-11-18 14:50:59.022991	2020-11-18 14:51:07.984253	1	1	133004	\N	\N	1	f	12	88	0	f
292	2020-11-18 14:50:56.154892	2020-11-18 14:51:03.649544	1	1	17875	\N	\N	1	f	12	94	0	f
317	2020-11-18 14:54:13.264989	2020-11-18 15:14:39.900595	17	17	121482	\N	\N	1	f	14	\N	0	f
306	2020-11-18 14:50:59.146271	2020-11-18 15:11:37.517169	1	1	409332	\N	\N	1	f	12	\N	0	f
330	2020-11-18 14:54:16.281573	2020-11-18 15:14:41.136614	17	17	409332	\N	\N	1	f	14	\N	0	f
294	2020-11-18 14:50:56.52922	2020-11-18 14:51:04.577607	1	1	46131	\N	\N	1	f	12	79	0	f
296	2020-11-18 14:50:56.887643	2020-11-18 14:51:04.993637	1	1	84197	\N	\N	1	f	12	92	0	f
328	2020-11-18 14:54:15.984989	2020-11-18 14:54:16.022178	17	17	0	\N	\N	1	t	14	\N	0	f
320	2020-11-18 14:54:13.910476	2020-11-18 14:54:21.72305	17	17	84197	\N	\N	1	f	14	92	0	f
323	2020-11-18 14:54:14.462243	2020-11-18 14:54:22.446988	17	17	210623	\N	\N	1	f	14	99	0	f
322	2020-11-18 14:54:14.276364	2020-11-18 14:54:22.143955	17	17	57335	\N	\N	1	f	14	79	0	f
324	2020-11-18 14:54:14.740019	2020-11-18 14:54:23.036393	17	17	606138	\N	\N	1	f	14	96	0	f
327	2020-11-18 14:54:15.698479	2020-11-18 14:54:23.976569	17	17	11317	\N	\N	1	f	14	74	0	f
326	2020-11-18 14:54:15.398098	2020-11-18 14:54:23.719023	17	17	205748	\N	\N	1	f	14	79	0	f
325	2020-11-18 14:54:15.091418	2020-11-18 14:54:23.327608	17	17	260422	\N	\N	1	f	14	100	0	f
329	2020-11-18 14:54:16.156506	2020-11-18 14:54:24.662995	17	17	133004	\N	\N	1	f	14	88	0	f
293	2020-11-18 14:50:56.273576	2020-11-18 15:11:36.363426	1	1	121482	\N	\N	1	f	12	\N	0	f
569	2021-08-23 14:58:59.042257	2021-08-23 14:59:05.238035	30	30	205748	\N	\N	1	f	24	79	0	f
371	2021-05-20 14:13:31.570376	2021-05-20 14:13:31.594782	21	21	0	\N	\N	1	t	16	\N	0	f
331	2020-11-18 14:54:16.454472	2020-11-18 14:54:16.487228	17	17	0	\N	\N	1	t	14	\N	0	f
341	2021-05-19 20:01:56.800539	2021-05-19 20:02:05.90501	20	20	598154	\N	\N	1	f	15	97	0	f
332	2020-11-18 14:54:16.622074	2020-11-18 14:54:16.670111	17	17	51251	\N	\N	1	t	14	\N	0	f
362	2021-05-19 20:02:03.824498	2021-05-19 20:02:11.963116	20	20	82352	\N	\N	1	f	15	79	0	f
360	2021-05-19 20:02:03.313856	2021-05-19 20:02:03.351382	20	20	62515	\N	\N	1	t	15	\N	0	f
351	2021-05-19 20:02:00.034324	2021-05-19 20:02:08.682434	20	20	260422	\N	\N	1	f	15	100	0	f
334	2020-11-18 14:54:17.114907	2020-11-18 14:54:17.155158	17	17	62515	\N	\N	1	t	14	\N	0	f
343	2021-05-19 20:01:57.157572	2021-05-19 20:22:18.097033	20	20	121482	\N	\N	1	f	15	\N	0	f
356	2021-05-19 20:02:02.606782	2021-05-19 20:22:18.905668	20	20	409332	\N	\N	1	f	15	\N	0	f
352	2021-05-19 20:02:00.890224	2021-05-19 20:02:09.054697	20	20	205748	\N	\N	1	f	15	79	0	f
333	2020-11-18 14:54:16.812237	2020-11-18 14:54:26.072057	17	17	24285	\N	\N	1	f	14	\N	0	f
335	2020-11-18 14:54:17.404238	2020-11-18 14:54:26.550688	17	17	70324	\N	\N	1	f	14	79	0	f
336	2020-11-18 14:54:17.681444	2020-11-18 14:54:26.890918	17	17	82352	\N	\N	1	f	14	79	0	f
353	2021-05-19 20:02:02.066554	2021-05-19 20:02:09.284301	20	20	11317	\N	\N	1	f	15	74	0	f
357	2021-05-19 20:02:02.747897	2021-05-19 20:02:02.774946	20	20	0	\N	\N	1	t	15	\N	0	f
345	2021-05-19 20:01:57.466963	2021-05-19 20:01:57.488502	20	20	0	\N	\N	1	t	15	\N	0	f
338	2021-04-08 15:16:39.311302	2021-04-08 15:16:39.341153	4	4	0	\N	\N	1	t	7	\N	0	f
342	2021-05-19 20:01:57.003487	2021-05-19 20:02:06.140206	20	20	17875	\N	\N	1	f	15	94	0	f
370	2021-05-20 14:13:31.435853	2021-05-20 14:13:39.92312	21	21	84197	\N	\N	1	f	16	92	0	f
363	2021-05-20 14:13:30.351084	2021-05-20 14:13:38.111153	21	21	534404	\N	\N	1	f	16	99	0	f
355	2021-05-19 20:02:02.494714	2021-05-19 20:02:09.761848	20	20	133004	\N	\N	1	f	15	88	0	f
347	2021-05-19 20:01:57.676489	2021-05-19 20:01:57.695234	20	20	0	\N	\N	1	t	15	\N	0	f
365	2021-05-20 14:13:30.748985	2021-05-20 14:13:38.92507	21	21	598154	\N	\N	1	f	16	97	0	f
358	2021-05-19 20:02:02.907962	2021-05-19 20:02:02.944466	20	20	51251	\N	\N	1	t	15	\N	0	f
368	2021-05-20 14:13:31.204604	2021-05-20 14:13:39.650339	21	21	46131	\N	\N	1	f	16	79	0	f
354	2021-05-19 20:02:02.335514	2021-05-19 20:02:02.372346	20	20	0	\N	\N	1	t	15	\N	0	f
344	2021-05-19 20:01:57.346537	2021-05-19 20:02:06.928244	20	20	46131	\N	\N	1	f	15	79	0	f
364	2021-05-20 14:13:30.550429	2021-05-20 14:13:38.476589	21	21	596394	\N	\N	1	f	16	99	0	f
346	2021-05-19 20:01:57.570261	2021-05-19 20:02:07.327674	20	20	84197	\N	\N	1	f	15	92	0	f
359	2021-05-19 20:02:03.068318	2021-05-19 20:02:11.219642	20	20	24285	\N	\N	1	f	15	\N	0	f
366	2021-05-20 14:13:30.931867	2021-05-20 14:13:39.079342	21	21	17875	\N	\N	1	f	16	94	0	f
348	2021-05-19 20:01:57.802153	2021-05-19 20:02:07.72356	20	20	57335	\N	\N	1	f	15	79	0	f
339	2021-05-19 20:01:55.967234	2021-05-19 20:02:04.852909	20	20	534404	\N	\N	1	f	15	99	0	f
340	2021-05-19 20:01:56.567994	2021-05-19 20:02:05.301449	20	20	596394	\N	\N	1	f	15	99	0	f
361	2021-05-19 20:02:03.563405	2021-05-19 20:02:11.623704	20	20	70324	\N	\N	1	f	15	79	0	f
349	2021-05-19 20:01:58.756406	2021-05-19 20:02:08.009568	20	20	210623	\N	\N	1	f	15	99	0	f
350	2021-05-19 20:01:59.371745	2021-05-19 20:02:08.417848	20	20	606138	\N	\N	1	f	15	96	0	f
369	2021-05-20 14:13:31.320042	2021-05-20 14:13:31.346084	21	21	0	\N	\N	1	t	16	\N	0	f
372	2021-05-20 14:13:31.67997	2021-05-20 14:13:40.180792	21	21	57335	\N	\N	1	f	16	79	0	f
378	2021-05-20 14:13:32.720232	2021-05-20 14:13:32.739405	21	21	0	\N	\N	1	t	16	\N	0	f
374	2021-05-20 14:13:31.969802	2021-05-20 14:13:40.669225	21	21	606138	\N	\N	1	f	16	96	0	f
373	2021-05-20 14:13:31.784787	2021-05-20 14:13:40.367817	21	21	210623	\N	\N	1	f	16	99	0	f
375	2021-05-20 14:13:32.148917	2021-05-20 14:13:40.843551	21	21	260422	\N	\N	1	f	16	100	0	f
376	2021-05-20 14:13:32.345979	2021-05-20 14:13:41.096637	21	21	205748	\N	\N	1	f	16	79	0	f
377	2021-05-20 14:13:32.538973	2021-05-20 14:13:41.242131	21	21	11317	\N	\N	1	f	16	74	0	f
367	2021-05-20 14:13:31.01573	2021-05-20 14:34:27.941843	21	21	121482	\N	\N	1	f	16	\N	0	f
419	2021-05-20 14:13:45.284429	2021-05-20 14:13:45.30227	23	23	0	\N	\N	1	t	18	\N	0	f
379	2021-05-20 14:13:32.824626	2021-05-20 14:13:41.598848	21	21	133004	\N	\N	1	f	16	88	0	f
402	2021-05-20 14:13:36.409596	2021-05-20 14:13:36.431491	22	22	0	\N	\N	1	t	17	\N	0	f
413	2021-05-20 14:13:44.105623	2021-05-20 14:13:52.981484	23	23	598154	\N	\N	1	f	18	97	0	f
396	2021-05-20 14:13:35.357008	2021-05-20 14:13:48.937104	22	22	57335	\N	\N	1	f	17	79	0	f
381	2021-05-20 14:13:33.002012	2021-05-20 14:13:33.023243	21	21	0	\N	\N	1	t	16	\N	0	f
390	2021-05-20 14:13:34.678202	2021-05-20 14:13:46.56371	22	22	17875	\N	\N	1	f	17	94	0	f
382	2021-05-20 14:13:33.104974	2021-05-20 14:13:33.141481	21	21	51251	\N	\N	1	t	16	\N	0	f
404	2021-05-20 14:13:36.595937	2021-05-20 14:34:29.383346	22	22	409332	\N	\N	1	f	17	\N	0	f
411	2021-05-20 14:13:43.748626	2021-05-20 14:13:52.116312	23	23	534404	\N	\N	1	f	18	99	0	f
384	2021-05-20 14:13:33.416353	2021-05-20 14:13:33.442059	21	21	62515	\N	\N	1	t	16	\N	0	f
397	2021-05-20 14:13:35.477683	2021-05-20 14:13:49.127515	22	22	210623	\N	\N	1	f	17	99	0	f
392	2021-05-20 14:13:34.938877	2021-05-20 14:13:48.361378	22	22	46131	\N	\N	1	f	17	79	0	f
391	2021-05-20 14:13:34.760533	2021-05-20 14:34:28.862574	22	22	121482	\N	\N	1	f	17	\N	0	f
393	2021-05-20 14:13:35.047155	2021-05-20 14:13:35.066955	22	22	0	\N	\N	1	t	17	\N	0	f
403	2021-05-20 14:13:36.516473	2021-05-20 14:13:50.398217	22	22	133004	\N	\N	1	f	17	88	0	f
418	2021-05-20 14:13:45.182051	2021-05-20 14:13:54.018994	23	23	84197	\N	\N	1	f	18	92	0	f
398	2021-05-20 14:13:35.65813	2021-05-20 14:13:49.433473	22	22	606138	\N	\N	1	f	17	96	0	f
416	2021-05-20 14:13:44.980158	2021-05-20 14:13:53.746963	23	23	46131	\N	\N	1	f	18	79	0	f
412	2021-05-20 14:13:43.920994	2021-05-20 14:13:52.518065	23	23	596394	\N	\N	1	f	18	99	0	f
415	2021-05-20 14:13:44.348942	2021-05-20 14:34:30.257273	23	23	121482	\N	\N	1	f	18	\N	0	f
399	2021-05-20 14:13:35.854502	2021-05-20 14:13:49.64901	22	22	260422	\N	\N	1	f	17	100	0	f
408	2021-05-20 14:13:37.14099	2021-05-20 14:13:37.185547	22	22	62515	\N	\N	1	t	17	\N	0	f
394	2021-05-20 14:13:35.151893	2021-05-20 14:13:48.632497	22	22	84197	\N	\N	1	f	17	92	0	f
405	2021-05-20 14:13:36.707217	2021-05-20 14:13:36.732157	22	22	0	\N	\N	1	t	17	\N	0	f
395	2021-05-20 14:13:35.255969	2021-05-20 14:13:35.276289	22	22	0	\N	\N	1	t	17	\N	0	f
400	2021-05-20 14:13:36.047462	2021-05-20 14:13:49.884055	22	22	205748	\N	\N	1	f	17	79	0	f
383	2021-05-20 14:13:33.240283	2021-05-20 14:13:42.488298	21	21	24285	\N	\N	1	f	16	\N	0	f
406	2021-05-20 14:13:36.81973	2021-05-20 14:13:36.851724	22	22	51251	\N	\N	1	t	17	\N	0	f
401	2021-05-20 14:13:36.22597	2021-05-20 14:13:50.029476	22	22	11317	\N	\N	1	f	17	74	0	f
385	2021-05-20 14:13:33.595758	2021-05-20 14:13:42.754543	21	21	70324	\N	\N	1	f	16	79	0	f
409	2021-05-20 14:13:37.342229	2021-05-20 14:13:51.576946	22	22	70324	\N	\N	1	f	17	79	0	f
407	2021-05-20 14:13:36.941	2021-05-20 14:13:51.310425	22	22	24285	\N	\N	1	f	17	\N	0	f
414	2021-05-20 14:13:44.278136	2021-05-20 14:13:53.141897	23	23	17875	\N	\N	1	f	18	94	0	f
386	2021-05-20 14:13:33.780218	2021-05-20 14:13:42.962948	21	21	82352	\N	\N	1	f	16	79	0	f
410	2021-05-20 14:13:37.544132	2021-05-20 14:13:51.786639	22	22	82352	\N	\N	1	f	17	79	0	f
387	2021-05-20 14:13:34.122674	2021-05-20 14:13:43.301405	22	22	534404	\N	\N	1	f	17	99	0	f
388	2021-05-20 14:13:34.31108	2021-05-20 14:13:43.655212	22	22	596394	\N	\N	1	f	17	99	0	f
417	2021-05-20 14:13:45.08126	2021-05-20 14:13:45.100997	23	23	0	\N	\N	1	t	18	\N	0	f
389	2021-05-20 14:13:34.490546	2021-05-20 14:13:44.830325	22	22	598154	\N	\N	1	f	17	97	0	f
421	2021-05-20 14:13:45.505948	2021-05-20 14:13:54.483622	23	23	210623	\N	\N	1	f	18	99	0	f
420	2021-05-20 14:13:45.381504	2021-05-20 14:13:54.299772	23	23	57335	\N	\N	1	f	18	79	0	f
422	2021-05-20 14:13:45.679357	2021-05-20 14:13:54.78883	23	23	606138	\N	\N	1	f	18	96	0	f
423	2021-05-20 14:13:45.85463	2021-05-20 14:13:55.050129	23	23	260422	\N	\N	1	f	18	100	0	f
424	2021-05-20 14:13:46.026804	2021-05-20 14:13:55.384687	23	23	205748	\N	\N	1	f	18	79	0	f
425	2021-05-20 14:13:46.201121	2021-05-20 14:13:55.551067	23	23	11317	\N	\N	1	f	18	74	0	f
380	2021-05-20 14:13:32.895998	2021-05-20 14:34:28.619535	21	21	409332	\N	\N	1	f	16	\N	0	f
473	2021-05-20 14:31:17.820808	2021-05-20 14:31:23.446934	25	25	11317	\N	\N	1	f	20	74	0	f
455	2021-05-20 14:22:05.682666	2021-05-20 14:22:12.428352	24	24	24285	\N	\N	1	f	19	\N	0	f
433	2021-05-20 14:13:47.781396	2021-05-20 14:13:57.180941	23	23	70324	\N	\N	1	f	18	79	0	f
426	2021-05-20 14:13:46.379071	2021-05-20 14:13:46.41082	23	23	0	\N	\N	1	t	18	\N	0	f
443	2021-05-20 14:22:03.969914	2021-05-20 14:22:03.995616	24	24	0	\N	\N	1	t	19	\N	0	f
434	2021-05-20 14:13:47.957768	2021-05-20 14:13:57.390532	23	23	82352	\N	\N	1	f	18	79	0	f
438	2021-05-20 14:22:03.337961	2021-05-20 14:22:08.89482	24	24	17875	\N	\N	1	f	19	94	0	f
429	2021-05-20 14:13:46.813841	2021-05-20 14:13:46.832514	23	23	0	\N	\N	1	t	18	\N	0	f
454	2021-05-20 14:22:05.571712	2021-05-20 14:22:05.598084	24	24	51251	\N	\N	1	t	19	\N	0	f
430	2021-05-20 14:13:46.914685	2021-05-20 14:13:46.938456	23	23	51251	\N	\N	1	t	18	\N	0	f
446	2021-05-20 14:22:04.400116	2021-05-20 14:22:10.554536	24	24	606138	\N	\N	1	f	19	96	0	f
465	2021-05-20 14:31:16.589511	2021-05-20 14:31:16.614463	25	25	0	\N	\N	1	t	20	\N	0	f
450	2021-05-20 14:22:05.166446	2021-05-20 14:22:05.188553	24	24	0	\N	\N	1	t	19	\N	0	f
432	2021-05-20 14:13:47.208212	2021-05-20 14:13:47.230571	23	23	62515	\N	\N	1	t	18	\N	0	f
447	2021-05-20 14:22:04.601246	2021-05-20 14:22:10.744029	24	24	260422	\N	\N	1	f	19	100	0	f
452	2021-05-20 14:22:05.351871	2021-05-20 14:42:33.424945	24	24	409332	\N	\N	1	f	19	\N	0	f
457	2021-05-20 14:22:06.058061	2021-05-20 14:22:12.733824	24	24	70324	\N	\N	1	f	19	79	0	f
448	2021-05-20 14:22:04.786883	2021-05-20 14:22:10.990515	24	24	205748	\N	\N	1	f	19	79	0	f
427	2021-05-20 14:13:46.645177	2021-05-20 14:13:55.902166	23	23	133004	\N	\N	1	f	18	88	0	f
439	2021-05-20 14:22:03.435801	2021-05-20 14:42:32.840651	24	24	121482	\N	\N	1	f	19	\N	0	f
431	2021-05-20 14:13:47.02738	2021-05-20 14:13:56.818729	23	23	24285	\N	\N	1	f	18	\N	0	f
460	2021-05-20 14:31:15.82142	2021-05-20 14:31:20.576237	25	25	596394	\N	\N	1	f	20	99	0	f
458	2021-05-20 14:22:06.240295	2021-05-20 14:22:12.948838	24	24	82352	\N	\N	1	f	19	79	0	f
449	2021-05-20 14:22:04.974784	2021-05-20 14:22:11.155062	24	24	11317	\N	\N	1	f	19	74	0	f
441	2021-05-20 14:22:03.740299	2021-05-20 14:22:03.761184	24	24	0	\N	\N	1	t	19	\N	0	f
462	2021-05-20 14:31:16.207059	2021-05-20 14:31:21.185707	25	25	17875	\N	\N	1	f	20	94	0	f
459	2021-05-20 14:31:15.618954	2021-05-20 14:31:20.218446	25	25	534404	\N	\N	1	f	20	99	0	f
453	2021-05-20 14:22:05.465224	2021-05-20 14:22:05.48507	24	24	0	\N	\N	1	t	19	\N	0	f
440	2021-05-20 14:22:03.623489	2021-05-20 14:22:09.484357	24	24	46131	\N	\N	1	f	19	79	0	f
451	2021-05-20 14:22:05.277642	2021-05-20 14:22:11.519906	24	24	133004	\N	\N	1	f	19	88	0	f
442	2021-05-20 14:22:03.849893	2021-05-20 14:22:09.769805	24	24	84197	\N	\N	1	f	19	92	0	f
456	2021-05-20 14:22:05.871797	2021-05-20 14:22:05.898064	24	24	62515	\N	\N	1	t	19	\N	0	f
464	2021-05-20 14:31:16.47093	2021-05-20 14:31:21.77797	25	25	46131	\N	\N	1	f	20	79	0	f
435	2021-05-20 14:22:02.742134	2021-05-20 14:22:07.937946	24	24	534404	\N	\N	1	f	19	99	0	f
444	2021-05-20 14:22:04.083966	2021-05-20 14:22:10.0486	24	24	57335	\N	\N	1	f	19	79	0	f
463	2021-05-20 14:31:16.292227	2021-05-20 14:51:34.891611	25	25	121482	\N	\N	1	f	20	\N	0	f
436	2021-05-20 14:22:02.948776	2021-05-20 14:22:08.289286	24	24	596394	\N	\N	1	f	19	99	0	f
445	2021-05-20 14:22:04.210528	2021-05-20 14:22:10.241149	24	24	210623	\N	\N	1	f	19	99	0	f
437	2021-05-20 14:22:03.149102	2021-05-20 14:22:08.743455	24	24	598154	\N	\N	1	f	19	97	0	f
461	2021-05-20 14:31:16.020487	2021-05-20 14:31:21.032103	25	25	598154	\N	\N	1	f	20	97	0	f
466	2021-05-20 14:31:16.701991	2021-05-20 14:31:22.056724	25	25	84197	\N	\N	1	f	20	92	0	f
467	2021-05-20 14:31:16.840926	2021-05-20 14:31:16.864796	25	25	0	\N	\N	1	t	20	\N	0	f
468	2021-05-20 14:31:16.950441	2021-05-20 14:31:22.333874	25	25	57335	\N	\N	1	f	20	79	0	f
469	2021-05-20 14:31:17.0608	2021-05-20 14:31:22.523681	25	25	210623	\N	\N	1	f	20	99	0	f
428	2021-05-20 14:13:46.714881	2021-05-20 14:34:30.572465	23	23	409332	\N	\N	1	f	18	\N	0	f
470	2021-05-20 14:31:17.250226	2021-05-20 14:31:22.838314	25	25	606138	\N	\N	1	f	20	96	0	f
471	2021-05-20 14:31:17.446436	2021-05-20 14:31:23.02748	25	25	260422	\N	\N	1	f	20	100	0	f
472	2021-05-20 14:31:17.634162	2021-05-20 14:31:23.288399	25	25	205748	\N	\N	1	f	20	79	0	f
490	2021-05-20 15:17:39.937417	2021-05-20 15:17:39.964223	26	26	0	\N	\N	1	t	21	\N	0	f
481	2021-05-20 14:31:18.929345	2021-05-20 14:31:25.04171	25	25	70324	\N	\N	1	f	20	79	0	f
474	2021-05-20 14:31:18.041257	2021-05-20 14:31:18.063473	25	25	0	\N	\N	1	t	20	\N	0	f
482	2021-05-20 14:31:19.109064	2021-05-20 14:31:25.261509	25	25	82352	\N	\N	1	f	20	79	0	f
502	2021-05-20 15:17:41.721342	2021-05-20 15:17:41.745265	26	26	0	\N	\N	1	t	21	\N	0	f
515	2021-05-28 15:08:29.940803	2021-05-28 15:08:58.554084	27	27	84197	\N	\N	1	f	22	92	0	f
483	2021-05-20 14:41:37.213887	2021-05-20 14:41:37.240267	24	\N	0	\N	\N	1	t	19	\N	0	f
477	2021-05-20 14:31:18.33825	2021-05-20 14:31:18.359095	25	25	0	\N	\N	1	t	20	\N	0	f
476	2021-05-20 14:31:18.22922	2021-05-20 14:51:35.172145	25	25	409332	\N	\N	1	f	20	\N	0	f
478	2021-05-20 14:31:18.450859	2021-05-20 14:31:18.476098	25	25	51251	\N	\N	1	t	20	\N	0	f
498	2021-05-20 15:17:41.210187	2021-05-20 15:17:46.422259	26	26	11317	\N	\N	1	f	21	74	0	f
485	2021-05-20 15:17:39.119355	2021-05-20 15:17:43.476637	26	26	596394	\N	\N	1	f	21	99	0	f
492	2021-05-20 15:17:40.184976	2021-05-20 15:17:40.213276	26	26	0	\N	\N	1	t	21	\N	0	f
480	2021-05-20 14:31:18.747428	2021-05-20 14:31:18.773285	25	25	62515	\N	\N	1	t	20	\N	0	f
493	2021-05-20 15:17:40.302907	2021-05-20 15:17:45.294379	26	26	57335	\N	\N	1	f	21	79	0	f
486	2021-05-20 15:17:39.325857	2021-05-20 15:17:43.928758	26	26	598154	\N	\N	1	f	21	97	0	f
507	2021-05-20 15:17:42.540851	2021-05-20 15:17:48.242315	26	26	82352	\N	\N	1	f	21	79	0	f
500	2021-05-20 15:17:41.533129	2021-05-20 15:17:46.785517	26	26	133004	\N	\N	1	f	21	88	0	f
494	2021-05-20 15:17:40.439693	2021-05-20 15:17:45.508541	26	26	210623	\N	\N	1	f	21	99	0	f
475	2021-05-20 14:31:18.150452	2021-05-20 14:31:23.826488	25	25	133004	\N	\N	1	f	20	88	0	f
488	2021-05-20 15:17:39.624507	2021-05-20 15:37:54.111944	26	26	121482	\N	\N	1	f	21	\N	0	f
501	2021-05-20 15:17:41.6131	2021-05-20 15:37:54.548036	26	26	409332	\N	\N	1	f	21	\N	0	f
479	2021-05-20 14:31:18.56614	2021-05-20 14:31:24.766384	25	25	24285	\N	\N	1	f	20	\N	0	f
514	2021-05-28 15:08:29.312342	2021-05-28 15:08:29.385369	27	27	0	\N	\N	1	t	22	\N	0	f
503	2021-05-20 15:17:41.832397	2021-05-20 15:17:41.856512	26	26	51251	\N	\N	1	t	21	\N	0	f
495	2021-05-20 15:17:40.63903	2021-05-20 15:17:45.833223	26	26	606138	\N	\N	1	f	21	96	0	f
504	2021-05-20 15:17:41.947178	2021-05-20 15:17:47.740901	26	26	24285	\N	\N	1	f	21	\N	0	f
499	2021-05-20 15:17:41.400015	2021-05-20 15:17:41.440152	26	26	0	\N	\N	1	t	21	\N	0	f
487	2021-05-20 15:17:39.530894	2021-05-20 15:17:44.087214	26	26	17875	\N	\N	1	f	21	94	0	f
509	2021-05-28 15:08:19.873921	2021-05-28 15:08:52.300275	27	27	596394	\N	\N	1	f	22	99	0	f
508	2021-05-28 15:08:17.802168	2021-05-28 15:08:50.990562	27	27	534404	\N	\N	1	f	22	99	0	f
496	2021-05-20 15:17:40.846747	2021-05-20 15:17:46.027754	26	26	260422	\N	\N	1	f	21	100	0	f
511	2021-05-28 15:08:25.343717	2021-05-28 15:08:54.210135	27	27	17875	\N	\N	1	f	22	94	0	f
506	2021-05-20 15:17:42.338571	2021-05-20 15:17:48.024257	26	26	70324	\N	\N	1	f	21	79	0	f
505	2021-05-20 15:17:42.141361	2021-05-20 15:17:42.171271	26	26	62515	\N	\N	1	t	21	\N	0	f
489	2021-05-20 15:17:39.818829	2021-05-20 15:17:44.682222	26	26	46131	\N	\N	1	f	21	79	0	f
484	2021-05-20 15:17:38.914644	2021-05-20 15:17:43.109128	26	26	534404	\N	\N	1	f	21	99	0	f
497	2021-05-20 15:17:41.02951	2021-05-20 15:17:46.266206	26	26	205748	\N	\N	1	f	21	79	0	f
513	2021-05-28 15:08:27.362003	2021-05-28 15:08:56.765097	27	27	46131	\N	\N	1	f	22	79	0	f
491	2021-05-20 15:17:40.05465	2021-05-20 15:17:45.001786	26	26	84197	\N	\N	1	f	21	92	0	f
510	2021-05-28 15:08:22.898814	2021-05-28 15:08:53.052302	27	27	598154	\N	\N	1	f	22	97	0	f
517	2021-05-28 15:08:31.460139	2021-05-28 15:08:59.180684	27	27	57335	\N	\N	1	f	22	79	0	f
516	2021-05-28 15:08:30.70487	2021-05-28 15:08:30.79545	27	27	0	\N	\N	1	t	22	\N	0	f
518	2021-05-28 15:08:32.186948	2021-05-28 15:08:59.733275	27	27	210623	\N	\N	1	f	22	99	0	f
512	2021-05-28 15:08:25.967013	2021-05-28 15:29:43.932234	27	27	121482	\N	\N	1	f	22	\N	0	f
519	2021-05-28 15:08:32.768244	2021-05-28 15:09:03.064922	27	27	606138	\N	\N	1	f	22	96	0	f
520	2021-05-28 15:08:33.186847	2021-05-28 15:09:03.510673	27	27	260422	\N	\N	1	f	22	100	0	f
553	2021-06-02 16:10:43.516521	2021-06-02 16:10:43.569429	28	28	62515	\N	\N	1	t	23	\N	0	f
524	2021-05-28 15:08:37.377893	2021-05-28 15:09:04.727067	27	27	133004	\N	\N	1	f	22	88	0	f
538	2021-06-02 16:10:40.063734	2021-06-02 16:10:40.12508	28	28	0	\N	\N	1	t	23	\N	0	f
550	2021-06-02 16:10:42.879477	2021-06-02 16:10:42.924934	28	28	0	\N	\N	1	t	23	\N	0	f
528	2021-05-28 15:08:43.079185	2021-05-28 15:09:06.438345	27	27	24285	\N	\N	1	f	22	\N	0	f
523	2021-05-28 15:08:35.572693	2021-05-28 15:08:35.715037	27	27	0	\N	\N	1	t	22	\N	0	f
563	2021-08-23 14:58:57.890415	2021-08-23 14:59:03.83588	30	30	84197	\N	\N	1	f	24	92	0	f
530	2021-05-28 15:08:45.127101	2021-05-28 15:09:06.866792	27	27	70324	\N	\N	1	f	22	79	0	f
539	2021-06-02 16:10:40.281944	2021-06-02 16:10:48.491047	28	28	84197	\N	\N	1	f	23	92	0	f
531	2021-05-28 15:08:46.482799	2021-05-28 15:09:07.4173	27	27	82352	\N	\N	1	f	22	79	0	f
526	2021-05-28 15:08:40.46386	2021-05-28 15:08:40.530967	27	27	0	\N	\N	1	t	22	\N	0	f
525	2021-05-28 15:08:39.541271	2021-05-28 15:29:53.425086	27	27	409332	\N	\N	1	f	22	\N	0	f
527	2021-05-28 15:08:42.210964	2021-05-28 15:08:42.270337	27	27	51251	\N	\N	1	t	22	\N	0	f
540	2021-06-02 16:10:40.595038	2021-06-02 16:10:40.643621	28	28	0	\N	\N	1	t	23	\N	0	f
546	2021-06-02 16:10:42.110112	2021-06-02 16:10:51.041273	28	28	11317	\N	\N	1	f	23	74	0	f
529	2021-05-28 15:08:44.188243	2021-05-28 15:08:44.255835	27	27	62515	\N	\N	1	t	22	\N	0	f
555	2021-06-02 16:10:44.002807	2021-06-02 16:10:53.709074	28	28	82352	\N	\N	1	f	23	79	0	f
541	2021-06-02 16:10:40.772017	2021-06-02 16:10:48.930121	28	28	57335	\N	\N	1	f	23	79	0	f
533	2021-06-02 16:10:38.54638	2021-06-02 16:10:45.992918	28	28	596394	\N	\N	1	f	23	99	0	f
536	2021-06-02 16:10:39.54824	2021-06-02 16:31:00.043213	28	28	121482	\N	\N	1	f	23	\N	0	f
548	2021-06-02 16:10:42.574163	2021-06-02 16:10:51.598018	28	28	133004	\N	\N	1	f	23	88	0	f
522	2021-05-28 15:08:34.956943	2021-05-28 15:09:04.225816	27	27	11317	\N	\N	1	f	22	74	0	f
549	2021-06-02 16:10:42.701255	2021-06-02 16:31:02.095405	28	28	409332	\N	\N	1	f	23	\N	0	f
551	2021-06-02 16:10:43.06039	2021-06-02 16:10:43.105088	28	28	51251	\N	\N	1	t	23	\N	0	f
562	2021-08-23 14:58:57.468319	2021-08-23 14:58:57.553044	30	30	0	\N	\N	1	t	24	\N	0	f
542	2021-06-02 16:10:41.052882	2021-06-02 16:10:49.318392	28	28	210623	\N	\N	1	f	23	99	0	f
547	2021-06-02 16:10:42.403178	2021-06-02 16:10:42.458174	28	28	0	\N	\N	1	t	23	\N	0	f
543	2021-06-02 16:10:41.312775	2021-06-02 16:10:49.883953	28	28	606138	\N	\N	1	f	23	96	0	f
552	2021-06-02 16:10:43.24244	2021-06-02 16:10:52.867817	28	28	24285	\N	\N	1	f	23	\N	0	f
557	2021-08-23 14:58:55.414872	2021-08-23 14:59:02.079488	30	30	596394	\N	\N	1	f	24	99	0	f
534	2021-06-02 16:10:38.881383	2021-06-02 16:10:46.750813	28	28	598154	\N	\N	1	f	23	97	0	f
556	2021-08-23 14:58:54.802681	2021-08-23 14:59:01.554055	30	30	534404	\N	\N	1	f	24	99	0	f
544	2021-06-02 16:10:41.571409	2021-06-02 16:10:50.242325	28	28	260422	\N	\N	1	f	23	100	0	f
535	2021-06-02 16:10:39.193594	2021-06-02 16:10:47.028432	28	28	17875	\N	\N	1	f	23	94	0	f
554	2021-06-02 16:10:43.758829	2021-06-02 16:10:53.355652	28	28	70324	\N	\N	1	f	23	79	0	f
559	2021-08-23 14:58:55.826139	2021-08-23 14:59:02.800603	30	30	17875	\N	\N	1	f	24	94	0	f
545	2021-06-02 16:10:41.840105	2021-06-02 16:10:50.721696	28	28	205748	\N	\N	1	f	23	79	0	f
532	2021-06-02 16:10:37.917688	2021-06-02 16:10:45.29998	28	28	534404	\N	\N	1	f	23	99	0	f
537	2021-06-02 16:10:39.831387	2021-06-02 16:10:47.995036	28	28	46131	\N	\N	1	f	23	79	0	f
561	2021-08-23 14:58:57.185751	2021-08-23 14:59:03.527555	30	30	46131	\N	\N	1	f	24	79	0	f
558	2021-08-23 14:58:55.625445	2021-08-23 14:59:02.618588	30	30	598154	\N	\N	1	f	24	97	0	f
565	2021-08-23 14:58:58.176456	2021-08-23 14:59:04.143357	30	30	57335	\N	\N	1	f	24	79	0	f
564	2021-08-23 14:58:58.039881	2021-08-23 14:58:58.07037	30	30	0	\N	\N	1	t	24	\N	0	f
566	2021-08-23 14:58:58.320194	2021-08-23 14:59:04.365052	30	30	210623	\N	\N	1	f	24	99	0	f
560	2021-08-23 14:58:56.819395	2021-08-23 15:19:13.438883	30	30	121482	\N	\N	1	f	24	\N	0	f
567	2021-08-23 14:58:58.578455	2021-08-23 14:59:04.754895	30	30	606138	\N	\N	1	f	24	96	0	f
568	2021-08-23 14:58:58.820823	2021-08-23 14:59:04.967152	30	30	260422	\N	\N	1	f	24	100	0	f
601	2021-08-23 17:20:19.34108	2021-08-23 17:20:19.364992	31	31	62515	\N	\N	1	t	25	\N	0	f
572	2021-08-23 14:58:59.615573	2021-08-23 14:59:05.845202	30	30	133004	\N	\N	1	f	24	88	0	f
587	2021-08-23 17:20:17.408671	2021-08-23 17:20:22.092285	31	31	84197	\N	\N	1	f	25	92	0	f
598	2021-08-23 17:20:18.945844	2021-08-23 17:20:18.9652	31	31	0	\N	\N	1	t	25	\N	0	f
586	2021-08-23 17:20:17.300624	2021-08-23 17:20:17.32326	31	31	0	\N	\N	1	t	25	\N	0	f
571	2021-08-23 14:58:59.501647	2021-08-23 14:58:59.52709	30	30	0	\N	\N	1	t	24	\N	0	f
576	2021-08-23 14:59:00.063079	2021-08-23 14:59:06.856077	30	30	24285	\N	\N	1	f	24	\N	0	f
578	2021-08-23 14:59:00.484179	2021-08-23 14:59:07.184542	30	30	70324	\N	\N	1	f	24	79	0	f
594	2021-08-23 17:20:18.474772	2021-08-23 17:20:23.459581	31	31	11317	\N	\N	1	f	25	74	0	f
574	2021-08-23 14:58:59.817585	2021-08-23 14:58:59.840178	30	30	0	\N	\N	1	t	24	\N	0	f
579	2021-08-23 14:59:00.687974	2021-08-23 14:59:07.434568	30	30	82352	\N	\N	1	f	24	79	0	f
573	2021-08-23 14:58:59.70466	2021-08-23 15:19:14.129155	30	30	409332	\N	\N	1	f	24	\N	0	f
575	2021-08-23 14:58:59.937752	2021-08-23 14:58:59.96457	30	30	51251	\N	\N	1	t	24	\N	0	f
588	2021-08-23 17:20:17.522863	2021-08-23 17:20:17.545743	31	31	0	\N	\N	1	t	25	\N	0	f
589	2021-08-23 17:20:17.633159	2021-08-23 17:20:22.348679	31	31	57335	\N	\N	1	f	25	79	0	f
577	2021-08-23 14:59:00.262742	2021-08-23 14:59:00.292317	30	30	62515	\N	\N	1	t	24	\N	0	f
581	2021-08-23 17:20:16.526504	2021-08-23 17:20:20.632735	31	31	596394	\N	\N	1	f	25	99	0	f
603	2021-08-23 17:20:19.700564	2021-08-23 17:20:25.217389	31	31	82352	\N	\N	1	f	25	79	0	f
596	2021-08-23 17:20:18.764375	2021-08-23 17:20:23.813436	31	31	133004	\N	\N	1	f	25	88	0	f
599	2021-08-23 17:20:19.053344	2021-08-23 17:20:19.077309	31	31	51251	\N	\N	1	t	25	\N	0	f
584	2021-08-23 17:20:17.005188	2021-08-23 17:40:29.471404	31	31	121482	\N	\N	1	f	25	\N	0	f
570	2021-08-23 14:58:59.279305	2021-08-23 14:59:05.420087	30	30	11317	\N	\N	1	f	24	74	0	f
597	2021-08-23 17:20:18.841773	2021-08-23 17:40:30.204406	31	31	409332	\N	\N	1	f	25	\N	0	f
590	2021-08-23 17:20:17.741135	2021-08-23 17:20:22.546254	31	31	210623	\N	\N	1	f	25	99	0	f
591	2021-08-23 17:20:17.916747	2021-08-23 17:20:22.873036	31	31	606138	\N	\N	1	f	25	96	0	f
600	2021-08-23 17:20:19.166455	2021-08-23 17:20:24.724866	31	31	24285	\N	\N	1	f	25	\N	0	f
595	2021-08-23 17:20:18.654987	2021-08-23 17:20:18.676011	31	31	0	\N	\N	1	t	25	\N	0	f
582	2021-08-23 17:20:16.727357	2021-08-23 17:20:21.07253	31	31	598154	\N	\N	1	f	25	97	0	f
604	2021-08-25 20:48:15.861541	2021-08-25 20:48:16.002265	30	\N	13104	\N	\N	1	t	24	\N	0	f
605	2021-08-25 20:48:15.9578	2021-08-25 20:48:16.009368	30	\N	0	\N	\N	1	t	24	\N	0	f
592	2021-08-23 17:20:18.105662	2021-08-23 17:20:23.069702	31	31	260422	\N	\N	1	f	25	100	0	f
583	2021-08-23 17:20:16.920198	2021-08-23 17:20:21.226696	31	31	17875	\N	\N	1	f	25	94	0	f
602	2021-08-23 17:20:19.52601	2021-08-23 17:20:24.992503	31	31	70324	\N	\N	1	f	25	79	0	f
593	2021-08-23 17:20:18.290002	2021-08-23 17:20:23.303629	31	31	205748	\N	\N	1	f	25	79	0	f
580	2021-08-23 17:20:16.337114	2021-08-23 17:20:20.271159	31	31	534404	\N	\N	1	f	25	99	0	f
585	2021-08-23 17:20:17.185247	2021-08-23 17:20:21.823246	31	31	46131	\N	\N	1	f	25	79	0	f
\.


--
-- Data for Name: asset_text_data; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.asset_text_data (id, data, asset_id, created_at, updated_at, data_vector) FROM stdin;
1	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	162	2019-11-13 14:18:38.133061	2019-11-13 14:18:38.133061	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
2	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	186	2019-11-25 14:29:53.41219	2019-11-25 14:29:53.41219	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
3	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	210	2019-11-25 18:21:20.072189	2019-11-25 18:21:20.072189	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
4	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	234	2020-05-12 16:34:15.182629	2020-05-12 16:34:15.182629	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
5	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	258	2020-06-10 15:04:44.025782	2020-06-10 15:04:44.025782	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
6	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	282	2020-10-07 19:40:57.425734	2020-10-07 19:40:57.425734	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
7	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	306	2020-11-18 15:11:37.428292	2020-11-18 15:11:37.428292	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
8	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	330	2020-11-18 15:14:40.991605	2020-11-18 15:14:40.991605	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
9	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	356	2021-05-19 20:22:18.820504	2021-05-19 20:22:18.820504	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
10	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	380	2021-05-20 14:34:28.500834	2021-05-20 14:34:28.500834	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
11	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	404	2021-05-20 14:34:29.322456	2021-05-20 14:34:29.322456	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
12	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	428	2021-05-20 14:34:30.542006	2021-05-20 14:34:30.542006	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
13	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	452	2021-05-20 14:42:33.277948	2021-05-20 14:42:33.277948	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
14	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	476	2021-05-20 14:51:35.123633	2021-05-20 14:51:35.123633	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
15	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	501	2021-05-20 15:37:54.47909	2021-05-20 15:37:54.47909	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
16	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	525	2021-05-28 15:29:53.239912	2021-05-28 15:29:53.239912	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
17	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	549	2021-06-02 16:31:02.00628	2021-06-02 16:31:02.00628	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
18	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	573	2021-08-23 15:19:14.08282	2021-08-23 15:19:14.08282	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
19	\nBlue native PAGE\nIlka Wittig1, Hans-Peter Braun2 & Hermann Scha?gger1\n\n1Molekulare Bioenergetik, Zentrum der Biologischen Chemie, Universita?tsklinikum Frankfurt, Theodor-Stern-Kai 7, Haus 26, D-60590 Frankfurt, Germany. 2Institut fu?r\nPflanzengenetik, Universita?t Hannover, Herrenha?user Str. 2, D-30419 Hannover, Germany. Correspondence should be addressed to H.S. (schagger@zbc.kgu.de).\n\nPublished online 27 June 2006; doi:10.1038/nprot.2006.62\n\nBlue native PAGE (BN-PAGE) can be used for one-step isolation of protein complexes from biological membranes and total cell and\n\ntissue homogenates. It can also be used to determine native protein masses and oligomeric states and to identify physiological\n\nprotein?protein interactions. Native complexes are recovered from gels by electroelution or diffusion and are used for 2D\n\ncrystallization and electron microscopy or analyzed by in-gel activity assays or by native electroblotting and immunodetection. In\n\nthis protocol, we describe methodology to perform BN-PAGE followed by (i) native extraction or native electroblotting of separated\n\nproteins, or (ii) a second dimension of tricine-SDS-PAGE or modified BN-PAGE, or (iii) a second dimension of isoelectric focusing\n\n(IEF) followed by a third dimension of tricine-SDS-PAGE for the separation of subunits of complexes. These protocols for 2D and 3D\n\nPAGE can be completed in 2 and 3 days.\n\nINTRODUCTION\nBN-PAGE was developed for the separation of mitochondrial\nmembrane proteins and complexes in the mass range of 10 kDa\nto 10 MDa1?3, as exemplified in Figure 1. It is used for the one-step\nisolation of microgram amounts of membrane protein complexes\nfrom biological membranes4?8 and total cell and tissue homo-\ngenates9?13; for clinical diagnostics of human mitochondrial\ndisorders10,11,13?15; to determine native masses and oligomeric\nstates2,14?18; to determine the stoichiometry of a multiprotein\ncomplex by an antibody-based gel-shift method19; to identify\nphysiological protein?protein interactions20?25; for 2D crystalliza-\ntion and electron microscopy26, in-gel activity assays27?30, native\nelectroblotting and immunodetection15; for studies of neurotrans-\nmitter assembly31, protein import32,33 and apoptosis research9; to\nisolate supramolecular physiological protein assemblies15?25, and\nmany more tasks. It is a convenient and inexpensive technique\nbased on a few simple principles.\n\nNonionic detergents are used for solubilization of biological\nmembranes. The choice of a specific nonionic detergent depends\non the detergent stability of the protein complexes of interest.\nOne of the mildest detergents is digitonin. It has been used to\nisolate supramolecular associations of multiprotein complexes,\nthus identifying physiological protein?protein interactions with-\nout using chemical crosslinking. Dodecylmaltoside has stronger\ndelipidating properties compared to digitonin. It is a mild neutral\ndetergent that is suited for isolation of membrane proteins\nand individual complexes but commonly dissociates labile hydro-\nphobic interactions. Triton X-100 shows intermediate behavior\nwith mitochondrial protein complexes. Under high-detergent\nconditions, it solubilizes membrane protein complexes, similar to\ndodecylmaltoside21. Under low-detergent conditions, it preserves\n\nrespiratory supercomplexes21 and dimeric ATP synthase20, similar\nto digitonin.\n\nFollowing solubilization of biological membranes and centrifu-\ngation, the anionic dye Coomassie blue G-250 is added to the\nsupernatant. This dye is sufficiently soluble in water, but it can also\nbind to membrane proteins because of its hydrophobic properties.\nBinding a large number of dye molecules imposes a charge shift on\nthe proteins that causes even basic proteins to migrate to the anode\nat pH 7.5 during BN-PAGE2. However, proteins are not separated\naccording to the charge/mass ratio but according to size in\nacrylamide gradient gels. Protein migration gradually decelerates\nwith running distance and with decreasing pore size of the gradient\ngel. Individual proteins must stop almost completely when they\napproach their size-dependent specific pore-size limit.\n\nBecause negatively charged protein surfaces repel each other,\nthe tendency of membrane proteins to aggregate is considerably\nreduced. Furthermore, membrane surface areas lose their hydro-\nphobic character upon binding the dye, which converts membrane\nproteins into water-soluble proteins. This means that no detergent\nis required in the BN gels once Coomassie dye has occupied protein\nsurfaces. Therefore, the risk of denaturation of detergent-sensitive\nproteins is minimized during BN-PAGE.\n\nNative proteins and complexes migrate as blue bands through\nBN gels. This facilitates excision of specific bands and recovery of\nblue stained native proteins by electroelution. The ease of recovery\nis especially advantageous when proteomic work that does not\n\n16%\n\n3.5% P 10,000\n\n1,000\n\n490\n700\n\n200\n\n130\n\n68\n48\n\n30\n\n10\n\n7\n\n2,500\n\nkDa\n\nkDa\n\nO\n\nI\nV\nIII\n\nIV\n\nII\n\nBN\n-P\n\nAG\nE\n\nP O I V III IV BN-PAGE\n\nSD\nS-\n\nPA\nG\n\nE\n\na b\n\nFigure 1 | Separation of dodecylmaltoside-solubilized mitochondrial\ncomplexes exemplifying the native mass range (o100 kDa toB10 MDa)\ncovered by BN-PAGE. (a) Solubilized bovine heart mitochondrial complexes\nI?V, pyruvate dehydrogenase complex (P), and oxoglutarate dehydrogenase\n\ncomplex (O) were separated according to their indicated masses on a linear\n\n3.5-16% acrylamide gradient gel for BN-PAGE. (b) Subunits of the native\ncomplexes were separated by tricine-SDS-PAGE using a 13% T, 3% C gel type.\n\nArrows mark the 96 kDa and 48 kDa subunits of the oxoglutarate\n\ndehydrogenase complex.\n\n418 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\ndemand homogeneously dissolved enzymes is to follow34,35. BN\ngels have also been used to isolate native proteins for 2D crystal-\nlization and electron microscopic analyses26.\n\nBN-PAGE is compatible with in-gel activity staining proce-\ndures27?30 and in-gel detection of fluorescent labels like the\nCyDye fluorophors36. However, interference of Coomassie blue\nwith activity measurements and fluorescent detection in general\ncannot be excluded. In these cases, a ?colorless native? (CN)-PAGE\nsystem might be used instead of BN-PAGE6,37?39. In its present\nform, the resolution capacity of CN-PAGE, with some exceptions,\nis low compared to BN-PAGE, and no suitable protocols for\nelectroelution and electroblotting of proteins from CN gels are\navailable so far. However, current investigations promise that\nthe resolution of CN-PAGE can be improved considerably, and\n\nelectroblotting of CN gels can work satisfactorily (I.W. and H.S.,\nunpublished data).\n\nSupramolecular assemblies that are retained during 1D\nBN-PAGE can be dissociated into the individual complexes by\napplying an orthogonal modified BN-PAGE for the second native\ndimension, thereby identifying the interaction partners and their\nstoichiometric ratio6,21,23,29. This second dimension of BN-PAGE,\nas exemplified in Figure 2, is less mild because low detergent\namounts are added to the cathode buffer. The mixture of nonionic\ndetergent and anionic Coomassie dye has some resemblance to\nan anionic detergent: it can dissociate detergent-labile associations\nbut keeps individual respiratory complexes intact.\n\nProtein subunits of complexes can be finally identified by various\ndenaturing techniques: SDS-PAGE40?42, doubled SDS-PAGE\n(dSDS-PAGE)34 and IEF/SDS 3D PAGE43, as exemplified in\nFigure 3.\n\nMATERIALS\nREAGENTS\n.1 M imidazole (Sigma); store at 7 1C. m CRITICAL Use imidazole instead of\n\nthe previously used Bis-Tris buffer, since Bis-Tris interferes with commonly\nused protein determination methods.\n\n.1 M imidazole/HCl, pH 7.0; store at 7 1C. m CRITICAL Mark this stock\nsolution with a color differing from that of the non?pH-adjusted stock\nabove to avoid confusion.\n\n.20% dodecyl-b-D-maltoside (wt/vol) (Fluka), dissolved in water; store 1-ml\naliquots at ?20 1C\n\n.20% Triton X-100 (wt/vol) (Fluka), dissolved in water; store 1-ml aliquots at\n?20 1C\n\n.20% digitonin (wt/vol) (cat no. 37006, 450% purity, used without\nrecrystallization) (Fluka), dissolved in water; store 0.1?1 ml aliquots at\n?20 1C. m CRITICAL Heating (470 1C) is required for some lots of\ndigitonin, but avoid repeated freezing/thawing because this can lead to\ninsolubility of digitonin at temperatureso50 1C.\n\n.1 M tricine (Fluka); store at 7 1C.\n\n.Glycerol (Fluka)\n\n.2 M 6-aminohexanoic acid (Fluka); store at 7 1C.\n\n.Acrylamide (twice-crystallized) (Serva)\n\n.Bis-acrylamide (twice-crystallized) (Serva)\n\n.5% (wt/vol) stock of Coomassie blue G-250 (Serva) (suspend in 500 mM\n6-aminohexanoic acid and store at 7 1C)\n\n.Ponceau S/glycerol stock solution (0.1% Ponceau S, 50% glycerol, (wt/vol))\n\n. ?Lysis? solution for IEF (optional): 8 M urea, 4% Triton X-100, 40 mM\nTris-base, 50 mM DTT, 0.1 mM PMSF\n\n.Methanol (for electroblotting of native proteins from BN gels)\n\n.Acetic acid (for electroblotting of native proteins from BN gels)\n\n.Mercaptoethanol (for tricine-SDS-PAGE)\nEQUIPMENT\n.Vertical electrophoresis apparatus according to Studier44 without special\n\ncooling for BN and 2D SDS gels. Most commercially available vertical\nprotein gel electrophoresis systems are also suitable for BN-PAGE, e.g., the\nSE 400 vertical unit (GE Healthcare) or the Protean II unit (Bio-Rad).\n\n.Power supply (600 V, 500 mA) for BN-PAGE, 2D SDS-PAGE, and\nelectroblotting of BN and SDS gels. For electroblotting, the power supply\nshould have a minimal load resistance of 30 O or less.\n\n.Semidry blotter with glassy carbon electrodes\n\n.PVDF membranes (Immobilon P, Millipore)\n\nD\n\nM\n\n2\n\n2.5\n2 1 0 I M III\n\n6 8 (g/g)\n\n1\n0 I\n\nM\n\nIII\nI\n\nM\n\nIII\n\nIV\n\nII\n\nIV\n\nII\n\n1-D BN-PAGE\n\n2-\nD\n\n B\nN\n\n-P\nAG\n\nE \n(+D\n\nD\nM\n\n)\n\n1-\nD\n\n B\nN\n\n-P\nAG\n\nE\n\na b\n\nFigure 2 | Separation of supramolecular assemblies of oxidative\nphosphorylation complexes by 1D BN-PAGE and identification of their\n\nconstituent individual complexes by 2D BN/BN-PAGE. (a) Bovine sonic\nparticles were solubilized by digitonin using the indicated digitonin/protein\n\nratios (g/g). The solubilized complexes were separated by 1D BN-PAGE.\n\nI, II, III and IV indicate mitochondrial complexes I?IV. 0, 1 and 2 indicate\n\nrespiratory supercomplexes containing monomeric complex I, dimeric complex\n\nIII, and zero (0), one (1) or two (2) copies of complex IV, respectively.\n\nM and D indicate monomeric (M) and dimeric (D) ATP synthase. (b) The lane\nfor the digitonin/protein ratio of 8 (gram/gram or g/g) from a was processed\nby 2D BN/BN-PAGE using 0.02% dodecylmaltoside added to cathode buffer B\n\n(2D BN/BN-PAGE + DDM). Respiratory supercomplexes were thereby\n\ndissociated into the individual constituent complexes I, III and IV. ?g/g?.\n\n55\n\n4.6 5.3 6.0 6.7 8.7 IEF\n\n30\n\n19\n\n11\n\nSD\nS-\n\nPA\nG\n\nE\n\nBN\n-P\n\nAG\nE\n\na b\n\nFigure 3 | Analysis of subunits of the mitochondrial ATP synthase complex\nfrom Arabidopsis by 3D BN/IEF/SDS-PAGE. (a) Mitochondrial protein complexes\nwere first separated by BN-PAGE. Afterwards, a 600-kDa band (marked)\n\nrepresenting the ATP synthase complex was cut out from the gel, electroeluted\n\nand separated from Coomassie blue. (b) Finally, the subunits of the ATP\nsynthase complex were resolved by IEF/SDS-PAGE. The isoelectric points and\n\nmolecular masses of standard proteins are given above and to the left of the\n\ngel. Reproduced from Werhahn and Braun43 with permission.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 419\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n.Whatman chromatographic papers (17 CHR)\n\n.Motor-driven tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer\n\n.Peristaltic pump for casting acrylamide gradient gels\n\n.Gradient mixer for use with a magnetic stirrer\n\n.H-shaped elutor vessel (for electroelution of native proteins) can be built accord-\ning to Hunkapiller et al. 45 or purchased from C.B.S. Scientific Co., Del Mar, CA.\n\nREAGENT SETUP\nAcrylamide-bisacrylamide mixture (AB-3) Prepare AB-3 mix containing 48 g\nacrylamide and 1.5 g bisacrylamide per 100 ml. Store at 7 1C. The AB-3 mix is defined\nas 49.5% T, 3% C according to Hjerten46. ?% T? indicates the total concentration\nof acrylamide and bisacrylamide monomers. ?% C? indicates the percentage of cross-\nlinker (bisacrylamide) to total monomer. Unless otherwise stated, 3% C is used\nthroughout. ! CAUTION Acrylamide and bisacrylamide are highly neurotoxic. When\nhandling these chemicals, wear gloves and use a pipetting aid.\n\nElectrode and gel buffers Prepare buffers as described in Table 1.\nSolubilization buffers for BN-PAGE The solubilization buffers for BN-PAGE\nshould be prepared as described in Table 2. Bis-Tris buffer was used in the\noriginal papers1,2, but since Bis-Tris disturbs almost all protein determination\nassays, it is replaced here by imidazole buffer. Solubilization buffer A is\ncommonly used to solubilize mitochondria and bacterial membranes. Higher\nsalt concentrations should be avoided, since high salt can lead to considerable\nstacking of proteins in the sample wells of BN-PAGE, and highly concentrated\nmembrane proteins tend to aggregate. If especially low ionic strength\nconditions are desired, 500 mM 6-aminohexanoic acid in buffer B can be used\ninstead of 50 mM sodium chloride. Low concentrations of 6-aminohexanoic\nacid (2 mM) and EDTA (1 mM) are commonly added to all buffers for\nprotease inhibition.\n\nPROCEDURE\nSelecting and casting gel ? TIMINGB2 h\n1| Select and cast the appropriate gel type for BN-PAGE. Common gel types are acrylamide gradient gels, although, very rarely,\nnon-gradient uniform acrylamide gels have been used for BN-PAGE47.\nm CRITICAL STEP Uniform acrylamide gels can be optimal for separation of two protein complexes with similar masses, but\nseveral attempts may be required to find the special acrylamide concentration for the specific narrow mass range.\n\nFirst, cast gradient separation gels in the cold (4?7 1C) using a gradient mixer. The amount of each reagent to use to compose\none gel (0.16 ? 14 ? 14 cm) is given in Table 3. Using a long cannula that reaches to the bottom of the gel, add 1 ml of\nwater to the bottom, and then pump in the acrylamide solution beginning with the low acrylamide solution that is continuously\nunderlain with solutions of increasing acrylamide concentration and density. Alternatively, if the gel-casting device includes a\nhole in the spacers between the two glass plates, the gradient gel also can be pumped from the bottom. Using a larger volume of\nthe 4% acrylamide solution than of the 13% acrylamide solution containing glycerol assures that the two solutions initially are\nnot mixed when the connecting tube is opened, and a linear gradient is obtained. 4-13% acrylamide gradient gels cover the\nprotein mass range 10 kDa to 3 MDa. However, proteins smaller than 100 kDa are not well resolved. 3-13% acrylamide gradient\ngels extend the mass range up to 10 MDa. A 3% acrylamide sample gel is used for this 3-13% acrylamide gradient gel type.\n\n2| Transfer the ?gel? to room temperature for polymerization. Polymerization takes approximately 30 min.\n\n3| Remove water from the top of the gradient gel after polymerization and cast the sample gel at room temperature. Add the\nappropriate sample gel comb. Gels are used either as ?preparative gels? using the total 14-cm gel width for sample application or\nas ?analytical gels? containing 0.5-cm or 1.0-cm sample wells.\n\n4| Leave the gel for approximately 15 min to polymerize.\n\n5| Remove the sample gel comb and overlay the gel with gel buffer (1?).\n? PAUSE POINT The gel can be stored at 4 1C for 1 month if\nthe overlay buffer is renewed atB3-day intervals.\n\nIsolation and storage of biological membranes\n6| Membrane fractions of interest are isolated according to\nstandard procedures.22,37,48,49 Fractions may include total\nmembranes of cells or membranes of subcellular fractions:\ne.g., organelles. Organelles often can be efficiently prepared by\ncombining differential centrifugations and density\ncentrifugations22,37,48,49.\n\nTABLE 1 | Electrode and gel buffers for BN-PAGE.\n\nCathode buffer B Cathode buffer B/10 Anode buffer Gel buffer (3?)\nTricine (mM) 50 50 ? ?\nImidazole (mM) 7.5 7.5 25 75\nCoomassie blue G-250 (%) 0.02 0.002 ? ?\n6-Aminohexanoic acid (M) ? ? ? 1.5\npH B7.0a B7.0a 7.0b 7.0b\n\nSlightly blue cathode buffer B/10 and all other solutions except deeply blue cathode buffer B should be stored at 7 1C. 6-Aminohexanoic acid is not essential for BN-PAGE, but\nit is an efficient and inexpensive serine protease inhibitor. Using 500 mM concentration in gels prevents gel drying during prolonged storage. m CRITICAL Deeply blue cathode\nbuffer B should be stirred for several hours before use and stored at room temperature, since Coomassie dye can form aggregates at low temperature, which can prevent proteins\nfrom entering the gel. aNo pH correction except with tricine or imidazole. bpH adjusted with HCl.\n\nTABLE 2 | Solubilization buffers for BN-PAGE.\n\nSolubilization\nbuffer A\n\nSolubilization\nbuffer B\n\nSodium chloride (mM) 50 ?\nImidazole/HCl (mM) 50 50\n6-Aminohexanoic acid (mM) 2 500\nEDTA (mM) 1 1\npH (at 41C) 7.0 7.0\n\n420 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n7| Suspend biological membranes in buffers containing carbohydrate or glycerol (e.g., 250 mM sucrose, or 400 mM sorbitol, or\n10% glycerol) to avoid dissociation of multiprotein complexes upon freezing.\n\n8| Shock-freeze aliquots in liquid nitrogen and store at ?80 1C. Ideally, the salt concentration in these membrane suspensions\nshould be low (0?50 mM NaCl). Avoid potassium and divalent cations because these ions precipitate Coomassie dye and\nproteins together with the dye. It is preferable to use 50 mM imidazole/HCl, pH 7.0. Other buffers, e.g., sodium phosphate\nor sodium-MOPS, are tolerated at 5?20 mM concentrations.\n? PAUSE POINT Biological membranes can be stored at ?80 1C for several months.\n\nSolubilization of biological membranes ? TIMINGB1 h\n9| Solubilize membranes as described in (A) for mitochondria and bacterial membranes, (B) for chloroplasts and (C) for\nmammalian cells and tissues.\n(A) Solubilization of mitochondria and bacterial membranes for BN-PAGE\n\n(i) Pellet aliquots of mitochondria (400 mg protein) by centrifuging for 10 min at 10,000?20,000g. Bacterial membrane\nfragments often require ultracentrifugation: e.g., 30 min at 100,000g. Centrifugation is required unless protein\nconcentrations are larger than 40 mg ml?1.\n\nm CRITICAL STEP Dilute high-density samples several-fold with water to allow for sedimentation by centrifugation.\n(ii) Add 40 ml solubilization buffer A (Table 2) to 400 mg of pelleted yeast mitochondria, mammalian mitochondria or bacterial\n\nmembranes.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add one of the detergents given in Table 4, using the volumes stated per 400 mg protein.\n? TROUBLESHOOTING\n(v) Leave sample to solubilize for 5?10 min.\n(vi) Centrifuge for 15 min at 100,000g, or for 20 min at 20,000g if the sample contains multiprotein complexes of interest\n\nlarger than 5 MDa. Retain the supernatant. In order to separate 5?10 MDa multiprotein complexes, use 3-13% or\n3.5-13% acrylamide gradient gels for BN-PAGE.\n\n(vii) Add 5 ml 50% glycerol to the supernatant.\n(viii) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio\n\nof 8 (gram/gram): i.e., for yeast mitochondria, use 1 ml if you are using dodecylmaltoside, 2.4 ml if you are using Triton\nX-100 and 3 ml if you are using digitonin. For mammalian mitochondria use 2.5 ml if you are using dodecylmaltoside,\n3 ml if you are using Triton X-100 and 6 ml if you are using digitonin. For bacterial membranes use 1 ml if you are using\ndodecylmaltoside, 2 ml if you are using Triton X-100 and 4 ml if you are using digitonin.\n\n(ix) Load around 20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n? TROUBLESHOOTING\n\nTABLE 3 | Composition of a sample gel and 4% and 13% acrylamide mixtures to prepare an acrylamide gradient gel.\n\nSample gel Gradient separation gel\n\n3.5% acrylamide 4% acrylamide 13% acrylamide\n\nAB-3 mix 0.44 ml 1.5 ml 3.9 ml\nGel buffer 3x 2 ml 6 ml 5 ml\nGlycerol ? ? 3 g\nWater 3.4 ml 10.4 ml 3 ml\nTotal volume 6 ml 18 ml 15 ml\n\n10% APS 50 ml 100 ml 75 ml\nTEMED 5 ml 10 ml 7.5 ml\nm CRITICAL 10% aqueous ammonium persulfate solution (APS) must be made up fresh or stored frozen. m CRITICAL Use cold solutions (o7 1C) to avoid untimely\npolymerization in the tubings used.\n\nTABLE 4 | Quantity of detergent required to solubilize the membranes.\n\nYeast mitochondria Mammalian mitochondria Bacterial membranes\n\nDodecylmaltoside (20%) 2.0 ml (1.0 g/g) 5 ml (2.5 g/g) 2 ml (1.0 g/g)\nTriton X-100 (20%) 4.8 ml (2.4 g/g) 6 ml (3.0 g/g) 4 ml (2.0 g/g)\nDigitonin (20%) 6.0 ml (3.0 g/g) 12 ml (6.0 g/g) 8 ml (4.0 g/g)\nThe volumes of added detergent and the detergent/protein ratios chosen gram/gram; (indicated in brackets) are shown for solubilization of yeast mitochondria\n(Saccharomyces cerevisiae and Yarrowia lipolytica), mammalian mitochondria (bovine, human, rat, mouse, pig), and bacterial plasma membranes (Paracoccus denitrificans).\nSlightly modified protocols have been used for mitochondria of Arabidopsis, potato and other plants23.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 421\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(B) Solubilization of chloroplasts for BN-PAGE\n(i) Pellet chloroplasts (500 mg protein) by 10 min centrifugation at 20,000g.\n(ii) Add 50 ml solubilization buffer A to the pelleted chloroplasts.\n(iii) Homogenize the membranes by twirling a tiny spatula before adding detergent.\n(iv) Add 2.5 ml 20% dodecylmaltoside or 20% Triton X-100 (to give a final concentration of 1.0 g/g) or 5 ml 20% digitonin\n\n(to give a final concentration of 2.0 g/g).\n(v) Leave mixture to solubilize for 5?10 min.\n(vi) Add 5 ml 50% glycerol.\n(vii) Add 5% Coomassie dye to give a detergent/dye ratio of 8 (i.e., 1.25 ml if you are using dodecylmaltoside or Triton X-100\n\nand 2.5 ml if you are using digitonin).\n(viii) LoadB25 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n(C) Homogenization and solubilization of mammalian cells and tissues for BN-PAGE.\n\n(i) Break up 10?50 mg tissue specimens with forceps and homogenize in 500 ml sucrose buffer (250 mM sucrose, 20 mM\nimidazole/HCl, pH 7.0) using a motor-driven, tightly fitting 0.5?1 ml glass/Teflon Potter-Elvehjem homogenizer (500 rpm;\n10?20 strokes). Similarly, use 20?40 strokes (500 rpm) to homogenize 10?50 mg sedimented cells (e.g., mouse embryonic\nstems cells, human fibroblast and lymphoblast cell lines, 143B osteosarcoma and derived rho zero cell lines) in diluted\nsucrose buffer (83 mM sucrose, 6.6 mM imidazole/HCl, pH 7.0).\n! CAUTION Wear protective gloves.\n\n(ii) Divide homogenized samples into aliquots corresponding to 20 mg wet weight cells, 5 mg wet weight heart muscle, 15 mg\nwet weight skeletal muscle or 10 mg wet weight kidney, brain or liver, and centrifuge for 10 min at 10,000?20,000g to\nobtain a pellet containing nuclei, mitochondria and larger cell fragments.\n? PAUSE POINT Pellets can be stored at ?80 1C following shock freezing by liquid nitrogen.\n\n(iii) Add 35 ml solubilization buffer A to the pellet recovered from 20 mg wet weight cells, 5 mg wet weight heart muscle,\n15 mg wet weight skeletal muscle or 10 mg wet weight kidney, brain or liver.\n\n(iv) Homogenize by twirling a tiny spatula before adding detergent.\n(v) Add dodecylmaltoside or digitonin. If using 20% dodecylmaltoside stock, add 5 ml for cells or skeletal muscle, 2 ml for\n\nheart muscle, or 10 ml for kidney, brain or liver. If using 20% digitonin stock, add 10 ml for cells or skeletal muscle, 4 ml\nfor heart muscle, or 20 ml for kidney, brain or liver.\n\n(vi) Solubilize for 5?10 min.\n(vii) Centrifuge for 15 min at 100,000g. Retain the supernatant.\n(viii) Add 5 ml 50% glycerol.\n(ix) Add Coomassie dye from the 5% Coomassie blue G-250 dye stock suspension to give a detergent/dye ratio of 8: i.e., if you\n\nare using dodecylmaltoside add 2.5 ml to cells or skeletal muscle, 1 ml to heart muscle or 5 ml to kidney, brain and liver.\nIf you are using digitonin add 5 ml to cells or skeletal muscle, 2 ml to heart muscle or 10 ml to kidney, brain or liver.\n\n(x) LoadB20 ml each to two 0.16 ? 0.5 cm sample wells for BN-PAGE.\n\nPreparation of chromatographically prepurified proteins for BN-PAGE\n10| Use samples with low ionic strength for separation by BN-PAGE (ideally in 50 mM NaCl, 50 mM imidazole/HCl, pH 7) to\navoid protein aggregation in the sample gel during electrophoresis. Use dialysis or other desalting techniques (e.g., centrifuge\nfilter units with appropriate cutoff limit, especially for high salt fractions from ion exchangers).\n? TROUBLESHOOTING\n\n11| Add 5% glycerol and 0.01% Ponceau S from the Ponceau S/glycerol stock solution but no Coomassie blue dye to\nchromatographically prepurified samples to avoid dissociation of detergent-labile subunits.\n? TROUBLESHOOTING\n\nElectrophoresis conditions ? TIMING 2?4 h\n12| Perform BN-PAGE at 4?7 1C, since broadening of bands is observed at room temperature. Use cathode buffer B, and set\npower supply at 100 V until the sample has entered the gel.\n\n13| Continue electrophoresis with the current limited to 15 mA and voltage limited to 500 V (for gel dimension of 0.16 ?\n14 ? 14 cm).\n14| After the blue running front has moved about one-third of the desired total running distance, remove cathode buffer B by\nsuction pump (buffer B can be reused two or three times). Continue the run using slightly blue cathode buffer B/10 for better\ndetection of faint protein bands, and to improve native blotting (less Coomassie dye competes under these conditions with\nprotein binding to PVDF membranes).\n\n422 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n15| Stop electrophoresis after 2?4 h. Use short runs if native extraction of proteins or 2D native electrophoresis is to\nfollow. At that point the protein has not yet reached its pore size limit. Therefore it still has some mobility and can be\nefficiently extracted.\nm CRITICAL STEP Coomassie dye diffuses, and faint protein bands that are just detected immediately after the end of\nelectrophoresis may disappear after 1 d. Therefore, excise bands immediately following the electrophoresis run and store the\nexcised bands in sealed tubes at 4 1C if electroelution or elution by diffusion is desired.\n? PAUSE POINT The blue gel or the excised bands can be stored at 4 1C for several days.\n\n16| The BN gel can then be fixed and Coomassie and silver stained using a previously published protocol for staining of\ntricine-SDS gels42. The migration distances of individual bands relative to marker proteins can be used to estimate native masses\nof membrane proteins. Alternatively, using non-fixed BN gels, blue bands can be excised and electroeluted, as described in\noption (A); extracted by diffusion, as described in option (B); or electroblotted, as described in option (C). Elution by diffusion\nis an alternative to electroelution, especially suited for smaller gel volumes. Alternatively a second dimension of BN-PAGE can\nbe performed, as described in option (D), a second dimension of tricine-SDS-PAGE, as described in (E); or a second dimension of\nIEF, as described in option (F). A third dimension of tricine-SDS-PAGE can be performed, as described in (E) and (F), after 2D\nBN/BN-PAGE or 2D BN/IEF-PAGE, respectively. For further techniques associated with tricine-SDS-PAGE, like staining and\nelectroblotting, see tricine-SDS-PAGE in ref. 42.\n? TROUBLESHOOTING\n(A) Electroelution of native proteins ? TIMING 4?16 h (preferentially overnight)\n\n(i) Excise blue protein bands from the BN gel (typically 0.3?0.5 ml volume per band from each preparative gel).\n(ii) Mash the gel by squeezing it repeatedly from one syringe to the next.\n(iii) Seal both lower ends of an H-shaped elutor vessel built according to Hunkapiller et al.45 with low-cutoff (e.g., 2 kDa)\n\ndialysis membranes, because of their mechanical stability.\n(iv) Squeeze the gel debris into the cathodic arm of the elutor vessel.\n(v) Fill both arms of the H-shaped vessel and the connecting tube with electrode buffer (25 mM tricine, 3.75 mM imidazole,\n\npH 7.0, and 5 mM 6-aminohexanoic acid for protease inhibition).\n(vi) Extract proteins with voltage limited to 500 V for about 4 h.\n\n! CAUTION Limit current to 2 mA per elutor vessel to prevent damage if a high-salt buffer was used erroneously. Extract\nproteins at 100 V overnight when protein aggregation must be avoided (e.g., for electron microscopic single-particle analysis).\n\n(vii) Remove clear buffer using Pasteur pipettes with the glass tips covered by silicon tubing, and collect the dark blue\nprotein/Coomassie layer.\n\n(viii) Add 10% glycerol before shock-freezing and storing at ?80 1C.\n? PAUSE POINT The blue sample containing glycerol can be stored at ?80 1C for several months.\n\n(B) Extraction by diffusion of native proteins ? TIMING 5?10 h\n(i) Excise and mash blue protein bands as described above, or use a microhomogenizer for 1.5 ml cups. Extraction by diffusion\n\nis preferentially used for small gel volumes and has been used for a proteomic study35.\n(ii) Add water (ten times the gel volume), place the cup or sealed tube in a mixer, and let diffuse for 4 h or overnight.\n(iii) Centrifuge for 10 min at 20,000g to pellet the gel debris, and proceed with the bluish supernatant.\n(iv) Concentrate the solution by vacuum centrifugation (without heating) to approach the volume of the extracted gel piece.\n\nThe concentrated solution then has a similar composition as the BN gel (500 mM 6-aminohexanoic acid, 25 mM imida-\nzole/HCl, pH 7.0). If considerably higher protein concentrations are desired, e.g., for electron microscopic single-particle\nanalysis, omit 6-aminohexanoic acid for casting the BN gels.\n\n(C) Electroblotting of native proteins from BN gels ? TIMING 5 h\n(i) Prepare electroblotting buffer (50 mM tricine, 7.5 mM imidazole; the resulting pH is around 7.0).\n(ii) Wet PVDF membrane with methanol and transfer it to electroblotting buffer.\n(iii) Soak a 3-mm stack of Whatman chromatographic papers with electroblotting buffer. The lower electrode is the cathode in\n\nthis arrangement.\n(iv) Place the gel and then the PVDF membrane on top.\n(v) Soak another 3-mm stack of chromatographic papers with electroblotting buffer, put it on top, and mount the anode.\n(vi) Place a 5-kg load on top.\n(vii) Transfer for 3 h at 4 1C (preferred) or at room temperature. Set voltage to 20 V (actual voltage is around 7 V) and limit\n\ncurrent to 0.5 mA cm?2 gel area.\n(viii) Use 25% methanol, 10% acetic acid for background destaining; protein bands are often immediately visible after the transfer.\n(ix) Rinse with water, dry and document the blot.\n(x) For western blot analysis, destain the bands on the PVDF membrane completely with methanol (for about 3 min), and\n\ntransfer then to western blot buffer.\n? TROUBLESHOOTING\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 423\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(D) Two-dimensional blue native PAGE (2D BN/BN-PAGE; native in two dimensions) ? TIMING 5?6 h\n(i) Prepare buffers for the second dimension of modified BN-PAGE. Anode and gel buffers are the same as used for 1D\n\nBN-PAGE. Add 0.02% dodecylmaltoside or 0.03% Triton X-100 to cathode buffer B (see comment in D, step viii).\n(ii) Excise a 0.5- to 1-cm-wide lane from the first dimension BN gel (gel thickness isB1.6 mm), rinse the strip for several\n\nseconds with water, and place it on a glass plate at the usual position for sample gels.\n(iii) Mount spacers (slightly thinner than for first-dimension gels: e.g., 1.4 mm), place the second glass plate on top, and fix\n\nwith clamps.\n(iv) Remove excess water and, after a few minutes, bring the gel to the upright position. (Waiting a few minutes avoids gliding\n\nof the gel strip.)\n(v) Insert a long cannula through the gaps between the 1D gel strip and spacers, and cast a steep 4-16% acrylamide\n\ngradient gel (analogous to Step 1, Table 3) with some water on top of the gradient.\n(vi) Following polymerization, add more water to cover the 1D native gel strip, and push the strip down to the separating gel\n\nusing appropriate plastic cards.\n(vii) Remove the water and fill the gaps between gel strip and spacers with a 10% acrylamide native gel (analogous to Step 1,\n\nTable 3).\n(viii) Use cathode buffer B containing either 0.02% dodecylmaltoside or 0.03% Triton X-100, and start second-dimension\n\nBN-PAGE under the running conditions of first-dimension BN-PAGE (current limited to 15 mA; voltage increases gradually\nduring the run from about 200 V to 500 V). In contrast to 1D BN gels, the 2D BN/BN gels should be stopped late in order\nto focus streaking protein bands finally into sharp protein spots. Cathode buffer B containing 0.02% dodecylmaltoside\nseems to dissociate physiological supramolecular structures more efficiently than cathode buffer B containing 0.03%\nTriton X-100.\n\n(ix) There are several ways to proceed after 2D BN/BN-PAGE. Whole 2D BN/BN gels can be used for native blotting,\nas described in Step 16 (C); horizontal and vertical gel strips from 2D BN/BN-PAGE can be used for a third dimension of\ntricine-SDS-PAGE, as described in Step 16 (E), and blue protein spots from 2D BN/BN-PAGE can be extracted, e.g., by\ndiffusion (Step 16 (B)) and then used for separation by IEF and SDS-PAGE, as described in Step 16 (F). Furthermore,\n2D BN/BN PAGE is compatible with in-gel enzyme activity assays24,30.\n\n(E) Tricine-SDS-PAGE for resolution of subunits in the second or third dimension ? TIMING 3?6 h or overnight\n(i) Place 0.5-cm gel strips from 1D BN-PAGE or 2D BN/BN-PAGE on glass plates.\n(ii) Soak strips for 15 min with 1% SDS (or for 60?120 min with 1% SDS, 1% mercaptoethanol whenever cleavage of disulfide\n\nbridges is important).\n(iii) Rinse briefly with water, especially when mercaptoethanol, an efficient inhibitor of polymerization, is used.\n(iv) Mount spacers, which can be considerably thinner than the native gel (e.g., 0.7 mm for 1.4-mm native gels), and put the\n\nsecond glass plate on top. The 0.5-cm lanes of the native gel thereby are squeezed to a width of about 1 cm and will not\nmove when the glass plates are brought to a vertical position.\n\n(v) Pour the separating gel mixture between the glass plates, so that a 5-mm gap to the native gel strip is left, and overlay\nwith water to fill the gap. Commonly a 10% T, 3% C tricine-SDS gel mixture is used to resolve proteins in the\n5?100 kDa mass range and for efficient electroblotting. A 16% T, 3% C tricine-SDS gel mixture is used for the 5?70 kDa\nmass range and to obtain sharper bands compared to 10% gels. For experimental details, see recent tricine-SDS-PAGE\nprotocols41,42.\n\n(vi) Following polymerization, gently push the native gel strip down to the separating gel using an 0.6-mm plastic card.\nRemove water and fill the gaps to the left and right of the native gel strip using a 10% acrylamide native gel mixture\n(analogous to Step 1, Table 3).\n\n(vii) Start electrophoresis at room temperature, using an apparatus without special cooling and gels with dimensions\n0.07 ? 14 ? 14 cm, with a maximal voltage of 200 V and current limited to 50 mA. When the current falls below 50 mA,\nthe voltage can be increased with the current still limited to 50 mA. The run time for 10% and 16% gels is 3?4 h\nand 5?6 h, respectively.\n\n? TROUBLESHOOTING\n(F) Three-dimensional BN/IEF/SDS-PAGE ? TIMINGB20 h\n\n(i) Electroelute proteins from BN gels as described above.\n(ii) Precipitate proteins by adding a 10% (vol/vol) trichloroacetic acid (TCA) solution (final TCA concentration: 3.5%);\n\nincubate for 10 min at 0 1C and centrifuge at 13,000g for 10 min at 0 1C.\n(iii) Resuspend proteins in acetone supplemented with 20 mM DTT, 1 mM PMSF; incubate for 1 h at ?20 1C; and centrifuge at\n\n13,000g for 15 min at 4 1C.\n(iv) Repeat Step iii.\n(v) Dry the pellet using a vacuum centrifuge.\n(vi) Resuspend the pellet in ?lysis solution? for IEF (see REAGENTS).\n\n424 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\n(vii) Carry out isoelectric focusing (IEF) according to standard procedures. All basic strategies for IEF are suitable for the\nseparation of proteins electroeluted from Blue native gels, like tube-gel separations using mobile ampholytes50 or\nseparations on gel strips using immobilized pH gradients51.\n\n(viii) Horizontally transfer the IEF gel strip onto an SDS polyacrylamide gel and carry out SDS-PAGE according to standard\nprocedures40,52. Three-dimensional BN/IEF/SDS-PAGE allows separation of isoforms of subunits of protein complexes,\nwhich often have very similar molecular masses but differ with respect to their isoelectric points. Especially in mammals\nand higher plants, subunits of protein complexes are sometimes encoded by small protein families. For instance, the\npreprotein translocase of the outer mitochondrial membrane from Arabidopsis includes four isoforms for the pre-protein\nreceptor TOM20, which can be separated nicely by 3D BN/IEF/SDS-PAGE43.\n\n? TIMING\nCasting gel: 2 h\nSample preparation and loading: 1 h\n1D blue native electrophoresis: 2?4 h, followed by fixing (0.5 h) and Coomassie staining (1 h) or the following optional steps:\nStep 16 (A) electroelution: 4?16 h (preferentially overnight)\nStep 16 (B) recovery by diffusion/concentration: 5?10 h\nStep 16 (C) native blotting: 5 h\nStep 16 (D) 2D BN/BN-PAGE (native in two dimensions): 5?6 h, followed (optionally) by 3D SDS-PAGE (3?6 h or overnight),\nfixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (E) 2D tricine-SDS-PAGE: 3?6 h or overnight followed by fixing (0.5 h) and Coomassie staining (1 h)\nStep 16 (F) IEF: 16 h, followed by tricine-SDS-PAGE (3?6 h or overnight), fixing (0.5 h) and Coomassie staining (1 h)\n\nTROUBLESHOOTING\nTroubleshooting advice can be found in Table 5.\n\nTABLE 5 | Troubleshooting table.\n\nPROBLEM SOLUTION\n\nStep 9 (A)\nHow to optimize membrane solubilization and\nto identify potential detergent sensitivity of\nmembrane protein complexes.\n\nFor initial BN analyses of specific biological membranes, test at least three detergent/\nprotein ratios: a normal detergent/protein ratio as given in Step 9 (A), Table 4, a second\none with half the detergent, and a third one with doubled the detergent. This helps to find\nthe optimal detergent amount for membrane solubilization and may indicate whether a\nspecific enzyme is detergent sensitive with respect to the physiological state of association\nand the catalytic activity. For example, yeast ATP synthase might be dimeric using low\nTriton X-100 but monomeric using high Triton X-100. Monomeric and dimeric ATP synthase\ndiffer by three dimer-specific subunits, which can be analyzed by second-dimension SDS-\nPAGE. Another possibility to preserve physiological oligomeric states is to use digitonin\ninstead of Triton X-100 or dodecylmaltoside. Digitonin seems to be less delipidating, and\nthe digitonin/protein ratio chosen is less critical.\n\nStep 9 (A)\nUpper and lower loading limits. Using isolated mitochondria, the maximum protein load for 0.16 ? 1.0 cm sample wells\n\nis around 400 mg of total protein. Blue aggregates that can prevent proteins from\nentering the gel can appear in the sample gel during electrophoresis if the load is too\nhigh. In that situation, keep the solubilization conditions as described but apply half\nthe sample volumes.\n\nThe minimal load to BN gels depends on the sensitivity of the protein detection\nmethod. Be sure that the detergent/protein ratios are chosen as described for solubilization\nof normal protein amounts, and use low final volumes for low protein amounts\nin order to keep detergent concentrations clearly above the critical micelle\nconcentration (cmc).\n\nStep 10\nProtein stacking and aggregation. Stacking of\nproteins (concentration to sharp bands in the\nsample gel) increases with the salt concentration\nin the sample. Unfortunately, membrane\nprotein complexes tend to aggregate upon\nconcentration.\n\nFor this reason, and because high salt precipitates Coomassie dye, salt concentrations must\nbe kept low. Lithium and sodium salts should be used preferentially to avoid precipitation of\nCoomassie dye and Coomassie?protein complexes by potassium and divalent cations.\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 425\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nStep 11\nDissociating properties of detergent/dye mixtures.\nChromatographically prepurified membrane pro-\nteins often contain reduced amounts of boundary\nlipid and excess detergent. Mixing excess detergent\nwith added Coomassie dye generates anionic\ndetergent-like properties and dissociates\ndetergent-labile subunits from multiprotein\ncomplexes.\n\nDo not add Coomassie dye to chromatographically pre-purified membrane proteins before\napplication to BN-PAGE. Coomassie dye from the cathode buffer that reaches sample\nproteins later during BN-PAGE seems to cause less dissociation. This might be explained by\npreseparation of acidic proteins from the bulk of neutral detergent during electrophoresis\nwhile, at the same time, Coomassie dye is retarded by incorporation into detergent/lipid\nmicelles in the sample well. After saturation with dye, these mixed micelles may catch up\nwith the protein in the separating gel where it is no longer exposed to excess detergent.\nPreseparation is restricted to acidic proteins (intrinsic pIo 7) that migrate to the anode\n(in the absence of Coomassie dye) while the bulk of neutral detergent stays immobile in the\nsample well until it becomes charged by Coomassie dye.\n\nStep 16\nProblems with determination of native masses by\nBN-PAGE. BN-PAGE can be used for rapid mass\nestimation of native membrane proteins and com-\nplexes and, with some limitations, for water-soluble\nproteins2. The error commonly is in the 10?20%\nrange, but larger deviations have also been\nreported, especially with highly basic membrane\nproteins, since large numbers of basic amino acid\nresidues often cannot be sufficiently overcome by\nnegative charges of protein bound Coomassie dye\nmolecules2,19,54,55. In such cases (i.e., in the\nabsence of a clear excess of negative charges) a\nshift to higher apparent protein masses is observed.\nApparent deviations from calculated masses (sum-\nming up the masses of subunits) may also result\nfrom glycosylation7. Some deviations, however,\ncannot be assigned to the above mentioned protein\nspecific problems but rather to inappropriate use of\nBN-PAGE.\n\nAs a general rule for mass determination by BN-PAGE, use the same detergent for\nsolubilization of your membrane protein of interest and for the membrane proteins used for\nmass calibration (e.g., the well-characterized mitochondrial oxidative phosphorylation\ncomplexes), since membrane protein migration distances can vary considerably with the\ndetergent used. This variation presumably correlates with variable detergent-dependent\namounts of boundary lipid. In the original work on mass determination by BN-PAGE2,\nsome water-soluble proteins were identified that can substitute for membrane\nprotein markers; i.e., these water-soluble proteins matched the calibration curve of\ndodecylmaltoside-solubilized membrane protein complexes. Since the migration distances\nof dodecylmaltoside-solubilized and digitonin-solubilized membrane protein complexes\ndiffer considerably, it is not possible to use the same set of water-soluble proteins to\nestimate the native masses of digitonin-solublized membrane proteins. Ignoring this\nfact can lead to considerable errors.\n\nStep 16 (C)\nInefficient transfer of proteins by native\nelectroblotting.\n\nSimilar to native electroelution and extraction, stop BN-PAGE early for native electroblot-\nting to ensure efficient transfer of proteins. Do not use cathode buffer B as the final buffer\nbefore electroblotting; rather, use B/10, since Coomassie dye binds to PVDF membranes and\nthereby reduces protein binding capacity. Do not use nitrocellulose membranes, since these\nmembranes cannot be destained under the conditions described above.3\n\nStep 16 (C)\nElectroblotting conditions may depend on the\nspecific apparatus used.\n\nLimit current to 0.5 mA cm?2 gel area. The actual voltage may vary with the apparatus used\n(and the thickness of the paper stacks).\n\nStep 16 (E)\nChoice of electrophoresis system. Although the\nLaemmli Tris-glycine system52 exhibits excellent\nresolution in 1D SDS-PAGE, it is not advisable to use\nit for the special type of 2D or 3D electrophoresis\ndiscussed here. Protein bands often streak, and the\nelectrotransfer of hydrophobic proteins to PVDF\nmembranes is problematic, because a higher-\npercentage acrylamide concentration must be used\nfor glycine SDS-PAGE than for tricine-SDS-PAGE to\ncover the same separation range.\n\nUse tricine-SDS gels for the 2D and 3D electrophoresis types described here.\n\nStep 16 (E)\nIn some situations, SDS-PAGE may not provide\nsufficient resolution (e.g., for mass spectrometric\nidentification of hydrophobic proteins in a mixture\nof common proteins).\n\nIn this case, use doubled SDS-PAGE (dSDS-PAGE)34, a technique that uses two orthogonal\nSDS-PAGEs with strongly differing gel types for 1D and 2D separation. Similarly, dSDS-PAGE\nusing nonreducing conditions for 1D resolution and reducing conditions for 2D resolution is\nideal for identifying the protein interaction partners after chemical crosslinking by\ncrosslinkers containing cleavable disulfide bonds.\n\nTABLE 5 | Troubleshooting table (continued).\n\n426 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nANTICIPATED RESULTS\nBlue native PAGE, similar to coimmunoprecipitation, has the advantage that detergent must be added only once (for\nsolubilization), and this detergent generally is not harmful to the proteins, owing to the presence of lipid from the solubilized\nmembranes. Unlike coimmunoprecipitation, BN-PAGE does not require antibodies to isolate the desired protein but seems to\nbe less mild because of the anionic properties of added Coomassie dye.\n\nBN-PAGE has mostly been used for protein separation on the analytical (microgram) scale. It covers a mass range of native\nproteins and complexes fromo100 kDa toB10 MDa, as exemplified in Figure 1. On the preparative (up to milligram) scale,\nit has been used, e.g., for Edman degradation of the subunits of oxidative phosphorylation complexes53. Using biological\nmembranes with highly abundant protein complexes of interest, it is often possible to identify protein complexes as blue bands\nduring 1D BN-PAGE and to identify subunits by Coomassie staining following 2D SDS-PAGE. Using sources with less-abundant\nproteins of interest, one might have to use silver staining, immunodetection or detection of covalently linked CyDye\nfluorophores to identify the proteins of interest. However, this is no great problem and essentially doesn?t involve different\nhandling, with the exception of attempting to enrich the desired protein before solubilization for BN-PAGE.\n\nIn recent years, BN-PAGE has also been used to identify and isolate larger physiological protein assemblies: e.g.,\nsupramolecular associations of ATP synthase and respiratory chain supercomplexes, as exemplified in Figure 2. The identity of\nthe constituent complexes is relatively easy to determine by 2D BN/BN-PAGE (Fig. 2b); i.e., by a 2D system that is native in\nboth dimensions. The supercomplexes separated in the first-dimension gel are dissociated into the individual complexes in the\nsecond-dimension gel, which is operated under less mild conditions than 1D BN-PAGE.\n\nIn parallel with the progress in analytical and mass spectrometric techniques, the isolation protocols can be scaled down\nor combined to multidimensional techniques, as exemplified in Figure 3, which shows the analysis of a multisubunit complex\nby 3D (BN/IEF/SDS) PAGE. This combination is quite a powerful approach for characterizing different isoforms of subunits\nof protein complexes that would not be accessible by common SDS-PAGE.\n\nACKNOWLEDGMENTS This work was supported by the Deutsche\nForschungsgemeinschaft (DFG), Sonderforschungsbereich 628, Project P13 to H.S.\nand DFG grant Br1829-7/2 to H.P.B.\n\nCOMPETING INTERESTS STATEMENT The authors declare that they have no\ncompeting financial interests.\n\nPublished online at http://www.natureprotocols.com\nReprints and permissions information is available online at http://npg.nature.com/\nreprintsandpermissions/\n\n1. Scha?gger, H. & von Jagow, G. Blue native electrophoresis for isolation of\nmembrane protein complexes in enzymatically active form. Anal. Biochem.\n199, 223?231 (1991).\n\n2. Scha?gger, H., Cramer, W.A. & von Jagow, G. Analysis of molecular masses and\noligomeric states of protein complexes by blue native electrophoresis and\nisolation of membrane protein complexes by two-dimensional native\nelectrophoresis. Anal. Biochem. 217, 220?230 (1994).\n\n3. Scha?gger, H. Blue native electrophoresis, in Membrane Protein Purification and\nCrystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow,\nG. & Scha?gger, H.) 5.105?5.130 (Academic, San Diego, USA, 2003).\n\n4. Ku?gler, M., Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Analysis of the\nchloroplast protein complexes by blue-native polyacrylamide gelelectrophoresis.\nPhotosynth. Res. 53, 35?44 (1997).\n\n5. Scha?gger, H. Blue native gels to isolate protein complexes from mitochondria.\nMethods Cell Biol. 65, 231?244 (2001).\n\n6. Pfeiffer, K. et al. Cardiolipin stabilizes respiratory chain supercomplexes.\nJ. Biol. Chem. 278, 52873?52880 (2003).\n\n7. Ludwig, J. et al. Identification and characterization of a novel 9.2 kDa membrane\nsector associated protein of vacuolar proton-ATPase from chromaffin granules.\nJ. Biol. Chem. 273, 10939?10947 (1998).\n\n8. Scha?gger, H. Native electrophoresis for isolation of mitochondrial\noxidative phosphorylation protein complexes. Methods Enzymol. 260,\n190?202 (1995).\n\n9. Vahsen, N. et al. AIF deficiency compromises oxidative phosphorylation.\nEMBO J. 23, 4679?4689 (2004).\n\n10. Acin-Perez, R. et al. Respiratory complex III is required to maintain complex I in\nmammalian mitochondria. Mol. Cell 13, 805?815 (2004).\n\n11. Scha?gger, H. et al. Significance of respirasomes for the assembly/stability of\nhuman respiratory chain complex I. J. Biol. Chem. 279, 36349?36353 (2004).\n\n12. Camacho-Carvajal, M., Wollscheid, B., Aebersold, R., Steimle, V. & Schamel, W.A.\nTwo-dimensional blue native/SDS gel electrophoresis of multi-protein complexes\nfrom whole cellular lysates. Mol. Cell. Proteomics 3, 176?182 (2004).\n\n13. Scha?gger, H. et al. Electrophoretic separation of multiprotein complexes from\nblood platelets and cell lines: Technique for the analysis of diseases with defects\nin oxidative phosphorylation. Electrophoresis 17, 709?714 (1996).\n\n14. Scha?gger, H. Quantification of oxidative phosphorylation enzymes after blue\nnative electrophoresis and two-dimensional resolution: Normal complex I protein\namounts in Parkinson?s disease conflict with reduced catalytic activities.\nElectrophoresis 16, 763?770 (1995).\n\n15. Carrozzo, R. et al. Subcomplexes of human ATP synthase mark mitochondrial\nbiosynthesis disorders. Ann. Neurol. 59, 265?275 (2006).\n\n16. Dudkina, N.V., Heinemeyer, J., Keegstra, W., Boekema, E.J. & Braun, H.P.\nStructure of dimeric ATP synthase from mitochondria: An angular association of\nmonomers induces the strong curvature of the inner membrane. FEBS Lett. 579,\n5769?5772 (2005).\n\n17. Dudkina, N.V., Eubel, H., Keegstra, W., Boekema, E.J. & Braun, H.P. Structure of a\nmitochondrial supercomplex formed by respiratory-chain complexes I and III.\nProc. Natl. Acad. Sci. USA 102, 3225?3229 (2005).\n\n18. Minauro-Sanmiguel, F., Wilkens, S. & Garcia, J.J. Structure of dimeric\nmitochondrial ATP synthase: novel FO bridging features and the structural basis of\nmitochondrial cristae biogenesis. Proc. Natl. Acad. Sci. USA 102, 12356?12358\n(2005).\n\n19. Schamel, W.W. et al. Coexistence of multivalent and monovalent TCRs explains\nhigh sensitivity and wide range of response. J. Exp. Med. 202, 493?503 (2005).\n\n20. Arnold, I., Pfeiffer, K., Neupert, W., Stuart, R.A. & Scha?gger, H. Yeast\nmitochondrial F1FO-ATP synthase exists as a dimer: identification of three\ndimer-specific subunits. EMBO J. 17, 7170?7178 (1998).\n\n21. Scha?gger, H. & Pfeiffer, K. Supercomplexes in the respiratory chains of yeast\nand mammalian mitochondria. EMBO J. 19, 1777?1783 (2000).\n\n22. Stroh, A. et al. Assembly of respiratory chain complexes I, III, and IV into NADH\noxidase supercomplex stabilizes Complex I in Paracoccus denitrificans. J. Biol.\nChem. 279, 5000?5007 (2004).\n\n23. Eubel, H., Ja?nsch, L. & Braun, H.P. New insights into the respiratory chain of plant\nmitochondria: supercomplexes and a unique composition of complex II. Plant\nPhysiol. 133, 274?286 (2003).\n\n24. Eubel, H., Heinemeyer, J. & Braun, H.P. Identification and characterization of\nrespirasomes in potato mitochondria. Plant Physiol. 134, 1450?1459 (2004).\n\n25. Paumard, P. et al. The ATP synthase is involved in generating mitochondrial\ncristae morphology. EMBO J. 21, 221?230 (2002).\n\n26. Poetsch, A., Neff, D., Seelert, H., Scha?gger, H. & Dencher, N.A. Dye removal,\ncatalytic activity and 2D-crystallization of chloroplast H+-ATP synthase\npurified by blue native electrophoresis. Biochim. Biophys. Acta 1466, 339?349\n(2000).\n\n27. Zerbetto, E., Vergani, L. & Dabbeni-Sala, F. Quantification of muscle\nmitochondrial oxidative phosphorylation enzymes via histochemical\n\nNATURE PROTOCOLS | VOL.1 NO.1 | 2006 | 427\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n\nstaining of blue native polyacrylamide gels. Electrophoresis 18,\n2059?2064 (1997).\n\n28. Jung, C., Higgins, C.M.J. & Xu, Z. Measuring the quantity of mitochondrial\nelectron transport chain complexes in tissues of central nervous system\nusing Blue native polyacrylamide gel electrophoresis. Anal. Biochem. 286,\n214?223 (2000).\n\n29. Eubel, H., Heinemeyer, J., Sunderhaus, S. & Braun, H.P. Respiratory chain super-\ncomplexes in plant mitochondria. Plant Physiol. Biochem. 42, 937?942 (2004).\n\n30. Sunderhaus, S. et al. Carbonic anhydrase subunits form a matrix-exposed domain\nattached to the membrane arm of mitochondrial complex I in plants. J. Biol.\nChem. 281, 6482?6488 (2006).\n\n31. Griffon, N. et al. Molecular determinants of glycine receptor subunit assembly.\nEMBO J. 18, 4711?4721 (1999).\n\n32. Dietmeyer, K. et al. Tom 5 functionally links mitochondrial preprotein receptors to\nthe general import pore. Nature 388, 195?200 (1997).\n\n33. Ja?nsch, L., Kruft, V., Schmitz, U.K. & Braun, H.P. Unique composition of the\npreprotein translocase of the outer mitochondrial membrane from plants. J. Biol.\nChem. 273, 17251?17257 (1998).\n\n34. Rais, I., Karas, M. & Scha?gger, H. Two-dimensional electrophoresis for the\nisolation of integral membrane proteins and mass spectrometric identification.\nProteomics 4, 2567?2571 (2004).\n\n35. Fandino, A.S. et al. LC-nanospray-MS/MS analysis of hydrophobic proteins from\nmembrane protein complexes isolated by blue-native electrophoresis. J. Mass\nSpectrom. 40, 1223?1231 (2005).\n\n36. Perales, M. et al. Disruption of a nuclear gene encoding a mitochondrial\ngamma carbonic anhydrase reduces complex I and supercomplex I+III2 levels\nand alters mitochondrial physiology in Arabidopsis. J. Mol. Biol. 350,\n263?277 (2005).\n\n37. Wittig, I. & Scha?gger, H. Advantages and limitations of clear native\npolyacrylamide gel electrophoresis. Proteomics 5, 4338?4346 (2005).\n\n38. Gavin, P.D., Devenish, R.J. & Prescott, M. FRET reveals changes in the F1-stator\nstalk interaction during activity of F1FO-ATP synthase. Biochim. Biophys. Acta\n1607, 167?179 (2003).\n\n39. Gavin, P.D., Prescott, M. & Devenish, R.J. Yeast F1FO-ATP synthase complex\ninteractions in vivo can occur in the absence of the dimer specific subunit e.\nJ. Bioenerg. Biomembr. 37, 55?66 (2005).\n\n40. Scha?gger, H. & von Jagow, G. Tricine-sodium dodecyl sulfate polyacrylamide gel\nelectrophoresis for the separation of proteins in the range from 1?100 kDalton.\nAnal. Biochem. 166, 368?379 (1987).\n\n41. Scha?gger, H. SDS electrophoresis techniques, in Membrane Protein Purification\nand Crystallization. A Practical Guide 2nd edn. (eds. Hunte, C., von Jagow, G.\n& Scha?gger, H.) 4.85?4.103 (Academic, San Diego, USA, 2003).\n\n42. Scha?gger, H. Tricine?SDS-PAGE. Nat. Protocols 1, 16?22 (2006).\n43. Werhahn, W. & Braun, H.P. Biochemical dissection of the mitochondrial proteome\n\nfrom Arabidopsis thaliana by three-dimensional gel electrophoresis.\nElectrophoresis 23, 640?646 (2002).\n\n44. Studier, F.W. Analysis of bacteriophage T7 early RNAs and proteins on slab gels.\nJ. Mol. Biol. 79, 237?248 (1973).\n\n45. Hunkapiller, M.W., Lujan, E., Ostrander, F. & Hood, L.E. Isolation of microgram\nquantities of proteins from polyacrylamide gels for amino acid sequence analysis.\nMethods Enzymol. 91, 227?236 (1983).\n\n46. Hjerten, S. Chromatographic separation according to size of macromolecules and\ncell particles on columns of agarose suspensions. Arch. Biochem. Biophys. 99,\n466?475 (1962).\n\n47. Reif, S., Voos, W. & Rassow, J. Intramitochondrial dimerization of citrate\nsynthase characterized by blue native electrophoresis. Anal. Biochem. 288,\n97?99 (2001).\n\n48. Meisinger, C., Pfanner, N. & Truscott, K.N. Isolation of yeast mitochondria.\nMethods Mol. Biol. 313, 33?39 (2006).\n\n49. Heinemeyer, J., Eubel, H., Wehmho?ner, D., Ja?nsch, L. & Braun, H.P. Proteomic\napproach to characterize the supramolecular organization of photosystems in\nhigher plants. Phytochemistry 65, 1683?1692 (2004).\n\n50. Klose, J. & Kobalz, U. Two-dimensional electrophoresis of proteins: an updated\nprotocol and implications for a functional analysis of the genome. Electrophoresis\n16, 1034?1059 (1995).\n\n51. Go?rg, A. & Weiss, W., High-resolution two-dimensional electrophoresis of proteins\nusing immobilized pH gradients. in Cell Biology. A Laboratory Handbook (ed. Celis,\nJ.) 386?397 (Academic, New York, 1998).\n\n52. Laemmli, U.K. Cleavage of structural proteins during assembly of the head of\nbacteriophage T4. Nature 227, 680?685 (1970).\n\n53. Scha?gger, H., Brandt, U., Gencic, S. & von Jagow, G. Ubiquinol-cytochrome\nc-reductase from human and bovine mitochondria. Methods Enzymol. 260,\n82?96 (1995).\n\n54. Moro, F., Sirrenberg, C., Schneider, H.C., Neupert, W. & Brunner, M. The TIM17.23\npreprotein translocase of mitochondria: composition and function in protein\ntransport into the matrix. EMBO J. 18, 3667?3675 (1999).\n\n55. Schamel, W.W. & Reth, M. Monomeric and oligomeric complexes of the B cell\nantigen receptor. Immunity 13, 5?14 (2000).\n\n428 | VOL.1 NO.1 | 2006 | NATURE PROTOCOLS\n\nPROTOCOL\n\n?\n 2\n\n00\n6 \n\nN\nat\n\nur\ne \n\nP\nub\n\nlis\nhi\n\nng\n G\n\nro\nup\n\n  h\ntt\n\np\n:/\n\n/w\nw\n\nw\n.n\n\nat\nur\n\ne.\nco\n\nm\n/n\n\nat\nur\n\nep\nro\n\nto\nco\n\nls\n\n\n	597	2021-08-23 17:40:30.10643	2021-08-23 17:40:30.10643	 '-100':465,1289,1426,3253,3281,3309,3480,3657,3717,5368,5574,5660,6662,6669,6707 '-13':2427,2452,2474,3191,3194 '-16':835,5492 '-250':516,1391,2679,3221,4056 '-3':1970,1973,1992,3367 '-30419':45 '-60590':30 '-7':8193 '/2':8194 '/ms':9235 '/n':912,1893,2867,3595,4400,5310,6254,6951,7784,9020,9827 '/nprot.2006.62':62 '/w':903,1884,2858,3586,4391,5301,6245,6942,7775,9011,9818 '0':1564,1572,1666,1682,2928,6153,6164 '0.002':2681 '0.01':4205 '0.02':1724,2680,5362,5568,5642 '0.03':5365,5571,5657 '0.07':6061 '0.1':1318,1409,1435 '0.16':2306,3326,3732,4126,4294,6740 '0.3':4723 '0.44':3369 '0.5':1911,2541,3327,3733,3786,4127,4724,5215,5382,5783,5868,6425,6487,6510,6535,7536 '0.6':6012 '0.7':5852 '00':886,1867,2841,3569,4374,5284,6228,6925,7758,8994,9801 '000':742,744,3042,3889 '000g':3044,3056,3157,3164,3623,3891,4036,5010,6159,6190 '1':250,743,785,1204,1236,1280,1296,1319,1352,1563,1571,1583,1598,1667,1684,1912,2061,2191,2209,2328,2571,2583,2646,2826,2827,3069,3238,3294,3787,4081,4477,5384,5499,5558,5805,5813,5815,5883,6042,6176,6181,6413,6430,6492,6515,6540,7894,8224,9403,9456 '1.0':2544,3468,3476,3664,6741 '1.25':3707 '1.4':5433,5855 '1.5':1980,2686,3371,4957 '10':240,243,741,752,2436,2462,2894,3038,3041,3148,3185,3402,3413,3419,3618,3686,3757,3796,3808,3874,3885,3888,3948,3990,4003,4027,4113,4144,4904,4940,5006,5225,5551,5932,5983,6034,6102,6138,6150,6161,6453,6845,7185,8461 '10.1038':61 '10.4':3390 '100':1984,2445,3055,3156,3406,4035,4265,4863,5949,9404 '102':8704,8740 '1034':9656 '1059':9657 '10939':8422 '10947':8423 '11':286,1758,4200,6960,8486 '12':3500,4238,8511 '120':5810 '1223':9254 '1231':9255 '12356':8741 '12358':8742 '13':278,287,858,2400,3346,3364,4274,6158,6189,8482,8547,9789 '130':748 '133':8885 '134':8906 '14':296,2307,2308,2529,4295,4296,4298,6062,6063,8580,9791 '143b':3824 '1450':8907 '1459':8908 '1466':8964 '15':288,2554,3153,3400,3867,3941,4032,4282,4408,5597,5802,6192,8617 '16':738,4524,4709,5685,5711,5727,5744,5958,6104,6438,6442,6447,6456,6463,6495,6518,6521,7142,7447,7522,7559,7654,8613,8637,9457,9655 '1607':9341 '166':9408 '167':9342 '1683':9628 '1692':9629 '17':1904,8576,8674,8801 '17251':9195 '17257':9196 '176':8544 '1777':8824 '1783':8825 '179':9343 '18':297,3398,8708,9035,9142,9769 '182':8545 '19':1757,7223,8744,8823 '190':8442 '195':9165 '1962':9559 '1970':9712 '1973':9505 '1983':9534 '1987':9411 '199':8246 '1991':8249 '1994':8290 '1995':8444,8616,9658,9740 '1996':8579 '1997':8355,9038,9167 '1998':8424,8804,9197,9692 '1999':9145,9772 '1c':1211,1244,1285,1301,1324,1329,1351,1359,1371,1404,1989,2290,2581,2714,2916,2984,3439,3911,4246,4497,4520,4914,4927,5194,6154,6165,6185,6196 '1d':1079,1618,1652,5357,5482,5519,5615,5788,6415,7576,7708,7720,7939,8103 '1molekulare':13 '2':43,216,754,885,1126,1362,1560,1562,1588,1608,1669,1687,1866,2082,2093,2187,2479,2797,2822,2840,3097,3301,3378,3474,3568,3984,4107,4235,4373,4412,4770,4843,5218,5283,6227,6407,6419,6924,7539,7757,8046,8250,8993,9800 '2.0':3466,3492,3677 '2.4':3245,3484 '2.5':1561,3266,3472,3649,3719,4074 '20':1268,1284,1286,1300,1302,1323,2972,3043,3160,3163,3321,3465,3481,3495,3622,3651,3654,3669,3774,3797,3801,3856,3890,3930,3973,3999,4016,5009,5203,6173,6184,7186,8770 '200':747,5607,6070,9166 '2000':8826,8967,9072,9792 '2001':8375,9582 '2002':8930,9484 '2003':8324,8392,8888,9344,9445 '2004':8460,8485,8510,8546,8857,8909,9095,9225,9630 '2005':8673,8707,8743,8769,9256,9293,9313,9378 '2006':59,881,1863,2836,3565,4369,5280,6223,6921,7753,8636,8990,9127,9459,9600,9796 '202':8443,8766 '2059':9036 '2064':9037 '21':1112,8805,8927 '214':9070 '217':8287 '22':8827,9458 '220':8288 '221':8928 '223':8247,9071 '227':9532,9709 '23':1113,8457,8858,9481 '230':8289,8929 '231':8248,8373 '236':9533 '237':9503 '24':8889 '244':8374 '248':9504 '25':321,357,2674,4809,5057,5223,8910 '250':2886,3771 '2567':9223 '2571':9224 '26':28,8931 '260':8441,9737 '263':9291 '265':8634 '27':57,8968 '273':8421,9194 '274':8886 '275':8635 '277':9292 '278':8389 '279':8507,8854 '28':9039 '281':9124 '286':8887,9069 '288':9579 '29':1114,9073 '2b':8062 '2d':123,208,323,939,1476,1520,1630,1721,1731,4424,4674,4677,5325,5619,5669,5672,5693,5718,5747,5793,6465,6497,7592,7646,7710,7726,7951,8059,8066,8949 '2d-crystallization':8948 '2institut':33 '2nd':8307,9428 '3':218,245,860,1201,1771,1999,2028,2314,2439,2451,2465,2473,2493,2665,3190,3255,3273,3337,3385,3392,5129,5162,5190,5266,5501,5560,5776,5934,5960,6044,6107,6481,6502,6529,8134,8291,8543 '3.0':3488,3498 '3.4':3388 '3.5':739,834,3193,3360,6147 '3.75':4812 '3.9':3373 '30':334,751,959,1543,1756,2491,3052,5759,9096 '31':9128 '313':9597 '32':9146 '3225':8705 '3229':8706 '33':347,9168,9598 '339':8965 '34':1192,7692,9198 '349':8966 '35':927,3920,8353,9226 '350':9290 '36':9257 '36349':8508 '36353':8509 '3667':9770 '3675':9771 '368':9409 '37':1005,2612,2642,9294,9375 '37006':1307 '379':9410 '38':9314 '386':9687 '388':9164 '39':1006,9345,9599 '397':9688 '3d':210,1195,1784,6402,6477,7594,7648,8144 '3x':3377 '4':1423,2288,2394,2426,2548,2580,3131,3311,3344,3362,3449,3490,4010,4236,4244,4413,4496,4519,4708,4837,4999,5193,5491,6108,6195,6420,6441,6601,8325,9222 '4.0':3506 '4.103':9440 '4.6':1750 '4.8':3482 '4.85':9439 '40':1427,3066,3091,3802,9253,9379 '400':1497,2890,3032,3099,3137,6747 '41':9412 '418':878 '419':1864 '41c':2830 '42':1184,4700,5995,9092,9446 '420':2833 '421':3566 '422':4366 '423':5281 '424':6220 '425':6922 '426':7750 '427':8991 '428':9793 '43':9460 '4338':9311 '4346':9312 '44':8354,9485 '45':1953,9506 '450':1308 '46':9535 '466':9557 '4679':8458 '4689':8459 '47':9560 '470':1328 '4711':9143 '4721':9144 '475':9558 '48':750,870,1976,2613,2643,9583 '49':2614,2644,9601 '49.5':1997 '490':745 '493':8767 '5':1384,2558,2971,3147,3175,3184,3204,3217,3382,3411,3470,3667,3685,3690,3696,3861,3935,3977,4026,4042,4052,4087,4100,4202,4818,4939,5096,5182,5332,5912,5948,5971,6111,6452,6460,6471,6550,6552,7746,8356,9152,9310,9790 '5.105':8318 '5.130':8319 '5.3':1751 '50':1412,1432,2177,2668,2669,2813,2816,2817,2929,2954,3206,3404,3626,3692,3758,3809,4044,4159,4162,5102,6076,6083,6096,9631 '500':755,1395,1514,2164,2736,2823,3614,3767,3794,3804,4288,4833,5052,5610 '5000':8855 '5007':8856 '503':8768 '51':9659 '52':6328,9693 '52873':8390 '52880':8391 '53':8352,9713 '54':7224,9741 '55':1749,7225,9376,9773 '5769':8671 '5772':8672 '579':8670 '59':8633 '6':887,1364,1397,1568,1868,2166,2184,2601,2682,2715,2818,2842,3283,3380,3396,3486,3570,4375,4820,5054,5079,5285,5333,5777,6112,6229,6472,6482,6503,6530,6926,7759,8376,8995,9802 '6.0':1752,3496,3502 '6.6':3839 '6.7':1753 '60':5809 '600':1512,1799 '628':8184 '640':9482 '646':9483 '6482':9125 '6488':9126 '65':8372,9627 '66':9377 '68':749 '680':9710 '685':9711 '7':26,753,1210,1243,1358,1370,1403,1988,2289,2713,2875,4166,4245,5209,7110,8393 '7.0':1240,2690,2692,2831,2832,2958,3778,3843,4816,5062,5113 '7.5':568,2672,2673,3415,5105 '70':5972 '700':746 '709':8577 '714':8578 '7170':8802 '7178':8803 '75':2675,3408 '763':8614 '770':8615 '79':9502 '8':270,1420,1569,1712,2904,3231,3504,3705,4066,8425 '8.7':1754 '80':2915,2983,3910,4913,4926 '805':8483 '815':8484 '82':9738 '83':3836 '9':2994,6558,6598,6725,8445 '9.2':8404 '91':9531 '937':9093 '942':9094 '96':867,9739 '97':9580 '99':9556,9581 'a.s':9228 'ab':1969,1972,1991,3366 'above.3':7520 'absenc':7118,7232,9365 'abund':7922,7960 'acad':8701,8737 'academ':8320,9441,9689 'access':8167 'accord':578,584,826,1467,1947,2001,2608,4760,6268,6324,9540 'acet':1447,5226 'aceton':6170 'acid':1366,1399,1448,2168,2186,2684,2717,2820,4822,5056,5081,5227,6141,7048,7106,7207,9526 'acin':8463 'acin-perez':8462 'acknowledg':8173 'acrylamid':588,836,1372,1379,1923,1966,1978,2010,2034,2225,2235,2248,2272,2340,2346,2356,2395,2401,2428,2453,2466,2475,3195,3347,3352,3361,3363,3365,5493,5552,6035,7617 'acrylamide-bisacrylamid':1965 'acta':8963,9340 'activ':134,332,955,978,5757,6649,8242,8611,8946,9332 'actual':5205,7543 'ad':518,1135,1726,2195,3120,3511,3645,3964,6136,6985,7807,7858 'add':2327,2513,3090,3123,3203,3212,3625,3648,3689,3695,3919,3967,3976,4002,4041,4047,4073,4099,4201,4903,4979,5361,5513,7005 'address':51 'adjust':1262,2793 'advantag':731,7802,9300 'advic':6544 'advis':7583 'aebersold':8518 'afterward':1797 'ag':766,1593,1603,1766 'agaros':9551 'aggreg':639,2155,2772,4170,4868,6753,6849,6875 'aid':2050 'aif':8450 'al':1952,8380,8397,8449,8467,8491,8552,8621,8748,8831,8914,9100,9132,9150,9230,9261 'al.45':4764 'aliquot':1282,1298,1321,2908,3029,3853 'allow':3084,6333 'almost':611,2100 'also':91,532,931,1488,2381,7192,7262,8021 'alter':9282 'altern':2360,4569,4608,4617 'although':2228,7565 'amino':7206,9525 'aminohexano':1365,1398,2167,2185,2683,2716,2819,4821,5055,5080 'ammonium':3421 'amount':262,1133,2296,6624,6821,6830,6974,7350,8602 'ampholytes50':6296 'anal':8244,8285,9067,9406,9577 'analog':5496,5555,6039 'analys':6578 'analyses26':945 'analysi':1772,4879,5077,5253,8138,8259,8338,8567,9236,9488,9528,9650 'analyt':2538,7873,8113 'analyz':129,6685 'angular':8656 'anhydras':9102,9272 'anion':511,1146,1154,6989,7855 'ann':8631 'ano':2784 'anod':565,2661,5178,5347,7115 'anoth':5161,6692 'antibodi':309,7840 'antibody-bas':308 'anticip':7792 'antigen':9786 'ap':3403,3424 'apoptosi':349 'appar':7244,7249 'apparatus':1466,6053,7531,7549 'appear':6763 'appli':1091,6786 'applic':2535,7016 'approach':615,5032,8153,9615 'appropri':2214,2515,4188,5533 'approxim':2490,2553 'aqueous':3420 'arabidopsi':1782,3556,6385,9286,9472 'arch':9553 'area':646,5220,7541 'arm':4787,4795,9114 'arnold':8771 'around':3320,5112,5208,6746 'arrang':5147 'arrow':864 'assay':135,2104 'assays24':5758 'assays27':333 'assembl':1074,1612,8031,8832,9139,9701 'assemblies15':356 'assembly/stability':8497 'assembly31':344 'assign':7271 'associ':416,1162,4683,6645,8034,8408,8657 'assur':2405 'atb3':2592 'atb3-day':2591 'atp':497,1702,1778,1805,1826,6653,6673,8036,8625,8651,8721,8786,8916,8954,9336,9355 'atpas':8414 'attach':9110 'attempt':2264,8002 'author':8201 'avail':1042,1481,8219 'avoid':1266,1338,2129,2897,2932,3441,4168,4224,4871,5466,6903 'b':783,843,1271,1590,1600,1606,1704,1730,1769,1820,2171,2657,2691,2693,2708,2753,2809,3006,3603,4259,4319,4324,4595,4931,5372,5565,5640,5655,5728,6448,7482,8517,9784 'b/10':2660,2698,4340,7491 'b7.0a':2688,2689 'background':5229 'bacteri':2122,3004,3021,3045,3108,3291,3462,3542 'bacteriophag':9490,9706 'band':706,715,1801,4250,4347,4463,4481,4491,4514,4554,4577,4717,4728,4948,5232,5256,5632,5980,6856,7599,7937 'base':310,369,1431 'basi':8730 'basic':559,6273,7198,7205 'becom':7136 'begin':2342 'behavior':468 'better':4342 'bind':533,542,653,4362,7495,7503 'bio':1508 'bio-rad':1507 'biochem':8245,8286,9068,9091,9407,9465,9554,9578 'biochim':8961,9338 'bioenerg':9373 'bioenergetik':14 'biogenesi':8734 'biol':8371,8387,8419,8505,8852,9122,9192,9289,9501,9596 'biolog':81,268,382,505,2599,2877,2977,2990,6581,7918,9680 'biologischen':17 'biomembr':9374 'biophi':8962,9339,9555 'biosynthesi':8629 'bis':1222,1227,1378,2084,2097 'bis-acrylamid':1377 'bis-tri':1221,1226,2083,2096 'bisacrylamid':1967,1982,2012,2021,2036 'blood':8559 'blot':4352,5248,5252,5273,5680,6459 'blotter':1548 'blue':1,63,514,705,719,976,1389,1819,2677,2695,2705,2750,3219,4054,4217,4301,4337,4509,4576,4715,4899,4918,4946,5322,5714,6286,6416,6752,7794,7936,8231,8270,8295,8346,8360,8527,8590,8958,9030,9062,9247,9574 'blue-nat':8345,9246 'bluish':5020 'bn':67,151,177,222,571,674,697,708,764,775,805,841,928,947,1003,1026,1081,1096,1120,1445,1455,1474,1492,1518,1527,1586,1620,1654,1764,1795,2066,2073,2146,2219,2242,2653,2723,2802,3025,3200,3333,3609,3739,3752,4133,4142,4155,4241,4526,4574,4623,4720,5050,5085,5093,5345,5359,5393,5581,5592,5616,5790,6127,6577,6795,7019,7034,7151,7154,7286,7297,7363,7464,7835,7862,7941,8012,8018,8105 'bn-page':66,150,176,221,696,774,804,840,946,1025,1080,1095,1119,1491,1517,1585,1619,1653,1794,2065,2072,2145,2218,2652,2722,2801,3024,3199,3332,3608,3738,3751,4132,4141,4154,4240,4622,5344,5358,5580,5591,5789,7018,7033,7150,7153,7285,7296,7463,7834,7861,7940,8011,8017,8104 'bn-page2':570,7362 'bn-page47':2241 'bn-page6':1002 'bn/bn':5620,5673,5748 'bn/bn-page':1631,1722,1732,4675,5326,5670,5694,5719,5794,6466,8060 'bn/ief-page':4678 'bn/ief/sds':8145 'bn/ief/sds-page':1785,6120,6332 'bn/ief/sds-page43':6403 'boekema':8644,8681 'bond':7744 'bottom':2323,2334,2387 'bound':7219 'boundari':6976,7352 'bovin':809,1633,3536,9733 'bph':2792 'br1829':8192 'bracket':3521 'brain':3879,3953,3994,4020,4091,4117 'brandt':9717 'braun':8336,8646,8683,8864,8894,9080,9176,9463,9612 'braun2':9 'braun43':1856 'break':3755 'bridg':5821,8725 'briefli':5826 'bring':5455 'broaden':4248 'brought':5894 'brunner':9750 'buffer':1139,1224,1729,2054,2056,2063,2070,2086,2111,2113,2170,2198,2570,2588,2650,2656,2659,2662,2664,2697,2707,2752,2799,2805,2808,2880,2960,3094,3376,3629,3770,3835,3923,4258,4318,4323,4339,4808,4856,4883,5101,5125,5138,5170,5274,5337,5350,5371,5564,5639,5654,7026,7481,7486 'built':1946,4759 'bulk':7052,7124 'c':861,2000,2014,2029,3010,3741,4602,5087,5686,5935,5961,6457,7448,7523,8311,9041,9432,9585,9728,9745 'c-reductas':9727 'c.b.s':1957 'c.m.j':9043 'ca':1962 'calcul':7252 'calibr':7319,7387 'camacho':8513 'camacho-carvaj':8512 'cannot':985,7210,7269,7513 'cannula':2318,5476 'capac':1013,7504 'carbohydr':2882 'carbon':1551,9101,9271 'card':5535,6015 'cardiolipin':8381 'carri':6263,6319 'carrozzo':8618 'carvaj':8514 'case':990,7228,7683 'cast':1922,2205,2212,2281,2365,2506,5083,5488,6405 'cat':1305 'catalyt':6648,8610,8945 'catch':7084 'cathod':1138,1728,2655,2658,2696,2706,2751,4257,4317,4338,4786,5144,5370,5563,5638,5653,7025,7480 'cation':2936,6916 'caus':557,7038 'caution':2033,3844,4839 'celi':9685 'cell':85,273,2621,3013,3747,3812,3817,3822,3830,3860,3901,3934,3980,4006,4077,4103,8370,8481,8541,8562,9546,9679,9785 'cellular':8538 'central':9058 'centrifu':508 'centrifug':2638,3036,3057,3088,3151,3620,3883,4030,4184,5004,5028,6156,6187,6208 'centrifugations22':2641 'cerevisia':3530 'chain':8040,8384,8501,8694,8815,8835,8871,9053,9083 'chang':9323 'charact':651 'character':7324,8155,8400,8898,9572,9617 'charg':551,627,7137,7216,7239 'charge/mass':581 'chem':8388,8420,8506,8853,9123,9193 'chemi':18 'chemic':429,2043,7737 'chlorid':2180,2811 'chloroplast':3008,3606,3613,3634,8341,8952 'choic':385,7561 'chosen':3517,6720,6813 'chr':1905 'chromaffin':8416 'chromatograph':1902,4137,4220,5134,5166,6966,7009,9538 'citrat':9570 'clamp':5445 'clear':4882,6837,7235,9304 'cleavabl':7742 'cleavag':5818,9696 'clinic':280 'cm':2309,2530,2542,2545,3328,3734,4128,4297,5217,5386,5784,5869,5884,6064,6742,7538 'cm-wide':5385 'cmc':6843 'cn':994,1016,1039,1054,1063 'cn-page':1015,1053 'co':910,918,1891,1899,1959,2865,2873,3593,3601,4398,4406,5308,5316,6252,6260,6949,6957,7782,7790,9018,9026,9825,9833 'coexist':8749 'coimmunoprecipit':7799,7833 'cold':2287,3436 'collect':4896 'color':1253 'colorless':992 'column':9549 'com':7168 'comb':2518,2563 'combin':2636,8126,8148 'comment':5374 'commerci':1480 'common':457,1231,2116,2194,2221,5930,7181,7679,8169 'compar':436,1023,5981 'compat':950,5751 'compet':4356,8197,8207 'complet':214,612,5261 'complex':79,110,204,234,266,305,399,419,455,472,482,702,792,812,817,822,848,877,1089,1167,1172,1616,1628,1648,1663,1675,1678,1690,1743,1780,1789,1807,1828,2258,2901,3170,3188,6341,6366,6574,6872,6911,7002,7328,7395,7410,7885,7924,7934,8052,8088,8142,8162,8239,8268,8278,8343,8366,8438,8469,8475,8502,8535,8557,8599,8695,8836,8846,8881,9054,9085,9117,9243,9274,9357,9781 'complexes53':7916 'compos':2303 'composit':3338,5047,8879,9179,9758 'compromis':8452 'concentr':2008,2126,2150,2182,2273,2357,2738,2920,2974,3062,3662,3675,5023,5041,5067,6146,6836,6842,6853,6865,6877,6889,7618 'condit':477,490,2161,4233,4359,5586,6782,7518,7525,7718,7724,8101 'conflict':8607 'confus':1267 'connect':2416,4804 'consider':641,1059,2136,5064,5845,7336,7412,7444 'constitu':1626,1742,8051 'contain':1673,1975,2403,2540,2881,3168,3896,4920,5566,5641,5656,6972,7741 'continu':2350,4275,4332,7749 'contrast':5613 'conveni':365 'convert':657 'cool':1472,6056 'coomassi':513,677,975,1147,1388,1818,2676,2768,2941,3213,3218,3697,4048,4053,4216,4354,4457,4533,6428,6490,6513,6538,6886,6906,6909,6986,7006,7021,7063,7120,7139,7220,7493,7859,7948 'copi':1688 'correct':2786 'correl':7344 'correspond':48,3854 'coval':7975 'cover':802,2431,4891,5517,7634,7877 'cramer':8254 'crista':8733,8923 'critic':1213,1246,1326,2245,2748,3071,3418,3434,4455,6723,6840 'cross':2019 'crosslink':430,7738,7740 'crystal':124,940,1375,1382,8303,8950,9424 'crystalliza':324 'cup':4959,4988 'current':1046,4279,4841,5213,5594,6073,6080,6092,7534 'curv':7388 'curvatur':8663 'cut':1809 'cutoff':4189,4768 'cydy':970,7977 'cytochrom':9726 'd':29,44,1272,1558,1584,1589,1595,1596,1599,1695,1701,4478,4632,5318,5376,6464,8935,9608 'dabbeni':8974 'dabbeni-sala':8973 'damag':4850 'dark':4898 'data':1072 'day':219,2593,4523 'ddm':1733 'debri':4783,5015 'deceler':594 'declar':8202 'decreas':600 'deepli':2704,2749 'defect':8571 'defici':8451 'defin':1995 'degrad':7909 'dehydrogenas':816,821,876 'del':1960 'delipid':434,6715 'demand':920 'denatur':687,1179 'dencher':8941 'denitrifican':3546,8850 'densiti':2359,2640,3076 'depend':391,619,6797,7349,7527 'der':16 'deriv':3827 'desalt':4181 'describ':146,2058,2079,2998,4584,4592,4599,4629,4642,4652,4668,4950,5682,5708,5741,6130,6784,6815,7519,7651 'desir':2163,4312,4505,5069,7844,8006 'destain':5230,5254,7515 'detail':5987 'detect':964,982,4343,4467,6804,7973 'deterg':376,390,394,406,444,476,489,669,690,1132,1144,1155,1160,3121,3127,3452,3512,3646,3965,4228,6569,6587,6608,6616,6623,6636,6835,6980,6983,6991,6997,7055,7101,7127,7302,7339,7348,7804,7814 'detergent-depend':7347 'detergent-labil':1159,4227,6996 'detergent-lik':6990 'detergent-sensit':689 'detergent/dye':3228,3702,4063,6964 'detergent/lipid':7070 'detergent/protein':3515,6592,6810 'determin':95,290,299,1234,2103,7145,7294,7360,8057,9134 'deutsch':8180 'develop':225 'devenish':9317,9350 'deviat':7190,7250,7267 'devic':2366 'dfg':8182,8190 'diagnost':281 'dialysi':4178,4772 'diego':8322,9443 'dietmey':9147 'differ':1254,6350,6675,7411,7704,7996,8156 'differenti':2637 'diffus':118,4459,4503,4590,4605,4934,4962,4997,5725 'diffusion/concentration':6451 'digitonin':408,438,501,1303,1336,1348,1639,3261,3289,3317,3494,3670,3725,3970,4000,4098,6702,6710,7406,7434 'digitonin-solubil':7405 'digitonin-solubl':7433 'digitonin/protein':1643,1709,6718 'dilut':3073,3833 'dimens':168,183,192,1102,1117,4292,4620,4635,4648,4658,5330,5341,5392,5430,5579,5590,5701,5774,6060,6470,6689,8073,8081,8093 'dimension':5321,6119,6331,8282,8526,8596,9208,9477,9638,9670 'dimer':496,1677,1700,6657,6672,6679,8650,8719,8791,8796,9368,9568 'dimer-specif':6678,8795 'disappear':4475 'discuss':7596 'diseas':8569,8606 'disord':8630 'disorders10':285 'disrupt':9262 'dissect':9466 'dissoci':458,1085,1158,1738,2898,4225,5646,6961,6995,7040,8084 'dissolv':922,1276,1292,1314 'distanc':597,4315,4551,7333,7399 'disturb':2099 'disulfid':5820,7743 'dival':2935,6915 'divid':3849 'document':5246 'dodecyl':1270,9389 'dodecyl-b-d-maltosid':1269 'dodecylmaltosid':431,789,1725,3244,3272,3300,3464,3652,3713,3968,3974,4072,5363,5569,5643,6709,7391,7402 'dodecylmaltoside-solubil':788,7390,7401 'dodecylmaltoside21':485 'doesn':7993 'doi':60 'domain':9109 'doubl':1185,6614,7685 'dri':2743,5244,6202 'driven':1908,3783 'dsds':1190,7690,7714 'dsds-page':1189,7689,7713 'dtt':1434,6175 'dudkina':8638,8675 'dures27':958 'dye':512,523,547,655,678,1148,2769,2942,2948,3214,3222,3698,4049,4057,4218,4355,4458,6887,6907,6987,7007,7022,7064,7079,7121,7140,7221,7494,7860,8943 'e':767,781,891,909,1594,1604,1763,1767,1872,1890,2846,2864,3574,3592,4379,4397,4644,4670,5289,5307,5712,5760,6233,6251,6496,6930,6948,7560,7655,7763,7781,8970,8999,9017,9371,9510,9806,9824 'e.g':1494,2627,2885,2961,3051,3813,4183,4769,4872,5070,5432,5723,5851,7320,7667,7906,8032 'e.j':8645,8682 'earli':7466,9492 'eas':726 'easi':8055 'ed':8309,9430,9684 'edman':7908 'edn':8308,9429 'edta':2190,2824 'effici':2633,2729,4452,5651,5833,5955,7473 'either':2522,5567 'electroblot':139,160,336,1035,1061,1440,1450,1525,1532,4597,4692,5088,5100,5124,5137,5169,5956,7455,7469,7488,7524 'electrod':1552,2051,2647,4807,5141 'electroelut':116,724,1033,1814,1940,4499,4582,4610,4703,6124,6284,6440,7459 'electron':126,327,943,4874,5072,9051 'electrophoresi':1465,1485,4176,4232,4276,4410,4426,4473,4485,6047,6418,6769,7057,7563,7595,7649,8233,8272,8284,8297,8430,8530,8575,8592,8612,8960,9034,9066,9209,9249,9308,9393,9417,9479,9480,9576,9639,9654,9671 'electrophoret':8553 'electrotransf':7604 'elut':4501,4603 'elutor':1937,4757,4790,4846 'elvehjem':1917,3792 'embo':8455,8799,8821,8925,9140,9767 'embryon':3815 'encod':6369,9267 'end':4471,4751 'enrich':8004 'ensur':7472 'enter':2781,4271,6759 'enzym':923,5756,6634,8588,8983 'enzymat':8241 'enzymol':8440,9530,9736 'ep':915,1896,2870,3598,4403,5313,6257,6954,7787,9023,9830 'equip':1463 'erron':4859 'error':7180,7445 'especi':730,2157,4191,4611,5829,6357,7195 'essenti':2720,7992 'estim':4563,7162,7428 'et':1951,4763,8379,8396,8448,8466,8490,8551,8620,8747,8830,8913,9099,9131,9149,9229,9260 'eubel':8677,8859,8890,9074,9604 'even':558 'exampl':6651 'excel':7573 'except':1020,2703,2787,8000 'excess':5448,6979,6982,7100,7236 'exchang':4198 'excis':712,4480,4490,4513,4580,4714,4943,5380 'exclud':987 'exemplifi':247,793,1123,1198,7891,8043,8131 'exhibit':7572 'exist':8788 'exp':8764 'experiment':5986 'explain':7044,8755 'expos':7098,9108 'extend':2456 'extract':157,4420,4453,4588,4827,4860,4932,4960,5037,5722,7461 'f':4655,4672,5745,6116,6519,8712,8976,9512,9743 'f.w':9487 'f1':9327 'f1-stator':9326 'f1fo':8785,9335,9354 'f1fo-atp':8784,9334,9353 'facilit':711 'fact':7440 'faint':4345,4461 'fall':6081 'famili':6373 'fandino':9227 'far':1044 'featur':8726 'feb':8668 'fibroblast':3819 'fig':8061 'figur':249,784,1125,1200,1607,1770,7893,8045,8133 'fill':4793,5541,5927,6019 'filter':4185 'final':1175,1821,3661,3674,5633,6144,6825,7485 'financi':8208 'find':2269,6620 'first':1791,2280,5391,5429,5589,8080 'first-dimens':5428,5588,8079 'fit':1910,3785 'fix':4531,4573,5443,6424,6486,6509,6534 'fluka':1275,1291,1313,1355,1361,1367 'fluoresc':966,981 'fluorophor':7978 'fluorophors36':971 'fo':8724 'focus':186,5629,6266 'fold':3080 'follow':153,188,502,3912,4429,4483,5511,5997,6422,6434,6474,6507,6523,7950 'follow34':926 'forcep':3763 'form':1010,2771,8243,8690,9104 'forschungsgemeinschaft':8181 'found':6547 'four':6387 'fraction':2603,2615,2626,4195 'fragment':3047,3902 'frankfurt':21,31 'freez':2903,2907,3914,4909 'freezing/thawing':1340 'fresh':3429 'fret':9321 'fromo100':7886 'front':4303 'frozen':3432 'fu':34 'function':9153,9649,9760 'furthermor':643,5746 'g':515,780,897,1390,1762,1878,1977,1981,2678,2852,3220,3386,3580,4055,4385,5295,6239,6936,7769,8230,8258,8314,9005,9385,9435,9723,9812 'g/g':1570,1645,1715,1748,3469,3473,3477,3485,3489,3493,3499,3503,3507,3665,3678 'gamma':9270 'gap':5479,5543,5914,5929,6021 'garcia':8715 'gation':509 'gavin':9315,9346 'ge':1500 'gel':114,133,312,331,590,606,675,709,838,862,929,954,963,1040,1064,1446,1456,1478,1484,1530,1813,1851,1925,2053,2206,2215,2222,2227,2236,2249,2284,2305,2326,2364,2380,2430,2455,2468,2477,2482,2502,2509,2517,2519,2525,2531,2539,2551,2562,2567,2569,2575,2649,2663,2740,2742,2783,3197,3342,3354,3356,3359,3375,4174,4273,4291,4510,4527,4575,4615,4721,4732,4736,4782,4968,4984,5014,5038,5051,5086,5094,5151,5219,5349,5394,5395,5420,5431,5457,5470,5483,5495,5521,5531,5545,5554,5617,5621,5674,5690,5755,5785,5850,5858,5874,5903,5918,5939,5965,5984,6003,6009,6030,6037,6058,6105,6128,6288,6292,6300,6311,6317,6406,6761,6767,6796,6860,7092,7540,7643,7705,8082,8094,8362,8529,9033,9065,9307,9392,9478,9498,9523 'gel-cast':2363 'gel-shift':311 'gelelectrophoresi':8349 'gels42':4548 'genates9':277 'gencic':9719 'gene':9266 'general':984,7290,7815,9160 'generat':6988,8921 'genom':9653 'gentl':5999 'germani':32,47 'gger':8226,8252,8293,8316,8358,8427,8488,8549,8582,8780,8807,8939,9204,9298,9381,9414,9437,9448,9715 'gger1':12 'give':3226,3659,3672,3700,4061 'given':1843,2311,3128,6595 'glass':2376,4889,5412,5438,5796,5863,5891,5907 'glass/teflon':1914,3789 'glassi':1550 'gler':8327 'glide':5467 'glove':2045,3847 'glycerol':1360,1413,2404,2884,2895,3207,3384,3693,4045,4203,4905,4921 'glycin':7570,7623,9136 'glycosylation7':7265 'go':9660 'gradient':589,605,837,1924,1926,2226,2233,2282,2293,2379,2423,2429,2454,2476,2501,3196,3353,3357,5494,5509,9677 'gradients51':6305 'gradual':593,5601 'gram/gram':1713,3232,3518 'grant':8191 'granul':8417 'great':7989 'griffon':9129 'guid':8306,9427 'h':900,1881,1935,2208,2855,2993,3583,4237,4388,4414,4710,4755,4799,4838,4941,5000,5097,5191,5298,5334,5778,6109,6113,6122,6182,6242,6408,6414,6421,6426,6431,6443,6454,6461,6473,6483,6488,6493,6504,6511,6516,6522,6531,6536,6541,6939,7772,8227,8253,8294,8317,8359,8428,8489,8550,8583,8678,8781,8808,8860,8891,8937,8940,8953,9008,9075,9205,9299,9382,9415,9438,9449,9605,9716,9815 'h-shape':1934,4754,4798 'h.c':9747 'h.p':8337,8647,8684,8865,8895,9081,9177,9464,9613 'h.p.b':8196 'h.s':53,1070,8188 'half':6606,6787 'han':7 'handbook':9683 'handl':2041,7997 'hannov':39,46 'hans-pet':6 'harm':7818 'haus':27 'hcl':2795 'head':9704 'healthcar':1501 'heart':810,3865,3939,3987,4013,4084,4110 'heat':1327,5030 'heinemey':8640,8892,9076,9602 'help':6618 'hermann':10 'herrenha':40 'hi':895,1876,2850,3578,4383,5293,6237,6934,7767,9003,9810 'higgin':9042 'high':475,2038,2131,2149,3075,4193,4854,6666,6775,6883,7197,7921,8756,9666 'high-dens':3074 'high-deterg':474 'high-resolut':9665 'high-salt':4853 'higher':2124,5065,6361,7243,7615,9624 'histochem':8985 'hjerten':9536 'hjerten46':2003 'hole':2369 'homo':276 'homogen':88,921,1918,3111,3636,3742,3765,3793,3807,3850,3957 'hood':9513 'horizont':5687,6307 'hour':2759 'howev':573,972,1045,2441,7268,7985 'human':283,3537,3818,8499,8624,9731 'hunkapil':1950,4762,9507 'hunt':8310,9431 'hydro':460,649 'hydrophob':540,7606,7673,9238 'i.e':3233,3706,4067,7229,7379,8063 'i.w':1068 'ideal':2917,4157,7729 'ident':8048 'identif':1623,7671,8398,8792,8896,9220 'identifi':104,316,421,1104,1176,6567,7371,7731,7932,7945,7980,8025 'ief':187,1418,1755,4650,5735,6217,6267,6276,6310,6520 'ief/sds':1194 'ief/sds-page':1832 'ignor':7438 'ii':165,763,1505,1580,1582,1657,3089,3624,3848,4733,4978,5114,5379,5798,6132,8882 'iii':180,761,772,1567,1575,1578,1658,1679,1745,3110,3635,3918,4747,5003,5126,5421,5824,6166,6200,8470,8698,8838 'iii2':9279 'ilka':4 'imida':5059 'imidazol':1206,1215,2110,2670,2791,4814,5107 'imidazole/hcl':1238,2814,2956,3776,3841,4164 'immedi':4468,4482,5235 'immobil':6303,7129,9675 'immobilon':1555 'immun':9788 'immunodetect':141,7971 'immunodetection15':338 'implic':9646 'import':5823,9161 'import32':346 'impos':549 'improv':1058,4350 'in-gel':131,329,952,961,5753 'inappropri':7282 'includ':2367,2617,6386 'incorpor':7068 'increas':2355,5600,6089,6861 'incub':6148,6179 'indic':829,1642,1661,1670,1696,2005,2015,3519,6630 'individu':454,607,1088,1165,1627,1741,4553,8087 'induc':8660 'ineffici':7449 'inexpens':367,2731 'inform':8217 'ing':1948 'inhibit':2201,4825 'inhibitor':2734,5834 'initi':2410,6576 'inner':8666 'insert':5473 'insight':8867 'insolubl':1346 'instanc':6375 'instead':1000,1216,2175,6703 'intact':1168 'integr':9214 'interact':108,425,462,1106,7734,9330,9358 'interactions20':320 'interest':401,2605,3172,7310,7926,7963,7984,8198,8209 'interfer':973,1229 'intermedi':467 'interv':2594 'intramitochondri':9567 'intrins':7108 'introduct':220 'investig':1047 'involv':7995,8919 'ion':2939,4197 'ionic':2159,4149 'isb1.6':5397 'isoelectr':185,1834,6265,6355 'isoform':6336,6388,8157 'isol':76,259,352,414,449,935,2595,2607,6733,7842,8027,8119,8235,8274,8364,8432,9212,9244,9515,9590 'iv':762,773,1579,1581,1660,1665,1691,1747,3122,3647,3956,4779,5022,5148,5446,5839,6197,8840 'ix':3318,4046,5240,5661 'j':8386,8395,8418,8456,8504,8641,8763,8800,8822,8851,8893,8926,9077,9121,9141,9191,9250,9287,9372,9499,9566,9603,9633,9686,9768 'j.j':8716 'ja':8329,8861,9169,9609 'jagow':8229,8257,8313,9384,9434,9722 'june':58 'jung':9040 'k':8378,8774,8810,9148 'k.n':9589 'kai':25 'kara':9201 'kda':241,756,757,799,868,871,1800,2437,2446,4771,5950,5973,7887,8405 'kdalton':9405 'keegstra':8642,8679 'keep':1164,6779,6834 'kept':6892 'kg':5183 'kidney':3878,3952,3993,4019,4090,4116 'klose':9632 'kobalz':9634 'kruft':8332,9172 'ku':8326 'l':8331,8863,8972,9171,9611 'l.e':9514 'label':967 'labil':459,1161,4229,6998 'laboratori':9682 'laemm':7567,9694 'lane':1706,5388,5870 'larg':544,7202 'larger':2390,3064,3173,3900,7189,8028 'late':5625 'later':7031 'layer':4901 'lc':9232 'lc-nanospray-m':9231 'lead':1344,2134,7442 'least':6585 'leav':2549,3142,3680 'left':1848,5921,6024 'less':1128,1546,4353,6714,6722,7039,7850,7959,8099 'less-abund':7958 'let':4996 'lett':8669 'level':9280 'like':968,4689,6289,6992 'limit':624,4190,4280,4286,4442,4831,4840,5212,5595,6074,6094,6731,7173,7533,9302 'line':3823,3831,8563 'linear':833,2422 'link':7976,9154 'linker':2020 'lipid':6977,7353,7827 'lipolytica':3533 'liquid':2910,3916 'lis':894,1875,2849,3577,4382,5292,6236,6933,7766,9002,9809 'lithium':6894 'liver':3881,3955,3996,4022,4093,4119 'lizat':941 'load':1540,3319,5184,6412,6730,6738,6772,6793 'loadb20':4121 'loadb25':3727 'long':2317,5475 'longer':7097 'lose':647 'lot':1334 'low':488,1022,1131,2158,2181,2345,2774,2927,4148,4767,6659,6824,6828,6893 'low-cutoff':4766 'low-deterg':487 'lower':4750,5140,6729 'ls':919,1900,2874,3602,4407,5317,6261,6958,7791,9027,9834 'ludwig':8394 'lujan':9509 'lymphoblast':3821 'lysat':8539 'lysi':1415,6214 'm':911,1205,1212,1237,1245,1325,1353,1363,1421,1559,1566,1574,1577,1597,1693,1698,1892,2244,2685,2747,2866,3070,3417,3433,3594,4399,4454,5309,6253,6950,7783,8328,8515,9019,9202,9259,9320,9349,9751,9777,9826 'm.w':9508 'ma':1515,4283,4844,5216,5598,6077,6084,6097,7537 'macromolecul':9544 'made':3427 'magnet':1932 'maintain':8474 'maltosid':1273 'mammal':6359 'mammalian':3012,3105,3263,3460,3534,3746,8478,8819 'mani':359 'mar':1961 'mark':865,1247,1802,8627 'marker':4557,7378 'mash':4734,4945 'mass':98,237,292,796,830,1838,2261,2278,2434,2458,4565,5951,5974,6348,7148,7161,7246,7253,7258,7293,7318,7359,7431,7669,7879,8115,8262,9218,9251 'match':7385 'materi':1202 'matrix':9107,9766 'matrix-expos':9106 'maxim':6067 'maximum':6736 'may':2265,2616,4474,6629,7083,7261,7526,7545,7662 'mda':801,2440,2463,3176,3186,7889 'mda1':244 'mean':666 'measur':979,9046 'mechan':4777 'med':8765 'meising':9584 'membran':82,231,264,383,451,480,506,535,636,644,658,1554,2123,2151,2600,2602,2619,2623,2878,2923,2978,2991,2996,3005,3022,3046,3109,3113,3292,3457,3463,3544,3638,4365,4567,4773,5117,5156,5260,6383,6563,6572,6582,6626,6870,6968,7013,7165,7199,7307,7314,7330,7376,7393,7408,7436,7498,7509,7512,7610,7831,7919,8237,8276,8299,8406,8667,9113,9188,9215,9241,9420 'membranes4':269 'mention':7275 'mercaptoethanol':1457,5816,5831 'methanol':1438,5119,5224,5263 'method':1235,6805,8369,8439,9529,9594,9735 'method19':314 'methodolog':147 'mg':3033,3067,3100,3138,3615,3759,3810,3857,3862,3868,3875,3931,3936,3942,3949,6748 'micell':6841,7071,7082 'microgram':261,7874,9517 'microhomogen':4955 'microscop':944,4875,5073 'microscopi':127 'microscopy26':328 'might':997,6655,7042,7965 'migrat':562,592,703,4550,7112,7332,7398 'mild':442,1129,7851,8100 'mildest':405 'milligram':7900 'millipor':1557 'min':2492,2555,3039,3053,3149,3154,3161,3619,3687,3886,4028,4033,5007,5267,5803,5811,6151,6162,6193 'minauro':8710 'minauro-sanmiguel':8709 'ming':7255 'minim':694,1539,6792 'minut':5454,5465 'mitochondri':230,284,470,791,811,1662,1777,1787,6382,7325,8434,8628,8688,8720,8732,8783,8922,8980,9050,9116,9155,9187,9269,9283,9469 'mitochondria':2120,3002,3019,3031,3104,3106,3236,3264,3459,3461,3528,3535,3554,3898,6734,8368,8479,8654,8820,8874,8903,9088,9593,9734,9757 'mitter':343 'mix':1974,1993,2413,3368,6981,7081 'mixer':1927,2294,4994 'mixtur':1141,1968,3348,3681,5904,5940,5966,6038,6965,7677 'ml':1281,1297,1320,1913,1985,2329,3068,3092,3205,3239,3246,3256,3267,3274,3284,3295,3302,3312,3322,3370,3372,3374,3379,3381,3383,3389,3391,3393,3397,3399,3401,3405,3407,3409,3412,3414,3416,3467,3471,3475,3483,3487,3491,3497,3501,3505,3627,3650,3668,3691,3708,3720,3728,3768,3788,3921,3978,3985,3991,4004,4011,4017,4043,4075,4082,4088,4101,4108,4114,4122,4725,4958 'mm':1396,1428,1433,1436,2165,2178,2188,2192,2667,2671,2737,2812,2815,2821,2825,2887,2891,2930,2955,2973,3772,3775,3837,3840,4160,4163,4810,4813,4819,5053,5058,5103,5106,5130,5163,5398,5434,5853,5856,5913,6013,6174,6177 'mobil':4448,6295 'modifi':175,1094,3548,5343 'mol':8480,8540,9288,9500,9595 'molecul':548 'molecular':1837,6347,8261,9133 'molecules2':7222 'monom':2013,2024,8659 'monomer':1674,1697,6664,6670,9778 'monoval':8753 'month':2584,2987,4930 'mop':2967 'moro':9742 'morpholog':8924 'most':7865 'motor':1907,3782 'motor-driven':1906,3781 'mount':5176,5422,5840 'mous':3539,3814 'move':4305,5888 'ms':9234 'multi':8533 'multi-protein':8532 'multidimension':8128 'multiprotein':304,418,2900,3169,3187,7001,8556 'multisubunit':8141 'multival':8751 'muscl':3866,3872,3940,3946,3983,3988,4009,4014,4080,4085,4106,4111,8979 'must':609,3425,4869,6890,7619,7805 'n':888,906,1591,1601,1869,1887,2843,2861,3571,3589,4376,4394,5286,5304,6230,6248,6927,6945,7760,7778,8447,8996,9014,9130,9587,9803,9821 'n.a':8942 'n.v':8639,8676 'nacl':2931,4161 'nadh':8842 'nanospray':9233 'narrow':2277 'nat':9454 'nativ':2,64,96,109,138,156,159,291,335,699,721,795,847,936,993,1101,1442,1452,1942,4351,4419,4425,4564,4705,4936,5090,5323,5327,5520,5553,5679,5849,5857,5873,5917,6002,6029,6036,6287,6417,6458,6467,7147,7164,7430,7454,7458,7468,7795,7882,8070,8232,8271,8283,8296,8347,8361,8429,8591,8959,9031,9063,9248,9305,9575 'native/sds':8528 'natl':8700,8736 'natur':882,1859,2837,3561,4370,5276,6224,6917,7754,8986,9163,9708,9797 'neff':8934 'negat':626,7215,7238 'ner':9607 'nervous':9059 'neupert':8775,9748 'neurol':8632 'neurotox':2039 'neurotran':342 'neutral':443,7054,7126 'new':8866,9690 'next':4746 'ng':896,1877,2851,3579,4384,5294,6238,6935,7768,9004,9811 'nice':6400 'nitrocellulos':7508 'nitrogen':2911,3917 'no.1':880,1862,2835,3564,4368,5279,6222,6920,7752,8989,9795 'non':1259,2232,4572 'non-fix':4571 'non-gradi':2231 'nonion':375,389,1143 'nonreduc':7717 'normal':6591,6819,8598 'novel':8403,8723 'npg.nature.com':8222 'nsch':8330,8862,9170,9610 'nuclear':9265 'nuclei':3897 'number':545,7203 'o':758,769,823,1544 'o100':798 'o7':3438 'observ':4252,7248 'obtain':2425,3893,5978 'occupi':680 'occur':9362 'often':2630,3048,5234,6343,6971,7209,7600,7929 'oligomer':100,294,6697,8264,9780 'omit':5078 'one':74,257,402,1683,2304,3124,4308,4742,6604,6612,7964 'one-step':73,256 'one-third':4307 'onlin':56,8211,8220 'onto':6313 'open':2419 'oper':8097 'optim':2252,6562,6622 'option':1419,4586,4594,4601,4631,4654,6435,6475 'order':3181,5627,6832 'organ':9620 'organell':2628,2629 'origin':2091,7356 'orthogon':1093,7698 'osteosarcoma':3825 'ostrand':9511 'otherwis':2026 'outer':6381,9186 'overcom':7213 'overlay':2565,2587,5923 'overnight':4712,4865,5002,5780,6445,6485,6506,6533 'owe':7822 'oxid':1614,7326,7914,8435,8453,8573,8586,8981 'oxidas':8843 'oxoglutar':820,875 'p':740,765,768,818,892,902,1556,1592,1602,1765,1873,1883,2847,2857,3575,3585,4380,4390,5290,5300,6234,6244,6931,6941,7764,7774,8912,9000,9010,9807,9817 'p.d':9316,9347 'p13':8186 'pa':779,1761 'page':3,65,68,152,173,178,197,211,223,698,776,806,842,855,948,995,1017,1027,1055,1082,1097,1121,1188,1191,1462,1493,1519,1523,1587,1621,1655,1796,2067,2074,2147,2220,2654,2724,2803,3026,3201,3334,3610,3740,3753,4134,4143,4156,4242,4624,4640,4663,4688,4697,5324,5346,5360,5582,5593,5706,5739,5749,5764,5791,5993,6323,6480,6501,6528,6691,7020,7035,7152,7155,7287,7298,7465,7579,7626,7632,7661,7688,7691,7701,7715,7796,7836,7863,7942,7954,8013,8019,8106,8146,8172,9453 'page2':572,7364 'page40':1183 'page43':1196 'page47':2243 'page6':1004 'paper':1903,5135,5167,7556 'papers1':2092 'paracoccus':3545,8849 'parallel':8108 'parkinson':8604 'particl':1635,4878,5076,9547 'partner':1107,7735 'pasteur':4885 'paumard':8911 'paus':2572,2975,3903,4506,4915 'pellet':3028,3102,3612,3633,3895,3905,3927,5012,6204,6212 'per':1983,3136,4727,4845 'peral':9258 'percentag':2017,7616 'perez':8464 'perform':149,4239,4627,4666 'peristalt':1919 'permiss':1858,8216 'persulf':3422 'peter':8 'pfanner':9586 'pfeiffer':8377,8773,8809 'pflanzengenetik':36 'ph':567,1239,1261,2687,2785,2828,2957,3777,3842,4165,4815,5061,5110,6304,9676 'ph-adjust':1260 'phobic':461,650 'phosphat':2963 'phosphoryl':1615,7327,7915,8436,8454,8574,8587,8982 'photosynth':8350 'photosystem':9622 'physiol':8884,8905,9090 'physiolog':105,317,354,422,5647,6642,6696,8029,9284 'phytochemistri':9626 'piec':5039 'pig':3540 'pio':7109 'pipet':2049 'pipett':4886 'place':4986,5149,5180,5408,5435,5782 'plant':6362,8873,8883,8904,9087,9089,9120,9190,9625 'plants23':3560 'plasma':3543 'plastic':5534,6014 'plate':2377,5413,5439,5797,5864,5892,5908 'platelet':8560 'plex':7169 'pmsf':1437,6178 'poetsch':8932 'point':1835,2573,2976,3904,4432,4507,4916,6356 'polyacrylamid':6316,8348,9032,9064,9306,9391,9522 'polymer':2487,2488,2504,2557,3443,5512,5836,5998 'ponceau':1405,1410,4206,4210 'pore':601,622,4440,9162 'pore-s':621 'posit':5417,5461,5898 'possibl':6693,7416,7930 'potassium':2933,6913 'potato':3557,8902 'potenti':6568 'potter':1916,3791 'potter-elvehjem':1915,3790 'pour':5900 'power':1510,1534,4262,8152 'practic':8305,9426 'pre':6392,7011 'pre-protein':6391 'pre-purifi':7010 'precipit':2940,6133,6885,6904 'prefer':2951,5195 'preferenti':4711,4964,6444,6901 'prepar':1971,2055,2077,2524,2634,3350,4135,4731,5099,5336,6410,7897 'preprotein':6377,9156,9182,9754 'prepurifi':4138,4221,6967 'prescott':9319,9348 'presenc':7825 'present':1009 'presepar':7046,7102 'preserv':492,6695 'presum':7343 'prevent':2741,2778,4849,6756 'previous':1219,4539 'principl':374 'pro':6969 'problem':6555,7143,7278,7990 'problemat':7612 'proc':8699,8735 'proce':957 'procedur':2202,6271 'procedures.22':2611 'procedures40':6327 'proceed':5017,5667 'process':1719 'progress':8111 'project':8185 'prolong':2745 'promis':1048 'properti':435,541,6962,6993,7856 'protean':1504 'proteas':2200,2733,4824 'protect':3846 'protein':78,97,106,107,163,232,265,318,319,345,355,398,423,424,452,471,481,536,555,560,574,591,608,628,637,659,664,681,692,700,722,937,1037,1169,1233,1443,1453,1483,1788,1841,1943,2102,2139,2152,2257,2433,2442,2779,2944,3034,3061,3139,3616,4139,4169,4346,4361,4422,4434,4462,4558,4568,4706,4716,4828,4861,4867,4937,4947,5066,5091,5231,5631,5636,5715,5945,6125,6134,6168,6283,6340,6365,6372,6393,6573,6588,6737,6751,6757,6803,6820,6829,6846,6852,6871,6910,7014,7030,7049,7088,7107,7166,7200,7218,7245,7276,7308,7315,7331,7369,7377,7384,7394,7409,7426,7437,7452,7476,7502,7598,7607,7674,7680,7733,7821,7845,7869,7883,7923,7933,7961,7982,8007,8030,8161,8238,8267,8277,8300,8342,8365,8409,8437,8534,8601,9216,9239,9242,9398,9421,9495,9520,9641,9673,9699,9762 'protein/coomassie':4900 'proteins2':7178 'proteom':733,4976,8542,9221,9309,9470,9614 'protocol':144,206,883,884,1031,1860,1865,2838,2839,3549,3562,3567,4371,4372,4541,5277,5282,6225,6226,6918,6923,7755,7756,8120,8987,8992,9455,9644,9798,9799 'protocols41':5994 'proton':8413 'proton-atpas':8412 'provid':7664 'publish':55,4540,8210 'pump':1920,2337,2384,4322 'purchas':1955 'purif':8301,9422 'purifi':7012,8956 'puriti':1309 'push':5524,6000 'put':5171,5860 'pvdf':1553,4364,5116,5155,5259,7497,7609 'pyruv':815 'quantif':8584,8977 'quantiti':3450,9048,9518 'quit':8150 'r':35,8465,8519,8619 'r.a':8778 'r.j':9318,9351 'rad':1509 'rai':9199 'rang':238,797,2279,2435,2459,5952,5975,7187,7638,7880,8760,9401 'rapid':7160 'rare':2230 'rassow':9565 'rat':3538 'rather':7280,7489 'ratio':582,1644,1710,3229,3516,3703,4064,6589,6593,6719,6811 'ratio6':1111 'reach':2320,4438,7028 'reagent':1203,1963,2299,6219 'reason':6880 'recent':5989,8015 'receptor':6394,9137,9157,9787 'recov':112,3928 'recoveri':717,728,6449 'recrystal':1312 'reduc':642,6973,7501,7723,8609,9273 'reductas':9729 'ref':4699 'reif':9561 'relat':4555,8054 'remov':2494,2559,4316,4881,5447,5537,6016,8944 'renew':2590 'repeat':1339,4740,6198 'repel':630 'replac':2107 'report':7194 'repres':1803 'reprint':8214 'reprintsandpermiss':8223 'reproduc':1852 'requir':671,1331,2267,3049,3059,3453,7839,8472 'res':8351 'research9':350 'resembl':1151 'residu':7208 'resist':1541 'resolut':1012,1051,5766,7574,7666,7721,7727,8597,9667 'resolv':1830,2450,5944 'respect':1692,4679,6114,6352,6639 'respirasom':8494,8900 'respiratori':493,1166,1671,1734,8039,8383,8468,8500,8693,8814,8834,8870,9082 'respiratory-chain':8692 'respons':8762 'restrict':7104 'result':5109,7263,7793 'resuspend':6167,6210 'retain':1077,3177,4037 'retard':7066 'reth':9776 'reus':4327 'reveal':9322 'rg':9661 'rho':3828 'right':6026 'rins':5241,5399,5825 'risk':685 'rnas':9493 'ro':898,916,1879,1897,2853,2871,3581,3599,4386,4404,5296,5314,6240,6258,6937,6955,7770,7788,9006,9024,9813,9831 'room':2484,2511,2765,4254,5198,6049 'rpm':3795,3805 'rule':7291 'run':596,4302,4314,4334,4417,4486,5585,5604,6099 's/glycerol':1406,4211 'saccharomyc':3529 'sala':8975 'salt':2125,2132,2919,4194,4855,6864,6884,6888,6897 'sampl':2142,2467,2508,2516,2534,2546,2561,3077,3143,3167,3329,3341,3355,3735,3851,4129,4146,4173,4222,4269,4919,5419,6409,6743,6766,6789,6859,6868,7029,7074,7132 'san':8321,9442 'sanmiguel':8711 'satisfactorili':1067 'satur':7077 'scale':7875,7901,8123 'scha':11,8225,8251,8292,8315,8357,8426,8487,8548,8581,8779,8806,8938,9203,9297,9380,9413,9436,9447,9714 'schagger@zbc.kgu.de':54 'schamel':8522,8745,9774 'schmitz':8334,9174 'schneider':9746 'sci':8702,8738 'scientif':1958 'sd':777,1759 'sds':172,196,854,1182,1187,1461,1477,1522,1529,4547,4639,4662,4687,4696,5705,5738,5763,5806,5814,5938,5964,5992,6315,6322,6479,6500,6527,6690,7578,7625,7631,7642,7660,7687,7700,7953,8171,9416,9452 'sds-page':1186,1521,5737,6321,6478,7577,7624,7659,7686,7699,7952,8170,9451 'sds-page40':1181 'se':1496 'seal':4493,4748,4990 'second':167,182,1100,1116,4619,4634,4647,5340,5404,5437,5578,5771,5862,6603,6688,8092 'second-dimens':5577,6687,8091 'sector':8407 'sediment':3086,3811 'see':4693,5373,5988,6218 'seelert':8936 'seem':5644,6711,7036,7847 'select':2203,2210 'semidri':1547 'sensit':691,6570,6637,6800,8757 'separ':162,200,228,577,786,825,850,1609,1650,1792,1816,2254,2283,3183,3358,4152,5530,5733,5902,6008,6281,6293,6298,6334,6399,7091,7637,7711,7870,8076,8554,9396,9539 'sequenc':9527 'serin':2732 'serva':1376,1383,1392 'set':4261,5200,7421 'setup':1964 'sever':2263,2758,2986,3079,4522,4929,5403,5664 'several-fold':3078 'shape':1936,4756,4800 'sharp':5635,6855 'sharper':5979 'shift':313,552,7241 'shock':2906,3913,4908 'shock-freez':2905,4907 'short':4416 'show':466,8136 'shown':3523 'sigma':1207 'signific':8492 'silicon':4893 'silver':4535,7969 'similar':483,499,2260,3799,5046,6346,7456,7712,7797 'simpl':373 'sinc':1225,2095,2130,2767,4247,7201,7329,7396,7492,7510 'singl':4877,5075 'single-particl':4876,5074 'sirrenberg':9744 'situat':6778,7658 'size':586,602,618,623,4441,9542 'size-depend':617 'skelet':3871,3945,3982,4008,4079,4105 'slab':9497 'slight':2694,3547,4336,5424 'small':4967,6371 'smaller':2443,4614 'soak':5127,5160,5799 'sodium':2179,2810,2962,2966,6896,9388 'sodium-mop':2965 'solubil':380,479,503,790,808,1637,1647,2062,2069,2112,2119,2798,2804,2807,2988,2995,3017,3093,3145,3455,3525,3604,3628,3683,3744,3922,4024,6564,6627,6781,6817,7304,7392,7403,7407,7811,7830,8009 'solubl':526,663,7177,7368,7383,7425,7435 'solut':1250,1408,1416,2341,2347,2353,2396,2402,2409,2702,3423,3437,4213,5025,5042,6143,6215,6556 'sometim':6368 'sonderforschungsbereich':8183 'sonic':1634 'sorbitol':2892 'sourc':7956 'spacer':2372,5423,5486,5548,5841 'spatula':3118,3643,3962 'special':1471,2271,6055,7589 'specif':388,620,714,2276,6580,6633,6680,7277,7530,8797,9369 'specimen':3761 'spectrom':9252 'spectrometr':7670,8116,9219 'spot':5637,5716 'squeez':4738,4780,5877 'stabil':395,4778,8382,8845 'stack':2137,5131,5164,6847,6850,7557 'stain':720,956,4536,4543,4690,6429,6491,6514,6539,7949,7970,9028 'stalk':9329 'standard':1840,2610,6270,6326 'start':5576,6046 'state':101,2027,3135,6643,6698,8265 'statement':8199 'states2':295 'stator':9328 'stay':7128 'steep':5490 'steiml':8520 'stem':3816 'step':75,258,2246,3072,4456,5377,5498,5557,5684,5710,5726,5743,6041,6199,6436,6437,6446,6455,6462,6494,6517,6557,6597,6724,6844,6959,7141,7446,7521,7558,7653 'stern':24 'still':4445,6093 'stir':2756 'stirrer':1933 'stock':1249,1263,1386,1407,3223,3975,4001,4058,4212 'stoichiometr':1110 'stoichiometri':301 'stop':610,4409,5624,7462 'storag':2597,2746 'store':1208,1241,1279,1295,1317,1356,1368,1401,1986,2578,2711,2763,2913,2981,3431,3908,4488,4517,4911,4924 'str':42 'strategi':6274 'streak':5630,7601 'strength':2160,4150 'strip':5401,5471,5484,5522,5526,5546,5691,5786,5800,5919,6004,6031,6301,6312 'stroh':8828 'stroke':3798,3803 'strong':7703,8662 'stronger':433 'structur':5649,8648,8685,8717,8729,9698 'stuart':8777 'studi':340 'studier':9486 'studier44':1469 'study35':4977 'subcellular':2625 'subcomplex':8622 'substitut':7374 'subunit':202,844,872,1170,1774,1823,4230,5768,6338,6363,6681,6999,7260,7912,7946,8159,8798,9103,9138,9370 'sucros':2888,3769,3773,3834,3838 'suction':4321 'suffici':525,7212,7665 'suit':447,4612 'suitabl':1030,1489,6278 'sulfat':9390 'sum':7254 'sunderhaus':9078,9097 'super':9084 'supercomplex':1672,1735,8041,8075,8385,8689,8811,8844,8875,9277 'supercomplexes21':494 'supernat':521,3179,3210,4039,5021 'supplement':6171 'suppli':1511,1535,4263 'support':8177 'supramolecular':353,415,1073,1611,5648,8033,9619 'sure':6807 'surfac':629,645,682 'suspend':1393,2876 'suspens':2924,3224,4059,9552 'synthas':1703,1779,1806,1827,6654,6674,8037,8626,8652,8722,8787,8917,8955,9337,9356,9571 'synthase20':498 'syring':4743 'system':996,1486,7564,8067,9060 'system52':7571 't4':9707 't7':9491 'tabl':2060,2081,2313,2645,2796,3096,3130,3336,3448,5500,5559,6043,6549,6551,6554,6600,7745,7748 'take':2489 'task':361 'tca':6142,6145 'tcrs':8754 'techniqu':368,1180,4182,4682,7694,8117,8129,8564,9418 'tein':6970 'teme':3410 'temperatur':2485,2512,2766,2775,4255,5199,6050 'temperatureso50':1350 'ten':4981 'tend':2153,6873 'tendenc':634 'test':6583 'thaliana':9473 'theodor':23 'theodor-stern-kai':22 'therebi':1103,1737,5875,7500 'therefor':683,4443,4479 'thick':5396,7553 'thinner':5425,5846 'third':191,4309,4657,5700,5773,6611 'three':4330,6118,6330,6586,6677,8794,9476 'three-dimension':6117,6329,9475 'throughout':2032 'thus':420 'tight':1909,3784 'tim17.23':9753 'time':4234,4331,4707,4938,4982,5095,5331,5775,6100,6404,7062 'timingb1':2992 'timingb2':2207 'timingb20':6121 'ting':7470 'tini':3117,3642,3961 'tion':325 'tip':4890 'tissu':87,275,3015,3749,3760,9056 'tob10':800,7888 'togeth':2945 'toler':2969 'tom':9151 'tom20':6395 'top':2498,5158,5174,5186,5441,5506,5866 'total':84,272,2007,2023,2528,2618,3394,4313,6750 'transfer':2480,5121,5188,5239,5269,6308,7450,7474 'translocas':6378,9183,9755 'transport':9052,9763 'trichloroacet':6140 'tricin':171,195,853,1354,1460,2666,2789,4546,4638,4661,4686,4695,4811,5104,5704,5762,5937,5963,5991,6499,6526,7630,7641,9387,9450 'tricine-sd':4545,5936,5962,7640 'tricine-sds-pag':170,194,852,1459,4637,4660,4685,4694,5703,5761,5990,6498,6525,7629 'tricine-sodium':9386 'tris':1223,1228,1430,2085,2098,7569 'tris-bas':1429 'tris-glycin':7568 'triton':463,1287,1424,3251,3279,3307,3478,3655,3715,5366,5572,5658,6660,6667,6705 'troubleshoot':3140,3335,4199,4231,4701,5275,6115,6542,6543,6553,7747 'truscott':9588 'tsklinikum':20 'tt':901,1882,2856,3584,4389,5299,6243,6940,7773,9009,9816 'tube':2417,3446,4494,4805,4894,4991,6291 'tube-gel':6290 'twice':1374,1381 'twice-crystal':1373,1380 'twirl':3115,3640,3959 'two':1686,2256,2375,2408,3325,3731,4125,4328,5320,5329,6469,7697,8281,8525,8595,9207,9637,9669 'two-dimension':5319,8280,8524,8594,9206,9636,9668 'type':863,2216,2223,2478,7590,7650,7706 'typic':4722 'u':9635,9718 'u.k':8335,9175,9695 'ub':893,1874,2848,3576,4381,5291,6235,6932,7765,9001,9808 'ubiquinol':9725 'ubiquinol-cytochrom':9724 'ultracentrifug':3050 'underlain':2351 'unfortun':6869 'uniform':2234,2247 'uniqu':8878,9178 'unit':1499,1506,4186 'universita':19,37 'unless':2025,3060 'unlik':7832 'unpublish':1071 'untim':3442 'updat':9643 'upon':652,2902,6876 'upper':6727 'upright':5460 'ur':890,908,914,1871,1889,1895,2845,2863,2869,3573,3591,3597,4378,4396,4402,5288,5306,5312,6232,6250,6256,6929,6947,6953,7762,7780,7786,8998,9016,9022,9805,9823,9829 'urea':1422 'usa':8323,8703,8739,9444 'use':71,93,121,253,378,412,428,856,933,999,1214,1220,1232,1310,1640,1723,1929,2031,2047,2088,2117,2174,2239,2291,2301,2315,2388,2470,2521,2526,2735,2761,2953,3132,3189,3237,3243,3250,3260,3265,3271,3278,3288,3293,3299,3306,3316,3435,3447,3552,3712,3724,3779,3800,3972,3998,4071,4097,4145,4177,4256,4335,4415,4537,4561,4570,4858,4884,4953,4965,4973,5222,5355,5532,5562,5677,5697,5731,5838,5942,5968,6010,6032,6051,6205,6294,6302,6658,6665,6701,6732,6823,6900,7158,7283,7299,7316,7340,7418,7479,7490,7507,7532,7550,7585,7621,7639,7684,7696,7716,7867,7905,7917,7955,7968,8023,9061,9674 'user':41 'usual':5416 'v':760,771,814,1513,3141,3679,3966,4266,4289,4792,4834,4864,5159,5204,5210,5472,5608,5611,5899,6071,6201,8333,8521,9173 'vacuolar':8411 'vacuum':5027,6207 'vahsen':8446 'vari':7335,7546 'variabl':7346 'variat':7342 'various':1178 ' 'vergani':8971 ' 'vertic':1464,1482,1498,5689,5897 ' 'vessel':1938,4758,4791,4801,4847 'vi':3150,3688,4023,4826,5179,5510,5996,6209 'via':8984 'vii':3202,3694,4029,4880,5187,5536,6045,6262 'viii':3211,3726,4040,4902,5221,5378,5561,6306 'visibl':5236 'vivo':9360 'vol.1':879,1861,2834,3563,4367,5278,6221,6919,7751,8988,9794 'vol/vol':6139 'voltag':4285,4830,5201,5206,5599,6068,6086,7544 'volum':2391,3134,3395,3509,4616,4726,4969,4985,5034,6790,6826 'von':8228,8256,8312,9383,9433,9721 'voo':9563 'w':904,905,1885,1886,2859,2860,3587,3588,4392,4393,5302,5303,6246,6247,6943,6944,7776,7777,8643,8680,8776,9012,9013,9462,9564,9664,9749,9819,9820 'w.a':8255,8523 'w.w':8746,9775 'wait':5462 'water':528,662,1278,1294,1316,2331,2495,3082,3387,4980,5243,5406,5449,5504,5515,5539,5828,5925,6017,7176,7367,7382,7424 'water-solubl':661,7175,7366,7381,7423 'way':5665 'wear':2044,3845 'wehmho':9606 'weight':3859,3864,3870,3877,3933,3938,3944,3951 'weiss':9663 'well':2143,2449,2547,3330,3736,4130,6744,7075,7133,7323 'well-character':7322 'werhahn':1854,9461 'western':5251,5272 'wet':3858,3863,3869,3876,3932,3937,3943,3950,5115 'whatman':1901,5133 'whenev':5817 'whether':6631 'whole':5671,8537 'wide':5387,8759 'width':2532,5880 'wilken':8713 'without':1311,1470,5029,6054 'wittig':9295 'wittig1':5 'wollscheid':8516 'work':734,1066,7357,8175 'would':8164 'wt/vol':1274,1290,1304,1385,1414 'www.natureprotocols.com':8213 'x':464,1288,1425,3252,3280,3308,3479,3656,3716,4120,5249,5367,5573,5659,6661,6668,6706 'xu':9044 'yarrowia':3532 'year':8016 'yeast':3103,3235,3458,3527,6652,8782,8817,9352,9592 'yet':4437 'york':9691 'z':9045 'zentrum':15 'zerbetto':8969 'zero':1681,3829 'zole/hcl':5060
\.


--
-- Data for Name: protocols; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.protocols (id, name, authors, description, added_by_id, my_module_id, team_id, protocol_type, parent_id, parent_updated_at, archived_by_id, archived_on, restored_by_id, restored_on, created_at, updated_at, published_on, nr_of_linked_children) FROM stdin;
1	\N	\N	\N	\N	1	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.267783	2019-05-30 12:40:53.267783	\N	0
2	\N	\N	\N	\N	2	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.287092	2019-05-30 12:40:53.287092	\N	0
3	\N	\N	\N	\N	3	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.303356	2019-05-30 12:40:53.303356	\N	0
4	\N	\N	\N	\N	4	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.312743	2019-05-30 12:40:53.312743	\N	0
5	\N	\N	\N	\N	5	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.322161	2019-05-30 12:40:53.322161	\N	0
6	\N	\N	\N	\N	6	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.331336	2019-05-30 12:40:53.331336	\N	0
7	\N	\N	\N	\N	7	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.340377	2019-05-30 12:40:53.340377	\N	0
8	\N	\N	\N	\N	8	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.349053	2019-05-30 12:40:53.349053	\N	0
9	\N	\N	\N	\N	9	1	0	\N	\N	\N	\N	\N	\N	2019-05-30 12:40:53.358163	2019-05-30 12:40:53.358163	\N	0
10	\N	\N	\N	\N	10	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.559303	2019-08-27 15:15:00.559303	\N	0
11	\N	\N	\N	\N	11	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.644243	2019-08-27 15:15:00.644243	\N	0
12	\N	\N	\N	\N	12	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.743148	2019-08-27 15:15:00.743148	\N	0
13	\N	\N	\N	\N	13	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.767723	2019-08-27 15:15:00.767723	\N	0
14	\N	\N	\N	\N	14	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.809475	2019-08-27 15:15:00.809475	\N	0
15	\N	\N	\N	\N	15	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.828045	2019-08-27 15:15:00.828045	\N	0
16	\N	\N	\N	\N	16	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.858102	2019-08-27 15:15:00.858102	\N	0
17	\N	\N	\N	\N	17	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.883671	2019-08-27 15:15:00.883671	\N	0
18	\N	\N	\N	\N	18	3	0	\N	\N	\N	\N	\N	\N	2019-08-27 15:15:00.959377	2019-08-27 15:15:00.959377	\N	0
19	\N	\N	\N	\N	19	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.645239	2019-08-28 07:30:58.645239	\N	0
20	\N	\N	\N	\N	20	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.703934	2019-08-28 07:30:58.703934	\N	0
21	\N	\N	\N	\N	21	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.735427	2019-08-28 07:30:58.735427	\N	0
22	\N	\N	\N	\N	22	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.763907	2019-08-28 07:30:58.763907	\N	0
23	\N	\N	\N	\N	23	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.814031	2019-08-28 07:30:58.814031	\N	0
24	\N	\N	\N	\N	24	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.846893	2019-08-28 07:30:58.846893	\N	0
25	\N	\N	\N	\N	25	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.882212	2019-08-28 07:30:58.882212	\N	0
26	\N	\N	\N	\N	26	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:58.931262	2019-08-28 07:30:58.931262	\N	0
27	\N	\N	\N	\N	27	2	0	\N	\N	\N	\N	\N	\N	2019-08-28 07:30:59.009523	2019-08-28 07:30:59.009523	\N	0
28	\N	\N	\N	\N	28	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:53.642076	2019-08-28 08:34:53.642076	\N	0
29	\N	\N	\N	\N	29	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:53.673829	2019-08-28 08:34:53.673829	\N	0
30	\N	\N	\N	\N	30	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:53.700966	2019-08-28 08:34:53.700966	\N	0
31	\N	\N	\N	\N	31	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:53.735502	2019-08-28 08:34:53.735502	\N	0
32	\N	\N	\N	\N	32	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:53.815851	2019-08-28 08:34:53.815851	\N	0
33	\N	\N	\N	\N	33	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:54.10857	2019-08-28 08:34:54.10857	\N	0
34	\N	\N	\N	\N	34	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:54.188419	2019-08-28 08:34:54.188419	\N	0
35	\N	\N	\N	\N	35	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:54.319644	2019-08-28 08:34:54.319644	\N	0
36	\N	\N	\N	\N	36	4	0	\N	\N	\N	\N	\N	\N	2019-08-28 08:34:54.399617	2019-08-28 08:34:54.399617	\N	0
37	\N	\N	\N	\N	37	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.405891	2019-08-29 12:51:23.405891	\N	0
38	\N	\N	\N	\N	38	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.531334	2019-08-29 12:51:23.531334	\N	0
39	\N	\N	\N	\N	39	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.565844	2019-08-29 12:51:23.565844	\N	0
40	\N	\N	\N	\N	40	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.677616	2019-08-29 12:51:23.677616	\N	0
41	\N	\N	\N	\N	41	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.717351	2019-08-29 12:51:23.717351	\N	0
42	\N	\N	\N	\N	42	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.772425	2019-08-29 12:51:23.772425	\N	0
43	\N	\N	\N	\N	43	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.813417	2019-08-29 12:51:23.813417	\N	0
44	\N	\N	\N	\N	44	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:23.837851	2019-08-29 12:51:23.837851	\N	0
45	\N	\N	\N	\N	45	5	0	\N	\N	\N	\N	\N	\N	2019-08-29 12:51:24.151298	2019-08-29 12:51:24.151298	\N	0
46	\N	\N	\N	\N	46	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.431756	2019-09-12 18:26:18.431756	\N	0
47	\N	\N	\N	\N	47	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.459453	2019-09-12 18:26:18.459453	\N	0
48	\N	\N	\N	\N	48	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.472959	2019-09-12 18:26:18.472959	\N	0
49	\N	\N	\N	\N	49	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.484912	2019-09-12 18:26:18.484912	\N	0
50	\N	\N	\N	\N	50	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.496252	2019-09-12 18:26:18.496252	\N	0
51	\N	\N	\N	\N	51	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.50772	2019-09-12 18:26:18.50772	\N	0
52	\N	\N	\N	\N	52	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.519786	2019-09-12 18:26:18.519786	\N	0
53	\N	\N	\N	\N	53	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.530863	2019-09-12 18:26:18.530863	\N	0
54	\N	\N	\N	\N	54	6	0	\N	\N	\N	\N	\N	\N	2019-09-12 18:26:18.543486	2019-09-12 18:26:18.543486	\N	0
55	\N	\N	\N	\N	55	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.622188	2019-11-13 13:57:41.622188	\N	0
56	\N	\N	\N	\N	56	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.663409	2019-11-13 13:57:41.663409	\N	0
57	\N	\N	\N	\N	57	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.69902	2019-11-13 13:57:41.69902	\N	0
58	\N	\N	\N	\N	58	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.717167	2019-11-13 13:57:41.717167	\N	0
59	\N	\N	\N	\N	59	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.735125	2019-11-13 13:57:41.735125	\N	0
60	\N	\N	\N	\N	60	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.768306	2019-11-13 13:57:41.768306	\N	0
61	\N	\N	\N	\N	61	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.785588	2019-11-13 13:57:41.785588	\N	0
62	\N	\N	\N	\N	62	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.809158	2019-11-13 13:57:41.809158	\N	0
63	\N	\N	\N	\N	63	8	0	\N	\N	\N	\N	\N	\N	2019-11-13 13:57:41.835352	2019-11-13 13:57:41.835352	\N	0
64	\N	\N	\N	\N	64	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:22.685961	2019-11-25 14:09:22.685961	\N	0
65	\N	\N	\N	\N	65	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:22.751661	2019-11-25 14:09:22.751661	\N	0
66	\N	\N	\N	\N	66	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:22.812742	2019-11-25 14:09:22.812742	\N	0
67	\N	\N	\N	\N	67	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:22.860983	2019-11-25 14:09:22.860983	\N	0
68	\N	\N	\N	\N	68	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:22.896312	2019-11-25 14:09:22.896312	\N	0
69	\N	\N	\N	\N	69	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:23.058757	2019-11-25 14:09:23.058757	\N	0
70	\N	\N	\N	\N	70	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:23.102239	2019-11-25 14:09:23.102239	\N	0
71	\N	\N	\N	\N	71	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:23.143015	2019-11-25 14:09:23.143015	\N	0
72	\N	\N	\N	\N	72	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:09:23.196094	2019-11-25 14:09:23.196094	\N	0
73	\N	\N	\N	\N	73	3	0	\N	\N	\N	\N	\N	\N	2019-11-25 14:12:02.984185	2019-11-25 14:12:02.984185	\N	0
75	\N	\N	\N	\N	75	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 17:01:05.096955	2019-11-25 17:01:05.096955	\N	0
76	\N	\N	\N	\N	76	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 17:01:05.109474	2019-11-25 17:01:05.109474	\N	0
77	\N	\N	\N	\N	77	7	0	\N	\N	\N	\N	\N	\N	2019-11-25 17:01:05.118811	2019-11-25 17:01:05.118811	\N	0
79	\N	\N	\N	\N	78	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.467568	2019-11-25 18:00:52.467568	\N	0
80	\N	\N	\N	\N	79	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.49257	2019-11-25 18:00:52.49257	\N	0
81	\N	\N	\N	\N	80	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.510911	2019-11-25 18:00:52.510911	\N	0
82	\N	\N	\N	\N	81	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.52886	2019-11-25 18:00:52.52886	\N	0
83	\N	\N	\N	\N	82	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.548734	2019-11-25 18:00:52.548734	\N	0
84	\N	\N	\N	\N	83	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.564723	2019-11-25 18:00:52.564723	\N	0
85	\N	\N	\N	\N	84	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.582263	2019-11-25 18:00:52.582263	\N	0
86	\N	\N	\N	\N	85	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.599096	2019-11-25 18:00:52.599096	\N	0
87	\N	\N	\N	\N	86	9	0	\N	\N	\N	\N	\N	\N	2019-11-25 18:00:52.617746	2019-11-25 18:00:52.617746	\N	0
88	\N	\N	\N	\N	87	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.00347	2020-05-12 16:13:02.00347	\N	0
89	\N	\N	\N	\N	88	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.061232	2020-05-12 16:13:02.061232	\N	0
90	\N	\N	\N	\N	89	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.11903	2020-05-12 16:13:02.11903	\N	0
91	\N	\N	\N	\N	90	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.135747	2020-05-12 16:13:02.135747	\N	0
92	\N	\N	\N	\N	91	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.164695	2020-05-12 16:13:02.164695	\N	0
93	\N	\N	\N	\N	92	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.231577	2020-05-12 16:13:02.231577	\N	0
94	\N	\N	\N	\N	93	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.247678	2020-05-12 16:13:02.247678	\N	0
95	\N	\N	\N	\N	94	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.262091	2020-05-12 16:13:02.262091	\N	0
96	\N	\N	\N	\N	95	10	0	\N	\N	\N	\N	\N	\N	2020-05-12 16:13:02.286207	2020-05-12 16:13:02.286207	\N	0
97	\N	\N	\N	\N	96	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.055399	2020-06-10 14:43:17.055399	\N	0
98	\N	\N	\N	\N	97	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.110561	2020-06-10 14:43:17.110561	\N	0
99	\N	\N	\N	\N	98	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.147922	2020-06-10 14:43:17.147922	\N	0
100	\N	\N	\N	\N	99	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.163143	2020-06-10 14:43:17.163143	\N	0
101	\N	\N	\N	\N	100	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.178817	2020-06-10 14:43:17.178817	\N	0
102	\N	\N	\N	\N	101	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.224097	2020-06-10 14:43:17.224097	\N	0
103	\N	\N	\N	\N	102	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.239651	2020-06-10 14:43:17.239651	\N	0
104	\N	\N	\N	\N	103	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.253654	2020-06-10 14:43:17.253654	\N	0
105	\N	\N	\N	\N	104	11	0	\N	\N	\N	\N	\N	\N	2020-06-10 14:43:17.269698	2020-06-10 14:43:17.269698	\N	0
107	\N	\N	<div>Wheeeeee</div>	7	105	5	1	106	2020-06-10 18:14:51.055053	\N	\N	\N	\N	2020-06-10 18:15:54.818214	2020-06-10 18:14:51.055053	\N	0
106	Test	\N	<div>Wheeeeee</div>	7	\N	5	3	\N	\N	\N	\N	\N	\N	2020-06-10 18:14:01.055947	2020-06-10 18:14:51.055053	2020-06-10 18:14:01.055947	1
109	\N	\N	\N	\N	106	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.128175	2020-10-07 19:20:25.128175	\N	0
110	\N	\N	\N	\N	107	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.188962	2020-10-07 19:20:25.188962	\N	0
111	\N	\N	\N	\N	108	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.226693	2020-10-07 19:20:25.226693	\N	0
112	\N	\N	\N	\N	109	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.244418	2020-10-07 19:20:25.244418	\N	0
113	\N	\N	\N	\N	110	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.261585	2020-10-07 19:20:25.261585	\N	0
114	\N	\N	\N	\N	111	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.343051	2020-10-07 19:20:25.343051	\N	0
115	\N	\N	\N	\N	112	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.360647	2020-10-07 19:20:25.360647	\N	0
116	\N	\N	\N	\N	113	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.376209	2020-10-07 19:20:25.376209	\N	0
117	\N	\N	\N	\N	114	13	0	\N	\N	\N	\N	\N	\N	2020-10-07 19:20:25.397433	2020-10-07 19:20:25.397433	\N	0
118	\N	\N	\N	\N	115	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.259032	2020-11-18 14:50:52.259032	\N	0
119	\N	\N	\N	\N	116	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.323235	2020-11-18 14:50:52.323235	\N	0
120	\N	\N	\N	\N	117	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.397488	2020-11-18 14:50:52.397488	\N	0
121	\N	\N	\N	\N	118	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.438265	2020-11-18 14:50:52.438265	\N	0
122	\N	\N	\N	\N	119	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.474067	2020-11-18 14:50:52.474067	\N	0
123	\N	\N	\N	\N	120	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.513791	2020-11-18 14:50:52.513791	\N	0
124	\N	\N	\N	\N	121	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.546771	2020-11-18 14:50:52.546771	\N	0
125	\N	\N	\N	\N	122	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.582599	2020-11-18 14:50:52.582599	\N	0
126	\N	\N	\N	\N	123	12	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:50:52.618422	2020-11-18 14:50:52.618422	\N	0
127	\N	\N	\N	\N	124	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.773473	2020-11-18 14:53:24.773473	\N	0
128	\N	\N	\N	\N	125	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.798697	2020-11-18 14:53:24.798697	\N	0
129	\N	\N	\N	\N	126	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.825654	2020-11-18 14:53:24.825654	\N	0
130	\N	\N	\N	\N	127	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.844593	2020-11-18 14:53:24.844593	\N	0
131	\N	\N	\N	\N	128	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.861021	2020-11-18 14:53:24.861021	\N	0
132	\N	\N	\N	\N	129	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.881397	2020-11-18 14:53:24.881397	\N	0
133	\N	\N	\N	\N	130	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.897171	2020-11-18 14:53:24.897171	\N	0
134	\N	\N	\N	\N	131	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.912813	2020-11-18 14:53:24.912813	\N	0
135	\N	\N	\N	\N	132	14	0	\N	\N	\N	\N	\N	\N	2020-11-18 14:53:24.928778	2020-11-18 14:53:24.928778	\N	0
137	\N	\N	\N	\N	134	7	0	\N	\N	\N	\N	\N	\N	2021-04-08 13:20:10.121134	2021-04-08 13:20:10.121134	\N	0
138	\N	\N	\N	\N	135	7	0	\N	\N	\N	\N	\N	\N	2021-04-08 13:20:10.127933	2021-04-08 13:20:10.127933	\N	0
139	\N	\N	\N	\N	136	7	0	\N	\N	\N	\N	\N	\N	2021-04-08 13:20:10.133811	2021-04-08 13:20:10.133811	\N	0
205	\N	\N	\N	\N	197	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.371208	2021-05-20 14:30:30.371208	\N	0
141	SOP 2338.08 - Chlorine in Water by LaMotte TRACER Pocket Tester	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>Chlorine in Water by LaMotte TRACER Pocket Tester</strong></div>\r\n<div><strong>Discusses the method used to analyze residual chlorine in reagent water.</strong></div>\r\n<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2021-04-08 14:44:48.020205	2021-04-08 14:46:30.067905	2021-04-08 14:44:48.020205	0
143	SOP 2338.15 - Conductivity	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>Conductivity</strong></div>\r\n<div><strong>Discusses the method used to analyze the conductivity of reagent water</strong></div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div>&nbsp;</div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2021-04-08 14:50:47.230398	2021-04-08 14:55:42.757185	2021-04-08 14:50:47.230398	0
140	Biological Hygiene Plan (BHP)	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>Biological Hygiene Plan (BHP)</strong></div>\r\n<div><strong>Discussed the Biohazardous Precautions in a Microbiological Laboratory</strong></div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2021-04-08 14:42:18.450155	2021-04-08 14:43:57.743885	2021-04-08 14:42:18.450155	0
78	Biological Quality Assurance Plan (BQAP)	\N	<div>&nbsp;</div>\r\n<div>Biological Quality Assurance Plan</div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div>&nbsp;</div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2019-11-25 17:39:31.371667	2021-04-08 14:40:13.26154	2019-11-25 17:39:31.371667	0
136	\N	\N	<div> </div>\r\n<div> </div>\r\n<div><strong>Steam Sterilization (Autoclave) and Decontamination</strong></div>\r\n<div><strong>Discusses the use of the Autoclave, sterilization times, QC and disposal.</strong></div>\r\n<div> </div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div> </div>	4	133	7	1	142	2021-04-08 14:54:44.458365	\N	\N	\N	\N	2021-04-08 13:20:10.110812	2021-04-08 14:54:44.458365	\N	0
206	\N	\N	\N	\N	198	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.381088	2021-05-20 14:30:30.381088	\N	0
207	\N	\N	\N	\N	199	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.391443	2021-05-20 14:30:30.391443	\N	0
208	TOC Solutions	\N	\N	24	\N	19	3	\N	\N	\N	\N	\N	\N	2021-05-20 14:34:33.206931	2021-05-20 14:41:37.26511	2021-05-20 14:34:33.206931	0
209	\N	\N	\N	\N	200	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.702233	2021-05-20 15:17:11.702233	\N	0
210	\N	\N	\N	\N	201	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.713802	2021-05-20 15:17:11.713802	\N	0
211	\N	\N	\N	\N	202	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.728012	2021-05-20 15:17:11.728012	\N	0
212	\N	\N	\N	\N	203	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.7389	2021-05-20 15:17:11.7389	\N	0
213	\N	\N	\N	\N	204	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.749837	2021-05-20 15:17:11.749837	\N	0
214	\N	\N	\N	\N	205	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.774229	2021-05-20 15:17:11.774229	\N	0
215	\N	\N	\N	\N	206	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.785093	2021-05-20 15:17:11.785093	\N	0
217	\N	\N	\N	\N	208	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.803838	2021-05-20 15:17:11.803838	\N	0
218	\N	\N	\N	\N	209	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:31:37.556462	2021-05-20 15:31:37.556462	\N	0
219	\N	\N	\N	\N	210	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:31:37.561641	2021-05-20 15:31:37.561641	\N	0
220	\N	\N	\N	\N	211	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:31:37.566229	2021-05-20 15:31:37.566229	\N	0
144	SOP 2338.09 - UV Light Intensity Checks	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>UV Light Intensity Checks</strong></div>\r\n<div><strong>Discusses the method used for verifying the light intensities of the Handheld UV lights and the BSC UV Lights.</strong></div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div>&nbsp;</div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2021-04-08 14:52:33.26689	2021-04-08 14:54:04.418698	2021-04-08 14:52:33.26689	0
108	SOP 2338.12 - The Determination of Total Coliforms and E. coli in Water by ColiLERT® for Drinking Water, Surface Water and Waste Water under Standard Methods for the Examination of Water and Wastewater, Standard Methods Online, 9223B-04	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>The Deermination of Total oliforms and <em>E.coli</em> in Water by ColiLert&reg; for Drinking Water, Surface Water and Waste Water under Standard Methods for the Examination of Water and Wastwater, Standard Methods Online, 9223B-04</strong></div>\r\n<div>&nbsp;</div>\r\n<div><strong>Discusses the method used to analyze Total Coliforms and <em>E. coli</em>. HPC method for reagent water QC is in the Attachments.</strong></div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div>&nbsp;</div>\r\n<div>&nbsp;</div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2020-06-12 11:32:47.718086	2021-04-08 14:58:03.076846	2020-06-12 11:32:47.718086	0
181	\N	\N	\N	\N	173	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:33.98417	2021-05-20 14:13:33.98417	\N	0
74	\N	\N	<div> </div>\r\n<div> </div>\r\n<div><strong>Steam Sterilization (Autoclave) and Decontamination</strong></div>\r\n<div><strong>Discusses the use of the Autoclave, sterilization times, QC and disposal.</strong></div>\r\n<div> </div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div> </div>	4	74	7	1	142	2021-04-08 14:54:44.458365	\N	\N	\N	\N	2019-11-25 17:01:05.072384	2021-04-08 14:54:44.458365	\N	0
142	SOP 2338.11 - Steam Sterilization (Autoclave) and Decontamination	\N	<div>&nbsp;</div>\r\n<div>&nbsp;</div>\r\n<div><strong>Steam Sterilization (Autoclave) and Decontamination</strong></div>\r\n<div><strong>Discusses the use of the Autoclave, sterilization times, QC and disposal.</strong></div>\r\n<div>&nbsp;</div>\r\n<div><span style="background-color: #ffffff;">http://r7atwork.r07.epa.gov/intranet/pdf/ENST_SOP.pdf</span></div>\r\n<div>&nbsp;</div>	4	\N	7	3	\N	\N	\N	\N	\N	\N	2021-04-08 14:49:12.220392	2021-04-08 14:54:44.458365	2021-04-08 14:49:12.220392	2
145	\N	\N	\N	\N	137	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.725472	2021-05-19 20:01:51.725472	\N	0
146	\N	\N	\N	\N	138	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.75954	2021-05-19 20:01:51.75954	\N	0
147	\N	\N	\N	\N	139	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.779592	2021-05-19 20:01:51.779592	\N	0
148	\N	\N	\N	\N	140	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.795958	2021-05-19 20:01:51.795958	\N	0
149	\N	\N	\N	\N	141	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.811162	2021-05-19 20:01:51.811162	\N	0
150	\N	\N	\N	\N	142	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.836101	2021-05-19 20:01:51.836101	\N	0
151	\N	\N	\N	\N	143	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.854684	2021-05-19 20:01:51.854684	\N	0
152	\N	\N	\N	\N	144	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.87145	2021-05-19 20:01:51.87145	\N	0
153	\N	\N	\N	\N	145	15	0	\N	\N	\N	\N	\N	\N	2021-05-19 20:01:51.88728	2021-05-19 20:01:51.88728	\N	0
154	\N	\N	\N	\N	146	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:26:02.272994	2021-05-20 10:26:02.272994	\N	0
155	\N	\N	\N	\N	147	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:26:02.279389	2021-05-20 10:26:02.279389	\N	0
156	\N	\N	\N	\N	148	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:31:30.163202	2021-05-20 10:31:30.163202	\N	0
157	\N	\N	\N	\N	149	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:31:30.168056	2021-05-20 10:31:30.168056	\N	0
158	\N	\N	\N	\N	150	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:31:30.172452	2021-05-20 10:31:30.172452	\N	0
159	\N	\N	\N	\N	151	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:31:30.178526	2021-05-20 10:31:30.178526	\N	0
160	\N	\N	\N	\N	152	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 10:31:30.186085	2021-05-20 10:31:30.186085	\N	0
161	\N	\N	\N	\N	153	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 13:07:49.523619	2021-05-20 13:07:49.523619	\N	0
162	\N	\N	\N	\N	154	1	0	\N	\N	\N	\N	\N	\N	2021-05-20 13:07:49.529434	2021-05-20 13:07:49.529434	\N	0
163	\N	\N	\N	\N	155	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.452518	2021-05-20 14:12:30.452518	\N	0
164	\N	\N	\N	\N	156	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.467582	2021-05-20 14:12:30.467582	\N	0
165	\N	\N	\N	\N	157	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.47913	2021-05-20 14:12:30.47913	\N	0
166	\N	\N	\N	\N	158	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.489492	2021-05-20 14:12:30.489492	\N	0
167	\N	\N	\N	\N	159	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.500645	2021-05-20 14:12:30.500645	\N	0
168	\N	\N	\N	\N	160	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.512063	2021-05-20 14:12:30.512063	\N	0
169	\N	\N	\N	\N	161	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.522657	2021-05-20 14:12:30.522657	\N	0
170	\N	\N	\N	\N	162	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.532312	2021-05-20 14:12:30.532312	\N	0
171	\N	\N	\N	\N	163	16	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:30.542684	2021-05-20 14:12:30.542684	\N	0
172	\N	\N	\N	\N	164	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.154072	2021-05-20 14:12:57.154072	\N	0
173	\N	\N	\N	\N	165	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.16373	2021-05-20 14:12:57.16373	\N	0
174	\N	\N	\N	\N	166	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.175665	2021-05-20 14:12:57.175665	\N	0
175	\N	\N	\N	\N	167	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.185806	2021-05-20 14:12:57.185806	\N	0
176	\N	\N	\N	\N	168	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.196297	2021-05-20 14:12:57.196297	\N	0
177	\N	\N	\N	\N	169	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.207612	2021-05-20 14:12:57.207612	\N	0
178	\N	\N	\N	\N	170	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.217337	2021-05-20 14:12:57.217337	\N	0
179	\N	\N	\N	\N	171	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.229219	2021-05-20 14:12:57.229219	\N	0
180	\N	\N	\N	\N	172	17	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:12:57.239968	2021-05-20 14:12:57.239968	\N	0
182	\N	\N	\N	\N	174	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:33.992167	2021-05-20 14:13:33.992167	\N	0
183	\N	\N	\N	\N	175	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.001866	2021-05-20 14:13:34.001866	\N	0
184	\N	\N	\N	\N	176	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.012322	2021-05-20 14:13:34.012322	\N	0
185	\N	\N	\N	\N	177	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.023681	2021-05-20 14:13:34.023681	\N	0
186	\N	\N	\N	\N	178	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.034718	2021-05-20 14:13:34.034718	\N	0
187	\N	\N	\N	\N	179	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.046603	2021-05-20 14:13:34.046603	\N	0
188	\N	\N	\N	\N	180	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.056108	2021-05-20 14:13:34.056108	\N	0
189	\N	\N	\N	\N	181	18	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:13:34.066021	2021-05-20 14:13:34.066021	\N	0
190	\N	\N	\N	\N	182	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.543917	2021-05-20 14:21:20.543917	\N	0
191	\N	\N	\N	\N	183	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.553995	2021-05-20 14:21:20.553995	\N	0
192	\N	\N	\N	\N	184	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.567707	2021-05-20 14:21:20.567707	\N	0
193	\N	\N	\N	\N	185	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.579873	2021-05-20 14:21:20.579873	\N	0
194	\N	\N	\N	\N	186	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.592656	2021-05-20 14:21:20.592656	\N	0
195	\N	\N	\N	\N	187	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.603952	2021-05-20 14:21:20.603952	\N	0
196	\N	\N	\N	\N	188	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.615769	2021-05-20 14:21:20.615769	\N	0
197	\N	\N	\N	\N	189	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.628481	2021-05-20 14:21:20.628481	\N	0
198	\N	\N	\N	\N	190	19	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:21:20.638171	2021-05-20 14:21:20.638171	\N	0
199	\N	\N	\N	\N	191	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.310018	2021-05-20 14:30:30.310018	\N	0
200	\N	\N	\N	\N	192	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.318166	2021-05-20 14:30:30.318166	\N	0
201	\N	\N	\N	\N	193	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.328758	2021-05-20 14:30:30.328758	\N	0
202	\N	\N	\N	\N	194	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.339102	2021-05-20 14:30:30.339102	\N	0
203	\N	\N	\N	\N	195	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.349785	2021-05-20 14:30:30.349785	\N	0
204	\N	\N	\N	\N	196	20	0	\N	\N	\N	\N	\N	\N	2021-05-20 14:30:30.360576	2021-05-20 14:30:30.360576	\N	0
216	\N	\N	\N	\N	207	21	0	\N	\N	\N	\N	\N	\N	2021-05-20 15:17:11.794488	2021-05-20 15:17:11.794488	\N	0
223	WIZ Field Calibration Procedures	\N	<div>This is the step by step protocol for the&nbsp; reagent refresh, and field calibration of the WIZ</div>	21	\N	16	3	\N	\N	\N	\N	\N	\N	2021-05-27 17:27:16.348187	2021-06-01 19:55:02.737723	2021-05-27 17:27:16.348187	0
233	\N	\N	\N	\N	221	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:01.815262	2021-06-02 16:10:01.815262	\N	0
234	\N	\N	\N	\N	222	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:01.882439	2021-06-02 16:10:01.882439	\N	0
235	\N	\N	\N	\N	223	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:01.946893	2021-06-02 16:10:01.946893	\N	0
236	\N	\N	\N	\N	224	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:02.096414	2021-06-02 16:10:02.096414	\N	0
237	\N	\N	\N	\N	225	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:02.210297	2021-06-02 16:10:02.210297	\N	0
238	\N	\N	\N	\N	226	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:02.461886	2021-06-02 16:10:02.461886	\N	0
239	\N	\N	\N	\N	227	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:02.511872	2021-06-02 16:10:02.511872	\N	0
240	\N	\N	\N	\N	228	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:02.54232	2021-06-02 16:10:02.54232	\N	0
241	\N	\N	\N	\N	229	23	0	\N	\N	\N	\N	\N	\N	2021-06-02 16:10:02.577486	2021-06-02 16:10:02.577486	\N	0
242	\N	\N	\N	\N	230	22	0	\N	\N	\N	\N	\N	\N	2021-06-10 16:07:24.175121	2021-06-10 16:07:24.175121	\N	0
243	\N	\N	\N	\N	231	22	0	\N	\N	\N	\N	\N	\N	2021-06-10 16:07:24.190089	2021-06-10 16:07:24.190089	\N	0
224	\N	\N	\N	\N	212	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:44.875476	2021-05-28 15:07:44.875476	\N	0
225	\N	\N	\N	\N	213	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:44.960626	2021-05-28 15:07:44.960626	\N	0
226	\N	\N	\N	\N	214	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.001446	2021-05-28 15:07:45.001446	\N	0
227	\N	\N	\N	\N	215	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.014538	2021-05-28 15:07:45.014538	\N	0
228	\N	\N	\N	\N	216	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.026229	2021-05-28 15:07:45.026229	\N	0
229	\N	\N	\N	\N	217	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.053986	2021-05-28 15:07:45.053986	\N	0
230	\N	\N	\N	\N	218	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.068232	2021-05-28 15:07:45.068232	\N	0
231	\N	\N	\N	\N	219	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.081672	2021-05-28 15:07:45.081672	\N	0
232	\N	\N	\N	\N	220	22	0	\N	\N	\N	\N	\N	\N	2021-05-28 15:07:45.107735	2021-05-28 15:07:45.107735	\N	0
245	\N	\N	\N	\N	233	22	0	\N	\N	\N	\N	\N	\N	2021-06-10 16:07:24.203853	2021-06-10 16:07:24.203853	\N	0
246	\N	\N	\N	\N	234	22	0	\N	\N	\N	\N	\N	\N	2021-06-10 16:07:24.208502	2021-06-10 16:07:24.208502	\N	0
244	\N	\N	<p><strong>Requirement:</strong></p>\r\n<table width="100%">\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p>&middot;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Plastic container 100ml</p>\r\n<p>&middot;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Vials 1 DRAM</p>\r\n<p>&middot;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Syringe 3 ml</p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p>&nbsp;</p>\r\n<table width="100%">\r\n<tbody>\r\n<tr>\r\n<td>\r\n<p>&middot;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Filters 0.2 um</p>\r\n<p>&middot;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; PhosVer 3 (HACH) powder pillow</p>\r\n<p>&middot;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Pipette 5ul, 1ml, 10ml</p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<p><strong>1 L PO</strong><sub>4</sub><strong> solution preparation </strong></p>\r\n<ol>\r\n<li>For of 1000 ppm\r\n<ul>\r\n<li>Weight 1.642 g of NaH<sub>2</sub>PO<sub>4</sub>2H<sub>2</sub>O: <strong><em>Measured</em></strong>:&nbsp; <u>_________________________</u></li>\r\n</ul>\r\n</li>\r\n<li>For of 4000 ppm\r\n<ul>\r\n<li>Weight 6.568 g of NaH<sub>2</sub>PO<sub>4</sub>2H<sub>2</sub>O: <strong><em>Measured</em></strong>:&nbsp; <u>_________________________</u></li>\r\n</ul>\r\n</li>\r\n</ol>\r\n<p>&nbsp;</p>\r\n<ul>\r\n<li>Mix at 200 RPM for 10 min with 1L of MiliQ Water in a volumetric flask</li>\r\n<li>Check concentration of PO<sub>4</sub> in model solution using UV spectrophotometer (HACH DR1900)\r\n<ol>\r\n<li>Setting &agrave; Program &agrave; &ldquo;490 P React PV&rdquo; (Measurement range* : 0 - 2.5 ppm)</li>\r\n<li>Dilute the model solution:\r\n<ul>\r\n<li><em>Example: Mix 20 uL of 1000ppm with 100 ml of water </em></li>\r\n</ul>\r\n</li>\r\n</ol>\r\n</li>\r\n</ul>\r\n<p><em>or&nbsp;&nbsp; &nbsp;Mix&nbsp; 5 uL of 4000ppm with 100 ml of water </em></p>\r\n<p>&nbsp;</p>\r\n<p><strong><em>Measured concentration: &nbsp;<u>&shy;&shy;&shy;_______________________________</u></em></strong></p>\r\n<p><strong>&nbsp;</strong><strong>&nbsp;</strong></p>\r\n<p><strong>&nbsp;</strong><strong>Adsorption isotherm test</strong></p>\r\n<ol>\r\n<li>Weight 1 g of sample and record the weight in Table 2</li>\r\n<li>Mix with 100 ml of model solution in a plastic container</li>\r\n<li>Place the sample on a mixer at medium (200 RPM) setting</li>\r\n<li>Collect 2ml of sample at time t in a vial for analysis</li>\r\n</ol>	\N	232	22	0	\N	\N	\N	\N	\N	\N	2021-06-10 16:07:24.196243	2021-06-10 18:49:53.009898	\N	0
221	WIZ Calibration	\N	\N	21	\N	16	3	\N	\N	\N	\N	\N	\N	2021-05-20 15:51:51.588009	2021-06-01 16:51:11.85111	2021-05-20 15:51:51.588009	0
247	WIZ Data collection and storage	\N	\N	21	\N	16	3	\N	\N	\N	\N	\N	\N	2021-06-24 17:21:36.567806	2021-06-24 17:21:36.567806	2021-06-24 17:21:36.567806	0
248	\N	\N	\N	\N	235	22	0	\N	\N	\N	\N	\N	\N	2021-07-06 15:50:12.380169	2021-07-06 15:50:12.380169	\N	0
249	\N	\N	\N	\N	236	22	0	\N	\N	\N	\N	\N	\N	2021-07-06 15:50:12.398964	2021-07-06 15:50:12.398964	\N	0
250	\N	\N	\N	\N	237	22	0	\N	\N	\N	\N	\N	\N	2021-07-06 15:50:12.406688	2021-07-06 15:50:12.406688	\N	0
251	\N	\N	\N	\N	238	22	0	\N	\N	\N	\N	\N	\N	2021-07-06 15:50:12.411309	2021-07-06 15:50:12.411309	\N	0
252	\N	\N	\N	\N	239	22	0	\N	\N	\N	\N	\N	\N	2021-07-12 18:41:37.10375	2021-07-12 18:41:37.10375	\N	0
253	\N	\N	\N	\N	240	22	0	\N	\N	\N	\N	\N	\N	2021-07-12 18:44:37.27416	2021-07-12 18:44:37.27416	\N	0
254	Field Sample Collection Report	\N	\N	21	\N	16	3	\N	\N	\N	\N	\N	\N	2021-07-21 19:12:50.810905	2021-07-21 19:12:50.810905	2021-07-21 19:12:50.810905	0
222	Reagent Preparation for WIZ	\N	<div>This is the protocol for formulating reagents to refresh the WIZ reagents, and for calibration in the field.</div>	21	\N	16	3	\N	\N	\N	\N	\N	\N	2021-05-26 15:16:55.10377	2021-07-28 19:42:12.325034	2021-05-26 15:16:55.10377	0
255	\N	\N	\N	\N	241	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.672594	2021-08-23 14:58:27.672594	\N	0
256	\N	\N	\N	\N	242	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.703778	2021-08-23 14:58:27.703778	\N	0
257	\N	\N	\N	\N	243	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.729853	2021-08-23 14:58:27.729853	\N	0
258	\N	\N	\N	\N	244	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.751112	2021-08-23 14:58:27.751112	\N	0
259	\N	\N	\N	\N	245	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.769329	2021-08-23 14:58:27.769329	\N	0
260	\N	\N	\N	\N	246	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.787949	2021-08-23 14:58:27.787949	\N	0
261	\N	\N	\N	\N	247	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.801153	2021-08-23 14:58:27.801153	\N	0
262	\N	\N	\N	\N	248	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.814694	2021-08-23 14:58:27.814694	\N	0
263	\N	\N	\N	\N	249	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 14:58:27.83045	2021-08-23 14:58:27.83045	\N	0
264	\N	\N	\N	\N	250	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.19669	2021-08-23 17:19:39.19669	\N	0
265	\N	\N	\N	\N	251	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.207027	2021-08-23 17:19:39.207027	\N	0
266	\N	\N	\N	\N	252	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.219228	2021-08-23 17:19:39.219228	\N	0
267	\N	\N	\N	\N	253	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.230788	2021-08-23 17:19:39.230788	\N	0
268	\N	\N	\N	\N	254	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.241171	2021-08-23 17:19:39.241171	\N	0
269	\N	\N	\N	\N	255	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.252933	2021-08-23 17:19:39.252933	\N	0
270	\N	\N	\N	\N	256	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.265312	2021-08-23 17:19:39.265312	\N	0
271	\N	\N	\N	\N	257	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.276023	2021-08-23 17:19:39.276023	\N	0
272	\N	\N	\N	\N	258	25	0	\N	\N	\N	\N	\N	\N	2021-08-23 17:19:39.287217	2021-08-23 17:19:39.287217	\N	0
273	\N	\N	\N	\N	259	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.79126	2021-08-23 19:46:47.79126	\N	0
274	\N	\N	\N	\N	260	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.796706	2021-08-23 19:46:47.796706	\N	0
275	\N	\N	\N	\N	261	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.801116	2021-08-23 19:46:47.801116	\N	0
276	\N	\N	\N	\N	262	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.805513	2021-08-23 19:46:47.805513	\N	0
277	\N	\N	\N	\N	263	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.809999	2021-08-23 19:46:47.809999	\N	0
278	\N	\N	\N	\N	264	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.81449	2021-08-23 19:46:47.81449	\N	0
279	\N	\N	\N	\N	265	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.819035	2021-08-23 19:46:47.819035	\N	0
280	\N	\N	\N	\N	266	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.823313	2021-08-23 19:46:47.823313	\N	0
281	\N	\N	\N	\N	267	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.827386	2021-08-23 19:46:47.827386	\N	0
282	\N	\N	\N	\N	268	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.831954	2021-08-23 19:46:47.831954	\N	0
283	\N	\N	\N	\N	269	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.836659	2021-08-23 19:46:47.836659	\N	0
284	\N	\N	\N	\N	270	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.840938	2021-08-23 19:46:47.840938	\N	0
285	\N	\N	\N	\N	271	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.845027	2021-08-23 19:46:47.845027	\N	0
286	\N	\N	\N	\N	272	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.849243	2021-08-23 19:46:47.849243	\N	0
287	\N	\N	\N	\N	273	24	0	\N	\N	\N	\N	\N	\N	2021-08-23 19:46:47.853502	2021-08-23 19:46:47.853502	\N	0
289	QC Creation	\N	<div>Creation of the method blank (MB), the lab fortified blank (LFB), and the Matrix Spike Sample (MS).</div>	30	\N	24	3	\N	\N	\N	\N	\N	\N	2021-08-24 13:48:38.718133	2021-08-24 17:46:58.459206	2021-08-24 13:48:38.718133	0
290	\N	\N	\N	\N	274	24	0	\N	\N	\N	\N	\N	\N	2021-08-25 16:08:53.521665	2021-08-25 16:08:53.521665	\N	0
288	Word and Excel document test	\N	\N	30	\N	24	3	\N	\N	\N	\N	\N	\N	2021-08-23 20:54:28.190703	2021-08-25 20:48:16.031602	2021-08-23 20:54:28.190703	0
291	\N	\N	\N	\N	275	24	0	\N	\N	\N	\N	\N	\N	2021-08-26 20:19:40.428108	2021-08-26 20:19:40.428108	\N	0
292	\N	\N	\N	\N	276	24	0	\N	\N	\N	\N	\N	\N	2021-08-26 20:19:40.433005	2021-08-26 20:19:40.433005	\N	0
293	Moisture Content Determination	\N	\N	30	\N	24	3	\N	\N	\N	\N	\N	\N	2021-08-26 20:19:59.222395	2021-08-26 20:38:33.036002	2021-08-26 20:19:59.222395	0
294	TEST Moisture Determination	\N	\N	30	\N	25	3	\N	\N	\N	\N	\N	\N	2021-08-26 20:54:22.228296	2021-08-26 20:55:32.936245	2021-08-26 20:54:22.228296	0
295	1214 IDOC	\N	\N	30	\N	24	3	\N	\N	\N	\N	\N	\N	2021-09-17 13:37:21.051444	2021-09-17 14:45:35.982954	2021-09-17 13:37:21.051444	0
\.


--
-- Data for Name: steps; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.steps (id, name, description, "position", completed, completed_on, user_id, created_at, updated_at, last_modified_by_id, protocol_id, assets_view_mode) FROM stdin;
805	TOC Stock Standards		0	f	\N	24	2021-05-20 14:41:37.12876	2021-05-20 14:41:37.218389	24	208	0
848	Reagent Preparation Day 1	<div>Steps for Reagent Preparation on Day 1</div>	2	f	\N	21	2021-05-27 14:51:11.853523	2021-05-27 16:52:20.079643	21	222	0
441	Sampling	<div>Make sure samples meet sample criteria</div>	0	f	\N	7	2020-06-10 18:14:51.024263	2020-06-10 18:14:51.024263	7	106	0
1015	MB Creation	<div>Labeled one Falcon tube as "MB" and 100uL of surrogate solution was added (edited)</div>\n<div>&nbsp;</div>	0	f	\N	30	2021-08-24 14:20:20.654218	2021-08-24 17:46:58.436307	30	289	0
1019	Analytical balance calibration.		0	f	\N	30	2021-08-26 20:26:39.447032	2021-08-26 20:26:39.453056	30	293	0
1022	LFB Creation	<div>Label 4 15 mL falcon tubes as:</div>\n<ul>\n<li>LFB IDOC 1</li>\n<li>LFB IDOC 2</li>\n<li>LFB IDOC 3</li>\n<li>LFB IDOC 4</li>\n</ul>	1	f	\N	30	2021-09-17 14:45:10.499396	2021-09-17 14:45:10.499396	30	295	0
557	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	17	2020-11-14 19:33:51.936498	2020-11-18 14:53:26.116062	17	133	0
1016	LFB Creation	<div>Labeled one Falcon tube as "LFB" and placed 100uL of surrogate solution inside.</div>	1	f	\N	30	2021-08-24 14:21:04.729013	2021-08-24 14:21:04.729013	30	289	0
1017	MS Creation	<div>Labeled one Falcon tube as "MS" and placed 100uL of surrogate solution inside.</div>	2	f	\N	30	2021-08-24 14:21:37.612009	2021-08-24 14:21:37.612009	30	289	0
1021	MB Creation	<div>Label 4 15 mL falcon tubes as:</div>\n<ul>\n<li>MB IDOC 1</li>\n<li>MB IDOC 2</li>\n<li>MB IDOC 3</li>\n<li>MB IDOC 4</li>\n</ul>	0	f	\N	30	2021-09-17 14:44:02.377003	2021-09-17 14:44:20.232281	30	295	0
1018	Calculations Word Doc and Sonde SOP Test		0	f	\N	30	2021-08-25 20:48:15.763855	2021-08-25 20:48:15.961449	30	288	0
1023	Water addition	<div>Add 10mL of reagent water to each falcon tube using a calibrated pipette</div>	2	f	\N	30	2021-09-17 14:45:35.970037	2021-09-17 14:45:35.970037	30	295	0
847	Chemicals used for reagent preparation		1	f	\N	21	2021-05-27 14:50:14.152843	2021-05-27 14:50:14.281933	21	222	0
846	Pre-preparation:  Glassware and ancillary material	<div>&nbsp;</div>\n<div>Prepare glassware, chemicals, water, and ice bath.</div>	0	f	\N	21	2021-05-26 16:37:18.805255	2021-05-26 16:37:18.937112	21	222	0
849	Field Day Reagent Preparation	<div>Reagents to prepare on the day that the service will be conducted.&nbsp; These are typically reagents with short shelf lives in diluted form.&nbsp; Of the standards, the NO2 has a short shelf life when diluted, the internal standard which does not contain NO2 is also prepared on this day to minimize the number of times that the standards are accessed.</div>	3	f	\N	21	2021-05-27 17:00:12.275031	2021-05-27 19:55:55.172011	21	222	0
1020	Analytical Balance Calibration		0	f	\N	30	2021-08-26 20:55:14.76726	2021-08-26 20:55:14.774315	30	294	0
621	Before you start	Follow the guidelines bellow.	0	t	2021-05-15 07:45:09.165699	21	2021-05-15 03:02:37.214025	2021-05-20 14:34:27.94485	21	166	0
850	Prepare reagents and equipment		0	f	\N	21	2021-05-27 17:54:47.131092	2021-05-27 19:19:27.303688	21	223	0
891	Travel to site.	<div>Travel to WIZ location, currently the intake structure of the Clermont County, Bob McEwen Water Treatment Plant.</div>	0	f	\N	21	2021-06-01 16:23:59.34412	2021-06-01 16:23:59.34412	21	221	0
893	Travel to WIZ location	<div>Entry at site requires opening locks at entrance to the equestrian area driveway, opening the lock and gate at the exit of the horsemanship driveway, and unlocking the gate to the causeway.&nbsp; Then driving down the causeway (usually in reverse, to permit easier access to the trunk.&nbsp; Then unlock the door to the intake structure.&nbsp; Use the toggle to turn the lock to the open position, and secure the keys.&nbsp; The trapdoor to the basement of the structure is located on the far side of the structure.&nbsp; There is a metal handle, that serves as both key, and handle on the light switch on the wall near the trapdoor.&nbsp; Turn on this light switch, it operates the lights in the basement, and they require some time to warm up.&nbsp; Use the handle to open the door.&nbsp; Make sure the door is locked in the open position and return the handle to the light switch.&nbsp; There is an extension on the ladder that should be raised and locked in the up position.&nbsp; This is for stability when climbing up or down the ladder.&nbsp; On the rack near the crane in the middle of the room is a rope with a snap swivel, on an orange plastic H-frame rope winder.&nbsp; Rig the cooler containing reagent bottles, and any other gear to be lowered to the basement to this line.&nbsp; Lower the line to the basement.&nbsp; Ensure that any persons are clear of the hatch and that the material being lowered will not fall on them if dropped.</div>	1	f	\N	21	2021-06-01 17:01:34.122022	2021-06-01 17:13:31.901293	21	223	0
892	Site Entry	<div>Entry at site requires opening locks at entrance to the equestrian area driveway, opening the lock and gate at the exit of the horsemanship driveway, and unlocking the gate to the causeway.&nbsp; Then driving down the causeway (usually in reverse, to permit easier access to the trunk.&nbsp; Then unlock the door to the intake structure.&nbsp; Use the toggle to turn the lock to the open position, and secure the keys.&nbsp; The trapdoor to the basement of the structure is located on the far side of the structure.&nbsp; There is a metal handle, that serves as both key, and handle on the light switch on the wall near the trapdoor.&nbsp; Turn on this light switch, it operates the lights in the basement, and they require some time to warm up.&nbsp; Use the handle to open the door.&nbsp; Make sure the door is locked in the open position and return the handle to the light switch.&nbsp; There is an extension on the ladder that should be raised and locked in the up position.&nbsp; This is for stability when climbing up or down the ladder.&nbsp; On the rack near the crane in the middle of the room is a rope with a snap swivel, on an orange plastic H-frame rope winder.&nbsp; Rig the cooler containing reagent bottles, and any other gear to be lowered to the basement to this line.&nbsp; Lower the line to the basement.&nbsp; Ensure that any persons are clear of the hatch and that the material being lowered will not fall on them if dropped.</div>\n<div>&nbsp;</div>	1	f	\N	21	2021-06-01 16:51:11.709582	2021-06-01 16:51:11.740805	21	221	0
894	WIZ calibration and reagent replacement	<div>This is a step by step instruction for refreshing the WIZ reagents and re-calibrating the WIZ.</div>	2	f	\N	21	2021-06-01 17:17:54.244243	2021-06-01 17:17:54.25786	21	223	0
767	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2021-05-19 10:59:35.842755	25	2021-05-19 09:18:23.13606	2021-05-20 14:30:30.657462	25	200	0
793	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	t	2021-05-18 21:21:19.298144	25	2021-05-18 12:39:44.65071	2021-05-20 14:30:30.873279	25	203	0
983	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	t	2021-08-22 01:42:52.7322	31	2021-08-21 19:20:04.349639	2021-08-23 17:19:39.661849	31	266	0
1	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	t	2019-05-27 02:55:33.739304	1	2019-05-26 19:43:45.465908	2019-05-30 12:40:53.824483	1	1	0
2	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FBT/3~rep_item~3]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FBT/5~rep_item~5]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FBT/3~rep_item~3]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FBT/5~rep_item~5]</span>.\n          </p>\n        </body>\n      </html>	0	t	2019-05-30 07:04:47.054088	1	2019-05-30 00:27:37.333087	2019-05-30 12:41:26.579044	1	2	0
3	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2019-05-30 08:14:20.839167	1	2019-05-30 00:42:46.445392	2019-05-30 12:40:53.855554	1	2	0
4	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	t	2019-05-30 05:01:48.33197	1	2019-05-30 00:13:13.709189	2019-05-30 12:40:53.860356	1	2	0
5	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	t	2019-05-26 19:18:42.839697	1	2019-05-26 10:13:24.810063	2019-05-30 12:40:53.909843	1	3	0
6	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2019-05-26 14:00:16.812552	1	2019-05-26 11:16:30.528983	2019-05-30 12:40:53.912719	1	3	0
7	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2019-05-26 13:39:54.7135	1	2019-05-26 08:26:51.899038	2019-05-30 12:40:53.918562	1	3	0
8	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	1	2019-05-26 11:51:06.232995	2019-05-30 12:40:53.920217	1	3	0
9	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	1	2019-05-26 09:20:38.330347	2019-05-30 12:40:53.923033	1	3	0
10	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	1	2019-05-26 07:12:49.683563	2019-05-30 12:40:53.925791	1	3	0
11	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	1	2019-05-26 08:07:33.184675	2019-05-30 12:40:53.928591	1	3	0
12	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	1	2019-05-26 08:55:28.672497	2019-05-30 12:40:53.931214	1	3	0
13	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2019-05-26 14:49:10.175583	1	2019-05-26 07:11:21.619824	2019-05-30 12:40:53.93706	1	3	0
14	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	1	2019-05-26 07:25:52.804916	2019-05-30 12:40:53.9387	1	3	0
15	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	1	2019-05-26 11:35:35.490153	2019-05-30 12:40:53.941261	1	3	0
16	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	1	2019-05-26 07:51:42.4495	2019-05-30 12:40:53.94392	1	3	0
17	Before you start	Follow the guidelines bellow.	0	f	\N	1	2019-05-29 23:53:35.093956	2019-05-30 12:41:21.723824	1	4	0
18	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	1	2019-05-29 21:47:30.227858	2019-05-30 12:40:53.980899	1	4	0
19	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	1	2019-05-30 01:13:13.925248	2019-05-30 12:40:53.983487	1	4	0
20	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	1	2019-05-29 23:08:59.070287	2019-05-30 12:40:54.058686	1	4	0
21	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	1	2019-05-30 01:10:05.034446	2019-05-30 12:40:53.989069	1	4	0
22	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	1	2019-05-29 23:19:33.533168	2019-05-30 12:40:53.991702	1	4	0
23	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	1	2019-05-29 21:41:26.738588	2019-05-30 12:40:53.994269	1	4	0
24	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2019-05-30 06:06:31.166669	1	2019-05-29 23:02:53.613482	2019-05-30 12:40:53.999806	1	4	0
25	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2019-05-30 08:59:05.480541	1	2019-05-30 00:41:54.038923	2019-05-30 12:40:54.004101	1	4	0
26	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	1	2019-05-30 00:00:51.707508	2019-05-30 12:40:54.005674	1	4	0
27	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	1	2019-05-30 00:52:17.664434	2019-05-30 12:40:54.008415	1	4	0
28	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	1	2019-05-25 03:50:55.482326	2019-05-30 12:40:54.080932	1	5	0
29	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	1	2019-05-25 05:35:14.27766	2019-05-30 12:40:54.091558	1	5	0
30	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	1	2019-05-25 07:08:59.535702	2019-05-30 12:40:54.086029	1	5	0
31	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	1	2019-05-28 13:28:52.642803	2019-05-30 12:41:22.104892	1	6	0
32	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2019-05-28 19:41:34.878817	1	2019-05-28 11:01:06.30918	2019-05-30 12:41:28.237554	1	6	0
33	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2019-05-28 19:46:36.972913	1	2019-05-28 11:02:00.071439	2019-05-30 12:41:28.741484	1	6	0
34	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	1	2019-05-27 21:55:35.951659	2019-05-30 12:41:30.169798	1	7	0
35	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	1	2019-05-27 18:58:03.415289	2019-05-30 12:40:54.244645	1	7	0
36	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2019-05-28 03:12:41.271048	1	2019-05-27 18:50:12.49557	2019-05-30 12:40:54.214517	1	7	0
37	Eliminate results that have positive NTCs	And repeat procedure.	3	t	2019-05-28 05:09:18.465186	1	2019-05-27 19:26:10.391107	2019-05-30 12:40:54.219015	1	7	0
38	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	1	2019-05-27 21:48:54.515778	2019-05-30 12:41:24.180704	1	7	0
39	Excel results	Write results in excel file.	5	t	2019-05-27 22:50:03.492693	1	2019-05-27 21:32:26.949075	2019-05-30 12:41:24.302726	1	7	0
40	Template for ddCq analysis	Sample information here.	0	f	\N	1	2019-05-27 19:48:29.566314	2019-05-30 12:41:30.704941	1	8	0
41	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	4	2019-08-27 07:11:04.684446	2019-08-27 15:15:02.74594	4	10	0
61	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	4	2019-08-26 00:01:59.762837	2019-08-27 15:15:03.334978	4	13	0
674	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	22	2021-05-20 12:57:37.858479	2021-05-20 14:12:57.724889	22	176	0
42	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#PCH/3~rep_item~8]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#PCH/5~rep_item~A]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#PCH/3~rep_item~8]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#PCH/5~rep_item~A]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	4	2019-08-26 06:43:10.481606	2019-08-27 15:15:39.748252	4	11	0
43	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2019-08-26 08:26:43.613186	4	2019-08-26 07:11:36.017349	2019-08-27 15:15:02.9847	4	11	0
44	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	4	2019-08-26 07:29:10.229367	2019-08-27 15:15:03.071219	4	11	0
45	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	4	2019-08-25 13:43:49.737133	2019-08-27 15:15:03.213728	4	12	0
46	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	4	2019-08-25 14:59:16.876849	2019-08-27 15:15:03.217024	4	12	0
47	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2019-08-25 19:10:41.820761	4	2019-08-25 12:52:56.048969	2019-08-27 15:15:03.224019	4	12	0
48	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	4	2019-08-25 15:26:15.001314	2019-08-27 15:15:03.225905	4	12	0
49	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	4	2019-08-25 16:47:31.372738	2019-08-27 15:15:03.229066	4	12	0
50	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2019-08-25 17:33:20.676906	4	2019-08-25 14:31:39.812661	2019-08-27 15:15:03.237608	4	12	0
51	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	4	2019-08-25 16:52:01.706626	2019-08-27 15:15:03.240083	4	12	0
52	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2019-08-25 22:42:50.984579	4	2019-08-25 16:45:14.680717	2019-08-27 15:15:03.24953	4	12	0
53	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2019-08-25 21:03:39.874176	4	2019-08-25 16:13:32.380115	2019-08-27 15:15:03.256113	4	12	0
54	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	4	2019-08-25 14:05:43.433557	2019-08-27 15:15:03.258004	4	12	0
55	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	4	2019-08-25 16:59:27.446867	2019-08-27 15:15:03.265887	4	12	0
56	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	4	2019-08-25 13:45:37.694414	2019-08-27 15:15:03.269255	4	12	0
57	Before you start	Follow the guidelines bellow.	0	t	2019-08-25 21:35:41.13091	4	2019-08-25 20:21:16.772439	2019-08-27 15:15:41.404731	4	13	0
854	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	27	2021-05-22 16:45:18.690963	2021-05-28 15:07:53.121357	27	225	0
855	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	27	2021-05-23 03:14:28.434435	2021-05-28 15:07:53.194795	27	226	0
58	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2019-08-26 02:42:50.319442	4	2019-08-25 22:12:47.815735	2019-08-27 15:15:03.324278	4	13	0
59	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	4	2019-08-25 21:34:46.89372	2019-08-27 15:15:03.326332	4	13	0
60	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2019-08-25 23:32:40.089868	4	2019-08-25 20:11:37.218972	2019-08-27 15:15:03.333234	4	13	0
62	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	4	2019-08-25 22:51:25.754205	2019-08-27 15:15:03.338037	4	13	0
63	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	4	2019-08-25 20:44:13.517443	2019-08-27 15:15:03.341293	4	13	0
64	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	4	2019-08-25 22:00:56.175135	2019-08-27 15:15:03.490578	4	13	0
65	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2019-08-26 03:27:26.794308	4	2019-08-25 23:01:45.822086	2019-08-27 15:15:03.352565	4	13	0
66	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	t	2019-08-26 06:50:46.594492	4	2019-08-25 23:22:49.587609	2019-08-27 15:15:03.358212	4	13	0
67	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	4	2019-08-25 21:12:54.606364	2019-08-27 15:15:03.360056	4	13	0
68	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	4	2019-08-21 16:58:49.453803	2019-08-27 15:15:03.513579	4	14	0
69	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	t	2019-08-22 02:00:42.053214	4	2019-08-21 16:53:09.433937	2019-08-27 15:15:03.521102	4	14	0
70	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	4	2019-08-21 19:57:59.609041	2019-08-27 15:15:03.522998	4	14	0
71	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	4	2019-08-22 11:30:09.892501	2019-08-27 15:15:41.853036	4	15	0
72	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	4	2019-08-22 14:47:59.85702	2019-08-27 15:15:42.373593	4	15	0
73	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	4	2019-08-22 10:08:03.479841	2019-08-27 15:15:42.727809	4	15	0
856	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	27	2021-05-23 02:41:22.333505	2021-05-28 15:07:53.198809	27	226	0
74	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	4	2019-08-22 01:34:00.523577	2019-08-27 15:15:44.739177	4	16	0
75	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	4	2019-08-22 01:30:21.84151	2019-08-27 15:15:03.774915	4	16	0
76	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	4	2019-08-22 02:42:11.676595	2019-08-27 15:15:03.730257	4	16	0
77	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	4	2019-08-22 00:26:52.159184	2019-08-27 15:15:03.733369	4	16	0
78	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	4	2019-08-22 02:08:09.459089	2019-08-27 15:15:44.850562	4	16	0
79	Excel results	Write results in excel file.	5	f	\N	4	2019-08-22 04:34:02.948306	2019-08-27 15:15:45.146096	4	16	0
80	Template for ddCq analysis	Sample information here.	0	f	\N	4	2019-08-23 06:08:37.650281	2019-08-27 15:15:45.490123	4	17	0
81	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	t	2019-08-25 15:13:23.696291	1	2019-08-25 07:11:43.89299	2019-08-28 07:31:00.461748	1	19	0
161	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	7	2019-08-26 17:48:04.217615	2019-08-29 12:51:26.346985	7	37	0
843	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	26	2021-05-15 00:11:18.434493	2021-05-20 15:37:54.551337	26	215	0
82	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#EZP/3~rep_item~D]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#EZP/5~rep_item~F]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#EZP/3~rep_item~D]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#EZP/5~rep_item~F]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	1	2019-08-25 09:07:00.942003	2019-08-28 07:31:02.381731	1	20	0
83	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	1	2019-08-25 07:44:38.244628	2019-08-28 07:31:00.520574	1	20	0
84	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	t	2019-08-25 15:25:33.691534	1	2019-08-25 08:09:06.537857	2019-08-28 07:31:00.598048	1	20	0
85	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	t	2019-08-24 12:16:47.58582	1	2019-08-24 06:38:46.069315	2019-08-28 07:31:00.752311	1	21	0
86	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	1	2019-08-24 09:14:47.170917	2019-08-28 07:31:00.756658	1	21	0
87	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	1	2019-08-24 09:05:13.798921	2019-08-28 07:31:00.760908	1	21	0
88	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	1	2019-08-24 06:41:21.107754	2019-08-28 07:31:00.765354	1	21	0
89	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	1	2019-08-24 09:13:33.838075	2019-08-28 07:31:00.769358	1	21	0
90	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2019-08-24 12:17:42.604466	1	2019-08-24 04:28:44.402549	2019-08-28 07:31:00.78955	1	21	0
91	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	1	2019-08-24 07:38:54.377623	2019-08-28 07:31:00.793913	1	21	0
92	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	1	2019-08-24 06:46:29.389218	2019-08-28 07:31:00.797996	1	21	0
93	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2019-08-24 13:01:25.467198	1	2019-08-24 04:47:36.150117	2019-08-28 07:31:00.80686	1	21	0
94	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	1	2019-08-24 04:56:37.121839	2019-08-28 07:31:00.810863	1	21	0
95	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	1	2019-08-24 08:53:32.86207	2019-08-28 07:31:00.815127	1	21	0
96	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	1	2019-08-24 04:43:40.474388	2019-08-28 07:31:00.819396	1	21	0
97	Before you start	Follow the guidelines bellow.	0	t	2019-08-23 00:38:15.193457	1	2019-08-22 14:51:31.707073	2019-08-28 07:31:03.529524	1	22	0
857	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	27	2021-05-23 06:10:10.260842	2021-05-28 15:07:53.201081	27	226	0
98	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	1	2019-08-22 15:34:38.519841	2019-08-28 07:31:00.879772	1	22	0
99	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	1	2019-08-22 17:42:30.828616	2019-08-28 07:31:00.884066	1	22	0
100	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	1	2019-08-22 14:52:26.489724	2019-08-28 07:31:01.172628	1	22	0
101	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	1	2019-08-22 16:59:05.544193	2019-08-28 07:31:00.892821	1	22	0
102	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	1	2019-08-22 17:51:41.506811	2019-08-28 07:31:00.905937	1	22	0
103	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2019-08-22 23:54:04.481501	1	2019-08-22 15:06:03.725332	2019-08-28 07:31:00.92218	1	22	0
104	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2019-08-22 18:50:28.140947	1	2019-08-22 14:23:05.011971	2019-08-28 07:31:01.184513	1	22	0
105	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2019-08-22 18:14:16.315444	1	2019-08-22 17:41:46.118062	2019-08-28 07:31:00.945374	1	22	0
106	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	1	2019-08-22 16:36:04.699585	2019-08-28 07:31:00.951984	1	22	0
107	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	1	2019-08-22 17:51:26.209749	2019-08-28 07:31:00.96492	1	22	0
108	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	1	2019-08-24 06:38:46.782178	2019-08-28 07:31:01.209396	1	23	0
109	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	1	2019-08-24 06:46:35.811141	2019-08-28 07:31:01.225443	1	23	0
110	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	1	2019-08-24 04:29:20.843813	2019-08-28 07:31:01.217423	1	23	0
111	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	1	2019-08-23 03:32:03.796318	2019-08-28 07:31:04.046276	1	24	0
112	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	1	2019-08-23 02:56:19.418896	2019-08-28 07:31:04.487462	1	24	0
113	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	1	2019-08-23 02:31:41.611578	2019-08-28 07:31:04.800687	1	24	0
114	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	1	2019-08-26 09:34:36.572876	2019-08-28 07:31:06.578392	1	25	0
115	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	1	2019-08-26 13:18:01.200033	2019-08-28 07:31:01.526647	1	25	0
116	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	1	2019-08-26 09:46:49.329841	2019-08-28 07:31:01.456684	1	25	0
117	Eliminate results that have positive NTCs	And repeat procedure.	3	t	2019-08-26 13:18:03.237949	1	2019-08-26 13:01:15.194311	2019-08-28 07:31:01.465805	1	25	0
118	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	1	2019-08-26 09:59:39.9601	2019-08-28 07:31:06.713079	1	25	0
119	Excel results	Write results in excel file.	5	t	2019-08-26 23:00:47.674892	1	2019-08-26 13:21:45.40109	2019-08-28 07:31:06.865481	1	25	0
120	Template for ddCq analysis	Sample information here.	0	f	\N	1	2019-08-23 14:40:16.971766	2019-08-28 07:31:07.342682	1	26	0
121	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	6	2019-08-23 01:20:10.812711	2019-08-28 08:34:56.209068	6	28	0
844	Excel results	Write results in excel file.	5	f	\N	26	2021-05-14 22:44:15.759916	2021-05-20 15:17:41.7277	26	215	0
122	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RLY/3~rep_item~I]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RLY/5~rep_item~K]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RLY/3~rep_item~I]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RLY/5~rep_item~K]</span>.\n          </p>\n        </body>\n      </html>	0	t	2019-08-25 08:46:52.481242	6	2019-08-25 01:07:53.085244	2019-08-28 08:35:48.388399	6	29	0
123	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2019-08-25 06:10:11.059235	6	2019-08-25 01:43:53.925011	2019-08-28 08:34:56.274178	6	29	0
124	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	6	2019-08-24 23:44:18.677132	2019-08-28 08:34:56.315222	6	29	0
125	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	6	2019-08-25 14:55:01.500795	2019-08-28 08:34:56.423059	6	30	0
126	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	6	2019-08-25 16:23:23.219723	2019-08-28 08:34:56.426006	6	30	0
127	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	6	2019-08-25 16:05:23.748305	2019-08-28 08:34:56.430201	6	30	0
128	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	6	2019-08-25 14:08:21.664884	2019-08-28 08:34:56.433208	6	30	0
129	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	6	2019-08-25 15:05:50.3801	2019-08-28 08:34:56.438221	6	30	0
130	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	6	2019-08-25 14:26:46.808644	2019-08-28 08:34:56.442295	6	30	0
131	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2019-08-25 19:18:25.694068	6	2019-08-25 14:09:35.986456	2019-08-28 08:34:56.454641	6	30	0
132	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2019-08-25 14:52:37.938831	6	2019-08-25 14:46:15.586011	2019-08-28 08:34:56.460133	6	30	0
133	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	6	2019-08-25 12:42:41.014657	2019-08-28 08:34:56.461896	6	30	0
134	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	6	2019-08-25 14:23:21.443595	2019-08-28 08:34:56.464811	6	30	0
135	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	6	2019-08-25 13:14:47.308801	2019-08-28 08:34:56.478429	6	30	0
136	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	t	2019-08-25 19:48:48.588441	6	2019-08-25 15:08:44.18109	2019-08-28 08:34:56.488807	6	30	0
137	Before you start	Follow the guidelines bellow.	0	f	\N	6	2019-08-27 07:30:49.579236	2019-08-28 08:35:49.727436	6	31	0
858	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	27	2021-05-23 06:07:55.168611	2021-05-28 15:07:53.251352	27	226	0
138	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	6	2019-08-27 08:57:05.589804	2019-08-28 08:34:56.545149	6	31	0
139	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	6	2019-08-27 06:25:20.160092	2019-08-28 08:34:56.551151	6	31	0
140	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2019-08-27 10:32:20.759144	6	2019-08-27 06:09:02.644461	2019-08-28 08:34:56.643853	6	31	0
141	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	6	2019-08-27 07:54:54.188064	2019-08-28 08:34:56.557263	6	31	0
142	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	6	2019-08-27 09:31:04.127467	2019-08-28 08:34:56.561609	6	31	0
143	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	6	2019-08-27 05:54:50.525679	2019-08-28 08:34:56.565684	6	31	0
144	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2019-08-27 15:00:12.893687	6	2019-08-27 09:18:35.948374	2019-08-28 08:34:56.572034	6	31	0
145	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2019-08-27 11:03:44.42021	6	2019-08-27 09:01:48.901065	2019-08-28 08:34:56.577128	6	31	0
146	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	6	2019-08-27 06:04:47.235786	2019-08-28 08:34:56.578853	6	31	0
147	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	6	2019-08-27 07:10:02.417886	2019-08-28 08:34:56.584269	6	31	0
148	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	6	2019-08-25 17:57:33.055864	2019-08-28 08:34:56.682441	6	32	0
149	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	t	2019-08-25 23:41:35.271764	6	2019-08-25 17:01:04.882622	2019-08-28 08:34:56.690593	6	32	0
150	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2019-08-25 15:46:03.365431	6	2019-08-25 15:37:52.138325	2019-08-28 08:34:56.692344	6	32	0
151	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	6	2019-08-23 17:27:29.874089	2019-08-28 08:35:50.467491	6	33	0
152	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	6	2019-08-23 18:53:03.673077	2019-08-28 08:35:50.882794	6	33	0
153	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	6	2019-08-23 19:45:55.677004	2019-08-28 08:35:51.544035	6	33	0
859	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	27	2021-05-23 04:38:36.611753	2021-05-28 15:07:53.508726	27	226	0
154	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2019-08-26 12:42:31.196705	6	2019-08-26 07:53:40.085813	2019-08-28 08:35:53.773111	6	34	0
155	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	6	2019-08-26 05:32:13.392978	2019-08-28 08:34:57.24116	6	34	0
156	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	6	2019-08-26 07:56:24.423533	2019-08-28 08:34:57.17198	6	34	0
157	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	6	2019-08-26 06:36:12.963838	2019-08-28 08:34:57.174821	6	34	0
158	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	6	2019-08-26 04:36:13.08945	2019-08-28 08:35:53.886819	6	34	0
159	Excel results	Write results in excel file.	5	f	\N	6	2019-08-26 09:29:07.12965	2019-08-28 08:35:54.24212	6	34	0
160	Template for ddCq analysis	Sample information here.	0	f	\N	6	2019-08-23 03:22:13.730159	2019-08-28 08:35:54.707312	6	35	0
845	Template for ddCq analysis	Sample information here.	0	f	\N	26	2021-05-15 09:19:01.535415	2021-05-20 15:17:47.742276	26	216	0
162	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZAT/3~rep_item~N]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZAT/5~rep_item~P]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZAT/3~rep_item~N]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZAT/5~rep_item~P]</span>.\n          </p>\n        </body>\n      </html>	0	t	2019-08-27 13:28:23.234741	7	2019-08-27 06:47:36.555459	2019-08-29 12:52:30.301258	7	38	0
163	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2019-08-27 08:34:31.583726	7	2019-08-27 05:30:27.078767	2019-08-29 12:51:26.539146	7	38	0
164	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	7	2019-08-27 03:55:28.986809	2019-08-29 12:51:26.681509	7	38	0
165	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	t	2019-08-28 23:49:57.523854	7	2019-08-28 22:49:57.46664	2019-08-29 12:51:26.838496	7	39	0
166	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	7	2019-08-29 01:10:09.58338	2019-08-29 12:51:26.840319	7	39	0
167	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	7	2019-08-29 00:10:40.185028	2019-08-29 12:51:26.843146	7	39	0
168	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	7	2019-08-29 02:19:40.275578	2019-08-29 12:51:26.849683	7	39	0
169	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	7	2019-08-29 00:17:30.504593	2019-08-29 12:51:26.852717	7	39	0
170	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	7	2019-08-29 01:32:34.327206	2019-08-29 12:51:26.858979	7	39	0
171	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	7	2019-08-28 23:46:01.796997	2019-08-29 12:51:26.863835	7	39	0
172	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	7	2019-08-28 23:41:48.561427	2019-08-29 12:51:26.866651	7	39	0
173	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	7	2019-08-29 01:47:19.39583	2019-08-29 12:51:26.873206	7	39	0
174	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	7	2019-08-28 23:06:27.134049	2019-08-29 12:51:26.876115	7	39	0
175	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	7	2019-08-29 01:56:51.403858	2019-08-29 12:51:26.878799	7	39	0
176	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	7	2019-08-29 00:50:31.905619	2019-08-29 12:51:26.885163	7	39	0
177	Before you start	Follow the guidelines bellow.	0	f	\N	7	2019-08-24 02:22:56.274597	2019-08-29 12:52:33.525795	7	40	0
860	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	27	2021-05-23 05:32:33.292347	2021-05-28 15:07:53.544795	27	226	0
178	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2019-08-24 08:06:31.461043	7	2019-08-24 01:02:10.603607	2019-08-29 12:51:27.040487	7	40	0
179	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	t	2019-08-24 08:30:39.70315	7	2019-08-24 02:08:17.335536	2019-08-29 12:51:27.080673	7	40	0
180	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	7	2019-08-24 01:53:05.793779	2019-08-29 12:51:27.340823	7	40	0
991	Before you start	Follow the guidelines bellow.	0	f	\N	31	2021-08-20 18:55:27.463393	2021-08-23 17:40:29.473886	31	267	0
181	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	7	2019-08-24 02:22:04.544963	2019-08-29 12:51:27.11321	7	40	0
182	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	7	2019-08-24 00:42:56.506155	2019-08-29 12:51:27.125289	7	40	0
183	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	7	2019-08-23 23:57:56.928305	2019-08-29 12:51:27.143576	7	40	0
184	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	7	2019-08-23 23:11:12.965865	2019-08-29 12:51:27.352713	7	40	0
185	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	7	2019-08-24 02:52:25.321919	2019-08-29 12:51:27.171772	7	40	0
186	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	7	2019-08-24 01:22:56.144251	2019-08-29 12:51:27.174564	7	40	0
187	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2019-08-24 10:47:56.37126	7	2019-08-24 03:46:37.411269	2019-08-29 12:51:27.189971	7	40	0
188	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	t	2019-08-26 20:36:02.823168	7	2019-08-26 12:51:24.72381	2019-08-29 12:51:27.478295	7	41	0
189	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	7	2019-08-26 16:56:15.749574	2019-08-29 12:51:27.55156	7	41	0
190	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2019-08-26 19:23:04.601061	7	2019-08-26 15:22:29.615327	2019-08-29 12:51:27.515433	7	41	0
191	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	t	2020-06-10 18:30:01.19161	7	2019-08-24 06:31:33.050865	2020-06-10 18:30:01.197777	7	42	0
192	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2020-06-10 18:30:04.649561	7	2019-08-24 06:09:58.235015	2020-06-10 18:30:04.653416	7	42	0
193	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	7	2019-08-24 04:53:57.800866	2019-08-29 12:52:36.734618	7	42	0
861	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	27	2021-05-23 03:27:48.997143	2021-05-28 15:07:53.861901	27	226	0
194	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	7	2019-08-25 17:43:30.969948	2019-08-29 12:52:41.408134	7	43	0
195	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2019-08-25 18:44:09.882526	7	2019-08-25 18:37:43.838467	2019-08-29 12:51:28.924189	7	43	0
196	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2019-08-25 21:27:25.7922	7	2019-08-25 15:23:59.47783	2019-08-29 12:51:28.687764	7	43	0
197	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	7	2019-08-25 17:04:03.178531	2019-08-29 12:51:28.690044	7	43	0
198	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	7	2019-08-25 14:50:00.543205	2019-08-29 12:52:41.682957	7	43	0
199	Excel results	Write results in excel file.	5	t	2019-08-26 00:55:02.215959	7	2019-08-25 16:18:04.983299	2019-08-29 12:52:42.398235	7	43	0
200	Template for ddCq analysis	Sample information here.	0	f	\N	7	2019-08-28 11:04:27.541684	2019-08-29 12:52:43.327114	7	44	0
201	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	8	2019-09-11 07:14:23.650604	2019-09-12 18:26:18.978311	8	46	0
202	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#LJK/3~rep_item~T]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#LJK/5~rep_item~V]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#LJK/3~rep_item~T]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#LJK/5~rep_item~V]</span>.\n          </p>\n        </body>\n      </html>	0	t	2019-09-11 07:54:23.576873	8	2019-09-10 23:01:48.967724	2019-09-12 18:26:27.096437	8	47	0
203	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	8	2019-09-10 19:51:46.87129	2019-09-12 18:26:19.024959	8	47	0
204	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	8	2019-09-10 21:43:01.09445	2019-09-12 18:26:19.054506	8	47	0
205	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	t	2019-09-07 13:31:45.525226	8	2019-09-07 10:50:01.072642	2019-09-12 18:26:19.099106	8	48	0
206	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2019-09-07 17:14:56.699826	8	2019-09-07 14:50:34.899004	2019-09-12 18:26:19.100934	8	48	0
207	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	8	2019-09-07 13:02:34.99865	2019-09-12 18:26:19.10458	8	48	0
208	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2019-09-07 15:11:16.990902	8	2019-09-07 14:19:42.197733	2019-09-12 18:26:19.110591	8	48	0
209	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	8	2019-09-07 10:27:19.080308	2019-09-12 18:26:19.112259	8	48	0
210	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	8	2019-09-07 14:35:45.763472	2019-09-12 18:26:19.115099	8	48	0
211	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	8	2019-09-07 12:55:29.047186	2019-09-12 18:26:19.117813	8	48	0
212	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2019-09-07 16:05:52.454216	8	2019-09-07 13:51:12.303204	2019-09-12 18:26:19.120577	8	48	0
213	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	8	2019-09-07 13:58:29.934359	2019-09-12 18:26:19.12353	8	48	0
214	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	8	2019-09-07 12:03:12.971714	2019-09-12 18:26:19.126307	8	48	0
215	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	8	2019-09-07 13:04:49.179987	2019-09-12 18:26:19.129459	8	48	0
216	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	8	2019-09-07 11:47:25.094955	2019-09-12 18:26:19.13319	8	48	0
217	Before you start	Follow the guidelines bellow.	0	f	\N	8	2019-09-11 01:19:02.873953	2019-09-12 18:26:28.314521	8	49	0
862	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2021-05-23 10:07:38.334155	27	2021-05-23 05:37:59.189002	2021-05-28 15:07:54.105518	27	226	0
218	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	8	2019-09-11 02:51:19.692969	2019-09-12 18:26:19.175363	8	49	0
219	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	8	2019-09-11 02:06:58.183477	2019-09-12 18:26:19.178203	8	49	0
220	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2019-09-11 07:26:06.615666	8	2019-09-11 00:17:30.424507	2019-09-12 18:26:19.284948	8	49	0
221	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	8	2019-09-11 03:15:18.261763	2019-09-12 18:26:19.184203	8	49	0
222	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	8	2019-09-11 03:07:39.470162	2019-09-12 18:26:19.18709	8	49	0
223	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2019-09-11 03:40:23.530642	8	2019-09-10 23:38:36.742546	2019-09-12 18:26:19.193295	8	49	0
224	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2019-09-11 09:28:22.6558	8	2019-09-11 04:27:11.784942	2019-09-12 18:26:19.198367	8	49	0
225	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2019-09-11 14:22:08.848978	8	2019-09-11 04:32:49.574579	2019-09-12 18:26:19.20325	8	49	0
226	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	8	2019-09-10 23:43:22.22098	2019-09-12 18:26:19.204914	8	49	0
227	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2019-09-11 03:42:12.117852	8	2019-09-11 01:55:21.129092	2019-09-12 18:26:19.211106	8	49	0
228	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	t	2019-09-11 18:10:44.414328	8	2019-09-11 08:32:42.542571	2019-09-12 18:26:19.31606	8	50	0
229	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	8	2019-09-11 09:33:43.63486	2019-09-12 18:26:19.330129	8	50	0
230	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2019-09-11 10:30:11.410618	8	2019-09-11 08:33:01.469794	2019-09-12 18:26:19.32409	8	50	0
231	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	8	2019-09-07 00:39:56.295598	2019-09-12 18:26:28.765408	8	51	0
232	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	8	2019-09-06 22:56:53.077945	2019-09-12 18:26:29.131447	8	51	0
233	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	8	2019-09-06 22:37:32.844369	2019-09-12 18:26:29.462669	8	51	0
863	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	27	2021-05-23 03:51:53.409406	2021-05-28 15:07:54.107841	27	226	0
234	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	8	2019-09-12 14:05:20.169681	2019-09-12 18:26:30.96358	8	52	0
235	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2019-09-12 14:20:44.407338	8	2019-09-12 14:00:23.881871	2019-09-12 18:26:19.471664	8	52	0
236	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	8	2019-09-12 11:06:48.338696	2019-09-12 18:26:19.473394	8	52	0
237	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	8	2019-09-12 13:59:44.917844	2019-09-12 18:26:19.476378	8	52	0
238	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	8	2019-09-12 12:29:40.810877	2019-09-12 18:26:31.070035	8	52	0
239	Excel results	Write results in excel file.	5	f	\N	8	2019-09-12 12:27:35.386625	2019-09-12 18:26:31.228378	8	52	0
240	Template for ddCq analysis	Sample information here.	0	f	\N	8	2019-09-07 20:35:28.41842	2019-09-12 18:26:31.56205	8	53	0
241	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	9	2019-11-13 07:55:37.388339	2019-11-13 13:57:42.577716	9	55	0
242	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MSZ/3~rep_item~Y]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MSZ/5~rep_item~a]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MSZ/3~rep_item~Y]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MSZ/5~rep_item~a]</span>.\n          </p>\n        </body>\n      </html>	0	t	2019-11-08 19:05:56.980077	9	2019-11-08 11:50:16.558518	2019-11-13 13:58:19.917164	9	56	0
243	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2019-11-08 11:30:27.50493	9	2019-11-08 09:53:01.745741	2019-11-13 13:57:42.63998	9	56	0
244	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	9	2019-11-08 09:51:02.902915	2019-11-13 13:57:42.678062	9	56	0
245	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	9	2019-11-10 04:16:43.410541	2019-11-13 13:57:42.728795	9	57	0
246	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	9	2019-11-10 01:38:18.33905	2019-11-13 13:57:42.733545	9	57	0
247	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2019-11-10 01:49:08.029345	9	2019-11-10 00:46:10.080017	2019-11-13 13:57:42.740433	9	57	0
248	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2019-11-10 04:21:10.114756	9	2019-11-10 00:44:43.878753	2019-11-13 13:57:42.751783	9	57	0
249	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	9	2019-11-10 01:01:05.017409	2019-11-13 13:57:42.756933	9	57	0
250	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	9	2019-11-10 04:39:33.092311	2019-11-13 13:57:42.763751	9	57	0
251	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	9	2019-11-10 01:02:51.110358	2019-11-13 13:57:42.770749	9	57	0
252	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	9	2019-11-10 03:46:45.661316	2019-11-13 13:57:42.775615	9	57	0
253	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2019-11-10 10:57:21.484343	9	2019-11-10 02:09:59.538063	2019-11-13 13:57:42.78414	9	57	0
254	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	9	2019-11-10 01:07:45.189977	2019-11-13 13:57:42.788179	9	57	0
255	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	9	2019-11-10 04:07:36.047501	2019-11-13 13:57:42.794572	9	57	0
256	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	t	2019-11-10 13:45:33.333237	9	2019-11-10 04:46:24.175498	2019-11-13 13:57:42.799873	9	57	0
257	Before you start	Follow the guidelines bellow.	0	t	2019-11-10 08:34:26.601593	9	2019-11-10 08:17:46.880785	2019-11-13 14:18:36.903025	9	58	0
864	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-23 07:15:45.891988	27	2021-05-23 02:31:14.079939	2021-05-28 15:07:54.110353	27	226	0
258	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2019-11-10 14:05:43.290902	9	2019-11-10 10:47:20.250732	2019-11-13 13:57:42.870962	9	58	0
259	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	9	2019-11-10 08:17:12.250753	2019-11-13 13:57:42.873782	9	58	0
260	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	9	2019-11-10 10:40:56.662323	2019-11-13 13:57:43.01775	9	58	0
261	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2019-11-10 16:14:46.418569	9	2019-11-10 06:15:27.52753	2019-11-13 13:57:42.894939	9	58	0
262	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	9	2019-11-10 06:57:25.774882	2019-11-13 13:57:42.897924	9	58	0
263	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2019-11-10 15:18:12.736547	9	2019-11-10 10:02:20.86307	2019-11-13 13:57:42.904302	9	58	0
264	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	9	2019-11-10 09:39:57.438516	2019-11-13 13:57:43.029162	9	58	0
265	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	9	2019-11-10 08:21:38.34986	2019-11-13 13:57:42.913129	9	58	0
266	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	9	2019-11-10 07:34:10.314648	2019-11-13 13:57:42.916718	9	58	0
267	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	9	2019-11-10 10:00:11.686328	2019-11-13 13:57:42.922618	9	58	0
268	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	t	2019-11-12 04:49:33.734867	9	2019-11-11 19:29:39.634016	2019-11-13 13:57:43.059554	9	59	0
269	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	9	2019-11-11 19:06:16.980816	2019-11-13 13:57:43.078855	9	59	0
270	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	9	2019-11-11 17:42:38.272415	2019-11-13 13:57:43.06889	9	59	0
271	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	9	2019-11-12 11:49:17.526814	2019-11-13 13:58:14.545666	9	60	0
272	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2019-11-12 16:39:41.431574	9	2019-11-12 10:52:42.359787	2019-11-13 13:58:23.032718	9	60	0
273	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	9	2019-11-12 13:57:11.72679	2019-11-13 13:58:23.756752	9	60	0
274	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2019-11-12 15:01:25.411296	9	2019-11-12 09:48:42.432783	2019-11-13 13:58:25.908354	9	61	0
275	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	9	2019-11-12 11:02:25.668916	2019-11-13 13:57:43.386773	9	61	0
276	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	9	2019-11-12 07:40:27.927026	2019-11-13 13:57:43.330021	9	61	0
277	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	9	2019-11-12 08:57:50.965114	2019-11-13 13:57:43.334545	9	61	0
278	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	9	2019-11-12 10:07:14.650196	2019-11-13 14:18:38.228253	9	61	0
279	Excel results	Write results in excel file.	5	f	\N	9	2019-11-12 10:58:48.293396	2019-11-13 13:58:17.561663	9	61	0
280	Template for ddCq analysis	Sample information here.	0	f	\N	9	2019-11-13 10:21:44.189031	2019-11-13 13:58:27.287071	9	62	0
281	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	t	2019-11-23 02:45:27.66483	4	2019-11-22 18:19:47.745957	2019-11-25 14:09:27.218292	4	64	0
282	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#KOK/3~rep_item~d]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#KOK/5~rep_item~f]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#KOK/3~rep_item~d]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#KOK/5~rep_item~f]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	4	2019-11-23 03:47:41.438522	2019-11-25 14:09:35.75239	4	65	0
283	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	4	2019-11-23 04:49:58.02027	2019-11-25 14:09:27.293868	4	65	0
284	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	4	2019-11-23 03:59:18.395088	2019-11-25 14:09:27.389469	4	65	0
285	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	4	2019-11-19 19:45:31.974582	2019-11-25 14:09:27.572191	4	66	0
286	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	4	2019-11-19 23:14:14.342084	2019-11-25 14:09:27.581804	4	66	0
287	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	4	2019-11-19 20:15:58.888401	2019-11-25 14:09:27.603577	4	66	0
288	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	4	2019-11-19 23:51:56.22965	2019-11-25 14:09:27.62038	4	66	0
289	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	4	2019-11-19 22:00:27.95024	2019-11-25 14:09:27.636512	4	66	0
290	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	4	2019-11-19 20:40:31.680667	2019-11-25 14:09:27.649102	4	66	0
291	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	4	2019-11-19 21:24:22.798689	2019-11-25 14:09:27.666964	4	66	0
292	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2019-11-20 01:05:41.001879	4	2019-11-19 19:09:44.756365	2019-11-25 14:09:27.684762	4	66	0
293	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2019-11-20 07:36:19.920918	4	2019-11-19 22:29:25.270713	2019-11-25 14:09:27.719247	4	66	0
294	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2019-11-20 00:54:28.949177	4	2019-11-19 22:42:53.518297	2019-11-25 14:09:27.731201	4	66	0
295	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2019-11-20 05:02:07.827141	4	2019-11-19 19:53:02.514425	2019-11-25 14:09:27.759592	4	66	0
296	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	4	2019-11-19 23:26:38.803979	2019-11-25 14:09:27.769709	4	66	0
297	Before you start	Follow the guidelines bellow.	0	t	2019-11-24 12:50:49.674835	4	2019-11-24 04:37:26.037674	2019-11-25 14:29:52.33627	4	67	0
865	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	27	2021-05-23 02:52:49.09949	2021-05-28 15:07:54.15331	27	226	0
298	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2019-11-24 11:27:41.795956	4	2019-11-24 07:49:00.764436	2019-11-25 14:09:28.050246	4	67	0
299	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	t	2019-11-24 08:56:48.999056	4	2019-11-24 06:54:01.923725	2019-11-25 14:09:28.08428	4	67	0
300	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	4	2019-11-24 06:42:28.515194	2019-11-25 14:09:28.318822	4	67	0
445	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	16	2020-10-06 15:06:08.654508	2020-10-07 19:20:26.701395	16	110	0
301	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2019-11-24 15:52:38.799249	4	2019-11-24 06:59:18.270927	2019-11-25 14:09:28.139487	4	67	0
302	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	t	2019-11-24 15:42:50.062991	4	2019-11-24 08:35:34.282288	2019-11-25 14:09:28.175965	4	67	0
303	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	4	2019-11-24 08:54:43.580322	2019-11-25 14:09:28.182132	4	67	0
304	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	4	2019-11-24 07:06:41.532549	2019-11-25 14:09:28.331308	4	67	0
305	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	4	2019-11-24 04:01:22.45572	2019-11-25 14:09:28.194839	4	67	0
306	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	4	2019-11-24 04:08:17.648701	2019-11-25 14:09:28.2013	4	67	0
307	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2019-11-24 14:54:24.97132	4	2019-11-24 08:06:32.993972	2019-11-25 14:09:28.207272	4	67	0
308	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	4	2019-11-21 07:35:29.10959	2019-11-25 14:09:28.365009	4	68	0
309	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	t	2019-11-21 09:41:20.186753	4	2019-11-21 09:18:44.899367	2019-11-25 14:09:28.405118	4	68	0
310	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2019-11-21 15:24:44.895391	4	2019-11-21 06:39:53.284423	2019-11-25 14:09:28.390986	4	68	0
311	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	4	2019-11-24 08:17:00.354392	2019-11-25 14:09:30.905588	4	69	0
312	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	4	2019-11-24 09:52:39.080902	2019-11-25 14:09:38.672559	4	69	0
313	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	4	2019-11-24 07:38:21.448222	2019-11-25 14:09:39.412702	4	69	0
314	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	4	2019-11-25 06:08:20.180919	2019-11-25 14:09:41.565161	4	70	0
315	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	4	2019-11-25 07:02:56.420208	2019-11-25 14:09:28.713408	4	70	0
316	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	4	2019-11-25 09:35:59.190251	2019-11-25 14:09:28.654684	4	70	0
317	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	4	2019-11-25 06:07:26.082404	2019-11-25 14:09:28.660623	4	70	0
318	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	4	2019-11-25 10:25:31.968073	2019-11-25 14:29:53.514063	4	70	0
319	Excel results	Write results in excel file.	5	f	\N	4	2019-11-25 08:56:23.651783	2019-11-25 14:09:33.407249	4	70	0
320	Template for ddCq analysis	Sample information here.	0	f	\N	4	2019-11-21 11:33:43.00889	2019-11-25 14:09:42.895445	4	71	0
321	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	t	2019-11-22 17:37:37.678545	10	2019-11-22 09:15:42.943417	2019-11-25 18:00:53.119026	10	79	0
322	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#HOY/3~rep_item~i]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#HOY/5~rep_item~k]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#HOY/3~rep_item~i]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#HOY/5~rep_item~k]</span>.\n          </p>\n        </body>\n      </html>	0	t	2019-11-20 11:01:20.174231	10	2019-11-20 10:53:28.362847	2019-11-25 18:01:05.889238	10	80	0
323	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	10	2019-11-20 06:49:25.432455	2019-11-25 18:00:53.161634	10	80	0
324	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	t	2019-11-20 10:33:53.001675	10	2019-11-20 09:54:04.521251	2019-11-25 18:00:53.174664	10	80	0
325	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	10	2019-11-24 16:19:52.915373	2019-11-25 18:00:53.2574	10	81	0
326	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	10	2019-11-24 15:48:52.798111	2019-11-25 18:00:53.263859	10	81	0
327	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	10	2019-11-24 12:14:37.185801	2019-11-25 18:00:53.268635	10	81	0
328	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2019-11-24 21:33:13.55765	10	2019-11-24 11:56:06.488698	2019-11-25 18:00:53.285246	10	81	0
329	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	10	2019-11-24 15:32:40.686377	2019-11-25 18:00:53.28872	10	81	0
330	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2019-11-24 22:59:49.291194	10	2019-11-24 15:41:26.701669	2019-11-25 18:00:53.300154	10	81	0
331	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	10	2019-11-24 14:43:09.714627	2019-11-25 18:00:53.303979	10	81	0
332	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2019-11-24 19:56:08.385838	10	2019-11-24 14:05:58.158923	2019-11-25 18:00:53.313764	10	81	0
333	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	10	2019-11-24 12:36:11.637668	2019-11-25 18:00:53.319012	10	81	0
334	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2019-11-24 21:46:30.395795	10	2019-11-24 15:35:00.104032	2019-11-25 18:00:53.329868	10	81	0
335	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	10	2019-11-24 12:11:08.626788	2019-11-25 18:00:53.332878	10	81	0
336	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	10	2019-11-24 16:28:52.791128	2019-11-25 18:00:53.339769	10	81	0
337	Before you start	Follow the guidelines bellow.	0	t	2019-11-25 01:20:27.419137	10	2019-11-24 16:12:08.463316	2019-11-25 18:21:19.152257	10	82	0
866	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	t	2021-05-23 01:51:09.394232	27	2021-05-23 01:49:53.614855	2021-05-28 15:07:54.480969	27	226	0
338	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	10	2019-11-24 17:37:51.290998	2019-11-25 18:00:53.414794	10	82	0
339	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	10	2019-11-24 18:03:12.793545	2019-11-25 18:00:53.419954	10	82	0
340	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	10	2019-11-24 18:05:27.37124	2019-11-25 18:00:53.556372	10	82	0
341	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	10	2019-11-24 18:05:22.122957	2019-11-25 18:00:53.433263	10	82	0
342	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	10	2019-11-24 13:41:39.338303	2019-11-25 18:00:53.438671	10	82	0
343	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2019-11-24 22:15:28.125481	10	2019-11-24 17:21:35.045012	2019-11-25 18:00:53.448559	10	82	0
344	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	10	2019-11-24 15:17:34.44376	2019-11-25 18:00:53.571648	10	82	0
345	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2019-11-25 02:54:31.114769	10	2019-11-24 17:22:11.436038	2019-11-25 18:00:53.462995	10	82	0
346	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	10	2019-11-24 15:26:05.611367	2019-11-25 18:00:53.465703	10	82	0
347	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	10	2019-11-24 16:37:37.265211	2019-11-25 18:00:53.470275	10	82	0
348	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	10	2019-11-21 12:38:57.58597	2019-11-25 18:00:53.601648	10	83	0
349	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	10	2019-11-21 11:47:21.145857	2019-11-25 18:00:53.623855	10	83	0
350	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2019-11-21 21:38:43.772529	10	2019-11-21 14:54:23.337232	2019-11-25 18:00:53.612994	10	83	0
351	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	10	2019-11-20 08:01:39.71984	2019-11-25 18:01:00.759408	10	84	0
352	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	10	2019-11-20 05:19:01.973714	2019-11-25 18:01:08.767006	10	84	0
353	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	10	2019-11-20 06:42:55.773349	2019-11-25 18:01:09.468756	10	84	0
867	Before you start	Follow the guidelines bellow.	0	t	2021-05-28 06:11:13.516602	27	2021-05-27 21:35:16.32719	2021-05-28 15:29:43.981519	27	227	0
354	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2019-11-24 11:26:11.20312	10	2019-11-24 09:04:14.198112	2019-11-25 18:01:11.478482	10	85	0
355	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	10	2019-11-24 08:22:47.472491	2019-11-25 18:00:53.9168	10	85	0
356	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	10	2019-11-24 08:42:41.993149	2019-11-25 18:00:53.859364	10	85	0
357	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	10	2019-11-24 11:11:27.027233	2019-11-25 18:00:53.865295	10	85	0
358	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	10	2019-11-24 08:44:29.965087	2019-11-25 18:21:20.165503	10	85	0
359	Excel results	Write results in excel file.	5	f	\N	10	2019-11-24 08:20:30.703157	2019-11-25 18:01:03.851458	10	85	0
360	Template for ddCq analysis	Sample information here.	0	f	\N	10	2019-11-20 01:17:06.721375	2019-11-25 18:01:12.770129	10	86	0
361	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	11	2020-05-08 07:17:10.531787	2020-05-12 16:13:04.81445	11	88	0
362	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYB/3~rep_item~n]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYB/5~rep_item~p]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYB/3~rep_item~n]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYB/5~rep_item~p]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	11	2020-05-06 17:32:05.501063	2020-05-12 16:13:55.887846	11	89	0
363	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	11	2020-05-06 21:24:13.522515	2020-05-12 16:13:05.00301	11	89	0
364	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	11	2020-05-06 18:42:51.515015	2020-05-12 16:13:05.529913	11	89	0
365	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	11	2020-05-09 04:11:47.725788	2020-05-12 16:13:06.076368	11	90	0
366	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	11	2020-05-09 01:52:10.680088	2020-05-12 16:13:06.111058	11	90	0
367	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	11	2020-05-09 04:58:14.744976	2020-05-12 16:13:06.143698	11	90	0
368	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2020-05-09 11:18:19.918538	11	2020-05-09 03:40:16.370974	2020-05-12 16:13:06.191262	11	90	0
369	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	11	2020-05-09 06:45:44.248309	2020-05-12 16:13:06.36617	11	90	0
370	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2020-05-09 06:28:51.129932	11	2020-05-09 04:55:08.143433	2020-05-12 16:13:06.377393	11	90	0
371	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	11	2020-05-09 05:42:04.569629	2020-05-12 16:13:06.381111	11	90	0
372	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	11	2020-05-09 03:47:36.408327	2020-05-12 16:13:06.385331	11	90	0
373	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	11	2020-05-09 03:49:31.702629	2020-05-12 16:13:06.388961	11	90	0
374	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	11	2020-05-09 04:35:05.343585	2020-05-12 16:13:06.392918	11	90	0
375	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	11	2020-05-09 02:45:38.485627	2020-05-12 16:13:06.396579	11	90	0
376	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	t	2020-05-09 11:23:16.328826	11	2020-05-09 02:18:33.495764	2020-05-12 16:13:06.40109	11	90	0
377	Before you start	Follow the guidelines bellow.	0	t	2020-05-10 06:04:31.534339	11	2020-05-10 02:45:57.928209	2020-05-12 16:34:13.868549	11	91	0
378	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2020-05-10 02:18:58.054467	11	2020-05-10 01:16:07.217949	2020-05-12 16:13:06.459583	11	91	0
379	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	11	2020-05-10 01:26:25.72858	2020-05-12 16:13:06.462712	11	91	0
380	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	11	2020-05-10 04:11:49.963075	2020-05-12 16:13:06.586302	11	91	0
381	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2020-05-10 09:54:57.946933	11	2020-05-10 00:16:56.7349	2020-05-12 16:13:06.476284	11	91	0
382	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	11	2020-05-10 01:36:20.150616	2020-05-12 16:13:06.479056	11	91	0
383	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	11	2020-05-10 02:14:15.454858	2020-05-12 16:13:06.483419	11	91	0
384	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	11	2020-05-10 02:15:48.163618	2020-05-12 16:13:06.595352	11	91	0
385	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2020-05-10 03:28:18.885517	11	2020-05-10 03:22:37.827802	2020-05-12 16:13:06.495879	11	91	0
386	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	11	2020-05-10 04:41:58.465564	2020-05-12 16:13:06.498057	11	91	0
387	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2020-05-10 06:40:20.131401	11	2020-05-10 03:35:09.219108	2020-05-12 16:13:06.502841	11	91	0
388	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	11	2020-05-07 02:11:41.376519	2020-05-12 16:13:06.620415	11	92	0
389	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	11	2020-05-06 22:00:00.3747	2020-05-12 16:13:06.641629	11	92	0
390	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	11	2020-05-06 21:46:26.092558	2020-05-12 16:13:06.63163	11	92	0
391	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	t	2020-05-12 00:54:45.833417	11	2020-05-12 00:22:44.177365	2020-05-12 16:13:50.352498	11	93	0
392	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2020-05-12 05:09:28.002494	11	2020-05-12 00:17:31.515889	2020-05-12 16:13:59.024414	11	93	0
393	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	11	2020-05-11 21:19:20.72692	2020-05-12 16:13:59.792061	11	93	0
394	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2020-05-10 10:33:46.37244	11	2020-05-10 04:28:35.712494	2020-05-12 16:14:02.171475	11	94	0
395	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2020-05-10 07:10:48.333027	11	2020-05-10 01:05:21.260318	2020-05-12 16:13:06.966628	11	94	0
396	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	11	2020-05-10 04:07:32.740013	2020-05-12 16:13:06.917822	11	94	0
397	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	11	2020-05-10 04:52:16.428653	2020-05-12 16:13:06.92209	11	94	0
398	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	11	2020-05-10 01:43:49.845092	2020-05-12 16:34:15.458482	11	94	0
399	Excel results	Write results in excel file.	5	f	\N	11	2020-05-10 01:53:55.802535	2020-05-12 16:13:53.093909	11	94	0
400	Template for ddCq analysis	Sample information here.	0	f	\N	11	2020-05-12 05:09:44.971702	2020-05-12 16:14:03.681239	11	95	0
401	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	12	2020-06-09 18:47:34.490173	2020-06-10 14:43:17.990474	12	97	0
422	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	12	2020-06-06 16:11:57.234179	2020-06-10 14:43:18.300646	12	100	0
402	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RTK/3~rep_item~s]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RTK/5~rep_item~u]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RTK/3~rep_item~s]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#RTK/5~rep_item~u]</span>.\n          </p>\n        </body>\n      </html>	0	t	2020-06-05 18:20:46.825711	12	2020-06-05 12:01:38.628276	2020-06-10 14:44:25.672707	12	98	0
403	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	12	2020-06-05 10:15:19.212811	2020-06-10 14:43:18.07115	12	98	0
404	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	12	2020-06-05 09:50:46.472267	2020-06-10 14:43:18.106074	12	98	0
405	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	12	2020-06-08 10:17:02.321647	2020-06-10 14:43:18.167444	12	99	0
406	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	12	2020-06-08 13:12:10.154632	2020-06-10 14:43:18.171852	12	99	0
407	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	12	2020-06-08 14:00:48.399495	2020-06-10 14:43:18.176162	12	99	0
408	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2020-06-08 17:33:13.540764	12	2020-06-08 12:38:03.012497	2020-06-10 14:43:18.184014	12	99	0
409	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	t	2020-06-08 12:12:14.81592	12	2020-06-08 11:04:27.369108	2020-06-10 14:43:18.190522	12	99	0
410	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	12	2020-06-08 12:10:37.736506	2020-06-10 14:43:18.192751	12	99	0
411	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2020-06-08 14:53:39.492687	12	2020-06-08 11:47:11.300894	2020-06-10 14:43:18.196606	12	99	0
412	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	12	2020-06-08 11:02:36.176951	2020-06-10 14:43:18.200196	12	99	0
413	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2020-06-08 14:34:31.935457	12	2020-06-08 12:51:50.454562	2020-06-10 14:43:18.208802	12	99	0
414	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2020-06-08 11:00:49.813299	12	2020-06-08 10:23:45.751691	2020-06-10 14:43:18.21508	12	99	0
415	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	12	2020-06-08 12:09:14.073985	2020-06-10 14:43:18.21856	12	99	0
416	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	12	2020-06-08 11:44:23.934363	2020-06-10 14:43:18.222125	12	99	0
417	Before you start	Follow the guidelines bellow.	0	f	\N	12	2020-06-06 15:09:16.458109	2020-06-10 15:04:42.991258	12	100	0
418	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2020-06-06 20:11:55.896556	12	2020-06-06 17:51:36.031292	2020-06-10 14:43:18.280106	12	100	0
419	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	12	2020-06-06 14:57:01.804066	2020-06-10 14:43:18.285035	12	100	0
420	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	12	2020-06-06 15:34:32.154987	2020-06-10 14:43:18.430885	12	100	0
421	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2020-06-06 13:26:42.710656	12	2020-06-06 13:17:27.156235	2020-06-10 14:43:18.298525	12	100	0
423	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	12	2020-06-06 16:37:25.516383	2020-06-10 14:43:18.306463	12	100	0
424	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	12	2020-06-06 14:27:35.06588	2020-06-10 14:43:18.440102	12	100	0
425	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2020-06-06 21:29:20.290754	12	2020-06-06 13:59:36.630022	2020-06-10 14:43:18.317621	12	100	0
426	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	t	2020-06-07 01:21:48.635965	12	2020-06-06 16:22:32.150601	2020-06-10 14:43:18.326612	12	100	0
427	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	12	2020-06-06 15:12:30.380397	2020-06-10 14:43:18.330243	12	100	0
428	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	12	2020-06-05 23:22:04.874451	2020-06-10 14:43:18.461811	12	101	0
429	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	12	2020-06-05 21:41:24.884818	2020-06-10 14:43:18.483985	12	101	0
430	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	12	2020-06-05 21:09:36.2481	2020-06-10 14:43:18.472698	12	101	0
431	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	12	2020-06-08 09:59:26.287874	2020-06-10 14:44:20.999096	12	102	0
432	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	12	2020-06-08 12:59:29.223347	2020-06-10 14:44:28.463568	12	102	0
433	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	12	2020-06-08 12:47:07.429244	2020-06-10 14:44:29.560551	12	102	0
434	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	12	2020-06-05 17:47:36.321295	2020-06-10 14:44:32.89061	12	103	0
435	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	12	2020-06-05 18:50:33.485895	2020-06-10 14:43:18.726738	12	103	0
436	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	12	2020-06-05 16:45:51.116174	2020-06-10 14:43:18.677389	12	103	0
437	Eliminate results that have positive NTCs	And repeat procedure.	3	t	2020-06-05 16:42:24.537476	12	2020-06-05 16:22:02.116279	2020-06-10 14:43:18.685097	12	103	0
438	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	12	2020-06-05 17:24:02.686084	2020-06-10 15:04:44.192148	12	103	0
439	Excel results	Write results in excel file.	5	t	2020-06-05 21:29:07.430641	12	2020-06-05 17:21:16.800358	2020-06-10 14:44:23.30204	12	103	0
440	Template for ddCq analysis	Sample information here.	0	t	2020-06-05 07:06:42.987235	12	2020-06-04 23:33:44.006818	2020-06-10 14:44:34.548209	12	104	0
442	Sampling	<div>Make sure samples meet sample criteria</div>	0	f	\N	7	2020-06-10 18:18:36.473777	2020-06-10 18:18:36.473777	7	107	0
443	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	16	2020-10-07 11:05:04.349526	2020-10-07 19:20:26.639336	16	109	0
444	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DPC/3~rep_item~x]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DPC/5~rep_item~z]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DPC/3~rep_item~x]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DPC/5~rep_item~z]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	16	2020-10-06 12:33:26.639422	2020-10-07 19:20:40.443503	16	110	0
446	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	16	2020-10-06 10:42:12.462394	2020-10-07 19:20:26.753422	16	110	0
447	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	16	2020-10-02 09:20:36.421261	2020-10-07 19:20:26.833608	16	111	0
448	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2020-10-02 17:08:42.713837	16	2020-10-02 12:49:44.073561	2020-10-07 19:20:26.842169	16	111	0
449	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2020-10-02 13:50:11.807754	16	2020-10-02 10:54:09.064966	2020-10-07 19:20:26.848558	16	111	0
450	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	16	2020-10-02 08:29:03.523738	2020-10-07 19:20:26.851027	16	111	0
451	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	16	2020-10-02 10:53:22.932009	2020-10-07 19:20:26.856026	16	111	0
452	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	16	2020-10-02 11:32:08.817097	2020-10-07 19:20:26.860553	16	111	0
453	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	16	2020-10-02 09:44:11.350497	2020-10-07 19:20:26.864679	16	111	0
454	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2020-10-02 20:35:54.403432	16	2020-10-02 11:58:58.786339	2020-10-07 19:20:26.876157	16	111	0
455	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	16	2020-10-02 10:37:15.065074	2020-10-07 19:20:26.880728	16	111	0
456	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2020-10-02 18:55:36.401705	16	2020-10-02 12:44:58.270052	2020-10-07 19:20:26.890877	16	111	0
457	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	16	2020-10-02 11:49:16.169593	2020-10-07 19:20:26.894901	16	111	0
458	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	16	2020-10-02 13:02:07.019204	2020-10-07 19:20:26.902549	16	111	0
459	Before you start	Follow the guidelines bellow.	0	t	2020-10-06 11:16:40.43757	16	2020-10-06 08:18:15.174447	2020-10-07 19:40:56.501411	16	112	0
460	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	16	2020-10-06 11:04:12.292249	2020-10-07 19:20:26.976977	16	112	0
461	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	16	2020-10-06 09:04:49.028889	2020-10-07 19:20:26.982734	16	112	0
462	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	16	2020-10-06 10:34:34.786133	2020-10-07 19:20:27.166186	16	112	0
463	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	16	2020-10-06 10:53:04.715696	2020-10-07 19:20:26.992521	16	112	0
464	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	16	2020-10-06 10:12:45.169961	2020-10-07 19:20:26.999494	16	112	0
465	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	16	2020-10-06 07:21:30.361768	2020-10-07 19:20:27.033493	16	112	0
466	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	16	2020-10-06 07:39:58.749739	2020-10-07 19:20:27.177793	16	112	0
467	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	16	2020-10-06 07:24:13.799951	2020-10-07 19:20:27.054015	16	112	0
468	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	16	2020-10-06 08:32:42.394852	2020-10-07 19:20:27.059773	16	112	0
469	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	16	2020-10-06 11:06:01.464638	2020-10-07 19:20:27.064159	16	112	0
470	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	16	2020-10-07 11:57:51.097583	2020-10-07 19:20:27.205007	16	113	0
471	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	16	2020-10-07 13:38:47.725168	2020-10-07 19:20:27.226418	16	113	0
472	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	16	2020-10-07 13:51:48.415237	2020-10-07 19:20:27.215237	16	113	0
473	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	t	2020-10-06 04:08:32.117892	16	2020-10-05 21:57:39.935721	2020-10-07 19:20:35.128915	16	114	0
474	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	16	2020-10-05 21:03:24.265289	2020-10-07 19:20:43.541874	16	114	0
475	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	16	2020-10-05 20:21:14.037262	2020-10-07 19:20:44.256062	16	114	0
476	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	16	2020-10-06 07:30:21.90165	2020-10-07 19:20:46.338163	16	115	0
477	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	16	2020-10-06 10:31:35.352355	2020-10-07 19:20:27.601022	16	115	0
478	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	16	2020-10-06 07:34:52.704067	2020-10-07 19:20:27.532264	16	115	0
479	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	16	2020-10-06 10:42:27.438824	2020-10-07 19:20:27.537487	16	115	0
480	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	16	2020-10-06 09:17:18.475277	2020-10-07 19:40:57.547827	16	115	0
481	Excel results	Write results in excel file.	5	f	\N	16	2020-10-06 06:48:59.416918	2020-10-07 19:20:37.743498	16	115	0
482	Template for ddCq analysis	Sample information here.	0	t	2020-10-02 04:59:56.304319	16	2020-10-02 02:13:30.074519	2020-10-07 19:20:47.763837	16	116	0
483	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	1	2020-11-15 18:58:07.166245	2020-11-18 14:50:53.771415	1	118	0
484	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYV/3~rep_item~12]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYV/5~rep_item~14]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYV/3~rep_item~12]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#DYV/5~rep_item~14]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	1	2020-11-17 11:31:00.033667	2020-11-18 14:51:02.034102	1	119	0
485	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2020-11-17 17:51:24.331527	1	2020-11-17 13:14:25.758157	2020-11-18 14:50:53.853942	1	119	0
486	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	1	2020-11-17 14:11:55.531408	2020-11-18 14:50:53.937794	1	119	0
487	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	1	2020-11-16 00:37:52.490507	2020-11-18 14:50:54.024227	1	120	0
488	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	1	2020-11-16 02:42:44.915272	2020-11-18 14:50:54.032934	1	120	0
489	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2020-11-16 11:54:40.041342	1	2020-11-16 02:17:19.110498	2020-11-18 14:50:54.045565	1	120	0
490	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	1	2020-11-16 04:28:44.928176	2020-11-18 14:50:54.052144	1	120	0
491	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	1	2020-11-16 04:12:48.863968	2020-11-18 14:50:54.059614	1	120	0
492	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	1	2020-11-16 02:20:44.969501	2020-11-18 14:50:54.107391	1	120	0
493	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	1	2020-11-16 04:34:08.082048	2020-11-18 14:50:54.113887	1	120	0
494	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	1	2020-11-16 00:21:23.003671	2020-11-18 14:50:54.11931	1	120	0
495	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	1	2020-11-16 01:55:47.964445	2020-11-18 14:50:54.1249	1	120	0
496	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	1	2020-11-16 03:35:35.611003	2020-11-18 14:50:54.13043	1	120	0
497	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2020-11-16 03:59:32.454153	1	2020-11-16 03:19:51.611799	2020-11-18 14:50:54.142144	1	120	0
498	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	1	2020-11-16 03:23:53.894506	2020-11-18 14:50:54.147324	1	120	0
499	Before you start	Follow the guidelines bellow.	0	t	2020-11-14 07:18:12.93619	1	2020-11-14 03:11:55.420306	2020-11-18 15:11:36.372788	1	121	0
500	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	1	2020-11-13 23:57:02.275894	2020-11-18 14:50:54.221305	1	121	0
501	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	1	2020-11-13 23:15:34.422655	2020-11-18 14:50:54.227656	1	121	0
502	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2020-11-14 10:33:50.764602	1	2020-11-14 03:05:43.305223	2020-11-18 14:50:54.396154	1	121	0
503	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	1	2020-11-14 00:55:48.291908	2020-11-18 14:50:54.245544	1	121	0
504	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	1	2020-11-14 00:36:45.517078	2020-11-18 14:50:54.251476	1	121	0
505	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	1	2020-11-13 22:32:43.177907	2020-11-18 14:50:54.257205	1	121	0
528	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2020-11-17 02:31:21.370396	17	2020-11-16 20:57:40.121323	2020-11-18 14:53:25.572221	17	129	0
547	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	17	2020-11-15 21:55:03.256464	2020-11-18 14:53:25.709135	17	130	0
506	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2020-11-14 03:11:43.385398	1	2020-11-13 23:18:45.188125	2020-11-18 14:50:54.408923	1	121	0
507	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2020-11-14 10:29:04.584453	1	2020-11-14 01:22:37.181779	2020-11-18 14:50:54.279765	1	121	0
508	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	1	2020-11-14 02:42:36.706198	2020-11-18 14:50:54.285135	1	121	0
509	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	1	2020-11-14 01:18:42.593532	2020-11-18 14:50:54.29197	1	121	0
510	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	1	2020-11-12 17:56:42.59751	2020-11-18 14:50:54.44219	1	122	0
511	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	1	2020-11-12 18:12:57.014612	2020-11-18 14:50:54.468612	1	122	0
512	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2020-11-12 22:53:11.747893	1	2020-11-12 17:19:45.795301	2020-11-18 14:50:54.459529	1	122	0
513	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	1	2020-11-14 00:01:44.245156	2020-11-18 14:50:56.735875	1	123	0
514	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2020-11-14 01:35:04.437474	1	2020-11-14 00:25:51.42551	2020-11-18 14:51:04.995828	1	123	0
515	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	1	2020-11-13 21:38:44.78875	2020-11-18 14:51:05.828486	1	123	0
516	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2020-11-14 02:14:00.533429	1	2020-11-14 01:58:21.397237	2020-11-18 14:51:07.986608	1	124	0
517	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2020-11-14 09:32:28.380039	1	2020-11-14 03:42:40.34961	2020-11-18 14:50:54.806215	1	124	0
518	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2020-11-14 12:08:07.042081	1	2020-11-14 03:04:10.971591	2020-11-18 14:50:54.744561	1	124	0
519	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	1	2020-11-14 02:22:57.552822	2020-11-18 14:50:54.750839	1	124	0
520	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	1	2020-11-14 02:04:28.608544	2020-11-18 15:11:37.519269	1	124	0
521	Excel results	Write results in excel file.	5	f	\N	1	2020-11-14 00:42:19.205663	2020-11-18 14:50:59.325302	1	124	0
522	Template for ddCq analysis	Sample information here.	0	t	2020-11-14 19:20:59.191506	1	2020-11-14 15:16:05.31509	2020-11-18 14:51:09.343685	1	125	0
523	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	17	2020-11-13 10:34:22.084805	2020-11-18 14:53:25.458654	17	127	0
524	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROW/3~rep_item~17]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROW/5~rep_item~19]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROW/3~rep_item~17]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROW/5~rep_item~19]</span>.\n          </p>\n        </body>\n      </html>	0	t	2020-11-18 11:55:26.384073	17	2020-11-18 04:22:40.426735	2020-11-18 14:54:18.658012	17	128	0
525	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	17	2020-11-18 03:03:34.935984	2020-11-18 14:53:25.489511	17	128	0
526	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	17	2020-11-18 06:35:48.765799	2020-11-18 14:53:25.523747	17	128	0
527	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	t	2020-11-17 04:17:13.117465	17	2020-11-16 21:46:17.710292	2020-11-18 14:53:25.566192	17	129	0
529	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	17	2020-11-16 23:31:38.621652	2020-11-18 14:53:25.574351	17	129	0
530	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	17	2020-11-16 21:14:51.271641	2020-11-18 14:53:25.57805	17	129	0
531	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	17	2020-11-16 19:49:19.301556	2020-11-18 14:53:25.581914	17	129	0
532	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	17	2020-11-16 18:44:09.798061	2020-11-18 14:53:25.586214	17	129	0
533	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2020-11-17 01:35:11.562535	17	2020-11-16 20:38:42.116318	2020-11-18 14:53:25.596898	17	129	0
534	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	17	2020-11-16 23:31:00.753299	2020-11-18 14:53:25.60083	17	129	0
535	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	17	2020-11-16 23:34:45.534018	2020-11-18 14:53:25.606742	17	129	0
536	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	17	2020-11-16 19:50:14.235615	2020-11-18 14:53:25.614636	17	129	0
537	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2020-11-17 03:17:33.211879	17	2020-11-16 20:22:30.585248	2020-11-18 14:53:25.619892	17	129	0
538	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	17	2020-11-16 22:08:27.639828	2020-11-18 14:53:25.625619	17	129	0
539	Before you start	Follow the guidelines bellow.	0	f	\N	17	2020-11-15 22:45:18.917511	2020-11-18 15:14:39.909472	17	130	0
540	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	17	2020-11-15 20:52:33.84154	2020-11-18 14:53:25.676498	17	130	0
541	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	17	2020-11-15 22:40:50.036426	2020-11-18 14:53:25.680195	17	130	0
542	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	17	2020-11-15 21:03:10.306071	2020-11-18 14:53:25.809063	17	130	0
543	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	17	2020-11-15 23:52:05.622597	2020-11-18 14:53:25.690267	17	130	0
544	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	17	2020-11-15 22:23:04.00382	2020-11-18 14:53:25.693876	17	130	0
545	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	17	2020-11-15 21:56:58.583465	2020-11-18 14:53:25.697417	17	130	0
546	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2020-11-16 05:59:24.350871	17	2020-11-15 21:54:37.612835	2020-11-18 14:53:25.820769	17	130	0
548	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	t	2020-11-16 03:20:39.291752	17	2020-11-15 21:27:12.101996	2020-11-18 14:53:25.71732	17	130	0
549	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	17	2020-11-15 22:58:32.052701	2020-11-18 14:53:25.739293	17	130	0
550	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	17	2020-11-16 00:13:26.304147	2020-11-18 14:53:25.84953	17	131	0
551	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	17	2020-11-15 20:37:37.437418	2020-11-18 14:53:25.869586	17	131	0
552	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	17	2020-11-15 22:34:39.179141	2020-11-18 14:53:25.859288	17	131	0
553	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	17	2020-11-17 14:31:11.822495	2020-11-18 14:54:13.736198	17	132	0
554	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2020-11-17 23:04:35.983202	17	2020-11-17 13:45:10.407887	2020-11-18 14:54:21.725518	17	132	0
555	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	17	2020-11-17 14:39:09.858659	2020-11-18 14:54:22.44911	17	132	0
556	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2020-11-15 02:22:09.191399	17	2020-11-14 18:34:48.681167	2020-11-18 14:54:24.666241	17	133	0
558	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2020-11-14 22:23:57.652678	17	2020-11-14 16:46:48.020452	2020-11-18 14:53:26.070502	17	133	0
559	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	17	2020-11-14 18:48:18.720616	2020-11-18 14:53:26.072749	17	133	0
560	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	t	2020-11-14 18:44:06.3365	17	2020-11-14 17:26:07.791464	2020-11-18 15:14:41.138737	17	133	0
561	Excel results	Write results in excel file.	5	t	2020-11-15 00:17:33.333565	17	2020-11-14 15:54:30.204085	2020-11-18 14:54:16.46534	17	133	0
562	Template for ddCq analysis	Sample information here.	0	f	\N	17	2020-11-13 02:07:05.839755	2020-11-18 14:54:26.074885	17	134	0
565	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	20	2021-05-17 05:46:32.47976	2021-05-19 20:01:52.347714	20	145	0
566	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROD/3~rep_item~1C]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROD/5~rep_item~1E]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROD/3~rep_item~1C]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ROD/5~rep_item~1E]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	20	2021-05-16 04:40:01.241626	2021-05-19 20:02:04.855052	20	146	0
567	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2021-05-16 13:04:56.604498	20	2021-05-16 06:26:53.302574	2021-05-19 20:01:52.397656	20	146	0
568	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	20	2021-05-16 06:21:09.78404	2021-05-19 20:01:52.42439	20	146	0
569	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	20	2021-05-19 06:35:40.386553	2021-05-19 20:01:52.45459	20	147	0
570	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	20	2021-05-19 10:43:00.07846	2021-05-19 20:01:52.458137	20	147	0
571	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	20	2021-05-19 10:16:08.645618	2021-05-19 20:01:52.460637	20	147	0
572	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	20	2021-05-19 09:07:36.280886	2021-05-19 20:01:52.463188	20	147	0
573	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	t	2021-05-19 12:59:11.803539	20	2021-05-19 09:32:10.099442	2021-05-19 20:01:52.469335	20	147	0
574	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	20	2021-05-19 06:46:36.008508	2021-05-19 20:01:52.471045	20	147	0
575	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	20	2021-05-19 06:49:40.05048	2021-05-19 20:01:52.473617	20	147	0
576	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2021-05-19 16:28:22.618457	20	2021-05-19 06:34:30.535577	2021-05-19 20:01:52.482321	20	147	0
577	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	20	2021-05-19 08:07:54.87233	2021-05-19 20:01:52.483908	20	147	0
578	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-19 12:07:15.417356	20	2021-05-19 07:34:22.99902	2021-05-19 20:01:52.486433	20	147	0
579	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2021-05-19 11:38:49.943187	20	2021-05-19 09:59:35.824528	2021-05-19 20:01:52.49155	20	147	0
580	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	20	2021-05-19 07:50:23.232639	2021-05-19 20:01:52.493282	20	147	0
581	Before you start	Follow the guidelines bellow.	0	f	\N	20	2021-05-19 17:26:11.933338	2021-05-19 20:22:18.100547	20	148	0
582	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	20	2021-05-19 19:07:15.901352	2021-05-19 20:01:52.537096	20	148	0
583	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	20	2021-05-19 16:44:39.316001	2021-05-19 20:01:52.540715	20	148	0
584	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	20	2021-05-19 17:31:14.007388	2021-05-19 20:01:52.67548	20	148	0
585	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	20	2021-05-19 16:50:05.718362	2021-05-19 20:01:52.594518	20	148	0
586	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	20	2021-05-19 16:17:00.84778	2021-05-19 20:01:52.598433	20	148	0
587	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	20	2021-05-19 17:39:07.517913	2021-05-19 20:01:52.60245	20	148	0
588	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2021-05-19 19:04:40.627283	20	2021-05-19 18:58:44.754336	2021-05-19 20:01:52.607979	20	148	0
589	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	20	2021-05-19 19:04:13.24678	2021-05-19 20:01:52.61056	20	148	0
590	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	20	2021-05-19 15:31:12.865532	2021-05-19 20:01:52.612866	20	148	0
591	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	20	2021-05-19 17:37:16.694627	2021-05-19 20:01:52.616436	20	148	0
592	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	20	2021-05-16 19:02:00.839793	2021-05-19 20:01:52.69839	20	149	0
593	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	20	2021-05-16 20:37:42.890491	2021-05-19 20:01:52.710031	20	149	0
594	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	20	2021-05-16 21:18:41.554265	2021-05-19 20:01:52.702705	20	149	0
595	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	20	2021-05-16 09:27:10.434229	2021-05-19 20:01:57.476649	20	150	0
596	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	20	2021-05-16 12:16:45.766532	2021-05-19 20:02:07.329746	20	150	0
597	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2021-05-16 11:44:20.424948	20	2021-05-16 10:13:43.681221	2021-05-19 20:02:08.013463	20	150	0
662	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	22	2021-05-15 17:37:45.572569	2021-05-20 14:12:57.64532	22	175	0
598	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	20	2021-05-17 10:12:23.617188	2021-05-19 20:02:09.764118	20	151	0
599	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2021-05-17 13:38:50.286199	20	2021-05-17 08:58:56.462008	2021-05-19 20:01:52.855989	20	151	0
600	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2021-05-17 10:45:18.939562	20	2021-05-17 09:45:43.313584	2021-05-19 20:01:52.828857	20	151	0
601	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	20	2021-05-17 08:24:37.335892	2021-05-19 20:01:52.830499	20	151	0
602	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	20	2021-05-17 11:11:19.712436	2021-05-19 20:22:18.907118	20	151	0
603	Excel results	Write results in excel file.	5	f	\N	20	2021-05-17 07:43:01.360292	2021-05-19 20:02:02.757264	20	151	0
604	Template for ddCq analysis	Sample information here.	0	f	\N	20	2021-05-19 07:13:16.85352	2021-05-19 20:02:11.221691	20	152	0
605	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	21	2021-05-20 02:49:16.99545	2021-05-20 14:12:30.914131	21	163	0
606	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FDL/3~rep_item~1H]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FDL/5~rep_item~1J]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FDL/3~rep_item~1H]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#FDL/5~rep_item~1J]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	21	2021-05-19 09:54:04.395339	2021-05-20 14:13:38.112708	21	164	0
607	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2021-05-19 12:42:21.105662	21	2021-05-19 10:45:42.265672	2021-05-20 14:12:30.938156	21	164	0
608	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	21	2021-05-19 09:16:46.42868	2021-05-20 14:12:30.959622	21	164	0
609	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	21	2021-05-19 15:19:16.275812	2021-05-20 14:12:30.985535	21	165	0
610	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2021-05-19 15:25:27.561701	21	2021-05-19 15:21:03.496478	2021-05-20 14:12:30.990565	21	165	0
611	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	21	2021-05-19 16:27:30.51267	2021-05-20 14:12:30.992541	21	165	0
612	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2021-05-19 15:13:59.02744	21	2021-05-19 13:43:11.100607	2021-05-20 14:12:30.997719	21	165	0
613	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	t	2021-05-19 13:13:17.59258	21	2021-05-19 12:57:16.998467	2021-05-20 14:12:31.002485	21	165	0
614	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2021-05-19 14:40:55.171886	21	2021-05-19 13:14:03.178714	2021-05-20 14:12:31.004257	21	165	0
615	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	21	2021-05-19 12:58:54.283773	2021-05-20 14:12:31.006827	21	165	0
616	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	21	2021-05-19 16:46:29.004608	2021-05-20 14:12:31.009435	21	165	0
617	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	21	2021-05-19 17:12:22.977424	2021-05-20 14:12:31.011874	21	165	0
618	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-20 00:31:33.762542	21	2021-05-19 14:38:46.434578	2021-05-20 14:12:31.016873	21	165	0
619	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2021-05-19 17:16:21.256387	21	2021-05-19 17:12:40.619878	2021-05-20 14:12:31.026679	21	165	0
620	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	t	2021-05-19 17:22:07.327458	21	2021-05-19 16:51:21.780129	2021-05-20 14:12:31.030912	21	165	0
622	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2021-05-15 04:12:31.175629	21	2021-05-15 03:49:58.614844	2021-05-20 14:12:31.067221	21	166	0
623	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	t	2021-05-15 12:01:27.152675	21	2021-05-15 02:13:21.091575	2021-05-20 14:12:31.07187	21	166	0
624	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2021-05-15 11:28:28.127014	21	2021-05-15 02:42:09.39792	2021-05-20 14:12:31.076714	21	166	0
625	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	21	2021-05-15 03:17:35.166247	2021-05-20 14:12:31.078396	21	166	0
626	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	21	2021-05-15 02:32:44.879754	2021-05-20 14:12:31.08094	21	166	0
627	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2021-05-15 10:09:54.27535	21	2021-05-15 04:44:12.26961	2021-05-20 14:12:31.086178	21	166	0
628	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	21	2021-05-15 02:11:37.957739	2021-05-20 14:12:31.151884	21	166	0
629	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2021-05-15 14:46:09.641103	21	2021-05-15 06:19:40.615086	2021-05-20 14:12:31.093108	21	166	0
630	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	21	2021-05-15 04:15:48.426555	2021-05-20 14:12:31.094909	21	166	0
653	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	22	2021-05-19 15:13:43.910742	2021-05-20 14:12:57.595393	22	174	0
631	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	21	2021-05-15 02:57:13.9391	2021-05-20 14:12:31.097632	21	166	0
632	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	21	2021-05-15 02:30:12.656608	2021-05-20 14:12:31.16747	21	167	0
633	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	21	2021-05-14 22:26:28.100066	2021-05-20 14:12:31.179428	21	167	0
634	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2021-05-15 01:17:40.482215	21	2021-05-15 00:34:32.489561	2021-05-20 14:12:31.174876	21	167	0
635	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	t	2021-05-15 08:26:01.534945	21	2021-05-15 06:04:52.611289	2021-05-20 14:13:31.333329	21	168	0
636	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	21	2021-05-15 01:44:14.111219	2021-05-20 14:13:39.92465	21	168	0
637	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2021-05-15 06:24:28.65712	21	2021-05-15 02:07:04.583715	2021-05-20 14:13:40.369264	21	168	0
702	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	23	2021-05-18 17:25:56.081043	2021-05-20 14:13:34.463523	23	184	0
638	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2021-05-16 10:19:13.05402	21	2021-05-16 06:33:45.482864	2021-05-20 14:13:41.600629	21	169	0
639	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	21	2021-05-16 07:14:05.396466	2021-05-20 14:12:31.338722	21	169	0
640	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	21	2021-05-16 08:16:23.400733	2021-05-20 14:12:31.306875	21	169	0
641	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	21	2021-05-16 05:58:25.027501	2021-05-20 14:12:31.309835	21	169	0
642	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	21	2021-05-16 04:48:08.747966	2021-05-20 14:34:28.624848	21	169	0
643	Excel results	Write results in excel file.	5	f	\N	21	2021-05-16 08:23:26.153573	2021-05-20 14:13:33.007955	21	169	0
644	Template for ddCq analysis	Sample information here.	0	f	\N	21	2021-05-16 12:08:31.031068	2021-05-20 14:13:42.489586	21	170	0
645	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	22	2021-05-16 16:59:29.185163	2021-05-20 14:12:57.53794	22	172	0
646	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZIO/3~rep_item~1M]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZIO/5~rep_item~1O]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZIO/3~rep_item~1M]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#ZIO/5~rep_item~1O]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	22	2021-05-17 18:49:38.775294	2021-05-20 14:13:43.302894	22	173	0
647	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	22	2021-05-17 18:46:09.333863	2021-05-20 14:12:57.547431	22	173	0
648	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	t	2021-05-18 04:59:07.723729	22	2021-05-17 22:14:11.090334	2021-05-20 14:12:57.569174	22	173	0
649	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	22	2021-05-19 18:07:29.110283	2021-05-20 14:12:57.583019	22	174	0
650	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2021-05-20 03:55:18.376972	22	2021-05-19 18:57:12.200862	2021-05-20 14:12:57.58792	22	174	0
651	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	22	2021-05-19 18:22:12.666433	2021-05-20 14:12:57.58935	22	174	0
652	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	22	2021-05-19 14:59:18.173309	2021-05-20 14:12:57.591677	22	174	0
654	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2021-05-19 23:30:07.05652	22	2021-05-19 15:44:32.147175	2021-05-20 14:12:57.601354	22	174	0
655	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	22	2021-05-19 15:48:30.461284	2021-05-20 14:12:57.602857	22	174	0
656	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	22	2021-05-19 18:54:33.460734	2021-05-20 14:12:57.605309	22	174	0
657	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	22	2021-05-19 15:04:09.19952	2021-05-20 14:12:57.607634	22	174	0
658	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-19 23:08:13.690466	22	2021-05-19 16:10:11.84034	2021-05-20 14:12:57.612555	22	174	0
659	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	22	2021-05-19 17:26:02.867307	2021-05-20 14:12:57.614026	22	174	0
660	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	22	2021-05-19 19:41:03.965478	2021-05-20 14:12:57.616391	22	174	0
661	Before you start	Follow the guidelines bellow.	0	f	\N	22	2021-05-15 16:56:33.111229	2021-05-20 14:34:28.866398	22	175	0
663	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	22	2021-05-15 20:28:19.600236	2021-05-20 14:12:57.647705	22	175	0
664	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2021-05-15 23:55:56.430528	22	2021-05-15 17:51:33.975562	2021-05-20 14:12:57.653565	22	175	0
665	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	22	2021-05-15 20:59:15.556741	2021-05-20 14:12:57.654951	22	175	0
666	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	22	2021-05-15 18:52:09.219823	2021-05-20 14:12:57.657319	22	175	0
667	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2021-05-15 19:45:52.557546	22	2021-05-15 17:58:02.465279	2021-05-20 14:12:57.659878	22	175	0
668	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	22	2021-05-15 19:22:46.02763	2021-05-20 14:12:57.704831	22	175	0
669	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	22	2021-05-15 18:11:12.490778	2021-05-20 14:12:57.665064	22	175	0
670	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	t	2021-05-15 22:47:29.293251	22	2021-05-15 20:26:37.3021	2021-05-20 14:12:57.669827	22	175	0
671	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	22	2021-05-15 17:10:19.419247	2021-05-20 14:12:57.671293	22	175	0
672	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	t	2021-05-20 13:39:47.268431	22	2021-05-20 13:30:01.34186	2021-05-20 14:12:57.721254	22	176	0
673	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	22	2021-05-20 12:49:42.482464	2021-05-20 14:12:57.729622	22	176	0
675	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	22	2021-05-17 01:54:00.699692	2021-05-20 14:13:35.053902	22	177	0
676	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	22	2021-05-17 00:21:19.477074	2021-05-20 14:13:48.633995	22	177	0
677	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	22	2021-05-17 03:50:13.825172	2021-05-20 14:13:49.129575	22	177	0
742	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	24	2021-05-15 19:32:25.865939	2021-05-20 14:21:21.017787	24	193	0
678	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	22	2021-05-19 01:05:36.979438	2021-05-20 14:13:50.399864	22	178	0
679	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	22	2021-05-18 23:07:57.320768	2021-05-20 14:12:57.867011	22	178	0
680	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	22	2021-05-19 03:00:00.761009	2021-05-20 14:12:57.836709	22	178	0
681	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	22	2021-05-18 23:13:38.610758	2021-05-20 14:12:57.839191	22	178	0
682	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	22	2021-05-19 02:18:03.252261	2021-05-20 14:34:29.386822	22	178	0
683	Excel results	Write results in excel file.	5	t	2021-05-19 07:10:22.862681	22	2021-05-18 22:35:40.950773	2021-05-20 14:13:36.716263	22	178	0
684	Template for ddCq analysis	Sample information here.	0	t	2021-05-18 12:11:58.781461	22	2021-05-18 09:42:51.535711	2021-05-20 14:13:51.311903	22	179	0
685	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	t	2021-05-17 12:39:44.893313	23	2021-05-17 11:35:15.791757	2021-05-20 14:13:34.337791	23	181	0
686	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#QEA/3~rep_item~1R]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#QEA/5~rep_item~1T]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#QEA/3~rep_item~1R]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#QEA/5~rep_item~1T]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	23	2021-05-16 06:56:35.152286	2021-05-20 14:13:52.117706	23	182	0
687	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	23	2021-05-16 09:55:49.854542	2021-05-20 14:13:34.351573	23	182	0
688	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	23	2021-05-16 09:46:43.467243	2021-05-20 14:13:34.370942	23	182	0
689	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	23	2021-05-15 08:17:21.530541	2021-05-20 14:13:34.410001	23	183	0
690	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	23	2021-05-15 08:27:05.850613	2021-05-20 14:13:34.412351	23	183	0
691	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	23	2021-05-15 06:53:19.057854	2021-05-20 14:13:34.415294	23	183	0
692	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	23	2021-05-15 09:21:23.200366	2021-05-20 14:13:34.417708	23	183	0
693	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	t	2021-05-15 17:11:06.121454	23	2021-05-15 09:51:59.212307	2021-05-20 14:13:34.422919	23	183	0
694	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	23	2021-05-15 08:40:39.842237	2021-05-20 14:13:34.424422	23	183	0
695	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2021-05-15 17:43:32.357381	23	2021-05-15 09:32:48.626075	2021-05-20 14:13:34.429596	23	183	0
696	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2021-05-15 13:51:54.210989	23	2021-05-15 07:00:41.470021	2021-05-20 14:13:34.43106	23	183	0
697	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	23	2021-05-15 09:31:14.26576	2021-05-20 14:13:34.433506	23	183	0
698	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-15 16:02:17.271525	23	2021-05-15 06:35:40.924195	2021-05-20 14:13:34.438262	23	183	0
699	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	23	2021-05-15 09:07:45.644576	2021-05-20 14:13:34.439665	23	183	0
700	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	23	2021-05-15 09:29:33.987318	2021-05-20 14:13:34.442326	23	183	0
701	Before you start	Follow the guidelines bellow.	0	f	\N	23	2021-05-18 17:35:30.772541	2021-05-20 14:34:30.259682	23	184	0
799	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	25	2021-05-14 18:40:08.569662	2021-05-20 14:30:31.061801	25	205	0
703	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	t	2021-05-19 01:32:15.087912	23	2021-05-18 16:27:55.808076	2021-05-20 14:13:34.468963	23	184	0
704	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	23	2021-05-18 15:57:02.118594	2021-05-20 14:13:34.523677	23	184	0
705	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	23	2021-05-18 17:30:36.201813	2021-05-20 14:13:34.473149	23	184	0
706	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	23	2021-05-18 15:11:18.903221	2021-05-20 14:13:34.475773	23	184	0
707	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	23	2021-05-18 14:58:16.074096	2021-05-20 14:13:34.478081	23	184	0
708	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	23	2021-05-18 17:13:59.787423	2021-05-20 14:13:34.531543	23	184	0
709	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2021-05-18 20:07:36.486485	23	2021-05-18 15:47:14.3552	2021-05-20 14:13:34.485582	23	184	0
710	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	23	2021-05-18 15:19:30.667485	2021-05-20 14:13:34.487123	23	184	0
711	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2021-05-18 20:15:47.247995	23	2021-05-18 16:39:21.536585	2021-05-20 14:13:34.492071	23	184	0
712	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	23	2021-05-18 09:45:11.447572	2021-05-20 14:13:34.545962	23	185	0
713	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	23	2021-05-18 13:24:18.800115	2021-05-20 14:13:34.560049	23	185	0
714	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	23	2021-05-18 14:15:23.809849	2021-05-20 14:13:34.552013	23	185	0
715	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	23	2021-05-15 23:06:18.803622	2021-05-20 14:13:45.088646	23	186	0
716	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2021-05-16 05:53:49.601809	23	2021-05-16 01:51:54.265691	2021-05-20 14:13:54.02042	23	186	0
717	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	23	2021-05-15 23:08:49.338576	2021-05-20 14:13:54.485042	23	186	0
800	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	25	2021-05-14 20:48:25.17121	2021-05-20 14:30:31.021234	25	205	0
801	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	25	2021-05-14 18:42:21.322142	2021-05-20 14:30:31.023715	25	205	0
718	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2021-05-19 20:24:28.648823	23	2021-05-19 11:33:06.966295	2021-05-20 14:13:55.903691	23	187	0
719	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2021-05-19 11:07:19.712325	23	2021-05-19 11:05:13.721222	2021-05-20 14:13:34.688442	23	187	0
720	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	23	2021-05-19 12:23:25.308208	2021-05-20 14:13:34.689981	23	187	0
721	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	23	2021-05-19 07:29:02.089346	2021-05-20 14:13:34.693803	23	187	0
722	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	t	2021-05-19 13:04:18.174636	23	2021-05-19 11:49:30.163457	2021-05-20 14:34:30.573902	23	187	0
723	Excel results	Write results in excel file.	5	f	\N	23	2021-05-19 11:49:28.996786	2021-05-20 14:13:46.819481	23	187	0
724	Template for ddCq analysis	Sample information here.	0	t	2021-05-19 07:30:47.651272	23	2021-05-19 04:08:27.850461	2021-05-20 14:13:56.820306	23	188	0
725	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	24	2021-05-20 05:12:21.466071	2021-05-20 14:21:20.907787	24	190	0
726	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#VRK/3~rep_item~1W]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#VRK/5~rep_item~1Y]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#VRK/3~rep_item~1W]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#VRK/5~rep_item~1Y]</span>.\n          </p>\n        </body>\n      </html>	0	t	2021-05-17 04:52:55.361123	24	2021-05-16 20:39:41.123784	2021-05-20 14:22:07.93939	24	191	0
727	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	24	2021-05-16 18:09:59.200844	2021-05-20 14:21:20.92007	24	191	0
728	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	24	2021-05-16 21:40:48.115316	2021-05-20 14:21:20.943143	24	191	0
729	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	24	2021-05-14 16:02:42.190068	2021-05-20 14:21:20.95883	24	192	0
730	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	24	2021-05-14 18:05:01.851508	2021-05-20 14:21:20.961378	24	192	0
731	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	24	2021-05-14 14:54:58.575725	2021-05-20 14:21:20.963705	24	192	0
732	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2021-05-14 23:21:01.195349	24	2021-05-14 14:47:50.83869	2021-05-20 14:21:20.969875	24	192	0
733	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	24	2021-05-14 16:27:29.884283	2021-05-20 14:21:20.971853	24	192	0
734	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	24	2021-05-14 18:47:06.736351	2021-05-20 14:21:20.974698	24	192	0
735	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	24	2021-05-14 17:56:39.755263	2021-05-20 14:21:20.977383	24	192	0
736	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	24	2021-05-14 18:00:20.824051	2021-05-20 14:21:20.979722	24	192	0
737	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2021-05-14 19:32:12.895178	24	2021-05-14 14:55:39.311772	2021-05-20 14:21:20.983012	24	192	0
738	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	24	2021-05-14 16:21:49.383968	2021-05-20 14:21:20.985627	24	192	0
739	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	24	2021-05-14 15:36:47.541161	2021-05-20 14:21:20.987887	24	192	0
740	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	t	2021-05-14 22:25:53.849126	24	2021-05-14 15:47:47.638288	2021-05-20 14:21:20.993438	24	192	0
741	Before you start	Follow the guidelines bellow.	0	f	\N	24	2021-05-15 21:36:48.853071	2021-05-20 14:42:32.844443	24	193	0
802	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	25	2021-05-14 18:31:09.334495	2021-05-20 14:51:35.174293	25	205	0
803	Excel results	Write results in excel file.	5	f	\N	25	2021-05-14 18:58:45.147657	2021-05-20 14:31:18.344149	25	205	0
804	Template for ddCq analysis	Sample information here.	0	f	\N	25	2021-05-18 08:00:03.846982	2021-05-20 14:31:24.767805	25	206	0
743	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	t	2021-05-16 03:58:47.937018	24	2021-05-15 23:21:40.958315	2021-05-20 14:21:21.021378	24	193	0
744	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2021-05-16 01:52:39.738574	24	2021-05-15 21:46:06.864942	2021-05-20 14:21:21.026994	24	193	0
745	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	24	2021-05-15 19:45:53.729924	2021-05-20 14:21:21.029275	24	193	0
746	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	24	2021-05-15 23:12:36.743832	2021-05-20 14:21:21.032033	24	193	0
747	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	24	2021-05-15 21:42:13.38502	2021-05-20 14:21:21.034698	24	193	0
748	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	24	2021-05-15 21:21:20.920667	2021-05-20 14:21:21.085019	24	193	0
749	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	24	2021-05-15 20:26:28.975006	2021-05-20 14:21:21.041314	24	193	0
750	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	t	2021-05-15 22:05:49.388181	24	2021-05-15 20:53:25.805655	2021-05-20 14:21:21.046653	24	193	0
751	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	24	2021-05-15 19:36:53.29268	2021-05-20 14:21:21.048625	24	193	0
752	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	24	2021-05-19 21:09:13.374471	2021-05-20 14:21:21.100208	24	194	0
753	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	24	2021-05-19 17:56:29.634739	2021-05-20 14:21:21.110771	24	194	0
754	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	24	2021-05-19 21:25:59.772068	2021-05-20 14:21:21.104679	24	194	0
755	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	24	2021-05-15 19:31:43.622258	2021-05-20 14:22:03.74858	24	195	0
756	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2021-05-16 03:14:33.139342	24	2021-05-15 20:30:37.17353	2021-05-20 14:22:09.771257	24	195	0
757	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2021-05-16 01:20:25.79026	24	2021-05-15 16:11:56.581233	2021-05-20 14:22:10.242656	24	195	0
806	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	26	2021-05-16 04:04:55.012399	2021-05-20 15:17:12.068707	26	209	0
827	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	26	2021-05-18 08:04:13.106781	2021-05-20 15:17:12.18618	26	212	0
758	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	24	2021-05-18 22:31:48.072302	2021-05-20 14:22:11.521511	24	196	0
759	QA Checklist	Please perform the following checklist before and after you start working.	1	t	2021-05-19 03:28:09.779477	24	2021-05-19 00:15:05.255876	2021-05-20 14:21:21.241107	24	196	0
760	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	24	2021-05-18 22:24:31.576667	2021-05-20 14:21:21.242976	24	196	0
761	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	24	2021-05-18 22:30:50.700162	2021-05-20 14:21:21.247761	24	196	0
762	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	24	2021-05-18 22:58:57.284285	2021-05-20 14:42:33.428436	24	196	0
763	Excel results	Write results in excel file.	5	f	\N	24	2021-05-18 20:15:44.60612	2021-05-20 14:22:05.471781	24	196	0
764	Template for ddCq analysis	Sample information here.	0	f	\N	24	2021-05-19 01:15:50.760924	2021-05-20 14:22:12.429733	24	197	0
765	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	t	2021-05-19 10:38:07.825984	25	2021-05-19 06:52:17.69985	2021-05-20 14:30:30.640502	25	199	0
766	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#YKK/3~rep_item~1b]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#YKK/5~rep_item~1d]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#YKK/3~rep_item~1b]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#YKK/5~rep_item~1d]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	25	2021-05-19 10:52:44.920508	2021-05-20 14:31:20.220219	25	200	0
768	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	25	2021-05-19 12:34:36.619348	2021-05-20 14:30:30.678927	25	200	0
769	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	t	2021-05-17 01:56:56.000579	25	2021-05-16 20:15:27.95729	2021-05-20 14:30:30.693717	25	201	0
770	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	25	2021-05-16 20:10:58.267417	2021-05-20 14:30:30.696279	25	201	0
771	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2021-05-16 16:56:18.679341	25	2021-05-16 16:05:55.261587	2021-05-20 14:30:30.701249	25	201	0
772	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	25	2021-05-16 16:46:03.676856	2021-05-20 14:30:30.703034	25	201	0
773	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	25	2021-05-16 17:16:07.940352	2021-05-20 14:30:30.708315	25	201	0
774	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	25	2021-05-16 16:42:18.151096	2021-05-20 14:30:30.712261	25	201	0
775	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2021-05-16 22:09:14.413181	25	2021-05-16 20:32:35.538554	2021-05-20 14:30:30.719475	25	201	0
776	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2021-05-16 19:05:55.148008	25	2021-05-16 19:00:43.072615	2021-05-20 14:30:30.725673	25	201	0
777	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	25	2021-05-16 18:56:05.432446	2021-05-20 14:30:30.728212	25	201	0
778	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-17 03:44:58.449918	25	2021-05-16 20:33:07.120095	2021-05-20 14:30:30.732854	25	201	0
779	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2021-05-16 20:16:30.335737	25	2021-05-16 19:26:56.903835	2021-05-20 14:30:30.738661	25	201	0
780	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	25	2021-05-16 18:03:52.215116	2021-05-20 14:30:30.741489	25	201	0
781	Before you start	Follow the guidelines bellow.	0	t	2021-05-15 04:50:50.462123	25	2021-05-15 00:14:05.22959	2021-05-20 14:51:34.89621	25	202	0
782	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	25	2021-05-15 01:29:02.788339	2021-05-20 14:30:30.776522	25	202	0
783	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	25	2021-05-15 00:30:59.52844	2021-05-20 14:30:30.780825	25	202	0
784	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	25	2021-05-15 01:33:47.270079	2021-05-20 14:30:30.844565	25	202	0
785	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2021-05-14 23:40:32.250727	25	2021-05-14 21:36:37.500579	2021-05-20 14:30:30.7929	25	202	0
786	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	25	2021-05-14 21:41:30.024569	2021-05-20 14:30:30.794764	25	202	0
787	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2021-05-15 00:13:30.868681	25	2021-05-14 22:42:47.957742	2021-05-20 14:30:30.800521	25	202	0
788	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2021-05-15 04:12:03.634789	25	2021-05-14 22:43:01.261505	2021-05-20 14:30:30.80495	25	202	0
789	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	25	2021-05-14 23:14:51.289803	2021-05-20 14:30:30.806807	25	202	0
790	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	25	2021-05-15 00:58:35.190792	2021-05-20 14:30:30.810037	25	202	0
791	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	25	2021-05-15 01:58:11.10601	2021-05-20 14:30:30.81485	25	202	0
792	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	25	2021-05-18 12:39:13.707586	2021-05-20 14:30:30.867872	25	203	0
794	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2021-05-18 12:34:51.069144	25	2021-05-18 11:59:34.50852	2021-05-20 14:30:30.879138	25	203	0
795	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	t	2021-05-18 06:27:19.495128	25	2021-05-17 22:55:40.083401	2021-05-20 14:31:16.601241	25	204	0
796	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	25	2021-05-18 01:46:34.512903	2021-05-20 14:31:22.058118	25	204	0
797	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2021-05-18 08:34:46.221963	25	2021-05-17 23:55:49.647311	2021-05-20 14:31:22.525103	25	204	0
798	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	t	2021-05-14 23:56:13.506597	25	2021-05-14 18:46:34.294427	2021-05-20 14:31:23.828157	25	205	0
807	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MQU/3~rep_item~1g]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MQU/5~rep_item~1i]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MQU/3~rep_item~1g]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#MQU/5~rep_item~1i]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	26	2021-05-15 08:12:15.720506	2021-05-20 15:17:43.110552	26	210	0
808	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2021-05-15 18:19:50.229275	26	2021-05-15 08:58:50.211229	2021-05-20 15:17:12.08122	26	210	0
809	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	26	2021-05-15 06:01:08.402787	2021-05-20 15:17:12.103882	26	210	0
810	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	26	2021-05-15 13:44:25.560085	2021-05-20 15:17:12.118189	26	211	0
811	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2021-05-15 16:21:09.406139	26	2021-05-15 12:06:19.899903	2021-05-20 15:17:12.123652	26	211	0
812	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	26	2021-05-15 13:36:10.715291	2021-05-20 15:17:12.125328	26	211	0
813	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	26	2021-05-15 09:20:04.776055	2021-05-20 15:17:12.127983	26	211	0
814	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	26	2021-05-15 12:47:22.906556	2021-05-20 15:17:12.130256	26	211	0
815	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	f	\N	26	2021-05-15 13:05:59.672553	2021-05-20 15:17:12.132511	26	211	0
816	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2021-05-15 15:40:04.285462	26	2021-05-15 10:49:19.852837	2021-05-20 15:17:12.138078	26	211	0
817	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	26	2021-05-15 13:41:02.042197	2021-05-20 15:17:12.139495	26	211	0
818	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	26	2021-05-15 09:17:20.293694	2021-05-20 15:17:12.143672	26	211	0
819	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	26	2021-05-15 10:05:28.661755	2021-05-20 15:17:12.146108	26	211	0
820	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	26	2021-05-15 09:26:09.429942	2021-05-20 15:17:12.148629	26	211	0
821	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	26	2021-05-15 11:54:14.945496	2021-05-20 15:17:12.151135	26	211	0
822	Before you start	Follow the guidelines bellow.	0	f	\N	26	2021-05-18 06:05:47.609349	2021-05-20 15:37:54.114638	26	212	0
823	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	26	2021-05-18 08:04:26.380128	2021-05-20 15:17:12.174853	26	212	0
841	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	26	2021-05-14 21:51:14.926604	2021-05-20 15:17:12.393799	26	215	0
842	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	26	2021-05-14 22:37:08.496965	2021-05-20 15:17:12.396288	26	215	0
824	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	26	2021-05-18 05:11:12.760469	2021-05-20 15:17:12.178276	26	212	0
825	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	26	2021-05-18 04:13:43.141932	2021-05-20 15:17:12.236873	26	212	0
826	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	26	2021-05-18 06:03:58.914014	2021-05-20 15:17:12.183697	26	212	0
828	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2021-05-18 06:24:45.057509	26	2021-05-18 04:56:09.800748	2021-05-20 15:17:12.190995	26	212	0
829	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	26	2021-05-18 08:04:36.476943	2021-05-20 15:17:12.242924	26	212	0
830	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2021-05-18 07:06:54.143356	26	2021-05-18 05:48:20.319323	2021-05-20 15:17:12.198608	26	212	0
831	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	26	2021-05-18 06:12:32.492451	2021-05-20 15:17:12.200262	26	212	0
832	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	26	2021-05-18 08:09:01.272897	2021-05-20 15:17:12.204084	26	212	0
833	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	26	2021-05-18 05:31:03.913158	2021-05-20 15:17:12.259173	26	213	0
834	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	t	2021-05-18 08:27:56.841033	26	2021-05-18 06:27:00.508552	2021-05-20 15:17:12.264584	26	213	0
835	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2021-05-18 12:20:21.070682	26	2021-05-18 08:28:28.848399	2021-05-20 15:17:12.26923	26	213	0
836	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	26	2021-05-17 12:52:41.129917	2021-05-20 15:17:39.94874	26	214	0
837	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	t	2021-05-17 20:13:39.785693	26	2021-05-17 12:43:06.267423	2021-05-20 15:17:45.003267	26	214	0
838	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2021-05-18 00:44:33.146325	26	2021-05-17 15:27:14.429407	2021-05-20 15:17:45.509962	26	214	0
839	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	26	2021-05-15 00:21:42.206267	2021-05-20 15:17:46.786966	26	215	0
840	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	26	2021-05-15 00:02:20.256331	2021-05-20 15:17:12.424063	26	215	0
851	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	27	2021-05-25 20:43:29.280507	2021-05-28 15:07:53.003302	27	224	0
852	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#OJT/3~rep_item~1l]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#OJT/5~rep_item~1n]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#OJT/3~rep_item~1l]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#OJT/5~rep_item~1n]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	27	2021-05-22 19:56:10.241346	2021-05-28 15:08:50.993442	27	225	0
853	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2021-05-23 00:55:44.602358	27	2021-05-22 17:10:53.219713	2021-05-28 15:07:53.08703	27	225	0
868	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	27	2021-05-27 22:27:20.811452	2021-05-28 15:07:55.175216	27	227	0
869	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	27	2021-05-28 00:59:50.20714	2021-05-28 15:07:55.18235	27	227	0
870	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	27	2021-05-27 22:15:33.392144	2021-05-28 15:07:55.298	27	227	0
871	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	27	2021-05-28 00:49:44.083368	2021-05-28 15:07:55.192674	27	227	0
872	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	f	\N	27	2021-05-28 01:58:51.886316	2021-05-28 15:07:55.196989	27	227	0
873	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	27	2021-05-27 22:43:52.766025	2021-05-28 15:07:55.201877	27	227	0
874	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2021-05-28 05:43:25.901968	27	2021-05-27 23:47:20.648623	2021-05-28 15:07:55.210272	27	227	0
875	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2021-05-27 23:28:22.840955	27	2021-05-27 22:38:11.56758	2021-05-28 15:07:55.216204	27	227	0
876	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	t	2021-05-28 09:23:53.659294	27	2021-05-27 23:45:57.946281	2021-05-28 15:07:55.224309	27	227	0
877	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	27	2021-05-27 23:03:53.625974	2021-05-28 15:07:55.22626	27	227	0
878	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	27	2021-05-26 14:00:50.07971	2021-05-28 15:07:55.330955	27	228	0
879	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	27	2021-05-26 16:40:16.660616	2021-05-28 15:07:55.345009	27	228	0
880	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2021-05-26 16:02:05.868815	27	2021-05-26 15:35:33.371534	2021-05-28 15:07:55.337637	27	228	0
881	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	27	2021-05-25 01:24:14.272375	2021-05-28 15:08:29.329034	27	229	0
882	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	27	2021-05-24 23:55:56.998322	2021-05-28 15:08:58.555698	27	229	0
883	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	27	2021-05-25 01:09:34.392592	2021-05-28 15:08:59.735131	27	229	0
884	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	27	2021-05-22 20:10:25.330959	2021-05-28 15:09:04.728558	27	230	0
885	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	27	2021-05-22 21:31:48.414933	2021-05-28 15:07:58.0745	27	230	0
886	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	27	2021-05-22 20:20:10.526877	2021-05-28 15:07:57.938945	27	230	0
887	Eliminate results that have positive NTCs	And repeat procedure.	3	t	2021-05-23 08:01:21.541306	27	2021-05-22 22:08:02.445505	2021-05-28 15:07:58.043913	27	230	0
888	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	27	2021-05-22 23:28:14.559344	2021-05-28 15:29:53.432554	27	230	0
889	Excel results	Write results in excel file.	5	f	\N	27	2021-05-22 22:13:40.733129	2021-05-28 15:08:40.477808	27	230	0
890	Template for ddCq analysis	Sample information here.	0	f	\N	27	2021-05-24 13:11:36.337009	2021-05-28 15:09:06.44038	27	231	0
895	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	28	2021-06-02 11:23:39.679756	2021-06-02 16:10:03.993165	28	233	0
896	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#AUG/3~rep_item~1q]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#AUG/5~rep_item~1s]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#AUG/3~rep_item~1q]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#AUG/5~rep_item~1s]</span>.\n          </p>\n        </body>\n      </html>	0	t	2021-05-30 16:37:26.148077	28	2021-05-30 15:28:56.89361	2021-06-02 16:10:45.339011	28	234	0
897	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	t	2021-05-31 00:55:45.035813	28	2021-05-30 15:50:26.05407	2021-06-02 16:10:04.113661	28	234	0
898	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	t	2021-05-30 16:25:14.956302	28	2021-05-30 12:18:14.70551	2021-06-02 16:10:04.196057	28	234	0
899	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	28	2021-05-31 19:36:42.804864	2021-06-02 16:10:04.492513	28	235	0
900	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	28	2021-05-31 19:06:14.337127	2021-06-02 16:10:04.50129	28	235	0
901	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	t	2021-06-01 03:50:48.051361	28	2021-05-31 21:01:06.660457	2021-06-02 16:10:04.521896	28	235	0
902	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	28	2021-05-31 18:20:08.368292	2021-06-02 16:10:04.529612	28	235	0
903	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	t	2021-06-01 04:38:32.02547	28	2021-05-31 19:32:36.108543	2021-06-02 16:10:04.554525	28	235	0
904	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2021-06-01 03:23:54.100123	28	2021-05-31 18:23:24.528306	2021-06-02 16:10:04.575375	28	235	0
905	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2021-05-31 19:49:36.829079	28	2021-05-31 16:23:15.006727	2021-06-02 16:10:04.590411	28	235	0
906	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2021-06-01 00:24:22.070578	28	2021-05-31 20:14:49.598631	2021-06-02 16:10:04.595553	28	235	0
907	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	28	2021-05-31 20:00:52.270125	2021-06-02 16:10:04.604707	28	235	0
908	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	t	2021-05-31 23:23:00.992517	28	2021-05-31 18:25:02.785407	2021-06-02 16:10:04.625683	28	235	0
909	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2021-06-01 03:47:59.771214	28	2021-05-31 19:53:20.442668	2021-06-02 16:10:04.646293	28	235	0
910	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	28	2021-05-31 19:37:54.195922	2021-06-02 16:10:04.650388	28	235	0
911	Before you start	Follow the guidelines bellow.	0	f	\N	28	2021-05-31 13:52:09.545738	2021-06-02 16:31:00.0513	28	236	0
912	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	28	2021-05-31 15:59:07.01103	2021-06-02 16:10:04.763125	28	236	0
913	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	28	2021-05-31 13:11:35.11885	2021-06-02 16:10:04.79802	28	236	0
914	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2021-05-31 18:46:12.795228	28	2021-05-31 15:59:04.200037	2021-06-02 16:10:04.822852	28	236	0
915	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	f	\N	28	2021-05-31 14:53:02.629866	2021-06-02 16:10:04.829771	28	236	0
916	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	t	2021-06-01 02:28:23.39865	28	2021-05-31 17:34:53.927518	2021-06-02 16:10:04.846541	28	236	0
917	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	28	2021-05-31 14:59:31.691689	2021-06-02 16:10:04.863032	28	236	0
918	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2021-05-31 22:53:19.447006	28	2021-05-31 14:44:50.11268	2021-06-02 16:10:04.89071	28	236	0
919	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	t	2021-05-31 20:54:32.460842	28	2021-05-31 16:17:19.238812	2021-06-02 16:10:04.908432	28	236	0
920	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	28	2021-05-31 15:53:24.251027	2021-06-02 16:10:04.925272	28	236	0
921	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2021-05-31 22:08:16.109103	28	2021-05-31 13:49:51.97047	2021-06-02 16:10:04.954324	28	236	0
922	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	28	2021-05-31 20:39:37.654044	2021-06-02 16:10:06.519932	28	237	0
923	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	28	2021-05-31 19:35:02.622937	2021-06-02 16:10:06.564723	28	237	0
924	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	28	2021-05-31 21:19:38.806999	2021-06-02 16:10:06.546139	28	237	0
925	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	28	2021-05-28 11:47:39.576418	2021-06-02 16:10:40.095139	28	238	0
926	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	28	2021-05-28 14:18:23.904154	2021-06-02 16:10:48.493062	28	238	0
927	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	t	2021-05-28 22:14:24.244843	28	2021-05-28 13:27:28.994932	2021-06-02 16:10:49.320378	28	238	0
928	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	28	2021-06-02 15:14:29.83347	2021-06-02 16:10:51.5994	28	239	0
929	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	28	2021-06-02 13:33:17.051461	2021-06-02 16:10:07.392827	28	239	0
930	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	f	\N	28	2021-06-02 15:56:15.334514	2021-06-02 16:10:07.176355	28	239	0
931	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	28	2021-06-02 13:31:26.617136	2021-06-02 16:10:07.187843	28	239	0
932	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	28	2021-06-02 13:08:27.944941	2021-06-02 16:31:02.096557	28	239	0
933	Excel results	Write results in excel file.	5	t	2021-06-02 15:41:13.170206	28	2021-06-02 15:38:50.763544	2021-06-02 16:10:42.903038	28	239	0
934	Template for ddCq analysis	Sample information here.	0	t	2021-05-30 07:19:50.88877	28	2021-05-30 01:34:52.14401	2021-06-02 16:10:52.869145	28	240	0
935	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	30	2021-08-21 09:39:13.482163	2021-08-23 14:58:28.35495	30	255	0
936	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#CKC/3~rep_item~1v]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#CKC/5~rep_item~1x]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#CKC/3~rep_item~1v]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#CKC/5~rep_item~1x]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	30	2021-08-19 17:55:06.726809	2021-08-23 14:59:01.556783	30	256	0
937	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	30	2021-08-19 16:42:53.124539	2021-08-23 14:58:28.397232	30	256	0
938	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	30	2021-08-19 14:26:45.777982	2021-08-23 14:58:28.426248	30	256	0
939	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	30	2021-08-21 17:27:42.536014	2021-08-23 14:58:28.464445	30	257	0
940	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	f	\N	30	2021-08-21 17:53:38.533368	2021-08-23 14:58:28.469029	30	257	0
941	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	30	2021-08-21 17:09:44.906	2021-08-23 14:58:28.473921	30	257	0
942	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	t	2021-08-21 18:58:26.272393	30	2021-08-21 15:33:23.407856	2021-08-23 14:58:28.479976	30	257	0
943	Buffer addition	Add 450 μL Buffer RLT or Buffer RLC to a maximum of 100 mg tissue powder. Vortex vigorously.	4	f	\N	30	2021-08-21 14:29:27.545268	2021-08-23 14:58:28.482667	30	257	0
944	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2021-08-22 01:30:56.786556	30	2021-08-21 17:56:14.544536	2021-08-23 14:58:28.488036	30	257	0
945	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	t	2021-08-21 15:21:33.664491	30	2021-08-21 13:21:51.539295	2021-08-23 14:58:28.493654	30	257	0
946	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	t	2021-08-21 21:23:46.645963	30	2021-08-21 13:31:39.592763	2021-08-23 14:58:28.498193	30	257	0
947	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	t	2021-08-21 23:59:36.315472	30	2021-08-21 14:15:08.220291	2021-08-23 14:58:28.503133	30	257	0
948	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	30	2021-08-21 13:06:49.456982	2021-08-23 14:58:28.505365	30	257	0
949	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	t	2021-08-21 23:11:55.503946	30	2021-08-21 13:43:20.319679	2021-08-23 14:58:28.508032	30	257	0
950	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	30	2021-08-21 16:15:11.638568	2021-08-23 14:58:28.511427	30	257	0
951	Before you start	Follow the guidelines bellow.	0	f	\N	30	2021-08-19 04:18:53.982914	2021-08-23 15:19:13.44216	30	258	0
952	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	t	2021-08-19 14:32:35.408202	30	2021-08-19 05:37:47.37945	2021-08-23 14:58:28.550708	30	258	0
953	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	30	2021-08-19 04:26:18.092824	2021-08-23 14:58:28.552498	30	258	0
954	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	f	\N	30	2021-08-19 07:49:14.033595	2021-08-23 14:58:28.677245	30	258	0
955	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2021-08-19 15:24:56.039302	30	2021-08-19 06:03:29.516739	2021-08-23 14:58:28.561784	30	258	0
956	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	t	2021-08-19 07:15:00.871188	30	2021-08-19 04:00:29.06336	2021-08-23 14:58:28.563832	30	258	0
957	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	t	2021-08-19 09:18:30.791689	30	2021-08-19 03:58:06.120418	2021-08-23 14:58:28.569567	30	258	0
958	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	f	\N	30	2021-08-19 03:42:57.835039	2021-08-23 14:58:28.683266	30	258	0
959	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	30	2021-08-19 03:28:16.655548	2021-08-23 14:58:28.574024	30	258	0
960	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	30	2021-08-19 05:42:08.815118	2021-08-23 14:58:28.61096	30	258	0
961	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	f	\N	30	2021-08-19 07:50:39.986419	2021-08-23 14:58:28.616126	30	258	0
962	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	30	2021-08-21 07:27:22.097059	2021-08-23 14:58:28.70093	30	259	0
963	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	f	\N	30	2021-08-21 07:33:29.13284	2021-08-23 14:58:28.711747	30	259	0
964	RT reaction	25°C for 10 min 37°C for 2 h	2	t	2021-08-21 06:37:26.663647	30	2021-08-21 05:46:56.173598	2021-08-23 14:58:28.706397	30	259	0
965	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	f	\N	30	2021-08-20 12:27:01.070628	2021-08-23 14:58:57.491513	30	260	0
966	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	30	2021-08-20 12:24:01.017077	2021-08-23 14:59:03.837535	30	260	0
967	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	30	2021-08-20 11:20:14.951337	2021-08-23 14:59:04.366639	30	260	0
968	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	30	2021-08-20 23:03:57.207231	2021-08-23 14:59:05.846972	30	261	0
969	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	30	2021-08-21 00:35:15.275559	2021-08-23 14:58:28.862807	30	261	0
970	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2021-08-21 10:19:54.747837	30	2021-08-21 02:41:11.465968	2021-08-23 14:58:28.831264	30	261	0
971	Eliminate results that have positive NTCs	And repeat procedure.	3	t	2021-08-20 23:32:35.036053	30	2021-08-20 22:45:02.085841	2021-08-23 14:58:28.836456	30	261	0
972	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	30	2021-08-21 02:48:55.313376	2021-08-23 15:19:14.130356	30	261	0
973	Excel results	Write results in excel file.	5	f	\N	30	2021-08-21 00:38:30.478324	2021-08-23 14:58:59.82461	30	261	0
974	Template for ddCq analysis	Sample information here.	0	f	\N	30	2021-08-21 14:19:19.867695	2021-08-23 14:59:06.857863	30	262	0
975	Gene expression	Compare response of PVYNTN, cab4 and PR1 genes in mock/virus inoculated potatoes & in time	0	f	\N	31	2021-08-20 03:03:18.661432	2021-08-23 17:19:39.588215	31	264	0
976	Inoculation of potatoes	<html>\n        <body>\n          <p>50% of samples should be mock inoculated\n          <span class=\\"atwho-inserted\\"contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#NXT/3~rep_item~20]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#NXT/5~rep_item~22]</span>\n          while other 50% with PVY NTN virus\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#NXT/3~rep_item~20]</span>\n          <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"\n            data-atwho-at-query=\\"#\\">[#NXT/5~rep_item~22]</span>.\n          </p>\n        </body>\n      </html>	0	f	\N	31	2021-08-19 06:59:38.552724	2021-08-23 17:20:20.272622	31	265	0
977	Store samples	Collect samples in <strong>2ml tubes</strong> and put them in <strong>liquid nitrogen</strong> and store at <strong>80°C</strong>.	1	f	\N	31	2021-08-19 03:32:45.164577	2021-08-23 17:19:39.599398	31	265	0
978	Collection of potatoes	50% of PVYNTN inoculated potatos and 50% of Mock inoculated potatos collect 1 day post inocullation while other halph of samples collect 6 days post inoculation.	2	f	\N	31	2021-08-19 03:51:23.930111	2021-08-23 17:19:39.625609	31	265	0
979	Homogenization of the material	Use tissue lyser: 1 min on step 3.	0	f	\N	31	2021-08-21 20:19:37.929977	2021-08-23 17:19:39.644303	31	266	0
980	Isolation of RNA with RNeasy plant mini kit	Disrupt a maximum of 100 mg plant material according to step 3 or 4.	1	t	2021-08-21 20:39:08.136364	31	2021-08-21 19:30:55.472568	2021-08-23 17:19:39.647455	31	266	0
981	Disruption with mortar and pestle	Immediately place tissue in liquid nitrogen. Grind thoroughly. Decant tissue powder and liquid nitrogen into RNase-free, liquid-nitrogen–cooled, 2 mL microcentrifuge tube (not supplied). Allow the liquid nitrogen to evaporate, but do not allow the tissue to thaw. Proceed immediately to step 5.	2	f	\N	31	2021-08-21 23:20:53.415632	2021-08-23 17:19:39.651929	31	266	0
982	Disruption	<html><body><p>Disruption using the TissueLyser II, TissueLyser LT, or TissueRuptor.<br><br>For detailed information on disruption of plant tissues for purification of RNA, see TissueLyser Handbook, TissueLyser LT Handbook, or TissueRuptor Handbook. (The RNeasy Mini Handbook will be updated with this option.)</p></body></html>	3	f	\N	31	2021-08-21 21:12:50.171599	2021-08-23 17:19:39.65576	31	266	0
984	Transfer the lysate to a QIAshredder spin column	Transfer the lysate to a QIAshredder spin column placed in a 2 mL collection tube. Centrifuge for 2 min at full speed. Transfer the supernatant of the flow-through to a new microcentrifuge tube (not supplied) without disturbing the cell-debris pellet.	5	t	2021-08-22 06:53:50.098767	31	2021-08-21 22:20:10.582189	2021-08-23 17:19:39.666903	31	266	0
985	Addition of ethanol	Add 0.5 volume of ethanol (96–100%) to the cleared lysate, and mix immediately by pipetting. Do not centrifuge.	6	f	\N	31	2021-08-21 18:48:10.006199	2021-08-23 17:19:39.668726	31	266	0
986	Transfer the sample	Transfer the sample (usually 650 μl), with any precipitate, to an RNeasy Mini spin column (pink) in a 2 mL collection tube (supplied). Close the lid, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flowthrough.	7	f	\N	31	2021-08-21 22:55:38.605044	2021-08-23 17:19:39.671311	31	266	0
987	Add 700 μL Buffer RW1 to the RNeasy spin column.	Close the lid, andcentrifuge for 15 s at ≥8000 x g. Discard the flow-through.	8	f	\N	31	2021-08-21 22:16:34.454561	2021-08-23 17:19:39.673818	31	266	0
988	Addition of buffer	Add 500 μL Buffer RPE to the RNeasy spin column. Close the lid, and centrifuge for 2 min at ≥8000 x g.	9	f	\N	31	2021-08-21 20:40:28.826128	2021-08-23 17:19:39.67629	31	266	0
989	Place the RNeasy spin column in a new 1.5 ml collection tube	Add 30–50 μL RNase-free water directly to the spin column membrane. Close the lid, and centrifuge for 1 min at ≥8000 x g to elute the RNA.	10	f	\N	31	2021-08-21 21:24:09.913894	2021-08-23 17:19:39.678758	31	266	0
990	If the expected RNA yield is >30 μg	<html><body><p>If the expected RNA yield is &gt;30 μg, repeat step 9 using another 30–50 μL of RNase-free water.<br>Alternatively, use the eluate from step 9 (if high RNA concentration is required). Reuse the collection tube from step 12.</p></body></html>	11	f	\N	31	2021-08-21 23:22:19.388684	2021-08-23 17:19:39.681401	31	266	0
992	Excise the tissue sample from the animal or remove it from storage.	<html><body><p>Remove RNAlater stabilized tissues from the reagent using forceps. Determine the amount of tissue. Do not use more than 30 mg.<br>Weighing tissue is the most accurate way to determine the amount.<br>Note: If the tissues were stored in RNAlater Reagent at –20°C, be sure to remove any crystals that may have formed.</p></body></html>	1	f	\N	31	2021-08-20 16:16:21.534889	2021-08-23 17:19:39.709787	31	267	0
993	Weigh the piece to be used, and place it into a suitably sized vessel for disruption and homogenization.	<html><body><p>RNA in harvested tissues is not protected until the tissues are treated with RNAlater RNA Stabilization Reagent, flash-frozen, or disrupted and homogenized in step 3. Frozen tissues should not be allowed to thaw during handling. The relevant procedures should be carried out as quickly as possible.<br><br>Note: Remaining fresh tissues can be placed into RNAlater RNA Stabilization Reagent to stabilize RNA (see protocol on page 34). However, previously frozen tissues thaw too slowly in the reagent, preventing the reagent from diffusing into the tissues quickly enough to prevent RNA degradation.</p></body></html>	2	f	\N	31	2021-08-20 15:53:20.581916	2021-08-23 17:19:39.712819	31	267	0
994	Disrupt the tissue and homogenize the lysate in Buffer RLT	<html><body><p>See “Disrupting and homogenizing starting material”, pages 18–21, for more details on disruption and homogenization.<br><br><strong>Note</strong>: Ensure that β-ME is added to Buffer RLT before use (see “Things to do before starting”).<br><br>After storage in RNAlater RNA Stabilization Reagent, tissues may become slightly harder than fresh or thawed tissues. Disruption and homogenization using standard methods is usually not a problem. For easier disruption and homogenization, we recommend using 600 µl Buffer RLT.<br><br><strong>Note</strong>: Incomplete homogenization leads to significantly reduced RNA yields and can cause clogging of the RNeasy spin column. Homogenization with the TissueLyser LT, TissueLyser II, and rotor–stator homogenizers generally results in higher RNA yields than with other methods.</p></body></html>	3	t	2021-08-20 22:49:20.818091	31	2021-08-20 16:48:32.958714	2021-08-23 17:19:39.718853	31	267	0
995	Disruption and homogenization using a rotor–stator homogenizer	Place the weighed (fresh, frozen, or RNAlater stabilized) tissue in a suitably sized vessel. Add the appropriate volume of Buffer RLT (see Table 8). Immediately disrupt and homogenize the tissue using a conventional rotor–stator homogenizer until it is uniformly homogeneous (usually 20–40 s).	4	t	2021-08-21 00:07:56.885972	31	2021-08-20 17:25:47.423121	2021-08-23 17:19:39.72343	31	267	0
996	Centrifuge the lysate for 3 min at full speed	<html><body><p>Carefully remove the supernatant by pipetting, and transfer it to a new microcentrifuge tube (not supplied).<br><br>Use only this supernatant (lysate) in subsequent steps. In some preparations, very small amounts of insoluble material will be present after the 3 min centrifugation, making the pellet invisible.</p></body></html>	5	t	2021-08-20 19:41:01.589596	31	2021-08-20 18:51:04.491587	2021-08-23 17:19:39.728565	31	267	0
997	Add 1 volume of 70% ethanol* to the cleared lysate, and mix immediately by pipetting	<html><body><span style="text-decoration: underline;">Do not centrifuge.</span><br><br><strong>Note</strong>: The volume of lysate may be less than 350 µL or 600 µL due to loss during homogenization and centrifugation in steps 3 and 4.<br><br><strong>Note</strong>: Precipitates may be visible after addition of ethanol. This does not affect the procedure.</body></html>	6	f	\N	31	2021-08-20 17:19:29.153145	2021-08-23 17:19:39.730326	31	267	0
998	Transfer up to 700 µL of the sample	<html><body><p>Transfer up to 700 µL of the sample, including any precipitate that may have formed, to an RNeasy spin column placed in a 2 mL collection tube (supplied).<br>Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm). Discard the flow-through. If the sample volume exceeds 700 µL, centrifuge successive aliquots in the same RNeasy spin column. Discard the flow-through after each centrifugation.</p></body></html>	7	t	2021-08-20 22:25:02.213519	31	2021-08-20 15:40:40.506421	2021-08-23 17:19:39.736019	31	267	0
999	Add 700 µL Buffer RW1 to the RNeasy spin column	Close the lid gently, and centrifuge for 15 s at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane. Discard the flow-through.	8	f	\N	31	2021-08-20 17:02:35.135002	2021-08-23 17:19:39.738175	31	267	0
1000	Add 500 µL Buffer RPE to the RNeasy spin column	<html><body><p>Close the lid gently, and centrifuge for 2 min at ≥8000 x g (≥10,000 rpm) to wash the spin column membrane.<br><br>The long centrifugation dries the spin column membrane, ensuring that no ethanol is carried over during RNA elution. Residual ethanol may interfere with downstream reactions.<br><br><strong>Note</strong>: After centrifugation, carefully remove the RNeasy spin column from the collection tube so that the column does not contact the flow-through. Otherwise, carryover of ethanol will occur.</p></body></html>	9	f	\N	31	2021-08-20 16:42:11.723833	2021-08-23 17:19:39.741304	31	267	0
1001	Place the RNeasy spin column in a new 1.5 mL collection tube (supplied)	Add 30–50 µL RNase-free water directly to the spin column membrane. Close the lid gently, and centrifuge for 1 min at ≥8000 x g (≥10,000 rpm) to elute the RNA.	10	t	2021-08-20 15:35:16.605171	31	2021-08-20 14:54:41.760092	2021-08-23 17:19:39.746598	31	267	0
1002	RNA denaturation	1 ug of RNA denature at 80°C for 5 min --> ice	0	f	\N	31	2021-08-19 13:43:49.327864	2021-08-23 17:19:39.800564	31	268	0
1003	Prepare mastermix for RT	High Capacity cDNA Reverse Transcription Kit (Applied Biosystems)	1	t	2021-08-19 15:38:00.408781	31	2021-08-19 10:03:57.06725	2021-08-23 17:19:39.806122	31	268	0
1004	RT reaction	25°C for 10 min 37°C for 2 h	2	f	\N	31	2021-08-19 09:44:50.730624	2021-08-23 17:19:39.807933	31	268	0
1005	Master Mix Preparation	PCR Master Mix includes Nuclease-Free Water and PCR Master Mix, 2X. PCR Master Mix is a premixed, ready-to-use solution containing Taq DNA Polymerase, dNTPs, MgCl2 and reaction buffers at optimal concentrations for efficient amplification of DNA templates by PCR.	0	t	2021-08-21 19:45:35.3627	31	2021-08-21 16:21:48.125848	2021-08-23 17:20:17.306977	31	269	0
1006	Setup of the 96-well Plate	<html><body><p>It is recommended to use pre-defined Excel templates or PlatR Pietting Assistant to customize pipetting scheme. Make sure to always include all necessary controls.&nbsp;Especially the negative controls.<br><br>Template of the 96-well plate.</p></body></html>	1	f	\N	31	2021-08-21 19:26:11.934242	2021-08-23 17:20:22.093707	31	269	0
1007	Reaction setup - NEW TITLE	<html><body><p>The Applied Biosystems 7900HT Fast Real-Time PCR System (7900HT Fast System) uses fluorescent-based PCR chemistries to provide:</p><ul><li>Quantitative detection of nucleic acid sequences using real-time analysis</li><li>Qualitative detection of nucleic acid sequences using end-point and dissociation-curve analysis</li></ul><p>You can perform several assay types on the 7900HT Fast System using reactions plates in the 96-well, 384-well, or TaqMan® Low Density Array format. This guide describes the allelic discrimination assay.</p></body></html>	2	f	\N	31	2021-08-21 18:44:16.382318	2021-08-23 17:20:22.547683	31	269	0
1008	Native PAGE protocol	Protein electrophoresis is a method for analysing the proteins in a fluid or an extract. The electrophoresis may be performed with a small volume of sample in a number of alternative ways with or without a supporting medium: SDS polyacrylamide gel electrophoresis (in short: gel electrophoresis, PAGE, or SDS-electrophoresis), free-flow electrophoresis, electrofocusing, isotachophoresis, affinity electrophoresis, immunoelectrophoresis, counterelectrophoresis, and capillary electrophoresis. Each method has many variations with individual advantages and limitations. Gel electrophoresis is often performed in combination with electroblotting immunoblotting to give additional information about a specific protein. Because of practical limitations, protein electrophoresis is generally not suited as a preparative method.	0	f	\N	31	2021-08-20 08:05:49.101157	2021-08-23 17:20:23.815088	31	270	0
1009	QA Checklist	Please perform the following checklist before and after you start working.	1	f	\N	31	2021-08-20 03:23:11.464044	2021-08-23 17:19:40.00568	31	270	0
1010	Check negative controls NTC	<html><body><div class="row">They have to be negative when using TaqMan assays.<br><ul><li class="col-xs-12">If they are positive when using SYBR assays check also melitng curve where signal comes from</li><li class="col-xs-12">If it is primer dimer result is negative</li><li class="col-xs-12">If it is specific signal it is positive.</li></ul></div></body></html>	2	t	2021-08-20 07:55:36.13351	31	2021-08-20 04:59:41.162377	2021-08-23 17:19:39.964326	31	270	0
1011	Eliminate results that have positive NTCs	And repeat procedure.	3	f	\N	31	2021-08-20 05:14:10.896868	2021-08-23 17:19:39.966986	31	270	0
1012	Native-Page Protocol	Follow a Nature protocol attached and a PDF.	4	f	\N	31	2021-08-20 06:10:11.208244	2021-08-23 17:40:30.205648	31	270	0
1013	Excel results	Write results in excel file.	5	f	\N	31	2021-08-20 04:20:29.295154	2021-08-23 17:20:18.951774	31	270	0
1014	Template for ddCq analysis	Sample information here.	0	t	2021-08-20 10:36:47.007046	31	2021-08-20 10:11:08.807768	2021-08-23 17:20:24.726209	31	271	0
\.


--
-- Data for Name: checklists; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.checklists (id, name, step_id, created_at, updated_at, created_by_id, last_modified_by_id) FROM stdin;
1	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	17	2019-05-30 12:40:54.039625	2019-05-30 12:40:54.039625	\N	\N
2	Things to do before starting	17	2019-05-30 12:40:54.050071	2019-05-30 12:40:54.050071	\N	\N
3	Optional	24	2019-05-30 12:40:54.063746	2019-05-30 12:40:54.063746	\N	\N
4	Mastermix	29	2019-05-30 12:40:54.090527	2019-05-30 12:40:54.090527	\N	\N
5	QA checklist	31	2019-05-30 12:40:54.123822	2019-05-30 12:40:54.123822	\N	\N
6	QA Native PAGE	35	2019-05-30 12:40:54.243626	2019-05-30 12:40:54.243626	\N	\N
7	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	57	2019-08-27 15:15:03.433928	2019-08-27 15:15:03.433928	\N	\N
8	Things to do before starting	57	2019-08-27 15:15:03.460274	2019-08-27 15:15:03.460274	\N	\N
9	Optional	64	2019-08-27 15:15:03.489299	2019-08-27 15:15:03.489299	\N	\N
10	Mastermix	69	2019-08-27 15:15:03.530243	2019-08-27 15:15:03.530243	\N	\N
11	QA checklist	71	2019-08-27 15:15:03.568022	2019-08-27 15:15:03.568022	\N	\N
12	QA Native PAGE	75	2019-08-27 15:15:03.773331	2019-08-27 15:15:03.773331	\N	\N
13	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	97	2019-08-28 07:31:01.093967	2019-08-28 07:31:01.093967	\N	\N
14	Things to do before starting	97	2019-08-28 07:31:01.147044	2019-08-28 07:31:01.147044	\N	\N
15	Optional	104	2019-08-28 07:31:01.18287	2019-08-28 07:31:01.18287	\N	\N
16	Mastermix	109	2019-08-28 07:31:01.224261	2019-08-28 07:31:01.224261	\N	\N
17	QA checklist	111	2019-08-28 07:31:01.262884	2019-08-28 07:31:01.262884	\N	\N
18	QA Native PAGE	115	2019-08-28 07:31:01.524832	2019-08-28 07:31:01.524832	\N	\N
19	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	137	2019-08-28 08:34:56.60514	2019-08-28 08:34:56.60514	\N	\N
20	Things to do before starting	137	2019-08-28 08:34:56.630867	2019-08-28 08:34:56.630867	\N	\N
21	Optional	144	2019-08-28 08:34:56.650743	2019-08-28 08:34:56.650743	\N	\N
22	Mastermix	149	2019-08-28 08:34:56.698115	2019-08-28 08:34:56.698115	\N	\N
23	QA checklist	151	2019-08-28 08:34:56.749161	2019-08-28 08:34:56.749161	\N	\N
24	QA Native PAGE	155	2019-08-28 08:34:57.239795	2019-08-28 08:34:57.239795	\N	\N
25	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	177	2019-08-29 12:51:27.240271	2019-08-29 12:51:27.240271	\N	\N
26	Things to do before starting	177	2019-08-29 12:51:27.292734	2019-08-29 12:51:27.292734	\N	\N
27	Optional	184	2019-08-29 12:51:27.351336	2019-08-29 12:51:27.351336	\N	\N
28	Mastermix	189	2019-08-29 12:51:27.536673	2019-08-29 12:51:27.536673	\N	\N
29	QA checklist	191	2019-08-29 12:51:27.709357	2019-08-29 12:51:27.709357	\N	\N
30	QA Native PAGE	195	2019-08-29 12:51:28.918028	2019-08-29 12:51:28.918028	\N	\N
31	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	217	2019-09-12 18:26:19.2545	2019-09-12 18:26:19.2545	\N	\N
32	Things to do before starting	217	2019-09-12 18:26:19.273281	2019-09-12 18:26:19.273281	\N	\N
33	Optional	224	2019-09-12 18:26:19.292449	2019-09-12 18:26:19.292449	\N	\N
34	Mastermix	229	2019-09-12 18:26:19.328757	2019-09-12 18:26:19.328757	\N	\N
35	QA checklist	231	2019-09-12 18:26:19.363041	2019-09-12 18:26:19.363041	\N	\N
36	QA Native PAGE	235	2019-09-12 18:26:19.529155	2019-09-12 18:26:19.529155	\N	\N
37	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	257	2019-11-13 13:57:42.9683	2019-11-13 13:57:42.9683	\N	\N
38	Things to do before starting	257	2019-11-13 13:57:43.000341	2019-11-13 13:57:43.000341	\N	\N
39	Optional	264	2019-11-13 13:57:43.025801	2019-11-13 13:57:43.025801	\N	\N
40	Mastermix	269	2019-11-13 13:57:43.076926	2019-11-13 13:57:43.076926	\N	\N
41	QA checklist	271	2019-11-13 13:57:43.131965	2019-11-13 13:57:43.131965	\N	\N
42	QA Native PAGE	275	2019-11-13 13:57:43.385179	2019-11-13 13:57:43.385179	\N	\N
43	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	297	2019-11-25 14:09:28.271971	2019-11-25 14:09:28.271971	\N	\N
44	Things to do before starting	297	2019-11-25 14:09:28.30234	2019-11-25 14:09:28.30234	\N	\N
45	Optional	304	2019-11-25 14:09:28.32993	2019-11-25 14:09:28.32993	\N	\N
46	Mastermix	309	2019-11-25 14:09:28.40368	2019-11-25 14:09:28.40368	\N	\N
47	QA checklist	311	2019-11-25 14:09:28.455248	2019-11-25 14:09:28.455248	\N	\N
48	QA Native PAGE	315	2019-11-25 14:09:28.711867	2019-11-25 14:09:28.711867	\N	\N
49	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	337	2019-11-25 18:00:53.513996	2019-11-25 18:00:53.513996	\N	\N
50	Things to do before starting	337	2019-11-25 18:00:53.532777	2019-11-25 18:00:53.532777	\N	\N
51	Optional	344	2019-11-25 18:00:53.569886	2019-11-25 18:00:53.569886	\N	\N
52	Mastermix	349	2019-11-25 18:00:53.622137	2019-11-25 18:00:53.622137	\N	\N
53	QA checklist	351	2019-11-25 18:00:53.676971	2019-11-25 18:00:53.676971	\N	\N
54	QA Native PAGE	355	2019-11-25 18:00:53.915208	2019-11-25 18:00:53.915208	\N	\N
55	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	377	2020-05-12 16:13:06.540219	2020-05-12 16:13:06.540219	\N	\N
56	Things to do before starting	377	2020-05-12 16:13:06.572181	2020-05-12 16:13:06.572181	\N	\N
57	Optional	384	2020-05-12 16:13:06.593951	2020-05-12 16:13:06.593951	\N	\N
58	Mastermix	389	2020-05-12 16:13:06.639935	2020-05-12 16:13:06.639935	\N	\N
59	QA checklist	391	2020-05-12 16:13:06.71022	2020-05-12 16:13:06.71022	\N	\N
60	QA Native PAGE	395	2020-05-12 16:13:06.965232	2020-05-12 16:13:06.965232	\N	\N
61	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	417	2020-06-10 14:43:18.390381	2020-06-10 14:43:18.390381	\N	\N
62	Things to do before starting	417	2020-06-10 14:43:18.418173	2020-06-10 14:43:18.418173	\N	\N
63	Optional	424	2020-06-10 14:43:18.438821	2020-06-10 14:43:18.438821	\N	\N
64	Mastermix	429	2020-06-10 14:43:18.482711	2020-06-10 14:43:18.482711	\N	\N
65	QA checklist	431	2020-06-10 14:43:18.537002	2020-06-10 14:43:18.537002	\N	\N
66	QA Native PAGE	435	2020-06-10 14:43:18.725428	2020-06-10 14:43:18.725428	\N	\N
67	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	459	2020-10-07 19:20:27.116652	2020-10-07 19:20:27.116652	\N	\N
68	Things to do before starting	459	2020-10-07 19:20:27.147838	2020-10-07 19:20:27.147838	\N	\N
69	Optional	466	2020-10-07 19:20:27.176231	2020-10-07 19:20:27.176231	\N	\N
70	Mastermix	471	2020-10-07 19:20:27.224794	2020-10-07 19:20:27.224794	\N	\N
71	QA checklist	473	2020-10-07 19:20:27.273161	2020-10-07 19:20:27.273161	\N	\N
72	QA Native PAGE	477	2020-10-07 19:20:27.599313	2020-10-07 19:20:27.599313	\N	\N
73	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	499	2020-11-18 14:50:54.346177	2020-11-18 14:50:54.346177	\N	\N
74	Things to do before starting	499	2020-11-18 14:50:54.379843	2020-11-18 14:50:54.379843	\N	\N
75	Optional	506	2020-11-18 14:50:54.40766	2020-11-18 14:50:54.40766	\N	\N
76	Mastermix	511	2020-11-18 14:50:54.46728	2020-11-18 14:50:54.46728	\N	\N
77	QA checklist	513	2020-11-18 14:50:54.525681	2020-11-18 14:50:54.525681	\N	\N
78	QA Native PAGE	517	2020-11-18 14:50:54.804803	2020-11-18 14:50:54.804803	\N	\N
79	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	539	2020-11-18 14:53:25.778896	2020-11-18 14:53:25.778896	\N	\N
80	Things to do before starting	539	2020-11-18 14:53:25.793075	2020-11-18 14:53:25.793075	\N	\N
81	Optional	546	2020-11-18 14:53:25.818802	2020-11-18 14:53:25.818802	\N	\N
82	Mastermix	551	2020-11-18 14:53:25.867749	2020-11-18 14:53:25.867749	\N	\N
83	QA checklist	553	2020-11-18 14:53:25.919564	2020-11-18 14:53:25.919564	\N	\N
84	QA Native PAGE	557	2020-11-18 14:53:26.1147	2020-11-18 14:53:26.1147	\N	\N
85	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	581	2021-05-19 20:01:52.641202	2021-05-19 20:01:52.641202	\N	\N
86	Things to do before starting	581	2021-05-19 20:01:52.665224	2021-05-19 20:01:52.665224	\N	\N
87	Optional	588	2021-05-19 20:01:52.680952	2021-05-19 20:01:52.680952	\N	\N
88	Mastermix	593	2021-05-19 20:01:52.709176	2021-05-19 20:01:52.709176	\N	\N
89	QA checklist	595	2021-05-19 20:01:52.736749	2021-05-19 20:01:52.736749	\N	\N
90	QA Native PAGE	599	2021-05-19 20:01:52.855017	2021-05-19 20:01:52.855017	\N	\N
91	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	621	2021-05-20 14:12:31.122191	2021-05-20 14:12:31.122191	\N	\N
92	Things to do before starting	621	2021-05-20 14:12:31.134788	2021-05-20 14:12:31.134788	\N	\N
93	Optional	628	2021-05-20 14:12:31.150897	2021-05-20 14:12:31.150897	\N	\N
94	Mastermix	633	2021-05-20 14:12:31.178431	2021-05-20 14:12:31.178431	\N	\N
95	QA checklist	635	2021-05-20 14:12:31.206785	2021-05-20 14:12:31.206785	\N	\N
96	QA Native PAGE	639	2021-05-20 14:12:31.337795	2021-05-20 14:12:31.337795	\N	\N
97	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	661	2021-05-20 14:12:57.680274	2021-05-20 14:12:57.680274	\N	\N
98	Things to do before starting	661	2021-05-20 14:12:57.689093	2021-05-20 14:12:57.689093	\N	\N
99	Optional	668	2021-05-20 14:12:57.704052	2021-05-20 14:12:57.704052	\N	\N
100	Mastermix	673	2021-05-20 14:12:57.728791	2021-05-20 14:12:57.728791	\N	\N
101	QA checklist	675	2021-05-20 14:12:57.751237	2021-05-20 14:12:57.751237	\N	\N
102	QA Native PAGE	679	2021-05-20 14:12:57.864836	2021-05-20 14:12:57.864836	\N	\N
103	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	701	2021-05-20 14:13:34.503319	2021-05-20 14:13:34.503319	\N	\N
104	Things to do before starting	701	2021-05-20 14:13:34.51347	2021-05-20 14:13:34.51347	\N	\N
105	Optional	708	2021-05-20 14:13:34.530618	2021-05-20 14:13:34.530618	\N	\N
106	Mastermix	713	2021-05-20 14:13:34.559012	2021-05-20 14:13:34.559012	\N	\N
107	QA checklist	715	2021-05-20 14:13:34.592004	2021-05-20 14:13:34.592004	\N	\N
108	QA Native PAGE	719	2021-05-20 14:13:34.724243	2021-05-20 14:13:34.724243	\N	\N
109	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	741	2021-05-20 14:21:21.058605	2021-05-20 14:21:21.058605	\N	\N
110	Things to do before starting	741	2021-05-20 14:21:21.069303	2021-05-20 14:21:21.069303	\N	\N
111	Optional	748	2021-05-20 14:21:21.084114	2021-05-20 14:21:21.084114	\N	\N
112	Mastermix	753	2021-05-20 14:21:21.109508	2021-05-20 14:21:21.109508	\N	\N
113	QA checklist	755	2021-05-20 14:21:21.13479	2021-05-20 14:21:21.13479	\N	\N
114	QA Native PAGE	759	2021-05-20 14:21:21.322161	2021-05-20 14:21:21.322161	\N	\N
115	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	781	2021-05-20 14:30:30.826006	2021-05-20 14:30:30.826006	\N	\N
116	Things to do before starting	781	2021-05-20 14:30:30.83581	2021-05-20 14:30:30.83581	\N	\N
117	Optional	788	2021-05-20 14:30:30.849621	2021-05-20 14:30:30.849621	\N	\N
118	Mastermix	793	2021-05-20 14:30:30.887793	2021-05-20 14:30:30.887793	\N	\N
119	QA checklist	795	2021-05-20 14:30:30.92254	2021-05-20 14:30:30.92254	\N	\N
120	QA Native PAGE	799	2021-05-20 14:30:31.060323	2021-05-20 14:30:31.060323	\N	\N
121	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	822	2021-05-20 15:17:12.215092	2021-05-20 15:17:12.215092	\N	\N
122	Things to do before starting	822	2021-05-20 15:17:12.227237	2021-05-20 15:17:12.227237	\N	\N
123	Optional	829	2021-05-20 15:17:12.242096	2021-05-20 15:17:12.242096	\N	\N
124	Mastermix	834	2021-05-20 15:17:12.27292	2021-05-20 15:17:12.27292	\N	\N
125	QA checklist	836	2021-05-20 15:17:12.30307	2021-05-20 15:17:12.30307	\N	\N
126	QA Native PAGE	840	2021-05-20 15:17:12.423126	2021-05-20 15:17:12.423126	\N	\N
127	Glassware for WIZ reagents	846	2021-05-26 16:37:18.903986	2021-05-26 16:37:18.903986	\N	\N
128	Chemicals for WIZ reagent preparation	847	2021-05-27 14:50:14.254925	2021-05-27 14:50:14.254925	\N	\N
129	PO4 Color Acid Reagent First Step	848	2021-05-27 16:52:19.419548	2021-05-27 16:52:19.419548	\N	\N
130	Wash Solution HCl	848	2021-05-27 16:52:19.569689	2021-05-27 16:52:19.569689	\N	\N
131	NO3:VCl3 Reduction Reagent	848	2021-05-27 16:52:19.589627	2021-05-27 16:52:19.589627	\N	\N
132	NOxSAA	848	2021-05-27 16:52:19.708935	2021-05-27 16:52:19.708935	\N	\N
133	NOxNED	848	2021-05-27 16:52:19.788843	2021-05-27 16:52:19.788843	\N	\N
134	PO4 Color Acid Reagent Second Step	848	2021-05-27 16:52:19.834046	2021-05-27 16:52:19.834046	\N	\N
135	Ascorbic Acid Reducing Reagent for PO4	848	2021-05-27 16:52:19.953123	2021-05-27 16:52:19.953123	\N	\N
136	Cleanup	848	2021-05-27 16:52:20.077688	2021-05-27 16:52:20.077688	\N	\N
137	Reagents and supplies for field	850	2021-05-27 19:19:27.285416	2021-05-27 19:19:27.285416	\N	\N
138	External Calibrant, 50 PPB N as NO2, 500 PPB N as NO3 and 300 PPB P as PO4	849	2021-05-27 19:55:55.057215	2021-05-27 19:55:55.057215	\N	\N
139	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	867	2021-05-28 15:07:55.238107	2021-05-28 15:07:55.238107	\N	\N
140	Things to do before starting	867	2021-05-28 15:07:55.284296	2021-05-28 15:07:55.284296	\N	\N
141	Optional	874	2021-05-28 15:07:55.307523	2021-05-28 15:07:55.307523	\N	\N
142	Mastermix	879	2021-05-28 15:07:55.344091	2021-05-28 15:07:55.344091	\N	\N
143	QA checklist	881	2021-05-28 15:07:55.379867	2021-05-28 15:07:55.379867	\N	\N
144	QA Native PAGE	885	2021-05-28 15:07:58.073482	2021-05-28 15:07:58.073482	\N	\N
145	Entry into WIZ location	892	2021-06-01 16:51:11.73076	2021-06-01 16:51:11.73076	\N	\N
146	Travel to WIZ location	893	2021-06-01 17:13:31.895724	2021-06-01 17:13:31.895724	\N	\N
147	WIZ Calibration	894	2021-06-01 17:17:54.253613	2021-06-01 17:17:54.253613	\N	\N
148	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	911	2021-06-02 16:10:05.018411	2021-06-02 16:10:05.018411	\N	\N
149	Things to do before starting	911	2021-06-02 16:10:05.094864	2021-06-02 16:10:05.094864	\N	\N
150	Optional	918	2021-06-02 16:10:05.205053	2021-06-02 16:10:05.205053	\N	\N
151	Mastermix	923	2021-06-02 16:10:06.562301	2021-06-02 16:10:06.562301	\N	\N
152	QA checklist	925	2021-06-02 16:10:06.673089	2021-06-02 16:10:06.673089	\N	\N
153	QA Native PAGE	929	2021-06-02 16:10:07.387078	2021-06-02 16:10:07.387078	\N	\N
154	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	951	2021-08-23 14:58:28.644159	2021-08-23 14:58:28.644159	\N	\N
155	Things to do before starting	951	2021-08-23 14:58:28.668526	2021-08-23 14:58:28.668526	\N	\N
156	Optional	958	2021-08-23 14:58:28.682496	2021-08-23 14:58:28.682496	\N	\N
157	Mastermix	963	2021-08-23 14:58:28.710929	2021-08-23 14:58:28.710929	\N	\N
158	QA checklist	965	2021-08-23 14:58:28.739095	2021-08-23 14:58:28.739095	\N	\N
159	QA Native PAGE	969	2021-08-23 14:58:28.861445	2021-08-23 14:58:28.861445	\N	\N
160	If using the RNeasy Kit for the first time, read "Important Notes" in the attached file	991	2021-08-23 17:19:39.755649	2021-08-23 17:19:39.755649	\N	\N
161	Things to do before starting	991	2021-08-23 17:19:39.768625	2021-08-23 17:19:39.768625	\N	\N
162	Optional	998	2021-08-23 17:19:39.784037	2021-08-23 17:19:39.784037	\N	\N
163	Mastermix	1003	2021-08-23 17:19:39.814099	2021-08-23 17:19:39.814099	\N	\N
164	QA checklist	1005	2021-08-23 17:19:39.85106	2021-08-23 17:19:39.85106	\N	\N
165	QA Native PAGE	1009	2021-08-23 17:19:40.004442	2021-08-23 17:19:40.004442	\N	\N
166	Analytical Balance Calibration	1019	2021-08-26 20:26:39.450491	2021-08-26 20:26:39.450491	\N	\N
167	Analytical Balance Calibration	1020	2021-08-26 20:55:14.770912	2021-08-26 20:55:14.770912	\N	\N
\.


--
-- Data for Name: checklist_items; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.checklist_items (id, text, checked, checklist_id, created_at, updated_at, created_by_id, last_modified_by_id, "position") FROM stdin;
1	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	1	2019-05-30 12:40:54.041483	2019-05-30 12:40:54.041483	\N	\N	0
2	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	1	2019-05-30 12:40:54.042958	2019-05-30 12:40:54.042958	\N	\N	1
3	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	1	2019-05-30 12:40:54.044103	2019-05-30 12:40:54.044103	\N	\N	2
4	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	1	2019-05-30 12:40:54.045147	2019-05-30 12:40:54.045147	\N	\N	3
5	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	1	2019-05-30 12:40:54.046119	2019-05-30 12:40:54.046119	\N	\N	4
6	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	1	2019-05-30 12:40:54.047108	2019-05-30 12:40:54.047108	\N	\N	5
7	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	1	2019-05-30 12:40:54.048048	2019-05-30 12:40:54.048048	\N	\N	6
8	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	2	2019-05-30 12:40:54.051264	2019-05-30 12:40:54.051264	\N	\N	0
9	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	2	2019-05-30 12:40:54.052427	2019-05-30 12:40:54.052427	\N	\N	1
10	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	2	2019-05-30 12:40:54.05329	2019-05-30 12:40:54.05329	\N	\N	2
11	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	3	2019-05-30 12:40:54.064955	2019-05-30 12:40:54.064955	\N	\N	0
12	H2O to 12.5 uL	f	4	2019-05-30 12:40:54.091796	2019-05-30 12:40:54.091796	\N	\N	0
13	Optional: Luciferase mRNA (denatured)	f	4	2019-05-30 12:40:54.092711	2019-05-30 12:40:54.092711	\N	\N	1
14	Reverse transcriptase	f	4	2019-05-30 12:40:54.093559	2019-05-30 12:40:54.093559	\N	\N	2
15	RNase inhibitor	f	4	2019-05-30 12:40:54.09441	2019-05-30 12:40:54.09441	\N	\N	3
16	Random Primers	f	4	2019-05-30 12:40:54.095196	2019-05-30 12:40:54.095196	\N	\N	4
17	dNTP mix	f	4	2019-05-30 12:40:54.096042	2019-05-30 12:40:54.096042	\N	\N	5
18	RT buffer	f	4	2019-05-30 12:40:54.096885	2019-05-30 12:40:54.096885	\N	\N	6
19	Make sure the UV light was on at least for 20 minutes before you started to work	f	5	2019-05-30 12:40:54.125097	2019-05-30 12:40:54.125097	\N	\N	0
20	Write down LOT numbers of reagents used	f	5	2019-05-30 12:40:54.126062	2019-05-30 12:40:54.126062	\N	\N	1
21	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	5	2019-05-30 12:40:54.126992	2019-05-30 12:40:54.126992	\N	\N	2
22	Always use designated separate chambers for pipetting samples and reagents	f	5	2019-05-30 12:40:54.127902	2019-05-30 12:40:54.127902	\N	\N	3
23	Change lab coats when switching chambers	f	5	2019-05-30 12:40:54.128752	2019-05-30 12:40:54.128752	\N	\N	4
24	Clean surfaces with 70% ethanol or RNA remover	f	5	2019-05-30 12:40:54.129584	2019-05-30 12:40:54.129584	\N	\N	5
25	Turn on the UV light	f	5	2019-05-30 12:40:54.130417	2019-05-30 12:40:54.130417	\N	\N	6
26	Check buffer stock & prepare new stock if needed	f	6	2019-05-30 12:40:54.244898	2019-05-30 12:40:54.244898	\N	\N	0
27	Check stock of reagents & order new stock if needed	f	6	2019-05-30 12:40:54.245824	2019-05-30 12:40:54.245824	\N	\N	1
28	Use gloves at all times	f	6	2019-05-30 12:40:54.246646	2019-05-30 12:40:54.246646	\N	\N	2
29	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	7	2019-08-27 15:15:03.442461	2019-08-27 15:15:03.442461	\N	\N	0
30	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	7	2019-08-27 15:15:03.450043	2019-08-27 15:15:03.450043	\N	\N	1
31	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	7	2019-08-27 15:15:03.451732	2019-08-27 15:15:03.451732	\N	\N	2
32	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	7	2019-08-27 15:15:03.453939	2019-08-27 15:15:03.453939	\N	\N	3
33	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	7	2019-08-27 15:15:03.455093	2019-08-27 15:15:03.455093	\N	\N	4
34	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	7	2019-08-27 15:15:03.456316	2019-08-27 15:15:03.456316	\N	\N	5
35	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	7	2019-08-27 15:15:03.457429	2019-08-27 15:15:03.457429	\N	\N	6
79	Change lab coats when switching chambers	f	17	2019-08-28 07:31:01.268603	2019-08-28 07:31:01.268603	\N	\N	4
80	Clean surfaces with 70% ethanol or RNA remover	f	17	2019-08-28 07:31:01.269652	2019-08-28 07:31:01.269652	\N	\N	5
81	Turn on the UV light	f	17	2019-08-28 07:31:01.270669	2019-08-28 07:31:01.270669	\N	\N	6
36	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	8	2019-08-27 15:15:03.462115	2019-08-27 15:15:03.462115	\N	\N	0
37	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	8	2019-08-27 15:15:03.46413	2019-08-27 15:15:03.46413	\N	\N	1
38	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	8	2019-08-27 15:15:03.465742	2019-08-27 15:15:03.465742	\N	\N	2
39	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	9	2019-08-27 15:15:03.490878	2019-08-27 15:15:03.490878	\N	\N	0
40	H2O to 12.5 uL	f	10	2019-08-27 15:15:03.531879	2019-08-27 15:15:03.531879	\N	\N	0
41	Optional: Luciferase mRNA (denatured)	f	10	2019-08-27 15:15:03.533108	2019-08-27 15:15:03.533108	\N	\N	1
42	Reverse transcriptase	f	10	2019-08-27 15:15:03.534175	2019-08-27 15:15:03.534175	\N	\N	2
43	RNase inhibitor	f	10	2019-08-27 15:15:03.535275	2019-08-27 15:15:03.535275	\N	\N	3
44	Random Primers	f	10	2019-08-27 15:15:03.536488	2019-08-27 15:15:03.536488	\N	\N	4
45	dNTP mix	f	10	2019-08-27 15:15:03.539037	2019-08-27 15:15:03.539037	\N	\N	5
46	RT buffer	f	10	2019-08-27 15:15:03.54009	2019-08-27 15:15:03.54009	\N	\N	6
47	Make sure the UV light was on at least for 20 minutes before you started to work	f	11	2019-08-27 15:15:03.569761	2019-08-27 15:15:03.569761	\N	\N	0
48	Write down LOT numbers of reagents used	f	11	2019-08-27 15:15:03.570903	2019-08-27 15:15:03.570903	\N	\N	1
49	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	11	2019-08-27 15:15:03.572035	2019-08-27 15:15:03.572035	\N	\N	2
50	Always use designated separate chambers for pipetting samples and reagents	f	11	2019-08-27 15:15:03.573063	2019-08-27 15:15:03.573063	\N	\N	3
51	Change lab coats when switching chambers	f	11	2019-08-27 15:15:03.5743	2019-08-27 15:15:03.5743	\N	\N	4
52	Clean surfaces with 70% ethanol or RNA remover	f	11	2019-08-27 15:15:03.575332	2019-08-27 15:15:03.575332	\N	\N	5
53	Turn on the UV light	f	11	2019-08-27 15:15:03.576347	2019-08-27 15:15:03.576347	\N	\N	6
54	Check buffer stock & prepare new stock if needed	f	12	2019-08-27 15:15:03.775255	2019-08-27 15:15:03.775255	\N	\N	0
55	Check stock of reagents & order new stock if needed	f	12	2019-08-27 15:15:03.77643	2019-08-27 15:15:03.77643	\N	\N	1
56	Use gloves at all times	f	12	2019-08-27 15:15:03.777626	2019-08-27 15:15:03.777626	\N	\N	2
57	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	13	2019-08-28 07:31:01.100007	2019-08-28 07:31:01.100007	\N	\N	0
58	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	13	2019-08-28 07:31:01.121118	2019-08-28 07:31:01.121118	\N	\N	1
59	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	13	2019-08-28 07:31:01.123589	2019-08-28 07:31:01.123589	\N	\N	2
60	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	13	2019-08-28 07:31:01.130722	2019-08-28 07:31:01.130722	\N	\N	3
61	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	13	2019-08-28 07:31:01.13226	2019-08-28 07:31:01.13226	\N	\N	4
62	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	13	2019-08-28 07:31:01.133644	2019-08-28 07:31:01.133644	\N	\N	5
63	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	13	2019-08-28 07:31:01.134855	2019-08-28 07:31:01.134855	\N	\N	6
64	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	14	2019-08-28 07:31:01.148624	2019-08-28 07:31:01.148624	\N	\N	0
65	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	14	2019-08-28 07:31:01.155909	2019-08-28 07:31:01.155909	\N	\N	1
66	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	14	2019-08-28 07:31:01.160331	2019-08-28 07:31:01.160331	\N	\N	2
67	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	15	2019-08-28 07:31:01.184811	2019-08-28 07:31:01.184811	\N	\N	0
68	H2O to 12.5 uL	f	16	2019-08-28 07:31:01.225735	2019-08-28 07:31:01.225735	\N	\N	0
69	Optional: Luciferase mRNA (denatured)	f	16	2019-08-28 07:31:01.226839	2019-08-28 07:31:01.226839	\N	\N	1
70	Reverse transcriptase	f	16	2019-08-28 07:31:01.227902	2019-08-28 07:31:01.227902	\N	\N	2
71	RNase inhibitor	f	16	2019-08-28 07:31:01.228895	2019-08-28 07:31:01.228895	\N	\N	3
72	Random Primers	f	16	2019-08-28 07:31:01.229911	2019-08-28 07:31:01.229911	\N	\N	4
73	dNTP mix	f	16	2019-08-28 07:31:01.23093	2019-08-28 07:31:01.23093	\N	\N	5
74	RT buffer	f	16	2019-08-28 07:31:01.231935	2019-08-28 07:31:01.231935	\N	\N	6
75	Make sure the UV light was on at least for 20 minutes before you started to work	f	17	2019-08-28 07:31:01.264293	2019-08-28 07:31:01.264293	\N	\N	0
76	Write down LOT numbers of reagents used	f	17	2019-08-28 07:31:01.265365	2019-08-28 07:31:01.265365	\N	\N	1
77	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	17	2019-08-28 07:31:01.266409	2019-08-28 07:31:01.266409	\N	\N	2
78	Always use designated separate chambers for pipetting samples and reagents	f	17	2019-08-28 07:31:01.267548	2019-08-28 07:31:01.267548	\N	\N	3
82	Check buffer stock & prepare new stock if needed	f	18	2019-08-28 07:31:01.526958	2019-08-28 07:31:01.526958	\N	\N	0
83	Check stock of reagents & order new stock if needed	f	18	2019-08-28 07:31:01.528084	2019-08-28 07:31:01.528084	\N	\N	1
84	Use gloves at all times	f	18	2019-08-28 07:31:01.529104	2019-08-28 07:31:01.529104	\N	\N	2
85	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	19	2019-08-28 08:34:56.60749	2019-08-28 08:34:56.60749	\N	\N	0
86	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	19	2019-08-28 08:34:56.612175	2019-08-28 08:34:56.612175	\N	\N	1
87	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	19	2019-08-28 08:34:56.618108	2019-08-28 08:34:56.618108	\N	\N	2
88	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	19	2019-08-28 08:34:56.619274	2019-08-28 08:34:56.619274	\N	\N	3
89	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	19	2019-08-28 08:34:56.620599	2019-08-28 08:34:56.620599	\N	\N	4
90	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	19	2019-08-28 08:34:56.621832	2019-08-28 08:34:56.621832	\N	\N	5
91	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	19	2019-08-28 08:34:56.623504	2019-08-28 08:34:56.623504	\N	\N	6
92	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	20	2019-08-28 08:34:56.632427	2019-08-28 08:34:56.632427	\N	\N	0
93	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	20	2019-08-28 08:34:56.634615	2019-08-28 08:34:56.634615	\N	\N	1
94	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	20	2019-08-28 08:34:56.635677	2019-08-28 08:34:56.635677	\N	\N	2
95	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	21	2019-08-28 08:34:56.652173	2019-08-28 08:34:56.652173	\N	\N	0
96	H2O to 12.5 uL	f	22	2019-08-28 08:34:56.701913	2019-08-28 08:34:56.701913	\N	\N	0
97	Optional: Luciferase mRNA (denatured)	f	22	2019-08-28 08:34:56.703013	2019-08-28 08:34:56.703013	\N	\N	1
98	Reverse transcriptase	f	22	2019-08-28 08:34:56.704069	2019-08-28 08:34:56.704069	\N	\N	2
99	RNase inhibitor	f	22	2019-08-28 08:34:56.705166	2019-08-28 08:34:56.705166	\N	\N	3
100	Random Primers	f	22	2019-08-28 08:34:56.706549	2019-08-28 08:34:56.706549	\N	\N	4
101	dNTP mix	f	22	2019-08-28 08:34:56.707577	2019-08-28 08:34:56.707577	\N	\N	5
102	RT buffer	f	22	2019-08-28 08:34:56.70877	2019-08-28 08:34:56.70877	\N	\N	6
103	Make sure the UV light was on at least for 20 minutes before you started to work	f	23	2019-08-28 08:34:56.751554	2019-08-28 08:34:56.751554	\N	\N	0
104	Write down LOT numbers of reagents used	f	23	2019-08-28 08:34:56.753208	2019-08-28 08:34:56.753208	\N	\N	1
105	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	23	2019-08-28 08:34:56.754629	2019-08-28 08:34:56.754629	\N	\N	2
106	Always use designated separate chambers for pipetting samples and reagents	f	23	2019-08-28 08:34:56.75594	2019-08-28 08:34:56.75594	\N	\N	3
107	Change lab coats when switching chambers	f	23	2019-08-28 08:34:56.757174	2019-08-28 08:34:56.757174	\N	\N	4
108	Clean surfaces with 70% ethanol or RNA remover	f	23	2019-08-28 08:34:56.758654	2019-08-28 08:34:56.758654	\N	\N	5
109	Turn on the UV light	f	23	2019-08-28 08:34:56.760155	2019-08-28 08:34:56.760155	\N	\N	6
110	Check buffer stock & prepare new stock if needed	f	24	2019-08-28 08:34:57.241468	2019-08-28 08:34:57.241468	\N	\N	0
111	Check stock of reagents & order new stock if needed	f	24	2019-08-28 08:34:57.242651	2019-08-28 08:34:57.242651	\N	\N	1
112	Use gloves at all times	f	24	2019-08-28 08:34:57.243795	2019-08-28 08:34:57.243795	\N	\N	2
113	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	25	2019-08-29 12:51:27.245346	2019-08-29 12:51:27.245346	\N	\N	0
114	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	25	2019-08-29 12:51:27.255052	2019-08-29 12:51:27.255052	\N	\N	1
115	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	25	2019-08-29 12:51:27.261809	2019-08-29 12:51:27.261809	\N	\N	2
116	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	25	2019-08-29 12:51:27.277989	2019-08-29 12:51:27.277989	\N	\N	3
117	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	25	2019-08-29 12:51:27.283196	2019-08-29 12:51:27.283196	\N	\N	4
118	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	25	2019-08-29 12:51:27.287758	2019-08-29 12:51:27.287758	\N	\N	5
119	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	25	2019-08-29 12:51:27.288978	2019-08-29 12:51:27.288978	\N	\N	6
120	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	26	2019-08-29 12:51:27.297084	2019-08-29 12:51:27.297084	\N	\N	0
121	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	26	2019-08-29 12:51:27.327155	2019-08-29 12:51:27.327155	\N	\N	1
122	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	26	2019-08-29 12:51:27.329147	2019-08-29 12:51:27.329147	\N	\N	2
123	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	27	2019-08-29 12:51:27.353031	2019-08-29 12:51:27.353031	\N	\N	0
124	H2O to 12.5 uL	f	28	2019-08-29 12:51:27.552117	2019-08-29 12:51:27.552117	\N	\N	0
125	Optional: Luciferase mRNA (denatured)	f	28	2019-08-29 12:51:27.55393	2019-08-29 12:51:27.55393	\N	\N	1
126	Reverse transcriptase	f	28	2019-08-29 12:51:27.565467	2019-08-29 12:51:27.565467	\N	\N	2
127	RNase inhibitor	f	28	2019-08-29 12:51:27.568099	2019-08-29 12:51:27.568099	\N	\N	3
128	Random Primers	f	28	2019-08-29 12:51:27.578161	2019-08-29 12:51:27.578161	\N	\N	4
129	dNTP mix	f	28	2019-08-29 12:51:27.587483	2019-08-29 12:51:27.587483	\N	\N	5
130	RT buffer	f	28	2019-08-29 12:51:27.588548	2019-08-29 12:51:27.588548	\N	\N	6
131	Make sure the UV light was on at least for 20 minutes before you started to work	f	29	2019-08-29 12:51:27.712241	2019-08-29 12:51:27.712241	\N	\N	0
132	Write down LOT numbers of reagents used	f	29	2019-08-29 12:51:27.731503	2019-08-29 12:51:27.731503	\N	\N	1
133	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	29	2019-08-29 12:51:27.740394	2019-08-29 12:51:27.740394	\N	\N	2
134	Always use designated separate chambers for pipetting samples and reagents	f	29	2019-08-29 12:51:27.74802	2019-08-29 12:51:27.74802	\N	\N	3
135	Change lab coats when switching chambers	f	29	2019-08-29 12:51:27.7514	2019-08-29 12:51:27.7514	\N	\N	4
136	Clean surfaces with 70% ethanol or RNA remover	f	29	2019-08-29 12:51:27.752443	2019-08-29 12:51:27.752443	\N	\N	5
137	Turn on the UV light	f	29	2019-08-29 12:51:27.753528	2019-08-29 12:51:27.753528	\N	\N	6
138	Check buffer stock & prepare new stock if needed	f	30	2019-08-29 12:51:28.924485	2019-08-29 12:51:28.924485	\N	\N	0
139	Check stock of reagents & order new stock if needed	f	30	2019-08-29 12:51:28.941617	2019-08-29 12:51:28.941617	\N	\N	1
140	Use gloves at all times	f	30	2019-08-29 12:51:28.942711	2019-08-29 12:51:28.942711	\N	\N	2
141	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	31	2019-09-12 18:26:19.259081	2019-09-12 18:26:19.259081	\N	\N	0
142	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	31	2019-09-12 18:26:19.264429	2019-09-12 18:26:19.264429	\N	\N	1
143	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	31	2019-09-12 18:26:19.26573	2019-09-12 18:26:19.26573	\N	\N	2
144	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	31	2019-09-12 18:26:19.266962	2019-09-12 18:26:19.266962	\N	\N	3
145	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	31	2019-09-12 18:26:19.268132	2019-09-12 18:26:19.268132	\N	\N	4
146	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	31	2019-09-12 18:26:19.26933	2019-09-12 18:26:19.26933	\N	\N	5
147	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	31	2019-09-12 18:26:19.270409	2019-09-12 18:26:19.270409	\N	\N	6
148	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	32	2019-09-12 18:26:19.274875	2019-09-12 18:26:19.274875	\N	\N	0
149	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	32	2019-09-12 18:26:19.276192	2019-09-12 18:26:19.276192	\N	\N	1
150	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	32	2019-09-12 18:26:19.277281	2019-09-12 18:26:19.277281	\N	\N	2
151	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	33	2019-09-12 18:26:19.293891	2019-09-12 18:26:19.293891	\N	\N	0
152	H2O to 12.5 uL	f	34	2019-09-12 18:26:19.330407	2019-09-12 18:26:19.330407	\N	\N	0
153	Optional: Luciferase mRNA (denatured)	f	34	2019-09-12 18:26:19.331535	2019-09-12 18:26:19.331535	\N	\N	1
154	Reverse transcriptase	f	34	2019-09-12 18:26:19.332646	2019-09-12 18:26:19.332646	\N	\N	2
155	RNase inhibitor	f	34	2019-09-12 18:26:19.333791	2019-09-12 18:26:19.333791	\N	\N	3
156	Random Primers	f	34	2019-09-12 18:26:19.33491	2019-09-12 18:26:19.33491	\N	\N	4
157	dNTP mix	f	34	2019-09-12 18:26:19.336088	2019-09-12 18:26:19.336088	\N	\N	5
158	RT buffer	f	34	2019-09-12 18:26:19.337146	2019-09-12 18:26:19.337146	\N	\N	6
159	Make sure the UV light was on at least for 20 minutes before you started to work	f	35	2019-09-12 18:26:19.364561	2019-09-12 18:26:19.364561	\N	\N	0
160	Write down LOT numbers of reagents used	f	35	2019-09-12 18:26:19.365676	2019-09-12 18:26:19.365676	\N	\N	1
161	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	35	2019-09-12 18:26:19.366818	2019-09-12 18:26:19.366818	\N	\N	2
162	Always use designated separate chambers for pipetting samples and reagents	f	35	2019-09-12 18:26:19.36788	2019-09-12 18:26:19.36788	\N	\N	3
163	Change lab coats when switching chambers	f	35	2019-09-12 18:26:19.368972	2019-09-12 18:26:19.368972	\N	\N	4
164	Clean surfaces with 70% ethanol or RNA remover	f	35	2019-09-12 18:26:19.370148	2019-09-12 18:26:19.370148	\N	\N	5
165	Turn on the UV light	f	35	2019-09-12 18:26:19.371207	2019-09-12 18:26:19.371207	\N	\N	6
166	Check buffer stock & prepare new stock if needed	f	36	2019-09-12 18:26:19.531767	2019-09-12 18:26:19.531767	\N	\N	0
167	Check stock of reagents & order new stock if needed	f	36	2019-09-12 18:26:19.532906	2019-09-12 18:26:19.532906	\N	\N	1
168	Use gloves at all times	f	36	2019-09-12 18:26:19.535653	2019-09-12 18:26:19.535653	\N	\N	2
169	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	37	2019-11-13 13:57:42.975038	2019-11-13 13:57:42.975038	\N	\N	0
170	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	37	2019-11-13 13:57:42.983021	2019-11-13 13:57:42.983021	\N	\N	1
171	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	37	2019-11-13 13:57:42.984875	2019-11-13 13:57:42.984875	\N	\N	2
172	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	37	2019-11-13 13:57:42.986305	2019-11-13 13:57:42.986305	\N	\N	3
173	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	37	2019-11-13 13:57:42.988075	2019-11-13 13:57:42.988075	\N	\N	4
174	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	37	2019-11-13 13:57:42.993984	2019-11-13 13:57:42.993984	\N	\N	5
175	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	37	2019-11-13 13:57:42.995521	2019-11-13 13:57:42.995521	\N	\N	6
176	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	38	2019-11-13 13:57:43.002412	2019-11-13 13:57:43.002412	\N	\N	0
177	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	38	2019-11-13 13:57:43.004322	2019-11-13 13:57:43.004322	\N	\N	1
178	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	38	2019-11-13 13:57:43.006773	2019-11-13 13:57:43.006773	\N	\N	2
179	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	39	2019-11-13 13:57:43.029539	2019-11-13 13:57:43.029539	\N	\N	0
180	H2O to 12.5 uL	f	40	2019-11-13 13:57:43.079174	2019-11-13 13:57:43.079174	\N	\N	0
181	Optional: Luciferase mRNA (denatured)	f	40	2019-11-13 13:57:43.080484	2019-11-13 13:57:43.080484	\N	\N	1
182	Reverse transcriptase	f	40	2019-11-13 13:57:43.08179	2019-11-13 13:57:43.08179	\N	\N	2
183	RNase inhibitor	f	40	2019-11-13 13:57:43.083056	2019-11-13 13:57:43.083056	\N	\N	3
184	Random Primers	f	40	2019-11-13 13:57:43.084316	2019-11-13 13:57:43.084316	\N	\N	4
185	dNTP mix	f	40	2019-11-13 13:57:43.085565	2019-11-13 13:57:43.085565	\N	\N	5
186	RT buffer	f	40	2019-11-13 13:57:43.087306	2019-11-13 13:57:43.087306	\N	\N	6
187	Make sure the UV light was on at least for 20 minutes before you started to work	f	41	2019-11-13 13:57:43.135182	2019-11-13 13:57:43.135182	\N	\N	0
188	Write down LOT numbers of reagents used	f	41	2019-11-13 13:57:43.136622	2019-11-13 13:57:43.136622	\N	\N	1
189	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	41	2019-11-13 13:57:43.137987	2019-11-13 13:57:43.137987	\N	\N	2
190	Always use designated separate chambers for pipetting samples and reagents	f	41	2019-11-13 13:57:43.13928	2019-11-13 13:57:43.13928	\N	\N	3
191	Change lab coats when switching chambers	f	41	2019-11-13 13:57:43.140559	2019-11-13 13:57:43.140559	\N	\N	4
192	Clean surfaces with 70% ethanol or RNA remover	f	41	2019-11-13 13:57:43.141788	2019-11-13 13:57:43.141788	\N	\N	5
193	Turn on the UV light	f	41	2019-11-13 13:57:43.142954	2019-11-13 13:57:43.142954	\N	\N	6
194	Check buffer stock & prepare new stock if needed	f	42	2019-11-13 13:57:43.387101	2019-11-13 13:57:43.387101	\N	\N	0
195	Check stock of reagents & order new stock if needed	f	42	2019-11-13 13:57:43.388951	2019-11-13 13:57:43.388951	\N	\N	1
196	Use gloves at all times	f	42	2019-11-13 13:57:43.39017	2019-11-13 13:57:43.39017	\N	\N	2
197	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	43	2019-11-25 14:09:28.278051	2019-11-25 14:09:28.278051	\N	\N	0
198	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	43	2019-11-25 14:09:28.28814	2019-11-25 14:09:28.28814	\N	\N	1
199	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	43	2019-11-25 14:09:28.289959	2019-11-25 14:09:28.289959	\N	\N	2
200	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	43	2019-11-25 14:09:28.291768	2019-11-25 14:09:28.291768	\N	\N	3
201	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	43	2019-11-25 14:09:28.293311	2019-11-25 14:09:28.293311	\N	\N	4
202	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	43	2019-11-25 14:09:28.294596	2019-11-25 14:09:28.294596	\N	\N	5
203	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	43	2019-11-25 14:09:28.295933	2019-11-25 14:09:28.295933	\N	\N	6
204	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	44	2019-11-25 14:09:28.304	2019-11-25 14:09:28.304	\N	\N	0
205	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	44	2019-11-25 14:09:28.305524	2019-11-25 14:09:28.305524	\N	\N	1
206	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	44	2019-11-25 14:09:28.306697	2019-11-25 14:09:28.306697	\N	\N	2
207	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	45	2019-11-25 14:09:28.331655	2019-11-25 14:09:28.331655	\N	\N	0
208	H2O to 12.5 uL	f	46	2019-11-25 14:09:28.405471	2019-11-25 14:09:28.405471	\N	\N	0
209	Optional: Luciferase mRNA (denatured)	f	46	2019-11-25 14:09:28.406865	2019-11-25 14:09:28.406865	\N	\N	1
210	Reverse transcriptase	f	46	2019-11-25 14:09:28.407985	2019-11-25 14:09:28.407985	\N	\N	2
211	RNase inhibitor	f	46	2019-11-25 14:09:28.409138	2019-11-25 14:09:28.409138	\N	\N	3
212	Random Primers	f	46	2019-11-25 14:09:28.410299	2019-11-25 14:09:28.410299	\N	\N	4
213	dNTP mix	f	46	2019-11-25 14:09:28.411438	2019-11-25 14:09:28.411438	\N	\N	5
214	RT buffer	f	46	2019-11-25 14:09:28.412922	2019-11-25 14:09:28.412922	\N	\N	6
215	Make sure the UV light was on at least for 20 minutes before you started to work	f	47	2019-11-25 14:09:28.456994	2019-11-25 14:09:28.456994	\N	\N	0
216	Write down LOT numbers of reagents used	f	47	2019-11-25 14:09:28.458633	2019-11-25 14:09:28.458633	\N	\N	1
217	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	47	2019-11-25 14:09:28.459898	2019-11-25 14:09:28.459898	\N	\N	2
218	Always use designated separate chambers for pipetting samples and reagents	f	47	2019-11-25 14:09:28.461161	2019-11-25 14:09:28.461161	\N	\N	3
219	Change lab coats when switching chambers	f	47	2019-11-25 14:09:28.462386	2019-11-25 14:09:28.462386	\N	\N	4
220	Clean surfaces with 70% ethanol or RNA remover	f	47	2019-11-25 14:09:28.463797	2019-11-25 14:09:28.463797	\N	\N	5
221	Turn on the UV light	f	47	2019-11-25 14:09:28.46514	2019-11-25 14:09:28.46514	\N	\N	6
222	Check buffer stock & prepare new stock if needed	f	48	2019-11-25 14:09:28.71372	2019-11-25 14:09:28.71372	\N	\N	0
223	Check stock of reagents & order new stock if needed	f	48	2019-11-25 14:09:28.714949	2019-11-25 14:09:28.714949	\N	\N	1
224	Use gloves at all times	f	48	2019-11-25 14:09:28.716238	2019-11-25 14:09:28.716238	\N	\N	2
225	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	49	2019-11-25 18:00:53.517071	2019-11-25 18:00:53.517071	\N	\N	0
226	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	49	2019-11-25 18:00:53.519098	2019-11-25 18:00:53.519098	\N	\N	1
227	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	49	2019-11-25 18:00:53.520975	2019-11-25 18:00:53.520975	\N	\N	2
228	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	49	2019-11-25 18:00:53.522694	2019-11-25 18:00:53.522694	\N	\N	3
229	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	49	2019-11-25 18:00:53.524082	2019-11-25 18:00:53.524082	\N	\N	4
230	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	49	2019-11-25 18:00:53.526559	2019-11-25 18:00:53.526559	\N	\N	5
231	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	49	2019-11-25 18:00:53.52779	2019-11-25 18:00:53.52779	\N	\N	6
232	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	50	2019-11-25 18:00:53.535014	2019-11-25 18:00:53.535014	\N	\N	0
233	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	50	2019-11-25 18:00:53.537029	2019-11-25 18:00:53.537029	\N	\N	1
234	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	50	2019-11-25 18:00:53.538765	2019-11-25 18:00:53.538765	\N	\N	2
235	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	51	2019-11-25 18:00:53.57198	2019-11-25 18:00:53.57198	\N	\N	0
236	H2O to 12.5 uL	f	52	2019-11-25 18:00:53.624292	2019-11-25 18:00:53.624292	\N	\N	0
237	Optional: Luciferase mRNA (denatured)	f	52	2019-11-25 18:00:53.626133	2019-11-25 18:00:53.626133	\N	\N	1
238	Reverse transcriptase	f	52	2019-11-25 18:00:53.627395	2019-11-25 18:00:53.627395	\N	\N	2
239	RNase inhibitor	f	52	2019-11-25 18:00:53.628798	2019-11-25 18:00:53.628798	\N	\N	3
240	Random Primers	f	52	2019-11-25 18:00:53.63005	2019-11-25 18:00:53.63005	\N	\N	4
241	dNTP mix	f	52	2019-11-25 18:00:53.63131	2019-11-25 18:00:53.63131	\N	\N	5
242	RT buffer	f	52	2019-11-25 18:00:53.632691	2019-11-25 18:00:53.632691	\N	\N	6
243	Make sure the UV light was on at least for 20 minutes before you started to work	f	53	2019-11-25 18:00:53.678687	2019-11-25 18:00:53.678687	\N	\N	0
244	Write down LOT numbers of reagents used	f	53	2019-11-25 18:00:53.680036	2019-11-25 18:00:53.680036	\N	\N	1
245	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	53	2019-11-25 18:00:53.682164	2019-11-25 18:00:53.682164	\N	\N	2
246	Always use designated separate chambers for pipetting samples and reagents	f	53	2019-11-25 18:00:53.68343	2019-11-25 18:00:53.68343	\N	\N	3
247	Change lab coats when switching chambers	f	53	2019-11-25 18:00:53.685067	2019-11-25 18:00:53.685067	\N	\N	4
248	Clean surfaces with 70% ethanol or RNA remover	f	53	2019-11-25 18:00:53.686877	2019-11-25 18:00:53.686877	\N	\N	5
249	Turn on the UV light	f	53	2019-11-25 18:00:53.688296	2019-11-25 18:00:53.688296	\N	\N	6
250	Check buffer stock & prepare new stock if needed	f	54	2019-11-25 18:00:53.917093	2019-11-25 18:00:53.917093	\N	\N	0
251	Check stock of reagents & order new stock if needed	f	54	2019-11-25 18:00:53.918717	2019-11-25 18:00:53.918717	\N	\N	1
252	Use gloves at all times	f	54	2019-11-25 18:00:53.91992	2019-11-25 18:00:53.91992	\N	\N	2
253	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	55	2020-05-12 16:13:06.549503	2020-05-12 16:13:06.549503	\N	\N	0
254	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	55	2020-05-12 16:13:06.557894	2020-05-12 16:13:06.557894	\N	\N	1
255	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	55	2020-05-12 16:13:06.560631	2020-05-12 16:13:06.560631	\N	\N	2
256	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	55	2020-05-12 16:13:06.56234	2020-05-12 16:13:06.56234	\N	\N	3
257	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	55	2020-05-12 16:13:06.565122	2020-05-12 16:13:06.565122	\N	\N	4
258	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	55	2020-05-12 16:13:06.567729	2020-05-12 16:13:06.567729	\N	\N	5
259	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	55	2020-05-12 16:13:06.569089	2020-05-12 16:13:06.569089	\N	\N	6
260	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	56	2020-05-12 16:13:06.573931	2020-05-12 16:13:06.573931	\N	\N	0
261	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	56	2020-05-12 16:13:06.576394	2020-05-12 16:13:06.576394	\N	\N	1
262	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	56	2020-05-12 16:13:06.577708	2020-05-12 16:13:06.577708	\N	\N	2
263	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	57	2020-05-12 16:13:06.595628	2020-05-12 16:13:06.595628	\N	\N	0
264	H2O to 12.5 uL	f	58	2020-05-12 16:13:06.641956	2020-05-12 16:13:06.641956	\N	\N	0
265	Optional: Luciferase mRNA (denatured)	f	58	2020-05-12 16:13:06.643354	2020-05-12 16:13:06.643354	\N	\N	1
266	Reverse transcriptase	f	58	2020-05-12 16:13:06.644934	2020-05-12 16:13:06.644934	\N	\N	2
267	RNase inhibitor	f	58	2020-05-12 16:13:06.646619	2020-05-12 16:13:06.646619	\N	\N	3
268	Random Primers	f	58	2020-05-12 16:13:06.648318	2020-05-12 16:13:06.648318	\N	\N	4
269	dNTP mix	f	58	2020-05-12 16:13:06.650009	2020-05-12 16:13:06.650009	\N	\N	5
270	RT buffer	f	58	2020-05-12 16:13:06.651578	2020-05-12 16:13:06.651578	\N	\N	6
271	Make sure the UV light was on at least for 20 minutes before you started to work	f	59	2020-05-12 16:13:06.713582	2020-05-12 16:13:06.713582	\N	\N	0
272	Write down LOT numbers of reagents used	f	59	2020-05-12 16:13:06.7151	2020-05-12 16:13:06.7151	\N	\N	1
273	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	59	2020-05-12 16:13:06.716604	2020-05-12 16:13:06.716604	\N	\N	2
274	Always use designated separate chambers for pipetting samples and reagents	f	59	2020-05-12 16:13:06.718166	2020-05-12 16:13:06.718166	\N	\N	3
275	Change lab coats when switching chambers	f	59	2020-05-12 16:13:06.719721	2020-05-12 16:13:06.719721	\N	\N	4
276	Clean surfaces with 70% ethanol or RNA remover	f	59	2020-05-12 16:13:06.721243	2020-05-12 16:13:06.721243	\N	\N	5
277	Turn on the UV light	f	59	2020-05-12 16:13:06.722712	2020-05-12 16:13:06.722712	\N	\N	6
278	Check buffer stock & prepare new stock if needed	f	60	2020-05-12 16:13:06.966962	2020-05-12 16:13:06.966962	\N	\N	0
279	Check stock of reagents & order new stock if needed	f	60	2020-05-12 16:13:06.968441	2020-05-12 16:13:06.968441	\N	\N	1
280	Use gloves at all times	f	60	2020-05-12 16:13:06.96989	2020-05-12 16:13:06.96989	\N	\N	2
281	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	61	2020-06-10 14:43:18.400351	2020-06-10 14:43:18.400351	\N	\N	0
282	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	61	2020-06-10 14:43:18.406833	2020-06-10 14:43:18.406833	\N	\N	1
283	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	61	2020-06-10 14:43:18.408281	2020-06-10 14:43:18.408281	\N	\N	2
284	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	61	2020-06-10 14:43:18.409645	2020-06-10 14:43:18.409645	\N	\N	3
285	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	61	2020-06-10 14:43:18.411471	2020-06-10 14:43:18.411471	\N	\N	4
286	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	61	2020-06-10 14:43:18.412802	2020-06-10 14:43:18.412802	\N	\N	5
287	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	61	2020-06-10 14:43:18.415061	2020-06-10 14:43:18.415061	\N	\N	6
288	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	62	2020-06-10 14:43:18.419751	2020-06-10 14:43:18.419751	\N	\N	0
289	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	62	2020-06-10 14:43:18.421221	2020-06-10 14:43:18.421221	\N	\N	1
290	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	62	2020-06-10 14:43:18.422961	2020-06-10 14:43:18.422961	\N	\N	2
291	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	63	2020-06-10 14:43:18.440404	2020-06-10 14:43:18.440404	\N	\N	0
292	H2O to 12.5 uL	f	64	2020-06-10 14:43:18.484288	2020-06-10 14:43:18.484288	\N	\N	0
293	Optional: Luciferase mRNA (denatured)	f	64	2020-06-10 14:43:18.485505	2020-06-10 14:43:18.485505	\N	\N	1
294	Reverse transcriptase	f	64	2020-06-10 14:43:18.486654	2020-06-10 14:43:18.486654	\N	\N	2
295	RNase inhibitor	f	64	2020-06-10 14:43:18.487809	2020-06-10 14:43:18.487809	\N	\N	3
296	Random Primers	f	64	2020-06-10 14:43:18.493143	2020-06-10 14:43:18.493143	\N	\N	4
297	dNTP mix	f	64	2020-06-10 14:43:18.494319	2020-06-10 14:43:18.494319	\N	\N	5
298	RT buffer	f	64	2020-06-10 14:43:18.495483	2020-06-10 14:43:18.495483	\N	\N	6
299	Make sure the UV light was on at least for 20 minutes before you started to work	f	65	2020-06-10 14:43:18.538619	2020-06-10 14:43:18.538619	\N	\N	0
300	Write down LOT numbers of reagents used	f	65	2020-06-10 14:43:18.539828	2020-06-10 14:43:18.539828	\N	\N	1
301	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	65	2020-06-10 14:43:18.541054	2020-06-10 14:43:18.541054	\N	\N	2
302	Always use designated separate chambers for pipetting samples and reagents	f	65	2020-06-10 14:43:18.542274	2020-06-10 14:43:18.542274	\N	\N	3
303	Change lab coats when switching chambers	f	65	2020-06-10 14:43:18.54346	2020-06-10 14:43:18.54346	\N	\N	4
304	Clean surfaces with 70% ethanol or RNA remover	f	65	2020-06-10 14:43:18.544686	2020-06-10 14:43:18.544686	\N	\N	5
305	Turn on the UV light	f	65	2020-06-10 14:43:18.545891	2020-06-10 14:43:18.545891	\N	\N	6
306	Check buffer stock & prepare new stock if needed	f	66	2020-06-10 14:43:18.727005	2020-06-10 14:43:18.727005	\N	\N	0
307	Check stock of reagents & order new stock if needed	f	66	2020-06-10 14:43:18.728282	2020-06-10 14:43:18.728282	\N	\N	1
308	Use gloves at all times	f	66	2020-06-10 14:43:18.729471	2020-06-10 14:43:18.729471	\N	\N	2
309	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	67	2020-10-07 19:20:27.125927	2020-10-07 19:20:27.125927	\N	\N	0
310	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	67	2020-10-07 19:20:27.134583	2020-10-07 19:20:27.134583	\N	\N	1
311	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	67	2020-10-07 19:20:27.136773	2020-10-07 19:20:27.136773	\N	\N	2
312	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	67	2020-10-07 19:20:27.138916	2020-10-07 19:20:27.138916	\N	\N	3
313	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	67	2020-10-07 19:20:27.140264	2020-10-07 19:20:27.140264	\N	\N	4
314	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	67	2020-10-07 19:20:27.141882	2020-10-07 19:20:27.141882	\N	\N	5
315	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	67	2020-10-07 19:20:27.143225	2020-10-07 19:20:27.143225	\N	\N	6
316	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	68	2020-10-07 19:20:27.152286	2020-10-07 19:20:27.152286	\N	\N	0
317	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	68	2020-10-07 19:20:27.153957	2020-10-07 19:20:27.153957	\N	\N	1
318	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	68	2020-10-07 19:20:27.155993	2020-10-07 19:20:27.155993	\N	\N	2
319	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	69	2020-10-07 19:20:27.178112	2020-10-07 19:20:27.178112	\N	\N	0
320	H2O to 12.5 uL	f	70	2020-10-07 19:20:27.226746	2020-10-07 19:20:27.226746	\N	\N	0
321	Optional: Luciferase mRNA (denatured)	f	70	2020-10-07 19:20:27.228142	2020-10-07 19:20:27.228142	\N	\N	1
322	Reverse transcriptase	f	70	2020-10-07 19:20:27.229357	2020-10-07 19:20:27.229357	\N	\N	2
323	RNase inhibitor	f	70	2020-10-07 19:20:27.230506	2020-10-07 19:20:27.230506	\N	\N	3
324	Random Primers	f	70	2020-10-07 19:20:27.231669	2020-10-07 19:20:27.231669	\N	\N	4
325	dNTP mix	f	70	2020-10-07 19:20:27.232962	2020-10-07 19:20:27.232962	\N	\N	5
326	RT buffer	f	70	2020-10-07 19:20:27.234882	2020-10-07 19:20:27.234882	\N	\N	6
327	Make sure the UV light was on at least for 20 minutes before you started to work	f	71	2020-10-07 19:20:27.274898	2020-10-07 19:20:27.274898	\N	\N	0
328	Write down LOT numbers of reagents used	f	71	2020-10-07 19:20:27.276667	2020-10-07 19:20:27.276667	\N	\N	1
329	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	71	2020-10-07 19:20:27.278386	2020-10-07 19:20:27.278386	\N	\N	2
330	Always use designated separate chambers for pipetting samples and reagents	f	71	2020-10-07 19:20:27.279662	2020-10-07 19:20:27.279662	\N	\N	3
331	Change lab coats when switching chambers	f	71	2020-10-07 19:20:27.280931	2020-10-07 19:20:27.280931	\N	\N	4
332	Clean surfaces with 70% ethanol or RNA remover	f	71	2020-10-07 19:20:27.28211	2020-10-07 19:20:27.28211	\N	\N	5
333	Turn on the UV light	f	71	2020-10-07 19:20:27.285121	2020-10-07 19:20:27.285121	\N	\N	6
334	Check buffer stock & prepare new stock if needed	f	72	2020-10-07 19:20:27.601332	2020-10-07 19:20:27.601332	\N	\N	0
335	Check stock of reagents & order new stock if needed	f	72	2020-10-07 19:20:27.602916	2020-10-07 19:20:27.602916	\N	\N	1
336	Use gloves at all times	f	72	2020-10-07 19:20:27.604915	2020-10-07 19:20:27.604915	\N	\N	2
337	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	73	2020-11-18 14:50:54.354905	2020-11-18 14:50:54.354905	\N	\N	0
338	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	73	2020-11-18 14:50:54.365026	2020-11-18 14:50:54.365026	\N	\N	1
339	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	73	2020-11-18 14:50:54.366764	2020-11-18 14:50:54.366764	\N	\N	2
340	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	73	2020-11-18 14:50:54.368263	2020-11-18 14:50:54.368263	\N	\N	3
341	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	73	2020-11-18 14:50:54.369657	2020-11-18 14:50:54.369657	\N	\N	4
342	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	73	2020-11-18 14:50:54.371003	2020-11-18 14:50:54.371003	\N	\N	5
343	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	73	2020-11-18 14:50:54.372646	2020-11-18 14:50:54.372646	\N	\N	6
344	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	74	2020-11-18 14:50:54.381521	2020-11-18 14:50:54.381521	\N	\N	0
345	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	74	2020-11-18 14:50:54.383008	2020-11-18 14:50:54.383008	\N	\N	1
346	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	74	2020-11-18 14:50:54.384429	2020-11-18 14:50:54.384429	\N	\N	2
347	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	75	2020-11-18 14:50:54.409215	2020-11-18 14:50:54.409215	\N	\N	0
348	H2O to 12.5 uL	f	76	2020-11-18 14:50:54.468956	2020-11-18 14:50:54.468956	\N	\N	0
349	Optional: Luciferase mRNA (denatured)	f	76	2020-11-18 14:50:54.47019	2020-11-18 14:50:54.47019	\N	\N	1
350	Reverse transcriptase	f	76	2020-11-18 14:50:54.471405	2020-11-18 14:50:54.471405	\N	\N	2
351	RNase inhibitor	f	76	2020-11-18 14:50:54.472519	2020-11-18 14:50:54.472519	\N	\N	3
352	Random Primers	f	76	2020-11-18 14:50:54.473773	2020-11-18 14:50:54.473773	\N	\N	4
353	dNTP mix	f	76	2020-11-18 14:50:54.474899	2020-11-18 14:50:54.474899	\N	\N	5
354	RT buffer	f	76	2020-11-18 14:50:54.476061	2020-11-18 14:50:54.476061	\N	\N	6
355	Make sure the UV light was on at least for 20 minutes before you started to work	f	77	2020-11-18 14:50:54.527847	2020-11-18 14:50:54.527847	\N	\N	0
356	Write down LOT numbers of reagents used	f	77	2020-11-18 14:50:54.529177	2020-11-18 14:50:54.529177	\N	\N	1
357	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	77	2020-11-18 14:50:54.53042	2020-11-18 14:50:54.53042	\N	\N	2
358	Always use designated separate chambers for pipetting samples and reagents	f	77	2020-11-18 14:50:54.53162	2020-11-18 14:50:54.53162	\N	\N	3
359	Change lab coats when switching chambers	f	77	2020-11-18 14:50:54.532856	2020-11-18 14:50:54.532856	\N	\N	4
360	Clean surfaces with 70% ethanol or RNA remover	f	77	2020-11-18 14:50:54.53405	2020-11-18 14:50:54.53405	\N	\N	5
361	Turn on the UV light	f	77	2020-11-18 14:50:54.535311	2020-11-18 14:50:54.535311	\N	\N	6
362	Check buffer stock & prepare new stock if needed	f	78	2020-11-18 14:50:54.806545	2020-11-18 14:50:54.806545	\N	\N	0
363	Check stock of reagents & order new stock if needed	f	78	2020-11-18 14:50:54.807802	2020-11-18 14:50:54.807802	\N	\N	1
364	Use gloves at all times	f	78	2020-11-18 14:50:54.808968	2020-11-18 14:50:54.808968	\N	\N	2
365	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	79	2020-11-18 14:53:25.781425	2020-11-18 14:53:25.781425	\N	\N	0
401	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	86	2021-05-19 20:01:52.667884	2021-05-19 20:01:52.667884	\N	\N	1
402	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	86	2021-05-19 20:01:52.669105	2021-05-19 20:01:52.669105	\N	\N	2
637	Prepare analytical balance with weighing paper (Vanadium III Chloride will react with plastic weighing boats).	f	131	2021-05-27 16:52:19.595495	2021-05-27 16:52:19.595495	\N	\N	1
366	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	79	2020-11-18 14:53:25.783354	2020-11-18 14:53:25.783354	\N	\N	1
367	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	79	2020-11-18 14:53:25.784968	2020-11-18 14:53:25.784968	\N	\N	2
368	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	79	2020-11-18 14:53:25.786333	2020-11-18 14:53:25.786333	\N	\N	3
369	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	79	2020-11-18 14:53:25.787765	2020-11-18 14:53:25.787765	\N	\N	4
370	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	79	2020-11-18 14:53:25.78911	2020-11-18 14:53:25.78911	\N	\N	5
371	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	79	2020-11-18 14:53:25.790398	2020-11-18 14:53:25.790398	\N	\N	6
372	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	80	2020-11-18 14:53:25.794595	2020-11-18 14:53:25.794595	\N	\N	0
373	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	80	2020-11-18 14:53:25.796479	2020-11-18 14:53:25.796479	\N	\N	1
374	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	80	2020-11-18 14:53:25.798538	2020-11-18 14:53:25.798538	\N	\N	2
375	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	81	2020-11-18 14:53:25.821157	2020-11-18 14:53:25.821157	\N	\N	0
376	H2O to 12.5 uL	f	82	2020-11-18 14:53:25.870042	2020-11-18 14:53:25.870042	\N	\N	0
377	Optional: Luciferase mRNA (denatured)	f	82	2020-11-18 14:53:25.871588	2020-11-18 14:53:25.871588	\N	\N	1
378	Reverse transcriptase	f	82	2020-11-18 14:53:25.873155	2020-11-18 14:53:25.873155	\N	\N	2
379	RNase inhibitor	f	82	2020-11-18 14:53:25.874668	2020-11-18 14:53:25.874668	\N	\N	3
380	Random Primers	f	82	2020-11-18 14:53:25.87625	2020-11-18 14:53:25.87625	\N	\N	4
381	dNTP mix	f	82	2020-11-18 14:53:25.877707	2020-11-18 14:53:25.877707	\N	\N	5
382	RT buffer	f	82	2020-11-18 14:53:25.879273	2020-11-18 14:53:25.879273	\N	\N	6
383	Make sure the UV light was on at least for 20 minutes before you started to work	f	83	2020-11-18 14:53:25.921626	2020-11-18 14:53:25.921626	\N	\N	0
384	Write down LOT numbers of reagents used	f	83	2020-11-18 14:53:25.92342	2020-11-18 14:53:25.92342	\N	\N	1
385	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	83	2020-11-18 14:53:25.925062	2020-11-18 14:53:25.925062	\N	\N	2
386	Always use designated separate chambers for pipetting samples and reagents	f	83	2020-11-18 14:53:25.926602	2020-11-18 14:53:25.926602	\N	\N	3
387	Change lab coats when switching chambers	f	83	2020-11-18 14:53:25.928203	2020-11-18 14:53:25.928203	\N	\N	4
388	Clean surfaces with 70% ethanol or RNA remover	f	83	2020-11-18 14:53:25.929774	2020-11-18 14:53:25.929774	\N	\N	5
389	Turn on the UV light	f	83	2020-11-18 14:53:25.931261	2020-11-18 14:53:25.931261	\N	\N	6
390	Check buffer stock & prepare new stock if needed	f	84	2020-11-18 14:53:26.116348	2020-11-18 14:53:26.116348	\N	\N	0
391	Check stock of reagents & order new stock if needed	f	84	2020-11-18 14:53:26.117686	2020-11-18 14:53:26.117686	\N	\N	1
392	Use gloves at all times	f	84	2020-11-18 14:53:26.118888	2020-11-18 14:53:26.118888	\N	\N	2
393	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	85	2021-05-19 20:01:52.649603	2021-05-19 20:01:52.649603	\N	\N	0
394	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	85	2021-05-19 20:01:52.655047	2021-05-19 20:01:52.655047	\N	\N	1
395	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	85	2021-05-19 20:01:52.656164	2021-05-19 20:01:52.656164	\N	\N	2
396	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	85	2021-05-19 20:01:52.657324	2021-05-19 20:01:52.657324	\N	\N	3
397	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	85	2021-05-19 20:01:52.658226	2021-05-19 20:01:52.658226	\N	\N	4
398	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	85	2021-05-19 20:01:52.659065	2021-05-19 20:01:52.659065	\N	\N	5
399	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	85	2021-05-19 20:01:52.663073	2021-05-19 20:01:52.663073	\N	\N	6
400	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	86	2021-05-19 20:01:52.666874	2021-05-19 20:01:52.666874	\N	\N	0
527	Change lab coats when switching chambers	f	113	2021-05-20 14:21:21.138948	2021-05-20 14:21:21.138948	\N	\N	4
403	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	87	2021-05-19 20:01:52.68222	2021-05-19 20:01:52.68222	\N	\N	0
404	H2O to 12.5 uL	f	88	2021-05-19 20:01:52.710221	2021-05-19 20:01:52.710221	\N	\N	0
405	Optional: Luciferase mRNA (denatured)	f	88	2021-05-19 20:01:52.711018	2021-05-19 20:01:52.711018	\N	\N	1
406	Reverse transcriptase	f	88	2021-05-19 20:01:52.712003	2021-05-19 20:01:52.712003	\N	\N	2
407	RNase inhibitor	f	88	2021-05-19 20:01:52.713038	2021-05-19 20:01:52.713038	\N	\N	3
408	Random Primers	f	88	2021-05-19 20:01:52.713983	2021-05-19 20:01:52.713983	\N	\N	4
409	dNTP mix	f	88	2021-05-19 20:01:52.715243	2021-05-19 20:01:52.715243	\N	\N	5
410	RT buffer	f	88	2021-05-19 20:01:52.716045	2021-05-19 20:01:52.716045	\N	\N	6
411	Make sure the UV light was on at least for 20 minutes before you started to work	f	89	2021-05-19 20:01:52.737796	2021-05-19 20:01:52.737796	\N	\N	0
412	Write down LOT numbers of reagents used	f	89	2021-05-19 20:01:52.738685	2021-05-19 20:01:52.738685	\N	\N	1
413	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	89	2021-05-19 20:01:52.739462	2021-05-19 20:01:52.739462	\N	\N	2
414	Always use designated separate chambers for pipetting samples and reagents	f	89	2021-05-19 20:01:52.740241	2021-05-19 20:01:52.740241	\N	\N	3
415	Change lab coats when switching chambers	f	89	2021-05-19 20:01:52.740993	2021-05-19 20:01:52.740993	\N	\N	4
416	Clean surfaces with 70% ethanol or RNA remover	f	89	2021-05-19 20:01:52.741761	2021-05-19 20:01:52.741761	\N	\N	5
417	Turn on the UV light	f	89	2021-05-19 20:01:52.742496	2021-05-19 20:01:52.742496	\N	\N	6
418	Check buffer stock & prepare new stock if needed	f	90	2021-05-19 20:01:52.856176	2021-05-19 20:01:52.856176	\N	\N	0
419	Check stock of reagents & order new stock if needed	f	90	2021-05-19 20:01:52.856963	2021-05-19 20:01:52.856963	\N	\N	1
420	Use gloves at all times	f	90	2021-05-19 20:01:52.857718	2021-05-19 20:01:52.857718	\N	\N	2
421	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	91	2021-05-20 14:12:31.126112	2021-05-20 14:12:31.126112	\N	\N	0
422	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	91	2021-05-20 14:12:31.127344	2021-05-20 14:12:31.127344	\N	\N	1
423	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	91	2021-05-20 14:12:31.129077	2021-05-20 14:12:31.129077	\N	\N	2
424	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	91	2021-05-20 14:12:31.129987	2021-05-20 14:12:31.129987	\N	\N	3
425	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	91	2021-05-20 14:12:31.130944	2021-05-20 14:12:31.130944	\N	\N	4
426	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	91	2021-05-20 14:12:31.131852	2021-05-20 14:12:31.131852	\N	\N	5
427	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	91	2021-05-20 14:12:31.132702	2021-05-20 14:12:31.132702	\N	\N	6
428	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	92	2021-05-20 14:12:31.136454	2021-05-20 14:12:31.136454	\N	\N	0
429	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	92	2021-05-20 14:12:31.137487	2021-05-20 14:12:31.137487	\N	\N	1
430	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	92	2021-05-20 14:12:31.138297	2021-05-20 14:12:31.138297	\N	\N	2
431	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	93	2021-05-20 14:12:31.152085	2021-05-20 14:12:31.152085	\N	\N	0
432	H2O to 12.5 uL	f	94	2021-05-20 14:12:31.179622	2021-05-20 14:12:31.179622	\N	\N	0
433	Optional: Luciferase mRNA (denatured)	f	94	2021-05-20 14:12:31.18043	2021-05-20 14:12:31.18043	\N	\N	1
434	Reverse transcriptase	f	94	2021-05-20 14:12:31.181195	2021-05-20 14:12:31.181195	\N	\N	2
435	RNase inhibitor	f	94	2021-05-20 14:12:31.181934	2021-05-20 14:12:31.181934	\N	\N	3
436	Random Primers	f	94	2021-05-20 14:12:31.182692	2021-05-20 14:12:31.182692	\N	\N	4
437	dNTP mix	f	94	2021-05-20 14:12:31.183478	2021-05-20 14:12:31.183478	\N	\N	5
438	RT buffer	f	94	2021-05-20 14:12:31.184265	2021-05-20 14:12:31.184265	\N	\N	6
439	Make sure the UV light was on at least for 20 minutes before you started to work	f	95	2021-05-20 14:12:31.207972	2021-05-20 14:12:31.207972	\N	\N	0
440	Write down LOT numbers of reagents used	f	95	2021-05-20 14:12:31.20884	2021-05-20 14:12:31.20884	\N	\N	1
441	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	95	2021-05-20 14:12:31.209643	2021-05-20 14:12:31.209643	\N	\N	2
442	Always use designated separate chambers for pipetting samples and reagents	f	95	2021-05-20 14:12:31.210488	2021-05-20 14:12:31.210488	\N	\N	3
443	Change lab coats when switching chambers	f	95	2021-05-20 14:12:31.211319	2021-05-20 14:12:31.211319	\N	\N	4
444	Clean surfaces with 70% ethanol or RNA remover	f	95	2021-05-20 14:12:31.212099	2021-05-20 14:12:31.212099	\N	\N	5
445	Turn on the UV light	f	95	2021-05-20 14:12:31.212949	2021-05-20 14:12:31.212949	\N	\N	6
446	Check buffer stock & prepare new stock if needed	f	96	2021-05-20 14:12:31.338921	2021-05-20 14:12:31.338921	\N	\N	0
447	Check stock of reagents & order new stock if needed	f	96	2021-05-20 14:12:31.339693	2021-05-20 14:12:31.339693	\N	\N	1
448	Use gloves at all times	f	96	2021-05-20 14:12:31.340629	2021-05-20 14:12:31.340629	\N	\N	2
528	Clean surfaces with 70% ethanol or RNA remover	f	113	2021-05-20 14:21:21.139735	2021-05-20 14:21:21.139735	\N	\N	5
529	Turn on the UV light	f	113	2021-05-20 14:21:21.140508	2021-05-20 14:21:21.140508	\N	\N	6
449	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	97	2021-05-20 14:12:57.681676	2021-05-20 14:12:57.681676	\N	\N	0
450	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	97	2021-05-20 14:12:57.683006	2021-05-20 14:12:57.683006	\N	\N	1
451	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	97	2021-05-20 14:12:57.683985	2021-05-20 14:12:57.683985	\N	\N	2
452	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	97	2021-05-20 14:12:57.684877	2021-05-20 14:12:57.684877	\N	\N	3
453	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	97	2021-05-20 14:12:57.68569	2021-05-20 14:12:57.68569	\N	\N	4
454	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	97	2021-05-20 14:12:57.686554	2021-05-20 14:12:57.686554	\N	\N	5
455	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	97	2021-05-20 14:12:57.687384	2021-05-20 14:12:57.687384	\N	\N	6
456	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	98	2021-05-20 14:12:57.690076	2021-05-20 14:12:57.690076	\N	\N	0
457	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	98	2021-05-20 14:12:57.69107	2021-05-20 14:12:57.69107	\N	\N	1
458	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	98	2021-05-20 14:12:57.691884	2021-05-20 14:12:57.691884	\N	\N	2
459	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	99	2021-05-20 14:12:57.705046	2021-05-20 14:12:57.705046	\N	\N	0
460	H2O to 12.5 uL	f	100	2021-05-20 14:12:57.729793	2021-05-20 14:12:57.729793	\N	\N	0
461	Optional: Luciferase mRNA (denatured)	f	100	2021-05-20 14:12:57.730577	2021-05-20 14:12:57.730577	\N	\N	1
462	Reverse transcriptase	f	100	2021-05-20 14:12:57.731292	2021-05-20 14:12:57.731292	\N	\N	2
463	RNase inhibitor	f	100	2021-05-20 14:12:57.732124	2021-05-20 14:12:57.732124	\N	\N	3
464	Random Primers	f	100	2021-05-20 14:12:57.732911	2021-05-20 14:12:57.732911	\N	\N	4
465	dNTP mix	f	100	2021-05-20 14:12:57.733656	2021-05-20 14:12:57.733656	\N	\N	5
466	RT buffer	f	100	2021-05-20 14:12:57.7344	2021-05-20 14:12:57.7344	\N	\N	6
467	Make sure the UV light was on at least for 20 minutes before you started to work	f	101	2021-05-20 14:12:57.752275	2021-05-20 14:12:57.752275	\N	\N	0
468	Write down LOT numbers of reagents used	f	101	2021-05-20 14:12:57.753073	2021-05-20 14:12:57.753073	\N	\N	1
469	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	101	2021-05-20 14:12:57.753923	2021-05-20 14:12:57.753923	\N	\N	2
470	Always use designated separate chambers for pipetting samples and reagents	f	101	2021-05-20 14:12:57.754806	2021-05-20 14:12:57.754806	\N	\N	3
471	Change lab coats when switching chambers	f	101	2021-05-20 14:12:57.755598	2021-05-20 14:12:57.755598	\N	\N	4
472	Clean surfaces with 70% ethanol or RNA remover	f	101	2021-05-20 14:12:57.756349	2021-05-20 14:12:57.756349	\N	\N	5
473	Turn on the UV light	f	101	2021-05-20 14:12:57.757199	2021-05-20 14:12:57.757199	\N	\N	6
474	Check buffer stock & prepare new stock if needed	f	102	2021-05-20 14:12:57.867429	2021-05-20 14:12:57.867429	\N	\N	0
475	Check stock of reagents & order new stock if needed	f	102	2021-05-20 14:12:57.868707	2021-05-20 14:12:57.868707	\N	\N	1
476	Use gloves at all times	f	102	2021-05-20 14:12:57.869823	2021-05-20 14:12:57.869823	\N	\N	2
477	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	103	2021-05-20 14:13:34.504936	2021-05-20 14:13:34.504936	\N	\N	0
478	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	103	2021-05-20 14:13:34.50698	2021-05-20 14:13:34.50698	\N	\N	1
479	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	103	2021-05-20 14:13:34.507991	2021-05-20 14:13:34.507991	\N	\N	2
480	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	103	2021-05-20 14:13:34.508845	2021-05-20 14:13:34.508845	\N	\N	3
481	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	103	2021-05-20 14:13:34.509762	2021-05-20 14:13:34.509762	\N	\N	4
482	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	103	2021-05-20 14:13:34.510562	2021-05-20 14:13:34.510562	\N	\N	5
483	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	103	2021-05-20 14:13:34.511315	2021-05-20 14:13:34.511315	\N	\N	6
530	Check buffer stock & prepare new stock if needed	f	114	2021-05-20 14:21:21.324218	2021-05-20 14:21:21.324218	\N	\N	0
531	Check stock of reagents & order new stock if needed	f	114	2021-05-20 14:21:21.326637	2021-05-20 14:21:21.326637	\N	\N	1
532	Use gloves at all times	f	114	2021-05-20 14:21:21.330611	2021-05-20 14:21:21.330611	\N	\N	2
484	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	104	2021-05-20 14:13:34.514516	2021-05-20 14:13:34.514516	\N	\N	0
485	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	104	2021-05-20 14:13:34.515584	2021-05-20 14:13:34.515584	\N	\N	1
486	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	104	2021-05-20 14:13:34.517089	2021-05-20 14:13:34.517089	\N	\N	2
487	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	105	2021-05-20 14:13:34.531736	2021-05-20 14:13:34.531736	\N	\N	0
488	H2O to 12.5 uL	f	106	2021-05-20 14:13:34.560269	2021-05-20 14:13:34.560269	\N	\N	0
489	Optional: Luciferase mRNA (denatured)	f	106	2021-05-20 14:13:34.561121	2021-05-20 14:13:34.561121	\N	\N	1
490	Reverse transcriptase	f	106	2021-05-20 14:13:34.562304	2021-05-20 14:13:34.562304	\N	\N	2
491	RNase inhibitor	f	106	2021-05-20 14:13:34.563111	2021-05-20 14:13:34.563111	\N	\N	3
492	Random Primers	f	106	2021-05-20 14:13:34.563895	2021-05-20 14:13:34.563895	\N	\N	4
493	dNTP mix	f	106	2021-05-20 14:13:34.56465	2021-05-20 14:13:34.56465	\N	\N	5
494	RT buffer	f	106	2021-05-20 14:13:34.565405	2021-05-20 14:13:34.565405	\N	\N	6
495	Make sure the UV light was on at least for 20 minutes before you started to work	f	107	2021-05-20 14:13:34.593043	2021-05-20 14:13:34.593043	\N	\N	0
496	Write down LOT numbers of reagents used	f	107	2021-05-20 14:13:34.593876	2021-05-20 14:13:34.593876	\N	\N	1
497	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	107	2021-05-20 14:13:34.594909	2021-05-20 14:13:34.594909	\N	\N	2
498	Always use designated separate chambers for pipetting samples and reagents	f	107	2021-05-20 14:13:34.595692	2021-05-20 14:13:34.595692	\N	\N	3
499	Change lab coats when switching chambers	f	107	2021-05-20 14:13:34.596465	2021-05-20 14:13:34.596465	\N	\N	4
500	Clean surfaces with 70% ethanol or RNA remover	f	107	2021-05-20 14:13:34.597207	2021-05-20 14:13:34.597207	\N	\N	5
501	Turn on the UV light	f	107	2021-05-20 14:13:34.598022	2021-05-20 14:13:34.598022	\N	\N	6
502	Check buffer stock & prepare new stock if needed	f	108	2021-05-20 14:13:34.725426	2021-05-20 14:13:34.725426	\N	\N	0
503	Check stock of reagents & order new stock if needed	f	108	2021-05-20 14:13:34.726349	2021-05-20 14:13:34.726349	\N	\N	1
504	Use gloves at all times	f	108	2021-05-20 14:13:34.727148	2021-05-20 14:13:34.727148	\N	\N	2
505	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	109	2021-05-20 14:21:21.060074	2021-05-20 14:21:21.060074	\N	\N	0
506	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	109	2021-05-20 14:21:21.061289	2021-05-20 14:21:21.061289	\N	\N	1
507	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	109	2021-05-20 14:21:21.062211	2021-05-20 14:21:21.062211	\N	\N	2
508	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	109	2021-05-20 14:21:21.063975	2021-05-20 14:21:21.063975	\N	\N	3
509	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	109	2021-05-20 14:21:21.064829	2021-05-20 14:21:21.064829	\N	\N	4
510	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	109	2021-05-20 14:21:21.066422	2021-05-20 14:21:21.066422	\N	\N	5
511	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	109	2021-05-20 14:21:21.067234	2021-05-20 14:21:21.067234	\N	\N	6
512	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	110	2021-05-20 14:21:21.070282	2021-05-20 14:21:21.070282	\N	\N	0
513	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	110	2021-05-20 14:21:21.071557	2021-05-20 14:21:21.071557	\N	\N	1
514	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	110	2021-05-20 14:21:21.072738	2021-05-20 14:21:21.072738	\N	\N	2
515	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	111	2021-05-20 14:21:21.085202	2021-05-20 14:21:21.085202	\N	\N	0
516	H2O to 12.5 uL	f	112	2021-05-20 14:21:21.110994	2021-05-20 14:21:21.110994	\N	\N	0
517	Optional: Luciferase mRNA (denatured)	f	112	2021-05-20 14:21:21.111825	2021-05-20 14:21:21.111825	\N	\N	1
518	Reverse transcriptase	f	112	2021-05-20 14:21:21.112626	2021-05-20 14:21:21.112626	\N	\N	2
519	RNase inhibitor	f	112	2021-05-20 14:21:21.113513	2021-05-20 14:21:21.113513	\N	\N	3
520	Random Primers	f	112	2021-05-20 14:21:21.114388	2021-05-20 14:21:21.114388	\N	\N	4
521	dNTP mix	f	112	2021-05-20 14:21:21.115162	2021-05-20 14:21:21.115162	\N	\N	5
522	RT buffer	f	112	2021-05-20 14:21:21.115957	2021-05-20 14:21:21.115957	\N	\N	6
523	Make sure the UV light was on at least for 20 minutes before you started to work	f	113	2021-05-20 14:21:21.135816	2021-05-20 14:21:21.135816	\N	\N	0
524	Write down LOT numbers of reagents used	f	113	2021-05-20 14:21:21.136648	2021-05-20 14:21:21.136648	\N	\N	1
525	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	113	2021-05-20 14:21:21.137426	2021-05-20 14:21:21.137426	\N	\N	2
526	Always use designated separate chambers for pipetting samples and reagents	f	113	2021-05-20 14:21:21.13818	2021-05-20 14:21:21.13818	\N	\N	3
533	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	115	2021-05-20 14:30:30.827558	2021-05-20 14:30:30.827558	\N	\N	0
534	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	115	2021-05-20 14:30:30.829174	2021-05-20 14:30:30.829174	\N	\N	1
535	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	115	2021-05-20 14:30:30.830151	2021-05-20 14:30:30.830151	\N	\N	2
536	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	115	2021-05-20 14:30:30.831162	2021-05-20 14:30:30.831162	\N	\N	3
537	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	115	2021-05-20 14:30:30.832024	2021-05-20 14:30:30.832024	\N	\N	4
538	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	115	2021-05-20 14:30:30.832912	2021-05-20 14:30:30.832912	\N	\N	5
539	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	115	2021-05-20 14:30:30.83374	2021-05-20 14:30:30.83374	\N	\N	6
540	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	116	2021-05-20 14:30:30.836948	2021-05-20 14:30:30.836948	\N	\N	0
541	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	116	2021-05-20 14:30:30.83796	2021-05-20 14:30:30.83796	\N	\N	1
542	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	116	2021-05-20 14:30:30.83882	2021-05-20 14:30:30.83882	\N	\N	2
543	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	117	2021-05-20 14:30:30.850753	2021-05-20 14:30:30.850753	\N	\N	0
544	H2O to 12.5 uL	f	118	2021-05-20 14:30:30.889357	2021-05-20 14:30:30.889357	\N	\N	0
545	Optional: Luciferase mRNA (denatured)	f	118	2021-05-20 14:30:30.890467	2021-05-20 14:30:30.890467	\N	\N	1
546	Reverse transcriptase	f	118	2021-05-20 14:30:30.891257	2021-05-20 14:30:30.891257	\N	\N	2
547	RNase inhibitor	f	118	2021-05-20 14:30:30.892078	2021-05-20 14:30:30.892078	\N	\N	3
548	Random Primers	f	118	2021-05-20 14:30:30.892876	2021-05-20 14:30:30.892876	\N	\N	4
549	dNTP mix	f	118	2021-05-20 14:30:30.893919	2021-05-20 14:30:30.893919	\N	\N	5
550	RT buffer	f	118	2021-05-20 14:30:30.894689	2021-05-20 14:30:30.894689	\N	\N	6
551	Make sure the UV light was on at least for 20 minutes before you started to work	f	119	2021-05-20 14:30:30.923836	2021-05-20 14:30:30.923836	\N	\N	0
552	Write down LOT numbers of reagents used	f	119	2021-05-20 14:30:30.924702	2021-05-20 14:30:30.924702	\N	\N	1
553	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	119	2021-05-20 14:30:30.925616	2021-05-20 14:30:30.925616	\N	\N	2
554	Always use designated separate chambers for pipetting samples and reagents	f	119	2021-05-20 14:30:30.926415	2021-05-20 14:30:30.926415	\N	\N	3
555	Change lab coats when switching chambers	f	119	2021-05-20 14:30:30.927212	2021-05-20 14:30:30.927212	\N	\N	4
556	Clean surfaces with 70% ethanol or RNA remover	f	119	2021-05-20 14:30:30.927995	2021-05-20 14:30:30.927995	\N	\N	5
557	Turn on the UV light	f	119	2021-05-20 14:30:30.929296	2021-05-20 14:30:30.929296	\N	\N	6
558	Check buffer stock & prepare new stock if needed	f	120	2021-05-20 14:30:31.062003	2021-05-20 14:30:31.062003	\N	\N	0
559	Check stock of reagents & order new stock if needed	f	120	2021-05-20 14:30:31.062839	2021-05-20 14:30:31.062839	\N	\N	1
560	Use gloves at all times	f	120	2021-05-20 14:30:31.063653	2021-05-20 14:30:31.063653	\N	\N	2
561	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	121	2021-05-20 15:17:12.216878	2021-05-20 15:17:12.216878	\N	\N	0
562	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	121	2021-05-20 15:17:12.218159	2021-05-20 15:17:12.218159	\N	\N	1
563	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	121	2021-05-20 15:17:12.220426	2021-05-20 15:17:12.220426	\N	\N	2
564	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	121	2021-05-20 15:17:12.221259	2021-05-20 15:17:12.221259	\N	\N	3
565	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	121	2021-05-20 15:17:12.222676	2021-05-20 15:17:12.222676	\N	\N	4
566	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	121	2021-05-20 15:17:12.223801	2021-05-20 15:17:12.223801	\N	\N	5
567	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	121	2021-05-20 15:17:12.225049	2021-05-20 15:17:12.225049	\N	\N	6
635	Mark as: Wash Solution HCl, C10, (Date), (Researcher)	f	130	2021-05-27 16:52:19.584829	2021-05-27 16:52:19.584829	\N	\N	5
636	Obtain 300 mL volumetric flask (a 250 mL flask, and a 50 mL flask may be used if a 300 mL flask is not available)	f	131	2021-05-27 16:52:19.593105	2021-05-27 16:52:19.593105	\N	\N	0
568	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	122	2021-05-20 15:17:12.228486	2021-05-20 15:17:12.228486	\N	\N	0
569	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	122	2021-05-20 15:17:12.229604	2021-05-20 15:17:12.229604	\N	\N	1
570	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	122	2021-05-20 15:17:12.230508	2021-05-20 15:17:12.230508	\N	\N	2
571	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	123	2021-05-20 15:17:12.24309	2021-05-20 15:17:12.24309	\N	\N	0
572	H2O to 12.5 uL	f	124	2021-05-20 15:17:12.274236	2021-05-20 15:17:12.274236	\N	\N	0
573	Optional: Luciferase mRNA (denatured)	f	124	2021-05-20 15:17:12.275218	2021-05-20 15:17:12.275218	\N	\N	1
574	Reverse transcriptase	f	124	2021-05-20 15:17:12.276142	2021-05-20 15:17:12.276142	\N	\N	2
575	RNase inhibitor	f	124	2021-05-20 15:17:12.276892	2021-05-20 15:17:12.276892	\N	\N	3
576	Random Primers	f	124	2021-05-20 15:17:12.277814	2021-05-20 15:17:12.277814	\N	\N	4
577	dNTP mix	f	124	2021-05-20 15:17:12.278754	2021-05-20 15:17:12.278754	\N	\N	5
578	RT buffer	f	124	2021-05-20 15:17:12.279588	2021-05-20 15:17:12.279588	\N	\N	6
579	Make sure the UV light was on at least for 20 minutes before you started to work	f	125	2021-05-20 15:17:12.304152	2021-05-20 15:17:12.304152	\N	\N	0
580	Write down LOT numbers of reagents used	f	125	2021-05-20 15:17:12.305029	2021-05-20 15:17:12.305029	\N	\N	1
581	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	125	2021-05-20 15:17:12.305838	2021-05-20 15:17:12.305838	\N	\N	2
582	Always use designated separate chambers for pipetting samples and reagents	f	125	2021-05-20 15:17:12.306604	2021-05-20 15:17:12.306604	\N	\N	3
583	Change lab coats when switching chambers	f	125	2021-05-20 15:17:12.307384	2021-05-20 15:17:12.307384	\N	\N	4
584	Clean surfaces with 70% ethanol or RNA remover	f	125	2021-05-20 15:17:12.30813	2021-05-20 15:17:12.30813	\N	\N	5
585	Turn on the UV light	f	125	2021-05-20 15:17:12.308935	2021-05-20 15:17:12.308935	\N	\N	6
586	Check buffer stock & prepare new stock if needed	f	126	2021-05-20 15:17:12.424297	2021-05-20 15:17:12.424297	\N	\N	0
587	Check stock of reagents & order new stock if needed	f	126	2021-05-20 15:17:12.425196	2021-05-20 15:17:12.425196	\N	\N	1
588	Use gloves at all times	f	126	2021-05-20 15:17:12.425967	2021-05-20 15:17:12.425967	\N	\N	2
589	150 mL Volumetric Flask	f	127	2021-05-26 16:37:18.945619	2021-05-26 16:37:18.945619	\N	\N	0
590	300 mL Volumetric Flask	f	127	2021-05-26 16:37:18.986136	2021-05-26 16:37:18.986136	\N	\N	1
591	150 mL  Volumetric Flask	f	127	2021-05-26 16:37:18.988951	2021-05-26 16:37:18.988951	\N	\N	2
592	150 mL Volumetric Flask	f	127	2021-05-26 16:37:18.997972	2021-05-26 16:37:18.997972	\N	\N	3
593	100 mL Volumetric Flask	f	127	2021-05-26 16:37:19.005217	2021-05-26 16:37:19.005217	\N	\N	4
594	100 mL Volumetric Flask	f	127	2021-05-26 16:37:19.013815	2021-05-26 16:37:19.013815	\N	\N	5
595	100 mL Volumetric Flask	f	127	2021-05-26 16:37:19.016156	2021-05-26 16:37:19.016156	\N	\N	6
596	100 mL Volumetric Flask	f	127	2021-05-26 16:37:19.020908	2021-05-26 16:37:19.020908	\N	\N	7
597	500 mL Volumetric Flask	f	127	2021-05-26 16:37:19.026034	2021-05-26 16:37:19.026034	\N	\N	8
598	25 mL Pipette	f	127	2021-05-26 16:37:19.031023	2021-05-26 16:37:19.031023	\N	\N	9
599	10 mL Pipette	f	127	2021-05-26 16:37:19.037533	2021-05-26 16:37:19.037533	\N	\N	10
600	20 uL Micropipette or Sringe	f	127	2021-05-26 16:37:19.047974	2021-05-26 16:37:19.047974	\N	\N	11
601	10 uL Micropipette or Syringe	f	127	2021-05-26 16:37:19.049915	2021-05-26 16:37:19.049915	\N	\N	12
602	25 uL Micropipette or Syringe	f	127	2021-05-26 16:37:19.053916	2021-05-26 16:37:19.053916	\N	\N	13
603	250 uL Micropipette or Syringe	f	127	2021-05-26 16:37:19.059713	2021-05-26 16:37:19.059713	\N	\N	14
604	150 uL Micropipette or Syringe	f	127	2021-05-26 16:37:19.06413	2021-05-26 16:37:19.06413	\N	\N	15
605	10 mL Graduated Cylinder	f	127	2021-05-26 16:37:19.067603	2021-05-26 16:37:19.067603	\N	\N	16
606	100 mL Graduated Cylinder	f	127	2021-05-26 16:37:19.074023	2021-05-26 16:37:19.074023	\N	\N	17
607	Weighing Boats	f	127	2021-05-26 16:37:19.075388	2021-05-26 16:37:19.075388	\N	\N	18
608	Weighing Papers	f	127	2021-05-26 16:37:19.080764	2021-05-26 16:37:19.080764	\N	\N	19
609	Spatula	f	127	2021-05-26 16:37:19.084727	2021-05-26 16:37:19.084727	\N	\N	20
610	Kimwipes	f	127	2021-05-26 16:37:19.090036	2021-05-26 16:37:19.090036	\N	\N	21
611	Ice Bucket	f	127	2021-05-27 13:52:23.271266	2021-05-27 13:52:23.271266	\N	\N	22
612	HCl	f	128	2021-05-27 14:50:14.286499	2021-05-27 14:50:14.286499	\N	\N	0
613	Vanadium Chloride	f	128	2021-05-27 14:50:14.310974	2021-05-27 14:50:14.310974	\N	\N	1
614	Sulfanilamide	f	128	2021-05-27 14:50:14.32181	2021-05-27 14:50:14.32181	\N	\N	2
616	Potassium Antimony Tartrate	f	128	2021-05-27 14:50:14.562255	2021-05-27 14:50:14.562255	\N	\N	4
617	Sodium Molybdate Dihydrate	f	128	2021-05-27 14:50:14.566192	2021-05-27 14:50:14.566192	\N	\N	5
618	Sulfuric Acid (H2SO4)	f	128	2021-05-27 14:50:14.576564	2021-05-27 14:50:14.576564	\N	\N	6
619	Ascorbic Acid	f	128	2021-05-27 14:50:14.586559	2021-05-27 14:50:14.586559	\N	\N	7
620	NO2 Standard	f	128	2021-05-27 14:50:14.597475	2021-05-27 14:50:14.597475	\N	\N	8
621	NO3 Standard	f	128	2021-05-27 14:50:14.600211	2021-05-27 14:50:14.600211	\N	\N	9
622	PO4 Standard	f	128	2021-05-27 14:50:14.607064	2021-05-27 14:50:14.607064	\N	\N	10
623	Sodium Hypochlorite Solution	f	128	2021-05-27 14:50:14.613932	2021-05-27 14:50:14.613932	\N	\N	11
624	Reagent Grade Water (about 2 liters)	f	128	2021-05-27 14:50:14.622015	2021-05-27 14:50:14.622015	\N	\N	12
625	In 100 mL volumetric flask	f	129	2021-05-27 16:52:19.537601	2021-05-27 16:52:19.537601	\N	\N	0
626	Add about 40 mL dH2O	f	129	2021-05-27 16:52:19.552209	2021-05-27 16:52:19.552209	\N	\N	1
627	Place in ice bath	f	129	2021-05-27 16:52:19.560574	2021-05-27 16:52:19.560574	\N	\N	2
628	Cautiously add, while swirling 10 mL sulfuric acid (H2SO4)	f	129	2021-05-27 16:52:19.562944	2021-05-27 16:52:19.562944	\N	\N	3
629	Chill until next step	f	129	2021-05-27 16:52:19.565528	2021-05-27 16:52:19.565528	\N	\N	4
630	Use 150 mL volumetric flask	f	130	2021-05-27 16:52:19.572413	2021-05-27 16:52:19.572413	\N	\N	0
631	Dispense about 120 mL dH2O to volumetric flask	f	130	2021-05-27 16:52:19.573345	2021-05-27 16:52:19.573345	\N	\N	1
632	Add 15 mL HCl to water, while swirling	f	130	2021-05-27 16:52:19.576664	2021-05-27 16:52:19.576664	\N	\N	2
633	QS to 150 mL with dH2O	f	130	2021-05-27 16:52:19.579761	2021-05-27 16:52:19.579761	\N	\N	3
634	Dispense to 150 mL bottle	f	130	2021-05-27 16:52:19.582027	2021-05-27 16:52:19.582027	\N	\N	4
638	Add 60 mL dH2O to volumetric flask	f	131	2021-05-27 16:52:19.600804	2021-05-27 16:52:19.600804	\N	\N	2
639	Weigh 2.4 g Vanadium III Chloride	f	131	2021-05-27 16:52:19.60544	2021-05-27 16:52:19.60544	\N	\N	3
640	Add Vanadium III Chloride to water, swirl to dissolve	f	131	2021-05-27 16:52:19.606819	2021-05-27 16:52:19.606819	\N	\N	4
641	Complete dissolution by addition of 25.2 mL concentrated HCl	f	131	2021-05-27 16:52:19.609234	2021-05-27 16:52:19.609234	\N	\N	5
642	QS to 300 mL with dH2O	f	131	2021-05-27 16:52:19.637733	2021-05-27 16:52:19.637733	\N	\N	6
643	Dispense to 300 mL bottle	f	131	2021-05-27 16:52:19.67818	2021-05-27 16:52:19.67818	\N	\N	7
644	Mark as: NO3:VCl3 Reduction Reagent, C2, (Date), (Researcher)	f	131	2021-05-27 16:52:19.693251	2021-05-27 16:52:19.693251	\N	\N	8
645	Add 75 mL dH2O to 150 mL volumetric flask	f	132	2021-05-27 16:52:19.717568	2021-05-27 16:52:19.717568	\N	\N	0
646	Weigh 3 g sulfanilamide	f	132	2021-05-27 16:52:19.720301	2021-05-27 16:52:19.720301	\N	\N	1
647	Add sulfanilamide to water in flask, swirl to dissolve	f	132	2021-05-27 16:52:19.725626	2021-05-27 16:52:19.725626	\N	\N	2
648	Pipette 15 mL HCl to flask, while swirling to complete disolution	f	132	2021-05-27 16:52:19.728072	2021-05-27 16:52:19.728072	\N	\N	3
649	QS to 150 mL	f	132	2021-05-27 16:52:19.731104	2021-05-27 16:52:19.731104	\N	\N	4
650	Dispense to 150 mL bottle	f	132	2021-05-27 16:52:19.732011	2021-05-27 16:52:19.732011	\N	\N	5
651	Mark as: NOxSAA, C3, (Date), (Researcher)	f	132	2021-05-27 16:52:19.776786	2021-05-27 16:52:19.776786	\N	\N	6
652	Add approximately 100 mL water to a 150 mL volumetric flask	f	133	2021-05-27 16:52:19.796208	2021-05-27 16:52:19.796208	\N	\N	0
653	Weigh 0.3 g N-(1-Naphthyl) ethylenediamine 2HCl	f	133	2021-05-27 16:52:19.804443	2021-05-27 16:52:19.804443	\N	\N	1
654	Add N-(1-Naphthyl) ethylenediamine 2HCl to water, and swirl to dissolve	f	133	2021-05-27 16:52:19.81035	2021-05-27 16:52:19.81035	\N	\N	2
655	QS to 150 mL	f	133	2021-05-27 16:52:19.820899	2021-05-27 16:52:19.820899	\N	\N	3
656	Dispense to 150 mL bottle	f	133	2021-05-27 16:52:19.829293	2021-05-27 16:52:19.829293	\N	\N	4
657	Mark as: NOxNED, C4, (Date), (Researcher)	f	133	2021-05-27 16:52:19.832289	2021-05-27 16:52:19.832289	\N	\N	5
658	Add about 40 mL water to a clean flask or beaker	f	134	2021-05-27 16:52:19.835241	2021-05-27 16:52:19.835241	\N	\N	0
659	Weigh 0.048 g of Potassium Antimony Tartrate	f	134	2021-05-27 16:52:19.839025	2021-05-27 16:52:19.839025	\N	\N	1
660	Add Potassium Antimony Tartrate to flask or beaker or water	f	134	2021-05-27 16:52:19.841003	2021-05-27 16:52:19.841003	\N	\N	2
661	Weigh 2.22 g Sodium Molybdate Dihydrate	f	134	2021-05-27 16:52:19.857072	2021-05-27 16:52:19.857072	\N	\N	3
662	Add Sodium Molybdate Dihydrate to flask or beaker with Potassium Antimony Tartrate and water and swirl to dissolve	f	134	2021-05-27 16:52:19.878931	2021-05-27 16:52:19.878931	\N	\N	4
663	Add dissolved salt solution to cooled acid solution	f	134	2021-05-27 16:52:19.888786	2021-05-27 16:52:19.888786	\N	\N	5
664	QS to 100 mL	f	134	2021-05-27 16:52:19.926516	2021-05-27 16:52:19.926516	\N	\N	6
665	Dispense to 100 mL bottle	f	134	2021-05-27 16:52:19.934455	2021-05-27 16:52:19.934455	\N	\N	7
666	Mark as: PO4 Color Acid Reagent, C1, (Date), (Researcher)	f	134	2021-05-27 16:52:19.943318	2021-05-27 16:52:19.943318	\N	\N	8
667	Add approximately 70 mL water to a 100 mL volumetric flask	f	135	2021-05-27 16:52:19.954946	2021-05-27 16:52:19.954946	\N	\N	0
668	Weigh 10 g Ascorbic Acid	f	135	2021-05-27 16:52:19.963598	2021-05-27 16:52:19.963598	\N	\N	1
669	Add Ascorbic Acid to water and swirl to dissolve	f	135	2021-05-27 16:52:20.031435	2021-05-27 16:52:20.031435	\N	\N	2
670	QS to 100 mL with dH2O	f	135	2021-05-27 16:52:20.047517	2021-05-27 16:52:20.047517	\N	\N	3
671	Dispense to a 100 mL bottle	f	135	2021-05-27 16:52:20.058264	2021-05-27 16:52:20.058264	\N	\N	4
672	Mark as: Ascorbic Acid Reducing Reagent for PO4, C5, (Date), (Researcher)	f	135	2021-05-27 16:52:20.071124	2021-05-27 16:52:20.071124	\N	\N	5
673	Dispose of ice	f	136	2021-05-27 16:52:20.085192	2021-05-27 16:52:20.085192	\N	\N	0
674	Dry ice bucket	f	136	2021-05-27 16:52:20.088214	2021-05-27 16:52:20.088214	\N	\N	1
675	Send all dirty glassware to glassware room	f	136	2021-05-27 16:52:20.095784	2021-05-27 16:52:20.095784	\N	\N	2
676	Store reagents in a safe location for trip to field	f	136	2021-05-27 16:52:20.099467	2021-05-27 16:52:20.099467	\N	\N	3
677	Cooler	f	137	2021-05-27 19:19:27.30742	2021-05-27 19:19:27.30742	\N	\N	0
678	Reagent water in instrument bag	f	137	2021-05-27 19:19:27.318181	2021-05-27 19:19:27.318181	\N	\N	1
687	Paper copy of data sheet to fill out	f	137	2021-05-27 19:19:27.342971	2021-05-27 19:19:27.342971	\N	\N	10
688	Pen	f	137	2021-05-27 19:19:27.359337	2021-05-27 19:19:27.359337	\N	\N	11
691	Add about 250 mL dH2O to a 500 mL volumetric Flask	f	138	2021-05-27 19:55:55.181573	2021-05-27 19:55:55.181573	\N	\N	0
692	Pipette 25 uL N as NO2 Stock (1000 PPM) into water in volumetric flask	f	138	2021-05-27 19:55:55.197316	2021-05-27 19:55:55.197316	\N	\N	1
693	Pipette 250 uL N as NO3 Stock (1000 PPM) into water in volumetric flask	f	138	2021-05-27 19:55:55.202354	2021-05-27 19:55:55.202354	\N	\N	2
694	Pipette 150uLK P as PO4 Stock (1000 PPM) into water in volumetric flask	f	138	2021-05-27 19:55:55.205921	2021-05-27 19:55:55.205921	\N	\N	3
695	Mix by swirling	f	138	2021-05-27 19:55:55.210343	2021-05-27 19:55:55.210343	\N	\N	4
696	QS to 500 mL	f	138	2021-05-27 19:55:55.213075	2021-05-27 19:55:55.213075	\N	\N	5
697	Dispense to 500 mL bottle	f	138	2021-05-27 19:55:55.216978	2021-05-27 19:55:55.216978	\N	\N	6
698	Mark as External Calibrant, 50 PPB N as NO2, 500 PPB N as NO3, 300 PPB P as PO4, (Date), (Researcher)	f	138	2021-05-27 19:55:55.222071	2021-05-27 19:55:55.222071	\N	\N	7
699	4 - 100 mL Screw Capped Bottles	f	127	2021-05-27 19:58:35.014342	2021-05-27 19:58:35.014342	\N	\N	23
700	3 - 150 mL Screw Capped Bottles	f	127	2021-05-27 19:58:35.077809	2021-05-27 19:58:35.077809	\N	\N	24
701	1 - 300 mL Screw Capped Bottle	f	127	2021-05-27 19:58:35.082272	2021-05-27 19:58:35.082272	\N	\N	25
702	1 - 500 mL Screw Capped Bottle	f	127	2021-05-27 19:58:35.090791	2021-05-27 19:58:35.090791	\N	\N	26
703	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	139	2021-05-28 15:07:55.254669	2021-05-28 15:07:55.254669	\N	\N	0
704	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	139	2021-05-28 15:07:55.269313	2021-05-28 15:07:55.269313	\N	\N	1
705	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	139	2021-05-28 15:07:55.273973	2021-05-28 15:07:55.273973	\N	\N	2
854	Fill out the calibration sheet to record if the balance is calibrated properly	f	166	2021-08-26 20:26:39.455382	2021-08-26 20:38:33.01167	\N	\N	2
706	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	139	2021-05-28 15:07:55.275574	2021-05-28 15:07:55.275574	\N	\N	3
707	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	139	2021-05-28 15:07:55.277874	2021-05-28 15:07:55.277874	\N	\N	4
708	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	139	2021-05-28 15:07:55.280132	2021-05-28 15:07:55.280132	\N	\N	5
709	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	139	2021-05-28 15:07:55.281127	2021-05-28 15:07:55.281127	\N	\N	6
710	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	140	2021-05-28 15:07:55.285631	2021-05-28 15:07:55.285631	\N	\N	0
711	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	140	2021-05-28 15:07:55.288215	2021-05-28 15:07:55.288215	\N	\N	1
712	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	140	2021-05-28 15:07:55.289671	2021-05-28 15:07:55.289671	\N	\N	2
713	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	141	2021-05-28 15:07:55.309161	2021-05-28 15:07:55.309161	\N	\N	0
714	H2O to 12.5 uL	f	142	2021-05-28 15:07:55.345214	2021-05-28 15:07:55.345214	\N	\N	0
715	Optional: Luciferase mRNA (denatured)	f	142	2021-05-28 15:07:55.346235	2021-05-28 15:07:55.346235	\N	\N	1
716	Reverse transcriptase	f	142	2021-05-28 15:07:55.346989	2021-05-28 15:07:55.346989	\N	\N	2
717	RNase inhibitor	f	142	2021-05-28 15:07:55.348209	2021-05-28 15:07:55.348209	\N	\N	3
718	Random Primers	f	142	2021-05-28 15:07:55.349278	2021-05-28 15:07:55.349278	\N	\N	4
719	dNTP mix	f	142	2021-05-28 15:07:55.350407	2021-05-28 15:07:55.350407	\N	\N	5
720	RT buffer	f	142	2021-05-28 15:07:55.351959	2021-05-28 15:07:55.351959	\N	\N	6
721	Make sure the UV light was on at least for 20 minutes before you started to work	f	143	2021-05-28 15:07:55.39888	2021-05-28 15:07:55.39888	\N	\N	0
722	Write down LOT numbers of reagents used	f	143	2021-05-28 15:07:55.400021	2021-05-28 15:07:55.400021	\N	\N	1
723	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	143	2021-05-28 15:07:55.462352	2021-05-28 15:07:55.462352	\N	\N	2
724	Always use designated separate chambers for pipetting samples and reagents	f	143	2021-05-28 15:07:55.911343	2021-05-28 15:07:55.911343	\N	\N	3
725	Change lab coats when switching chambers	f	143	2021-05-28 15:07:56.481407	2021-05-28 15:07:56.481407	\N	\N	4
726	Clean surfaces with 70% ethanol or RNA remover	f	143	2021-05-28 15:07:56.482573	2021-05-28 15:07:56.482573	\N	\N	5
727	Turn on the UV light	f	143	2021-05-28 15:07:56.483352	2021-05-28 15:07:56.483352	\N	\N	6
728	Check buffer stock & prepare new stock if needed	f	144	2021-05-28 15:07:58.074721	2021-05-28 15:07:58.074721	\N	\N	0
729	Check stock of reagents & order new stock if needed	f	144	2021-05-28 15:07:58.076287	2021-05-28 15:07:58.076287	\N	\N	1
730	Use gloves at all times	f	144	2021-05-28 15:07:58.077151	2021-05-28 15:07:58.077151	\N	\N	2
731	Labels	f	127	2021-06-01 15:53:36.933906	2021-06-01 15:53:36.933906	\N	\N	27
732	Marker	f	127	2021-06-01 15:53:36.952849	2021-06-01 15:53:36.952849	\N	\N	28
679	2 waste bags	f	137	2021-05-27 19:19:27.322689	2021-06-01 16:06:50.910666	\N	\N	2
680	Bottles for samples	f	137	2021-05-27 19:19:27.325994	2021-06-01 16:06:50.917701	\N	\N	6
681	Labels for sample bottles	f	137	2021-05-27 19:19:27.328418	2021-06-01 16:06:50.933867	\N	\N	7
682	Sharpie	f	137	2021-05-27 19:19:27.331166	2021-06-01 16:06:50.936122	\N	\N	8
684	Small tool set (including: wire cutters, open end adjustable wrench, needle nose pliers, flat and Phillips screw drivers, some Allen wrenches, knife, and maybe vice grips.  Other tools as experience shows.)	f	137	2021-05-27 19:19:27.334865	2021-06-01 16:06:50.943484	\N	\N	5
685	Zip ties	f	137	2021-05-27 19:19:27.335703	2021-06-01 16:06:50.944747	\N	\N	4
686	Reagents\r\n    Wash Solution HCl, C10\r\n    NO3:VCl3 Reduction Reagent, C2\r\n    NOxSAA, C3\r\n    NOxNED, C4\r\n    PO4 Color Acid Reagent, C1\r\n    Ascorbic Acid Reducing Reagent for PO4, C5\r\n    Internal Calibrant 200 PPB N as NO3, 100 PPB P as PO4, C6\r\n    Chlorine Wash Solution\r\n    Reagent Water Rinse (at least 2 wash bottles)\r\n    External Calibrant 50 PPB N as NO2, 500 PPB N as NO3, 300 PPB P as PO4	f	137	2021-05-27 19:19:27.33869	2021-06-01 16:06:50.948953	\N	\N	3
690	Additional safety and comfort supplies, such as drinking water (facilities are not available in the water intake structure).	f	137	2021-05-27 19:19:27.401118	2021-06-01 16:06:50.953975	\N	\N	12
689	Latex, or other chemical resistant gloves, safety glasses	f	137	2021-05-27 19:19:27.379293	2021-06-01 16:07:40.017377	\N	\N	13
683	Laboratory notebook	f	137	2021-05-27 19:19:27.333037	2021-06-01 16:11:36.128039	\N	\N	9
733	Load all gear and reagents into vehicle	f	145	2021-06-01 16:51:11.741793	2021-06-01 16:51:11.741793	\N	\N	0
734	Travel to	f	145	2021-06-01 16:51:11.766992	2021-06-01 16:51:11.766992	\N	\N	1
735	Laptop computer with WIZPanel installed, and power supply	f	137	2021-06-01 16:59:33.320866	2021-06-01 16:59:33.320866	\N	\N	14
736	Serial to USB cable	f	137	2021-06-01 16:59:33.329619	2021-06-01 16:59:33.329619	\N	\N	15
737	Keys to driveway and causeway locks and intake structure door (2 keys total),	f	137	2021-06-01 16:59:33.331307	2021-06-01 16:59:33.331307	\N	\N	16
738	Containers for carrying supplies and equipment that may be easily lowered on a rope into the basement where the WIZ is located.  Typically a cooler and backpack.	f	137	2021-06-01 16:59:33.333281	2021-06-01 16:59:33.333281	\N	\N	17
739	Opening lock at entrance to the equestrian area driveway, secure gate to fence and post using wire loop and chain respectively.  Keep lock in vehicle.	f	146	2021-06-01 17:13:31.901496	2021-06-01 17:13:31.901496	\N	\N	0
740	Travel to and open the lock and gate at the exit of the equestrian area driveway.  Open gate and place on open pillar.  keep lock in truck, or secure gate in open position with lock.	f	146	2021-06-01 17:13:31.905633	2021-06-01 17:13:31.905633	\N	\N	1
741	Travel to causeway entrance and unlock gate.  Keep lock in truck.	f	146	2021-06-01 17:13:31.907141	2021-06-01 17:13:31.907141	\N	\N	2
742	Turn vehicle around and reverse down causeway to permit easier access to vehicle trunk	f	146	2021-06-01 17:13:31.909718	2021-06-01 17:13:31.909718	\N	\N	3
743	Park vehicle and unlock the door to the intake structure.  Use the toggle to turn the lock to the open position, and secure the keys.  Carry supplies from vehicle into intake structure.	f	146	2021-06-01 17:13:31.910968	2021-06-01 17:13:31.910968	\N	\N	4
855	Calibrate balance using 1g weight	f	167	2021-08-26 20:55:14.774596	2021-08-26 20:55:14.774596	\N	\N	0
744	The trapdoor to the basement of the structure is located on the far side of the structure.  There is a metal handle, that serves as both key, and handle on the light switch on the wall near the trapdoor.  Turn on this light switch, it operates the lights in the basement, and they require some time to warm up.  Use the handle to open the door.  Make sure the door is locked in the open position and return the handle to the light switch.  There is an extension on the ladder that should be raised and locked in the up position.  This is for stability when climbing up or down the ladder.	f	146	2021-06-01 17:13:31.911953	2021-06-01 17:13:31.911953	\N	\N	5
745	On the rack near the crane in the middle of the room is a rope with a snap swivel, on an orange plastic H-frame rope winder.  Rig the cooler containing reagent bottles, and any other gear to be lowered to the basement to this line.  Lower the line to the basement.  Ensure that any persons are clear of the hatch and that the material being lowered will not fall on them if dropped.	f	146	2021-06-01 17:13:31.914327	2021-06-01 17:13:31.914327	\N	\N	6
747	Send code by text to turn WIZ off\r\nText WIZ 2 Cincinnati Ohio Telephone number (202) 765-5867\r\nSend the code: &key=123&WIZ=0	f	147	2021-06-01 17:17:54.264724	2021-06-01 17:22:34.660804	\N	\N	1
746	Plug in laptop and turn on	f	147	2021-06-01 17:17:54.258054	2021-06-01 19:55:02.65878	\N	\N	2
748	Once the WIZ has shut down, disconnect the top connetor to the WIZ electronics box.  Connect this to the calibration power supply.  Connect the serial port of the calibration power supply to the serial to USB cable that was brought to the field.  Connect the USB to the laptop	f	147	2021-06-01 17:22:34.666289	2021-06-01 19:55:02.66562	\N	\N	3
749	Visually observe WIZ for leaks or other problems.	f	147	2021-06-01 19:55:02.667511	2021-06-01 19:55:02.667511	\N	\N	0
750	Launch WIZPanel	f	147	2021-06-01 19:55:02.670335	2021-06-01 19:55:02.670335	\N	\N	4
751	Select appropriate port in WIZ panel in Main Tab, and select Connect	f	147	2021-06-01 19:55:02.671175	2021-06-01 19:55:02.671175	\N	\N	5
752	Select the "Operation" tab, select the "Results" tab within Operations.  Select "Read last position".  The box "To position" will populate with a number.  Record this number in lab form and/or notebook.	f	147	2021-06-01 19:55:02.672448	2021-06-01 19:55:02.672448	\N	\N	6
753	(Note there are only 450 positions in memory.  Once position 450 is filed the instrument begins writing again at 1)	f	147	2021-06-01 19:55:02.674419	2021-06-01 19:55:02.674419	\N	\N	7
754	Run a Blank.  Remove the sample tube from the connector to the filtrate bag.  Immerse this in reagent water. Select the "Direct commands" tab in the "Operation" tab of WIZPanel.  Select the "GENERAL" tab within this.  Select the command "START ANALYSIS".  Select the "Graph" tab.  Select the "Start Chart" button.  When the analysis has finished select the "Stop Chart" button.  Click on the "Results" tab.  enter the last result number from the previous step in the "From position" box, add 3 (1 each for each analyte, NO2, NO3, and PO4) and enter the result into the "To position" box.  Click the "Start Download" button and record the results on data sheet and/or notebook.	f	147	2021-06-01 19:55:02.676392	2021-06-01 19:55:02.676392	\N	\N	8
755	Remove the reagent water from the inlet, and replace with external calibrant	f	147	2021-06-01 19:55:02.678304	2021-06-01 19:55:02.678304	\N	\N	9
756	Select the "Direct commands" tab in the "Operation" tab of WIZPanel.  Select the "GENERAL" tab within this.  Select the command "START ANALYSIS".  Select the "Graph" tab.  Select the "Start Chart" button.  When the analysis has finished select the "Stop Chart" button.  Click on the "Results" tab.  enter the last result number from the previous step in the "From position" box, add 3 (1 each for each analyte, NO2, NO3, and PO4) and enter the result into the "To position" box.  Click the "Start Download" button and record the results on data sheet and/or notebook.	f	147	2021-06-01 19:55:02.68057	2021-06-01 19:55:02.68057	\N	\N	10
757	Connect bleach to water inlet and go to "GENERAL" tab.  Hit "Start Wash".  Go to Graph tab, and hit "Start Chart".  When the wash has finished, hit "Stop Chart" and return to the "Direct Commands" tab.	f	147	2021-06-01 19:55:02.682207	2021-06-01 19:55:02.682207	\N	\N	11
758	Repeat the bleach wash twice, for a total of 3 washes.	f	147	2021-06-01 19:55:02.684184	2021-06-01 19:55:02.684184	\N	\N	12
759	Remove bleach and seal bottle.  Reconnect reagent water.	f	147	2021-06-01 19:55:02.684959	2021-06-01 19:55:02.684959	\N	\N	13
760	Repeat wash process with reagent water 3 times.	f	147	2021-06-01 19:55:02.687277	2021-06-01 19:55:02.687277	\N	\N	14
761	When reagent water wash is complete replace reagents.  Replace each reagent, carefully placing the tube into the new bottle or reagents and sealing the top with parafilm.	f	147	2021-06-01 19:55:02.68803	2021-06-01 19:55:02.68803	\N	\N	15
762	Remove the waste line from the waste bags hanging on the WIZ frame.  Remove the caps from the waste bags that were transported to the field.  Place the caps from the new waste bags onto the old waste bags to ensure they do not leak.  Use a wire cutter or other device to remove the ties holding the waste bags onto the frame.  Place the filled waste bags in a safe location.  Attach the waste lines to new waste bags.  Fasten the new waste bags to the frame using zip ties.  Record this in the data sheet and/or laboratory notebook.	f	147	2021-06-01 19:55:02.690671	2021-06-01 19:55:02.690671	\N	\N	16
763	Remove the reagent water supply bag from the frame.  Remove the line from the reagent water supply bag, and attach to the new supply bag.  Attach the new water supply to the frame using zip ties.  Record this in the data sheet and/or laboratory notebook.	f	147	2021-06-01 19:55:02.693062	2021-06-01 19:55:02.693062	\N	\N	17
764	Prime all reagents for all methods by sending the entering 004 into the box on the "GENERAL" tab, next to the Send FXXX button.  Press the "Send FXXX" button to send this command.	f	147	2021-06-01 19:55:02.694929	2021-06-01 19:55:02.694929	\N	\N	18
765	Press the "START WASH" command.  Wait until the instrument returns to waiting status, and press "START WASH" again.	f	147	2021-06-01 19:55:02.697543	2021-06-01 19:55:02.697543	\N	\N	19
766	On the "GENERAL" tab in the "LED current setting" box, enter 0 into the "LED N." box. Press the button that says "READ", then press send.  In the "Sample and reference visualization", press start.  Record the values in the data sheet and/or laboratory notebook.	f	147	2021-06-01 19:55:02.699297	2021-06-01 19:55:02.699297	\N	\N	20
767	The Sample	f	147	2021-06-01 19:55:02.700096	2021-06-01 19:55:02.700096	\N	\N	21
768	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	148	2021-06-02 16:10:05.033853	2021-06-02 16:10:05.033853	\N	\N	0
769	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	148	2021-06-02 16:10:05.04234	2021-06-02 16:10:05.04234	\N	\N	1
770	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	148	2021-06-02 16:10:05.051581	2021-06-02 16:10:05.051581	\N	\N	2
771	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	148	2021-06-02 16:10:05.058013	2021-06-02 16:10:05.058013	\N	\N	3
772	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	148	2021-06-02 16:10:05.064539	2021-06-02 16:10:05.064539	\N	\N	4
773	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	148	2021-06-02 16:10:05.073515	2021-06-02 16:10:05.073515	\N	\N	5
774	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	148	2021-06-02 16:10:05.080547	2021-06-02 16:10:05.080547	\N	\N	6
775	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	149	2021-06-02 16:10:05.100753	2021-06-02 16:10:05.100753	\N	\N	0
776	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	149	2021-06-02 16:10:05.111885	2021-06-02 16:10:05.111885	\N	\N	1
777	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	149	2021-06-02 16:10:05.127113	2021-06-02 16:10:05.127113	\N	\N	2
778	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	150	2021-06-02 16:10:05.220637	2021-06-02 16:10:05.220637	\N	\N	0
779	H2O to 12.5 uL	f	151	2021-06-02 16:10:06.566217	2021-06-02 16:10:06.566217	\N	\N	0
780	Optional: Luciferase mRNA (denatured)	f	151	2021-06-02 16:10:06.568997	2021-06-02 16:10:06.568997	\N	\N	1
781	Reverse transcriptase	f	151	2021-06-02 16:10:06.572156	2021-06-02 16:10:06.572156	\N	\N	2
782	RNase inhibitor	f	151	2021-06-02 16:10:06.575451	2021-06-02 16:10:06.575451	\N	\N	3
783	Random Primers	f	151	2021-06-02 16:10:06.578806	2021-06-02 16:10:06.578806	\N	\N	4
784	dNTP mix	f	151	2021-06-02 16:10:06.581975	2021-06-02 16:10:06.581975	\N	\N	5
785	RT buffer	f	151	2021-06-02 16:10:06.586084	2021-06-02 16:10:06.586084	\N	\N	6
786	Make sure the UV light was on at least for 20 minutes before you started to work	f	152	2021-06-02 16:10:06.678466	2021-06-02 16:10:06.678466	\N	\N	0
787	Write down LOT numbers of reagents used	f	152	2021-06-02 16:10:06.682145	2021-06-02 16:10:06.682145	\N	\N	1
788	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	152	2021-06-02 16:10:06.685279	2021-06-02 16:10:06.685279	\N	\N	2
789	Always use designated separate chambers for pipetting samples and reagents	f	152	2021-06-02 16:10:06.689059	2021-06-02 16:10:06.689059	\N	\N	3
790	Change lab coats when switching chambers	f	152	2021-06-02 16:10:06.691747	2021-06-02 16:10:06.691747	\N	\N	4
791	Clean surfaces with 70% ethanol or RNA remover	f	152	2021-06-02 16:10:06.693903	2021-06-02 16:10:06.693903	\N	\N	5
792	Turn on the UV light	f	152	2021-06-02 16:10:06.697068	2021-06-02 16:10:06.697068	\N	\N	6
793	Check buffer stock & prepare new stock if needed	f	153	2021-06-02 16:10:07.393443	2021-06-02 16:10:07.393443	\N	\N	0
794	Check stock of reagents & order new stock if needed	f	153	2021-06-02 16:10:07.396119	2021-06-02 16:10:07.396119	\N	\N	1
795	Use gloves at all times	f	153	2021-06-02 16:10:07.399792	2021-06-02 16:10:07.399792	\N	\N	2
615	N-(1-Naphthyl) ethylenediamine 2HCl	f	128	2021-05-27 14:50:14.32642	2021-07-28 19:42:12.286292	\N	\N	3
796	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	154	2021-08-23 14:58:28.649412	2021-08-23 14:58:28.649412	\N	\N	0
797	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	154	2021-08-23 14:58:28.658127	2021-08-23 14:58:28.658127	\N	\N	1
798	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	154	2021-08-23 14:58:28.660755	2021-08-23 14:58:28.660755	\N	\N	2
799	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	154	2021-08-23 14:58:28.662167	2021-08-23 14:58:28.662167	\N	\N	3
800	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	154	2021-08-23 14:58:28.663165	2021-08-23 14:58:28.663165	\N	\N	4
801	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	154	2021-08-23 14:58:28.665431	2021-08-23 14:58:28.665431	\N	\N	5
802	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	154	2021-08-23 14:58:28.666348	2021-08-23 14:58:28.666348	\N	\N	6
803	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	155	2021-08-23 14:58:28.669593	2021-08-23 14:58:28.669593	\N	\N	0
804	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	155	2021-08-23 14:58:28.671106	2021-08-23 14:58:28.671106	\N	\N	1
805	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	155	2021-08-23 14:58:28.671877	2021-08-23 14:58:28.671877	\N	\N	2
856	Calibrate balance using 10g weight	f	167	2021-08-26 20:55:14.777104	2021-08-26 20:55:14.777104	\N	\N	1
857	Log on calibration sheet	f	167	2021-08-26 20:55:32.914579	2021-08-26 20:55:32.914579	\N	\N	2
806	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	156	2021-08-23 14:58:28.683421	2021-08-23 14:58:28.683421	\N	\N	0
807	H2O to 12.5 uL	f	157	2021-08-23 14:58:28.711936	2021-08-23 14:58:28.711936	\N	\N	0
808	Optional: Luciferase mRNA (denatured)	f	157	2021-08-23 14:58:28.713055	2021-08-23 14:58:28.713055	\N	\N	1
809	Reverse transcriptase	f	157	2021-08-23 14:58:28.714088	2021-08-23 14:58:28.714088	\N	\N	2
810	RNase inhibitor	f	157	2021-08-23 14:58:28.71497	2021-08-23 14:58:28.71497	\N	\N	3
811	Random Primers	f	157	2021-08-23 14:58:28.715825	2021-08-23 14:58:28.715825	\N	\N	4
812	dNTP mix	f	157	2021-08-23 14:58:28.716694	2021-08-23 14:58:28.716694	\N	\N	5
813	RT buffer	f	157	2021-08-23 14:58:28.717601	2021-08-23 14:58:28.717601	\N	\N	6
814	Make sure the UV light was on at least for 20 minutes before you started to work	f	158	2021-08-23 14:58:28.740164	2021-08-23 14:58:28.740164	\N	\N	0
815	Write down LOT numbers of reagents used	f	158	2021-08-23 14:58:28.741238	2021-08-23 14:58:28.741238	\N	\N	1
816	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	158	2021-08-23 14:58:28.742165	2021-08-23 14:58:28.742165	\N	\N	2
817	Always use designated separate chambers for pipetting samples and reagents	f	158	2021-08-23 14:58:28.743005	2021-08-23 14:58:28.743005	\N	\N	3
818	Change lab coats when switching chambers	f	158	2021-08-23 14:58:28.743857	2021-08-23 14:58:28.743857	\N	\N	4
819	Clean surfaces with 70% ethanol or RNA remover	f	158	2021-08-23 14:58:28.744727	2021-08-23 14:58:28.744727	\N	\N	5
820	Turn on the UV light	f	158	2021-08-23 14:58:28.745693	2021-08-23 14:58:28.745693	\N	\N	6
821	Check buffer stock & prepare new stock if needed	f	159	2021-08-23 14:58:28.86311	2021-08-23 14:58:28.86311	\N	\N	0
822	Check stock of reagents & order new stock if needed	f	159	2021-08-23 14:58:28.864236	2021-08-23 14:58:28.864236	\N	\N	1
823	Use gloves at all times	f	159	2021-08-23 14:58:28.86508	2021-08-23 14:58:28.86508	\N	\N	2
824	For optimal results, stabilize harvested tissues immediately in RNAlater RNA Stabilization Reagent (see protocol on page 34). Tissues can be stored in the reagent for up to 1 day at 37°C, 7 days at 15–25°C, or 4 weeks at 2–8°C, or archived at –20°C or –80°C.	f	160	2021-08-23 17:19:39.757481	2021-08-23 17:19:39.757481	\N	\N	0
825	Fresh, frozen, or RNAlater stabilized tissues can be used. Tissues can be stored at –70°C for several months. Flash-freeze tissues in liquid nitrogen, and immediately transfer to –70°C. Do not allow tissues to thaw during weighing or handling prior to disruption in Buffer RLT. Homogenized tissue lysates from step 4 can also be stored at –70°C for several months. Incubate frozen lysates at 37°C in a water bath until completely thawed and salts are dissolved before continuing with step 5. Avoid prolonged incubation, which may compromise RNA integrity.	f	160	2021-08-23 17:19:39.758968	2021-08-23 17:19:39.758968	\N	\N	1
826	If desired, more than 30 mg tissue can be disrupted and homogenized at the start of the procedure (increase the volume of Buffer RLT proportionately). Use a portion of the homogenate corresponding to no more than 30 mg tissue for RNA purification, and store the rest at –80°C.	f	160	2021-08-23 17:19:39.759892	2021-08-23 17:19:39.759892	\N	\N	2
827	Buffer RLT may form a precipitate upon storage. If necessary, redissolve by warming, and then place at room temperature (15–25°C).	f	160	2021-08-23 17:19:39.76125	2021-08-23 17:19:39.76125	\N	\N	3
828	Buffer RLT and Buffer RW1 contain a guanidine salt and are therefore not compatible with disinfecting reagents containing bleach. See page 6 for safety information.	f	160	2021-08-23 17:19:39.764088	2021-08-23 17:19:39.764088	\N	\N	4
829	Perform all steps of the procedure at room temperature. During the procedure, work quickly.	f	160	2021-08-23 17:19:39.765667	2021-08-23 17:19:39.765667	\N	\N	5
830	Perform all centrifugation steps at 20–25°C in a standard microcentrifuge. Ensure that the centrifuge does not cool below 20°C.	f	160	2021-08-23 17:19:39.766495	2021-08-23 17:19:39.766495	\N	\N	6
831	β-Mercaptoethanol (β-ME) must be added to Buffer RLT before use. Add 10 µl β-ME per 1 mL Buffer RLT. Dispense in a fume hood and wear appropriate protective clothing. Buffer RLT containing β-ME can be stored at room temperature (15–25°C) for up to 1 month. Alternatively, add 20 µL of 2 M dithiothreitol (DTT) per 1 mL Buffer RLT. The stock solution of 2 M DTT in water should be prepared fresh or frozen in single-use aliquots. Buffer RLT containing DTT can be stored at room temperature for up to 1 month.	f	161	2021-08-23 17:19:39.769842	2021-08-23 17:19:39.769842	\N	\N	0
832	Buffer RPE is supplied as a concentrate. Before using for the first time, add 4 volumes of ethanol (96–100%) as indicated on the bottle to obtain a working solution.	f	161	2021-08-23 17:19:39.771436	2021-08-23 17:19:39.771436	\N	\N	1
833	If performing optional on-column DNase digestion, prepare DNase I stock solution as described in Appendix D (page 67).	f	161	2021-08-23 17:19:39.772274	2021-08-23 17:19:39.772274	\N	\N	2
834	If performing optional on-column DNase digestion (see “Eliminating genomic DNA contamination”, page 21), follow steps D1–D4 (page 67) after performing this step.	f	162	2021-08-23 17:19:39.78524	2021-08-23 17:19:39.78524	\N	\N	0
835	H2O to 12.5 uL	f	163	2021-08-23 17:19:39.81585	2021-08-23 17:19:39.81585	\N	\N	0
836	Optional: Luciferase mRNA (denatured)	f	163	2021-08-23 17:19:39.816629	2021-08-23 17:19:39.816629	\N	\N	1
837	Reverse transcriptase	f	163	2021-08-23 17:19:39.817656	2021-08-23 17:19:39.817656	\N	\N	2
838	RNase inhibitor	f	163	2021-08-23 17:19:39.818651	2021-08-23 17:19:39.818651	\N	\N	3
839	Random Primers	f	163	2021-08-23 17:19:39.81935	2021-08-23 17:19:39.81935	\N	\N	4
840	dNTP mix	f	163	2021-08-23 17:19:39.820081	2021-08-23 17:19:39.820081	\N	\N	5
841	RT buffer	f	163	2021-08-23 17:19:39.82107	2021-08-23 17:19:39.82107	\N	\N	6
842	Make sure the UV light was on at least for 20 minutes before you started to work	f	164	2021-08-23 17:19:39.852117	2021-08-23 17:19:39.852117	\N	\N	0
843	Write down LOT numbers of reagents used	f	164	2021-08-23 17:19:39.853185	2021-08-23 17:19:39.853185	\N	\N	1
844	Use tips with filtes for pipetting samples; use tips without filters for pipetting reagents	f	164	2021-08-23 17:19:39.853932	2021-08-23 17:19:39.853932	\N	\N	2
845	Always use designated separate chambers for pipetting samples and reagents	f	164	2021-08-23 17:19:39.854669	2021-08-23 17:19:39.854669	\N	\N	3
846	Change lab coats when switching chambers	f	164	2021-08-23 17:19:39.85558	2021-08-23 17:19:39.85558	\N	\N	4
847	Clean surfaces with 70% ethanol or RNA remover	f	164	2021-08-23 17:19:39.856536	2021-08-23 17:19:39.856536	\N	\N	5
848	Turn on the UV light	f	164	2021-08-23 17:19:39.857668	2021-08-23 17:19:39.857668	\N	\N	6
849	Check buffer stock & prepare new stock if needed	f	165	2021-08-23 17:19:40.005881	2021-08-23 17:19:40.005881	\N	\N	0
850	Check stock of reagents & order new stock if needed	f	165	2021-08-23 17:19:40.006768	2021-08-23 17:19:40.006768	\N	\N	1
851	Use gloves at all times	f	165	2021-08-23 17:19:40.008026	2021-08-23 17:19:40.008026	\N	\N	2
852	Calibrate using a 1g weight	f	166	2021-08-26 20:26:39.453267	2021-08-26 20:26:39.453267	\N	\N	0
853	Calibrate using a 10g weight	f	166	2021-08-26 20:26:39.454493	2021-08-26 20:26:39.454493	\N	\N	1
\.


--
-- Data for Name: comments; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.comments (id, message, user_id, created_at, updated_at, last_modified_by_id, type, associated_id, unseen_by) FROM stdin;
1	I ' 've created a demo project	1	2019-05-30 07:11:09.615512	2019-05-30 12:40:53.207999	\N	ProjectComment	1	{}
2	We should have a meeting to discuss sampling parametrs soon.	1	2019-05-27 08:25:52.46382	2019-05-30 12:40:53.623984	\N	TaskComment	1	{}
3	I agree.	1	2019-05-26 18:50:00.736278	2019-05-30 12:40:53.626081	\N	TaskComment	1	{}
4	The samples have arrived.	1	2019-05-30 07:41:34.866553	2019-05-30 12:40:53.627987	\N	TaskComment	2	{}
5	Due date has been postponed for a day.	1	2019-05-26 07:34:32.640182	2019-05-30 12:40:53.62984	\N	TaskComment	3	{}
6	Please show Steve the RT procedure.	1	2019-05-25 05:36:53.391425	2019-05-30 12:40:53.631624	\N	TaskComment	5	{}
7	The results must be very definitive.	1	2019-05-29 01:55:42.414513	2019-05-30 12:40:53.633435	\N	TaskComment	6	{}
8	The due date here is flexible.	1	2019-05-27 16:42:38.027576	2019-05-30 12:40:53.635106	\N	TaskComment	8	{}
9	This looks well.	1	2019-05-30 11:13:21.958043	2019-05-30 12:40:53.849283	\N	StepComment	2	{}
10	Great job!	1	2019-05-30 08:43:55.461797	2019-05-30 12:40:53.854291	\N	StepComment	3	{}
11	Great job!	1	2019-05-30 08:57:22.907569	2019-05-30 12:40:53.859131	\N	StepComment	4	{}
12	Admin I have used different sample [#FBT/2~rep_item~2]	1	2019-05-30 02:09:11.807442	2019-05-30 12:40:53.875587	\N	StepComment	2	{}
13	Admin Please complete this by Monday.	1	2019-05-30 00:14:58.743834	2019-05-30 12:40:53.879255	\N	StepComment	4	{}
14	Great job!	1	2019-05-28 08:34:01.062914	2019-05-30 12:40:53.917282	\N	StepComment	7	{}
15	Thanks for getting this done.	1	2019-05-29 03:18:47.018633	2019-05-30 12:40:53.935944	\N	StepComment	13	{}
16	Admin Please check if results match results in [#qPCR~tsk~6]	1	2019-05-27 13:53:00.266864	2019-05-30 12:40:53.954893	\N	ResultComment	2	{}
17	Thanks for getting this done.	1	2019-05-30 11:17:02.365708	2019-05-30 12:40:53.998714	\N	StepComment	24	{}
18	Great job!	1	2019-05-30 12:16:25.79989	2019-05-30 12:40:54.003	\N	StepComment	25	{}
19	This looks well.	1	2019-05-29 04:35:37.680747	2019-05-30 12:40:54.103829	\N	StepComment	32	{}
20	Great job!	1	2019-05-29 16:40:00.21508	2019-05-30 12:40:54.213413	\N	StepComment	36	{}
21	Great job!	1	2019-05-30 05:11:23.901131	2019-05-30 12:40:54.217933	\N	StepComment	37	{}
22	This looks well.	1	2019-05-30 06:22:20.425549	2019-05-30 12:40:54.224835	\N	StepComment	39	{}
23	I actually ran it for 15 minutes and not 10.	1	2019-05-27 21:20:42.56345	2019-05-30 12:40:54.271218	\N	StepComment	40	{}
24	Admin Could you check if this is okay?	1	2019-05-26 19:25:28.565232	2019-05-30 12:41:21.640856	\N	ResultComment	12	{}
25	Admin please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#FBT/5~rep_item~5]</span> seems to be acting strange?	1	2019-05-29 10:54:21.802517	2019-05-30 12:41:23.751022	\N	ResultComment	17	{}
26	I ' 've created a demo project	4	2019-08-23 07:11:32.361573	2019-08-27 15:14:59.891982	\N	ProjectComment	2	{}
27	We should have a meeting to discuss sampling parametrs soon.	4	2019-08-27 13:04:00.146064	2019-08-27 15:15:01.716998	\N	TaskComment	10	{}
28	I agree.	4	2019-08-27 08:24:22.950563	2019-08-27 15:15:01.719709	\N	TaskComment	10	{}
29	The samples have arrived.	4	2019-08-27 02:15:10.257177	2019-08-27 15:15:01.721512	\N	TaskComment	11	{}
30	Due date has been postponed for a day.	4	2019-08-25 14:11:41.968582	2019-08-27 15:15:01.725742	\N	TaskComment	12	{}
31	Please show Steve the RT procedure.	4	2019-08-22 13:00:54.643573	2019-08-27 15:15:01.730962	\N	TaskComment	14	{}
32	The results must be very definitive.	4	2019-08-22 16:32:20.664546	2019-08-27 15:15:01.732772	\N	TaskComment	15	{}
33	The due date here is flexible.	4	2019-08-23 10:05:43.741385	2019-08-27 15:15:01.737368	\N	TaskComment	17	{}
34	Great job!	4	2019-08-26 21:24:28.596816	2019-08-27 15:15:02.980682	\N	StepComment	43	{}
35	Regina Klepikow I have used different sample [#PCH/2~rep_item~7]	4	2019-08-26 06:55:53.644318	2019-08-27 15:15:03.058848	\N	StepComment	42	{}
36	Regina Klepikow Please complete this by Monday.	4	2019-08-26 07:37:44.586011	2019-08-27 15:15:03.069249	\N	StepComment	44	{}
37	This looks well.	4	2019-08-26 08:55:49.174895	2019-08-27 15:15:03.222524	\N	StepComment	47	{}
38	Great job!	4	2019-08-27 13:59:46.426874	2019-08-27 15:15:03.236261	\N	StepComment	50	{}
39	This looks well.	4	2019-08-27 00:14:28.960234	2019-08-27 15:15:03.247607	\N	StepComment	52	{}
40	Great job!	4	2019-08-26 12:41:02.113893	2019-08-27 15:15:03.254793	\N	StepComment	53	{}
41	Regina Klepikow Please check if results match results in [#qPCR~tsk~F]	4	2019-08-26 07:04:38.351807	2019-08-27 15:15:03.283743	\N	ResultComment	23	{}
42	This looks well.	4	2019-08-26 04:43:16.442197	2019-08-27 15:15:03.31764	\N	StepComment	57	{}
43	This looks well.	4	2019-08-27 09:29:46.745151	2019-08-27 15:15:03.322964	\N	StepComment	58	{}
44	Great job!	4	2019-08-26 15:20:51.338271	2019-08-27 15:15:03.331907	\N	StepComment	60	{}
45	Thanks for getting this done.	4	2019-08-26 07:59:29.446957	2019-08-27 15:15:03.351273	\N	StepComment	65	{}
46	This looks well.	4	2019-08-27 07:46:29.316857	2019-08-27 15:15:03.357012	\N	StepComment	66	{}
47	Great job!	4	2019-08-25 19:15:45.903707	2019-08-27 15:15:03.519355	\N	StepComment	69	{}
48	I actually ran it for 15 minutes and not 10.	4	2019-08-23 07:35:12.357607	2019-08-27 15:15:03.826721	\N	StepComment	80	{}
49	Regina Klepikow Could you check if this is okay?	4	2019-08-26 15:13:44.181799	2019-08-27 15:15:41.29228	\N	ResultComment	33	{}
50	Regina Klepikow please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#PCH/5~rep_item~A]</span> seems to be acting strange?	4	2019-08-23 18:29:57.250483	2019-08-27 15:15:44.294497	\N	ResultComment	38	{}
51	I ' 've created a demo project	1	2019-08-24 01:56:17.402541	2019-08-28 07:30:58.609437	\N	ProjectComment	3	{}
52	We should have a meeting to discuss sampling parametrs soon.	1	2019-08-25 15:57:35.95505	2019-08-28 07:31:00.077285	\N	TaskComment	19	{}
53	I agree.	1	2019-08-25 23:07:18.717794	2019-08-28 07:31:00.093244	\N	TaskComment	19	{}
54	The samples have arrived.	1	2019-08-26 01:24:53.064732	2019-08-28 07:31:00.12175	\N	TaskComment	20	{}
55	Due date has been postponed for a day.	1	2019-08-24 23:46:44.52818	2019-08-28 07:31:00.157282	\N	TaskComment	21	{}
56	Please show Steve the RT procedure.	1	2019-08-24 19:02:49.436686	2019-08-28 07:31:00.176851	\N	TaskComment	23	{}
57	The results must be very definitive.	1	2019-08-23 02:04:26.08146	2019-08-28 07:31:00.18256	\N	TaskComment	24	{}
58	The due date here is flexible.	1	2019-08-24 05:27:16.363707	2019-08-28 07:31:00.194304	\N	TaskComment	26	{}
59	This looks well.	1	2019-08-26 04:35:48.026408	2019-08-28 07:31:00.396061	\N	StepComment	81	{}
60	Great job!	1	2019-08-28 01:15:47.083573	2019-08-28 07:31:00.528567	\N	StepComment	84	{}
61	Admin I have used different sample [#EZP/2~rep_item~C]	1	2019-08-25 10:49:21.484634	2019-08-28 07:31:00.590551	\N	StepComment	82	{}
62	Admin Please complete this by Monday.	1	2019-08-25 10:08:50.855566	2019-08-28 07:31:00.596925	\N	StepComment	84	{}
63	This looks well.	1	2019-08-27 04:07:42.538474	2019-08-28 07:31:00.805559	\N	StepComment	93	{}
64	Admin Please check if results match results in [#qPCR~tsk~O]	1	2019-08-25 23:25:53.732781	2019-08-28 07:31:00.827132	\N	ResultComment	44	{}
65	Great job!	1	2019-08-25 12:31:38.183742	2019-08-28 07:31:00.874573	\N	StepComment	97	{}
66	Great job!	1	2019-08-23 17:00:33.129259	2019-08-28 07:31:00.920952	\N	StepComment	103	{}
67	Great job!	1	2019-08-23 13:00:25.656674	2019-08-28 07:31:00.944167	\N	StepComment	105	{}
68	Great job!	1	2019-08-26 20:36:37.72044	2019-08-28 07:31:01.464535	\N	StepComment	117	{}
69	This looks well.	1	2019-08-27 06:04:32.406154	2019-08-28 07:31:01.477459	\N	StepComment	119	{}
70	I actually ran it for 15 minutes and not 10.	1	2019-08-23 16:19:34.654935	2019-08-28 07:31:01.576579	\N	StepComment	120	{}
71	Admin Could you check if this is okay?	1	2019-08-25 13:01:26.486445	2019-08-28 07:31:03.427947	\N	ResultComment	54	{}
72	Admin please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#EZP/5~rep_item~F]</span> seems to be acting strange?	1	2019-08-24 17:24:14.286969	2019-08-28 07:31:06.118185	\N	ResultComment	59	{}
73	I ' 've created a demo project	6	2019-08-26 13:06:14.337837	2019-08-28 08:34:53.585337	\N	ProjectComment	4	{}
74	We should have a meeting to discuss sampling parametrs soon.	6	2019-08-23 20:17:06.647153	2019-08-28 08:34:55.905383	\N	TaskComment	28	{}
75	I agree.	6	2019-08-23 14:57:07.000467	2019-08-28 08:34:55.943943	\N	TaskComment	28	{}
76	The samples have arrived.	6	2019-08-25 14:38:37.628209	2019-08-28 08:34:55.963874	\N	TaskComment	29	{}
77	Due date has been postponed for a day.	6	2019-08-26 11:24:40.505194	2019-08-28 08:34:55.966251	\N	TaskComment	30	{}
78	Please show Steve the RT procedure.	6	2019-08-26 08:44:37.469086	2019-08-28 08:34:55.968052	\N	TaskComment	32	{}
79	The results must be very definitive.	6	2019-08-24 09:50:37.475129	2019-08-28 08:34:55.984748	\N	TaskComment	33	{}
80	The due date here is flexible.	6	2019-08-23 01:37:26.291517	2019-08-28 08:34:55.986835	\N	TaskComment	35	{}
81	Thanks for getting this done.	6	2019-08-27 05:55:00.433492	2019-08-28 08:34:56.26464	\N	StepComment	122	{}
82	Great job!	6	2019-08-25 16:09:11.249933	2019-08-28 08:34:56.272832	\N	StepComment	123	{}
83	Raghuraman Venkatapathy I have used different sample [#RLY/2~rep_item~H]	6	2019-08-25 02:13:54.516939	2019-08-28 08:34:56.309129	\N	StepComment	122	{}
84	Raghuraman Venkatapathy Please complete this by Monday.	6	2019-08-25 00:48:18.740923	2019-08-28 08:34:56.31406	\N	StepComment	124	{}
85	Thanks for getting this done.	6	2019-08-26 19:15:46.466572	2019-08-28 08:34:56.451846	\N	StepComment	131	{}
86	Thanks for getting this done.	6	2019-08-27 10:39:57.119978	2019-08-28 08:34:56.458922	\N	StepComment	132	{}
87	This looks well.	6	2019-08-26 13:46:34.401388	2019-08-28 08:34:56.486465	\N	StepComment	136	{}
88	Raghuraman Venkatapathy Please check if results match results in [#qPCR~tsk~X]	6	2019-08-25 15:41:51.768888	2019-08-28 08:34:56.4976	\N	ResultComment	65	{}
89	Thanks for getting this done.	6	2019-08-28 01:40:45.099223	2019-08-28 08:34:56.570711	\N	StepComment	144	{}
90	This looks well.	6	2019-08-28 07:55:07.965473	2019-08-28 08:34:56.575857	\N	StepComment	145	{}
91	Great job!	6	2019-08-26 22:00:24.301479	2019-08-28 08:34:56.689333	\N	StepComment	149	{}
92	Thanks for getting this done.	6	2019-08-27 13:47:18.968808	2019-08-28 08:34:57.164543	\N	StepComment	154	{}
93	I actually ran it for 15 minutes and not 10.	6	2019-08-23 04:20:29.638452	2019-08-28 08:34:57.333982	\N	StepComment	160	{}
94	Raghuraman Venkatapathy Could you check if this is okay?	6	2019-08-25 15:03:37.32167	2019-08-28 08:35:49.609242	\N	ResultComment	75	{}
95	Raghuraman Venkatapathy please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#RLY/5~rep_item~K]</span> seems to be acting strange?	6	2019-08-24 12:41:55.875484	2019-08-28 08:35:53.251301	\N	ResultComment	80	{}
96	I ' 've created a demo project	7	2019-08-28 14:49:01.899127	2019-08-29 12:51:23.164999	\N	ProjectComment	5	{}
97	We should have a meeting to discuss sampling parametrs soon.	7	2019-08-27 03:48:29.741783	2019-08-29 12:51:25.610071	\N	TaskComment	37	{}
98	I agree.	7	2019-08-26 18:20:12.383335	2019-08-29 12:51:25.630216	\N	TaskComment	37	{}
99	The samples have arrived.	7	2019-08-27 05:31:01.770267	2019-08-29 12:51:25.638137	\N	TaskComment	38	{}
100	Due date has been postponed for a day.	7	2019-08-29 05:57:51.206486	2019-08-29 12:51:25.64891	\N	TaskComment	39	{}
101	Please show Steve the RT procedure.	7	2019-08-26 16:59:56.71974	2019-08-29 12:51:25.660674	\N	TaskComment	41	{}
102	The results must be very definitive.	7	2019-08-25 02:20:29.159879	2019-08-29 12:51:25.668873	\N	TaskComment	42	{}
103	The due date here is flexible.	7	2019-08-28 16:23:13.082765	2019-08-29 12:51:25.670603	\N	TaskComment	44	{}
104	Great job!	7	2019-08-28 20:54:45.674195	2019-08-29 12:51:26.513274	\N	StepComment	162	{}
105	Thanks for getting this done.	7	2019-08-29 01:25:27.728824	2019-08-29 12:51:26.518541	\N	StepComment	163	{}
106	Steven Baker I have used different sample [#ZAT/2~rep_item~M]	7	2019-08-27 07:26:50.900855	2019-08-29 12:51:26.675803	\N	StepComment	162	{}
107	Steven Baker Please complete this by Monday.	7	2019-08-27 05:30:29.642736	2019-08-29 12:51:26.680263	\N	StepComment	164	{}
108	This looks well.	7	2019-08-29 09:21:28.747104	2019-08-29 12:51:26.837216	\N	StepComment	165	{}
109	Steven Baker Please check if results match results in [#qPCR~tsk~g]	7	2019-08-29 03:57:17.940318	2019-08-29 12:51:26.939165	\N	ResultComment	86	{}
110	Thanks for getting this done.	7	2019-08-29 09:11:18.758898	2019-08-29 12:51:27.039201	\N	StepComment	178	{}
111	Thanks for getting this done.	7	2019-08-29 03:32:51.668618	2019-08-29 12:51:27.078997	\N	StepComment	179	{}
112	Thanks for getting this done.	7	2019-08-24 11:05:46.168928	2019-08-29 12:51:27.188643	\N	StepComment	187	{}
113	This looks well.	7	2019-08-27 19:47:34.046314	2019-08-29 12:51:27.470059	\N	StepComment	188	{}
114	This looks well.	7	2019-08-27 01:11:38.513743	2019-08-29 12:51:27.505687	\N	StepComment	190	{}
115	Thanks for getting this done.	7	2019-08-26 00:16:05.842768	2019-08-29 12:51:28.67433	\N	StepComment	196	{}
116	This looks well.	7	2019-08-29 07:48:41.596262	2019-08-29 12:51:28.741272	\N	StepComment	199	{}
117	I actually ran it for 15 minutes and not 10.	7	2019-08-28 12:21:53.344495	2019-08-29 12:51:29.090297	\N	StepComment	200	{}
118	Steven Baker Could you check if this is okay?	7	2019-08-29 12:38:54.365105	2019-08-29 12:52:33.043935	\N	ResultComment	96	{}
119	Steven Baker please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#ZAT/5~rep_item~P]</span> seems to be acting strange?	7	2019-08-25 02:08:48.548848	2019-08-29 12:52:40.323876	\N	ResultComment	101	{}
120	I ' 've created a demo project	8	2019-09-12 05:45:47.470992	2019-09-12 18:26:18.405289	\N	ProjectComment	6	{}
121	We should have a meeting to discuss sampling parametrs soon.	8	2019-09-11 18:53:12.143705	2019-09-12 18:26:18.852429	\N	TaskComment	46	{}
122	I agree.	8	2019-09-11 14:05:42.824692	2019-09-12 18:26:18.854415	\N	TaskComment	46	{}
123	The samples have arrived.	8	2019-09-11 06:50:17.327236	2019-09-12 18:26:18.856173	\N	TaskComment	47	{}
124	Due date has been postponed for a day.	8	2019-09-07 13:19:24.502413	2019-09-12 18:26:18.857872	\N	TaskComment	48	{}
125	Please show Steve the RT procedure.	8	2019-09-11 07:49:33.298582	2019-09-12 18:26:18.859635	\N	TaskComment	50	{}
126	The results must be very definitive.	8	2019-09-07 06:29:27.380371	2019-09-12 18:26:18.86135	\N	TaskComment	51	{}
127	The due date here is flexible.	8	2019-09-07 22:05:40.741363	2019-09-12 18:26:18.863032	\N	TaskComment	53	{}
128	Great job!	8	2019-09-12 06:50:15.130537	2019-09-12 18:26:19.02193	\N	StepComment	202	{}
129	Michael F Davis I have used different sample [#LJK/2~rep_item~S]	8	2019-09-10 23:53:13.096607	2019-09-12 18:26:19.047947	\N	StepComment	202	{}
130	Michael F Davis Please complete this by Monday.	8	2019-09-10 21:56:30.230127	2019-09-12 18:26:19.053349	\N	StepComment	204	{}
131	This looks well.	8	2019-09-11 15:36:54.194152	2019-09-12 18:26:19.097837	\N	StepComment	205	{}
132	Thanks for getting this done.	8	2019-09-09 02:36:47.816195	2019-09-12 18:26:19.10935	\N	StepComment	208	{}
133	Michael F Davis Please check if results match results in [#qPCR~tsk~p]	8	2019-09-07 23:39:40.635558	2019-09-12 18:26:19.147162	\N	ResultComment	107	{}
134	This looks well.	8	2019-09-12 04:49:03.341111	2019-09-12 18:26:19.192005	\N	StepComment	223	{}
135	Great job!	8	2019-09-11 10:11:18.154608	2019-09-12 18:26:19.197198	\N	StepComment	224	{}
136	Great job!	8	2019-09-12 05:30:30.459321	2019-09-12 18:26:19.202051	\N	StepComment	225	{}
137	Great job!	8	2019-09-12 09:47:16.379732	2019-09-12 18:26:19.209871	\N	StepComment	227	{}
138	Thanks for getting this done.	8	2019-09-12 15:43:00.958031	2019-09-12 18:26:19.314763	\N	StepComment	228	{}
139	This looks well.	8	2019-09-11 22:32:17.567794	2019-09-12 18:26:19.322734	\N	StepComment	230	{}
140	Thanks for getting this done.	8	2019-09-12 14:58:27.805312	2019-09-12 18:26:19.47035	\N	StepComment	235	{}
141	I actually ran it for 15 minutes and not 10.	8	2019-09-07 22:15:07.015465	2019-09-12 18:26:19.594672	\N	StepComment	240	{}
142	Michael F Davis Could you check if this is okay?	8	2019-09-10 04:57:45.112832	2019-09-12 18:26:28.213597	\N	ResultComment	117	{}
143	Michael F Davis please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#LJK/5~rep_item~V]</span> seems to be acting strange?	8	2019-09-07 02:10:56.937949	2019-09-12 18:26:30.526406	\N	ResultComment	122	{}
144	I ' 've created a demo project	9	2019-11-11 02:40:36.360355	2019-11-13 13:57:41.505749	\N	ProjectComment	22	{}
145	We should have a meeting to discuss sampling parametrs soon.	9	2019-11-13 08:49:25.962954	2019-11-13 13:57:42.288498	\N	TaskComment	55	{}
146	I agree.	9	2019-11-13 04:55:49.0439	2019-11-13 13:57:42.291444	\N	TaskComment	55	{}
147	The samples have arrived.	9	2019-11-08 10:51:24.886291	2019-11-13 13:57:42.293765	\N	TaskComment	56	{}
148	Due date has been postponed for a day.	9	2019-11-10 20:05:32.549947	2019-11-13 13:57:42.29619	\N	TaskComment	57	{}
149	Please show Steve the RT procedure.	9	2019-11-12 07:30:57.372631	2019-11-13 13:57:42.298443	\N	TaskComment	59	{}
150	The results must be very definitive.	9	2019-11-13 03:09:58.816014	2019-11-13 13:57:42.300909	\N	TaskComment	60	{}
151	The due date here is flexible.	9	2019-11-13 09:57:27.035223	2019-11-13 13:57:42.303117	\N	TaskComment	62	{}
152	Great job!	9	2019-11-11 22:21:13.299207	2019-11-13 13:57:42.637814	\N	StepComment	243	{}
153	Mustafa Bob I have used different sample [#MSZ/2~rep_item~X]	9	2019-11-08 13:31:10.551134	2019-11-13 13:57:42.667901	\N	StepComment	242	{}
154	Mustafa Bob Please complete this by Monday.	9	2019-11-08 11:34:33.098238	2019-11-13 13:57:42.672978	\N	StepComment	244	{}
155	This looks well.	9	2019-11-10 21:33:29.403222	2019-11-13 13:57:42.750088	\N	StepComment	248	{}
156	This looks well.	9	2019-11-12 16:33:17.375457	2019-11-13 13:57:42.782738	\N	StepComment	253	{}
157	Mustafa Bob Please check if results match results in [#qPCR~tsk~y]	9	2019-11-11 02:58:06.445367	2019-11-13 13:57:42.819117	\N	ResultComment	128	{}
158	Great job!	9	2019-11-12 00:18:46.209523	2019-11-13 13:57:42.857105	\N	StepComment	257	{}
159	This looks well.	9	2019-11-12 06:08:21.748531	2019-11-13 13:57:42.867923	\N	StepComment	258	{}
160	This looks well.	9	2019-11-11 02:23:57.767053	2019-11-13 13:57:42.890159	\N	StepComment	261	{}
161	This looks well.	9	2019-11-13 00:29:25.877152	2019-11-13 13:57:43.0575	\N	StepComment	268	{}
162	This looks well.	9	2019-11-13 01:53:21.920128	2019-11-13 13:57:43.099136	\N	StepComment	272	{}
163	Great job!	9	2019-11-13 00:15:31.95903	2019-11-13 13:57:43.320578	\N	StepComment	274	{}
164	I actually ran it for 15 minutes and not 10.	9	2019-11-13 10:37:55.946605	2019-11-13 13:57:43.440191	\N	StepComment	280	{}
165	Mustafa Bob Could you check if this is okay?	9	2019-11-12 01:10:28.301426	2019-11-13 13:58:13.628712	\N	ResultComment	138	{}
166	Mustafa Bob please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#MSZ/5~rep_item~a]</span> seems to be acting strange?	9	2019-11-13 05:45:13.120964	2019-11-13 13:58:16.82979	\N	ResultComment	143	{}
167	I ' 've created a demo project	4	2019-11-24 00:01:58.720801	2019-11-25 14:09:22.24237	\N	ProjectComment	23	{}
168	We should have a meeting to discuss sampling parametrs soon.	4	2019-11-23 15:45:02.499991	2019-11-25 14:09:25.322928	\N	TaskComment	64	{}
169	I agree.	4	2019-11-22 18:06:05.308791	2019-11-25 14:09:25.342963	\N	TaskComment	64	{}
170	The samples have arrived.	4	2019-11-23 07:57:10.266296	2019-11-25 14:09:25.417428	\N	TaskComment	65	{}
171	Due date has been postponed for a day.	4	2019-11-20 12:07:16.263043	2019-11-25 14:09:25.479663	\N	TaskComment	66	{}
172	Please show Steve the RT procedure.	4	2019-11-21 17:57:46.923687	2019-11-25 14:09:25.491415	\N	TaskComment	68	{}
173	The results must be very definitive.	4	2019-11-25 00:52:16.958162	2019-11-25 14:09:25.503319	\N	TaskComment	69	{}
174	The due date here is flexible.	4	2019-11-21 18:15:55.228507	2019-11-25 14:09:25.542004	\N	TaskComment	71	{}
175	Regina Klepikow I have used different sample [#KOK/2~rep_item~c]	4	2019-11-23 05:05:06.115321	2019-11-25 14:09:27.375332	\N	StepComment	282	{}
176	Regina Klepikow Please complete this by Monday.	4	2019-11-23 05:16:23.648457	2019-11-25 14:09:27.386521	\N	StepComment	284	{}
177	This looks well.	4	2019-11-21 07:02:53.671145	2019-11-25 14:09:27.681652	\N	StepComment	292	{}
178	This looks well.	4	2019-11-24 15:00:55.474748	2019-11-25 14:09:27.715405	\N	StepComment	293	{}
179	This looks well.	4	2019-11-21 11:08:13.448736	2019-11-25 14:09:27.752874	\N	StepComment	295	{}
180	Regina Klepikow Please check if results match results in [#qPCR~tsk~17]	4	2019-11-21 05:02:36.593578	2019-11-25 14:09:27.867327	\N	ResultComment	149	{}
181	Great job!	4	2019-11-24 13:42:15.689454	2019-11-25 14:09:28.021836	\N	StepComment	297	{}
182	Thanks for getting this done.	4	2019-11-24 23:51:04.749896	2019-11-25 14:09:28.047626	\N	StepComment	298	{}
183	This looks well.	4	2019-11-24 18:32:15.947583	2019-11-25 14:09:28.080494	\N	StepComment	299	{}
184	Great job!	4	2019-11-25 11:59:25.867295	2019-11-25 14:09:28.135054	\N	StepComment	301	{}
185	Great job!	4	2019-11-25 01:31:04.052493	2019-11-25 14:09:28.170635	\N	StepComment	302	{}
186	Thanks for getting this done.	4	2019-11-25 05:51:36.318707	2019-11-25 14:09:28.377827	\N	StepComment	309	{}
187	Great job!	4	2019-11-21 20:07:57.150625	2019-11-25 14:09:28.389583	\N	StepComment	310	{}
188	I actually ran it for 15 minutes and not 10.	4	2019-11-21 11:47:51.946417	2019-11-25 14:09:28.767769	\N	StepComment	320	{}
189	Regina Klepikow Could you check if this is okay?	4	2019-11-22 15:24:55.488579	2019-11-25 14:09:30.387277	\N	ResultComment	159	{}
190	Regina Klepikow please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#KOK/5~rep_item~f]</span> seems to be acting strange?	4	2019-11-25 07:30:23.121652	2019-11-25 14:09:32.684283	\N	ResultComment	164	{}
191	I ' 've created a demo project	10	2019-11-23 09:19:00.564306	2019-11-25 18:00:52.448099	\N	ProjectComment	26	{}
192	We should have a meeting to discuss sampling parametrs soon.	10	2019-11-22 07:28:44.446828	2019-11-25 18:00:52.974617	\N	TaskComment	78	{}
193	I agree.	10	2019-11-22 19:22:10.746751	2019-11-25 18:00:52.976988	\N	TaskComment	78	{}
194	The samples have arrived.	10	2019-11-20 17:58:16.406213	2019-11-25 18:00:52.979292	\N	TaskComment	79	{}
195	Due date has been postponed for a day.	10	2019-11-25 09:09:00.816738	2019-11-25 18:00:52.982298	\N	TaskComment	80	{}
196	Please show Steve the RT procedure.	10	2019-11-21 22:04:08.775154	2019-11-25 18:00:52.984274	\N	TaskComment	82	{}
197	The results must be very definitive.	10	2019-11-20 05:53:53.613307	2019-11-25 18:00:52.986384	\N	TaskComment	83	{}
198	The due date here is flexible.	10	2019-11-20 20:40:11.93243	2019-11-25 18:00:52.988297	\N	TaskComment	85	{}
199	This looks well.	10	2019-11-24 12:15:30.532712	2019-11-25 18:00:53.116273	\N	StepComment	321	{}
200	This looks well.	10	2019-11-21 22:20:10.553018	2019-11-25 18:00:53.154462	\N	StepComment	322	{}
201	Thanks for getting this done.	10	2019-11-20 10:37:06.154772	2019-11-25 18:00:53.173097	\N	StepComment	324	{}
202	Laura Webb I have used different sample [#HOY/2~rep_item~h]	10	2019-11-20 12:50:55.613586	2019-11-25 18:00:53.196904	\N	StepComment	322	{}
203	Laura Webb Please complete this by Monday.	10	2019-11-20 11:22:20.141485	2019-11-25 18:00:53.201917	\N	StepComment	324	{}
204	This looks well.	10	2019-11-25 15:08:22.865959	2019-11-25 18:00:53.282283	\N	StepComment	328	{}
205	Great job!	10	2019-11-25 16:52:38.024303	2019-11-25 18:00:53.298496	\N	StepComment	330	{}
206	Thanks for getting this done.	10	2019-11-25 00:45:57.454465	2019-11-25 18:00:53.312014	\N	StepComment	332	{}
207	Thanks for getting this done.	10	2019-11-25 10:14:52.560403	2019-11-25 18:00:53.32792	\N	StepComment	334	{}
208	Laura Webb Please check if results match results in [#qPCR~tsk~1L]	10	2019-11-24 23:40:35.312946	2019-11-25 18:00:53.365253	\N	ResultComment	170	{}
209	This looks well.	10	2019-11-25 14:49:03.311336	2019-11-25 18:00:53.409225	\N	StepComment	337	{}
210	Great job!	10	2019-11-25 04:04:18.832293	2019-11-25 18:00:53.44714	\N	StepComment	343	{}
211	This looks well.	10	2019-11-25 04:36:59.119155	2019-11-25 18:00:53.461175	\N	StepComment	345	{}
212	Great job!	10	2019-11-25 03:26:27.827448	2019-11-25 18:00:53.850367	\N	StepComment	354	{}
213	I actually ran it for 15 minutes and not 10.	10	2019-11-20 02:58:28.009821	2019-11-25 18:00:53.965315	\N	StepComment	360	{}
214	Laura Webb Could you check if this is okay?	10	2019-11-25 04:27:02.692158	2019-11-25 18:01:00.070114	\N	ResultComment	180	{}
215	Laura Webb please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#HOY/5~rep_item~k]</span> seems to be acting strange?	10	2019-11-21 10:58:08.543004	2019-11-25 18:01:03.07736	\N	ResultComment	185	{}
216	I ' 've created a demo project	11	2020-05-12 10:16:54.459514	2020-05-12 16:13:01.695648	\N	ProjectComment	27	{}
217	We should have a meeting to discuss sampling parametrs soon.	11	2020-05-08 06:12:25.895112	2020-05-12 16:13:02.911123	\N	TaskComment	87	{}
218	I agree.	11	2020-05-08 22:06:15.670166	2020-05-12 16:13:02.91444	\N	TaskComment	87	{}
219	The samples have arrived.	11	2020-05-07 11:34:33.758059	2020-05-12 16:13:02.917797	\N	TaskComment	88	{}
220	Due date has been postponed for a day.	11	2020-05-09 02:20:12.603404	2020-05-12 16:13:02.920797	\N	TaskComment	89	{}
221	Please show Steve the RT procedure.	11	2020-05-07 10:18:12.350332	2020-05-12 16:13:02.923324	\N	TaskComment	91	{}
222	The results must be very definitive.	11	2020-05-12 09:36:09.835793	2020-05-12 16:13:02.92527	\N	TaskComment	92	{}
223	The due date here is flexible.	11	2020-05-12 11:35:26.676097	2020-05-12 16:13:02.927595	\N	TaskComment	94	{}
224	Lynn Tran I have used different sample [#DYB/2~rep_item~m]	11	2020-05-06 18:57:25.613993	2020-05-12 16:13:05.42909	\N	StepComment	362	{}
225	Lynn Tran Please complete this by Monday.	11	2020-05-06 18:58:11.957578	2020-05-12 16:13:05.518009	\N	StepComment	364	{}
226	This looks well.	11	2020-05-09 18:05:34.713063	2020-05-12 16:13:06.374963	\N	StepComment	370	{}
227	Lynn Tran Please check if results match results in [#qPCR~tsk~1U]	11	2020-05-09 07:16:11.300359	2020-05-12 16:13:06.415781	\N	ResultComment	191	{}
228	Thanks for getting this done.	11	2020-05-10 17:53:00.971138	2020-05-12 16:13:06.449055	\N	StepComment	377	{}
229	Thanks for getting this done.	11	2020-05-11 15:33:50.125091	2020-05-12 16:13:06.458112	\N	StepComment	378	{}
230	This looks well.	11	2020-05-12 10:07:47.880877	2020-05-12 16:13:06.474814	\N	StepComment	381	{}
231	Great job!	11	2020-05-10 08:38:45.471908	2020-05-12 16:13:06.494424	\N	StepComment	385	{}
232	This looks well.	11	2020-05-12 14:03:51.969436	2020-05-12 16:13:06.662738	\N	StepComment	391	{}
233	Thanks for getting this done.	11	2020-05-12 11:15:02.447404	2020-05-12 16:13:06.673392	\N	StepComment	392	{}
234	Great job!	11	2020-05-11 06:23:52.666159	2020-05-12 16:13:06.902826	\N	StepComment	394	{}
235	I actually ran it for 15 minutes and not 10.	11	2020-05-12 06:43:23.121067	2020-05-12 16:13:07.008499	\N	StepComment	400	{}
236	Lynn Tran Could you check if this is okay?	11	2020-05-11 11:59:47.327018	2020-05-12 16:13:49.818657	\N	ResultComment	201	{}
237	Lynn Tran please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#DYB/5~rep_item~p]</span> seems to be acting strange?	11	2020-05-12 05:52:29.763934	2020-05-12 16:13:52.384458	\N	ResultComment	206	{}
238	I ' 've created a demo project	12	2020-06-07 13:46:56.496225	2020-06-10 14:43:16.894839	\N	ProjectComment	28	{}
239	We should have a meeting to discuss sampling parametrs soon.	12	2020-06-09 15:18:19.077194	2020-06-10 14:43:17.711712	\N	TaskComment	96	{}
240	I agree.	12	2020-06-10 10:11:21.249059	2020-06-10 14:43:17.713923	\N	TaskComment	96	{}
241	The samples have arrived.	12	2020-06-06 07:05:12.653498	2020-06-10 14:43:17.715833	\N	TaskComment	97	{}
242	Due date has been postponed for a day.	12	2020-06-09 06:07:40.876157	2020-06-10 14:43:17.717783	\N	TaskComment	98	{}
243	Please show Steve the RT procedure.	12	2020-06-06 08:18:50.931548	2020-06-10 14:43:17.719837	\N	TaskComment	100	{}
244	The results must be very definitive.	12	2020-06-08 15:18:34.098382	2020-06-10 14:43:17.721865	\N	TaskComment	101	{}
245	The due date here is flexible.	12	2020-06-04 21:57:37.767146	2020-06-10 14:43:17.724267	\N	TaskComment	103	{}
246	This looks well.	12	2020-06-09 06:42:28.854996	2020-06-10 14:43:18.067424	\N	StepComment	402	{}
247	Heath Mash I have used different sample [#RTK/2~rep_item~r]	12	2020-06-05 13:00:51.278794	2020-06-10 14:43:18.099375	\N	StepComment	402	{}
248	Heath Mash Please complete this by Monday.	12	2020-06-05 11:47:05.727983	2020-06-10 14:43:18.103597	\N	StepComment	404	{}
249	Great job!	12	2020-06-10 04:40:15.799772	2020-06-10 14:43:18.182524	\N	StepComment	408	{}
250	Thanks for getting this done.	12	2020-06-08 12:25:18.596553	2020-06-10 14:43:18.189086	\N	StepComment	409	{}
251	Thanks for getting this done.	12	2020-06-08 19:27:24.835393	2020-06-10 14:43:18.207369	\N	StepComment	413	{}
252	Great job!	12	2020-06-08 13:52:18.759188	2020-06-10 14:43:18.213739	\N	StepComment	414	{}
253	Heath Mash Please check if results match results in [#qPCR~tsk~1d]	12	2020-06-09 09:01:28.938157	2020-06-10 14:43:18.240021	\N	ResultComment	212	{}
254	Great job!	12	2020-06-06 14:38:37.906464	2020-06-10 14:43:18.297091	\N	StepComment	421	{}
255	This looks well.	12	2020-06-07 03:23:03.872264	2020-06-10 14:43:18.316201	\N	StepComment	425	{}
256	Great job!	12	2020-06-09 07:23:49.119804	2020-06-10 14:43:18.683673	\N	StepComment	437	{}
257	Great job!	12	2020-06-07 04:23:20.875645	2020-06-10 14:43:18.74771	\N	StepComment	440	{}
258	I actually ran it for 15 minutes and not 10.	12	2020-06-05 01:25:28.693355	2020-06-10 14:43:18.767253	\N	StepComment	440	{}
259	Heath Mash Could you check if this is okay?	12	2020-06-10 12:27:52.517234	2020-06-10 14:44:20.450928	\N	ResultComment	222	{}
260	Heath Mash please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#RTK/5~rep_item~u]</span> seems to be acting strange?	12	2020-06-08 11:30:26.318107	2020-06-10 14:44:22.66328	\N	ResultComment	227	{}
261	test	7	2020-06-10 18:25:03.592784	2020-06-10 18:25:03.592784	\N	ProjectComment	5	{}
262	I ' 've created a demo project	16	2020-10-07 19:01:45.57038	2020-10-07 19:20:24.937434	\N	ProjectComment	33	{}
263	We should have a meeting to discuss sampling parametrs soon.	16	2020-10-07 17:52:20.076351	2020-10-07 19:20:25.878883	\N	TaskComment	106	{}
264	I agree.	16	2020-10-07 11:45:36.382639	2020-10-07 19:20:25.88174	\N	TaskComment	106	{}
265	The samples have arrived.	16	2020-10-06 22:35:11.06319	2020-10-07 19:20:25.883893	\N	TaskComment	107	{}
266	Due date has been postponed for a day.	16	2020-10-03 03:17:48.75569	2020-10-07 19:20:25.886026	\N	TaskComment	108	{}
267	Please show Steve the RT procedure.	16	2020-10-07 15:11:11.710474	2020-10-07 19:20:25.888751	\N	TaskComment	110	{}
268	The results must be very definitive.	16	2020-10-06 17:51:08.343457	2020-10-07 19:20:25.890726	\N	TaskComment	111	{}
269	The due date here is flexible.	16	2020-10-02 08:22:00.01968	2020-10-07 19:20:25.892962	\N	TaskComment	113	{}
270	Kristie Rue I have used different sample [#DPC/2~rep_item~w]	16	2020-10-06 13:42:52.501052	2020-10-07 19:20:26.744629	\N	StepComment	444	{}
271	Kristie Rue Please complete this by Monday.	16	2020-10-06 11:14:47.460007	2020-10-07 19:20:26.751783	\N	StepComment	446	{}
272	Great job!	16	2020-10-06 18:07:16.892274	2020-10-07 19:20:26.840716	\N	StepComment	448	{}
273	Thanks for getting this done.	16	2020-10-06 15:17:16.169997	2020-10-07 19:20:26.847154	\N	StepComment	449	{}
274	Great job!	16	2020-10-04 18:21:23.904181	2020-10-07 19:20:26.874111	\N	StepComment	454	{}
275	This looks well.	16	2020-10-05 10:08:40.292688	2020-10-07 19:20:26.888924	\N	StepComment	456	{}
276	Kristie Rue Please check if results match results in [#qPCR~tsk~1n]	16	2020-10-04 06:35:03.471669	2020-10-07 19:20:26.924393	\N	ResultComment	233	{}
277	This looks well.	16	2020-10-07 17:43:41.407972	2020-10-07 19:20:26.971482	\N	StepComment	459	{}
278	Great job!	16	2020-10-07 05:49:58.597528	2020-10-07 19:20:27.241637	\N	StepComment	473	{}
279	I actually ran it for 15 minutes and not 10.	16	2020-10-02 02:42:02.266201	2020-10-07 19:20:27.650951	\N	StepComment	482	{}
280	Kristie Rue Could you check if this is okay?	16	2020-10-04 05:46:30.420542	2020-10-07 19:20:34.555765	\N	ResultComment	243	{}
349	Great job!	20	2021-05-19 19:56:46.717606	2021-05-19 20:01:52.606799	\N	StepComment	588	{}
281	Kristie Rue please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#DPC/5~rep_item~z]</span> seems to be acting strange?	16	2020-10-06 02:42:45.458174	2020-10-07 19:20:36.968277	\N	ResultComment	248	{}
282	I ' 've created a demo project	1	2020-11-17 16:22:12.527391	2020-11-18 14:50:52.169498	\N	ProjectComment	34	{}
283	We should have a meeting to discuss sampling parametrs soon.	1	2020-11-15 19:01:11.261309	2020-11-18 14:50:53.426543	\N	TaskComment	115	{}
284	I agree.	1	2020-11-16 01:15:16.662161	2020-11-18 14:50:53.431132	\N	TaskComment	115	{}
285	The samples have arrived.	1	2020-11-17 16:58:39.693013	2020-11-18 14:50:53.435506	\N	TaskComment	116	{}
286	Due date has been postponed for a day.	1	2020-11-16 11:43:52.689463	2020-11-18 14:50:53.439869	\N	TaskComment	117	{}
287	Please show Steve the RT procedure.	1	2020-11-13 05:01:39.513237	2020-11-18 14:50:53.44425	\N	TaskComment	119	{}
288	The results must be very definitive.	1	2020-11-14 17:23:00.899252	2020-11-18 14:50:53.448156	\N	TaskComment	120	{}
289	The due date here is flexible.	1	2020-11-15 11:21:46.256302	2020-11-18 14:50:53.451801	\N	TaskComment	122	{}
290	Thanks for getting this done.	1	2020-11-18 12:47:43.791087	2020-11-18 14:50:53.851552	\N	StepComment	485	{}
291	Admin I have used different sample [#DYV/2~rep_item~11]	1	2020-11-17 12:25:21.521656	2020-11-18 14:50:53.923133	\N	StepComment	484	{}
292	Admin Please complete this by Monday.	1	2020-11-17 15:18:39.333994	2020-11-18 14:50:53.935842	\N	StepComment	486	{}
293	Thanks for getting this done.	1	2020-11-17 11:54:58.033465	2020-11-18 14:50:54.04375	\N	StepComment	489	{}
294	This looks well.	1	2020-11-18 09:38:54.948031	2020-11-18 14:50:54.140774	\N	StepComment	497	{}
295	Admin Please check if results match results in [#qPCR~tsk~1w]	1	2020-11-16 11:45:53.10927	2020-11-18 14:50:54.165584	\N	ResultComment	254	{}
296	Great job!	1	2020-11-17 12:31:31.999101	2020-11-18 14:50:54.214265	\N	StepComment	499	{}
297	Great job!	1	2020-11-17 03:35:56.208817	2020-11-18 14:50:54.238271	\N	StepComment	502	{}
298	This looks well.	1	2020-11-14 16:00:30.252195	2020-11-18 14:50:54.266862	\N	StepComment	506	{}
299	Great job!	1	2020-11-18 03:17:39.10273	2020-11-18 14:50:54.278162	\N	StepComment	507	{}
300	Thanks for getting this done.	1	2020-11-14 22:37:06.69601	2020-11-18 14:50:54.458093	\N	StepComment	512	{}
301	Thanks for getting this done.	1	2020-11-17 05:16:54.744402	2020-11-18 14:50:54.49375	\N	StepComment	514	{}
302	Thanks for getting this done.	1	2020-11-14 05:19:02.658275	2020-11-18 14:50:54.719249	\N	StepComment	516	{}
303	This looks well.	1	2020-11-14 17:25:01.090524	2020-11-18 14:50:54.731122	\N	StepComment	517	{}
304	This looks well.	1	2020-11-15 23:39:13.333511	2020-11-18 14:50:54.74312	\N	StepComment	518	{}
305	This looks well.	1	2020-11-17 20:50:04.619853	2020-11-18 14:50:54.83611	\N	StepComment	522	{}
306	I actually ran it for 15 minutes and not 10.	1	2020-11-14 16:00:33.853779	2020-11-18 14:50:54.869115	\N	StepComment	522	{}
307	Admin Could you check if this is okay?	1	2020-11-18 05:08:32.865536	2020-11-18 14:50:56.183627	\N	ResultComment	264	{}
308	Admin please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#DYV/5~rep_item~14]</span> seems to be acting strange?	1	2020-11-14 15:10:35.660744	2020-11-18 14:50:58.542603	\N	ResultComment	269	{}
309	I ' 've created a demo project	17	2020-11-18 03:28:40.817404	2020-11-18 14:53:24.723433	\N	ProjectComment	35	{}
310	We should have a meeting to discuss sampling parametrs soon.	17	2020-11-14 04:02:31.563493	2020-11-18 14:53:25.2783	\N	TaskComment	124	{}
311	I agree.	17	2020-11-13 21:40:49.970812	2020-11-18 14:53:25.281377	\N	TaskComment	124	{}
312	The samples have arrived.	17	2020-11-18 04:38:55.407208	2020-11-18 14:53:25.285803	\N	TaskComment	125	{}
313	Due date has been postponed for a day.	17	2020-11-17 00:30:31.452953	2020-11-18 14:53:25.288606	\N	TaskComment	126	{}
314	Please show Steve the RT procedure.	17	2020-11-16 00:15:42.603736	2020-11-18 14:53:25.290612	\N	TaskComment	128	{}
315	The results must be very definitive.	17	2020-11-17 21:59:22.025887	2020-11-18 14:53:25.292587	\N	TaskComment	129	{}
316	The due date here is flexible.	17	2020-11-13 12:19:07.637408	2020-11-18 14:53:25.29455	\N	TaskComment	131	{}
317	Great job!	17	2020-11-18 13:33:53.813003	2020-11-18 14:53:25.484905	\N	StepComment	524	{}
318	Ronald Hermann I have used different sample [#ROW/2~rep_item~16]	17	2020-11-18 05:17:59.994772	2020-11-18 14:53:25.515348	\N	StepComment	524	{}
319	Ronald Hermann Please complete this by Monday.	17	2020-11-18 07:27:44.454738	2020-11-18 14:53:25.522214	\N	StepComment	526	{}
320	Thanks for getting this done.	17	2020-11-17 14:20:43.273997	2020-11-18 14:53:25.564613	\N	StepComment	527	{}
321	This looks well.	17	2020-11-17 14:44:17.217645	2020-11-18 14:53:25.570867	\N	StepComment	528	{}
322	This looks well.	17	2020-11-17 15:40:10.64226	2020-11-18 14:53:25.594847	\N	StepComment	533	{}
323	Ronald Hermann Please check if results match results in [#qPCR~tsk~25]	17	2020-11-18 06:27:26.495376	2020-11-18 14:53:25.64184	\N	ResultComment	275	{}
324	This looks well.	17	2020-11-16 15:47:07.738852	2020-11-18 14:53:25.715563	\N	StepComment	548	{}
325	Great job!	17	2020-11-18 05:58:30.60224	2020-11-18 14:53:25.893364	\N	StepComment	554	{}
326	Great job!	17	2020-11-18 03:32:58.614896	2020-11-18 14:53:26.059301	\N	StepComment	556	{}
327	This looks well.	17	2020-11-16 20:15:00.639303	2020-11-18 14:53:26.069129	\N	StepComment	558	{}
328	Thanks for getting this done.	17	2020-11-18 14:37:06.446912	2020-11-18 14:53:26.078729	\N	StepComment	560	{}
329	This looks well.	17	2020-11-16 06:52:03.866009	2020-11-18 14:53:26.085897	\N	StepComment	561	{}
330	I actually ran it for 15 minutes and not 10.	17	2020-11-13 02:23:48.048279	2020-11-18 14:53:26.156142	\N	StepComment	562	{}
331	Ronald Hermann Could you check if this is okay?	17	2020-11-17 22:18:23.722732	2020-11-18 14:54:13.167762	\N	ResultComment	285	{}
332	Ronald Hermann please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#ROW/5~rep_item~19]</span> seems to be acting strange?	17	2020-11-18 12:59:06.544219	2020-11-18 14:54:15.727422	\N	ResultComment	290	{}
333	[@Laura Webb~A]  I am going to try this for tracking purposes	4	2021-04-08 13:28:21.962685	2021-04-08 13:28:21.962685	\N	TaskComment	133	{}
334	I ' 've created a demo project	20	2021-05-18 20:49:22.170915	2021-05-19 20:01:51.669005	\N	ProjectComment	37	{}
335	We should have a meeting to discuss sampling parametrs soon.	20	2021-05-18 00:53:24.406995	2021-05-19 20:01:52.195035	\N	TaskComment	137	{}
336	I agree.	20	2021-05-17 08:57:29.322109	2021-05-19 20:01:52.198285	\N	TaskComment	137	{}
337	The samples have arrived.	20	2021-05-16 22:03:58.315622	2021-05-19 20:01:52.199697	\N	TaskComment	138	{}
338	Due date has been postponed for a day.	20	2021-05-19 12:06:20.476709	2021-05-19 20:01:52.20178	\N	TaskComment	139	{}
339	Please show Steve the RT procedure.	20	2021-05-17 07:49:49.844154	2021-05-19 20:01:52.202953	\N	TaskComment	141	{}
340	The results must be very definitive.	20	2021-05-16 23:01:15.222875	2021-05-19 20:01:52.20411	\N	TaskComment	142	{}
341	The due date here is flexible.	20	2021-05-19 14:13:33.470263	2021-05-19 20:01:52.205698	\N	TaskComment	144	{}
342	This looks well.	20	2021-05-18 02:25:51.643884	2021-05-19 20:01:52.396647	\N	StepComment	567	{}
343	Susan Thorneloe I have used different sample [#ROD/2~rep_item~1B]	20	2021-05-16 04:44:02.999051	2021-05-19 20:01:52.419595	\N	StepComment	566	{}
344	Susan Thorneloe Please complete this by Monday.	20	2021-05-16 06:32:33.293929	2021-05-19 20:01:52.423181	\N	StepComment	568	{}
345	Great job!	20	2021-05-19 19:44:47.180646	2021-05-19 20:01:52.468202	\N	StepComment	573	{}
346	Thanks for getting this done.	20	2021-05-19 18:05:04.741721	2021-05-19 20:01:52.480666	\N	StepComment	576	{}
347	This looks well.	20	2021-05-19 12:45:57.420208	2021-05-19 20:01:52.490487	\N	StepComment	579	{}
348	Susan Thorneloe Please check if results match results in [#qPCR~tsk~2I]	20	2021-05-19 17:39:00.981787	2021-05-19 20:01:52.504305	\N	ResultComment	299	{}
350	Great job!	20	2021-05-18 11:39:52.067909	2021-05-19 20:01:52.724311	\N	StepComment	597	{}
351	Great job!	20	2021-05-17 17:08:52.409156	2021-05-19 20:01:52.827943	\N	StepComment	600	{}
352	I actually ran it for 15 minutes and not 10.	20	2021-05-19 08:45:27.27355	2021-05-19 20:01:52.884469	\N	StepComment	604	{}
353	Susan Thorneloe Could you check if this is okay?	20	2021-05-19 14:16:10.474373	2021-05-19 20:01:57.072759	\N	ResultComment	309	{}
354	Susan Thorneloe please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#ROD/5~rep_item~1E]</span> seems to be acting strange?	20	2021-05-16 08:44:55.834249	2021-05-19 20:02:02.095337	\N	ResultComment	314	{}
355	I ' 've created a demo project	21	2021-05-17 09:32:04.314779	2021-05-20 14:12:30.443101	\N	ProjectComment	38	{}
356	We should have a meeting to discuss sampling parametrs soon.	21	2021-05-20 08:12:15.917368	2021-05-20 14:12:30.818177	\N	TaskComment	155	{}
357	I agree.	21	2021-05-20 09:07:14.89179	2021-05-20 14:12:30.819831	\N	TaskComment	155	{}
358	The samples have arrived.	21	2021-05-20 07:17:45.448755	2021-05-20 14:12:30.82246	\N	TaskComment	156	{}
359	Due date has been postponed for a day.	21	2021-05-19 22:00:56.975677	2021-05-20 14:12:30.823759	\N	TaskComment	157	{}
360	Please show Steve the RT procedure.	21	2021-05-15 01:08:57.395847	2021-05-20 14:12:30.824968	\N	TaskComment	159	{}
361	The results must be very definitive.	21	2021-05-15 09:36:51.451203	2021-05-20 14:12:30.826236	\N	TaskComment	160	{}
362	The due date here is flexible.	21	2021-05-17 09:02:21.330102	2021-05-20 14:12:30.827446	\N	TaskComment	162	{}
363	Thanks for getting this done.	21	2021-05-19 16:51:15.26101	2021-05-20 14:12:30.936643	\N	StepComment	607	{}
364	Alan Lindquist I have used different sample [#FDL/2~rep_item~1G]	21	2021-05-19 11:31:12.344001	2021-05-20 14:12:30.954594	\N	StepComment	606	{}
365	Alan Lindquist Please complete this by Monday.	21	2021-05-19 09:33:51.546326	2021-05-20 14:12:30.958685	\N	StepComment	608	{}
366	This looks well.	21	2021-05-20 13:58:15.531529	2021-05-20 14:12:30.989566	\N	StepComment	610	{}
367	Thanks for getting this done.	21	2021-05-20 13:57:32.625931	2021-05-20 14:12:30.996791	\N	StepComment	612	{}
368	Great job!	21	2021-05-20 12:01:26.398059	2021-05-20 14:12:31.001536	\N	StepComment	613	{}
369	Thanks for getting this done.	21	2021-05-20 05:12:01.456714	2021-05-20 14:12:31.01594	\N	StepComment	618	{}
370	This looks well.	21	2021-05-19 18:31:58.868626	2021-05-20 14:12:31.025718	\N	StepComment	619	{}
371	This looks well.	21	2021-05-20 14:08:22.879028	2021-05-20 14:12:31.029999	\N	StepComment	620	{}
372	Alan Lindquist Please check if results match results in [#qPCR~tsk~2a]	21	2021-05-20 11:02:14.940412	2021-05-20 14:12:31.039696	\N	ResultComment	320	{}
373	Thanks for getting this done.	21	2021-05-15 07:41:21.503888	2021-05-20 14:12:31.066243	\N	StepComment	622	{}
374	Thanks for getting this done.	21	2021-05-19 02:12:03.901463	2021-05-20 14:12:31.070942	\N	StepComment	623	{}
375	Great job!	21	2021-05-19 17:32:54.641321	2021-05-20 14:12:31.07578	\N	StepComment	624	{}
376	Great job!	21	2021-05-15 12:39:40.62452	2021-05-20 14:12:31.085223	\N	StepComment	627	{}
377	This looks well.	21	2021-05-18 09:10:16.853026	2021-05-20 14:12:31.092161	\N	StepComment	629	{}
378	This looks well.	21	2021-05-17 08:37:29.209488	2021-05-20 14:12:31.173953	\N	StepComment	634	{}
379	Great job!	21	2021-05-16 19:00:59.701919	2021-05-20 14:12:31.194439	\N	StepComment	637	{}
380	Great job!	21	2021-05-17 13:40:47.637009	2021-05-20 14:12:31.301772	\N	StepComment	638	{}
381	I actually ran it for 15 minutes and not 10.	21	2021-05-16 12:56:13.826595	2021-05-20 14:12:31.370473	\N	StepComment	644	{}
382	I ' 've created a demo project	22	2021-05-19 19:17:25.693679	2021-05-20 14:12:57.144257	\N	ProjectComment	39	{}
383	We should have a meeting to discuss sampling parametrs soon.	22	2021-05-16 16:10:12.981237	2021-05-20 14:12:57.472433	\N	TaskComment	164	{}
384	I agree.	22	2021-05-17 00:34:52.787934	2021-05-20 14:12:57.47382	\N	TaskComment	164	{}
385	The samples have arrived.	22	2021-05-17 23:46:35.805078	2021-05-20 14:12:57.475131	\N	TaskComment	165	{}
386	Due date has been postponed for a day.	22	2021-05-20 02:35:02.377988	2021-05-20 14:12:57.47807	\N	TaskComment	166	{}
387	Please show Steve the RT procedure.	22	2021-05-20 11:28:10.926724	2021-05-20 14:12:57.479313	\N	TaskComment	168	{}
388	The results must be very definitive.	22	2021-05-17 19:37:06.603011	2021-05-20 14:12:57.480601	\N	TaskComment	169	{}
389	The due date here is flexible.	22	2021-05-18 08:04:06.799624	2021-05-20 14:12:57.481785	\N	TaskComment	171	{}
390	Max Krause I have used different sample [#ZIO/2~rep_item~1L]	22	2021-05-17 20:27:55.475801	2021-05-20 14:12:57.565013	\N	StepComment	646	{}
391	Max Krause Please complete this by Monday.	22	2021-05-17 22:28:41.432892	2021-05-20 14:12:57.568272	\N	StepComment	648	{}
392	Great job!	22	2021-05-20 11:08:30.022608	2021-05-20 14:12:57.586986	\N	StepComment	650	{}
393	Thanks for getting this done.	22	2021-05-20 05:19:18.730109	2021-05-20 14:12:57.600262	\N	StepComment	654	{}
394	Great job!	22	2021-05-20 01:18:38.717006	2021-05-20 14:12:57.611511	\N	StepComment	658	{}
395	Max Krause Please check if results match results in [#qPCR~tsk~2j]	22	2021-05-19 20:45:05.861399	2021-05-20 14:12:57.620914	\N	ResultComment	329	{}
396	Thanks for getting this done.	22	2021-05-20 04:59:35.370209	2021-05-20 14:12:57.65259	\N	StepComment	664	{}
397	Great job!	22	2021-05-18 05:10:39.328632	2021-05-20 14:12:57.668907	\N	StepComment	670	{}
398	Thanks for getting this done.	22	2021-05-20 14:08:03.390659	2021-05-20 14:12:57.720305	\N	StepComment	672	{}
399	This looks well.	22	2021-05-20 12:37:44.299658	2021-05-20 14:12:57.845518	\N	StepComment	683	{}
400	I actually ran it for 15 minutes and not 10.	22	2021-05-18 11:02:52.392233	2021-05-20 14:12:57.894804	\N	StepComment	684	{}
401	Alan Lindquist Could you check if this is okay?	21	2021-05-19 18:38:34.075705	2021-05-20 14:13:30.95789	\N	ResultComment	339	{}
402	Alan Lindquist please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#FDL/5~rep_item~1J]</span> seems to be acting strange?	21	2021-05-15 03:47:23.465751	2021-05-20 14:13:32.558259	\N	ResultComment	344	{}
403	I ' 've created a demo project	23	2021-05-18 07:50:48.517086	2021-05-20 14:13:33.977516	\N	ProjectComment	40	{}
404	We should have a meeting to discuss sampling parametrs soon.	23	2021-05-17 15:32:03.363607	2021-05-20 14:13:34.274011	\N	TaskComment	173	{}
405	I agree.	23	2021-05-17 20:26:56.520828	2021-05-20 14:13:34.275276	\N	TaskComment	173	{}
406	The samples have arrived.	23	2021-05-16 06:15:00.132776	2021-05-20 14:13:34.276455	\N	TaskComment	174	{}
407	Due date has been postponed for a day.	23	2021-05-15 18:18:38.197005	2021-05-20 14:13:34.277605	\N	TaskComment	175	{}
408	Please show Steve the RT procedure.	23	2021-05-18 19:17:24.807275	2021-05-20 14:13:34.279749	\N	TaskComment	177	{}
409	The results must be very definitive.	23	2021-05-16 10:22:20.844941	2021-05-20 14:13:34.281277	\N	TaskComment	178	{}
410	The due date here is flexible.	23	2021-05-19 10:05:55.105522	2021-05-20 14:13:34.282454	\N	TaskComment	180	{}
411	Great job!	23	2021-05-20 08:07:35.935035	2021-05-20 14:13:34.336893	\N	StepComment	685	{}
412	Eunice Varughese I have used different sample [#QEA/2~rep_item~1Q]	23	2021-05-16 07:55:08.949088	2021-05-20 14:13:34.367412	\N	StepComment	686	{}
413	Eunice Varughese Please complete this by Monday.	23	2021-05-16 10:00:46.891552	2021-05-20 14:13:34.370131	\N	StepComment	688	{}
414	This looks well.	23	2021-05-16 23:39:34.558545	2021-05-20 14:13:34.42197	\N	StepComment	693	{}
415	This looks well.	23	2021-05-18 00:39:31.054082	2021-05-20 14:13:34.428676	\N	StepComment	695	{}
416	This looks well.	23	2021-05-19 10:52:26.270358	2021-05-20 14:13:34.437083	\N	StepComment	698	{}
417	Eunice Varughese Please check if results match results in [#qPCR~tsk~2s]	23	2021-05-16 06:52:51.011137	2021-05-20 14:13:34.446543	\N	ResultComment	352	{}
418	This looks well.	23	2021-05-19 07:18:33.393415	2021-05-20 14:13:34.46802	\N	StepComment	703	{}
419	Great job!	23	2021-05-20 10:53:16.947836	2021-05-20 14:13:34.484628	\N	StepComment	709	{}
420	Great job!	23	2021-05-19 20:25:59.689	2021-05-20 14:13:34.491171	\N	StepComment	711	{}
421	Thanks for getting this done.	23	2021-05-18 03:17:44.520292	2021-05-20 14:13:34.57214	\N	StepComment	716	{}
422	This looks well.	23	2021-05-20 08:23:22.478984	2021-05-20 14:13:34.687246	\N	StepComment	719	{}
424	This looks well.	23	2021-05-19 08:05:52.785674	2021-05-20 14:13:34.753674	\N	StepComment	724	{}
425	I actually ran it for 15 minutes and not 10.	23	2021-05-19 05:34:59.047929	2021-05-20 14:13:34.767002	\N	StepComment	724	{}
423	Max Krause Could you check if this is okay?	22	2021-05-20 09:26:22.379441	2021-05-20 14:13:34.697612	\N	ResultComment	354	{}
426	Max Krause please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#ZIO/5~rep_item~1O]</span> seems to be acting strange?	22	2021-05-17 04:25:11.275555	2021-05-20 14:13:36.244893	\N	ResultComment	365	{}
427	Eunice Varughese Could you check if this is okay?	23	2021-05-17 10:51:24.963388	2021-05-20 14:13:44.294324	\N	ResultComment	372	{}
428	Eunice Varughese please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#QEA/5~rep_item~1T]</span> seems to be acting strange?	23	2021-05-17 18:33:20.007376	2021-05-20 14:13:46.218147	\N	ResultComment	377	{}
429	I ' 've created a demo project	24	2021-05-18 15:47:10.424712	2021-05-20 14:21:20.534384	\N	ProjectComment	41	{}
430	We should have a meeting to discuss sampling parametrs soon.	24	2021-05-20 12:10:30.766024	2021-05-20 14:21:20.838109	\N	TaskComment	182	{}
431	I agree.	24	2021-05-20 06:04:34.967957	2021-05-20 14:21:20.839413	\N	TaskComment	182	{}
432	The samples have arrived.	24	2021-05-16 19:55:25.75222	2021-05-20 14:21:20.840786	\N	TaskComment	183	{}
433	Due date has been postponed for a day.	24	2021-05-14 16:31:01.441033	2021-05-20 14:21:20.842177	\N	TaskComment	184	{}
434	Please show Steve the RT procedure.	24	2021-05-20 09:52:31.670296	2021-05-20 14:21:20.843766	\N	TaskComment	186	{}
435	The results must be very definitive.	24	2021-05-15 22:00:56.67812	2021-05-20 14:21:20.845221	\N	TaskComment	187	{}
436	The due date here is flexible.	24	2021-05-19 20:53:55.671652	2021-05-20 14:21:20.846507	\N	TaskComment	189	{}
437	Great job!	24	2021-05-19 22:23:25.152806	2021-05-20 14:21:20.916923	\N	StepComment	726	{}
438	Maily Pham I have used different sample [#VRK/2~rep_item~1V]	24	2021-05-16 20:58:39.447627	2021-05-20 14:21:20.937632	\N	StepComment	726	{}
439	Maily Pham Please complete this by Monday.	24	2021-05-16 23:40:09.526246	2021-05-20 14:21:20.942098	\N	StepComment	728	{}
440	Great job!	24	2021-05-16 06:00:33.078551	2021-05-20 14:21:20.96891	\N	StepComment	732	{}
441	Great job!	24	2021-05-15 05:36:11.126888	2021-05-20 14:21:20.992469	\N	StepComment	740	{}
442	Maily Pham Please check if results match results in [#qPCR~tsk~31]	24	2021-05-15 09:56:33.368351	2021-05-20 14:21:20.997393	\N	ResultComment	383	{}
443	Thanks for getting this done.	24	2021-05-19 08:23:31.251627	2021-05-20 14:21:21.025995	\N	StepComment	744	{}
444	Thanks for getting this done.	24	2021-05-16 19:52:04.219781	2021-05-20 14:21:21.045637	\N	StepComment	750	{}
445	This looks well.	24	2021-05-19 07:13:47.477503	2021-05-20 14:21:21.239669	\N	StepComment	759	{}
446	I actually ran it for 15 minutes and not 10.	24	2021-05-19 02:42:36.798883	2021-05-20 14:21:21.391281	\N	StepComment	764	{}
447	Maily Pham Could you check if this is okay?	24	2021-05-16 08:18:32.027862	2021-05-20 14:22:03.361852	\N	ResultComment	393	{}
448	Maily Pham please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#VRK/5~rep_item~1Y]</span> seems to be acting strange?	24	2021-05-15 22:41:55.728172	2021-05-20 14:22:04.992432	\N	ResultComment	398	{}
449	I ' 've created a demo project	25	2021-05-20 11:51:40.617698	2021-05-20 14:30:30.302472	\N	ProjectComment	43	{}
450	We should have a meeting to discuss sampling parametrs soon.	25	2021-05-19 19:37:23.954542	2021-05-20 14:30:30.580918	\N	TaskComment	191	{}
451	I agree.	25	2021-05-20 00:16:13.957047	2021-05-20 14:30:30.582247	\N	TaskComment	191	{}
452	The samples have arrived.	25	2021-05-19 21:42:55.605833	2021-05-20 14:30:30.583438	\N	TaskComment	192	{}
453	Due date has been postponed for a day.	25	2021-05-17 06:18:05.264846	2021-05-20 14:30:30.584661	\N	TaskComment	193	{}
454	Please show Steve the RT procedure.	25	2021-05-18 15:28:03.163442	2021-05-20 14:30:30.585841	\N	TaskComment	195	{}
455	The results must be very definitive.	25	2021-05-18 04:17:32.886898	2021-05-20 14:30:30.587021	\N	TaskComment	196	{}
456	The due date here is flexible.	25	2021-05-19 05:00:38.134864	2021-05-20 14:30:30.588233	\N	TaskComment	198	{}
457	Thanks for getting this done.	25	2021-05-19 17:57:38.084806	2021-05-20 14:30:30.639489	\N	StepComment	765	{}
458	Great job!	25	2021-05-20 03:47:42.673719	2021-05-20 14:30:30.656555	\N	StepComment	767	{}
459	Anna Wade I have used different sample [#YKK/2~rep_item~1a]	25	2021-05-19 12:04:36.361799	2021-05-20 14:30:30.673833	\N	StepComment	766	{}
460	Anna Wade Please complete this by Monday.	25	2021-05-19 13:28:36.750018	2021-05-20 14:30:30.677951	\N	StepComment	768	{}
461	Great job!	25	2021-05-17 11:49:45.927901	2021-05-20 14:30:30.700301	\N	StepComment	771	{}
462	Great job!	25	2021-05-17 22:27:00.567151	2021-05-20 14:30:30.718537	\N	StepComment	775	{}
463	Great job!	25	2021-05-19 05:01:51.292763	2021-05-20 14:30:30.724133	\N	StepComment	776	{}
464	This looks well.	25	2021-05-20 04:32:21.826189	2021-05-20 14:30:30.737752	\N	StepComment	779	{}
465	Anna Wade Please check if results match results in [#qPCR~tsk~3A]	25	2021-05-17 14:00:00.515667	2021-05-20 14:30:30.748225	\N	ResultComment	404	{}
466	This looks well.	25	2021-05-20 01:28:26.430304	2021-05-20 14:30:30.773401	\N	StepComment	781	{}
467	This looks well.	25	2021-05-17 08:03:50.782994	2021-05-20 14:30:30.791978	\N	StepComment	785	{}
468	This looks well.	25	2021-05-15 14:04:02.388386	2021-05-20 14:30:30.799666	\N	StepComment	787	{}
469	Great job!	25	2021-05-17 14:39:34.646169	2021-05-20 14:30:30.804055	\N	StepComment	788	{}
470	Great job!	25	2021-05-19 15:51:47.790509	2021-05-20 14:30:30.872274	\N	StepComment	793	{}
471	Thanks for getting this done.	25	2021-05-19 07:00:45.728507	2021-05-20 14:30:30.878039	\N	StepComment	794	{}
472	Great job!	25	2021-05-19 15:11:39.628981	2021-05-20 14:30:30.899262	\N	StepComment	795	{}
473	I actually ran it for 15 minutes and not 10.	25	2021-05-18 09:50:06.10962	2021-05-20 14:30:31.089462	\N	StepComment	804	{}
474	Anna Wade Could you check if this is okay?	25	2021-05-17 07:21:08.021346	2021-05-20 14:31:16.229216	\N	ResultComment	414	{}
475	Anna Wade please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#YKK/5~rep_item~1d]</span> seems to be acting strange?	25	2021-05-17 23:02:31.552501	2021-05-20 14:31:17.840553	\N	ResultComment	419	{}
476	I ' 've created a demo project	26	2021-05-20 02:34:41.441796	2021-05-20 15:17:11.691837	\N	ProjectComment	44	{}
477	We should have a meeting to discuss sampling parametrs soon.	26	2021-05-16 13:41:04.348816	2021-05-20 15:17:12.000657	\N	TaskComment	200	{}
478	I agree.	26	2021-05-16 20:40:39.682297	2021-05-20 15:17:12.002068	\N	TaskComment	200	{}
479	The samples have arrived.	26	2021-05-15 10:33:34.816145	2021-05-20 15:17:12.003295	\N	TaskComment	201	{}
480	Due date has been postponed for a day.	26	2021-05-16 06:53:51.072227	2021-05-20 15:17:12.004558	\N	TaskComment	202	{}
481	Please show Steve the RT procedure.	26	2021-05-18 18:03:51.699151	2021-05-20 15:17:12.005821	\N	TaskComment	204	{}
482	The results must be very definitive.	26	2021-05-18 07:03:40.002179	2021-05-20 15:17:12.007813	\N	TaskComment	205	{}
483	The due date here is flexible.	26	2021-05-15 15:59:54.730936	2021-05-20 15:17:12.009246	\N	TaskComment	207	{}
484	This looks well.	26	2021-05-17 17:58:56.712168	2021-05-20 15:17:12.080227	\N	StepComment	808	{}
485	Jody Shoemaker I have used different sample [#MQU/2~rep_item~1f]	26	2021-05-15 08:27:51.041287	2021-05-20 15:17:12.098502	\N	StepComment	807	{}
486	Jody Shoemaker Please complete this by Monday.	26	2021-05-15 07:48:54.88365	2021-05-20 15:17:12.102978	\N	StepComment	809	{}
487	Great job!	26	2021-05-17 21:46:01.870687	2021-05-20 15:17:12.122481	\N	StepComment	811	{}
488	Thanks for getting this done.	26	2021-05-16 07:24:48.945286	2021-05-20 15:17:12.136868	\N	StepComment	816	{}
489	Jody Shoemaker Please check if results match results in [#qPCR~tsk~3J]	26	2021-05-16 08:28:55.471316	2021-05-20 15:17:12.156385	\N	ResultComment	425	{}
490	Great job!	26	2021-05-18 18:56:39.789484	2021-05-20 15:17:12.190043	\N	StepComment	828	{}
491	This looks well.	26	2021-05-18 08:12:02.778941	2021-05-20 15:17:12.19771	\N	StepComment	830	{}
492	This looks well.	26	2021-05-20 07:13:28.251071	2021-05-20 15:17:12.262952	\N	StepComment	834	{}
493	This looks well.	26	2021-05-20 10:40:47.073022	2021-05-20 15:17:12.26819	\N	StepComment	835	{}
494	This looks well.	26	2021-05-19 01:39:30.400035	2021-05-20 15:17:12.286104	\N	StepComment	837	{}
495	Great job!	26	2021-05-18 18:20:31.874017	2021-05-20 15:17:12.2907	\N	StepComment	838	{}
496	I actually ran it for 15 minutes and not 10.	26	2021-05-15 10:43:18.981233	2021-05-20 15:17:12.452421	\N	StepComment	845	{}
497	Jody Shoemaker Could you check if this is okay?	26	2021-05-15 19:04:10.608047	2021-05-20 15:17:39.556426	\N	ResultComment	435	{}
498	Jody Shoemaker please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#MQU/5~rep_item~1i]</span> seems to be acting strange?	26	2021-05-18 07:19:51.072586	2021-05-20 15:17:41.228616	\N	ResultComment	440	{}
499	I ' 've created a demo project	27	2021-05-27 20:43:21.254573	2021-05-28 15:07:44.70795	\N	ProjectComment	46	{}
500	We should have a meeting to discuss sampling parametrs soon.	27	2021-05-26 11:34:24.907972	2021-05-28 15:07:52.598432	\N	TaskComment	212	{}
501	I agree.	27	2021-05-26 14:27:33.248416	2021-05-28 15:07:52.619064	\N	TaskComment	212	{}
502	The samples have arrived.	27	2021-05-22 19:28:01.491441	2021-05-28 15:07:52.626312	\N	TaskComment	213	{}
503	Due date has been postponed for a day.	27	2021-05-23 07:32:52.103907	2021-05-28 15:07:52.774808	\N	TaskComment	214	{}
504	Please show Steve the RT procedure.	27	2021-05-27 06:46:05.440196	2021-05-28 15:07:52.77793	\N	TaskComment	216	{}
505	The results must be very definitive.	27	2021-05-25 14:41:56.689516	2021-05-28 15:07:52.780545	\N	TaskComment	217	{}
506	The due date here is flexible.	27	2021-05-25 12:17:46.436558	2021-05-28 15:07:52.782693	\N	TaskComment	219	{}
507	This looks well.	27	2021-05-26 23:39:17.182761	2021-05-28 15:07:53.084692	\N	StepComment	853	{}
508	Tae Lee I have used different sample [#OJT/2~rep_item~1k]	27	2021-05-22 21:28:53.677342	2021-05-28 15:07:53.113045	\N	StepComment	852	{}
509	Tae Lee Please complete this by Monday.	27	2021-05-22 18:44:28.142102	2021-05-28 15:07:53.119774	\N	StepComment	854	{}
510	Great job!	27	2021-05-24 14:14:15.632771	2021-05-28 15:07:54.103735	\N	StepComment	862	{}
511	Tae Lee Please check if results match results in [#qPCR~tsk~3V]	27	2021-05-23 13:27:49.918274	2021-05-28 15:07:54.548883	\N	ResultComment	446	{}
512	This looks well.	27	2021-05-28 07:18:39.226245	2021-05-28 15:07:55.171624	\N	StepComment	867	{}
513	Thanks for getting this done.	27	2021-05-28 12:07:22.638552	2021-05-28 15:07:55.209126	\N	StepComment	874	{}
514	This looks well.	27	2021-05-28 13:18:54.869478	2021-05-28 15:07:55.215047	\N	StepComment	875	{}
515	Great job!	27	2021-05-28 13:35:20.976085	2021-05-28 15:07:55.223185	\N	StepComment	876	{}
516	This looks well.	27	2021-05-24 20:55:31.591808	2021-05-28 15:07:58.042745	\N	StepComment	887	{}
517	I actually ran it for 15 minutes and not 10.	27	2021-05-24 13:53:07.89784	2021-05-28 15:07:58.119921	\N	StepComment	890	{}
518	Tae Lee Could you check if this is okay?	27	2021-05-25 18:38:16.068819	2021-05-28 15:08:25.426703	\N	ResultComment	456	{}
519	Tae Lee please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#OJT/5~rep_item~1n]</span> seems to be acting strange?	27	2021-05-26 06:54:51.219803	2021-05-28 15:08:35.008051	\N	ResultComment	461	{}
520	I ' 've created a demo project	28	2021-05-29 19:01:34.003144	2021-06-02 16:10:01.67851	\N	ProjectComment	54	{}
521	We should have a meeting to discuss sampling parametrs soon.	28	2021-06-02 09:06:55.584195	2021-06-02 16:10:03.231771	\N	TaskComment	221	{}
522	I agree.	28	2021-06-02 08:59:08.136412	2021-06-02 16:10:03.237302	\N	TaskComment	221	{}
523	The samples have arrived.	28	2021-05-30 19:33:15.012218	2021-06-02 16:10:03.242463	\N	TaskComment	222	{}
524	Due date has been postponed for a day.	28	2021-06-01 12:22:22.915828	2021-06-02 16:10:03.245737	\N	TaskComment	223	{}
525	Please show Steve the RT procedure.	28	2021-06-01 00:08:57.794546	2021-06-02 16:10:03.249327	\N	TaskComment	225	{}
526	The results must be very definitive.	28	2021-05-28 16:31:56.012953	2021-06-02 16:10:03.252415	\N	TaskComment	226	{}
527	The due date here is flexible.	28	2021-05-30 05:42:22.281915	2021-06-02 16:10:03.256509	\N	TaskComment	228	{}
528	This looks well.	28	2021-05-31 16:19:03.820451	2021-06-02 16:10:04.083738	\N	StepComment	896	{}
529	Great job!	28	2021-06-02 03:49:52.737726	2021-06-02 16:10:04.104829	\N	StepComment	897	{}
530	Thanks for getting this done.	28	2021-06-02 16:05:33.646894	2021-06-02 16:10:04.178253	\N	StepComment	898	{}
531	Dennis Fine I have used different sample [#AUG/2~rep_item~1p]	28	2021-05-30 15:40:24.894578	2021-06-02 16:10:04.307565	\N	StepComment	896	{}
532	Dennis Fine Please complete this by Monday.	28	2021-05-30 12:30:01.864358	2021-06-02 16:10:04.321518	\N	StepComment	898	{}
533	Great job!	28	2021-06-01 11:42:23.712822	2021-06-02 16:10:04.51667	\N	StepComment	901	{}
534	Thanks for getting this done.	28	2021-06-01 14:10:54.582913	2021-06-02 16:10:04.550093	\N	StepComment	903	{}
535	This looks well.	28	2021-06-01 19:06:21.017468	2021-06-02 16:10:04.573754	\N	StepComment	904	{}
536	Thanks for getting this done.	28	2021-06-02 07:24:04.173888	2021-06-02 16:10:04.587672	\N	StepComment	905	{}
537	Great job!	28	2021-06-02 11:52:20.015061	2021-06-02 16:10:04.622611	\N	StepComment	908	{}
538	Great job!	28	2021-06-01 11:13:55.675183	2021-06-02 16:10:04.64452	\N	StepComment	909	{}
539	Dennis Fine Please check if results match results in [#qPCR~tsk~3e]	28	2021-06-02 04:01:35.141894	2021-06-02 16:10:04.668196	\N	ResultComment	467	{}
540	Great job!	28	2021-06-02 02:52:43.264428	2021-06-02 16:10:04.820125	\N	StepComment	914	{}
541	This looks well.	28	2021-06-01 20:10:45.940813	2021-06-02 16:10:04.888336	\N	StepComment	918	{}
542	Great job!	28	2021-06-02 04:28:57.059882	2021-06-02 16:10:04.948727	\N	StepComment	921	{}
543	This looks well.	28	2021-05-30 14:51:18.453196	2021-06-02 16:10:06.634022	\N	StepComment	927	{}
544	This looks well.	28	2021-06-02 15:51:28.103088	2021-06-02 16:10:07.233131	\N	StepComment	933	{}
545	Thanks for getting this done.	28	2021-06-02 07:16:57.574568	2021-06-02 16:10:07.440967	\N	StepComment	934	{}
546	I actually ran it for 15 minutes and not 10.	28	2021-05-30 02:29:06.106513	2021-06-02 16:10:07.507184	\N	StepComment	934	{}
547	Dennis Fine Could you check if this is okay?	28	2021-06-01 14:05:58.884054	2021-06-02 16:10:39.261014	\N	ResultComment	477	{}
548	Dennis Fine please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#AUG/5~rep_item~1s]</span> seems to be acting strange?	28	2021-05-30 04:15:27.284241	2021-06-02 16:10:42.142907	\N	ResultComment	482	{}
549	Samples were received and placed in AWBERC 486	27	2021-06-10 16:08:13.665299	2021-06-10 16:08:13.665299	\N	TaskComment	230	{}
550	completed.	27	2021-06-14 19:22:09.654363	2021-06-14 19:22:09.654363	\N	TaskComment	231	{}
551	In progress.	27	2021-06-14 19:23:15.844727	2021-06-14 19:23:15.844727	\N	TaskComment	232	{}
552	completed	27	2021-06-16 15:59:23.761389	2021-06-16 15:59:23.761389	\N	TaskComment	234	{}
553	1. mix 100:0% wt \n2. mix 80:20%  wt\n3. mix 50:50% wt	27	2021-07-06 15:50:55.498107	2021-07-06 15:50:55.498107	\N	TaskComment	236	{}
555	Condition 1: B (100%)\nCondition 2: B (5%) in 2mm \nCondition 3: B (10%) in 2mm \nCondition 4: B (20%) in 2mm	27	2021-07-12 18:44:49.533312	2021-07-12 18:44:58.995654	\N	TaskComment	240	{}
554	Plan: \nCondition 1: A (0%) in 2mm (100%)\nCondition 2: A (5%) in 2mm  (not done correctly)\nCondition 3: A (10%) in 2mm \nCondition 4: A (100%) in 2mm (0%)	27	2021-07-12 18:44:01.732413	2021-07-15 14:29:46.363808	\N	TaskComment	239	{}
556	Actual: \nCondition 1: A (0%) in 2mm (100%)\n - 0g A, 1.0146g "2mm"\nCondition 2: A (#%) in 2mm \n - 0.5262 g A, 0.9518 g "2mm"\nCondition 3: A (10%) in 2mm \n - 0.1187 g A, 0.8950 g "2mm"\nCondition 4: A (100%) in 2mm (0%)\n - 1.0018 g A, 0 g "2mm"	27	2021-07-13 16:27:14.850124	2021-07-13 16:27:14.850124	\N	TaskComment	239	{}
557	I ' 've created a demo project	30	2021-08-17 17:20:39.371062	2021-08-23 14:58:27.566091	\N	ProjectComment	55	{}
558	We should have a meeting to discuss sampling parametrs soon.	30	2021-08-21 09:50:12.456651	2021-08-23 14:58:28.185028	\N	TaskComment	241	{}
559	I agree.	30	2021-08-21 17:32:33.734664	2021-08-23 14:58:28.186448	\N	TaskComment	241	{}
560	The samples have arrived.	30	2021-08-19 23:17:48.011026	2021-08-23 14:58:28.188225	\N	TaskComment	242	{}
561	Due date has been postponed for a day.	30	2021-08-22 00:54:36.122247	2021-08-23 14:58:28.189496	\N	TaskComment	243	{}
562	Please show Steve the RT procedure.	30	2021-08-21 05:53:46.9402	2021-08-23 14:58:28.190648	\N	TaskComment	245	{}
563	The results must be very definitive.	30	2021-08-20 14:59:26.113624	2021-08-23 14:58:28.191839	\N	TaskComment	246	{}
564	The due date here is flexible.	30	2021-08-21 15:47:54.168339	2021-08-23 14:58:28.192993	\N	TaskComment	248	{}
565	Aaron Tam I have used different sample [#CKC/2~rep_item~1u]	30	2021-08-19 18:05:47.873152	2021-08-23 14:58:28.420334	\N	StepComment	936	{}
566	Aaron Tam Please complete this by Monday.	30	2021-08-19 14:41:15.3886	2021-08-23 14:58:28.425291	\N	StepComment	938	{}
567	Great job!	30	2021-08-21 20:20:24.628835	2021-08-23 14:58:28.478832	\N	StepComment	942	{}
568	Thanks for getting this done.	30	2021-08-23 08:13:50.406332	2021-08-23 14:58:28.487102	\N	StepComment	944	{}
569	Great job!	30	2021-08-22 23:21:22.292597	2021-08-23 14:58:28.491948	\N	StepComment	945	{}
570	This looks well.	30	2021-08-22 14:01:29.631122	2021-08-23 14:58:28.497262	\N	StepComment	946	{}
571	This looks well.	30	2021-08-22 15:11:18.037521	2021-08-23 14:58:28.502194	\N	StepComment	947	{}
572	Aaron Tam Please check if results match results in [#qPCR~tsk~3y]	30	2021-08-22 07:09:43.367081	2021-08-23 14:58:28.521775	\N	ResultComment	489	{}
573	Great job!	30	2021-08-21 21:29:35.965336	2021-08-23 14:58:28.549759	\N	StepComment	952	{}
574	Great job!	30	2021-08-21 14:07:39.364959	2021-08-23 14:58:28.560762	\N	StepComment	955	{}
575	Great job!	30	2021-08-20 20:55:57.474529	2021-08-23 14:58:28.56857	\N	StepComment	957	{}
576	Great job!	30	2021-08-23 06:52:42.668461	2021-08-23 14:58:28.835393	\N	StepComment	971	{}
577	I actually ran it for 15 minutes and not 10.	30	2021-08-21 14:22:47.631841	2021-08-23 14:58:28.890561	\N	StepComment	974	{}
578	Aaron Tam Could you check if this is okay?	30	2021-08-21 14:19:42.902609	2021-08-23 14:58:56.648265	\N	ResultComment	499	{}
579	Aaron Tam please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#CKC/5~rep_item~1x]</span> seems to be acting strange?	30	2021-08-20 13:28:49.594997	2021-08-23 14:58:59.303308	\N	ResultComment	504	{}
580	I ' 've created a demo project	31	2021-08-17 15:21:29.434231	2021-08-23 17:19:39.187259	\N	ProjectComment	56	{}
581	We should have a meeting to discuss sampling parametrs soon.	31	2021-08-20 07:25:51.508707	2021-08-23 17:19:39.501376	\N	TaskComment	250	{}
582	I agree.	31	2021-08-20 04:18:21.411453	2021-08-23 17:19:39.503165	\N	TaskComment	250	{}
583	The samples have arrived.	31	2021-08-20 00:19:09.82641	2021-08-23 17:19:39.504608	\N	TaskComment	251	{}
584	Due date has been postponed for a day.	31	2021-08-22 13:20:34.647869	2021-08-23 17:19:39.506256	\N	TaskComment	252	{}
585	Please show Steve the RT procedure.	31	2021-08-19 19:58:01.904374	2021-08-23 17:19:39.507881	\N	TaskComment	254	{}
586	The results must be very definitive.	31	2021-08-22 09:06:46.255142	2021-08-23 17:19:39.509329	\N	TaskComment	255	{}
587	The due date here is flexible.	31	2021-08-20 15:34:12.168784	2021-08-23 17:19:39.510668	\N	TaskComment	257	{}
588	JAMES VOIT I have used different sample [#NXT/2~rep_item~1z]	31	2021-08-19 08:19:27.637389	2021-08-23 17:19:39.619697	\N	StepComment	976	{}
589	JAMES VOIT Please complete this by Monday.	31	2021-08-19 04:30:54.696427	2021-08-23 17:19:39.624218	\N	StepComment	978	{}
590	Great job!	31	2021-08-23 09:25:15.778269	2021-08-23 17:19:39.660748	\N	StepComment	983	{}
591	Thanks for getting this done.	31	2021-08-23 10:38:09.732038	2021-08-23 17:19:39.665888	\N	StepComment	984	{}
592	JAMES VOIT Please check if results match results in [#qPCR~tsk~47]	31	2021-08-22 10:45:43.447645	2021-08-23 17:19:39.687683	\N	ResultComment	510	{}
593	Great job!	31	2021-08-21 01:48:21.476123	2021-08-23 17:19:39.717383	\N	StepComment	994	{}
594	This looks well.	31	2021-08-23 16:11:32.408	2021-08-23 17:19:39.722382	\N	StepComment	995	{}
595	This looks well.	31	2021-08-21 23:31:29.690546	2021-08-23 17:19:39.727059	\N	StepComment	996	{}
596	This looks well.	31	2021-08-22 13:21:09.724744	2021-08-23 17:19:39.735079	\N	StepComment	998	{}
597	Thanks for getting this done.	31	2021-08-22 22:13:51.126979	2021-08-23 17:19:39.745614	\N	StepComment	1001	{}
598	Great job!	31	2021-08-23 01:47:01.199604	2021-08-23 17:19:39.80503	\N	StepComment	1003	{}
599	Thanks for getting this done.	31	2021-08-22 16:46:59.041001	2021-08-23 17:19:39.827672	\N	StepComment	1005	{}
600	Thanks for getting this done.	31	2021-08-22 14:27:47.790955	2021-08-23 17:19:39.963019	\N	StepComment	1010	{}
601	Thanks for getting this done.	31	2021-08-20 22:34:37.696695	2021-08-23 17:19:40.022154	\N	StepComment	1014	{}
602	I actually ran it for 15 minutes and not 10.	31	2021-08-20 11:32:13.845601	2021-08-23 17:19:40.042658	\N	StepComment	1014	{}
603	JAMES VOIT Could you check if this is okay?	31	2021-08-22 14:50:57.844597	2021-08-23 17:20:16.942651	\N	ResultComment	520	{}
604	JAMES VOIT please check the results again. <span class=\\"atwho-inserted\\" contenteditable=\\"false\\"data-atwho-at-query=\\"#\\">[#NXT/5~rep_item~22]</span> seems to be acting strange?	31	2021-08-22 20:49:18.995181	2021-08-23 17:20:18.492859	\N	ResultComment	525	{}
\.


--
-- Data for Name: connections; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.connections (id, input_id, output_id) FROM stdin;
1	2	1
2	3	2
3	4	3
4	5	4
5	6	5
6	7	6
7	8	7
8	11	10
9	12	11
10	13	12
11	14	13
12	15	14
13	16	15
14	17	16
15	20	19
16	21	20
17	22	21
18	23	22
19	24	23
20	25	24
21	26	25
22	29	28
23	30	29
24	31	30
25	32	31
26	33	32
27	34	33
28	35	34
29	38	37
30	39	38
31	40	39
32	41	40
33	42	41
34	43	42
35	44	43
36	47	46
37	48	47
38	49	48
39	50	49
40	51	50
41	52	51
42	53	52
43	56	55
44	57	56
45	58	57
46	59	58
47	60	59
48	61	60
49	62	61
50	65	64
51	66	65
52	67	66
53	68	67
54	69	68
55	70	69
56	71	70
57	75	74
58	76	75
59	77	76
60	79	78
61	80	79
62	81	80
63	82	81
64	83	82
65	84	83
66	85	84
67	88	87
68	89	88
69	90	89
70	91	90
71	92	91
72	93	92
73	94	93
74	97	96
75	98	97
76	99	98
77	100	99
78	101	100
79	102	101
80	103	102
81	107	106
82	108	107
83	109	108
84	110	109
85	111	110
86	112	111
87	113	112
88	116	115
89	117	116
90	118	117
91	119	118
92	120	119
93	121	120
94	122	121
95	125	124
96	126	125
97	127	126
98	128	127
99	129	128
100	130	129
101	131	130
102	138	137
103	139	138
104	140	139
105	141	140
106	142	141
107	143	142
108	144	143
109	147	146
114	149	148
115	150	149
116	151	150
117	152	151
118	154	153
119	156	155
120	157	156
121	158	157
122	159	158
123	160	159
124	161	160
125	162	161
126	165	164
127	166	165
128	167	166
129	168	167
130	169	168
131	170	169
132	171	170
133	174	173
134	175	174
135	176	175
136	177	176
137	178	177
138	179	178
139	180	179
140	183	182
141	184	183
142	185	184
143	186	185
144	187	186
145	188	187
146	189	188
147	192	191
148	193	192
149	194	193
150	195	194
151	196	195
152	197	196
153	198	197
154	201	200
155	202	201
156	203	202
157	204	203
158	205	204
159	206	205
160	207	206
161	213	212
162	214	213
163	215	214
164	216	215
165	217	216
166	218	217
167	219	218
168	222	221
169	223	222
170	224	223
171	225	224
172	226	225
173	227	226
174	228	227
175	242	241
176	243	242
177	244	243
178	245	244
179	246	245
180	247	246
181	248	247
182	251	250
183	252	251
184	253	252
185	254	253
186	255	254
187	256	255
188	257	256
189	262	259
190	263	259
191	271	263
192	259	260
193	259	261
194	272	261
195	270	272
196	265	271
197	262	264
198	273	264
199	270	273
200	270	265
201	266	270
202	267	266
203	268	267
204	269	268
205	276	275
\.



-- --
-- -- Data for Name: label_templates; Type: TABLE DATA; Schema: public; Owner: postgres
-- --

-- COPY public.label_templates (id, name, content, language_type, "default", size, created_at, updated_at) FROM stdin;
-- 1	SciNote Item	^XA\n^LH20,20\n^CF0,23\n^FO0,0^FD{{item_id}}^FS\n^FO0,7^BQN,2,4^FDMA{{item_id}}^FS\n^FO95,30^FB180,4,0,L^FD{{item_name}}^FS^FS\n^XZ\n	0	t	1" x 0.5" / 25.4mm x 12.7mm	2022-02-16 16:38:43.469087	2022-02-16 16:38:43.469087
-- \.


--
-- Data for Name: my_module_repository_rows; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.my_module_repository_rows (id, repository_row_id, my_module_id, assigned_by_id, created_at, updated_at) FROM stdin;
1	1	2	1	2019-05-30 12:40:53.417123	2019-05-30 12:40:53.417123
2	2	2	1	2019-05-30 12:40:53.420268	2019-05-30 12:40:53.420268
3	3	2	1	2019-05-30 12:40:53.423081	2019-05-30 12:40:53.423081
4	4	2	1	2019-05-30 12:40:53.425791	2019-05-30 12:40:53.425791
5	5	2	1	2019-05-30 12:40:53.428445	2019-05-30 12:40:53.428445
6	1	3	1	2019-05-30 12:40:53.444651	2019-05-30 12:40:53.444651
7	2	3	1	2019-05-30 12:40:53.451677	2019-05-30 12:40:53.451677
8	3	3	1	2019-05-30 12:40:53.454119	2019-05-30 12:40:53.454119
9	4	3	1	2019-05-30 12:40:53.456531	2019-05-30 12:40:53.456531
10	5	3	1	2019-05-30 12:40:53.459066	2019-05-30 12:40:53.459066
11	1	4	1	2019-05-30 12:40:53.474908	2019-05-30 12:40:53.474908
12	2	4	1	2019-05-30 12:40:53.480471	2019-05-30 12:40:53.480471
13	3	4	1	2019-05-30 12:40:53.483004	2019-05-30 12:40:53.483004
14	4	4	1	2019-05-30 12:40:53.485646	2019-05-30 12:40:53.485646
15	5	4	1	2019-05-30 12:40:53.48825	2019-05-30 12:40:53.48825
16	1	5	1	2019-05-30 12:40:53.511386	2019-05-30 12:40:53.511386
17	2	5	1	2019-05-30 12:40:53.517117	2019-05-30 12:40:53.517117
18	3	5	1	2019-05-30 12:40:53.519866	2019-05-30 12:40:53.519866
19	4	5	1	2019-05-30 12:40:53.522558	2019-05-30 12:40:53.522558
20	5	5	1	2019-05-30 12:40:53.525388	2019-05-30 12:40:53.525388
21	1	6	1	2019-05-30 12:40:53.541719	2019-05-30 12:40:53.541719
22	2	6	1	2019-05-30 12:40:53.547532	2019-05-30 12:40:53.547532
23	3	6	1	2019-05-30 12:40:53.550167	2019-05-30 12:40:53.550167
24	4	6	1	2019-05-30 12:40:53.552725	2019-05-30 12:40:53.552725
25	5	6	1	2019-05-30 12:40:53.555258	2019-05-30 12:40:53.555258
26	1	7	1	2019-05-30 12:40:53.571713	2019-05-30 12:40:53.571713
27	2	7	1	2019-05-30 12:40:53.57729	2019-05-30 12:40:53.57729
28	3	7	1	2019-05-30 12:40:53.57988	2019-05-30 12:40:53.57988
29	4	7	1	2019-05-30 12:40:53.582464	2019-05-30 12:40:53.582464
30	5	7	1	2019-05-30 12:40:53.584969	2019-05-30 12:40:53.584969
31	1	8	1	2019-05-30 12:40:53.600968	2019-05-30 12:40:53.600968
32	2	8	1	2019-05-30 12:40:53.606516	2019-05-30 12:40:53.606516
33	3	8	1	2019-05-30 12:40:53.609085	2019-05-30 12:40:53.609085
34	4	8	1	2019-05-30 12:40:53.61174	2019-05-30 12:40:53.61174
35	5	8	1	2019-05-30 12:40:53.614438	2019-05-30 12:40:53.614438
36	6	11	4	2019-08-27 15:15:01.387876	2019-08-27 15:15:01.387876
37	7	11	4	2019-08-27 15:15:01.392901	2019-08-27 15:15:01.392901
38	8	11	4	2019-08-27 15:15:01.396026	2019-08-27 15:15:01.396026
39	9	11	4	2019-08-27 15:15:01.399328	2019-08-27 15:15:01.399328
40	10	11	4	2019-08-27 15:15:01.402982	2019-08-27 15:15:01.402982
41	6	12	4	2019-08-27 15:15:01.423098	2019-08-27 15:15:01.423098
42	7	12	4	2019-08-27 15:15:01.432089	2019-08-27 15:15:01.432089
43	8	12	4	2019-08-27 15:15:01.436157	2019-08-27 15:15:01.436157
44	9	12	4	2019-08-27 15:15:01.440425	2019-08-27 15:15:01.440425
45	10	12	4	2019-08-27 15:15:01.44448	2019-08-27 15:15:01.44448
46	6	13	4	2019-08-27 15:15:01.481068	2019-08-27 15:15:01.481068
47	7	13	4	2019-08-27 15:15:01.487694	2019-08-27 15:15:01.487694
48	8	13	4	2019-08-27 15:15:01.490636	2019-08-27 15:15:01.490636
49	9	13	4	2019-08-27 15:15:01.493447	2019-08-27 15:15:01.493447
50	10	13	4	2019-08-27 15:15:01.496251	2019-08-27 15:15:01.496251
51	6	14	4	2019-08-27 15:15:01.51563	2019-08-27 15:15:01.51563
52	7	14	4	2019-08-27 15:15:01.522128	2019-08-27 15:15:01.522128
53	8	14	4	2019-08-27 15:15:01.52507	2019-08-27 15:15:01.52507
54	9	14	4	2019-08-27 15:15:01.527885	2019-08-27 15:15:01.527885
55	10	14	4	2019-08-27 15:15:01.530763	2019-08-27 15:15:01.530763
56	6	15	4	2019-08-27 15:15:01.551772	2019-08-27 15:15:01.551772
57	7	15	4	2019-08-27 15:15:01.558156	2019-08-27 15:15:01.558156
58	8	15	4	2019-08-27 15:15:01.562869	2019-08-27 15:15:01.562869
59	9	15	4	2019-08-27 15:15:01.56588	2019-08-27 15:15:01.56588
60	10	15	4	2019-08-27 15:15:01.568816	2019-08-27 15:15:01.568816
61	6	16	4	2019-08-27 15:15:01.58796	2019-08-27 15:15:01.58796
62	7	16	4	2019-08-27 15:15:01.593929	2019-08-27 15:15:01.593929
63	8	16	4	2019-08-27 15:15:01.596834	2019-08-27 15:15:01.596834
64	9	16	4	2019-08-27 15:15:01.599513	2019-08-27 15:15:01.599513
65	10	16	4	2019-08-27 15:15:01.602219	2019-08-27 15:15:01.602219
66	6	17	4	2019-08-27 15:15:01.648478	2019-08-27 15:15:01.648478
67	7	17	4	2019-08-27 15:15:01.669145	2019-08-27 15:15:01.669145
68	8	17	4	2019-08-27 15:15:01.677641	2019-08-27 15:15:01.677641
69	9	17	4	2019-08-27 15:15:01.68058	2019-08-27 15:15:01.68058
70	10	17	4	2019-08-27 15:15:01.688591	2019-08-27 15:15:01.688591
71	11	20	1	2019-08-28 07:30:59.282066	2019-08-28 07:30:59.282066
72	12	20	1	2019-08-28 07:30:59.300098	2019-08-28 07:30:59.300098
73	13	20	1	2019-08-28 07:30:59.3196	2019-08-28 07:30:59.3196
74	14	20	1	2019-08-28 07:30:59.330705	2019-08-28 07:30:59.330705
75	15	20	1	2019-08-28 07:30:59.340053	2019-08-28 07:30:59.340053
76	11	21	1	2019-08-28 07:30:59.375597	2019-08-28 07:30:59.375597
77	12	21	1	2019-08-28 07:30:59.390423	2019-08-28 07:30:59.390423
78	13	21	1	2019-08-28 07:30:59.400009	2019-08-28 07:30:59.400009
79	14	21	1	2019-08-28 07:30:59.411362	2019-08-28 07:30:59.411362
80	15	21	1	2019-08-28 07:30:59.423568	2019-08-28 07:30:59.423568
81	11	22	1	2019-08-28 07:30:59.457875	2019-08-28 07:30:59.457875
82	12	22	1	2019-08-28 07:30:59.47339	2019-08-28 07:30:59.47339
83	13	22	1	2019-08-28 07:30:59.483587	2019-08-28 07:30:59.483587
84	14	22	1	2019-08-28 07:30:59.496408	2019-08-28 07:30:59.496408
85	15	22	1	2019-08-28 07:30:59.507245	2019-08-28 07:30:59.507245
86	11	23	1	2019-08-28 07:30:59.539543	2019-08-28 07:30:59.539543
87	12	23	1	2019-08-28 07:30:59.556913	2019-08-28 07:30:59.556913
88	13	23	1	2019-08-28 07:30:59.567351	2019-08-28 07:30:59.567351
89	14	23	1	2019-08-28 07:30:59.576623	2019-08-28 07:30:59.576623
90	15	23	1	2019-08-28 07:30:59.587471	2019-08-28 07:30:59.587471
91	11	24	1	2019-08-28 07:30:59.622807	2019-08-28 07:30:59.622807
92	12	24	1	2019-08-28 07:30:59.637426	2019-08-28 07:30:59.637426
93	13	24	1	2019-08-28 07:30:59.647041	2019-08-28 07:30:59.647041
94	14	24	1	2019-08-28 07:30:59.657173	2019-08-28 07:30:59.657173
95	15	24	1	2019-08-28 07:30:59.667605	2019-08-28 07:30:59.667605
96	11	25	1	2019-08-28 07:30:59.796691	2019-08-28 07:30:59.796691
97	12	25	1	2019-08-28 07:30:59.813112	2019-08-28 07:30:59.813112
98	13	25	1	2019-08-28 07:30:59.821969	2019-08-28 07:30:59.821969
99	14	25	1	2019-08-28 07:30:59.832638	2019-08-28 07:30:59.832638
100	15	25	1	2019-08-28 07:30:59.84328	2019-08-28 07:30:59.84328
101	11	26	1	2019-08-28 07:30:59.872859	2019-08-28 07:30:59.872859
102	12	26	1	2019-08-28 07:30:59.886128	2019-08-28 07:30:59.886128
103	13	26	1	2019-08-28 07:30:59.905659	2019-08-28 07:30:59.905659
104	14	26	1	2019-08-28 07:30:59.925714	2019-08-28 07:30:59.925714
105	15	26	1	2019-08-28 07:30:59.942892	2019-08-28 07:30:59.942892
106	16	29	6	2019-08-28 08:34:54.532731	2019-08-28 08:34:54.532731
107	17	29	6	2019-08-28 08:34:54.542186	2019-08-28 08:34:54.542186
108	18	29	6	2019-08-28 08:34:54.554678	2019-08-28 08:34:54.554678
109	19	29	6	2019-08-28 08:34:54.566101	2019-08-28 08:34:54.566101
110	20	29	6	2019-08-28 08:34:54.586704	2019-08-28 08:34:54.586704
111	16	30	6	2019-08-28 08:34:54.663183	2019-08-28 08:34:54.663183
112	17	30	6	2019-08-28 08:34:54.719912	2019-08-28 08:34:54.719912
113	18	30	6	2019-08-28 08:34:54.728566	2019-08-28 08:34:54.728566
114	19	30	6	2019-08-28 08:34:54.740822	2019-08-28 08:34:54.740822
115	20	30	6	2019-08-28 08:34:54.778399	2019-08-28 08:34:54.778399
116	16	31	6	2019-08-28 08:34:55.32959	2019-08-28 08:34:55.32959
117	17	31	6	2019-08-28 08:34:55.351086	2019-08-28 08:34:55.351086
118	18	31	6	2019-08-28 08:34:55.403052	2019-08-28 08:34:55.403052
119	19	31	6	2019-08-28 08:34:55.41723	2019-08-28 08:34:55.41723
120	20	31	6	2019-08-28 08:34:55.423602	2019-08-28 08:34:55.423602
121	16	32	6	2019-08-28 08:34:55.602583	2019-08-28 08:34:55.602583
122	17	32	6	2019-08-28 08:34:55.617928	2019-08-28 08:34:55.617928
123	18	32	6	2019-08-28 08:34:55.622272	2019-08-28 08:34:55.622272
124	19	32	6	2019-08-28 08:34:55.626451	2019-08-28 08:34:55.626451
125	20	32	6	2019-08-28 08:34:55.632748	2019-08-28 08:34:55.632748
126	16	33	6	2019-08-28 08:34:55.669468	2019-08-28 08:34:55.669468
127	17	33	6	2019-08-28 08:34:55.708933	2019-08-28 08:34:55.708933
128	18	33	6	2019-08-28 08:34:55.713795	2019-08-28 08:34:55.713795
129	19	33	6	2019-08-28 08:34:55.716614	2019-08-28 08:34:55.716614
130	20	33	6	2019-08-28 08:34:55.721236	2019-08-28 08:34:55.721236
131	16	34	6	2019-08-28 08:34:55.760345	2019-08-28 08:34:55.760345
132	17	34	6	2019-08-28 08:34:55.808684	2019-08-28 08:34:55.808684
133	18	34	6	2019-08-28 08:34:55.812131	2019-08-28 08:34:55.812131
134	19	34	6	2019-08-28 08:34:55.815239	2019-08-28 08:34:55.815239
135	20	34	6	2019-08-28 08:34:55.817953	2019-08-28 08:34:55.817953
136	16	35	6	2019-08-28 08:34:55.868591	2019-08-28 08:34:55.868591
137	17	35	6	2019-08-28 08:34:55.874583	2019-08-28 08:34:55.874583
138	18	35	6	2019-08-28 08:34:55.896576	2019-08-28 08:34:55.896576
139	19	35	6	2019-08-28 08:34:55.900229	2019-08-28 08:34:55.900229
140	20	35	6	2019-08-28 08:34:55.903062	2019-08-28 08:34:55.903062
176	27	47	8	2019-09-12 18:26:18.619876	2019-09-12 18:26:18.619876
177	28	47	8	2019-09-12 18:26:18.62561	2019-09-12 18:26:18.62561
178	29	47	8	2019-09-12 18:26:18.628763	2019-09-12 18:26:18.628763
179	30	47	8	2019-09-12 18:26:18.631781	2019-09-12 18:26:18.631781
180	31	47	8	2019-09-12 18:26:18.634844	2019-09-12 18:26:18.634844
181	27	48	8	2019-09-12 18:26:18.654661	2019-09-12 18:26:18.654661
182	28	48	8	2019-09-12 18:26:18.663517	2019-09-12 18:26:18.663517
183	29	48	8	2019-09-12 18:26:18.666332	2019-09-12 18:26:18.666332
184	30	48	8	2019-09-12 18:26:18.669084	2019-09-12 18:26:18.669084
185	31	48	8	2019-09-12 18:26:18.671917	2019-09-12 18:26:18.671917
186	27	49	8	2019-09-12 18:26:18.69061	2019-09-12 18:26:18.69061
187	28	49	8	2019-09-12 18:26:18.696842	2019-09-12 18:26:18.696842
188	29	49	8	2019-09-12 18:26:18.699704	2019-09-12 18:26:18.699704
189	30	49	8	2019-09-12 18:26:18.702573	2019-09-12 18:26:18.702573
190	31	49	8	2019-09-12 18:26:18.705383	2019-09-12 18:26:18.705383
191	27	50	8	2019-09-12 18:26:18.724132	2019-09-12 18:26:18.724132
192	28	50	8	2019-09-12 18:26:18.730449	2019-09-12 18:26:18.730449
193	29	50	8	2019-09-12 18:26:18.733295	2019-09-12 18:26:18.733295
194	30	50	8	2019-09-12 18:26:18.736117	2019-09-12 18:26:18.736117
195	31	50	8	2019-09-12 18:26:18.738908	2019-09-12 18:26:18.738908
196	27	51	8	2019-09-12 18:26:18.759144	2019-09-12 18:26:18.759144
197	28	51	8	2019-09-12 18:26:18.765715	2019-09-12 18:26:18.765715
198	29	51	8	2019-09-12 18:26:18.768523	2019-09-12 18:26:18.768523
199	30	51	8	2019-09-12 18:26:18.77135	2019-09-12 18:26:18.77135
200	31	51	8	2019-09-12 18:26:18.774171	2019-09-12 18:26:18.774171
201	27	52	8	2019-09-12 18:26:18.79378	2019-09-12 18:26:18.79378
202	28	52	8	2019-09-12 18:26:18.799921	2019-09-12 18:26:18.799921
203	29	52	8	2019-09-12 18:26:18.802863	2019-09-12 18:26:18.802863
204	30	52	8	2019-09-12 18:26:18.806755	2019-09-12 18:26:18.806755
205	31	52	8	2019-09-12 18:26:18.809562	2019-09-12 18:26:18.809562
206	27	53	8	2019-09-12 18:26:18.828088	2019-09-12 18:26:18.828088
207	28	53	8	2019-09-12 18:26:18.834349	2019-09-12 18:26:18.834349
208	29	53	8	2019-09-12 18:26:18.837108	2019-09-12 18:26:18.837108
209	30	53	8	2019-09-12 18:26:18.839879	2019-09-12 18:26:18.839879
210	31	53	8	2019-09-12 18:26:18.843347	2019-09-12 18:26:18.843347
211	32	56	9	2019-11-13 13:57:41.932426	2019-11-13 13:57:41.932426
212	33	56	9	2019-11-13 13:57:41.940065	2019-11-13 13:57:41.940065
213	34	56	9	2019-11-13 13:57:41.94585	2019-11-13 13:57:41.94585
214	35	56	9	2019-11-13 13:57:41.950526	2019-11-13 13:57:41.950526
215	36	56	9	2019-11-13 13:57:41.954995	2019-11-13 13:57:41.954995
216	32	57	9	2019-11-13 13:57:41.983756	2019-11-13 13:57:41.983756
217	33	57	9	2019-11-13 13:57:41.993542	2019-11-13 13:57:41.993542
218	34	57	9	2019-11-13 13:57:41.996879	2019-11-13 13:57:41.996879
219	35	57	9	2019-11-13 13:57:42.004208	2019-11-13 13:57:42.004208
220	36	57	9	2019-11-13 13:57:42.007556	2019-11-13 13:57:42.007556
221	32	58	9	2019-11-13 13:57:42.035928	2019-11-13 13:57:42.035928
222	33	58	9	2019-11-13 13:57:42.04421	2019-11-13 13:57:42.04421
223	34	58	9	2019-11-13 13:57:42.048161	2019-11-13 13:57:42.048161
224	35	58	9	2019-11-13 13:57:42.051879	2019-11-13 13:57:42.051879
225	36	58	9	2019-11-13 13:57:42.054892	2019-11-13 13:57:42.054892
226	32	59	9	2019-11-13 13:57:42.082438	2019-11-13 13:57:42.082438
227	33	59	9	2019-11-13 13:57:42.089427	2019-11-13 13:57:42.089427
228	34	59	9	2019-11-13 13:57:42.093327	2019-11-13 13:57:42.093327
229	35	59	9	2019-11-13 13:57:42.098139	2019-11-13 13:57:42.098139
230	36	59	9	2019-11-13 13:57:42.102444	2019-11-13 13:57:42.102444
231	32	60	9	2019-11-13 13:57:42.139191	2019-11-13 13:57:42.139191
232	33	60	9	2019-11-13 13:57:42.15309	2019-11-13 13:57:42.15309
233	34	60	9	2019-11-13 13:57:42.1579	2019-11-13 13:57:42.1579
234	35	60	9	2019-11-13 13:57:42.165592	2019-11-13 13:57:42.165592
235	36	60	9	2019-11-13 13:57:42.170508	2019-11-13 13:57:42.170508
236	32	61	9	2019-11-13 13:57:42.200821	2019-11-13 13:57:42.200821
237	33	61	9	2019-11-13 13:57:42.20909	2019-11-13 13:57:42.20909
238	34	61	9	2019-11-13 13:57:42.212656	2019-11-13 13:57:42.212656
239	35	61	9	2019-11-13 13:57:42.215963	2019-11-13 13:57:42.215963
240	36	61	9	2019-11-13 13:57:42.221031	2019-11-13 13:57:42.221031
241	32	62	9	2019-11-13 13:57:42.248447	2019-11-13 13:57:42.248447
242	33	62	9	2019-11-13 13:57:42.256436	2019-11-13 13:57:42.256436
243	34	62	9	2019-11-13 13:57:42.262343	2019-11-13 13:57:42.262343
244	35	62	9	2019-11-13 13:57:42.267048	2019-11-13 13:57:42.267048
245	36	62	9	2019-11-13 13:57:42.271777	2019-11-13 13:57:42.271777
246	37	65	4	2019-11-25 14:09:23.367823	2019-11-25 14:09:23.367823
247	38	65	4	2019-11-25 14:09:23.387306	2019-11-25 14:09:23.387306
248	39	65	4	2019-11-25 14:09:23.398311	2019-11-25 14:09:23.398311
249	40	65	4	2019-11-25 14:09:23.41142	2019-11-25 14:09:23.41142
250	41	65	4	2019-11-25 14:09:23.423234	2019-11-25 14:09:23.423234
251	37	66	4	2019-11-25 14:09:23.473852	2019-11-25 14:09:23.473852
252	38	66	4	2019-11-25 14:09:23.496308	2019-11-25 14:09:23.496308
253	39	66	4	2019-11-25 14:09:23.508533	2019-11-25 14:09:23.508533
254	40	66	4	2019-11-25 14:09:23.524745	2019-11-25 14:09:23.524745
255	41	66	4	2019-11-25 14:09:23.555885	2019-11-25 14:09:23.555885
256	37	67	4	2019-11-25 14:09:23.602036	2019-11-25 14:09:23.602036
257	38	67	4	2019-11-25 14:09:23.619528	2019-11-25 14:09:23.619528
258	39	67	4	2019-11-25 14:09:23.634839	2019-11-25 14:09:23.634839
259	40	67	4	2019-11-25 14:09:23.650083	2019-11-25 14:09:23.650083
260	41	67	4	2019-11-25 14:09:23.665664	2019-11-25 14:09:23.665664
261	37	68	4	2019-11-25 14:09:23.708084	2019-11-25 14:09:23.708084
262	38	68	4	2019-11-25 14:09:23.725736	2019-11-25 14:09:23.725736
263	39	68	4	2019-11-25 14:09:23.737774	2019-11-25 14:09:23.737774
264	40	68	4	2019-11-25 14:09:23.753036	2019-11-25 14:09:23.753036
265	41	68	4	2019-11-25 14:09:23.765066	2019-11-25 14:09:23.765066
266	37	69	4	2019-11-25 14:09:23.807957	2019-11-25 14:09:23.807957
267	38	69	4	2019-11-25 14:09:23.827928	2019-11-25 14:09:23.827928
268	39	69	4	2019-11-25 14:09:23.83977	2019-11-25 14:09:23.83977
269	40	69	4	2019-11-25 14:09:23.851582	2019-11-25 14:09:23.851582
270	41	69	4	2019-11-25 14:09:23.864166	2019-11-25 14:09:23.864166
271	37	70	4	2019-11-25 14:09:23.908824	2019-11-25 14:09:23.908824
272	38	70	4	2019-11-25 14:09:23.924051	2019-11-25 14:09:23.924051
273	39	70	4	2019-11-25 14:09:23.936508	2019-11-25 14:09:23.936508
274	40	70	4	2019-11-25 14:09:23.947912	2019-11-25 14:09:23.947912
275	41	70	4	2019-11-25 14:09:23.960071	2019-11-25 14:09:23.960071
276	37	71	4	2019-11-25 14:09:24.682426	2019-11-25 14:09:24.682426
277	38	71	4	2019-11-25 14:09:24.868335	2019-11-25 14:09:24.868335
278	39	71	4	2019-11-25 14:09:24.967818	2019-11-25 14:09:24.967818
279	40	71	4	2019-11-25 14:09:24.9925	2019-11-25 14:09:24.9925
280	41	71	4	2019-11-25 14:09:25.027282	2019-11-25 14:09:25.027282
281	42	79	10	2019-11-25 18:00:52.676505	2019-11-25 18:00:52.676505
282	43	79	10	2019-11-25 18:00:52.681201	2019-11-25 18:00:52.681201
283	44	79	10	2019-11-25 18:00:52.684591	2019-11-25 18:00:52.684591
284	45	79	10	2019-11-25 18:00:52.687916	2019-11-25 18:00:52.687916
285	46	79	10	2019-11-25 18:00:52.691924	2019-11-25 18:00:52.691924
286	42	80	10	2019-11-25 18:00:52.718339	2019-11-25 18:00:52.718339
287	43	80	10	2019-11-25 18:00:52.725885	2019-11-25 18:00:52.725885
288	44	80	10	2019-11-25 18:00:52.729419	2019-11-25 18:00:52.729419
289	45	80	10	2019-11-25 18:00:52.732584	2019-11-25 18:00:52.732584
290	46	80	10	2019-11-25 18:00:52.73636	2019-11-25 18:00:52.73636
291	42	81	10	2019-11-25 18:00:52.764059	2019-11-25 18:00:52.764059
292	43	81	10	2019-11-25 18:00:52.771994	2019-11-25 18:00:52.771994
293	44	81	10	2019-11-25 18:00:52.775436	2019-11-25 18:00:52.775436
294	45	81	10	2019-11-25 18:00:52.779077	2019-11-25 18:00:52.779077
295	46	81	10	2019-11-25 18:00:52.782363	2019-11-25 18:00:52.782363
296	42	82	10	2019-11-25 18:00:52.807225	2019-11-25 18:00:52.807225
297	43	82	10	2019-11-25 18:00:52.814866	2019-11-25 18:00:52.814866
298	44	82	10	2019-11-25 18:00:52.81897	2019-11-25 18:00:52.81897
299	45	82	10	2019-11-25 18:00:52.822186	2019-11-25 18:00:52.822186
300	46	82	10	2019-11-25 18:00:52.825642	2019-11-25 18:00:52.825642
301	42	83	10	2019-11-25 18:00:52.85106	2019-11-25 18:00:52.85106
302	43	83	10	2019-11-25 18:00:52.859529	2019-11-25 18:00:52.859529
303	44	83	10	2019-11-25 18:00:52.863992	2019-11-25 18:00:52.863992
304	45	83	10	2019-11-25 18:00:52.867131	2019-11-25 18:00:52.867131
305	46	83	10	2019-11-25 18:00:52.871581	2019-11-25 18:00:52.871581
306	42	84	10	2019-11-25 18:00:52.897911	2019-11-25 18:00:52.897911
307	43	84	10	2019-11-25 18:00:52.90667	2019-11-25 18:00:52.90667
308	44	84	10	2019-11-25 18:00:52.910515	2019-11-25 18:00:52.910515
309	45	84	10	2019-11-25 18:00:52.914519	2019-11-25 18:00:52.914519
310	46	84	10	2019-11-25 18:00:52.918425	2019-11-25 18:00:52.918425
311	42	85	10	2019-11-25 18:00:52.944811	2019-11-25 18:00:52.944811
312	43	85	10	2019-11-25 18:00:52.95282	2019-11-25 18:00:52.95282
313	44	85	10	2019-11-25 18:00:52.956141	2019-11-25 18:00:52.956141
314	45	85	10	2019-11-25 18:00:52.960305	2019-11-25 18:00:52.960305
315	46	85	10	2019-11-25 18:00:52.964016	2019-11-25 18:00:52.964016
316	47	88	11	2020-05-12 16:13:02.520148	2020-05-12 16:13:02.520148
317	48	88	11	2020-05-12 16:13:02.526908	2020-05-12 16:13:02.526908
318	49	88	11	2020-05-12 16:13:02.530805	2020-05-12 16:13:02.530805
319	50	88	11	2020-05-12 16:13:02.535395	2020-05-12 16:13:02.535395
320	51	88	11	2020-05-12 16:13:02.539027	2020-05-12 16:13:02.539027
321	47	89	11	2020-05-12 16:13:02.563678	2020-05-12 16:13:02.563678
322	48	89	11	2020-05-12 16:13:02.574797	2020-05-12 16:13:02.574797
323	49	89	11	2020-05-12 16:13:02.581843	2020-05-12 16:13:02.581843
324	50	89	11	2020-05-12 16:13:02.587111	2020-05-12 16:13:02.587111
325	51	89	11	2020-05-12 16:13:02.590949	2020-05-12 16:13:02.590949
326	47	90	11	2020-05-12 16:13:02.620345	2020-05-12 16:13:02.620345
327	48	90	11	2020-05-12 16:13:02.632632	2020-05-12 16:13:02.632632
328	49	90	11	2020-05-12 16:13:02.637613	2020-05-12 16:13:02.637613
329	50	90	11	2020-05-12 16:13:02.642478	2020-05-12 16:13:02.642478
330	51	90	11	2020-05-12 16:13:02.647013	2020-05-12 16:13:02.647013
331	47	91	11	2020-05-12 16:13:02.670848	2020-05-12 16:13:02.670848
332	48	91	11	2020-05-12 16:13:02.681059	2020-05-12 16:13:02.681059
333	49	91	11	2020-05-12 16:13:02.684831	2020-05-12 16:13:02.684831
334	50	91	11	2020-05-12 16:13:02.689557	2020-05-12 16:13:02.689557
335	51	91	11	2020-05-12 16:13:02.694311	2020-05-12 16:13:02.694311
336	47	92	11	2020-05-12 16:13:02.721813	2020-05-12 16:13:02.721813
337	48	92	11	2020-05-12 16:13:02.732845	2020-05-12 16:13:02.732845
338	49	92	11	2020-05-12 16:13:02.737873	2020-05-12 16:13:02.737873
339	50	92	11	2020-05-12 16:13:02.742851	2020-05-12 16:13:02.742851
340	51	92	11	2020-05-12 16:13:02.747722	2020-05-12 16:13:02.747722
341	47	93	11	2020-05-12 16:13:02.777409	2020-05-12 16:13:02.777409
342	48	93	11	2020-05-12 16:13:02.788295	2020-05-12 16:13:02.788295
343	49	93	11	2020-05-12 16:13:02.794124	2020-05-12 16:13:02.794124
344	50	93	11	2020-05-12 16:13:02.799912	2020-05-12 16:13:02.799912
345	51	93	11	2020-05-12 16:13:02.806606	2020-05-12 16:13:02.806606
346	47	94	11	2020-05-12 16:13:02.850298	2020-05-12 16:13:02.850298
347	48	94	11	2020-05-12 16:13:02.861228	2020-05-12 16:13:02.861228
348	49	94	11	2020-05-12 16:13:02.867818	2020-05-12 16:13:02.867818
349	50	94	11	2020-05-12 16:13:02.873513	2020-05-12 16:13:02.873513
350	51	94	11	2020-05-12 16:13:02.881887	2020-05-12 16:13:02.881887
351	52	97	12	2020-06-10 14:43:17.381816	2020-06-10 14:43:17.381816
352	53	97	12	2020-06-10 14:43:17.3895	2020-06-10 14:43:17.3895
353	54	97	12	2020-06-10 14:43:17.392869	2020-06-10 14:43:17.392869
354	55	97	12	2020-06-10 14:43:17.396095	2020-06-10 14:43:17.396095
355	56	97	12	2020-06-10 14:43:17.399329	2020-06-10 14:43:17.399329
356	52	98	12	2020-06-10 14:43:17.421612	2020-06-10 14:43:17.421612
357	53	98	12	2020-06-10 14:43:17.431341	2020-06-10 14:43:17.431341
358	54	98	12	2020-06-10 14:43:17.434707	2020-06-10 14:43:17.434707
359	55	98	12	2020-06-10 14:43:17.437984	2020-06-10 14:43:17.437984
360	56	98	12	2020-06-10 14:43:17.441173	2020-06-10 14:43:17.441173
361	52	99	12	2020-06-10 14:43:17.46673	2020-06-10 14:43:17.46673
362	53	99	12	2020-06-10 14:43:17.474749	2020-06-10 14:43:17.474749
363	54	99	12	2020-06-10 14:43:17.480495	2020-06-10 14:43:17.480495
364	55	99	12	2020-06-10 14:43:17.483763	2020-06-10 14:43:17.483763
365	56	99	12	2020-06-10 14:43:17.486888	2020-06-10 14:43:17.486888
366	52	100	12	2020-06-10 14:43:17.522195	2020-06-10 14:43:17.522195
367	53	100	12	2020-06-10 14:43:17.530059	2020-06-10 14:43:17.530059
368	54	100	12	2020-06-10 14:43:17.534587	2020-06-10 14:43:17.534587
369	55	100	12	2020-06-10 14:43:17.538743	2020-06-10 14:43:17.538743
370	56	100	12	2020-06-10 14:43:17.542774	2020-06-10 14:43:17.542774
371	52	101	12	2020-06-10 14:43:17.600544	2020-06-10 14:43:17.600544
372	53	101	12	2020-06-10 14:43:17.607724	2020-06-10 14:43:17.607724
373	54	101	12	2020-06-10 14:43:17.610802	2020-06-10 14:43:17.610802
374	55	101	12	2020-06-10 14:43:17.614424	2020-06-10 14:43:17.614424
375	56	101	12	2020-06-10 14:43:17.617526	2020-06-10 14:43:17.617526
376	52	102	12	2020-06-10 14:43:17.640144	2020-06-10 14:43:17.640144
377	53	102	12	2020-06-10 14:43:17.646837	2020-06-10 14:43:17.646837
378	54	102	12	2020-06-10 14:43:17.649877	2020-06-10 14:43:17.649877
379	55	102	12	2020-06-10 14:43:17.652867	2020-06-10 14:43:17.652867
380	56	102	12	2020-06-10 14:43:17.655882	2020-06-10 14:43:17.655882
381	52	103	12	2020-06-10 14:43:17.680777	2020-06-10 14:43:17.680777
382	53	103	12	2020-06-10 14:43:17.687598	2020-06-10 14:43:17.687598
383	54	103	12	2020-06-10 14:43:17.690795	2020-06-10 14:43:17.690795
384	55	103	12	2020-06-10 14:43:17.693894	2020-06-10 14:43:17.693894
385	56	103	12	2020-06-10 14:43:17.696996	2020-06-10 14:43:17.696996
386	57	107	16	2020-10-07 19:20:25.545276	2020-10-07 19:20:25.545276
387	58	107	16	2020-10-07 19:20:25.552155	2020-10-07 19:20:25.552155
388	59	107	16	2020-10-07 19:20:25.557162	2020-10-07 19:20:25.557162
389	60	107	16	2020-10-07 19:20:25.56221	2020-10-07 19:20:25.56221
390	61	107	16	2020-10-07 19:20:25.567518	2020-10-07 19:20:25.567518
391	57	108	16	2020-10-07 19:20:25.612906	2020-10-07 19:20:25.612906
392	58	108	16	2020-10-07 19:20:25.625161	2020-10-07 19:20:25.625161
393	59	108	16	2020-10-07 19:20:25.629185	2020-10-07 19:20:25.629185
394	60	108	16	2020-10-07 19:20:25.632595	2020-10-07 19:20:25.632595
395	61	108	16	2020-10-07 19:20:25.636836	2020-10-07 19:20:25.636836
396	57	109	16	2020-10-07 19:20:25.664583	2020-10-07 19:20:25.664583
397	58	109	16	2020-10-07 19:20:25.671606	2020-10-07 19:20:25.671606
398	59	109	16	2020-10-07 19:20:25.676251	2020-10-07 19:20:25.676251
399	60	109	16	2020-10-07 19:20:25.679432	2020-10-07 19:20:25.679432
400	61	109	16	2020-10-07 19:20:25.682883	2020-10-07 19:20:25.682883
401	57	110	16	2020-10-07 19:20:25.707143	2020-10-07 19:20:25.707143
402	58	110	16	2020-10-07 19:20:25.714513	2020-10-07 19:20:25.714513
403	59	110	16	2020-10-07 19:20:25.717537	2020-10-07 19:20:25.717537
404	60	110	16	2020-10-07 19:20:25.720538	2020-10-07 19:20:25.720538
405	61	110	16	2020-10-07 19:20:25.723567	2020-10-07 19:20:25.723567
406	57	111	16	2020-10-07 19:20:25.751586	2020-10-07 19:20:25.751586
407	58	111	16	2020-10-07 19:20:25.760481	2020-10-07 19:20:25.760481
408	59	111	16	2020-10-07 19:20:25.763575	2020-10-07 19:20:25.763575
409	60	111	16	2020-10-07 19:20:25.772411	2020-10-07 19:20:25.772411
410	61	111	16	2020-10-07 19:20:25.776205	2020-10-07 19:20:25.776205
411	57	112	16	2020-10-07 19:20:25.800885	2020-10-07 19:20:25.800885
412	58	112	16	2020-10-07 19:20:25.808723	2020-10-07 19:20:25.808723
413	59	112	16	2020-10-07 19:20:25.811863	2020-10-07 19:20:25.811863
414	60	112	16	2020-10-07 19:20:25.815421	2020-10-07 19:20:25.815421
415	61	112	16	2020-10-07 19:20:25.81861	2020-10-07 19:20:25.81861
416	57	113	16	2020-10-07 19:20:25.844291	2020-10-07 19:20:25.844291
417	58	113	16	2020-10-07 19:20:25.851416	2020-10-07 19:20:25.851416
418	59	113	16	2020-10-07 19:20:25.854475	2020-10-07 19:20:25.854475
419	60	113	16	2020-10-07 19:20:25.857986	2020-10-07 19:20:25.857986
420	61	113	16	2020-10-07 19:20:25.862071	2020-10-07 19:20:25.862071
421	62	116	1	2020-11-18 14:50:52.736108	2020-11-18 14:50:52.736108
422	63	116	1	2020-11-18 14:50:52.751866	2020-11-18 14:50:52.751866
423	64	116	1	2020-11-18 14:50:52.763757	2020-11-18 14:50:52.763757
424	65	116	1	2020-11-18 14:50:52.776289	2020-11-18 14:50:52.776289
425	66	116	1	2020-11-18 14:50:52.789036	2020-11-18 14:50:52.789036
426	62	117	1	2020-11-18 14:50:52.832958	2020-11-18 14:50:52.832958
427	63	117	1	2020-11-18 14:50:52.851818	2020-11-18 14:50:52.851818
428	64	117	1	2020-11-18 14:50:52.865919	2020-11-18 14:50:52.865919
429	65	117	1	2020-11-18 14:50:52.878872	2020-11-18 14:50:52.878872
430	66	117	1	2020-11-18 14:50:52.889964	2020-11-18 14:50:52.889964
431	62	118	1	2020-11-18 14:50:52.932089	2020-11-18 14:50:52.932089
432	63	118	1	2020-11-18 14:50:52.948857	2020-11-18 14:50:52.948857
433	64	118	1	2020-11-18 14:50:52.962274	2020-11-18 14:50:52.962274
434	65	118	1	2020-11-18 14:50:52.976904	2020-11-18 14:50:52.976904
435	66	118	1	2020-11-18 14:50:52.991006	2020-11-18 14:50:52.991006
436	62	119	1	2020-11-18 14:50:53.033638	2020-11-18 14:50:53.033638
437	63	119	1	2020-11-18 14:50:53.048907	2020-11-18 14:50:53.048907
438	64	119	1	2020-11-18 14:50:53.060669	2020-11-18 14:50:53.060669
439	65	119	1	2020-11-18 14:50:53.073172	2020-11-18 14:50:53.073172
440	66	119	1	2020-11-18 14:50:53.085417	2020-11-18 14:50:53.085417
441	62	120	1	2020-11-18 14:50:53.129552	2020-11-18 14:50:53.129552
442	63	120	1	2020-11-18 14:50:53.145109	2020-11-18 14:50:53.145109
443	64	120	1	2020-11-18 14:50:53.157101	2020-11-18 14:50:53.157101
444	65	120	1	2020-11-18 14:50:53.171429	2020-11-18 14:50:53.171429
445	66	120	1	2020-11-18 14:50:53.184342	2020-11-18 14:50:53.184342
446	62	121	1	2020-11-18 14:50:53.229035	2020-11-18 14:50:53.229035
447	63	121	1	2020-11-18 14:50:53.249107	2020-11-18 14:50:53.249107
448	64	121	1	2020-11-18 14:50:53.262505	2020-11-18 14:50:53.262505
449	65	121	1	2020-11-18 14:50:53.27679	2020-11-18 14:50:53.27679
450	66	121	1	2020-11-18 14:50:53.29163	2020-11-18 14:50:53.29163
451	62	122	1	2020-11-18 14:50:53.337377	2020-11-18 14:50:53.337377
452	63	122	1	2020-11-18 14:50:53.353976	2020-11-18 14:50:53.353976
453	64	122	1	2020-11-18 14:50:53.381489	2020-11-18 14:50:53.381489
454	65	122	1	2020-11-18 14:50:53.392501	2020-11-18 14:50:53.392501
455	66	122	1	2020-11-18 14:50:53.40397	2020-11-18 14:50:53.40397
456	67	125	17	2020-11-18 14:53:24.992363	2020-11-18 14:53:24.992363
457	68	125	17	2020-11-18 14:53:24.996095	2020-11-18 14:53:24.996095
458	69	125	17	2020-11-18 14:53:24.999395	2020-11-18 14:53:24.999395
459	70	125	17	2020-11-18 14:53:25.004467	2020-11-18 14:53:25.004467
460	71	125	17	2020-11-18 14:53:25.008656	2020-11-18 14:53:25.008656
461	67	126	17	2020-11-18 14:53:25.035304	2020-11-18 14:53:25.035304
462	68	126	17	2020-11-18 14:53:25.044456	2020-11-18 14:53:25.044456
463	69	126	17	2020-11-18 14:53:25.048179	2020-11-18 14:53:25.048179
464	70	126	17	2020-11-18 14:53:25.05168	2020-11-18 14:53:25.05168
465	71	126	17	2020-11-18 14:53:25.055437	2020-11-18 14:53:25.055437
466	67	127	17	2020-11-18 14:53:25.079031	2020-11-18 14:53:25.079031
467	68	127	17	2020-11-18 14:53:25.086374	2020-11-18 14:53:25.086374
468	69	127	17	2020-11-18 14:53:25.089776	2020-11-18 14:53:25.089776
469	70	127	17	2020-11-18 14:53:25.092975	2020-11-18 14:53:25.092975
470	71	127	17	2020-11-18 14:53:25.096225	2020-11-18 14:53:25.096225
471	67	128	17	2020-11-18 14:53:25.118841	2020-11-18 14:53:25.118841
472	68	128	17	2020-11-18 14:53:25.127422	2020-11-18 14:53:25.127422
473	69	128	17	2020-11-18 14:53:25.147033	2020-11-18 14:53:25.147033
474	70	128	17	2020-11-18 14:53:25.150227	2020-11-18 14:53:25.150227
475	71	128	17	2020-11-18 14:53:25.153177	2020-11-18 14:53:25.153177
476	67	129	17	2020-11-18 14:53:25.174186	2020-11-18 14:53:25.174186
477	68	129	17	2020-11-18 14:53:25.18162	2020-11-18 14:53:25.18162
478	69	129	17	2020-11-18 14:53:25.184812	2020-11-18 14:53:25.184812
479	70	129	17	2020-11-18 14:53:25.187895	2020-11-18 14:53:25.187895
480	71	129	17	2020-11-18 14:53:25.190934	2020-11-18 14:53:25.190934
481	67	130	17	2020-11-18 14:53:25.212485	2020-11-18 14:53:25.212485
482	68	130	17	2020-11-18 14:53:25.219073	2020-11-18 14:53:25.219073
483	69	130	17	2020-11-18 14:53:25.222409	2020-11-18 14:53:25.222409
484	70	130	17	2020-11-18 14:53:25.225843	2020-11-18 14:53:25.225843
485	71	130	17	2020-11-18 14:53:25.228841	2020-11-18 14:53:25.228841
486	67	131	17	2020-11-18 14:53:25.250065	2020-11-18 14:53:25.250065
487	68	131	17	2020-11-18 14:53:25.257592	2020-11-18 14:53:25.257592
488	69	131	17	2020-11-18 14:53:25.260708	2020-11-18 14:53:25.260708
489	70	131	17	2020-11-18 14:53:25.264596	2020-11-18 14:53:25.264596
490	71	131	17	2020-11-18 14:53:25.268059	2020-11-18 14:53:25.268059
491	72	138	20	2021-05-19 20:01:51.963922	2021-05-19 20:01:51.963922
492	73	138	20	2021-05-19 20:01:51.969948	2021-05-19 20:01:51.969948
493	74	138	20	2021-05-19 20:01:51.9738	2021-05-19 20:01:51.9738
494	75	138	20	2021-05-19 20:01:51.978757	2021-05-19 20:01:51.978757
495	76	138	20	2021-05-19 20:01:51.982395	2021-05-19 20:01:51.982395
496	72	139	20	2021-05-19 20:01:52.003057	2021-05-19 20:01:52.003057
497	73	139	20	2021-05-19 20:01:52.011528	2021-05-19 20:01:52.011528
498	74	139	20	2021-05-19 20:01:52.014918	2021-05-19 20:01:52.014918
499	75	139	20	2021-05-19 20:01:52.018581	2021-05-19 20:01:52.018581
500	76	139	20	2021-05-19 20:01:52.021303	2021-05-19 20:01:52.021303
501	72	140	20	2021-05-19 20:01:52.043404	2021-05-19 20:01:52.043404
502	73	140	20	2021-05-19 20:01:52.072139	2021-05-19 20:01:52.072139
503	74	140	20	2021-05-19 20:01:52.075223	2021-05-19 20:01:52.075223
504	75	140	20	2021-05-19 20:01:52.077724	2021-05-19 20:01:52.077724
505	76	140	20	2021-05-19 20:01:52.080254	2021-05-19 20:01:52.080254
506	72	141	20	2021-05-19 20:01:52.095895	2021-05-19 20:01:52.095895
507	73	141	20	2021-05-19 20:01:52.10088	2021-05-19 20:01:52.10088
508	74	141	20	2021-05-19 20:01:52.102826	2021-05-19 20:01:52.102826
509	75	141	20	2021-05-19 20:01:52.104752	2021-05-19 20:01:52.104752
510	76	141	20	2021-05-19 20:01:52.106758	2021-05-19 20:01:52.106758
511	72	142	20	2021-05-19 20:01:52.124228	2021-05-19 20:01:52.124228
512	73	142	20	2021-05-19 20:01:52.129638	2021-05-19 20:01:52.129638
513	74	142	20	2021-05-19 20:01:52.131846	2021-05-19 20:01:52.131846
514	75	142	20	2021-05-19 20:01:52.134184	2021-05-19 20:01:52.134184
515	76	142	20	2021-05-19 20:01:52.136216	2021-05-19 20:01:52.136216
516	72	143	20	2021-05-19 20:01:52.150841	2021-05-19 20:01:52.150841
517	73	143	20	2021-05-19 20:01:52.155765	2021-05-19 20:01:52.155765
518	74	143	20	2021-05-19 20:01:52.157758	2021-05-19 20:01:52.157758
519	75	143	20	2021-05-19 20:01:52.160013	2021-05-19 20:01:52.160013
520	76	143	20	2021-05-19 20:01:52.162292	2021-05-19 20:01:52.162292
521	72	144	20	2021-05-19 20:01:52.177188	2021-05-19 20:01:52.177188
522	73	144	20	2021-05-19 20:01:52.182129	2021-05-19 20:01:52.182129
523	74	144	20	2021-05-19 20:01:52.184162	2021-05-19 20:01:52.184162
524	75	144	20	2021-05-19 20:01:52.186095	2021-05-19 20:01:52.186095
525	76	144	20	2021-05-19 20:01:52.188413	2021-05-19 20:01:52.188413
526	77	156	21	2021-05-20 14:12:30.579406	2021-05-20 14:12:30.579406
527	78	156	21	2021-05-20 14:12:30.581876	2021-05-20 14:12:30.581876
528	79	156	21	2021-05-20 14:12:30.583939	2021-05-20 14:12:30.583939
529	80	156	21	2021-05-20 14:12:30.585952	2021-05-20 14:12:30.585952
530	81	156	21	2021-05-20 14:12:30.588045	2021-05-20 14:12:30.588045
531	77	157	21	2021-05-20 14:12:30.602987	2021-05-20 14:12:30.602987
532	78	157	21	2021-05-20 14:12:30.609244	2021-05-20 14:12:30.609244
533	79	157	21	2021-05-20 14:12:30.612456	2021-05-20 14:12:30.612456
534	80	157	21	2021-05-20 14:12:30.616101	2021-05-20 14:12:30.616101
535	81	157	21	2021-05-20 14:12:30.618486	2021-05-20 14:12:30.618486
536	77	158	21	2021-05-20 14:12:30.648189	2021-05-20 14:12:30.648189
537	78	158	21	2021-05-20 14:12:30.655875	2021-05-20 14:12:30.655875
538	79	158	21	2021-05-20 14:12:30.664922	2021-05-20 14:12:30.664922
539	80	158	21	2021-05-20 14:12:30.678127	2021-05-20 14:12:30.678127
540	81	158	21	2021-05-20 14:12:30.682197	2021-05-20 14:12:30.682197
541	77	159	21	2021-05-20 14:12:30.704556	2021-05-20 14:12:30.704556
542	78	159	21	2021-05-20 14:12:30.711426	2021-05-20 14:12:30.711426
543	79	159	21	2021-05-20 14:12:30.714859	2021-05-20 14:12:30.714859
544	80	159	21	2021-05-20 14:12:30.717635	2021-05-20 14:12:30.717635
545	81	159	21	2021-05-20 14:12:30.725705	2021-05-20 14:12:30.725705
546	77	160	21	2021-05-20 14:12:30.745125	2021-05-20 14:12:30.745125
547	78	160	21	2021-05-20 14:12:30.751452	2021-05-20 14:12:30.751452
548	79	160	21	2021-05-20 14:12:30.755489	2021-05-20 14:12:30.755489
549	80	160	21	2021-05-20 14:12:30.758678	2021-05-20 14:12:30.758678
550	81	160	21	2021-05-20 14:12:30.761382	2021-05-20 14:12:30.761382
551	77	161	21	2021-05-20 14:12:30.775186	2021-05-20 14:12:30.775186
552	78	161	21	2021-05-20 14:12:30.779908	2021-05-20 14:12:30.779908
553	79	161	21	2021-05-20 14:12:30.782259	2021-05-20 14:12:30.782259
554	80	161	21	2021-05-20 14:12:30.784972	2021-05-20 14:12:30.784972
555	81	161	21	2021-05-20 14:12:30.786952	2021-05-20 14:12:30.786952
556	77	162	21	2021-05-20 14:12:30.800851	2021-05-20 14:12:30.800851
557	78	162	21	2021-05-20 14:12:30.805806	2021-05-20 14:12:30.805806
558	79	162	21	2021-05-20 14:12:30.807835	2021-05-20 14:12:30.807835
559	80	162	21	2021-05-20 14:12:30.809814	2021-05-20 14:12:30.809814
560	81	162	21	2021-05-20 14:12:30.811878	2021-05-20 14:12:30.811878
561	82	165	22	2021-05-20 14:12:57.266926	2021-05-20 14:12:57.266926
562	83	165	22	2021-05-20 14:12:57.269388	2021-05-20 14:12:57.269388
563	84	165	22	2021-05-20 14:12:57.271487	2021-05-20 14:12:57.271487
564	85	165	22	2021-05-20 14:12:57.273654	2021-05-20 14:12:57.273654
565	86	165	22	2021-05-20 14:12:57.275816	2021-05-20 14:12:57.275816
566	82	166	22	2021-05-20 14:12:57.290967	2021-05-20 14:12:57.290967
567	83	166	22	2021-05-20 14:12:57.295952	2021-05-20 14:12:57.295952
568	84	166	22	2021-05-20 14:12:57.299228	2021-05-20 14:12:57.299228
569	85	166	22	2021-05-20 14:12:57.302029	2021-05-20 14:12:57.302029
570	86	166	22	2021-05-20 14:12:57.304194	2021-05-20 14:12:57.304194
571	82	167	22	2021-05-20 14:12:57.318421	2021-05-20 14:12:57.318421
572	83	167	22	2021-05-20 14:12:57.323063	2021-05-20 14:12:57.323063
573	84	167	22	2021-05-20 14:12:57.325326	2021-05-20 14:12:57.325326
574	85	167	22	2021-05-20 14:12:57.327523	2021-05-20 14:12:57.327523
575	86	167	22	2021-05-20 14:12:57.330067	2021-05-20 14:12:57.330067
576	82	168	22	2021-05-20 14:12:57.34801	2021-05-20 14:12:57.34801
577	83	168	22	2021-05-20 14:12:57.35284	2021-05-20 14:12:57.35284
578	84	168	22	2021-05-20 14:12:57.355048	2021-05-20 14:12:57.355048
579	85	168	22	2021-05-20 14:12:57.357151	2021-05-20 14:12:57.357151
580	86	168	22	2021-05-20 14:12:57.359432	2021-05-20 14:12:57.359432
581	82	169	22	2021-05-20 14:12:57.391984	2021-05-20 14:12:57.391984
582	83	169	22	2021-05-20 14:12:57.402845	2021-05-20 14:12:57.402845
583	84	169	22	2021-05-20 14:12:57.40488	2021-05-20 14:12:57.40488
584	85	169	22	2021-05-20 14:12:57.407444	2021-05-20 14:12:57.407444
585	86	169	22	2021-05-20 14:12:57.410221	2021-05-20 14:12:57.410221
586	82	170	22	2021-05-20 14:12:57.42829	2021-05-20 14:12:57.42829
587	83	170	22	2021-05-20 14:12:57.434407	2021-05-20 14:12:57.434407
588	84	170	22	2021-05-20 14:12:57.436844	2021-05-20 14:12:57.436844
589	85	170	22	2021-05-20 14:12:57.43963	2021-05-20 14:12:57.43963
590	86	170	22	2021-05-20 14:12:57.442231	2021-05-20 14:12:57.442231
591	82	171	22	2021-05-20 14:12:57.459226	2021-05-20 14:12:57.459226
592	83	171	22	2021-05-20 14:12:57.464015	2021-05-20 14:12:57.464015
593	84	171	22	2021-05-20 14:12:57.465976	2021-05-20 14:12:57.465976
594	85	171	22	2021-05-20 14:12:57.467965	2021-05-20 14:12:57.467965
595	86	171	22	2021-05-20 14:12:57.470515	2021-05-20 14:12:57.470515
596	87	174	23	2021-05-20 14:13:34.094623	2021-05-20 14:13:34.094623
597	88	174	23	2021-05-20 14:13:34.097184	2021-05-20 14:13:34.097184
598	89	174	23	2021-05-20 14:13:34.099136	2021-05-20 14:13:34.099136
599	90	174	23	2021-05-20 14:13:34.101189	2021-05-20 14:13:34.101189
600	91	174	23	2021-05-20 14:13:34.103201	2021-05-20 14:13:34.103201
601	87	175	23	2021-05-20 14:13:34.117727	2021-05-20 14:13:34.117727
602	88	175	23	2021-05-20 14:13:34.127922	2021-05-20 14:13:34.127922
603	89	175	23	2021-05-20 14:13:34.13236	2021-05-20 14:13:34.13236
604	90	175	23	2021-05-20 14:13:34.135846	2021-05-20 14:13:34.135846
605	91	175	23	2021-05-20 14:13:34.137939	2021-05-20 14:13:34.137939
606	87	176	23	2021-05-20 14:13:34.155392	2021-05-20 14:13:34.155392
607	88	176	23	2021-05-20 14:13:34.160617	2021-05-20 14:13:34.160617
608	89	176	23	2021-05-20 14:13:34.162551	2021-05-20 14:13:34.162551
609	90	176	23	2021-05-20 14:13:34.164951	2021-05-20 14:13:34.164951
610	91	176	23	2021-05-20 14:13:34.167788	2021-05-20 14:13:34.167788
611	87	177	23	2021-05-20 14:13:34.184177	2021-05-20 14:13:34.184177
612	88	177	23	2021-05-20 14:13:34.188565	2021-05-20 14:13:34.188565
613	89	177	23	2021-05-20 14:13:34.190483	2021-05-20 14:13:34.190483
614	90	177	23	2021-05-20 14:13:34.192422	2021-05-20 14:13:34.192422
615	91	177	23	2021-05-20 14:13:34.194415	2021-05-20 14:13:34.194415
616	87	178	23	2021-05-20 14:13:34.208038	2021-05-20 14:13:34.208038
617	88	178	23	2021-05-20 14:13:34.212183	2021-05-20 14:13:34.212183
618	89	178	23	2021-05-20 14:13:34.214076	2021-05-20 14:13:34.214076
619	90	178	23	2021-05-20 14:13:34.216029	2021-05-20 14:13:34.216029
620	91	178	23	2021-05-20 14:13:34.218372	2021-05-20 14:13:34.218372
621	87	179	23	2021-05-20 14:13:34.232653	2021-05-20 14:13:34.232653
622	88	179	23	2021-05-20 14:13:34.237032	2021-05-20 14:13:34.237032
623	89	179	23	2021-05-20 14:13:34.239229	2021-05-20 14:13:34.239229
624	90	179	23	2021-05-20 14:13:34.241176	2021-05-20 14:13:34.241176
625	91	179	23	2021-05-20 14:13:34.243452	2021-05-20 14:13:34.243452
626	87	180	23	2021-05-20 14:13:34.261474	2021-05-20 14:13:34.261474
627	88	180	23	2021-05-20 14:13:34.265938	2021-05-20 14:13:34.265938
628	89	180	23	2021-05-20 14:13:34.267839	2021-05-20 14:13:34.267839
629	90	180	23	2021-05-20 14:13:34.269754	2021-05-20 14:13:34.269754
630	91	180	23	2021-05-20 14:13:34.272336	2021-05-20 14:13:34.272336
631	92	183	24	2021-05-20 14:21:20.665234	2021-05-20 14:21:20.665234
632	93	183	24	2021-05-20 14:21:20.667551	2021-05-20 14:21:20.667551
633	94	183	24	2021-05-20 14:21:20.669589	2021-05-20 14:21:20.669589
634	95	183	24	2021-05-20 14:21:20.671594	2021-05-20 14:21:20.671594
635	96	183	24	2021-05-20 14:21:20.673615	2021-05-20 14:21:20.673615
636	92	184	24	2021-05-20 14:21:20.688211	2021-05-20 14:21:20.688211
637	93	184	24	2021-05-20 14:21:20.695298	2021-05-20 14:21:20.695298
638	94	184	24	2021-05-20 14:21:20.697292	2021-05-20 14:21:20.697292
639	95	184	24	2021-05-20 14:21:20.699296	2021-05-20 14:21:20.699296
640	96	184	24	2021-05-20 14:21:20.701612	2021-05-20 14:21:20.701612
641	92	185	24	2021-05-20 14:21:20.716973	2021-05-20 14:21:20.716973
642	93	185	24	2021-05-20 14:21:20.721543	2021-05-20 14:21:20.721543
643	94	185	24	2021-05-20 14:21:20.723851	2021-05-20 14:21:20.723851
644	95	185	24	2021-05-20 14:21:20.726492	2021-05-20 14:21:20.726492
645	96	185	24	2021-05-20 14:21:20.728501	2021-05-20 14:21:20.728501
646	92	186	24	2021-05-20 14:21:20.743052	2021-05-20 14:21:20.743052
647	93	186	24	2021-05-20 14:21:20.74815	2021-05-20 14:21:20.74815
648	94	186	24	2021-05-20 14:21:20.750231	2021-05-20 14:21:20.750231
649	95	186	24	2021-05-20 14:21:20.752381	2021-05-20 14:21:20.752381
650	96	186	24	2021-05-20 14:21:20.75459	2021-05-20 14:21:20.75459
651	92	187	24	2021-05-20 14:21:20.770251	2021-05-20 14:21:20.770251
652	93	187	24	2021-05-20 14:21:20.774865	2021-05-20 14:21:20.774865
653	94	187	24	2021-05-20 14:21:20.776803	2021-05-20 14:21:20.776803
654	95	187	24	2021-05-20 14:21:20.778736	2021-05-20 14:21:20.778736
655	96	187	24	2021-05-20 14:21:20.781049	2021-05-20 14:21:20.781049
656	92	188	24	2021-05-20 14:21:20.798202	2021-05-20 14:21:20.798202
657	93	188	24	2021-05-20 14:21:20.803001	2021-05-20 14:21:20.803001
658	94	188	24	2021-05-20 14:21:20.805037	2021-05-20 14:21:20.805037
659	95	188	24	2021-05-20 14:21:20.807321	2021-05-20 14:21:20.807321
660	96	188	24	2021-05-20 14:21:20.8096	2021-05-20 14:21:20.8096
661	92	189	24	2021-05-20 14:21:20.824761	2021-05-20 14:21:20.824761
662	93	189	24	2021-05-20 14:21:20.829564	2021-05-20 14:21:20.829564
663	94	189	24	2021-05-20 14:21:20.831716	2021-05-20 14:21:20.831716
664	95	189	24	2021-05-20 14:21:20.833935	2021-05-20 14:21:20.833935
665	96	189	24	2021-05-20 14:21:20.836295	2021-05-20 14:21:20.836295
666	97	192	25	2021-05-20 14:30:30.416543	2021-05-20 14:30:30.416543
667	98	192	25	2021-05-20 14:30:30.419074	2021-05-20 14:30:30.419074
668	99	192	25	2021-05-20 14:30:30.421185	2021-05-20 14:30:30.421185
669	100	192	25	2021-05-20 14:30:30.423205	2021-05-20 14:30:30.423205
670	101	192	25	2021-05-20 14:30:30.425315	2021-05-20 14:30:30.425315
671	97	193	25	2021-05-20 14:30:30.43917	2021-05-20 14:30:30.43917
672	98	193	25	2021-05-20 14:30:30.443735	2021-05-20 14:30:30.443735
673	99	193	25	2021-05-20 14:30:30.445864	2021-05-20 14:30:30.445864
674	100	193	25	2021-05-20 14:30:30.447857	2021-05-20 14:30:30.447857
675	101	193	25	2021-05-20 14:30:30.449797	2021-05-20 14:30:30.449797
676	97	194	25	2021-05-20 14:30:30.464375	2021-05-20 14:30:30.464375
677	98	194	25	2021-05-20 14:30:30.469768	2021-05-20 14:30:30.469768
678	99	194	25	2021-05-20 14:30:30.471976	2021-05-20 14:30:30.471976
679	100	194	25	2021-05-20 14:30:30.473959	2021-05-20 14:30:30.473959
680	101	194	25	2021-05-20 14:30:30.476006	2021-05-20 14:30:30.476006
681	97	195	25	2021-05-20 14:30:30.49028	2021-05-20 14:30:30.49028
682	98	195	25	2021-05-20 14:30:30.494684	2021-05-20 14:30:30.494684
683	99	195	25	2021-05-20 14:30:30.49676	2021-05-20 14:30:30.49676
684	100	195	25	2021-05-20 14:30:30.498911	2021-05-20 14:30:30.498911
685	101	195	25	2021-05-20 14:30:30.501203	2021-05-20 14:30:30.501203
686	97	196	25	2021-05-20 14:30:30.515848	2021-05-20 14:30:30.515848
687	98	196	25	2021-05-20 14:30:30.520578	2021-05-20 14:30:30.520578
688	99	196	25	2021-05-20 14:30:30.522581	2021-05-20 14:30:30.522581
689	100	196	25	2021-05-20 14:30:30.524517	2021-05-20 14:30:30.524517
690	101	196	25	2021-05-20 14:30:30.526465	2021-05-20 14:30:30.526465
691	97	197	25	2021-05-20 14:30:30.542572	2021-05-20 14:30:30.542572
692	98	197	25	2021-05-20 14:30:30.546934	2021-05-20 14:30:30.546934
693	99	197	25	2021-05-20 14:30:30.549225	2021-05-20 14:30:30.549225
694	100	197	25	2021-05-20 14:30:30.551207	2021-05-20 14:30:30.551207
695	101	197	25	2021-05-20 14:30:30.55326	2021-05-20 14:30:30.55326
696	97	198	25	2021-05-20 14:30:30.568006	2021-05-20 14:30:30.568006
697	98	198	25	2021-05-20 14:30:30.572997	2021-05-20 14:30:30.572997
698	99	198	25	2021-05-20 14:30:30.575	2021-05-20 14:30:30.575
699	100	198	25	2021-05-20 14:30:30.577199	2021-05-20 14:30:30.577199
700	101	198	25	2021-05-20 14:30:30.579206	2021-05-20 14:30:30.579206
701	102	201	26	2021-05-20 15:17:11.829647	2021-05-20 15:17:11.829647
702	103	201	26	2021-05-20 15:17:11.832034	2021-05-20 15:17:11.832034
703	104	201	26	2021-05-20 15:17:11.834575	2021-05-20 15:17:11.834575
704	105	201	26	2021-05-20 15:17:11.836678	2021-05-20 15:17:11.836678
705	106	201	26	2021-05-20 15:17:11.838875	2021-05-20 15:17:11.838875
706	102	202	26	2021-05-20 15:17:11.854434	2021-05-20 15:17:11.854434
707	103	202	26	2021-05-20 15:17:11.859638	2021-05-20 15:17:11.859638
708	104	202	26	2021-05-20 15:17:11.86277	2021-05-20 15:17:11.86277
709	105	202	26	2021-05-20 15:17:11.864825	2021-05-20 15:17:11.864825
710	106	202	26	2021-05-20 15:17:11.86686	2021-05-20 15:17:11.86686
711	102	203	26	2021-05-20 15:17:11.881046	2021-05-20 15:17:11.881046
712	103	203	26	2021-05-20 15:17:11.885349	2021-05-20 15:17:11.885349
713	104	203	26	2021-05-20 15:17:11.887263	2021-05-20 15:17:11.887263
714	105	203	26	2021-05-20 15:17:11.889157	2021-05-20 15:17:11.889157
715	106	203	26	2021-05-20 15:17:11.891377	2021-05-20 15:17:11.891377
716	102	204	26	2021-05-20 15:17:11.906207	2021-05-20 15:17:11.906207
717	103	204	26	2021-05-20 15:17:11.911894	2021-05-20 15:17:11.911894
718	104	204	26	2021-05-20 15:17:11.913948	2021-05-20 15:17:11.913948
719	105	204	26	2021-05-20 15:17:11.915956	2021-05-20 15:17:11.915956
720	106	204	26	2021-05-20 15:17:11.917906	2021-05-20 15:17:11.917906
721	102	205	26	2021-05-20 15:17:11.934832	2021-05-20 15:17:11.934832
722	103	205	26	2021-05-20 15:17:11.940028	2021-05-20 15:17:11.940028
723	104	205	26	2021-05-20 15:17:11.942062	2021-05-20 15:17:11.942062
724	105	205	26	2021-05-20 15:17:11.944017	2021-05-20 15:17:11.944017
725	106	205	26	2021-05-20 15:17:11.946208	2021-05-20 15:17:11.946208
726	102	206	26	2021-05-20 15:17:11.960895	2021-05-20 15:17:11.960895
727	103	206	26	2021-05-20 15:17:11.966085	2021-05-20 15:17:11.966085
728	104	206	26	2021-05-20 15:17:11.968092	2021-05-20 15:17:11.968092
729	105	206	26	2021-05-20 15:17:11.970013	2021-05-20 15:17:11.970013
730	106	206	26	2021-05-20 15:17:11.972545	2021-05-20 15:17:11.972545
731	102	207	26	2021-05-20 15:17:11.988661	2021-05-20 15:17:11.988661
732	103	207	26	2021-05-20 15:17:11.993062	2021-05-20 15:17:11.993062
733	104	207	26	2021-05-20 15:17:11.995035	2021-05-20 15:17:11.995035
734	105	207	26	2021-05-20 15:17:11.99698	2021-05-20 15:17:11.99698
735	106	207	26	2021-05-20 15:17:11.998932	2021-05-20 15:17:11.998932
736	107	213	27	2021-05-28 15:07:45.868036	2021-05-28 15:07:45.868036
737	108	213	27	2021-05-28 15:07:45.886801	2021-05-28 15:07:45.886801
738	109	213	27	2021-05-28 15:07:45.904024	2021-05-28 15:07:45.904024
739	110	213	27	2021-05-28 15:07:45.909649	2021-05-28 15:07:45.909649
740	111	213	27	2021-05-28 15:07:45.92673	2021-05-28 15:07:45.92673
741	107	214	27	2021-05-28 15:07:46.035383	2021-05-28 15:07:46.035383
742	108	214	27	2021-05-28 15:07:46.190965	2021-05-28 15:07:46.190965
743	109	214	27	2021-05-28 15:07:46.230424	2021-05-28 15:07:46.230424
744	110	214	27	2021-05-28 15:07:46.236768	2021-05-28 15:07:46.236768
745	111	214	27	2021-05-28 15:07:46.24669	2021-05-28 15:07:46.24669
746	107	215	27	2021-05-28 15:07:46.371759	2021-05-28 15:07:46.371759
747	108	215	27	2021-05-28 15:07:46.423073	2021-05-28 15:07:46.423073
748	109	215	27	2021-05-28 15:07:46.452337	2021-05-28 15:07:46.452337
749	110	215	27	2021-05-28 15:07:46.481915	2021-05-28 15:07:46.481915
750	111	215	27	2021-05-28 15:07:46.509138	2021-05-28 15:07:46.509138
751	107	216	27	2021-05-28 15:07:46.740878	2021-05-28 15:07:46.740878
752	108	216	27	2021-05-28 15:07:46.831061	2021-05-28 15:07:46.831061
753	109	216	27	2021-05-28 15:07:46.870844	2021-05-28 15:07:46.870844
754	110	216	27	2021-05-28 15:07:46.932158	2021-05-28 15:07:46.932158
755	111	216	27	2021-05-28 15:07:46.957995	2021-05-28 15:07:46.957995
756	107	217	27	2021-05-28 15:07:47.123468	2021-05-28 15:07:47.123468
757	108	217	27	2021-05-28 15:07:50.312751	2021-05-28 15:07:50.312751
758	109	217	27	2021-05-28 15:07:50.955736	2021-05-28 15:07:50.955736
759	110	217	27	2021-05-28 15:07:50.989937	2021-05-28 15:07:50.989937
760	111	217	27	2021-05-28 15:07:51.41081	2021-05-28 15:07:51.41081
761	107	218	27	2021-05-28 15:07:52.218418	2021-05-28 15:07:52.218418
762	108	218	27	2021-05-28 15:07:52.292776	2021-05-28 15:07:52.292776
763	109	218	27	2021-05-28 15:07:52.324266	2021-05-28 15:07:52.324266
764	110	218	27	2021-05-28 15:07:52.330056	2021-05-28 15:07:52.330056
765	111	218	27	2021-05-28 15:07:52.339768	2021-05-28 15:07:52.339768
766	107	219	27	2021-05-28 15:07:52.40429	2021-05-28 15:07:52.40429
767	108	219	27	2021-05-28 15:07:52.435584	2021-05-28 15:07:52.435584
768	109	219	27	2021-05-28 15:07:52.451425	2021-05-28 15:07:52.451425
769	110	219	27	2021-05-28 15:07:52.467607	2021-05-28 15:07:52.467607
770	111	219	27	2021-05-28 15:07:52.477331	2021-05-28 15:07:52.477331
771	112	222	28	2021-06-02 16:10:02.72043	2021-06-02 16:10:02.72043
772	113	222	28	2021-06-02 16:10:02.737444	2021-06-02 16:10:02.737444
773	114	222	28	2021-06-02 16:10:02.745478	2021-06-02 16:10:02.745478
774	115	222	28	2021-06-02 16:10:02.750584	2021-06-02 16:10:02.750584
775	116	222	28	2021-06-02 16:10:02.758849	2021-06-02 16:10:02.758849
776	112	223	28	2021-06-02 16:10:02.819502	2021-06-02 16:10:02.819502
777	113	223	28	2021-06-02 16:10:02.839263	2021-06-02 16:10:02.839263
778	114	223	28	2021-06-02 16:10:02.845102	2021-06-02 16:10:02.845102
779	115	223	28	2021-06-02 16:10:02.850895	2021-06-02 16:10:02.850895
780	116	223	28	2021-06-02 16:10:02.858213	2021-06-02 16:10:02.858213
781	112	224	28	2021-06-02 16:10:02.901402	2021-06-02 16:10:02.901402
782	113	224	28	2021-06-02 16:10:02.920593	2021-06-02 16:10:02.920593
783	114	224	28	2021-06-02 16:10:02.925378	2021-06-02 16:10:02.925378
784	115	224	28	2021-06-02 16:10:02.932032	2021-06-02 16:10:02.932032
785	116	224	28	2021-06-02 16:10:02.940925	2021-06-02 16:10:02.940925
786	112	225	28	2021-06-02 16:10:02.986523	2021-06-02 16:10:02.986523
787	113	225	28	2021-06-02 16:10:02.998685	2021-06-02 16:10:02.998685
788	114	225	28	2021-06-02 16:10:03.014839	2021-06-02 16:10:03.014839
789	115	225	28	2021-06-02 16:10:03.019099	2021-06-02 16:10:03.019099
790	116	225	28	2021-06-02 16:10:03.023323	2021-06-02 16:10:03.023323
791	112	226	28	2021-06-02 16:10:03.055322	2021-06-02 16:10:03.055322
792	113	226	28	2021-06-02 16:10:03.0673	2021-06-02 16:10:03.0673
793	114	226	28	2021-06-02 16:10:03.072006	2021-06-02 16:10:03.072006
794	115	226	28	2021-06-02 16:10:03.075801	2021-06-02 16:10:03.075801
795	116	226	28	2021-06-02 16:10:03.079197	2021-06-02 16:10:03.079197
796	112	227	28	2021-06-02 16:10:03.117269	2021-06-02 16:10:03.117269
797	113	227	28	2021-06-02 16:10:03.12945	2021-06-02 16:10:03.12945
798	114	227	28	2021-06-02 16:10:03.135129	2021-06-02 16:10:03.135129
799	115	227	28	2021-06-02 16:10:03.142552	2021-06-02 16:10:03.142552
800	116	227	28	2021-06-02 16:10:03.148078	2021-06-02 16:10:03.148078
801	112	228	28	2021-06-02 16:10:03.18999	2021-06-02 16:10:03.18999
802	113	228	28	2021-06-02 16:10:03.202075	2021-06-02 16:10:03.202075
803	114	228	28	2021-06-02 16:10:03.20785	2021-06-02 16:10:03.20785
804	115	228	28	2021-06-02 16:10:03.21322	2021-06-02 16:10:03.21322
805	116	228	28	2021-06-02 16:10:03.223157	2021-06-02 16:10:03.223157
806	117	242	30	2021-08-23 14:58:27.937246	2021-08-23 14:58:27.937246
807	118	242	30	2021-08-23 14:58:27.944954	2021-08-23 14:58:27.944954
808	119	242	30	2021-08-23 14:58:27.947895	2021-08-23 14:58:27.947895
809	120	242	30	2021-08-23 14:58:27.950934	2021-08-23 14:58:27.950934
810	121	242	30	2021-08-23 14:58:27.953953	2021-08-23 14:58:27.953953
811	117	243	30	2021-08-23 14:58:27.97835	2021-08-23 14:58:27.97835
812	118	243	30	2021-08-23 14:58:27.988016	2021-08-23 14:58:27.988016
813	119	243	30	2021-08-23 14:58:27.991197	2021-08-23 14:58:27.991197
814	120	243	30	2021-08-23 14:58:27.993535	2021-08-23 14:58:27.993535
815	121	243	30	2021-08-23 14:58:27.998394	2021-08-23 14:58:27.998394
816	117	244	30	2021-08-23 14:58:28.019688	2021-08-23 14:58:28.019688
817	118	244	30	2021-08-23 14:58:28.026088	2021-08-23 14:58:28.026088
818	119	244	30	2021-08-23 14:58:28.029421	2021-08-23 14:58:28.029421
819	120	244	30	2021-08-23 14:58:28.03255	2021-08-23 14:58:28.03255
820	121	244	30	2021-08-23 14:58:28.035014	2021-08-23 14:58:28.035014
821	117	245	30	2021-08-23 14:58:28.056686	2021-08-23 14:58:28.056686
822	118	245	30	2021-08-23 14:58:28.06343	2021-08-23 14:58:28.06343
823	119	245	30	2021-08-23 14:58:28.066893	2021-08-23 14:58:28.066893
824	120	245	30	2021-08-23 14:58:28.069878	2021-08-23 14:58:28.069878
825	121	245	30	2021-08-23 14:58:28.072241	2021-08-23 14:58:28.072241
826	117	246	30	2021-08-23 14:58:28.095031	2021-08-23 14:58:28.095031
827	118	246	30	2021-08-23 14:58:28.12205	2021-08-23 14:58:28.12205
828	119	246	30	2021-08-23 14:58:28.124648	2021-08-23 14:58:28.124648
829	120	246	30	2021-08-23 14:58:28.127158	2021-08-23 14:58:28.127158
830	121	246	30	2021-08-23 14:58:28.129871	2021-08-23 14:58:28.129871
831	117	247	30	2021-08-23 14:58:28.143526	2021-08-23 14:58:28.143526
832	118	247	30	2021-08-23 14:58:28.147814	2021-08-23 14:58:28.147814
833	119	247	30	2021-08-23 14:58:28.149743	2021-08-23 14:58:28.149743
834	120	247	30	2021-08-23 14:58:28.151661	2021-08-23 14:58:28.151661
835	121	247	30	2021-08-23 14:58:28.153645	2021-08-23 14:58:28.153645
836	117	248	30	2021-08-23 14:58:28.167922	2021-08-23 14:58:28.167922
837	118	248	30	2021-08-23 14:58:28.172308	2021-08-23 14:58:28.172308
838	119	248	30	2021-08-23 14:58:28.174255	2021-08-23 14:58:28.174255
839	120	248	30	2021-08-23 14:58:28.176155	2021-08-23 14:58:28.176155
840	121	248	30	2021-08-23 14:58:28.178514	2021-08-23 14:58:28.178514
841	122	251	31	2021-08-23 17:19:39.317097	2021-08-23 17:19:39.317097
842	123	251	31	2021-08-23 17:19:39.320094	2021-08-23 17:19:39.320094
843	124	251	31	2021-08-23 17:19:39.32219	2021-08-23 17:19:39.32219
844	125	251	31	2021-08-23 17:19:39.324431	2021-08-23 17:19:39.324431
845	126	251	31	2021-08-23 17:19:39.326938	2021-08-23 17:19:39.326938
846	122	252	31	2021-08-23 17:19:39.343605	2021-08-23 17:19:39.343605
847	123	252	31	2021-08-23 17:19:39.349618	2021-08-23 17:19:39.349618
848	124	252	31	2021-08-23 17:19:39.352076	2021-08-23 17:19:39.352076
849	125	252	31	2021-08-23 17:19:39.354376	2021-08-23 17:19:39.354376
850	126	252	31	2021-08-23 17:19:39.356529	2021-08-23 17:19:39.356529
851	122	253	31	2021-08-23 17:19:39.373261	2021-08-23 17:19:39.373261
852	123	253	31	2021-08-23 17:19:39.379385	2021-08-23 17:19:39.379385
853	124	253	31	2021-08-23 17:19:39.381587	2021-08-23 17:19:39.381587
854	125	253	31	2021-08-23 17:19:39.383681	2021-08-23 17:19:39.383681
855	126	253	31	2021-08-23 17:19:39.38579	2021-08-23 17:19:39.38579
856	122	254	31	2021-08-23 17:19:39.399304	2021-08-23 17:19:39.399304
857	123	254	31	2021-08-23 17:19:39.403975	2021-08-23 17:19:39.403975
858	124	254	31	2021-08-23 17:19:39.406533	2021-08-23 17:19:39.406533
859	125	254	31	2021-08-23 17:19:39.408789	2021-08-23 17:19:39.408789
860	126	254	31	2021-08-23 17:19:39.411578	2021-08-23 17:19:39.411578
861	122	255	31	2021-08-23 17:19:39.428149	2021-08-23 17:19:39.428149
862	123	255	31	2021-08-23 17:19:39.433315	2021-08-23 17:19:39.433315
863	124	255	31	2021-08-23 17:19:39.435756	2021-08-23 17:19:39.435756
864	125	255	31	2021-08-23 17:19:39.437857	2021-08-23 17:19:39.437857
865	126	255	31	2021-08-23 17:19:39.440144	2021-08-23 17:19:39.440144
866	122	256	31	2021-08-23 17:19:39.457503	2021-08-23 17:19:39.457503
867	123	256	31	2021-08-23 17:19:39.462522	2021-08-23 17:19:39.462522
868	124	256	31	2021-08-23 17:19:39.465064	2021-08-23 17:19:39.465064
869	125	256	31	2021-08-23 17:19:39.468558	2021-08-23 17:19:39.468558
870	126	256	31	2021-08-23 17:19:39.470787	2021-08-23 17:19:39.470787
871	122	257	31	2021-08-23 17:19:39.487181	2021-08-23 17:19:39.487181
872	123	257	31	2021-08-23 17:19:39.492082	2021-08-23 17:19:39.492082
873	124	257	31	2021-08-23 17:19:39.49461	2021-08-23 17:19:39.49461
874	125	257	31	2021-08-23 17:19:39.497073	2021-08-23 17:19:39.497073
875	126	257	31	2021-08-23 17:19:39.499236	2021-08-23 17:19:39.499236
\.


-- --
-- -- Data for Name: my_module_status_consequences; Type: TABLE DATA; Schema: public; Owner: postgres
-- --

-- COPY public.my_module_status_consequences (id, my_module_status_id, type, created_at, updated_at) FROM stdin;
-- 1	2	MyModuleStatusConsequences::Uncompletion	2021-09-30 14:06:04.394026	2021-09-30 14:06:04.394026
-- 2	3	MyModuleStatusConsequences::Completion	2021-09-30 14:06:04.401779	2021-09-30 14:06:04.401779
-- \.


-- --
-- -- Data for Name: my_module_status_flows; Type: TABLE DATA; Schema: public; Owner: postgres
-- --

-- COPY public.my_module_status_flows (id, name, description, visibility, team_id, created_by_id, last_modified_by_id, created_at, updated_at) FROM stdin;
-- 1	SciNote Free default task flow	\N	0	\N	\N	\N	2021-09-30 14:06:04.358292	2021-09-30 14:06:04.358292
-- \.


--
-- Data for Name: my_module_tags; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.my_module_tags (id, my_module_id, tag_id, created_by_id) FROM stdin;
1	1	1	\N
2	2	2	\N
3	2	3	\N
4	2	4	\N
5	3	3	\N
6	3	5	\N
7	4	2	\N
8	5	2	\N
9	5	6	\N
10	6	2	\N
11	6	6	\N
12	6	4	\N
13	6	7	\N
14	6	8	\N
15	7	1	\N
16	7	9	\N
17	8	1	\N
18	8	9	\N
19	10	10	\N
20	11	11	\N
21	11	12	\N
22	11	13	\N
23	12	12	\N
24	12	14	\N
25	13	11	\N
26	14	11	\N
27	14	15	\N
28	15	11	\N
29	15	15	\N
30	15	13	\N
31	15	16	\N
32	15	17	\N
33	16	10	\N
34	16	18	\N
35	17	10	\N
36	17	18	\N
37	19	19	\N
38	20	20	\N
39	20	21	\N
40	20	22	\N
41	21	21	\N
42	21	23	\N
43	22	20	\N
44	23	20	\N
45	23	24	\N
46	24	20	\N
47	24	24	\N
48	24	22	\N
49	24	25	\N
50	24	26	\N
51	25	19	\N
52	25	27	\N
53	26	19	\N
54	26	27	\N
55	28	28	\N
56	29	29	\N
57	29	30	\N
58	29	31	\N
59	30	30	\N
60	30	32	\N
61	31	29	\N
62	32	29	\N
63	32	33	\N
64	33	29	\N
65	33	33	\N
66	33	31	\N
67	33	34	\N
68	33	35	\N
69	34	28	\N
70	34	36	\N
71	35	28	\N
72	35	36	\N
73	37	37	\N
74	38	38	\N
75	38	39	\N
76	38	40	\N
77	39	39	\N
78	39	41	\N
79	40	38	\N
80	41	38	\N
81	41	42	\N
82	42	38	\N
83	42	42	\N
84	42	40	\N
85	42	43	\N
86	42	44	\N
87	43	37	\N
88	43	45	\N
89	44	37	\N
90	44	45	\N
91	46	46	\N
92	47	47	\N
93	47	48	\N
94	47	49	\N
95	48	48	\N
96	48	50	\N
97	49	47	\N
98	50	47	\N
99	50	51	\N
100	51	47	\N
101	51	51	\N
102	51	49	\N
103	51	52	\N
104	51	53	\N
105	52	46	\N
106	52	54	\N
107	53	46	\N
108	53	54	\N
109	55	55	\N
110	56	56	\N
111	56	57	\N
112	56	58	\N
113	57	57	\N
114	57	59	\N
115	58	56	\N
116	59	56	\N
117	59	60	\N
118	60	56	\N
119	60	60	\N
120	60	58	\N
121	60	61	\N
122	60	62	\N
123	61	55	\N
124	61	63	\N
125	62	55	\N
126	62	63	\N
127	64	64	\N
128	65	65	\N
129	65	66	\N
130	65	67	\N
131	66	66	\N
132	66	68	\N
133	67	65	\N
134	68	65	\N
135	68	69	\N
136	69	65	\N
137	69	69	\N
138	69	67	\N
139	69	70	\N
140	69	71	\N
141	70	64	\N
142	70	72	\N
143	71	64	\N
144	71	72	\N
150	78	75	\N
151	79	76	\N
152	79	77	\N
153	79	78	\N
154	80	77	\N
155	80	79	\N
156	81	76	\N
157	82	76	\N
158	82	80	\N
159	83	76	\N
160	83	80	\N
161	83	78	\N
162	83	81	\N
163	83	82	\N
164	84	75	\N
165	84	83	\N
166	85	75	\N
167	85	83	\N
168	87	84	\N
169	88	85	\N
170	88	86	\N
171	88	87	\N
172	89	86	\N
173	89	88	\N
174	90	85	\N
175	91	85	\N
176	91	89	\N
177	92	85	\N
178	92	89	\N
179	92	87	\N
180	92	90	\N
181	92	91	\N
182	93	84	\N
183	93	92	\N
184	94	84	\N
185	94	92	\N
186	96	93	\N
187	97	94	\N
188	97	95	\N
189	97	96	\N
190	98	95	\N
191	98	97	\N
192	99	94	\N
193	100	94	\N
194	100	98	\N
195	101	94	\N
196	101	98	\N
197	101	96	\N
198	101	99	\N
199	101	100	\N
200	102	93	\N
201	102	101	\N
202	103	93	\N
203	103	101	\N
204	106	102	\N
205	107	103	\N
206	107	104	\N
207	107	105	\N
208	108	104	\N
209	108	106	\N
210	109	103	\N
211	110	103	\N
212	110	107	\N
213	111	103	\N
214	111	107	\N
215	111	105	\N
216	111	108	\N
217	111	109	\N
218	112	102	\N
219	112	110	\N
220	113	102	\N
221	113	110	\N
222	115	111	\N
223	116	112	\N
224	116	113	\N
225	116	114	\N
226	117	113	\N
227	117	115	\N
228	118	112	\N
229	119	112	\N
230	119	116	\N
231	120	112	\N
232	120	116	\N
233	120	114	\N
234	120	117	\N
235	120	118	\N
236	121	111	\N
237	121	119	\N
238	122	111	\N
239	122	119	\N
240	124	120	\N
241	125	121	\N
242	125	122	\N
243	125	123	\N
244	126	122	\N
245	126	124	\N
246	127	121	\N
247	128	121	\N
248	128	125	\N
249	129	121	\N
250	129	125	\N
251	129	123	\N
252	129	126	\N
253	129	127	\N
254	130	120	\N
255	130	128	\N
256	131	120	\N
257	131	128	\N
258	133	73	4
259	137	129	\N
260	138	130	\N
261	138	131	\N
262	138	132	\N
263	139	131	\N
264	139	133	\N
265	140	130	\N
266	141	130	\N
267	141	134	\N
268	142	130	\N
269	142	134	\N
270	142	132	\N
271	142	135	\N
272	142	136	\N
273	143	129	\N
274	143	137	\N
275	144	129	\N
276	144	137	\N
277	155	138	\N
278	156	139	\N
279	156	140	\N
280	156	141	\N
281	157	140	\N
282	157	142	\N
283	158	139	\N
284	159	139	\N
285	159	143	\N
286	160	139	\N
287	160	143	\N
288	160	141	\N
289	160	144	\N
290	160	145	\N
291	161	138	\N
292	161	146	\N
293	162	138	\N
294	162	146	\N
295	164	147	\N
296	165	148	\N
297	165	149	\N
298	165	150	\N
299	166	149	\N
300	166	151	\N
301	167	148	\N
302	168	148	\N
303	168	152	\N
304	169	148	\N
305	169	152	\N
306	169	150	\N
307	169	153	\N
308	169	154	\N
309	170	147	\N
310	170	155	\N
311	171	147	\N
312	171	155	\N
313	173	156	\N
314	174	157	\N
315	174	158	\N
316	174	159	\N
317	175	158	\N
318	175	160	\N
319	176	157	\N
320	177	157	\N
321	177	161	\N
322	178	157	\N
323	178	161	\N
324	178	159	\N
325	178	162	\N
326	178	163	\N
327	179	156	\N
328	179	164	\N
329	180	156	\N
330	180	164	\N
331	182	165	\N
332	183	166	\N
333	183	167	\N
334	183	168	\N
335	184	167	\N
336	184	169	\N
337	185	166	\N
338	186	166	\N
339	186	170	\N
340	187	166	\N
341	187	170	\N
342	187	168	\N
343	187	171	\N
344	187	172	\N
345	188	165	\N
346	188	173	\N
347	189	165	\N
348	189	173	\N
349	191	174	\N
350	192	175	\N
351	192	176	\N
352	192	177	\N
353	193	176	\N
354	193	178	\N
355	194	175	\N
356	195	175	\N
357	195	179	\N
358	196	175	\N
359	196	179	\N
360	196	177	\N
361	196	180	\N
362	196	181	\N
363	197	174	\N
364	197	182	\N
365	198	174	\N
366	198	182	\N
367	200	183	\N
368	201	184	\N
369	201	185	\N
370	201	186	\N
371	202	185	\N
372	202	187	\N
373	203	184	\N
374	204	184	\N
375	204	188	\N
376	205	184	\N
377	205	188	\N
378	205	186	\N
379	205	189	\N
380	205	190	\N
381	206	183	\N
382	206	191	\N
383	207	183	\N
384	207	191	\N
385	212	192	\N
386	213	193	\N
387	213	194	\N
388	213	195	\N
389	214	194	\N
390	214	196	\N
391	215	193	\N
392	216	193	\N
393	216	197	\N
394	217	193	\N
395	217	197	\N
396	217	195	\N
397	217	198	\N
398	217	199	\N
399	218	192	\N
400	218	200	\N
401	219	192	\N
402	219	200	\N
403	221	201	\N
404	222	202	\N
405	222	203	\N
406	222	204	\N
407	223	203	\N
408	223	205	\N
409	224	202	\N
410	225	202	\N
411	225	206	\N
412	226	202	\N
413	226	206	\N
414	226	204	\N
415	226	207	\N
416	226	208	\N
417	227	201	\N
418	227	209	\N
419	228	201	\N
420	228	209	\N
421	241	210	\N
422	242	211	\N
423	242	212	\N
424	242	213	\N
425	243	212	\N
426	243	214	\N
427	244	211	\N
428	245	211	\N
429	245	215	\N
430	246	211	\N
431	246	215	\N
432	246	213	\N
433	246	216	\N
434	246	217	\N
435	247	210	\N
436	247	218	\N
437	248	210	\N
438	248	218	\N
439	250	219	\N
440	251	220	\N
441	251	221	\N
442	251	222	\N
443	252	221	\N
444	252	223	\N
445	253	220	\N
446	254	220	\N
447	254	224	\N
448	255	220	\N
449	255	224	\N
450	255	222	\N
451	255	225	\N
452	255	226	\N
453	256	219	\N
454	256	227	\N
455	257	219	\N
456	257	227	\N
\.


--
-- Data for Name: notifications; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.notifications (id, title, message, type_of, generator_user_id, created_at, updated_at) FROM stdin;
1	<i>gonzalez.michael@epa.gov</i> was added as Normal user to team <strong>ECEB SW</strong> by <i>Admin</i>.	Team:  ECEB SW	0	\N	2019-07-22 11:21:59.603003	2019-07-22 11:21:59.603003
2	<i>venkatapathy.raghuraman@epa.gov</i> was added as Normal user to team <strong>ECEB SW</strong> by <i>Admin</i>.	Team:  ECEB SW	0	\N	2019-07-22 11:21:59.894458	2019-07-22 11:21:59.894458
3	<i>baker.steven@epa.gov</i> was added as Administrator to team <strong>R7-Molecular Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Molecular Lab	0	\N	2019-08-27 15:17:09.303568	2019-08-27 15:17:09.303568
4	<i>venkatapathy.raghuraman@epa.gov</i> was added as Administrator to team <strong>RTEB</strong> by <i>Raghuraman Venkatapathy</i>.	Team:  RTEB	0	\N	2019-08-28 08:36:02.451091	2019-08-28 08:36:02.451091
5	<i>Raghuraman Venkatapathy</i> was added as Administrator to team <strong>ECEB SW</strong> by <i>Admin</i>.	Team:  ECEB SW	0	\N	2019-08-28 09:15:40.74171	2019-08-28 09:15:40.74171
6	<i>Steven Baker</i> was added as Administrator to team <strong>R7-Molecular Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Molecular Lab	0	\N	2019-09-12 15:39:47.483171	2019-09-12 15:39:47.483171
7	<i>Michael F Davis</i> was added as Guest to team <strong>R7-Molecular Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Molecular Lab	0	\N	2019-10-21 16:32:02.964383	2019-10-21 16:32:02.964383
8	<i>Regina Klepikow</i> has completed the task <strong>Prep Autoclave</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/25">Micro QC</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/23/canvas">Autoclave Sterilization Check</a> | Task: <a href="https://ordscinote.epa.gov/modules/74/protocols">Prep Autoclave</a>	1	4	2019-11-25 17:07:20.421542	2019-11-25 17:07:20.421542
9	<i>Laura Webb</i> was added as Normal user to team <strong>R7-Microbiology Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Microbiology Lab	0	\N	2019-11-25 18:03:25.14189	2019-11-25 18:03:25.14189
10	<i>Steven Baker</i> was added as Normal user to team <strong>R7-Microbiology Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Microbiology Lab	0	\N	2020-06-12 11:55:49.183346	2020-06-12 11:55:49.183346
11	<i>gonzalez.michael@epa.gov</i> was added as Normal user to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2020-09-28 12:22:40.256165	2020-09-28 12:22:40.256165
12	<i>venkatapathy.raghuraman@epa.gov</i> was added as Normal user to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2020-09-28 12:22:40.356806	2020-09-28 12:22:40.356806
13	<i>foster.albert@epa.gov</i> was added as Normal user to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2020-09-28 12:22:40.749675	2020-09-28 12:22:40.749675
14	<i>allen.derrick01@epa.gov</i> was added as Normal user to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2020-09-28 12:22:40.915794	2020-09-28 12:22:40.915794
15	<i>gonzalez.michael@epa.gov</i> was added as Normal user to team <strong>My projects</strong> by <i>Admin</i>.	Team:  My projects	0	\N	2020-09-28 13:00:11.610366	2020-09-28 13:00:11.610366
16	<i>allen01.derrick@epa.gov</i> was added as Normal user to team <strong>My projects</strong> by <i>Admin</i>.	Team:  My projects	0	\N	2020-09-28 13:00:12.022591	2020-09-28 13:00:12.022591
17	<i>foster.albert@epa.gov</i> was added as Normal user to team <strong>My projects</strong> by <i>Admin</i>.	Team:  My projects	0	\N	2020-09-28 13:00:12.051521	2020-09-28 13:00:12.051521
18	<i>shan.yonggui@epa.gov</i> was added as Normal user to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2021-04-08 10:19:49.297972	2021-04-08 10:19:49.297972
19	<i>meyer.david@epa.gov</i> was added as Normal user to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2021-04-08 10:20:34.225105	2021-04-08 10:20:34.225105
20	Regina Klepikow mentioned you in a comment on task 2021 April - Quarterly Timing Check.	Project: <a href="https://ordscinote.epa.gov/projects/25">Micro QC</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/24/canvas">Autoclave Timing Check</a> | Task: <a href="https://ordscinote.epa.gov/modules/133/protocols">2021 April - Quarterly Timing Check</a>	0	\N	2021-04-08 13:28:21.974924	2021-04-08 13:28:21.974924
21	<i>Regina Klepikow</i> has completed the task <strong>2021 July - Quarterly Timing Check</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/25">Micro QC</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/24/canvas">Autoclave Timing Check</a> | Task: <a href="https://ordscinote.epa.gov/modules/134/protocols">2021 July - Quarterly Timing Check</a>	1	4	2021-04-08 14:19:03.583156	2021-04-08 14:19:03.583156
22	<i>Regina Klepikow</i> has completed the task <strong>Quarterly Timing Check</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/25">Micro QC</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/24/canvas">Autoclave Timing Check</a> | Task: <a href="https://ordscinote.epa.gov/modules/133/protocols">Quarterly Timing Check</a>	1	4	2021-04-08 14:21:00.030934	2021-04-08 14:21:00.030934
23	<i>Steven Baker</i> was removed from team <strong>R7-Microbiology Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Microbiology Lab	0	\N	2021-04-08 15:53:54.284188	2021-04-08 15:53:54.284188
24	<i>Steven Baker</i> was removed from team <strong>R7-Molecular Lab</strong> by <i>Regina Klepikow</i>.	Team:  R7-Molecular Lab	0	\N	2021-04-08 15:54:38.85064	2021-04-08 15:54:38.85064
25	<i>nadagouda.mallikarjuna@epa.gov</i> was added as Normal user to team <strong>TLee ' 's lab</strong> by <i>Tae Lee</i>.	Team:  TLee ' 's lab	0	\N	2021-06-10 16:09:28.270257	2021-06-10 16:09:28.270257
26	<i>Tae Lee</i> has completed the task <strong>Receive new samples (06/10/2021)</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/50/canvas">20210528 PO4 adsorption test (two samples)</a> | Task: <a href="https://ordscinote.epa.gov/modules/230/protocols">Receive new samples (06/10/2021)</a>	1	27	2021-06-10 18:45:11.139431	2021-06-10 18:45:11.139431
27	<i>Tae Lee</i> has completed the task <strong>Bake each sample at 250 C</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/50/canvas">20210610 PO4 adsorption test (two samples)</a> | Task: <a href="https://ordscinote.epa.gov/modules/231/protocols">Bake each sample at 250 C</a>	1	27	2021-06-14 19:22:13.948107	2021-06-14 19:22:13.948107
28	<i>Tae Lee</i> has completed the task <strong>Collect samples at t = 1,2,3,24 h</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/50/canvas">20210610 PO4 adsorption test (two samples)</a> | Task: <a href="https://ordscinote.epa.gov/modules/233/protocols">Collect samples at t = 1,2,3,24 h</a>	1	27	2021-06-16 15:58:53.66334	2021-06-16 15:58:53.66334
29	<i>Tae Lee</i> has completed the task <strong>Run adsorption test</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/50/canvas">20210610 PO4 adsorption test (two samples)</a> | Task: <a href="https://ordscinote.epa.gov/modules/232/protocols">Run adsorption test</a>	1	27	2021-06-16 15:59:07.735434	2021-06-16 15:59:07.735434
30	<i>Tae Lee</i> has completed the task <strong>Analyze samples using Spectraphotometer</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/50/canvas">20210610 PO4 adsorption test (two samples)</a> | Task: <a href="https://ordscinote.epa.gov/modules/234/protocols">Analyze samples using Spectraphotometer</a>	1	27	2021-06-16 15:59:27.133259	2021-06-16 15:59:27.133259
31	<i>Tae Lee</i> has completed the task <strong>Receive the new sample</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/52/canvas">20210706 Alumina adsorbent mix for P adsorption</a> | Task: <a href="https://ordscinote.epa.gov/modules/235/protocols">Receive the new sample</a>	1	27	2021-07-07 16:07:45.74873	2021-07-07 16:07:45.74873
32	<i>Tae Lee</i> has completed the task <strong>Mix with 5% biochar</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/52/canvas">20210706 Alumina adsorbent mix for P adsorption</a> | Task: <a href="https://ordscinote.epa.gov/modules/236/protocols">Mix with 5% biochar</a>	1	27	2021-07-07 16:08:20.493608	2021-07-07 16:08:20.493608
33	<i>Tae Lee</i> has completed the task <strong>Adsorption test</strong>. You can now start working on the next task in the workflow.	Project: <a href="https://ordscinote.epa.gov/projects/48">P Recovery</a> | Experiment: <a href="https://ordscinote.epa.gov/experiments/52/canvas">20210706 Alumina adsorbent mix for P adsorption</a> | Task: <a href="https://ordscinote.epa.gov/modules/237/protocols">Adsorption test</a>	1	27	2021-07-07 16:08:45.109014	2021-07-07 16:08:45.109014
34	<i>Aaron Tam</i> was added as Normal user to team <strong>FTOH ELN001</strong> by <i>JAMES VOIT</i>.	Team:  FTOH ELN001	0	\N	2021-08-25 13:47:15.131034	2021-08-25 13:47:15.131034
35	<i>JAMES VOIT</i> was added as Normal user to team <strong>FTOH ELN001 (TEST)</strong> by <i>Aaron Tam</i>.	Team:  FTOH ELN001 (TEST)	0	\N	2021-08-25 14:57:58.961719	2021-08-25 14:57:58.961719
36	<i>Nicholas Sylvest</i> was added as Administrator to team <strong>My projects</strong> by <i>Admin</i>.	Team:  My projects	0	\N	2021-11-16 15:17:40.516936	2021-11-16 15:17:40.516936
37	<i>Jacob Specht</i> was added as Normal user to team <strong>My projects</strong> by <i>Admin</i>.	Team:  My projects	0	\N	2022-02-09 13:05:39.480332	2022-02-09 13:05:39.480332
38	<i>Nicholas Sylvest</i> was added as Administrator to team <strong>Evaluating de-packaging technologies to remove film plastics from food</strong> by <i>Admin</i>.	Team:  Evaluating de-packaging technologies to remove film plastics from food	0	\N	2022-02-09 18:49:48.459655	2022-02-09 18:49:48.459655
\.


--
-- Data for Name: results; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.results (id, name, my_module_id, user_id, created_at, updated_at, archived, archived_on, last_modified_by_id, archived_by_id, restored_by_id, restored_on) FROM stdin;
3	DNA q results	4	1	2019-05-29 23:59:45.593048	2019-05-30 12:40:54.077122	f	\N	\N	\N	\N	\N
4	Sample distribution on the plate	6	1	2019-05-29 01:51:18.541415	2019-05-30 12:40:54.164408	f	\N	\N	\N	\N	\N
8	qPCR raw data	6	1	2019-05-29 08:32:08.485312	2019-05-30 12:40:54.202511	f	\N	\N	\N	\N	\N
9	qPCR results	7	1	2019-05-28 15:05:19.919081	2019-05-30 12:40:54.254473	f	\N	\N	\N	\N	\N
1	Number of samples	2	1	2019-05-30 09:16:37.21109	2019-05-30 12:40:53.907661	f	\N	\N	\N	\N	\N
2	Nanodrop results	3	1	2019-05-26 20:07:20.81334	2019-05-30 12:40:53.956128	f	\N	\N	\N	\N	\N
5	Data analysis	6	1	2019-05-30 02:27:36.685903	2019-05-30 12:40:54.189107	f	\N	\N	\N	\N	\N
6	Immunofluorescence summary	6	1	2019-05-28 14:38:17.131044	2019-05-30 12:40:54.192068	f	\N	\N	\N	\N	\N
7	Discussion	6	1	2019-05-28 11:50:24.325993	2019-05-30 12:40:54.195148	f	\N	\N	\N	\N	\N
18	Article	6	1	2019-05-30 03:20:15.579781	2019-05-30 12:41:23.967721	f	\N	\N	\N	\N	\N
19	Template for ddCq analysis	8	1	2019-05-28 09:51:21.181731	2019-05-30 12:41:24.889241	f	\N	\N	\N	\N	\N
10	Mock inoculated plant	2	1	2019-05-30 09:53:59.540262	2019-05-30 12:41:26.922946	f	\N	\N	\N	\N	\N
11	Plant	2	1	2019-05-30 08:11:29.5161	2019-05-30 12:41:27.349866	f	\N	\N	\N	\N	\N
12	Agarose gel electrophoresis of totRNA samples	3	1	2019-05-26 19:25:28.565232	2019-05-30 12:41:27.521284	f	\N	\N	\N	\N	\N
13	Result of RNA integrity	4	1	2019-05-30 09:28:52.033483	2019-05-30 12:41:27.892729	f	\N	\N	\N	\N	\N
14	Results	6	1	2019-05-29 01:12:08.380214	2019-05-30 12:41:29.074444	f	\N	\N	\N	\N	\N
15	Cromatogram	6	1	2019-05-29 04:53:08.142	2019-05-30 12:41:29.284217	f	\N	\N	\N	\N	\N
16	All results - curves	6	1	2019-05-30 01:38:37.340552	2019-05-30 12:41:29.523033	f	\N	\N	\N	\N	\N
17	Bacteria plates YPGA	6	1	2019-05-29 10:54:21.802517	2019-05-30 12:41:29.697472	f	\N	\N	\N	\N	\N
20	Dilution curve and efficiency	8	1	2019-05-28 22:45:11.716331	2019-05-30 12:41:31.026733	f	\N	\N	\N	\N	\N
21	Relative quantification results	8	1	2019-05-30 10:26:52.06136	2019-05-30 12:41:31.254436	f	\N	\N	\N	\N	\N
55	Result of RNA integrity	22	1	2019-08-23 01:35:23.548104	2019-08-28 07:31:03.795078	f	\N	\N	\N	\N	\N
24	DNA q results	13	4	2019-08-26 06:50:36.252948	2019-08-27 15:15:03.508137	f	\N	\N	\N	\N	\N
25	Sample distribution on the plate	15	4	2019-08-23 09:32:48.091785	2019-08-27 15:15:03.651842	f	\N	\N	\N	\N	\N
29	qPCR raw data	15	4	2019-08-22 21:40:14.072534	2019-08-27 15:15:03.720009	f	\N	\N	\N	\N	\N
30	qPCR results	16	4	2019-08-22 22:47:53.671504	2019-08-27 15:15:03.787661	f	\N	\N	\N	\N	\N
22	Number of samples	11	4	2019-08-26 13:48:41.394381	2019-08-27 15:15:03.210804	f	\N	\N	\N	\N	\N
23	Nanodrop results	12	4	2019-08-25 18:36:35.826403	2019-08-27 15:15:03.286022	f	\N	\N	\N	\N	\N
26	Data analysis	15	4	2019-08-24 13:29:04.465797	2019-08-27 15:15:03.703407	f	\N	\N	\N	\N	\N
27	Immunofluorescence summary	15	4	2019-08-23 22:28:41.781629	2019-08-27 15:15:03.707331	f	\N	\N	\N	\N	\N
28	Discussion	15	4	2019-08-22 12:27:10.404058	2019-08-27 15:15:03.711187	f	\N	\N	\N	\N	\N
31	Mock inoculated plant	11	4	2019-08-27 12:00:47.031348	2019-08-27 15:15:40.468133	f	\N	\N	\N	\N	\N
32	Plant	11	4	2019-08-26 10:44:23.741641	2019-08-27 15:15:40.803871	f	\N	\N	\N	\N	\N
44	Nanodrop results	21	1	2019-08-25 01:49:48.999406	2019-08-28 07:31:00.845004	f	\N	\N	\N	\N	\N
33	Agarose gel electrophoresis of totRNA samples	12	4	2019-08-26 15:13:44.181799	2019-08-27 15:15:41.294826	f	\N	\N	\N	\N	\N
34	Result of RNA integrity	13	4	2019-08-25 20:26:16.041759	2019-08-27 15:15:41.666237	f	\N	\N	\N	\N	\N
35	Results	15	4	2019-08-24 01:03:25.121202	2019-08-27 15:15:43.238284	f	\N	\N	\N	\N	\N
36	Cromatogram	15	4	2019-08-24 01:58:49.598004	2019-08-27 15:15:43.550178	f	\N	\N	\N	\N	\N
37	All results - curves	15	4	2019-08-26 07:51:08.625565	2019-08-27 15:15:43.844885	f	\N	\N	\N	\N	\N
38	Bacteria plates YPGA	15	4	2019-08-23 18:29:57.250483	2019-08-27 15:15:44.296128	f	\N	\N	\N	\N	\N
39	Article	15	4	2019-08-24 18:58:44.659175	2019-08-27 15:15:44.562653	f	\N	\N	\N	\N	\N
40	Template for ddCq analysis	17	4	2019-08-23 20:43:42.203288	2019-08-27 15:15:45.801042	f	\N	\N	\N	\N	\N
41	Dilution curve and efficiency	17	4	2019-08-24 18:35:49.134604	2019-08-27 15:15:46.219431	f	\N	\N	\N	\N	\N
42	Relative quantification results	17	4	2019-08-24 07:11:53.404709	2019-08-27 15:15:46.495159	f	\N	\N	\N	\N	\N
43	Number of samples	20	1	2019-08-26 06:12:32.620458	2019-08-28 07:31:00.747281	f	\N	\N	\N	\N	\N
45	DNA q results	22	1	2019-08-23 04:19:18.2938	2019-08-28 07:31:01.202618	f	\N	\N	\N	\N	\N
46	Sample distribution on the plate	24	1	2019-08-23 23:38:44.649747	2019-08-28 07:31:01.338257	f	\N	\N	\N	\N	\N
47	Data analysis	24	1	2019-08-24 08:19:10.08889	2019-08-28 07:31:01.392279	f	\N	\N	\N	\N	\N
48	Immunofluorescence summary	24	1	2019-08-24 01:30:30.449864	2019-08-28 07:31:01.398723	f	\N	\N	\N	\N	\N
49	Discussion	24	1	2019-08-26 14:31:13.184856	2019-08-28 07:31:01.406428	f	\N	\N	\N	\N	\N
50	qPCR raw data	24	1	2019-08-23 05:40:49.383188	2019-08-28 07:31:01.439506	f	\N	\N	\N	\N	\N
51	qPCR results	25	1	2019-08-27 05:46:24.478011	2019-08-28 07:31:01.540884	f	\N	\N	\N	\N	\N
52	Mock inoculated plant	20	1	2019-08-27 04:13:32.360334	2019-08-28 07:31:02.731895	f	\N	\N	\N	\N	\N
53	Plant	20	1	2019-08-26 19:46:31.579849	2019-08-28 07:31:03.137301	f	\N	\N	\N	\N	\N
54	Agarose gel electrophoresis of totRNA samples	21	1	2019-08-25 13:01:26.486445	2019-08-28 07:31:03.429507	f	\N	\N	\N	\N	\N
56	Results	24	1	2019-08-23 13:41:56.279984	2019-08-28 07:31:05.197469	f	\N	\N	\N	\N	\N
57	Cromatogram	24	1	2019-08-24 08:37:05.92084	2019-08-28 07:31:05.467657	f	\N	\N	\N	\N	\N
58	All results - curves	24	1	2019-08-24 01:31:44.731026	2019-08-28 07:31:05.728293	f	\N	\N	\N	\N	\N
61	Template for ddCq analysis	26	1	2019-08-23 16:38:13.56031	2019-08-28 07:31:07.61665	f	\N	\N	\N	\N	\N
59	Bacteria plates YPGA	24	1	2019-08-24 17:24:14.286969	2019-08-28 07:31:06.120198	f	\N	\N	\N	\N	\N
60	Article	24	1	2019-08-26 16:32:32.759548	2019-08-28 07:31:06.401197	f	\N	\N	\N	\N	\N
62	Dilution curve and efficiency	26	1	2019-08-24 18:36:23.049771	2019-08-28 07:31:07.875232	f	\N	\N	\N	\N	\N
63	Relative quantification results	26	1	2019-08-24 00:03:00.907162	2019-08-28 07:31:08.270619	f	\N	\N	\N	\N	\N
66	DNA q results	31	6	2019-08-27 15:12:59.216653	2019-08-28 08:34:56.674998	f	\N	\N	\N	\N	\N
67	Sample distribution on the plate	33	6	2019-08-23 21:27:21.664688	2019-08-28 08:34:56.892766	f	\N	\N	\N	\N	\N
71	qPCR raw data	33	6	2019-08-24 09:32:55.074848	2019-08-28 08:34:57.15671	f	\N	\N	\N	\N	\N
72	qPCR results	34	6	2019-08-26 14:22:15.042524	2019-08-28 08:34:57.257276	f	\N	\N	\N	\N	\N
64	Number of samples	29	6	2019-08-27 02:30:36.191508	2019-08-28 08:34:56.418694	f	\N	\N	\N	\N	\N
65	Nanodrop results	30	6	2019-08-25 13:14:06.10807	2019-08-28 08:34:56.498931	f	\N	\N	\N	\N	\N
68	Data analysis	33	6	2019-08-24 07:16:23.149385	2019-08-28 08:34:57.102285	f	\N	\N	\N	\N	\N
69	Immunofluorescence summary	33	6	2019-08-25 01:11:08.852161	2019-08-28 08:34:57.125812	f	\N	\N	\N	\N	\N
70	Discussion	33	6	2019-08-27 15:02:08.623604	2019-08-28 08:34:57.135316	f	\N	\N	\N	\N	\N
73	Mock inoculated plant	29	6	2019-08-26 20:49:59.917012	2019-08-28 08:35:48.779908	f	\N	\N	\N	\N	\N
74	Plant	29	6	2019-08-24 23:32:58.093077	2019-08-28 08:35:49.271192	f	\N	\N	\N	\N	\N
116	Plant	47	8	2019-09-11 08:34:00.618406	2019-09-12 18:26:27.911276	f	\N	\N	\N	\N	\N
75	Agarose gel electrophoresis of totRNA samples	30	6	2019-08-25 15:03:37.32167	2019-08-28 08:35:49.611451	f	\N	\N	\N	\N	\N
76	Result of RNA integrity	31	6	2019-08-28 02:26:20.251142	2019-08-28 08:35:50.209965	f	\N	\N	\N	\N	\N
77	Results	33	6	2019-08-23 22:26:57.365977	2019-08-28 08:35:51.876053	f	\N	\N	\N	\N	\N
78	Cromatogram	33	6	2019-08-24 14:19:51.954708	2019-08-28 08:35:52.427288	f	\N	\N	\N	\N	\N
79	All results - curves	33	6	2019-08-24 18:35:30.704134	2019-08-28 08:35:52.739817	f	\N	\N	\N	\N	\N
101	Bacteria plates YPGA	42	7	2019-08-25 02:08:48.548848	2019-08-29 12:52:40.325334	f	\N	\N	\N	\N	\N
80	Bacteria plates YPGA	33	6	2019-08-24 12:41:55.875484	2019-08-28 08:35:53.252688	f	\N	\N	\N	\N	\N
81	Article	33	6	2019-08-25 00:15:44.237293	2019-08-28 08:35:53.596535	f	\N	\N	\N	\N	\N
82	Template for ddCq analysis	35	6	2019-08-23 04:38:40.672669	2019-08-28 08:35:55.191863	f	\N	\N	\N	\N	\N
83	Dilution curve and efficiency	35	6	2019-08-23 23:48:47.830454	2019-08-28 08:35:55.523413	f	\N	\N	\N	\N	\N
84	Relative quantification results	35	6	2019-08-24 10:05:04.185618	2019-08-28 08:35:55.822984	f	\N	\N	\N	\N	\N
87	DNA q results	40	7	2019-08-24 14:10:35.912473	2019-08-29 12:51:27.409246	f	\N	\N	\N	\N	\N
88	Sample distribution on the plate	42	7	2019-08-25 04:03:20.082078	2019-08-29 12:51:28.085447	f	\N	\N	\N	\N	\N
92	qPCR raw data	42	7	2019-08-25 02:31:28.498024	2019-08-29 12:51:28.613517	f	\N	\N	\N	\N	\N
93	qPCR results	43	7	2019-08-25 18:08:13.795706	2019-08-29 12:51:28.976905	f	\N	\N	\N	\N	\N
85	Number of samples	38	7	2019-08-28 10:47:56.848087	2019-08-29 12:51:26.82919	f	\N	\N	\N	\N	\N
86	Nanodrop results	39	7	2019-08-29 02:03:44.622162	2019-08-29 12:51:26.940977	f	\N	\N	\N	\N	\N
89	Data analysis	42	7	2019-08-24 05:57:15.569561	2019-08-29 12:51:28.415226	f	\N	\N	\N	\N	\N
90	Immunofluorescence summary	42	7	2019-08-27 16:54:00.913301	2019-08-29 12:51:28.560167	f	\N	\N	\N	\N	\N
91	Discussion	42	7	2019-08-25 10:30:20.367651	2019-08-29 12:51:28.575071	f	\N	\N	\N	\N	\N
94	Mock inoculated plant	38	7	2019-08-27 09:42:55.351081	2019-08-29 12:52:31.523227	f	\N	\N	\N	\N	\N
95	Plant	38	7	2019-08-28 20:23:46.391813	2019-08-29 12:52:32.378549	f	\N	\N	\N	\N	\N
102	Article	42	7	2019-08-28 02:05:37.210522	2019-08-29 12:52:40.763282	f	\N	\N	\N	\N	\N
96	Agarose gel electrophoresis of totRNA samples	39	7	2019-08-29 12:38:54.365105	2019-08-29 12:52:33.058185	f	\N	\N	\N	\N	\N
97	Result of RNA integrity	40	7	2019-08-24 18:36:50.103256	2019-08-29 12:52:34.409816	f	\N	\N	\N	\N	\N
98	Results	42	7	2019-08-25 07:23:19.701519	2019-08-29 12:52:37.823256	f	\N	\N	\N	\N	\N
99	Cromatogram	42	7	2019-08-24 06:03:18.811359	2019-08-29 12:52:38.575266	f	\N	\N	\N	\N	\N
100	All results - curves	42	7	2019-08-25 14:26:55.319614	2019-08-29 12:52:39.4142	f	\N	\N	\N	\N	\N
103	Template for ddCq analysis	44	7	2019-08-28 15:14:36.192685	2019-08-29 12:52:43.861202	f	\N	\N	\N	\N	\N
104	Dilution curve and efficiency	44	7	2019-08-28 18:18:33.000745	2019-08-29 12:52:44.528759	f	\N	\N	\N	\N	\N
105	Relative quantification results	44	7	2019-08-28 12:23:22.84094	2019-08-29 12:52:45.226635	f	\N	\N	\N	\N	\N
108	DNA q results	49	8	2019-09-11 02:04:23.196192	2019-09-12 18:26:19.308132	f	\N	\N	\N	\N	\N
109	Sample distribution on the plate	51	8	2019-09-07 19:54:50.156082	2019-09-12 18:26:19.411754	f	\N	\N	\N	\N	\N
113	qPCR raw data	51	8	2019-09-06 23:40:12.716957	2019-09-12 18:26:19.460723	f	\N	\N	\N	\N	\N
114	qPCR results	52	8	2019-09-12 13:24:38.191569	2019-09-12 18:26:19.551156	f	\N	\N	\N	\N	\N
106	Number of samples	47	8	2019-09-11 07:03:26.293501	2019-09-12 18:26:19.092582	f	\N	\N	\N	\N	\N
107	Nanodrop results	48	8	2019-09-07 22:20:21.457743	2019-09-12 18:26:19.148496	f	\N	\N	\N	\N	\N
110	Data analysis	51	8	2019-09-09 00:33:28.8448	2019-09-12 18:26:19.443869	f	\N	\N	\N	\N	\N
111	Immunofluorescence summary	51	8	2019-09-09 14:10:53.780743	2019-09-12 18:26:19.447549	f	\N	\N	\N	\N	\N
112	Discussion	51	8	2019-09-09 07:55:57.192004	2019-09-12 18:26:19.451586	f	\N	\N	\N	\N	\N
115	Mock inoculated plant	47	8	2019-09-10 23:23:53.065995	2019-09-12 18:26:27.629276	f	\N	\N	\N	\N	\N
124	Template for ddCq analysis	53	8	2019-09-08 18:15:45.459187	2019-09-12 18:26:31.818017	f	\N	\N	\N	\N	\N
117	Agarose gel electrophoresis of totRNA samples	48	8	2019-09-10 04:57:45.112832	2019-09-12 18:26:28.216075	f	\N	\N	\N	\N	\N
118	Result of RNA integrity	49	8	2019-09-12 00:19:24.139156	2019-09-12 18:26:28.602484	f	\N	\N	\N	\N	\N
119	Results	51	8	2019-09-07 20:19:29.537298	2019-09-12 18:26:29.720857	f	\N	\N	\N	\N	\N
120	Cromatogram	51	8	2019-09-08 21:11:38.503213	2019-09-12 18:26:29.980477	f	\N	\N	\N	\N	\N
121	All results - curves	51	8	2019-09-06 23:43:50.83638	2019-09-12 18:26:30.250629	f	\N	\N	\N	\N	\N
122	Bacteria plates YPGA	51	8	2019-09-07 02:10:56.937949	2019-09-12 18:26:30.528176	f	\N	\N	\N	\N	\N
123	Article	51	8	2019-09-07 03:19:40.488014	2019-09-12 18:26:30.773976	f	\N	\N	\N	\N	\N
125	Dilution curve and efficiency	53	8	2019-09-08 18:40:57.309452	2019-09-12 18:26:32.089764	f	\N	\N	\N	\N	\N
126	Relative quantification results	53	8	2019-09-07 21:51:40.665955	2019-09-12 18:26:32.349124	f	\N	\N	\N	\N	\N
129	DNA q results	58	9	2019-11-11 04:15:50.798524	2019-11-13 13:57:43.049188	f	\N	\N	\N	\N	\N
130	Sample distribution on the plate	60	9	2019-11-13 05:55:47.598919	2019-11-13 13:57:43.2229	f	\N	\N	\N	\N	\N
134	qPCR raw data	60	9	2019-11-12 23:49:52.688732	2019-11-13 13:57:43.309257	f	\N	\N	\N	\N	\N
135	qPCR results	61	9	2019-11-13 05:17:14.599992	2019-11-13 13:57:43.404912	f	\N	\N	\N	\N	\N
127	Number of samples	56	9	2019-11-10 21:06:06.314469	2019-11-13 13:57:42.725377	f	\N	\N	\N	\N	\N
128	Nanodrop results	57	9	2019-11-10 22:29:06.747543	2019-11-13 13:57:42.820784	f	\N	\N	\N	\N	\N
131	Data analysis	60	9	2019-11-13 12:21:09.527744	2019-11-13 13:57:43.279415	f	\N	\N	\N	\N	\N
132	Immunofluorescence summary	60	9	2019-11-13 02:40:19.247397	2019-11-13 13:57:43.284865	f	\N	\N	\N	\N	\N
133	Discussion	60	9	2019-11-12 23:53:48.609193	2019-11-13 13:57:43.29022	f	\N	\N	\N	\N	\N
176	qPCR raw data	83	10	2019-11-20 15:21:27.399047	2019-11-25 18:00:53.840501	f	\N	\N	\N	\N	\N
177	qPCR results	84	10	2019-11-25 04:27:02.117221	2019-11-25 18:00:53.93258	f	\N	\N	\N	\N	\N
169	Number of samples	79	10	2019-11-22 18:06:42.376327	2019-11-25 18:00:53.253623	f	\N	\N	\N	\N	\N
144	Article	60	9	2019-11-13 11:06:11.270923	2019-11-13 13:58:17.113944	f	\N	\N	\N	\N	\N
145	Template for ddCq analysis	62	9	2019-11-13 12:34:03.621435	2019-11-13 13:58:18.191245	f	\N	\N	\N	\N	\N
165	Article	69	4	2019-11-24 08:17:05.123126	2019-11-25 14:09:32.947252	f	\N	\N	\N	\N	\N
136	Mock inoculated plant	56	9	2019-11-08 10:47:37.500175	2019-11-13 13:58:20.542883	f	\N	\N	\N	\N	\N
137	Plant	56	9	2019-11-09 08:55:27.744494	2019-11-13 13:58:21.367673	f	\N	\N	\N	\N	\N
138	Agarose gel electrophoresis of totRNA samples	57	9	2019-11-12 01:10:28.301426	2019-11-13 13:58:21.627304	f	\N	\N	\N	\N	\N
139	Result of RNA integrity	58	9	2019-11-12 05:14:04.759739	2019-11-13 13:58:22.571743	f	\N	\N	\N	\N	\N
140	Results	60	9	2019-11-13 13:01:37.359466	2019-11-13 13:58:24.34494	f	\N	\N	\N	\N	\N
141	Cromatogram	60	9	2019-11-12 15:56:10.184059	2019-11-13 13:58:24.655199	f	\N	\N	\N	\N	\N
142	All results - curves	60	9	2019-11-13 07:56:53.992447	2019-11-13 13:58:25.057761	f	\N	\N	\N	\N	\N
143	Bacteria plates YPGA	60	9	2019-11-13 05:45:13.120964	2019-11-13 13:58:25.311012	f	\N	\N	\N	\N	\N
146	Dilution curve and efficiency	62	9	2019-11-13 11:26:30.578774	2019-11-13 13:58:27.742718	f	\N	\N	\N	\N	\N
147	Relative quantification results	62	9	2019-11-13 10:27:16.464052	2019-11-13 13:58:28.144055	f	\N	\N	\N	\N	\N
148	Number of samples	65	4	2019-11-23 18:47:42.26901	2019-11-25 14:09:27.55951	f	\N	\N	\N	\N	\N
149	Nanodrop results	66	4	2019-11-20 09:44:33.57309	2019-11-25 14:09:27.932183	f	\N	\N	\N	\N	\N
150	DNA q results	67	4	2019-11-24 14:48:54.710827	2019-11-25 14:09:28.355952	f	\N	\N	\N	\N	\N
151	Sample distribution on the plate	69	4	2019-11-25 01:06:34.283564	2019-11-25 14:09:28.545321	f	\N	\N	\N	\N	\N
152	Data analysis	69	4	2019-11-24 15:44:49.870996	2019-11-25 14:09:28.603148	f	\N	\N	\N	\N	\N
153	Immunofluorescence summary	69	4	2019-11-24 11:03:16.860582	2019-11-25 14:09:28.610839	f	\N	\N	\N	\N	\N
154	Discussion	69	4	2019-11-24 17:13:43.754556	2019-11-25 14:09:28.619781	f	\N	\N	\N	\N	\N
155	qPCR raw data	69	4	2019-11-24 06:50:32.076652	2019-11-25 14:09:28.634068	f	\N	\N	\N	\N	\N
156	qPCR results	70	4	2019-11-25 06:25:57.770028	2019-11-25 14:09:28.730016	f	\N	\N	\N	\N	\N
170	Nanodrop results	80	10	2019-11-24 22:14:17.271717	2019-11-25 18:00:53.367475	f	\N	\N	\N	\N	\N
173	Data analysis	83	10	2019-11-23 10:20:02.675267	2019-11-25 18:00:53.815856	f	\N	\N	\N	\N	\N
166	Template for ddCq analysis	71	4	2019-11-21 11:27:46.870552	2019-11-25 14:09:34.045086	f	\N	\N	\N	\N	\N
174	Immunofluorescence summary	83	10	2019-11-23 16:13:55.087073	2019-11-25 18:00:53.822857	f	\N	\N	\N	\N	\N
175	Discussion	83	10	2019-11-21 01:21:57.750321	2019-11-25 18:00:53.82913	f	\N	\N	\N	\N	\N
157	Mock inoculated plant	65	4	2019-11-24 18:20:57.602668	2019-11-25 14:09:36.333631	f	\N	\N	\N	\N	\N
158	Plant	65	4	2019-11-23 19:39:38.154399	2019-11-25 14:09:37.059207	f	\N	\N	\N	\N	\N
159	Agarose gel electrophoresis of totRNA samples	66	4	2019-11-22 15:24:55.488579	2019-11-25 14:09:37.309045	f	\N	\N	\N	\N	\N
160	Result of RNA integrity	67	4	2019-11-24 04:39:37.318207	2019-11-25 14:09:38.242235	f	\N	\N	\N	\N	\N
161	Results	69	4	2019-11-25 06:26:01.749666	2019-11-25 14:09:39.979289	f	\N	\N	\N	\N	\N
162	Cromatogram	69	4	2019-11-25 03:28:15.015934	2019-11-25 14:09:40.309855	f	\N	\N	\N	\N	\N
163	All results - curves	69	4	2019-11-24 16:46:21.364321	2019-11-25 14:09:40.711531	f	\N	\N	\N	\N	\N
164	Bacteria plates YPGA	69	4	2019-11-25 07:30:23.121652	2019-11-25 14:09:40.988558	f	\N	\N	\N	\N	\N
167	Dilution curve and efficiency	71	4	2019-11-22 23:37:51.513828	2019-11-25 14:09:43.327908	f	\N	\N	\N	\N	\N
168	Relative quantification results	71	4	2019-11-24 01:46:54.818293	2019-11-25 14:09:43.693409	f	\N	\N	\N	\N	\N
184	All results - curves	83	10	2019-11-21 00:08:16.39651	2019-11-25 18:01:10.71177	f	\N	\N	\N	\N	\N
171	DNA q results	81	10	2019-11-24 18:00:14.910962	2019-11-25 18:00:53.594477	f	\N	\N	\N	\N	\N
172	Sample distribution on the plate	83	10	2019-11-20 22:33:04.967509	2019-11-25 18:00:53.754992	f	\N	\N	\N	\N	\N
185	Bacteria plates YPGA	83	10	2019-11-21 10:58:08.543004	2019-11-25 18:01:10.952803	f	\N	\N	\N	\N	\N
188	Dilution curve and efficiency	85	10	2019-11-20 11:22:13.109413	2019-11-25 18:01:13.230513	f	\N	\N	\N	\N	\N
189	Relative quantification results	85	10	2019-11-20 21:57:43.393906	2019-11-25 18:01:13.602298	f	\N	\N	\N	\N	\N
192	DNA q results	90	11	2020-05-10 04:06:43.889482	2020-05-12 16:13:06.614698	f	\N	\N	\N	\N	\N
193	Sample distribution on the plate	92	11	2020-05-12 05:47:53.176146	2020-05-12 16:13:06.785667	f	\N	\N	\N	\N	\N
186	Article	83	10	2019-11-20 20:08:29.006451	2019-11-25 18:01:03.329014	f	\N	\N	\N	\N	\N
187	Template for ddCq analysis	85	10	2019-11-20 09:06:16.328958	2019-11-25 18:01:04.449759	f	\N	\N	\N	\N	\N
178	Mock inoculated plant	79	10	2019-11-21 19:52:11.699065	2019-11-25 18:01:06.504447	f	\N	\N	\N	\N	\N
179	Plant	79	10	2019-11-22 16:36:47.868785	2019-11-25 18:01:07.218728	f	\N	\N	\N	\N	\N
180	Agarose gel electrophoresis of totRNA samples	80	10	2019-11-25 04:27:02.692158	2019-11-25 18:01:07.466487	f	\N	\N	\N	\N	\N
181	Result of RNA integrity	81	10	2019-11-24 22:37:13.06951	2019-11-25 18:01:08.341367	f	\N	\N	\N	\N	\N
182	Results	83	10	2019-11-20 16:12:42.117772	2019-11-25 18:01:10.052745	f	\N	\N	\N	\N	\N
183	Cromatogram	83	10	2019-11-20 17:36:51.181155	2019-11-25 18:01:10.348572	f	\N	\N	\N	\N	\N
197	qPCR raw data	92	11	2020-05-12 14:39:08.916828	2020-05-12 16:13:06.892162	f	\N	\N	\N	\N	\N
198	qPCR results	93	11	2020-05-11 00:00:25.490192	2020-05-12 16:13:06.980116	f	\N	\N	\N	\N	\N
190	Number of samples	88	11	2020-05-08 23:45:57.428985	2020-05-12 16:13:06.050718	f	\N	\N	\N	\N	\N
191	Nanodrop results	89	11	2020-05-09 02:52:38.441762	2020-05-12 16:13:06.417272	f	\N	\N	\N	\N	\N
194	Data analysis	92	11	2020-05-12 00:16:54.77403	2020-05-12 16:13:06.865975	f	\N	\N	\N	\N	\N
195	Immunofluorescence summary	92	11	2020-05-12 11:00:18.086556	2020-05-12 16:13:06.872049	f	\N	\N	\N	\N	\N
196	Discussion	92	11	2020-05-12 06:40:53.083737	2020-05-12 16:13:06.879226	f	\N	\N	\N	\N	\N
239	qPCR raw data	111	16	2020-10-06 05:26:29.847566	2020-10-07 19:20:27.510127	f	\N	\N	\N	\N	\N
240	qPCR results	112	16	2020-10-06 11:37:30.066667	2020-10-07 19:20:27.619724	f	\N	\N	\N	\N	\N
232	Number of samples	107	16	2020-10-07 07:09:21.134863	2020-10-07 19:20:26.829108	f	\N	\N	\N	\N	\N
233	Nanodrop results	108	16	2020-10-03 07:05:45.832298	2020-10-07 19:20:26.92869	f	\N	\N	\N	\N	\N
207	Article	92	11	2020-05-12 14:17:55.593023	2020-05-12 16:13:52.650198	f	\N	\N	\N	\N	\N
208	Template for ddCq analysis	94	11	2020-05-12 11:06:13.008496	2020-05-12 16:13:53.797207	f	\N	\N	\N	\N	\N
228	Article	101	12	2020-06-09 07:23:49.626805	2020-06-10 14:44:22.906541	f	\N	\N	\N	\N	\N
199	Mock inoculated plant	88	11	2020-05-07 16:51:09.477048	2020-05-12 16:13:56.454844	f	\N	\N	\N	\N	\N
200	Plant	88	11	2020-05-07 11:48:36.378433	2020-05-12 16:13:57.227578	f	\N	\N	\N	\N	\N
201	Agarose gel electrophoresis of totRNA samples	89	11	2020-05-11 11:59:47.327018	2020-05-12 16:13:57.47832	f	\N	\N	\N	\N	\N
202	Result of RNA integrity	90	11	2020-05-10 16:56:40.863878	2020-05-12 16:13:58.515054	f	\N	\N	\N	\N	\N
203	Results	92	11	2020-05-12 09:03:35.586291	2020-05-12 16:14:00.355593	f	\N	\N	\N	\N	\N
204	Cromatogram	92	11	2020-05-12 13:01:23.542836	2020-05-12 16:14:00.664042	f	\N	\N	\N	\N	\N
205	All results - curves	92	11	2020-05-11 20:52:56.984582	2020-05-12 16:14:01.113741	f	\N	\N	\N	\N	\N
206	Bacteria plates YPGA	92	11	2020-05-12 05:52:29.763934	2020-05-12 16:14:01.368884	f	\N	\N	\N	\N	\N
209	Dilution curve and efficiency	94	11	2020-05-12 10:31:41.377274	2020-05-12 16:14:04.166684	f	\N	\N	\N	\N	\N
210	Relative quantification results	94	11	2020-05-12 15:59:40.038871	2020-05-12 16:14:04.502661	f	\N	\N	\N	\N	\N
213	DNA q results	99	12	2020-06-07 04:36:34.381268	2020-06-10 14:43:18.45671	f	\N	\N	\N	\N	\N
214	Sample distribution on the plate	101	12	2020-06-09 08:58:37.479608	2020-06-10 14:43:18.597227	f	\N	\N	\N	\N	\N
218	qPCR raw data	101	12	2020-06-08 18:21:26.328582	2020-06-10 14:43:18.663994	f	\N	\N	\N	\N	\N
219	qPCR results	102	12	2020-06-06 14:00:34.456459	2020-06-10 14:43:18.739661	f	\N	\N	\N	\N	\N
211	Number of samples	97	12	2020-06-05 12:03:55.217012	2020-06-10 14:43:18.164669	f	\N	\N	\N	\N	\N
212	Nanodrop results	98	12	2020-06-09 01:41:04.204446	2020-06-10 14:43:18.241454	f	\N	\N	\N	\N	\N
215	Data analysis	101	12	2020-06-10 12:44:09.359765	2020-06-10 14:43:18.643881	f	\N	\N	\N	\N	\N
216	Immunofluorescence summary	101	12	2020-06-08 21:26:57.013628	2020-06-10 14:43:18.648867	f	\N	\N	\N	\N	\N
217	Discussion	101	12	2020-06-10 00:03:57.049095	2020-06-10 14:43:18.653631	f	\N	\N	\N	\N	\N
236	Data analysis	111	16	2020-10-06 12:24:48.504482	2020-10-07 19:20:27.475487	f	\N	\N	\N	\N	\N
237	Immunofluorescence summary	111	16	2020-10-06 04:33:50.821228	2020-10-07 19:20:27.484431	f	\N	\N	\N	\N	\N
229	Template for ddCq analysis	103	12	2020-06-05 17:05:13.163818	2020-06-10 14:44:23.881633	f	\N	\N	\N	\N	\N
238	Discussion	111	16	2020-10-06 21:01:18.762697	2020-10-07 19:20:27.495027	f	\N	\N	\N	\N	\N
251	Dilution curve and efficiency	113	16	2020-10-02 10:38:38.102817	2020-10-07 19:20:48.205037	f	\N	\N	\N	\N	\N
220	Mock inoculated plant	97	12	2020-06-06 01:48:05.045977	2020-06-10 14:44:26.211307	f	\N	\N	\N	\N	\N
221	Plant	97	12	2020-06-07 15:12:41.921924	2020-06-10 14:44:26.932982	f	\N	\N	\N	\N	\N
222	Agarose gel electrophoresis of totRNA samples	98	12	2020-06-10 12:27:52.517234	2020-06-10 14:44:27.155081	f	\N	\N	\N	\N	\N
223	Result of RNA integrity	99	12	2020-06-07 06:37:12.939669	2020-06-10 14:44:28.073603	f	\N	\N	\N	\N	\N
224	Results	101	12	2020-06-10 07:15:08.009067	2020-06-10 14:44:30.621336	f	\N	\N	\N	\N	\N
225	Cromatogram	101	12	2020-06-09 10:46:23.212951	2020-06-10 14:44:31.205168	f	\N	\N	\N	\N	\N
226	All results - curves	101	12	2020-06-08 18:02:43.596045	2020-06-10 14:44:31.877607	f	\N	\N	\N	\N	\N
227	Bacteria plates YPGA	101	12	2020-06-08 11:30:26.318107	2020-06-10 14:44:32.172267	f	\N	\N	\N	\N	\N
230	Dilution curve and efficiency	103	12	2020-06-06 06:50:31.081819	2020-06-10 14:44:34.939396	f	\N	\N	\N	\N	\N
231	Relative quantification results	103	12	2020-06-06 13:46:34.107847	2020-06-10 14:44:35.251572	f	\N	\N	\N	\N	\N
252	Relative quantification results	113	16	2020-10-04 10:42:01.233603	2020-10-07 19:20:48.534666	f	\N	\N	\N	\N	\N
234	DNA q results	109	16	2020-10-06 12:52:48.577669	2020-10-07 19:20:27.198805	f	\N	\N	\N	\N	\N
235	Sample distribution on the plate	111	16	2020-10-06 02:20:00.851316	2020-10-07 19:20:27.339247	f	\N	\N	\N	\N	\N
253	Number of samples	116	1	2020-11-18 08:03:08.576049	2020-11-18 14:50:54.015118	f	\N	\N	\N	\N	\N
255	DNA q results	118	1	2020-11-14 18:25:21.525837	2020-11-18 14:50:54.432945	f	\N	\N	\N	\N	\N
254	Nanodrop results	117	1	2020-11-16 04:39:37.329801	2020-11-18 14:50:54.187076	f	\N	\N	\N	\N	\N
241	Mock inoculated plant	107	16	2020-10-06 19:55:19.949894	2020-10-07 19:20:41.013135	f	\N	\N	\N	\N	\N
249	Article	111	16	2020-10-07 14:18:06.56894	2020-10-07 19:20:37.2613	f	\N	\N	\N	\N	\N
250	Template for ddCq analysis	113	16	2020-10-02 22:37:43.405074	2020-10-07 19:20:38.465492	f	\N	\N	\N	\N	\N
256	Sample distribution on the plate	120	1	2020-11-14 07:15:49.171601	2020-11-18 14:50:54.599076	f	\N	\N	\N	\N	\N
242	Plant	107	16	2020-10-07 06:14:11.599624	2020-10-07 19:20:41.859613	f	\N	\N	\N	\N	\N
243	Agarose gel electrophoresis of totRNA samples	108	16	2020-10-04 05:46:30.420542	2020-10-07 19:20:42.148071	f	\N	\N	\N	\N	\N
244	Result of RNA integrity	109	16	2020-10-06 20:10:03.426186	2020-10-07 19:20:43.097981	f	\N	\N	\N	\N	\N
245	Results	111	16	2020-10-07 14:40:37.128582	2020-10-07 19:20:44.791399	f	\N	\N	\N	\N	\N
246	Cromatogram	111	16	2020-10-07 09:21:45.643049	2020-10-07 19:20:45.106531	f	\N	\N	\N	\N	\N
247	All results - curves	111	16	2020-10-07 10:58:08.318659	2020-10-07 19:20:45.493151	f	\N	\N	\N	\N	\N
248	Bacteria plates YPGA	111	16	2020-10-06 02:42:45.458174	2020-10-07 19:20:45.729153	f	\N	\N	\N	\N	\N
257	Data analysis	120	1	2020-11-14 05:52:46.460576	2020-11-18 14:50:54.675499	f	\N	\N	\N	\N	\N
258	Immunofluorescence summary	120	1	2020-11-15 10:30:19.823912	2020-11-18 14:50:54.683801	f	\N	\N	\N	\N	\N
259	Discussion	120	1	2020-11-15 15:31:32.099019	2020-11-18 14:50:54.692613	f	\N	\N	\N	\N	\N
260	qPCR raw data	120	1	2020-11-14 12:17:53.917837	2020-11-18 14:50:54.705842	f	\N	\N	\N	\N	\N
261	qPCR results	121	1	2020-11-14 16:11:45.10358	2020-11-18 14:50:54.822851	f	\N	\N	\N	\N	\N
263	Plant	116	1	2020-11-17 23:05:01.484351	2020-11-18 14:51:03.404828	f	\N	\N	\N	\N	\N
264	Agarose gel electrophoresis of totRNA samples	117	1	2020-11-18 05:08:32.865536	2020-11-18 14:51:03.651877	f	\N	\N	\N	\N	\N
266	Results	120	1	2020-11-14 17:02:59.396144	2020-11-18 14:51:06.391862	f	\N	\N	\N	\N	\N
265	Result of RNA integrity	118	1	2020-11-15 15:07:22.546134	2020-11-18 14:51:04.57968	f	\N	\N	\N	\N	\N
286	Result of RNA integrity	127	17	2020-11-15 23:46:40.020593	2020-11-18 14:54:21.186096	f	\N	\N	\N	\N	\N
287	Results	129	17	2020-11-18 02:07:40.609777	2020-11-18 14:54:23.038788	f	\N	\N	\N	\N	\N
288	Cromatogram	129	17	2020-11-17 22:19:59.948171	2020-11-18 14:54:23.329607	f	\N	\N	\N	\N	\N
289	All results - curves	129	17	2020-11-18 07:55:19.738946	2020-11-18 14:54:23.721634	f	\N	\N	\N	\N	\N
270	Article	120	1	2020-11-14 18:34:36.718008	2020-11-18 14:50:58.863337	f	\N	\N	\N	\N	\N
271	Template for ddCq analysis	122	1	2020-11-15 06:05:40.672959	2020-11-18 14:50:59.943345	f	\N	\N	\N	\N	\N
290	Bacteria plates YPGA	129	17	2020-11-18 12:59:06.544219	2020-11-18 14:54:23.978628	f	\N	\N	\N	\N	\N
293	Dilution curve and efficiency	131	17	2020-11-14 00:47:25.562949	2020-11-18 14:54:26.553023	f	\N	\N	\N	\N	\N
262	Mock inoculated plant	116	1	2020-11-17 16:14:11.614448	2020-11-18 14:51:02.638489	f	\N	\N	\N	\N	\N
267	Cromatogram	120	1	2020-11-14 21:42:19.947104	2020-11-18 14:51:06.687204	f	\N	\N	\N	\N	\N
268	All results - curves	120	1	2020-11-13 22:31:09.528125	2020-11-18 14:51:07.150669	f	\N	\N	\N	\N	\N
269	Bacteria plates YPGA	120	1	2020-11-14 15:10:35.660744	2020-11-18 14:51:07.433823	f	\N	\N	\N	\N	\N
272	Dilution curve and efficiency	122	1	2020-11-15 07:38:37.519846	2020-11-18 14:51:09.77503	f	\N	\N	\N	\N	\N
273	Relative quantification results	122	1	2020-11-15 18:53:10.501053	2020-11-18 14:51:10.11068	f	\N	\N	\N	\N	\N
294	Relative quantification results	131	17	2020-11-15 01:31:05.185574	2020-11-18 14:54:26.893317	f	\N	\N	\N	\N	\N
276	DNA q results	127	17	2020-11-16 08:11:23.435287	2020-11-18 14:53:25.842333	f	\N	\N	\N	\N	\N
277	Sample distribution on the plate	129	17	2020-11-17 20:08:55.832407	2020-11-18 14:53:25.991055	f	\N	\N	\N	\N	\N
281	qPCR raw data	129	17	2020-11-18 03:07:11.775102	2020-11-18 14:53:26.050789	f	\N	\N	\N	\N	\N
282	qPCR results	130	17	2020-11-15 12:07:00.832875	2020-11-18 14:53:26.129267	f	\N	\N	\N	\N	\N
274	Number of samples	125	17	2020-11-18 08:34:32.689055	2020-11-18 14:53:25.558446	f	\N	\N	\N	\N	\N
275	Nanodrop results	126	17	2020-11-17 14:03:29.936494	2020-11-18 14:53:25.643627	f	\N	\N	\N	\N	\N
278	Data analysis	129	17	2020-11-18 08:46:42.621063	2020-11-18 14:53:26.030292	f	\N	\N	\N	\N	\N
279	Immunofluorescence summary	129	17	2020-11-18 02:37:37.930508	2020-11-18 14:53:26.035225	f	\N	\N	\N	\N	\N
280	Discussion	129	17	2020-11-17 13:13:17.87307	2020-11-18 14:53:26.040803	f	\N	\N	\N	\N	\N
321	DNA q results	158	21	2021-05-15 15:24:59.726389	2021-05-20 14:12:31.163649	f	\N	\N	\N	\N	\N
295	SOP 2338.11 - Quarterly Timing Checks - Attachment 3	133	4	2021-04-08 15:16:39.224248	2021-04-08 15:25:41.691691	t	2021-04-08 15:25:41	4	4	\N	\N
307	Mock inoculated plant	138	20	2021-05-17 20:46:27.3527	2021-05-19 20:02:05.304686	f	\N	\N	\N	\N	\N
296	SOP 2338.11 - Attachment 3 - Quaterly Timing Checks	133	4	2021-04-08 15:35:45.545931	2021-04-08 15:35:45.555505	f	\N	4	\N	\N	\N
322	Sample distribution on the plate	160	21	2021-05-15 19:42:46.910375	2021-05-20 14:12:31.251434	f	\N	\N	\N	\N	\N
291	Article	129	17	2020-11-18 14:49:08.817811	2020-11-18 14:54:15.989951	f	\N	\N	\N	\N	\N
292	Template for ddCq analysis	131	17	2020-11-13 18:30:18.645198	2020-11-18 14:54:17.11921	f	\N	\N	\N	\N	\N
283	Mock inoculated plant	125	17	2020-11-18 11:47:56.637958	2020-11-18 14:54:19.220916	f	\N	\N	\N	\N	\N
284	Plant	125	17	2020-11-18 04:30:14.276672	2020-11-18 14:54:19.949763	f	\N	\N	\N	\N	\N
285	Agarose gel electrophoresis of totRNA samples	126	17	2020-11-17 22:18:23.722732	2020-11-18 14:54:20.260148	f	\N	\N	\N	\N	\N
297	SOP 2338.11 - Attachment 4 - Generator Flush Log	74	4	2021-04-08 15:45:39.112206	2021-04-08 15:45:39.119422	f	\N	4	\N	\N	\N
300	DNA q results	140	20	2021-05-19 18:19:58.859442	2021-05-19 20:01:52.694328	f	\N	\N	\N	\N	\N
301	Sample distribution on the plate	142	20	2021-05-17 06:33:54.48019	2021-05-19 20:01:52.778289	f	\N	\N	\N	\N	\N
305	qPCR raw data	142	20	2021-05-16 16:17:54.05984	2021-05-19 20:01:52.817008	f	\N	\N	\N	\N	\N
306	qPCR results	143	20	2021-05-17 11:53:22.665428	2021-05-19 20:01:52.865335	f	\N	\N	\N	\N	\N
298	Number of samples	138	20	2021-05-18 02:03:23.71681	2021-05-19 20:01:52.452099	f	\N	\N	\N	\N	\N
299	Nanodrop results	139	20	2021-05-19 13:31:59.198291	2021-05-19 20:01:52.505375	f	\N	\N	\N	\N	\N
302	Data analysis	142	20	2021-05-17 00:39:07.919732	2021-05-19 20:01:52.803902	f	\N	\N	\N	\N	\N
303	Immunofluorescence summary	142	20	2021-05-18 21:29:00.275312	2021-05-19 20:01:52.806785	f	\N	\N	\N	\N	\N
304	Discussion	142	20	2021-05-19 13:42:29.518623	2021-05-19 20:01:52.81016	f	\N	\N	\N	\N	\N
308	Plant	138	20	2021-05-17 05:02:27.133877	2021-05-19 20:02:05.906892	f	\N	\N	\N	\N	\N
326	qPCR raw data	160	21	2021-05-15 04:23:56.373935	2021-05-20 14:12:31.296006	f	\N	\N	\N	\N	\N
315	Article	142	20	2021-05-16 10:04:24.198517	2021-05-19 20:02:02.34504	f	\N	\N	\N	\N	\N
316	Template for ddCq analysis	144	20	2021-05-19 09:03:20.244747	2021-05-19 20:02:03.319041	f	\N	\N	\N	\N	\N
327	qPCR results	161	21	2021-05-16 19:15:48.886448	2021-05-20 14:12:31.349011	f	\N	\N	\N	\N	\N
309	Agarose gel electrophoresis of totRNA samples	139	20	2021-05-19 14:16:10.474373	2021-05-19 20:02:06.143096	f	\N	\N	\N	\N	\N
310	Result of RNA integrity	140	20	2021-05-19 17:03:28.152676	2021-05-19 20:02:06.931415	f	\N	\N	\N	\N	\N
311	Results	142	20	2021-05-17 11:11:10.629967	2021-05-19 20:02:08.419858	f	\N	\N	\N	\N	\N
312	Cromatogram	142	20	2021-05-16 15:31:13.122526	2021-05-19 20:02:08.68479	f	\N	\N	\N	\N	\N
313	All results - curves	142	20	2021-05-19 08:54:40.641402	2021-05-19 20:02:09.057507	f	\N	\N	\N	\N	\N
314	Bacteria plates YPGA	142	20	2021-05-16 08:44:55.834249	2021-05-19 20:02:09.286511	f	\N	\N	\N	\N	\N
317	Dilution curve and efficiency	144	20	2021-05-19 10:02:21.619208	2021-05-19 20:02:11.625754	f	\N	\N	\N	\N	\N
318	Relative quantification results	144	20	2021-05-19 11:19:10.500672	2021-05-19 20:02:11.965172	f	\N	\N	\N	\N	\N
319	Number of samples	156	21	2021-05-20 14:09:40.938674	2021-05-20 14:12:30.983403	f	\N	\N	\N	\N	\N
320	Nanodrop results	157	21	2021-05-19 15:41:25.502176	2021-05-20 14:12:31.040888	f	\N	\N	\N	\N	\N
323	Data analysis	160	21	2021-05-17 02:35:00.290062	2021-05-20 14:12:31.279904	f	\N	\N	\N	\N	\N
324	Immunofluorescence summary	160	21	2021-05-16 01:35:57.027657	2021-05-20 14:12:31.283296	f	\N	\N	\N	\N	\N
325	Discussion	160	21	2021-05-16 00:43:50.560326	2021-05-20 14:12:31.286617	f	\N	\N	\N	\N	\N
330	DNA q results	167	22	2021-05-16 03:53:41.139034	2021-05-20 14:12:57.715408	f	\N	\N	\N	\N	\N
331	Sample distribution on the plate	169	22	2021-05-17 01:16:59.936726	2021-05-20 14:12:57.789421	f	\N	\N	\N	\N	\N
335	qPCR raw data	169	22	2021-05-17 02:10:08.155638	2021-05-20 14:12:57.827757	f	\N	\N	\N	\N	\N
336	qPCR results	170	22	2021-05-19 05:58:16.040427	2021-05-20 14:12:57.877401	f	\N	\N	\N	\N	\N
328	Number of samples	165	22	2021-05-19 21:58:13.970888	2021-05-20 14:12:57.581026	f	\N	\N	\N	\N	\N
329	Nanodrop results	166	22	2021-05-19 19:59:13.668471	2021-05-20 14:12:57.621842	f	\N	\N	\N	\N	\N
332	Data analysis	169	22	2021-05-19 14:40:25.022939	2021-05-20 14:12:57.816376	f	\N	\N	\N	\N	\N
333	Immunofluorescence summary	169	22	2021-05-17 19:41:12.576879	2021-05-20 14:12:57.818938	f	\N	\N	\N	\N	\N
334	Discussion	169	22	2021-05-19 06:43:34.208031	2021-05-20 14:12:57.821588	f	\N	\N	\N	\N	\N
384	DNA q results	185	24	2021-05-16 12:54:05.840005	2021-05-20 14:21:21.096738	f	\N	\N	\N	\N	\N
385	Sample distribution on the plate	187	24	2021-05-16 08:25:30.20429	2021-05-20 14:21:21.172693	f	\N	\N	\N	\N	\N
369	Relative quantification results	171	22	2021-05-20 00:06:10.967794	2021-05-20 14:13:51.788166	f	\N	\N	\N	\N	\N
370	Mock inoculated plant	174	23	2021-05-17 08:46:10.953815	2021-05-20 14:13:52.519551	f	\N	\N	\N	\N	\N
345	Article	160	21	2021-05-15 12:40:36.996304	2021-05-20 14:13:32.723447	f	\N	\N	\N	\N	\N
346	Template for ddCq analysis	162	21	2021-05-17 00:51:28.340244	2021-05-20 14:13:33.41983	f	\N	\N	\N	\N	\N
378	Article	178	23	2021-05-19 07:50:29.872776	2021-05-20 14:13:46.382715	f	\N	\N	\N	\N	\N
354	Agarose gel electrophoresis of totRNA samples	166	22	2021-05-20 09:26:22.379441	2021-05-20 14:13:46.565066	f	\N	\N	\N	\N	\N
379	Template for ddCq analysis	180	23	2021-05-20 02:56:25.413528	2021-05-20 14:13:47.211556	f	\N	\N	\N	\N	\N
353	DNA q results	176	23	2021-05-18 16:54:40.540532	2021-05-20 14:13:34.542611	f	\N	\N	\N	\N	\N
355	Sample distribution on the plate	178	23	2021-05-16 10:58:46.853077	2021-05-20 14:13:34.634728	f	\N	\N	\N	\N	\N
359	qPCR raw data	178	23	2021-05-16 01:18:16.029803	2021-05-20 14:13:34.677015	f	\N	\N	\N	\N	\N
366	Article	169	22	2021-05-19 16:25:52.626712	2021-05-20 14:13:36.412846	f	\N	\N	\N	\N	\N
371	Plant	174	23	2021-05-16 13:53:38.674426	2021-05-20 14:13:52.983211	f	\N	\N	\N	\N	\N
360	qPCR results	179	23	2021-05-19 23:36:31.248163	2021-05-20 14:13:34.734544	f	\N	\N	\N	\N	\N
350	Number of samples	174	23	2021-05-16 14:34:40.322219	2021-05-20 14:13:34.407748	f	\N	\N	\N	\N	\N
352	Nanodrop results	175	23	2021-05-16 03:43:35.232009	2021-05-20 14:13:34.447463	f	\N	\N	\N	\N	\N
356	Data analysis	178	23	2021-05-18 00:33:03.949033	2021-05-20 14:13:34.664382	f	\N	\N	\N	\N	\N
357	Immunofluorescence summary	178	23	2021-05-17 01:34:02.335487	2021-05-20 14:13:34.667067	f	\N	\N	\N	\N	\N
358	Discussion	178	23	2021-05-19 14:11:44.499652	2021-05-20 14:13:34.670211	f	\N	\N	\N	\N	\N
372	Agarose gel electrophoresis of totRNA samples	175	23	2021-05-17 10:51:24.963388	2021-05-20 14:13:53.143295	f	\N	\N	\N	\N	\N
373	Result of RNA integrity	176	23	2021-05-19 04:12:26.024777	2021-05-20 14:13:53.748429	f	\N	\N	\N	\N	\N
374	Results	178	23	2021-05-17 06:22:28.84056	2021-05-20 14:13:54.790188	f	\N	\N	\N	\N	\N
375	Cromatogram	178	23	2021-05-18 16:41:19.528774	2021-05-20 14:13:55.051538	f	\N	\N	\N	\N	\N
367	Template for ddCq analysis	171	22	2021-05-18 07:15:32.011264	2021-05-20 14:13:37.144883	f	\N	\N	\N	\N	\N
376	All results - curves	178	23	2021-05-19 12:07:26.162163	2021-05-20 14:13:55.386105	f	\N	\N	\N	\N	\N
337	Mock inoculated plant	156	21	2021-05-20 04:23:32.027954	2021-05-20 14:13:38.478448	f	\N	\N	\N	\N	\N
338	Plant	156	21	2021-05-20 01:16:39.225005	2021-05-20 14:13:38.926436	f	\N	\N	\N	\N	\N
339	Agarose gel electrophoresis of totRNA samples	157	21	2021-05-19 18:38:34.075705	2021-05-20 14:13:39.080713	f	\N	\N	\N	\N	\N
340	Result of RNA integrity	158	21	2021-05-16 07:14:11.937276	2021-05-20 14:13:39.651805	f	\N	\N	\N	\N	\N
341	Results	160	21	2021-05-16 20:37:56.490517	2021-05-20 14:13:40.670604	f	\N	\N	\N	\N	\N
342	Cromatogram	160	21	2021-05-15 14:22:04.218815	2021-05-20 14:13:40.844929	f	\N	\N	\N	\N	\N
343	All results - curves	160	21	2021-05-18 02:26:10.82275	2021-05-20 14:13:41.098487	f	\N	\N	\N	\N	\N
344	Bacteria plates YPGA	160	21	2021-05-15 03:47:23.465751	2021-05-20 14:13:41.243494	f	\N	\N	\N	\N	\N
347	Dilution curve and efficiency	162	21	2021-05-17 08:30:46.760173	2021-05-20 14:13:42.756633	f	\N	\N	\N	\N	\N
348	Relative quantification results	162	21	2021-05-18 23:54:21.490489	2021-05-20 14:13:42.964818	f	\N	\N	\N	\N	\N
349	Mock inoculated plant	165	22	2021-05-17 22:42:55.482962	2021-05-20 14:13:43.656769	f	\N	\N	\N	\N	\N
389	qPCR raw data	187	24	2021-05-16 05:39:38.665209	2021-05-20 14:21:21.230053	f	\N	\N	\N	\N	\N
351	Plant	165	22	2021-05-19 00:52:36.220234	2021-05-20 14:13:44.83181	f	\N	\N	\N	\N	\N
390	qPCR results	188	24	2021-05-18 23:36:24.342319	2021-05-20 14:21:21.345067	f	\N	\N	\N	\N	\N
382	Number of samples	183	24	2021-05-18 07:45:56.58209	2021-05-20 14:21:20.956597	f	\N	\N	\N	\N	\N
361	Result of RNA integrity	167	22	2021-05-16 21:57:39.206247	2021-05-20 14:13:48.362925	f	\N	\N	\N	\N	\N
362	Results	169	22	2021-05-17 15:59:26.527645	2021-05-20 14:13:49.434812	f	\N	\N	\N	\N	\N
363	Cromatogram	169	22	2021-05-17 22:39:28.610717	2021-05-20 14:13:49.650388	f	\N	\N	\N	\N	\N
364	All results - curves	169	22	2021-05-19 04:23:52.865005	2021-05-20 14:13:49.885438	f	\N	\N	\N	\N	\N
365	Bacteria plates YPGA	169	22	2021-05-17 04:25:11.275555	2021-05-20 14:13:50.030789	f	\N	\N	\N	\N	\N
368	Dilution curve and efficiency	171	22	2021-05-19 08:06:06.472544	2021-05-20 14:13:51.578293	f	\N	\N	\N	\N	\N
377	Bacteria plates YPGA	178	23	2021-05-17 18:33:20.007376	2021-05-20 14:13:55.552434	f	\N	\N	\N	\N	\N
380	Dilution curve and efficiency	180	23	2021-05-20 01:11:09.938858	2021-05-20 14:13:57.182411	f	\N	\N	\N	\N	\N
381	Relative quantification results	180	23	2021-05-19 05:59:00.874299	2021-05-20 14:13:57.392034	f	\N	\N	\N	\N	\N
392	Plant	183	24	2021-05-17 14:40:16.811915	2021-05-20 14:22:08.744812	f	\N	\N	\N	\N	\N
383	Nanodrop results	184	24	2021-05-14 18:02:48.846078	2021-05-20 14:21:20.998312	f	\N	\N	\N	\N	\N
386	Data analysis	187	24	2021-05-19 15:17:33.325898	2021-05-20 14:21:21.210663	f	\N	\N	\N	\N	\N
387	Immunofluorescence summary	187	24	2021-05-15 18:18:17.535595	2021-05-20 14:21:21.213285	f	\N	\N	\N	\N	\N
388	Discussion	187	24	2021-05-18 19:12:22.881427	2021-05-20 14:21:21.217624	f	\N	\N	\N	\N	\N
393	Agarose gel electrophoresis of totRNA samples	184	24	2021-05-16 08:18:32.027862	2021-05-20 14:22:08.896189	f	\N	\N	\N	\N	\N
396	Cromatogram	187	24	2021-05-16 03:52:53.503582	2021-05-20 14:22:10.745456	f	\N	\N	\N	\N	\N
394	Result of RNA integrity	185	24	2021-05-16 19:41:31.805947	2021-05-20 14:22:09.485918	f	\N	\N	\N	\N	\N
395	Results	187	24	2021-05-17 13:22:04.527198	2021-05-20 14:22:10.556099	f	\N	\N	\N	\N	\N
397	All results - curves	187	24	2021-05-16 08:51:28.557256	2021-05-20 14:22:10.996785	f	\N	\N	\N	\N	\N
398	Bacteria plates YPGA	187	24	2021-05-15 22:41:55.728172	2021-05-20 14:22:11.156656	f	\N	\N	\N	\N	\N
399	Article	187	24	2021-05-18 22:38:02.939465	2021-05-20 14:22:05.169818	f	\N	\N	\N	\N	\N
401	Dilution curve and efficiency	189	24	2021-05-19 15:26:55.551275	2021-05-20 14:22:12.735242	f	\N	\N	\N	\N	\N
400	Template for ddCq analysis	189	24	2021-05-19 02:37:52.811296	2021-05-20 14:22:05.875289	f	\N	\N	\N	\N	\N
391	Mock inoculated plant	183	24	2021-05-18 02:01:56.264203	2021-05-20 14:22:08.290786	f	\N	\N	\N	\N	\N
402	Relative quantification results	189	24	2021-05-19 08:09:57.600703	2021-05-20 14:22:12.950215	f	\N	\N	\N	\N	\N
431	qPCR raw data	205	26	2021-05-17 14:41:48.25952	2021-05-20 15:17:12.384878	f	\N	\N	\N	\N	\N
405	DNA q results	194	25	2021-05-15 17:05:13.616301	2021-05-20 14:30:30.863958	f	\N	\N	\N	\N	\N
406	Sample distribution on the plate	196	25	2021-05-18 01:54:28.114294	2021-05-20 14:30:30.973679	f	\N	\N	\N	\N	\N
410	qPCR raw data	196	25	2021-05-18 00:37:36.48825	2021-05-20 14:30:31.012042	f	\N	\N	\N	\N	\N
411	qPCR results	197	25	2021-05-14 22:01:27.289676	2021-05-20 14:30:31.070676	f	\N	\N	\N	\N	\N
403	Number of samples	192	25	2021-05-20 02:15:42.767227	2021-05-20 14:30:30.691035	f	\N	\N	\N	\N	\N
404	Nanodrop results	193	25	2021-05-17 12:21:24.856895	2021-05-20 14:30:30.750264	f	\N	\N	\N	\N	\N
407	Data analysis	196	25	2021-05-20 10:37:51.370953	2021-05-20 14:30:30.999648	f	\N	\N	\N	\N	\N
408	Immunofluorescence summary	196	25	2021-05-18 17:11:34.572524	2021-05-20 14:30:31.002607	f	\N	\N	\N	\N	\N
409	Discussion	196	25	2021-05-18 14:45:38.219333	2021-05-20 14:30:31.005756	f	\N	\N	\N	\N	\N
432	qPCR results	206	26	2021-05-15 00:31:22.415089	2021-05-20 15:17:12.433929	f	\N	\N	\N	\N	\N
424	Number of samples	201	26	2021-05-17 10:53:09.424493	2021-05-20 15:17:12.115944	f	\N	\N	\N	\N	\N
425	Nanodrop results	202	26	2021-05-15 22:00:41.30232	2021-05-20 15:17:12.15734	f	\N	\N	\N	\N	\N
428	Data analysis	205	26	2021-05-20 07:05:59.504962	2021-05-20 15:17:12.371303	f	\N	\N	\N	\N	\N
429	Immunofluorescence summary	205	26	2021-05-18 22:59:14.559707	2021-05-20 15:17:12.374778	f	\N	\N	\N	\N	\N
430	Discussion	205	26	2021-05-19 20:48:28.628248	2021-05-20 15:17:12.378112	f	\N	\N	\N	\N	\N
434	Plant	201	26	2021-05-15 23:36:48.106545	2021-05-20 15:17:43.930169	f	\N	\N	\N	\N	\N
420	Article	196	25	2021-05-18 13:27:28.375856	2021-05-20 14:31:18.045535	f	\N	\N	\N	\N	\N
421	Template for ddCq analysis	198	25	2021-05-19 04:26:38.833559	2021-05-20 14:31:18.750669	f	\N	\N	\N	\N	\N
449	Data analysis	217	27	2021-05-27 14:45:08.521767	2021-05-28 15:07:57.776167	f	\N	\N	\N	\N	\N
412	Mock inoculated plant	192	25	2021-05-20 11:39:43.553495	2021-05-20 14:31:20.577766	f	\N	\N	\N	\N	\N
413	Plant	192	25	2021-05-20 02:15:01.534794	2021-05-20 14:31:21.033596	f	\N	\N	\N	\N	\N
414	Agarose gel electrophoresis of totRNA samples	193	25	2021-05-17 07:21:08.021346	2021-05-20 14:31:21.187263	f	\N	\N	\N	\N	\N
415	Result of RNA integrity	194	25	2021-05-15 07:27:58.100329	2021-05-20 14:31:21.779471	f	\N	\N	\N	\N	\N
416	Results	196	25	2021-05-19 07:49:56.061973	2021-05-20 14:31:22.839968	f	\N	\N	\N	\N	\N
417	Cromatogram	196	25	2021-05-18 15:09:59.82753	2021-05-20 14:31:23.028883	f	\N	\N	\N	\N	\N
418	All results - curves	196	25	2021-05-18 23:18:33.21375	2021-05-20 14:31:23.289716	f	\N	\N	\N	\N	\N
419	Bacteria plates YPGA	196	25	2021-05-17 23:02:31.552501	2021-05-20 14:31:23.448537	f	\N	\N	\N	\N	\N
422	Dilution curve and efficiency	198	25	2021-05-19 14:25:49.813756	2021-05-20 14:31:25.043226	f	\N	\N	\N	\N	\N
423	Relative quantification results	198	25	2021-05-18 10:01:18.801702	2021-05-20 14:31:25.263046	f	\N	\N	\N	\N	\N
435	Agarose gel electrophoresis of totRNA samples	202	26	2021-05-15 19:04:10.608047	2021-05-20 15:17:44.088658	f	\N	\N	\N	\N	\N
426	DNA q results	203	26	2021-05-18 23:18:16.240677	2021-05-20 15:17:12.255175	f	\N	\N	\N	\N	\N
427	Sample distribution on the plate	205	26	2021-05-17 19:36:27.351458	2021-05-20 15:17:12.346009	f	\N	\N	\N	\N	\N
436	Result of RNA integrity	203	26	2021-05-18 06:37:36.501787	2021-05-20 15:17:44.683847	f	\N	\N	\N	\N	\N
450	Immunofluorescence summary	217	27	2021-05-28 05:30:04.521969	2021-05-28 15:07:57.814007	f	\N	\N	\N	\N	\N
451	Discussion	217	27	2021-05-24 23:03:16.881667	2021-05-28 15:07:57.834677	f	\N	\N	\N	\N	\N
456	Agarose gel electrophoresis of totRNA samples	214	27	2021-05-25 18:38:16.068819	2021-05-28 15:08:54.212013	f	\N	\N	\N	\N	\N
437	Results	205	26	2021-05-18 01:50:24.841331	2021-05-20 15:17:45.834598	f	\N	\N	\N	\N	\N
457	Result of RNA integrity	215	27	2021-05-28 00:31:57.656226	2021-05-28 15:08:56.76664	f	\N	\N	\N	\N	\N
441	Article	205	26	2021-05-17 20:01:34.466333	2021-05-20 15:17:41.403223	f	\N	\N	\N	\N	\N
442	Template for ddCq analysis	207	26	2021-05-15 20:28:55.795824	2021-05-20 15:17:42.14524	f	\N	\N	\N	\N	\N
462	Article	217	27	2021-05-28 12:21:56.837627	2021-05-28 15:08:35.577602	f	\N	\N	\N	\N	\N
433	Mock inoculated plant	201	26	2021-05-15 15:11:49.38052	2021-05-20 15:17:43.478357	f	\N	\N	\N	\N	\N
438	Cromatogram	205	26	2021-05-20 02:05:43.958404	2021-05-20 15:17:46.029118	f	\N	\N	\N	\N	\N
439	All results - curves	205	26	2021-05-19 11:29:46.402211	2021-05-20 15:17:46.268299	f	\N	\N	\N	\N	\N
440	Bacteria plates YPGA	205	26	2021-05-18 07:19:51.072586	2021-05-20 15:17:46.423619	f	\N	\N	\N	\N	\N
443	Dilution curve and efficiency	207	26	2021-05-16 09:29:40.96311	2021-05-20 15:17:48.025916	f	\N	\N	\N	\N	\N
444	Relative quantification results	207	26	2021-05-15 09:35:47.445722	2021-05-20 15:17:48.243744	f	\N	\N	\N	\N	\N
459	Cromatogram	217	27	2021-05-25 18:52:27.046085	2021-05-28 15:09:03.512416	f	\N	\N	\N	\N	\N
447	DNA q results	215	27	2021-05-28 07:35:06.883751	2021-05-28 15:07:55.325885	f	\N	\N	\N	\N	\N
448	Sample distribution on the plate	217	27	2021-05-25 08:53:02.959758	2021-05-28 15:07:57.18051	f	\N	\N	\N	\N	\N
452	qPCR raw data	217	27	2021-05-25 18:56:40.893964	2021-05-28 15:07:57.866236	f	\N	\N	\N	\N	\N
453	qPCR results	218	27	2021-05-23 05:31:14.862902	2021-05-28 15:07:58.095162	f	\N	\N	\N	\N	\N
445	Number of samples	213	27	2021-05-22 23:26:35.638116	2021-05-28 15:07:53.191844	f	\N	\N	\N	\N	\N
446	Nanodrop results	214	27	2021-05-23 11:29:18.212839	2021-05-28 15:07:54.550724	f	\N	\N	\N	\N	\N
460	All results - curves	217	27	2021-05-28 09:32:01.306096	2021-05-28 15:09:03.974002	f	\N	\N	\N	\N	\N
461	Bacteria plates YPGA	217	27	2021-05-26 06:54:51.219803	2021-05-28 15:09:04.227605	f	\N	\N	\N	\N	\N
464	Dilution curve and efficiency	219	27	2021-05-25 08:33:29.528799	2021-05-28 15:09:06.868822	f	\N	\N	\N	\N	\N
465	Relative quantification results	219	27	2021-05-25 09:31:03.65887	2021-05-28 15:09:07.418743	f	\N	\N	\N	\N	\N
463	Template for ddCq analysis	219	27	2021-05-25 05:12:05.027812	2021-05-28 15:08:44.194612	f	\N	\N	\N	\N	\N
454	Mock inoculated plant	213	27	2021-05-23 16:18:26.148126	2021-05-28 15:08:52.30229	f	\N	\N	\N	\N	\N
455	Plant	213	27	2021-05-25 03:45:22.686187	2021-05-28 15:08:53.05428	f	\N	\N	\N	\N	\N
458	Results	217	27	2021-05-26 03:28:15.733672	2021-05-28 15:09:03.071593	f	\N	\N	\N	\N	\N
468	DNA q results	224	28	2021-06-01 07:53:19.035449	2021-06-02 16:10:06.503877	f	\N	\N	\N	\N	\N
469	Sample distribution on the plate	226	28	2021-05-28 18:15:55.638775	2021-06-02 16:10:06.895272	f	\N	\N	\N	\N	\N
473	qPCR raw data	226	28	2021-05-28 21:22:33.937511	2021-06-02 16:10:07.138828	f	\N	\N	\N	\N	\N
474	qPCR results	227	28	2021-06-02 14:41:15.476294	2021-06-02 16:10:07.417994	f	\N	\N	\N	\N	\N
466	Number of samples	222	28	2021-05-31 15:20:31.659121	2021-06-02 16:10:04.48281	f	\N	\N	\N	\N	\N
467	Nanodrop results	223	28	2021-06-01 15:47:24.308603	2021-06-02 16:10:04.67329	f	\N	\N	\N	\N	\N
470	Data analysis	226	28	2021-05-30 13:15:00.645629	2021-06-02 16:10:07.086248	f	\N	\N	\N	\N	\N
471	Immunofluorescence summary	226	28	2021-06-01 01:16:10.425168	2021-06-02 16:10:07.099257	f	\N	\N	\N	\N	\N
472	Discussion	226	28	2021-05-29 16:16:50.234404	2021-06-02 16:10:07.114476	f	\N	\N	\N	\N	\N
512	Sample distribution on the plate	255	31	2021-08-22 03:02:23.36954	2021-08-23 17:19:39.899931	f	\N	\N	\N	\N	\N
516	qPCR raw data	255	31	2021-08-22 05:44:48.822794	2021-08-23 17:19:39.948216	f	\N	\N	\N	\N	\N
483	Article	226	28	2021-05-31 16:45:41.264436	2021-06-02 16:10:42.411155	f	\N	\N	\N	\N	\N
484	Template for ddCq analysis	228	28	2021-05-29 22:35:14.254923	2021-06-02 16:10:43.52305	f	\N	\N	\N	\N	\N
475	Mock inoculated plant	222	28	2021-05-31 02:16:22.836451	2021-06-02 16:10:45.995004	f	\N	\N	\N	\N	\N
476	Plant	222	28	2021-06-02 03:16:17.638891	2021-06-02 16:10:46.753317	f	\N	\N	\N	\N	\N
477	Agarose gel electrophoresis of totRNA samples	223	28	2021-06-01 14:05:58.884054	2021-06-02 16:10:47.030412	f	\N	\N	\N	\N	\N
478	Result of RNA integrity	224	28	2021-05-31 20:10:37.220614	2021-06-02 16:10:47.996858	f	\N	\N	\N	\N	\N
479	Results	226	28	2021-05-28 20:55:44.344597	2021-06-02 16:10:49.887011	f	\N	\N	\N	\N	\N
480	Cromatogram	226	28	2021-05-29 19:36:59.736437	2021-06-02 16:10:50.244222	f	\N	\N	\N	\N	\N
481	All results - curves	226	28	2021-05-30 21:20:23.392915	2021-06-02 16:10:50.726417	f	\N	\N	\N	\N	\N
482	Bacteria plates YPGA	226	28	2021-05-30 04:15:27.284241	2021-06-02 16:10:51.043435	f	\N	\N	\N	\N	\N
485	Dilution curve and efficiency	228	28	2021-05-30 12:54:41.41158	2021-06-02 16:10:53.357089	f	\N	\N	\N	\N	\N
486	Relative quantification results	228	28	2021-05-31 02:36:13.104887	2021-06-02 16:10:53.71046	f	\N	\N	\N	\N	\N
487	Capacity results	234	27	2021-06-23 15:38:02.185464	2021-06-23 15:38:02.297148	f	\N	27	\N	\N	\N
517	qPCR results	256	31	2021-08-20 04:54:44.623612	2021-08-23 17:19:40.015699	f	\N	\N	\N	\N	\N
490	DNA q results	244	30	2021-08-19 10:21:22.528439	2021-08-23 14:58:28.696885	f	\N	\N	\N	\N	\N
491	Sample distribution on the plate	246	30	2021-08-20 11:09:26.660443	2021-08-23 14:58:28.781734	f	\N	\N	\N	\N	\N
495	qPCR raw data	246	30	2021-08-20 10:15:30.013075	2021-08-23 14:58:28.822029	f	\N	\N	\N	\N	\N
496	qPCR results	247	30	2021-08-21 04:27:40.636751	2021-08-23 14:58:28.872514	f	\N	\N	\N	\N	\N
488	Number of samples	242	30	2021-08-20 04:59:14.324729	2021-08-23 14:58:28.46203	f	\N	\N	\N	\N	\N
489	Nanodrop results	243	30	2021-08-21 19:12:29.509435	2021-08-23 14:58:28.522995	f	\N	\N	\N	\N	\N
492	Data analysis	246	30	2021-08-22 18:02:04.784763	2021-08-23 14:58:28.808836	f	\N	\N	\N	\N	\N
493	Immunofluorescence summary	246	30	2021-08-22 04:54:32.610545	2021-08-23 14:58:28.811896	f	\N	\N	\N	\N	\N
494	Discussion	246	30	2021-08-21 22:31:45.396918	2021-08-23 14:58:28.815252	f	\N	\N	\N	\N	\N
509	Number of samples	251	31	2021-08-19 09:46:34.074006	2021-08-23 17:19:39.641452	f	\N	\N	\N	\N	\N
510	Nanodrop results	252	31	2021-08-22 02:24:21.727842	2021-08-23 17:19:39.688889	f	\N	\N	\N	\N	\N
505	Article	246	30	2021-08-21 19:20:03.803572	2021-08-23 14:58:59.50597	f	\N	\N	\N	\N	\N
506	Template for ddCq analysis	248	30	2021-08-22 09:37:12.587746	2021-08-23 14:59:00.266123	f	\N	\N	\N	\N	\N
513	Data analysis	255	31	2021-08-23 05:17:27.123168	2021-08-23 17:19:39.930314	f	\N	\N	\N	\N	\N
514	Immunofluorescence summary	255	31	2021-08-22 16:18:44.213888	2021-08-23 17:19:39.93543	f	\N	\N	\N	\N	\N
497	Mock inoculated plant	242	30	2021-08-19 15:06:12.417542	2021-08-23 14:59:02.081105	f	\N	\N	\N	\N	\N
498	Plant	242	30	2021-08-22 02:17:46.621905	2021-08-23 14:59:02.621022	f	\N	\N	\N	\N	\N
499	Agarose gel electrophoresis of totRNA samples	243	30	2021-08-21 14:19:42.902609	2021-08-23 14:59:02.802272	f	\N	\N	\N	\N	\N
500	Result of RNA integrity	244	30	2021-08-20 16:43:58.026618	2021-08-23 14:59:03.529775	f	\N	\N	\N	\N	\N
501	Results	246	30	2021-08-20 10:52:13.873517	2021-08-23 14:59:04.756684	f	\N	\N	\N	\N	\N
502	Cromatogram	246	30	2021-08-22 08:27:00.581202	2021-08-23 14:59:04.968822	f	\N	\N	\N	\N	\N
503	All results - curves	246	30	2021-08-21 09:14:18.348023	2021-08-23 14:59:05.239736	f	\N	\N	\N	\N	\N
504	Bacteria plates YPGA	246	30	2021-08-20 13:28:49.594997	2021-08-23 14:59:05.421792	f	\N	\N	\N	\N	\N
507	Dilution curve and efficiency	248	30	2021-08-22 02:07:47.460214	2021-08-23 14:59:07.186279	f	\N	\N	\N	\N	\N
508	Relative quantification results	248	30	2021-08-23 04:25:30.901685	2021-08-23 14:59:07.436154	f	\N	\N	\N	\N	\N
528	Dilution curve and efficiency	257	31	2021-08-21 13:07:13.270313	2021-08-23 17:20:24.993866	f	\N	\N	\N	\N	\N
511	DNA q results	253	31	2021-08-21 03:13:06.123344	2021-08-23 17:19:39.797065	f	\N	\N	\N	\N	\N
515	Discussion	255	31	2021-08-22 12:34:09.19306	2021-08-23 17:19:39.939107	f	\N	\N	\N	\N	\N
521	Result of RNA integrity	253	31	2021-08-21 19:07:26.386774	2021-08-23 17:20:21.824952	f	\N	\N	\N	\N	\N
522	Results	255	31	2021-08-22 12:01:46.39101	2021-08-23 17:20:22.874561	f	\N	\N	\N	\N	\N
523	Cromatogram	255	31	2021-08-23 04:12:09.62542	2021-08-23 17:20:23.071051	f	\N	\N	\N	\N	\N
529	Relative quantification results	257	31	2021-08-23 05:51:30.395215	2021-08-23 17:20:25.218721	f	\N	\N	\N	\N	\N
526	Article	255	31	2021-08-21 18:43:14.363836	2021-08-23 17:20:18.658104	f	\N	\N	\N	\N	\N
527	Template for ddCq analysis	257	31	2021-08-20 17:00:15.859134	2021-08-23 17:20:19.343932	f	\N	\N	\N	\N	\N
518	Mock inoculated plant	251	31	2021-08-19 07:12:57.018647	2021-08-23 17:20:20.634184	f	\N	\N	\N	\N	\N
519	Plant	251	31	2021-08-22 00:04:47.667428	2021-08-23 17:20:21.074031	f	\N	\N	\N	\N	\N
520	Agarose gel electrophoresis of totRNA samples	252	31	2021-08-22 14:50:57.844597	2021-08-23 17:20:21.228214	f	\N	\N	\N	\N	\N
524	All results - curves	255	31	2021-08-23 17:14:41.855221	2021-08-23 17:20:23.304929	f	\N	\N	\N	\N	\N
525	Bacteria plates YPGA	255	31	2021-08-22 20:49:18.995181	2021-08-23 17:20:23.460897	f	\N	\N	\N	\N	\N
\.


--
-- Data for Name: tables; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.tables (id, contents, created_at, updated_at, created_by_id, last_modified_by_id, data_vector, name, team_id) FROM stdin;
1	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-05-30 12:40:53.821161	2019-05-30 12:40:53.821161	1	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	1
2	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-05-30 12:40:53.863335	2019-05-30 12:40:53.863335	1	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		1
4	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-05-30 12:40:54.056742	2019-05-30 12:40:54.056742	1	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	1
6	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-05-30 12:40:54.115635	2019-05-30 12:40:54.115635	1	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	1
3	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-05-30 12:40:53.962963	2019-05-30 12:40:53.966482	1	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		1
5	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-05-30 12:40:54.072762	2019-05-30 12:40:54.075328	1	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		1
7	\\x7b2264617461223a5b5b224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c224e534b2f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c224e534b2f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c224e534b2f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c224e534b2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224351522f35202831303078295c74222c224351522f35202831303078295c74222c224351522f35202831303078295c74222c224351522f35202831303078295c74222c224351522f35202831303078295c74222c224351522f35202831303078295c74222c224351522f35202831303078295c74222c224351522f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c224351522f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224351522f31202831303078295c74222c224351522f31202831303078295c74222c224351522f31202831303078295c74222c224351522f31202831303078295c74222c224351522f31202831303078295c74222c224351522f31202831303078295c74222c224351522f31202831303078295c74222c224351522f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c224351522f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-05-30 12:40:54.159506	2019-05-30 12:40:54.162167	1	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'cqr/1':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'cqr/5':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'mix':27,31,61,65,95,99,129,133 'nsk/3':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'nsk/5':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		1
8	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-05-30 12:40:54.198005	2019-05-30 12:40:54.200704	1	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		1
9	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-05-30 12:40:54.250135	2019-05-30 12:40:54.252716	1	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		1
10	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-08-27 15:15:02.674561	2019-08-27 15:15:02.674561	4	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	3
28	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-08-28 08:34:56.191903	2019-08-28 08:34:56.191903	6	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	4
11	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-08-27 15:15:03.005517	2019-08-27 15:15:03.005517	4	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		3
163	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-20 14:21:20.905399	2021-05-20 14:21:20.905399	24	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	19
164	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-20 14:21:20.92611	2021-05-20 14:21:20.92611	24	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		19
13	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-08-27 15:15:03.47592	2019-08-27 15:15:03.47592	4	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	3
15	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-08-27 15:15:03.555668	2019-08-27 15:15:03.555668	4	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	3
166	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-20 14:21:21.076977	2021-05-20 14:21:21.076977	24	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	19
165	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-20 14:21:21.000265	2021-05-20 14:21:21.002222	24	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		19
210	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-08-23 14:58:28.529557	2021-08-23 14:58:28.532256	30	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		24
212	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-08-23 14:58:28.692435	2021-08-23 14:58:28.694848	30	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		24
214	\\x7b2264617461223a5b5b225353492f31202831303078295c74222c225353492f31202831303078295c74222c225353492f31202831303078295c74222c225353492f31202831303078295c74222c225353492f31202831303078295c74222c225353492f31202831303078295c74222c225353492f31202831303078295c74222c225353492f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c225353492f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c22454a552f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c22454a552f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c22454a552f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c22454a552f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225353492f34202831303078295c74222c225353492f34202831303078295c74222c225353492f34202831303078295c74222c225353492f34202831303078295c74222c225353492f34202831303078295c74222c225353492f34202831303078295c74222c225353492f34202831303078295c74222c225353492f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c225353492f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-08-23 14:58:28.778273	2021-08-23 14:58:28.779936	30	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'eju/3':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'eju/4':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'ssi/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'ssi/4':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242		24
215	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-08-23 14:58:28.817608	2021-08-23 14:58:28.819962	30	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		24
216	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-08-23 14:58:28.868782	2021-08-23 14:58:28.870544	30	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		24
12	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-08-27 15:15:03.295217	2019-08-27 15:15:03.300015	4	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		3
14	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-08-27 15:15:03.503195	2019-08-27 15:15:03.505718	4	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		3
16	\\x7b2264617461223a5b5b2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c2250425a2f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c2250425a2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c2250425a2f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c2250425a2f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c2250425a2f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c2250425a2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c2250425a2f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c2250425a2f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-27 15:15:03.646205	2019-08-27 15:15:03.649017	4	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'pbz/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'pbz/3':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'pbz/4':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'pbz/5':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'smpl':28,32,62,66,96,100,130,134		3
17	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-08-27 15:15:03.715338	2019-08-27 15:15:03.717715	4	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		3
18	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-27 15:15:03.782604	2019-08-27 15:15:03.785013	4	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		3
19	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-08-28 07:31:00.456855	2019-08-28 07:31:00.456855	1	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	2
20	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-08-28 07:31:00.535475	2019-08-28 07:31:00.535475	1	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		2
21	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-08-28 07:31:00.839302	2019-08-28 07:31:00.842739	1	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		2
22	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-08-28 07:31:01.169424	2019-08-28 07:31:01.169424	1	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	2
23	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-08-28 07:31:01.198182	2019-08-28 07:31:01.200385	1	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		2
29	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-08-28 08:34:56.280996	2019-08-28 08:34:56.280996	6	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		4
36	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-28 08:34:57.252647	2019-08-28 08:34:57.254976	6	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		4
168	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-20 14:21:21.127571	2021-05-20 14:21:21.127571	24	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	19
24	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-08-28 07:31:01.251218	2019-08-28 07:31:01.251218	1	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	2
25	\\x7b2264617461223a5b5b224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c224b474d2f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c224b474d2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b225158452f33202831303078295c74222c225158452f33202831303078295c74222c225158452f33202831303078295c74222c225158452f33202831303078295c74222c225158452f33202831303078295c74222c225158452f33202831303078295c74222c225158452f33202831303078295c74222c225158452f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c225158452f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c224b474d2f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c224b474d2f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225158452f35202831303078295c74222c225158452f35202831303078295c74222c225158452f35202831303078295c74222c225158452f35202831303078295c74222c225158452f35202831303078295c74222c225158452f35202831303078295c74222c225158452f35202831303078295c74222c225158452f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c225158452f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-28 07:31:01.333704	2019-08-28 07:31:01.335971	1	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'kgm/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'kgm/3':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'qxe/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'qxe/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'smpl':28,32,62,66,96,100,130,134		2
26	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-08-28 07:31:01.413271	2019-08-28 07:31:01.415478	1	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		2
27	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-28 07:31:01.535818	2019-08-28 07:31:01.538716	1	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		2
31	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-08-28 08:34:56.640236	2019-08-28 08:34:56.640236	6	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	4
167	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-20 14:21:21.092979	2021-05-20 14:21:21.094776	24	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		19
37	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-08-29 12:51:26.252223	2019-08-29 12:51:26.252223	7	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	5
74	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-11-25 18:00:53.179564	2019-11-25 18:00:53.179564	10	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		9
33	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-08-28 08:34:56.724482	2019-08-28 08:34:56.724482	6	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	4
32	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-08-28 08:34:56.670377	2019-08-28 08:34:56.67262	6	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		4
217	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-08-23 17:19:39.585679	2021-08-23 17:19:39.585679	31	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	25
42	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-08-29 12:51:27.636922	2019-08-29 12:51:27.636922	7	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	5
218	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-08-23 17:19:39.607428	2021-08-23 17:19:39.607428	31	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		25
220	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-08-23 17:19:39.775953	2021-08-23 17:19:39.775953	31	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	25
30	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-08-28 08:34:56.501269	2019-08-28 08:34:56.50605	6	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		4
34	\\x7b2264617461223a5b5b224444492f34202831303078295c74222c224444492f34202831303078295c74222c224444492f34202831303078295c74222c224444492f34202831303078295c74222c224444492f34202831303078295c74222c224444492f34202831303078295c74222c224444492f34202831303078295c74222c224444492f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c224444492f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224444492f33202831303078295c74222c224444492f33202831303078295c74222c224444492f33202831303078295c74222c224444492f33202831303078295c74222c224444492f33202831303078295c74222c224444492f33202831303078295c74222c224444492f33202831303078295c74222c224444492f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c224444492f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b225344552f31202831303078295c74222c225344552f31202831303078295c74222c225344552f31202831303078295c74222c225344552f31202831303078295c74222c225344552f31202831303078295c74222c225344552f31202831303078295c74222c225344552f31202831303078295c74222c225344552f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c225344552f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225344552f34202831303078295c74222c225344552f34202831303078295c74222c225344552f34202831303078295c74222c225344552f34202831303078295c74222c225344552f34202831303078295c74222c225344552f34202831303078295c74222c225344552f34202831303078295c74222c225344552f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c225344552f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-28 08:34:56.870382	2019-08-28 08:34:56.890038	6	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'ddi/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'ddi/4':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'sdu/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'sdu/4':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'smpl':28,32,62,66,96,100,130,134		4
35	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-08-28 08:34:57.147907	2019-08-28 08:34:57.152893	6	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		4
73	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-11-25 18:00:53.129834	2019-11-25 18:00:53.129834	10	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	9
169	\\x7b2264617461223a5b5b22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c22554b412f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c22554b412f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c22564f462f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c22564f462f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c22564f462f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c22564f462f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c22554b412f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c22554b412f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:21:21.168785	2021-05-20 14:21:21.170485	24	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'uka/3':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'uka/4':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'vof/4':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'vof/5':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186		19
38	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-08-29 12:51:26.550931	2019-08-29 12:51:26.550931	7	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		5
40	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-08-29 12:51:27.338226	2019-08-29 12:51:27.338226	7	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	5
170	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-20 14:21:21.221464	2021-05-20 14:21:21.227615	24	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		19
39	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-08-29 12:51:26.966826	2019-08-29 12:51:26.979902	7	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		5
41	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-08-29 12:51:27.382646	2019-08-29 12:51:27.384921	7	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		5
43	\\x7b2264617461223a5b5b224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c224b4f552f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c224b4f552f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b225255422f32202831303078295c74222c225255422f32202831303078295c74222c225255422f32202831303078295c74222c225255422f32202831303078295c74222c225255422f32202831303078295c74222c225255422f32202831303078295c74222c225255422f32202831303078295c74222c225255422f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c225255422f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b225255422f33202831303078295c74222c225255422f33202831303078295c74222c225255422f33202831303078295c74222c225255422f33202831303078295c74222c225255422f33202831303078295c74222c225255422f33202831303078295c74222c225255422f33202831303078295c74222c225255422f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c225255422f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c224b4f552f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c224b4f552f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-29 12:51:28.080645	2019-08-29 12:51:28.083065	7	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'kou/3':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'kou/4':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'rub/2':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'rub/3':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'smpl':28,32,62,66,96,100,130,134		5
44	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-08-29 12:51:28.579285	2019-08-29 12:51:28.610317	7	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		5
45	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-08-29 12:51:28.960173	2019-08-29 12:51:28.974469	7	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		5
46	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-09-12 18:26:18.970962	2019-09-12 18:26:18.970962	8	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	6
47	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-09-12 18:26:19.034888	2019-09-12 18:26:19.034888	8	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		6
49	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-09-12 18:26:19.282645	2019-09-12 18:26:19.282645	8	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	6
76	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-11-25 18:00:53.550082	2019-11-25 18:00:53.550082	10	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	9
171	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:21:21.334819	2021-05-20 14:21:21.339516	24	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		19
51	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-09-12 18:26:19.352592	2019-09-12 18:26:19.352592	8	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	6
222	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-08-23 17:19:39.841191	2021-08-23 17:19:39.841191	31	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	25
78	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-11-25 18:00:53.65998	2019-11-25 18:00:53.65998	10	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	9
48	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-09-12 18:26:19.155816	2019-09-12 18:26:19.1593	8	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		6
50	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-09-12 18:26:19.303488	2019-09-12 18:26:19.305749	8	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		6
101	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2020-10-07 19:20:26.714587	2020-10-07 19:20:26.714587	16	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		13
52	\\x7b2264617461223a5b5b22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c22554e412f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c22554e412f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c22554e412f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c22554e412f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c22575a562f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c22575a562f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c22554e412f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c22554e412f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-09-12 18:26:19.406646	2019-09-12 18:26:19.409193	8	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'una/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'una/3':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'una/5':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'wzv/5':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186		6
53	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-09-12 18:26:19.455235	2019-09-12 18:26:19.457479	8	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		6
54	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-09-12 18:26:19.544387	2019-09-12 18:26:19.547412	8	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		6
55	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-11-13 13:57:42.565048	2019-11-13 13:57:42.565048	9	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	8
172	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-20 14:30:30.643438	2021-05-20 14:30:30.643438	25	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	20
56	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-11-13 13:57:42.650068	2019-11-13 13:57:42.650068	9	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		8
173	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-20 14:30:30.663135	2021-05-20 14:30:30.663135	25	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		20
58	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-11-13 13:57:43.014615	2019-11-13 13:57:43.014615	9	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	8
175	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-20 14:30:30.84272	2021-05-20 14:30:30.84272	25	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	20
60	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-11-13 13:57:43.111906	2019-11-13 13:57:43.111906	9	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	8
63	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-11-13 13:57:43.398503	2019-11-13 13:57:43.401675	9	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		8
177	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-20 14:30:30.911929	2021-05-20 14:30:30.911929	25	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	20
75	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-11-25 18:00:53.378387	2019-11-25 18:00:53.381733	10	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		9
77	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-11-25 18:00:53.585016	2019-11-25 18:00:53.590651	10	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		9
79	\\x7b2264617461223a5b5b225646482f32202831303078295c74222c225646482f32202831303078295c74222c225646482f32202831303078295c74222c225646482f32202831303078295c74222c225646482f32202831303078295c74222c225646482f32202831303078295c74222c225646482f32202831303078295c74222c225646482f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c225646482f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b225646482f33202831303078295c74222c225646482f33202831303078295c74222c225646482f33202831303078295c74222c225646482f33202831303078295c74222c225646482f33202831303078295c74222c225646482f33202831303078295c74222c225646482f33202831303078295c74222c225646482f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c225646482f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b225643482f33202831303078295c74222c225643482f33202831303078295c74222c225643482f33202831303078295c74222c225643482f33202831303078295c74222c225643482f33202831303078295c74222c225643482f33202831303078295c74222c225643482f33202831303078295c74222c225643482f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c225643482f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225646482f31202831303078295c74222c225646482f31202831303078295c74222c225646482f31202831303078295c74222c225646482f31202831303078295c74222c225646482f31202831303078295c74222c225646482f31202831303078295c74222c225646482f31202831303078295c74222c225646482f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c225646482f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-11-25 18:00:53.748714	2019-11-25 18:00:53.751679	10	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'vch/3':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'vfh/1':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'vfh/2':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'vfh/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124		9
88	\\x7b2264617461223a5b5b225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c225a48492f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c225a48492f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c22554d412f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c22554d412f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c22554d412f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c22554d412f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c22554d412f32202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c22554d412f3220283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-05-12 16:13:06.780074	2020-05-12 16:13:06.782694	11	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'uma/1':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'uma/2':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'uma/3':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'zhi/5':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56		10
57	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-11-13 13:57:42.829576	2019-11-13 13:57:42.834408	9	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		8
59	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-11-13 13:57:43.043718	2019-11-13 13:57:43.046461	9	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		8
61	\\x7b2264617461223a5b5b225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c225a56452f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c225a56452f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224942552f33202831303078295c74222c224942552f33202831303078295c74222c224942552f33202831303078295c74222c224942552f33202831303078295c74222c224942552f33202831303078295c74222c224942552f33202831303078295c74222c224942552f33202831303078295c74222c224942552f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c224942552f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c225a56452f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c225a56452f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c225a56452f32202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c225a56452f3220283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-11-13 13:57:43.216674	2019-11-13 13:57:43.219345	9	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'ibu/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'zve/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'zve/2':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'zve/3':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56		8
62	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-11-13 13:57:43.302076	2019-11-13 13:57:43.30543	9	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		8
103	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2020-10-07 19:20:27.162719	2020-10-07 19:20:27.162719	16	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	13
224	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-08-23 17:19:39.942495	2021-08-23 17:19:39.945254	31	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		25
64	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2019-11-25 14:09:27.20469	2019-11-25 14:09:27.20469	4	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	7
65	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2019-11-25 14:09:27.313926	2019-11-25 14:09:27.313926	4	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		7
66	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2019-11-25 14:09:27.917046	2019-11-25 14:09:27.925247	4	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		7
67	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2019-11-25 14:09:28.315923	2019-11-25 14:09:28.315923	4	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	7
68	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2019-11-25 14:09:28.350529	2019-11-25 14:09:28.353257	4	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		7
69	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2019-11-25 14:09:28.439862	2019-11-25 14:09:28.439862	4	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	7
70	\\x7b2264617461223a5b5b224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c224d46572f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c224d46572f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c224d46572f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c224d46572f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c224d46572f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c224d46572f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224441422f35202831303078295c74222c224441422f35202831303078295c74222c224441422f35202831303078295c74222c224441422f35202831303078295c74222c224441422f35202831303078295c74222c224441422f35202831303078295c74222c224441422f35202831303078295c74222c224441422f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c224441422f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-11-25 14:09:28.539966	2019-11-25 14:09:28.54251	4	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'dab/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'mfw/1':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mfw/2':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'mfw/4':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		7
71	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-11-25 14:09:28.627079	2019-11-25 14:09:28.631519	4	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		7
176	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-20 14:30:30.860322	2021-05-20 14:30:30.862073	25	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		20
118	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2020-11-18 14:53:25.454024	2020-11-18 14:53:25.454024	17	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	14
72	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-11-25 14:09:28.724481	2019-11-25 14:09:28.727383	4	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		7
80	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2019-11-25 18:00:53.834538	2019-11-25 18:00:53.837501	10	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		9
81	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2019-11-25 18:00:53.926881	2019-11-25 18:00:53.929515	10	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		9
82	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2020-05-12 16:13:04.774345	2020-05-12 16:13:04.774345	11	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	10
83	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2020-05-12 16:13:05.026952	2020-05-12 16:13:05.026952	11	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		10
174	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-20 14:30:30.75228	2021-05-20 14:30:30.754269	25	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		20
85	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2020-05-12 16:13:06.582734	2020-05-12 16:13:06.582734	11	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	10
84	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2020-05-12 16:13:06.425683	2020-05-12 16:13:06.429073	11	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		10
86	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2020-05-12 16:13:06.607916	2020-05-12 16:13:06.611226	11	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		10
87	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2020-05-12 16:13:06.692278	2020-05-12 16:13:06.692278	11	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	10
178	\\x7b2264617461223a5b5b224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c224b4f4c2f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c224b4f4c2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c224b4f4c2f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c224b4f4c2f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c224f46492f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c224f46492f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c224b4f4c2f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c224b4f4c2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:30:30.969337	2021-05-20 14:30:30.971771	25	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'kol/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'kol/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'kol/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'ofi/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'smpl':28,32,62,66,96,100,130,134		20
179	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-20 14:30:31.008562	2021-05-20 14:30:31.010234	25	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		20
180	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:30:31.067275	2021-05-20 14:30:31.068924	25	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		20
89	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2020-05-12 16:13:06.88428	2020-05-12 16:13:06.886793	11	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		10
90	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-05-12 16:13:06.97496	2020-05-12 16:13:06.977537	11	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		10
91	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2020-06-10 14:43:17.974044	2020-06-10 14:43:17.974044	12	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	11
181	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-20 15:17:12.066318	2021-05-20 15:17:12.066318	26	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	21
92	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2020-06-10 14:43:18.08262	2020-06-10 14:43:18.08262	12	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		11
182	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-20 15:17:12.087082	2021-05-20 15:17:12.087082	26	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		21
94	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2020-06-10 14:43:18.427706	2020-06-10 14:43:18.427706	12	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	11
93	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2020-06-10 14:43:18.255211	2020-06-10 14:43:18.25969	12	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		11
100	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2020-10-07 19:20:26.618927	2020-10-07 19:20:26.618927	16	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	13
95	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2020-06-10 14:43:18.45103	2020-06-10 14:43:18.454259	12	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		11
184	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-20 15:17:12.235132	2021-05-20 15:17:12.235132	26	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	21
96	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2020-06-10 14:43:18.522879	2020-06-10 14:43:18.522879	12	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	11
186	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-20 15:17:12.294766	2021-05-20 15:17:12.294766	26	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	21
104	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2020-10-07 19:20:27.191913	2020-10-07 19:20:27.194878	16	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		13
97	\\x7b2264617461223a5b5b22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c22474a482f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c22474a482f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c224c495a2f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c224c495a2f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c22474a482f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c22474a482f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c224c495a2f32202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c224c495a2f3220283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-06-10 14:43:18.591723	2020-06-10 14:43:18.594321	12	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'gjh/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'gjh/4':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'liz/2':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'liz/4':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		11
98	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2020-06-10 14:43:18.658935	2020-06-10 14:43:18.661597	12	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		11
99	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-06-10 14:43:18.734313	2020-06-10 14:43:18.737204	12	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		11
119	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2020-11-18 14:53:25.499483	2020-11-18 14:53:25.499483	17	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		14
121	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2020-11-18 14:53:25.805545	2020-11-18 14:53:25.805545	17	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	14
105	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2020-10-07 19:20:27.25883	2020-10-07 19:20:27.25883	16	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	13
183	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-20 15:17:12.15923	2021-05-20 15:17:12.16106	26	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		21
185	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-20 15:17:12.251508	2021-05-20 15:17:12.253321	26	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		21
187	\\x7b2264617461223a5b5b22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c22504c4b2f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c22504c4b2f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c22504c4b2f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c22504c4b2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c22504c4b2f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c22504c4b2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c224e57582f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c224e57582f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 15:17:12.340916	2021-05-20 15:17:12.34357	26	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'nwx/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'plk/1':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'plk/4':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'plk/5':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'smpl':28,32,62,66,96,100,130,134		21
188	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-20 15:17:12.381063	2021-05-20 15:17:12.382974	26	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		21
189	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 15:17:12.429891	2021-05-20 15:17:12.43179	26	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		21
102	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2020-10-07 19:20:26.938901	2020-10-07 19:20:26.944288	16	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		13
130	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-19 20:01:52.673765	2021-05-19 20:01:52.673765	20	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	15
157	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-20 14:13:34.521571	2021-05-20 14:13:34.521571	23	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	18
225	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-08-23 17:19:40.01195	2021-08-23 17:19:40.013768	31	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		25
106	\\x7b2264617461223a5b5b2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c2254594d2f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c2254594d2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c2254594d2f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c2254594d2f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224351582f32202831303078295c74222c224351582f32202831303078295c74222c224351582f32202831303078295c74222c224351582f32202831303078295c74222c224351582f32202831303078295c74222c224351582f32202831303078295c74222c224351582f32202831303078295c74222c224351582f32202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c224351582f3220283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224351582f33202831303078295c74222c224351582f33202831303078295c74222c224351582f33202831303078295c74222c224351582f33202831303078295c74222c224351582f33202831303078295c74222c224351582f33202831303078295c74222c224351582f33202831303078295c74222c224351582f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c224351582f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-10-07 19:20:27.332834	2020-10-07 19:20:27.335611	16	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'cqx/2':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'cqx/3':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134 'tym/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'tym/4':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124		13
107	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2020-10-07 19:20:27.500672	2020-10-07 19:20:27.505045	16	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		13
108	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-10-07 19:20:27.612923	2020-10-07 19:20:27.616246	16	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		13
109	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2020-11-18 14:50:53.757205	2020-11-18 14:50:53.757205	1	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	12
190	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-28 15:07:52.985114	2021-05-28 15:07:52.985114	27	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	22
110	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2020-11-18 14:50:53.875111	2020-11-18 14:50:53.875111	1	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		12
111	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2020-11-18 14:50:54.17914	2020-11-18 14:50:54.182895	1	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		12
191	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-28 15:07:53.096726	2021-05-28 15:07:53.096726	27	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		22
112	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2020-11-18 14:50:54.393432	2020-11-18 14:50:54.393432	1	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	12
113	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2020-11-18 14:50:54.427049	2020-11-18 14:50:54.429887	1	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		12
193	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-28 15:07:55.295726	2021-05-28 15:07:55.295726	27	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	22
138	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-20 14:12:31.046682	2021-05-20 14:12:31.04927	21	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		16
114	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2020-11-18 14:50:54.507669	2020-11-18 14:50:54.507669	1	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	12
115	\\x7b2264617461223a5b5b2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c2247514e2f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c2247514e2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c22444b4f2f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c22444b4f2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c2247514e2f32202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c2247514e2f3220283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c2247514e2f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c2247514e2f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-11-18 14:50:54.593682	2020-11-18 14:50:54.596394	1	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'dko/1':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'gqn/2':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'gqn/3':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'gqn/5':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		12
116	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2020-11-18 14:50:54.700432	2020-11-18 14:50:54.703264	1	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		12
194	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-28 15:07:55.319201	2021-05-28 15:07:55.323427	27	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		22
117	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-11-18 14:50:54.817353	2020-11-18 14:50:54.820124	1	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		12
195	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-28 15:07:55.365798	2021-05-28 15:07:55.365798	27	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	22
140	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-20 14:12:31.159889	2021-05-20 14:12:31.161775	21	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		16
211	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-08-23 14:58:28.67558	2021-08-23 14:58:28.67558	30	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	24
123	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2020-11-18 14:53:25.905217	2020-11-18 14:53:25.905217	17	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	14
120	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2020-11-18 14:53:25.652004	2020-11-18 14:53:25.654904	17	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		14
122	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2020-11-18 14:53:25.835746	2020-11-18 14:53:25.839128	17	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		14
124	\\x7b2264617461223a5b5b2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c2246545a2f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c2246545a2f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c2246545a2f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c2246545a2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c2246545a2f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c2246545a2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224457422f32202831303078295c74222c224457422f32202831303078295c74222c224457422f32202831303078295c74222c224457422f32202831303078295c74222c224457422f32202831303078295c74222c224457422f32202831303078295c74222c224457422f32202831303078295c74222c224457422f32202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c224457422f3220283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-11-18 14:53:25.984668	2020-11-18 14:53:25.987812	17	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'dwb/2':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'ftz/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'ftz/2':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'ftz/5':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		14
125	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2020-11-18 14:53:26.044891	2020-11-18 14:53:26.047764	17	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		14
126	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2020-11-18 14:53:26.124052	2020-11-18 14:53:26.126641	17	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		14
127	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-19 20:01:52.334332	2021-05-19 20:01:52.334332	20	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	15
192	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-28 15:07:54.653068	2021-05-28 15:07:55.139791	27	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		22
128	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-19 20:01:52.407107	2021-05-19 20:01:52.407107	20	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		15
196	\\x7b2264617461223a5b5b224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c224a41582f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c224a41582f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224153452f33202831303078295c74222c224153452f33202831303078295c74222c224153452f33202831303078295c74222c224153452f33202831303078295c74222c224153452f33202831303078295c74222c224153452f33202831303078295c74222c224153452f33202831303078295c74222c224153452f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c224153452f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224153452f34202831303078295c74222c224153452f34202831303078295c74222c224153452f34202831303078295c74222c224153452f34202831303078295c74222c224153452f34202831303078295c74222c224153452f34202831303078295c74222c224153452f34202831303078295c74222c224153452f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c224153452f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c224a41582f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c224a41582f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-28 15:07:57.172152	2021-05-28 15:07:57.175157	27	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'ase/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'ase/4':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'jax/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'jax/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		22
197	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-28 15:07:57.83764	2021-05-28 15:07:57.839439	27	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		22
132	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-19 20:01:52.728692	2021-05-19 20:01:52.728692	20	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	15
198	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-28 15:07:58.082636	2021-05-28 15:07:58.084262	27	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		22
219	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-08-23 17:19:39.691315	2021-08-23 17:19:39.693683	31	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		25
221	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-08-23 17:19:39.792649	2021-08-23 17:19:39.795314	31	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		25
129	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-19 20:01:52.511934	2021-05-19 20:01:52.515553	20	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		15
131	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-19 20:01:52.689657	2021-05-19 20:01:52.692323	20	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		15
133	\\x7b2264617461223a5b5b2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c2249594a2f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c2249594a2f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c224c48442f35202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c224c48442f3520283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c224c48442f33202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c224c48442f3320283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c224c48442f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c224c48442f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-19 20:01:52.774665	2021-05-19 20:01:52.77644	20	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'iyj/2':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'lhd/3':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'lhd/4':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'lhd/5':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		15
142	\\x7b2264617461223a5b5b22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c22444a582f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c22444a582f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c224a5a462f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c224a5a462f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c224a5a462f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c224a5a462f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c22444a582f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c22444a582f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:12:31.246934	2021-05-20 14:12:31.248767	21	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'djx/1':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'djx/4':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'jzf/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'jzf/4':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		16
134	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-19 20:01:52.812758	2021-05-19 20:01:52.815229	20	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		15
135	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-19 20:01:52.860873	2021-05-19 20:01:52.862898	20	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		15
136	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-20 14:12:30.909987	2021-05-20 14:12:30.909987	21	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	16
199	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-06-02 16:10:03.930518	2021-06-02 16:10:03.930518	28	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	23
137	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-20 14:12:30.944533	2021-05-20 14:12:30.944533	21	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		16
200	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-06-02 16:10:04.21257	2021-06-02 16:10:04.21257	28	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		23
139	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-20 14:12:31.142773	2021-05-20 14:12:31.142773	21	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	16
202	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-06-02 16:10:05.154001	2021-06-02 16:10:05.154001	28	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	23
141	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-20 14:12:31.198659	2021-05-20 14:12:31.198659	21	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	16
204	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-06-02 16:10:06.642604	2021-06-02 16:10:06.642604	28	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	23
143	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-20 14:12:31.289505	2021-05-20 14:12:31.293949	21	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		16
144	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:12:31.344922	2021-05-20 14:12:31.34694	21	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		16
145	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-20 14:12:57.535421	2021-05-20 14:12:57.535421	22	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	17
146	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-20 14:12:57.553617	2021-05-20 14:12:57.553617	22	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		17
148	\\x7b2264617461223a5b5b22416d6f756e74206f66207374617274696e67206d6174657269616c20286d6729222c22566f6c756d65206f662042756666657220524c542028c2b56c29225d2c5b223c3230222c22333530225d2c5b223230202d203330222c22363030225d5d7d	2021-05-20 14:12:57.696655	2021-05-20 14:12:57.696655	22	\N	 '20':12,14 '265l':11 '30':15 '302':10 '350':13 '600':16 'amount':1 'buffer':8 'materi':4 'mg':5 'rlt':9 'start':3 'volum':6	Volumes of Buffer RLT for tissue disruption and homogenization	17
150	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-20 14:12:57.744243	2021-05-20 14:12:57.744243	22	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	17
201	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-06-02 16:10:04.683042	2021-06-02 16:10:04.691884	28	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		23
203	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-06-02 16:10:06.491623	2021-06-02 16:10:06.497758	28	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		23
205	\\x7b2264617461223a5b5b224247482f33202831303078295c74222c224247482f33202831303078295c74222c224247482f33202831303078295c74222c224247482f33202831303078295c74222c224247482f33202831303078295c74222c224247482f33202831303078295c74222c224247482f33202831303078295c74222c224247482f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c224247482f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c22504e4b2f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c22504e4b2f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c22504e4b2f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c22504e4b2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c22504e4b2f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c22504e4b2f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-06-02 16:10:06.8851	2021-06-02 16:10:06.889793	28	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'bgh/3':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'pnk/2':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'pnk/4':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'pnk/5':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'smpl':28,32,62,66,96,100,130,134		23
155	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-05-20 14:13:34.35794	2021-05-20 14:13:34.35794	23	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		18
209	\\x7b2264617461223a5b5b224e616d65222c2253616d706c652074797065225d2c5b224d6f636b2031647069222c22706f7461746f65206c6561766573225d2c5b224d6f636b2036647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2031647069222c22706f7461746f65206c6561766573225d2c5b22505659204e544e2036647069222c22706f7461746f65206c6561766573225d5d7d	2021-08-23 14:58:28.404352	2021-08-23 14:58:28.404352	30	\N	 '1dpi':5,14 '6dpi':9,19 'leav':7,11,16,21 'mock':4,8 'name':1 'ntn':13,18 'potato':6,10,15,20 'pvi':12,17 'sampl':2 'type':3		24
206	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-06-02 16:10:07.123951	2021-06-02 16:10:07.129859	28	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		23
207	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-06-02 16:10:07.408639	2021-06-02 16:10:07.413336	28	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		23
223	\\x7b2264617461223a5b5b224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c224f424e2f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c224f424e2f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c224f4e512f33202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c224f4e512f3320283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c224f4e512f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c224f4e512f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c224f4e512f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c224f4e512f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-08-23 17:19:39.895056	2021-08-23 17:19:39.897732	31	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'obn/2':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'onq/1':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'onq/3':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'onq/5':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'smpl':28,32,62,66,96,100,130,134		25
147	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-20 14:12:57.628432	2021-05-20 14:12:57.63097	22	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		17
149	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-20 14:12:57.712179	2021-05-20 14:12:57.713784	22	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		17
151	\\x7b2264617461223a5b5b224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c224e45442f34202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c224e45442f3420283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224253482f32202831303078295c74222c224253482f32202831303078295c74222c224253482f32202831303078295c74222c224253482f32202831303078295c74222c224253482f32202831303078295c74222c224253482f32202831303078295c74222c224253482f32202831303078295c74222c224253482f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c224253482f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c224e45442f31202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c224e45442f3120283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c224e45442f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c224e45442f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:12:57.785405	2021-05-20 14:12:57.787266	22	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'bsh/2':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'mix':27,31,61,65,95,99,129,133 'ned/1':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'ned/4':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'ned/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		17
152	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-20 14:12:57.824134	2021-05-20 14:12:57.825943	22	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		17
153	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:12:57.873431	2021-05-20 14:12:57.87534	22	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		17
154	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-05-20 14:13:34.340494	2021-05-20 14:13:34.340494	23	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	18
208	\\x7b2264617461223a5b5b2267726f75702f74696d65222c223120647069222c223620647069222c22222c22225d2c5b225056594e544e222c2231222c2231222c22222c22225d2c5b226d6f636b222c2231222c2231222c22222c22225d2c5b22222c22222c22222c22222c22225d5d7d	2021-08-23 14:58:28.342271	2021-08-23 14:58:28.342271	30	\N	 '1':2,7,8,10,11 '6':4 'dpi':3,5 'group/time':1 'mock':9 'pvyntn':6	Experiment design table	24
159	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-05-20 14:13:34.583899	2021-05-20 14:13:34.583899	23	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	18
213	\\x7b2264617461223a5b5b22446174653a222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b225265616374696f6e20766f6c756d653a222c22323020756c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22554d4d2f7265616374696f6e3a222c22313020c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b2273616d706c652f7265616374696f6e3a222c223620c2b56c222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22222c22225d2c5b224368656d6973747279222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22222c225461714d616e222c22222c22222c225359425220477265656e2049222c22222c22225d2c5b2247656e65222c22636f78222c22222c22222c2263616234222c22222c22222c225056592d4e544e222c22222c22222c2250522d31222c22222c22225d2c5b22222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e222c22636f6e632e2a222c2266696e616c20636f6e632e2a2a222c22c2b56c2f7265616374696f6e225d2c5b2250726f6265222c22313020c2b54d222c22313530206e4d222c22302e33222c222f222c222f222c222f222c223520c2b54d222c22313530206e4d222c22302e36222c222f222c222f222c222f225d2c5b224650222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d2c5b225250222c22333020c2b54d222c22333030206e4d222c22302e32222c22333020c2b54d222c22333030206e4d20222c22302e32222c22313020c2b54d222c22323530206e4d222c22302e35222c22333020c2b54d222c22333030206e4d20222c22302e32225d5d7d	2021-08-23 14:58:28.731136	2021-08-23 14:58:28.731136	30	\N	 '-1':30 '0.2':70,76,88,95,101,113 '0.3':57 '0.5':82,107 '0.6':63 '10':7,52,77,102 '150':55,61 '20':4 '250':80,105 '265l':9,13 '265l/reaction':35,40,45,50 '265m':54,60,67,73,79,85,92,98,104,110 '30':65,71,83,90,96,108 '300':68,74,86,93,99,111 '302':8,12,34,39,44,49,53,59,66,72,78,84,91,97,103,109 '5':58 '6':11 'cab4':25 'chemistri':14 'conc':31,33,36,38,41,43,46,48 'cox':24 'date':1 'final':32,37,42,47 'fp':64 'gene':23 'green':17,21 'nm':56,62,69,75,81,87,94,100,106,112 'ntn':28 'pr':29 'probe':51 'pvi':27 'pvy-ntn':26 'reaction':2 'rp':89 'sample/reaction':10 'sybr':16,20 'taqman':15,19 'ul':5 'umm/reaction':6 'volum':3	Realtime mastermix preparation - gene expression	24
156	\\x7b2264617461223a5b5b2253616d706c65206e616d65222c226e672f756c20222c223236302f32383020222c223236302f32333020225d2c5b22505659204e544e2036647069222c22312e3035222c22312e3339222c22312e3631225d2c5b22505659204e544e2031647069222c223332302e3532222c22322e3136222c22322e3236225d2c5b224d6f636b2036647069222c223930362e3139222c22322e3135222c22322e3236225d2c5b224d6f636b2031647069222c223934302e3232222c22322e3137222c22322e3237225d5d7d	2021-05-20 14:13:34.448999	2021-05-20 14:13:34.451003	23	\N	 '1.05':9 '1.39':10 '1.61':11 '1dpi':14,24 '2.15':21 '2.16':16 '2.17':26 '2.26':17,22 '2.27':27 '260/230':5 '260/280':4 '320.52':15 '6dpi':8,19 '906.19':20 '940.22':25 'mock':18,23 'name':2 'ng/ul':3 'ntn':7,13 'pvi':6,12 'sampl':1		18
158	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920225d2c5b2231222c22302c32225d2c5b2232222c22302c35225d2c5b2233222c22302c34225d2c5b2234222c22312c31225d2c5b2235222c2231322c35225d2c5b2236222c22302c3035225d2c5b2237222c2230225d2c5b2238222c2230225d2c5b2239222c2230225d2c5b223130222c2230225d2c5b223131222c2230225d2c5b223132222c22342c37225d2c5b223133222c22322c33225d2c5b223134222c22352c36225d2c5b223135222c22322c36225d2c5b223136222c22382c39225d2c5b223137222c22312c37225d2c5b223138222c2231225d5d7d	2021-05-20 14:13:34.539286	2021-05-20 14:13:34.540917	23	\N	 '0':5,8,11,20,23,25,27,29,31 '05':21 '1':2,4,14,15,48,51 '10':28 '11':30 '12':17,32 '13':35 '14':38 '15':41 '16':44 '17':47 '18':50 '2':6,7,36,42 '3':10,37 '4':12,13,33 '5':9,16,18,39 '6':19,40,43 '7':22,34,49 '8':24,45 '9':26,46 'quantiti':3 'sampl':1		18
160	\\x7b2264617461223a5b5b22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c22444f4c2f31202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313078295c74222c224d697820736d706c2028313078295c74225d2c5b22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c22444f4c2f3120283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c202831303078295c74222c224d697820736d706c202831303078295c74225d2c5b224355432f32202831303078295c74222c224355432f32202831303078295c74222c224355432f32202831303078295c74222c224355432f32202831303078295c74222c224355432f32202831303078295c74222c224355432f32202831303078295c74222c224355432f32202831303078295c74222c224355432f32202831303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c20283130303078295c74222c224d697820736d706c20283130303078295c74225d2c5b224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c224355432f3220283130303078295c74222c6e756c6c2c6e756c6c2c224d697820736d706c2028313030303078295c74222c224d697820736d706c2028313030303078295c74225d2c5b22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c22444f4c2f34202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c22444f4c2f3420283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c22444f4c2f35202831303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c22444f4c2f3520283130303078295c74222c6e756c6c2c6e756c6c2c224e544331222c224e544332225d2c5b6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:13:34.631204	2021-05-20 14:13:34.632868	23	\N	 '10000x':131,135 '1000x':36,39,42,45,48,51,54,57,97,101,104,107,110,113,116,119,122,125,166,169,172,175,178,181,184,187,222,225,228,231,234,237,240,243 '100x':2,5,8,11,14,17,20,23,63,67,70,73,76,79,82,85,88,91,138,141,144,147,150,153,156,159,194,197,200,203,206,209,212,215 '10x':29,33 'cuc/2':69,72,75,78,81,84,87,90,103,106,109,112,115,118,121,124 'dol/1':1,4,7,10,13,16,19,22,35,38,41,44,47,50,53,56 'dol/4':137,140,143,146,149,152,155,158,165,168,171,174,177,180,183,186 'dol/5':193,196,199,202,205,208,211,214,221,224,227,230,233,236,239,242 'mix':27,31,61,65,95,99,129,133 'ntc1':163,191,219,247 'ntc2':164,192,220,248 'null':25,26,59,60,93,94,127,128,161,162,189,190,217,218,245,246,249,250,251,252,253,254,255,256,257,258,259,260 'smpl':28,32,62,66,96,100,130,134		18
161	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c22436f6c756d6e2033222c22436f6c756d6e2034225d2c5b2232222c22302c32222c22302c34222c22302c3038225d2c5b2232222c22302c35222c2231222c22302c35225d2c5b2233222c22302c34222c22312c32222c22302c3438225d2c5b2234222c22312c31222c22342c34222c22342c3834225d2c5b2235222c2231322c35222c2236322c35222c223738312c3235225d2c5b2236222c22302c3035222c22302c33222c22302c303135225d2c5b2237222c2230222c2234222c2230225d2c5b2238222c2230222c2230222c2230225d2c5b2239222c2230222c2230222c2230225d2c5b223130222c2230222c2230222c2230225d2c5b223131222c2230222c2230222c2230225d2c5b223132222c22342c37222c2235362c34222c223236352c3038225d2c5b223133222c22322c33222c223330222c223639225d2c5b223134222c22352c36222c2237382c34222c223433392c3034225d2c5b223135222c22322c36222c223339222c223130312c34225d2c5b223136222c22382c39222c223134322c34222c22313236372c3336225d2c5b223137222c22312c37222c2232382c39222c2234392c3133225d2c5b223138222c2231222c223138222c223138225d2c5b22222c22222c22222c22225d5d7d	2021-05-20 14:13:34.673129	2021-05-20 14:13:34.675022	23	\N	 '0':9,11,13,16,19,22,26,43,45,47,50,52,54,55,56,58,59,60,62,63,64,66,67,68 '015':48 '04':87 '05':44 '08':14,75 '1':2,18,24,29,30,102,109 '10':61 '101':92 '11':65 '12':36,69 '1267':99 '13':76,107 '14':81 '142':97 '15':88 '16':94 '17':101 '18':108,110,111 '2':8,10,15,25,77,89 '25':41 '265':74 '28':104 '3':5,21,46,78 '30':79 '36':100 '39':91 '4':7,12,23,28,31,32,33,51,70,73,85,93,98 '439':86 '48':27 '49':106 '5':17,20,35,37,39,82 '56':72 '6':42,83,90 '62':38 '69':80 '7':49,71,103 '78':84 '781':40 '8':53,95 '84':34 '9':57,96,105 'column':4,6 'quantiti':3 'sampl':1		18
162	\\x7b2264617461223a5b5b2253616d706c652031222c225175616e7469747920222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2231222c22302c32222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2232222c22302c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2233222c22302c34222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2234222c22312c31222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2235222c2231322c35222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2236222c22302c3035222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2237222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2238222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b2239222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223130222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223131222c2230222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223132222c22342c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223133222c22322c33222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223134222c22352c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223135222c22322c36222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223136222c22382c39222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223137222c22312c37222c6e756c6c2c6e756c6c2c6e756c6c5d2c5b223138222c2231222c6e756c6c2c6e756c6c2c6e756c6c5d5d7d	2021-05-20 14:13:34.730991	2021-05-20 14:13:34.732691	23	\N	 '0':8,14,20,38,44,49,54,59,64 '05':39 '1':2,7,26,27,99,105 '10':58 '11':63 '12':32,68 '13':74 '14':80 '15':86 '16':92 '17':98 '18':104 '2':9,13,75,87 '3':19,76 '4':21,25,69 '5':15,31,33,81 '6':37,82,88 '7':43,70,100 '8':48,93 '9':53,94 'null':4,5,6,10,11,12,16,17,18,22,23,24,28,29,30,34,35,36,40,41,42,45,46,47,50,51,52,55,56,57,60,61,62,65,66,67,71,72,73,77,78,79,83,84,85,89,90,91,95,96,97,101,102,103,106,107,108 'quantiti':3 'sampl':1		18
\.


--
-- Data for Name: repositories; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repositories (id, team_id, created_by_id, name, created_at, updated_at, discarded_at, permission_level, type, parent_id, status, selected, my_module_id, archived, archived_on, restored_on, archived_by_id, restored_by_id) FROM stdin;
1	1	1	Samples	2019-05-30 12:40:52.926189	2019-05-30 12:40:52.926189	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
2	3	4	Samples	2019-08-27 15:14:58.380494	2019-08-27 15:14:58.380494	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
3	2	1	Samples	2019-08-28 07:30:56.863636	2019-08-28 07:30:56.863636	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
4	4	6	Samples	2019-08-28 08:34:51.573642	2019-08-28 08:34:51.573642	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
5	5	7	Samples	2019-08-29 12:51:21.193561	2019-08-29 12:51:21.193561	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
6	5	7	Reagents	2019-08-30 16:08:39.809599	2019-08-30 16:08:39.809599	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
7	5	7	Filters	2019-08-30 16:10:22.548698	2019-08-30 16:10:22.548698	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
8	5	7	Extracts	2019-08-30 16:10:29.424376	2019-08-30 16:10:29.424376	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
9	6	8	Samples	2019-09-12 18:26:18.098683	2019-09-12 18:26:18.098683	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
10	8	9	Samples	2019-11-13 13:57:40.937886	2019-11-13 13:57:40.937886	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
11	7	4	Samples	2019-11-25 14:09:20.832875	2019-11-25 14:09:20.832875	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
12	9	10	Samples	2019-11-25 18:00:52.13728	2019-11-25 18:00:52.13728	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
13	10	11	Samples	2020-05-12 16:13:00.832734	2020-05-12 16:13:00.832734	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
14	11	12	Samples	2020-06-10 14:43:16.398409	2020-06-10 14:43:16.398409	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
15	13	16	Samples	2020-10-07 19:20:23.672694	2020-10-07 19:20:23.672694	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
16	12	1	Samples	2020-11-18 14:50:51.52832	2020-11-18 14:50:51.52832	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
17	14	17	Samples	2020-11-18 14:53:24.30604	2020-11-18 14:53:24.30604	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
18	15	20	Samples	2021-05-19 20:01:51.399233	2021-05-19 20:01:51.399233	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
19	16	21	Samples	2021-05-20 14:12:30.260382	2021-05-20 14:12:30.260382	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
20	17	22	Samples	2021-05-20 14:12:57.006671	2021-05-20 14:12:57.006671	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
21	18	23	Samples	2021-05-20 14:13:33.854997	2021-05-20 14:13:33.854997	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
22	19	24	Samples	2021-05-20 14:21:20.37627	2021-05-20 14:21:20.37627	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
23	20	25	Samples	2021-05-20 14:30:30.175753	2021-05-20 14:30:30.175753	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
24	21	26	Samples	2021-05-20 15:17:11.550634	2021-05-20 15:17:11.550634	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
25	22	27	Samples	2021-05-28 15:07:38.000743	2021-05-28 15:07:38.000743	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
26	23	28	Samples	2021-06-02 16:10:00.948019	2021-06-02 16:10:00.948019	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
27	24	30	Samples	2021-08-23 14:58:27.181068	2021-08-23 14:58:27.181068	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
28	25	31	Samples	2021-08-23 17:19:39.047803	2021-08-23 17:19:39.047803	\N	0	Repository	\N	\N	\N	\N	f	\N	\N	\N	\N
\.



--
-- Data for Name: repository_cells; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repository_cells (id, repository_row_id, repository_column_id, value_type, value_id, created_at, updated_at) FROM stdin;
1	1	1	RepositoryListValue	1	2019-05-30 12:40:53.028216	2019-05-30 12:40:53.028216
2	1	2	RepositoryListValue	2	2019-05-30 12:40:53.033902	2019-05-30 12:40:53.033902
3	2	1	RepositoryListValue	3	2019-05-30 12:40:53.040705	2019-05-30 12:40:53.040705
4	2	2	RepositoryListValue	4	2019-05-30 12:40:53.045948	2019-05-30 12:40:53.045948
5	3	1	RepositoryListValue	5	2019-05-30 12:40:53.052721	2019-05-30 12:40:53.052721
6	3	2	RepositoryListValue	6	2019-05-30 12:40:53.057877	2019-05-30 12:40:53.057877
7	4	1	RepositoryListValue	7	2019-05-30 12:40:53.064601	2019-05-30 12:40:53.064601
8	4	2	RepositoryListValue	8	2019-05-30 12:40:53.069761	2019-05-30 12:40:53.069761
9	5	1	RepositoryListValue	9	2019-05-30 12:40:53.076347	2019-05-30 12:40:53.076347
10	5	2	RepositoryListValue	10	2019-05-30 12:40:53.081486	2019-05-30 12:40:53.081486
11	6	3	RepositoryListValue	11	2019-08-27 15:14:58.781972	2019-08-27 15:14:58.781972
12	6	4	RepositoryListValue	12	2019-08-27 15:14:58.79477	2019-08-27 15:14:58.79477
13	7	3	RepositoryListValue	13	2019-08-27 15:14:58.821221	2019-08-27 15:14:58.821221
14	7	4	RepositoryListValue	14	2019-08-27 15:14:58.833072	2019-08-27 15:14:58.833072
15	8	3	RepositoryListValue	15	2019-08-27 15:14:58.848952	2019-08-27 15:14:58.848952
16	8	4	RepositoryListValue	16	2019-08-27 15:14:58.861088	2019-08-27 15:14:58.861088
17	9	3	RepositoryListValue	17	2019-08-27 15:14:58.878261	2019-08-27 15:14:58.878261
18	9	4	RepositoryListValue	18	2019-08-27 15:14:58.889953	2019-08-27 15:14:58.889953
19	10	3	RepositoryListValue	19	2019-08-27 15:14:58.948798	2019-08-27 15:14:58.948798
20	10	4	RepositoryListValue	20	2019-08-27 15:14:58.973268	2019-08-27 15:14:58.973268
21	11	5	RepositoryListValue	21	2019-08-28 07:30:58.210305	2019-08-28 07:30:58.210305
22	11	6	RepositoryListValue	22	2019-08-28 07:30:58.218971	2019-08-28 07:30:58.218971
23	12	5	RepositoryListValue	23	2019-08-28 07:30:58.230554	2019-08-28 07:30:58.230554
24	12	6	RepositoryListValue	24	2019-08-28 07:30:58.238411	2019-08-28 07:30:58.238411
25	13	5	RepositoryListValue	25	2019-08-28 07:30:58.250943	2019-08-28 07:30:58.250943
26	13	6	RepositoryListValue	26	2019-08-28 07:30:58.258723	2019-08-28 07:30:58.258723
27	14	5	RepositoryListValue	27	2019-08-28 07:30:58.275382	2019-08-28 07:30:58.275382
28	14	6	RepositoryListValue	28	2019-08-28 07:30:58.284147	2019-08-28 07:30:58.284147
29	15	5	RepositoryListValue	29	2019-08-28 07:30:58.295669	2019-08-28 07:30:58.295669
30	15	6	RepositoryListValue	30	2019-08-28 07:30:58.304231	2019-08-28 07:30:58.304231
31	16	7	RepositoryListValue	31	2019-08-28 08:34:51.692322	2019-08-28 08:34:51.692322
32	16	8	RepositoryListValue	32	2019-08-28 08:34:51.700392	2019-08-28 08:34:51.700392
33	17	7	RepositoryListValue	33	2019-08-28 08:34:51.711089	2019-08-28 08:34:51.711089
34	17	8	RepositoryListValue	34	2019-08-28 08:34:51.71819	2019-08-28 08:34:51.71819
35	18	7	RepositoryListValue	35	2019-08-28 08:34:51.737453	2019-08-28 08:34:51.737453
36	18	8	RepositoryListValue	36	2019-08-28 08:34:51.745329	2019-08-28 08:34:51.745329
37	19	7	RepositoryListValue	37	2019-08-28 08:34:51.787054	2019-08-28 08:34:51.787054
38	19	8	RepositoryListValue	38	2019-08-28 08:34:52.26257	2019-08-28 08:34:52.26257
39	20	7	RepositoryListValue	39	2019-08-28 08:34:52.408953	2019-08-28 08:34:52.408953
40	20	8	RepositoryListValue	40	2019-08-28 08:34:52.49562	2019-08-28 08:34:52.49562
51	27	11	RepositoryListValue	51	2019-09-12 18:26:18.201014	2019-09-12 18:26:18.201014
52	27	12	RepositoryListValue	52	2019-09-12 18:26:18.209282	2019-09-12 18:26:18.209282
53	28	11	RepositoryListValue	53	2019-09-12 18:26:18.218002	2019-09-12 18:26:18.218002
54	28	12	RepositoryListValue	54	2019-09-12 18:26:18.225104	2019-09-12 18:26:18.225104
55	29	11	RepositoryListValue	55	2019-09-12 18:26:18.233888	2019-09-12 18:26:18.233888
56	29	12	RepositoryListValue	56	2019-09-12 18:26:18.240807	2019-09-12 18:26:18.240807
57	30	11	RepositoryListValue	57	2019-09-12 18:26:18.249435	2019-09-12 18:26:18.249435
58	30	12	RepositoryListValue	58	2019-09-12 18:26:18.255959	2019-09-12 18:26:18.255959
59	31	11	RepositoryListValue	59	2019-09-12 18:26:18.263745	2019-09-12 18:26:18.263745
60	31	12	RepositoryListValue	60	2019-09-12 18:26:18.269917	2019-09-12 18:26:18.269917
61	32	13	RepositoryListValue	61	2019-11-13 13:57:41.112926	2019-11-13 13:57:41.112926
62	32	14	RepositoryListValue	62	2019-11-13 13:57:41.123413	2019-11-13 13:57:41.123413
63	33	13	RepositoryListValue	63	2019-11-13 13:57:41.136439	2019-11-13 13:57:41.136439
64	33	14	RepositoryListValue	64	2019-11-13 13:57:41.14508	2019-11-13 13:57:41.14508
65	34	13	RepositoryListValue	65	2019-11-13 13:57:41.157177	2019-11-13 13:57:41.157177
66	34	14	RepositoryListValue	66	2019-11-13 13:57:41.169793	2019-11-13 13:57:41.169793
67	35	13	RepositoryListValue	67	2019-11-13 13:57:41.184558	2019-11-13 13:57:41.184558
68	35	14	RepositoryListValue	68	2019-11-13 13:57:41.194284	2019-11-13 13:57:41.194284
69	36	13	RepositoryListValue	69	2019-11-13 13:57:41.206923	2019-11-13 13:57:41.206923
70	36	14	RepositoryListValue	70	2019-11-13 13:57:41.215508	2019-11-13 13:57:41.215508
71	37	15	RepositoryListValue	71	2019-11-25 14:09:21.311929	2019-11-25 14:09:21.311929
72	37	16	RepositoryListValue	72	2019-11-25 14:09:21.326993	2019-11-25 14:09:21.326993
73	38	15	RepositoryListValue	73	2019-11-25 14:09:21.350887	2019-11-25 14:09:21.350887
74	38	16	RepositoryListValue	74	2019-11-25 14:09:21.364699	2019-11-25 14:09:21.364699
75	39	15	RepositoryListValue	75	2019-11-25 14:09:21.385187	2019-11-25 14:09:21.385187
76	39	16	RepositoryListValue	76	2019-11-25 14:09:21.408252	2019-11-25 14:09:21.408252
77	40	15	RepositoryListValue	77	2019-11-25 14:09:21.428603	2019-11-25 14:09:21.428603
78	40	16	RepositoryListValue	78	2019-11-25 14:09:21.439932	2019-11-25 14:09:21.439932
79	41	15	RepositoryListValue	79	2019-11-25 14:09:21.464004	2019-11-25 14:09:21.464004
80	41	16	RepositoryListValue	80	2019-11-25 14:09:21.480961	2019-11-25 14:09:21.480961
81	42	17	RepositoryListValue	81	2019-11-25 18:00:52.190118	2019-11-25 18:00:52.190118
82	42	18	RepositoryListValue	82	2019-11-25 18:00:52.20066	2019-11-25 18:00:52.20066
83	43	17	RepositoryListValue	83	2019-11-25 18:00:52.211235	2019-11-25 18:00:52.211235
84	43	18	RepositoryListValue	84	2019-11-25 18:00:52.219062	2019-11-25 18:00:52.219062
85	44	17	RepositoryListValue	85	2019-11-25 18:00:52.229294	2019-11-25 18:00:52.229294
86	44	18	RepositoryListValue	86	2019-11-25 18:00:52.237453	2019-11-25 18:00:52.237453
87	45	17	RepositoryListValue	87	2019-11-25 18:00:52.246908	2019-11-25 18:00:52.246908
88	45	18	RepositoryListValue	88	2019-11-25 18:00:52.255323	2019-11-25 18:00:52.255323
89	46	17	RepositoryListValue	89	2019-11-25 18:00:52.266842	2019-11-25 18:00:52.266842
90	46	18	RepositoryListValue	90	2019-11-25 18:00:52.278384	2019-11-25 18:00:52.278384
91	47	19	RepositoryListValue	91	2020-05-12 16:13:01.138699	2020-05-12 16:13:01.138699
92	47	20	RepositoryListValue	92	2020-05-12 16:13:01.150361	2020-05-12 16:13:01.150361
93	48	19	RepositoryListValue	93	2020-05-12 16:13:01.163586	2020-05-12 16:13:01.163586
94	48	20	RepositoryListValue	94	2020-05-12 16:13:01.173521	2020-05-12 16:13:01.173521
95	49	19	RepositoryListValue	95	2020-05-12 16:13:01.187055	2020-05-12 16:13:01.187055
96	49	20	RepositoryListValue	96	2020-05-12 16:13:01.201104	2020-05-12 16:13:01.201104
97	50	19	RepositoryListValue	97	2020-05-12 16:13:01.217682	2020-05-12 16:13:01.217682
98	50	20	RepositoryListValue	98	2020-05-12 16:13:01.226966	2020-05-12 16:13:01.226966
99	51	19	RepositoryListValue	99	2020-05-12 16:13:01.244115	2020-05-12 16:13:01.244115
100	51	20	RepositoryListValue	100	2020-05-12 16:13:01.260235	2020-05-12 16:13:01.260235
101	52	21	RepositoryListValue	101	2020-06-10 14:43:16.556923	2020-06-10 14:43:16.556923
102	52	22	RepositoryListValue	102	2020-06-10 14:43:16.567506	2020-06-10 14:43:16.567506
103	53	21	RepositoryListValue	103	2020-06-10 14:43:16.57681	2020-06-10 14:43:16.57681
104	53	22	RepositoryListValue	104	2020-06-10 14:43:16.584317	2020-06-10 14:43:16.584317
105	54	21	RepositoryListValue	105	2020-06-10 14:43:16.593302	2020-06-10 14:43:16.593302
106	54	22	RepositoryListValue	106	2020-06-10 14:43:16.600594	2020-06-10 14:43:16.600594
107	55	21	RepositoryListValue	107	2020-06-10 14:43:16.609282	2020-06-10 14:43:16.609282
108	55	22	RepositoryListValue	108	2020-06-10 14:43:16.616008	2020-06-10 14:43:16.616008
109	56	21	RepositoryListValue	109	2020-06-10 14:43:16.624562	2020-06-10 14:43:16.624562
110	56	22	RepositoryListValue	110	2020-06-10 14:43:16.631635	2020-06-10 14:43:16.631635
111	57	23	RepositoryListValue	111	2020-10-07 19:20:24.375938	2020-10-07 19:20:24.375938
112	57	24	RepositoryListValue	112	2020-10-07 19:20:24.396395	2020-10-07 19:20:24.396395
113	58	23	RepositoryListValue	113	2020-10-07 19:20:24.413517	2020-10-07 19:20:24.413517
114	58	24	RepositoryListValue	114	2020-10-07 19:20:24.427542	2020-10-07 19:20:24.427542
115	59	23	RepositoryListValue	115	2020-10-07 19:20:24.446072	2020-10-07 19:20:24.446072
116	59	24	RepositoryListValue	116	2020-10-07 19:20:24.458568	2020-10-07 19:20:24.458568
117	60	23	RepositoryListValue	117	2020-10-07 19:20:24.475196	2020-10-07 19:20:24.475196
118	60	24	RepositoryListValue	118	2020-10-07 19:20:24.487206	2020-10-07 19:20:24.487206
119	61	23	RepositoryListValue	119	2020-10-07 19:20:24.505495	2020-10-07 19:20:24.505495
120	61	24	RepositoryListValue	120	2020-10-07 19:20:24.517902	2020-10-07 19:20:24.517902
121	62	25	RepositoryListValue	121	2020-11-18 14:50:51.744543	2020-11-18 14:50:51.744543
122	62	26	RepositoryListValue	122	2020-11-18 14:50:51.756761	2020-11-18 14:50:51.756761
123	63	25	RepositoryListValue	123	2020-11-18 14:50:51.77065	2020-11-18 14:50:51.77065
124	63	26	RepositoryListValue	124	2020-11-18 14:50:51.780898	2020-11-18 14:50:51.780898
125	64	25	RepositoryListValue	125	2020-11-18 14:50:51.796472	2020-11-18 14:50:51.796472
126	64	26	RepositoryListValue	126	2020-11-18 14:50:51.806935	2020-11-18 14:50:51.806935
127	65	25	RepositoryListValue	127	2020-11-18 14:50:51.822294	2020-11-18 14:50:51.822294
128	65	26	RepositoryListValue	128	2020-11-18 14:50:51.832419	2020-11-18 14:50:51.832419
129	66	25	RepositoryListValue	129	2020-11-18 14:50:51.847733	2020-11-18 14:50:51.847733
130	66	26	RepositoryListValue	130	2020-11-18 14:50:51.858477	2020-11-18 14:50:51.858477
131	67	27	RepositoryListValue	131	2020-11-18 14:53:24.494828	2020-11-18 14:53:24.494828
132	67	28	RepositoryListValue	132	2020-11-18 14:53:24.502389	2020-11-18 14:53:24.502389
133	68	27	RepositoryListValue	133	2020-11-18 14:53:24.513031	2020-11-18 14:53:24.513031
134	68	28	RepositoryListValue	134	2020-11-18 14:53:24.520929	2020-11-18 14:53:24.520929
135	69	27	RepositoryListValue	135	2020-11-18 14:53:24.531501	2020-11-18 14:53:24.531501
136	69	28	RepositoryListValue	136	2020-11-18 14:53:24.539937	2020-11-18 14:53:24.539937
137	70	27	RepositoryListValue	137	2020-11-18 14:53:24.550358	2020-11-18 14:53:24.550358
138	70	28	RepositoryListValue	138	2020-11-18 14:53:24.557372	2020-11-18 14:53:24.557372
139	71	27	RepositoryListValue	139	2020-11-18 14:53:24.56744	2020-11-18 14:53:24.56744
140	71	28	RepositoryListValue	140	2020-11-18 14:53:24.575321	2020-11-18 14:53:24.575321
141	72	29	RepositoryListValue	141	2021-05-19 20:01:51.45892	2021-05-19 20:01:51.45892
142	72	30	RepositoryListValue	142	2021-05-19 20:01:51.468865	2021-05-19 20:01:51.468865
143	73	29	RepositoryListValue	143	2021-05-19 20:01:51.475998	2021-05-19 20:01:51.475998
144	73	30	RepositoryListValue	144	2021-05-19 20:01:51.481474	2021-05-19 20:01:51.481474
145	74	29	RepositoryListValue	145	2021-05-19 20:01:51.48865	2021-05-19 20:01:51.48865
146	74	30	RepositoryListValue	146	2021-05-19 20:01:51.494034	2021-05-19 20:01:51.494034
147	75	29	RepositoryListValue	147	2021-05-19 20:01:51.50111	2021-05-19 20:01:51.50111
148	75	30	RepositoryListValue	148	2021-05-19 20:01:51.506555	2021-05-19 20:01:51.506555
149	76	29	RepositoryListValue	149	2021-05-19 20:01:51.53846	2021-05-19 20:01:51.53846
150	76	30	RepositoryListValue	150	2021-05-19 20:01:51.543483	2021-05-19 20:01:51.543483
151	77	31	RepositoryListValue	151	2021-05-20 14:12:30.295344	2021-05-20 14:12:30.295344
152	77	32	RepositoryListValue	152	2021-05-20 14:12:30.300611	2021-05-20 14:12:30.300611
153	78	31	RepositoryListValue	153	2021-05-20 14:12:30.307052	2021-05-20 14:12:30.307052
154	78	32	RepositoryListValue	154	2021-05-20 14:12:30.311901	2021-05-20 14:12:30.311901
155	79	31	RepositoryListValue	155	2021-05-20 14:12:30.31861	2021-05-20 14:12:30.31861
156	79	32	RepositoryListValue	156	2021-05-20 14:12:30.323647	2021-05-20 14:12:30.323647
157	80	31	RepositoryListValue	157	2021-05-20 14:12:30.329963	2021-05-20 14:12:30.329963
158	80	32	RepositoryListValue	158	2021-05-20 14:12:30.33492	2021-05-20 14:12:30.33492
159	81	31	RepositoryListValue	159	2021-05-20 14:12:30.341755	2021-05-20 14:12:30.341755
160	81	32	RepositoryListValue	160	2021-05-20 14:12:30.346913	2021-05-20 14:12:30.346913
161	82	33	RepositoryListValue	161	2021-05-20 14:12:57.035737	2021-05-20 14:12:57.035737
162	82	34	RepositoryListValue	162	2021-05-20 14:12:57.040627	2021-05-20 14:12:57.040627
163	83	33	RepositoryListValue	163	2021-05-20 14:12:57.048169	2021-05-20 14:12:57.048169
164	83	34	RepositoryListValue	164	2021-05-20 14:12:57.052823	2021-05-20 14:12:57.052823
165	84	33	RepositoryListValue	165	2021-05-20 14:12:57.058725	2021-05-20 14:12:57.058725
166	84	34	RepositoryListValue	166	2021-05-20 14:12:57.063243	2021-05-20 14:12:57.063243
167	85	33	RepositoryListValue	167	2021-05-20 14:12:57.070293	2021-05-20 14:12:57.070293
168	85	34	RepositoryListValue	168	2021-05-20 14:12:57.075092	2021-05-20 14:12:57.075092
169	86	33	RepositoryListValue	169	2021-05-20 14:12:57.080903	2021-05-20 14:12:57.080903
170	86	34	RepositoryListValue	170	2021-05-20 14:12:57.085642	2021-05-20 14:12:57.085642
171	87	35	RepositoryListValue	171	2021-05-20 14:13:33.882805	2021-05-20 14:13:33.882805
172	87	36	RepositoryListValue	172	2021-05-20 14:13:33.887578	2021-05-20 14:13:33.887578
173	88	35	RepositoryListValue	173	2021-05-20 14:13:33.893886	2021-05-20 14:13:33.893886
174	88	36	RepositoryListValue	174	2021-05-20 14:13:33.90122	2021-05-20 14:13:33.90122
175	89	35	RepositoryListValue	175	2021-05-20 14:13:33.907355	2021-05-20 14:13:33.907355
176	89	36	RepositoryListValue	176	2021-05-20 14:13:33.9117	2021-05-20 14:13:33.9117
177	90	35	RepositoryListValue	177	2021-05-20 14:13:33.91725	2021-05-20 14:13:33.91725
178	90	36	RepositoryListValue	178	2021-05-20 14:13:33.921715	2021-05-20 14:13:33.921715
179	91	35	RepositoryListValue	179	2021-05-20 14:13:33.927162	2021-05-20 14:13:33.927162
180	91	36	RepositoryListValue	180	2021-05-20 14:13:33.931535	2021-05-20 14:13:33.931535
181	92	37	RepositoryListValue	181	2021-05-20 14:21:20.418934	2021-05-20 14:21:20.418934
182	92	38	RepositoryListValue	182	2021-05-20 14:21:20.426834	2021-05-20 14:21:20.426834
183	93	37	RepositoryListValue	183	2021-05-20 14:21:20.434148	2021-05-20 14:21:20.434148
184	93	38	RepositoryListValue	184	2021-05-20 14:21:20.440346	2021-05-20 14:21:20.440346
185	94	37	RepositoryListValue	185	2021-05-20 14:21:20.447709	2021-05-20 14:21:20.447709
186	94	38	RepositoryListValue	186	2021-05-20 14:21:20.454754	2021-05-20 14:21:20.454754
187	95	37	RepositoryListValue	187	2021-05-20 14:21:20.46192	2021-05-20 14:21:20.46192
188	95	38	RepositoryListValue	188	2021-05-20 14:21:20.467049	2021-05-20 14:21:20.467049
189	96	37	RepositoryListValue	189	2021-05-20 14:21:20.473679	2021-05-20 14:21:20.473679
190	96	38	RepositoryListValue	190	2021-05-20 14:21:20.48025	2021-05-20 14:21:20.48025
191	97	39	RepositoryListValue	191	2021-05-20 14:30:30.205584	2021-05-20 14:30:30.205584
192	97	40	RepositoryListValue	192	2021-05-20 14:30:30.210423	2021-05-20 14:30:30.210423
193	98	39	RepositoryListValue	193	2021-05-20 14:30:30.216411	2021-05-20 14:30:30.216411
194	98	40	RepositoryListValue	194	2021-05-20 14:30:30.222053	2021-05-20 14:30:30.222053
195	99	39	RepositoryListValue	195	2021-05-20 14:30:30.229117	2021-05-20 14:30:30.229117
196	99	40	RepositoryListValue	196	2021-05-20 14:30:30.234018	2021-05-20 14:30:30.234018
197	100	39	RepositoryListValue	197	2021-05-20 14:30:30.240046	2021-05-20 14:30:30.240046
198	100	40	RepositoryListValue	198	2021-05-20 14:30:30.244828	2021-05-20 14:30:30.244828
199	101	39	RepositoryListValue	199	2021-05-20 14:30:30.251399	2021-05-20 14:30:30.251399
200	101	40	RepositoryListValue	200	2021-05-20 14:30:30.256309	2021-05-20 14:30:30.256309
201	102	41	RepositoryListValue	201	2021-05-20 15:17:11.584982	2021-05-20 15:17:11.584982
202	102	42	RepositoryListValue	202	2021-05-20 15:17:11.590906	2021-05-20 15:17:11.590906
203	103	41	RepositoryListValue	203	2021-05-20 15:17:11.601141	2021-05-20 15:17:11.601141
204	103	42	RepositoryListValue	204	2021-05-20 15:17:11.605924	2021-05-20 15:17:11.605924
205	104	41	RepositoryListValue	205	2021-05-20 15:17:11.612006	2021-05-20 15:17:11.612006
206	104	42	RepositoryListValue	206	2021-05-20 15:17:11.61749	2021-05-20 15:17:11.61749
207	105	41	RepositoryListValue	207	2021-05-20 15:17:11.624706	2021-05-20 15:17:11.624706
208	105	42	RepositoryListValue	208	2021-05-20 15:17:11.629887	2021-05-20 15:17:11.629887
209	106	41	RepositoryListValue	209	2021-05-20 15:17:11.636457	2021-05-20 15:17:11.636457
210	106	42	RepositoryListValue	210	2021-05-20 15:17:11.641088	2021-05-20 15:17:11.641088
211	107	43	RepositoryListValue	211	2021-05-28 15:07:42.326867	2021-05-28 15:07:42.326867
212	107	44	RepositoryListValue	212	2021-05-28 15:07:42.366624	2021-05-28 15:07:42.366624
213	108	43	RepositoryListValue	213	2021-05-28 15:07:42.395829	2021-05-28 15:07:42.395829
214	108	44	RepositoryListValue	214	2021-05-28 15:07:42.410498	2021-05-28 15:07:42.410498
215	109	43	RepositoryListValue	215	2021-05-28 15:07:42.469119	2021-05-28 15:07:42.469119
216	109	44	RepositoryListValue	216	2021-05-28 15:07:42.557493	2021-05-28 15:07:42.557493
217	110	43	RepositoryListValue	217	2021-05-28 15:07:42.942658	2021-05-28 15:07:42.942658
218	110	44	RepositoryListValue	218	2021-05-28 15:07:42.978659	2021-05-28 15:07:42.978659
219	111	43	RepositoryListValue	219	2021-05-28 15:07:43.058694	2021-05-28 15:07:43.058694
220	111	44	RepositoryListValue	220	2021-05-28 15:07:43.092064	2021-05-28 15:07:43.092064
221	112	45	RepositoryListValue	221	2021-06-02 16:10:01.245429	2021-06-02 16:10:01.245429
222	112	46	RepositoryListValue	222	2021-06-02 16:10:01.269066	2021-06-02 16:10:01.269066
223	113	45	RepositoryListValue	223	2021-06-02 16:10:01.294608	2021-06-02 16:10:01.294608
224	113	46	RepositoryListValue	224	2021-06-02 16:10:01.311075	2021-06-02 16:10:01.311075
225	114	45	RepositoryListValue	225	2021-06-02 16:10:01.334213	2021-06-02 16:10:01.334213
226	114	46	RepositoryListValue	226	2021-06-02 16:10:01.347769	2021-06-02 16:10:01.347769
227	115	45	RepositoryListValue	227	2021-06-02 16:10:01.366598	2021-06-02 16:10:01.366598
228	115	46	RepositoryListValue	228	2021-06-02 16:10:01.382826	2021-06-02 16:10:01.382826
229	116	45	RepositoryListValue	229	2021-06-02 16:10:01.393322	2021-06-02 16:10:01.393322
230	116	46	RepositoryListValue	230	2021-06-02 16:10:01.406433	2021-06-02 16:10:01.406433
231	117	47	RepositoryListValue	231	2021-08-23 14:58:27.332626	2021-08-23 14:58:27.332626
232	117	48	RepositoryListValue	232	2021-08-23 14:58:27.345084	2021-08-23 14:58:27.345084
233	118	47	RepositoryListValue	233	2021-08-23 14:58:27.355812	2021-08-23 14:58:27.355812
234	118	48	RepositoryListValue	234	2021-08-23 14:58:27.361219	2021-08-23 14:58:27.361219
235	119	47	RepositoryListValue	235	2021-08-23 14:58:27.368633	2021-08-23 14:58:27.368633
236	119	48	RepositoryListValue	236	2021-08-23 14:58:27.373507	2021-08-23 14:58:27.373507
237	120	47	RepositoryListValue	237	2021-08-23 14:58:27.379394	2021-08-23 14:58:27.379394
238	120	48	RepositoryListValue	238	2021-08-23 14:58:27.383898	2021-08-23 14:58:27.383898
239	121	47	RepositoryListValue	239	2021-08-23 14:58:27.390684	2021-08-23 14:58:27.390684
240	121	48	RepositoryListValue	240	2021-08-23 14:58:27.395474	2021-08-23 14:58:27.395474
241	122	49	RepositoryListValue	241	2021-08-23 17:19:39.081155	2021-08-23 17:19:39.081155
242	122	50	RepositoryListValue	242	2021-08-23 17:19:39.086227	2021-08-23 17:19:39.086227
243	123	49	RepositoryListValue	243	2021-08-23 17:19:39.092902	2021-08-23 17:19:39.092902
244	123	50	RepositoryListValue	244	2021-08-23 17:19:39.098649	2021-08-23 17:19:39.098649
245	124	49	RepositoryListValue	245	2021-08-23 17:19:39.105591	2021-08-23 17:19:39.105591
246	124	50	RepositoryListValue	246	2021-08-23 17:19:39.111253	2021-08-23 17:19:39.111253
247	125	49	RepositoryListValue	247	2021-08-23 17:19:39.118551	2021-08-23 17:19:39.118551
248	125	50	RepositoryListValue	248	2021-08-23 17:19:39.124071	2021-08-23 17:19:39.124071
249	126	49	RepositoryListValue	249	2021-08-23 17:19:39.130589	2021-08-23 17:19:39.130589
250	126	50	RepositoryListValue	250	2021-08-23 17:19:39.135442	2021-08-23 17:19:39.135442
\.


--
-- Data for Name: repository_columns; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repository_columns (id, repository_id, created_by_id, name, data_type, created_at, updated_at, metadata, parent_id) FROM stdin;
1	1	1	Sample Types	2	2019-05-30 12:40:52.938646	2019-05-30 12:40:52.938646	{}	\N
2	1	1	Sample Groups	2	2019-05-30 12:40:52.948774	2019-05-30 12:40:52.948774	{}	\N
3	2	4	Sample Types	2	2019-08-27 15:14:58.434735	2019-08-27 15:14:58.434735	{}	\N
4	2	4	Sample Groups	2	2019-08-27 15:14:58.466204	2019-08-27 15:14:58.466204	{}	\N
5	3	1	Sample Types	2	2019-08-28 07:30:56.952114	2019-08-28 07:30:56.952114	{}	\N
6	3	1	Sample Groups	2	2019-08-28 07:30:57.181967	2019-08-28 07:30:57.181967	{}	\N
7	4	6	Sample Types	2	2019-08-28 08:34:51.589767	2019-08-28 08:34:51.589767	{}	\N
8	4	6	Sample Groups	2	2019-08-28 08:34:51.621695	2019-08-28 08:34:51.621695	{}	\N
9	5	7	Sample Types	2	2019-08-29 12:51:21.225138	2019-08-29 12:51:21.225138	{}	\N
10	5	7	Sample Groups	2	2019-08-29 12:51:21.267514	2019-08-29 12:51:21.267514	{}	\N
11	9	8	Sample Types	2	2019-09-12 18:26:18.120491	2019-09-12 18:26:18.120491	{}	\N
12	9	8	Sample Groups	2	2019-09-12 18:26:18.132236	2019-09-12 18:26:18.132236	{}	\N
13	10	9	Sample Types	2	2019-11-13 13:57:40.96612	2019-11-13 13:57:40.96612	{}	\N
14	10	9	Sample Groups	2	2019-11-13 13:57:40.986269	2019-11-13 13:57:40.986269	{}	\N
15	11	4	Sample Types	2	2019-11-25 14:09:20.877315	2019-11-25 14:09:20.877315	{}	\N
16	11	4	project id	2	2019-11-25 14:09:20.957447	2019-11-25 17:44:01.074534	{}	\N
17	12	10	Sample Types	2	2019-11-25 18:00:52.142288	2019-11-25 18:00:52.142288	{}	\N
18	12	10	Sample Groups	2	2019-11-25 18:00:52.148314	2019-11-25 18:00:52.148314	{}	\N
19	13	11	Sample Types	2	2020-05-12 16:13:00.859116	2020-05-12 16:13:00.859116	{}	\N
20	13	11	Sample Groups	2	2020-05-12 16:13:00.880044	2020-05-12 16:13:00.880044	{}	\N
21	14	12	Sample Types	2	2020-06-10 14:43:16.430864	2020-06-10 14:43:16.430864	{}	\N
22	14	12	Sample Groups	2	2020-06-10 14:43:16.452719	2020-06-10 14:43:16.452719	{}	\N
23	15	16	Sample Types	2	2020-10-07 19:20:23.710666	2020-10-07 19:20:23.710666	{}	\N
24	15	16	Sample Groups	2	2020-10-07 19:20:23.884242	2020-10-07 19:20:23.884242	{}	\N
25	16	1	Sample Types	2	2020-11-18 14:50:51.553174	2020-11-18 14:50:51.553174	{}	\N
26	16	1	Sample Groups	2	2020-11-18 14:50:51.584803	2020-11-18 14:50:51.584803	{}	\N
27	17	17	Sample Types	2	2020-11-18 14:53:24.336817	2020-11-18 14:53:24.336817	{}	\N
28	17	17	Sample Groups	2	2020-11-18 14:53:24.353551	2020-11-18 14:53:24.353551	{}	\N
29	18	20	Sample Types	2	2021-05-19 20:01:51.405099	2021-05-19 20:01:51.405099	{}	\N
30	18	20	Sample Groups	2	2021-05-19 20:01:51.411344	2021-05-19 20:01:51.411344	{}	\N
31	19	21	Sample Types	2	2021-05-20 14:12:30.265168	2021-05-20 14:12:30.265168	{}	\N
32	19	21	Sample Groups	2	2021-05-20 14:12:30.269201	2021-05-20 14:12:30.269201	{}	\N
33	20	22	Sample Types	2	2021-05-20 14:12:57.01057	2021-05-20 14:12:57.01057	{}	\N
34	20	22	Sample Groups	2	2021-05-20 14:12:57.014344	2021-05-20 14:12:57.014344	{}	\N
35	21	23	Sample Types	2	2021-05-20 14:13:33.85795	2021-05-20 14:13:33.85795	{}	\N
36	21	23	Sample Groups	2	2021-05-20 14:13:33.861254	2021-05-20 14:13:33.861254	{}	\N
37	22	24	Sample Types	2	2021-05-20 14:21:20.381207	2021-05-20 14:21:20.381207	{}	\N
38	22	24	Sample Groups	2	2021-05-20 14:21:20.386524	2021-05-20 14:21:20.386524	{}	\N
39	23	25	Sample Types	2	2021-05-20 14:30:30.178984	2021-05-20 14:30:30.178984	{}	\N
40	23	25	Sample Groups	2	2021-05-20 14:30:30.182645	2021-05-20 14:30:30.182645	{}	\N
41	24	26	Sample Types	2	2021-05-20 15:17:11.555013	2021-05-20 15:17:11.555013	{}	\N
42	24	26	Sample Groups	2	2021-05-20 15:17:11.559143	2021-05-20 15:17:11.559143	{}	\N
43	25	27	Sample Types	2	2021-05-28 15:07:38.050861	2021-05-28 15:07:38.050861	{}	\N
44	25	27	Sample Groups	2	2021-05-28 15:07:41.452165	2021-05-28 15:07:41.452165	{}	\N
45	26	28	Sample Types	2	2021-06-02 16:10:00.990638	2021-06-02 16:10:00.990638	{}	\N
46	26	28	Sample Groups	2	2021-06-02 16:10:01.019904	2021-06-02 16:10:01.019904	{}	\N
47	27	30	Sample Types	2	2021-08-23 14:58:27.22555	2021-08-23 14:58:27.22555	{}	\N
48	27	30	Sample Groups	2	2021-08-23 14:58:27.240128	2021-08-23 14:58:27.240128	{}	\N
49	28	31	Sample Types	2	2021-08-23 17:19:39.052452	2021-08-23 17:19:39.052452	{}	\N
50	28	31	Sample Groups	2	2021-08-23 17:19:39.056751	2021-08-23 17:19:39.056751	{}	\N
\.


--
-- Data for Name: repository_list_items; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repository_list_items (id, repository_column_id, data, created_by_id, last_modified_by_id, created_at, updated_at) FROM stdin;
1	1	Potato leaves	1	1	2019-05-30 12:40:52.964437	2019-05-30 12:40:52.964437
2	1	Tea leaves	1	1	2019-05-30 12:40:52.96721	2019-05-30 12:40:52.96721
3	1	Potato bug	1	1	2019-05-30 12:40:52.969479	2019-05-30 12:40:52.969479
4	2	Fodder	1	1	2019-05-30 12:40:52.971719	2019-05-30 12:40:52.971719
5	2	Nutrient	1	1	2019-05-30 12:40:52.973899	2019-05-30 12:40:52.973899
6	2	Seed	1	1	2019-05-30 12:40:52.976054	2019-05-30 12:40:52.976054
7	3	Potato leaves	4	4	2019-08-27 15:14:58.553576	2019-08-27 15:14:58.553576
8	3	Tea leaves	4	4	2019-08-27 15:14:58.564542	2019-08-27 15:14:58.564542
9	3	Potato bug	4	4	2019-08-27 15:14:58.568522	2019-08-27 15:14:58.568522
10	4	Fodder	4	4	2019-08-27 15:14:58.573219	2019-08-27 15:14:58.573219
11	4	Nutrient	4	4	2019-08-27 15:14:58.578053	2019-08-27 15:14:58.578053
12	4	Seed	4	4	2019-08-27 15:14:58.583515	2019-08-27 15:14:58.583515
13	5	Potato leaves	1	1	2019-08-28 07:30:57.233972	2019-08-28 07:30:57.233972
14	5	Tea leaves	1	1	2019-08-28 07:30:57.238988	2019-08-28 07:30:57.238988
15	5	Potato bug	1	1	2019-08-28 07:30:57.243229	2019-08-28 07:30:57.243229
16	6	Fodder	1	1	2019-08-28 07:30:57.24733	2019-08-28 07:30:57.24733
17	6	Nutrient	1	1	2019-08-28 07:30:57.25148	2019-08-28 07:30:57.25148
18	6	Seed	1	1	2019-08-28 07:30:57.255758	2019-08-28 07:30:57.255758
19	7	Potato leaves	6	6	2019-08-28 08:34:51.632789	2019-08-28 08:34:51.632789
20	7	Tea leaves	6	6	2019-08-28 08:34:51.638598	2019-08-28 08:34:51.638598
21	7	Potato bug	6	6	2019-08-28 08:34:51.641605	2019-08-28 08:34:51.641605
22	8	Fodder	6	6	2019-08-28 08:34:51.645326	2019-08-28 08:34:51.645326
23	8	Nutrient	6	6	2019-08-28 08:34:51.648475	2019-08-28 08:34:51.648475
24	8	Seed	6	6	2019-08-28 08:34:51.651433	2019-08-28 08:34:51.651433
25	9	Potato leaves	7	7	2019-08-29 12:51:21.330033	2019-08-29 12:51:21.330033
26	9	Tea leaves	7	7	2019-08-29 12:51:21.404026	2019-08-29 12:51:21.404026
27	9	Potato bug	7	7	2019-08-29 12:51:21.422077	2019-08-29 12:51:21.422077
28	10	Fodder	7	7	2019-08-29 12:51:21.43897	2019-08-29 12:51:21.43897
29	10	Nutrient	7	7	2019-08-29 12:51:21.468919	2019-08-29 12:51:21.468919
30	10	Seed	7	7	2019-08-29 12:51:21.475298	2019-08-29 12:51:21.475298
31	11	Potato leaves	8	8	2019-09-12 18:26:18.151142	2019-09-12 18:26:18.151142
32	11	Tea leaves	8	8	2019-09-12 18:26:18.156039	2019-09-12 18:26:18.156039
33	11	Potato bug	8	8	2019-09-12 18:26:18.158784	2019-09-12 18:26:18.158784
34	12	Fodder	8	8	2019-09-12 18:26:18.161756	2019-09-12 18:26:18.161756
35	12	Nutrient	8	8	2019-09-12 18:26:18.164622	2019-09-12 18:26:18.164622
36	12	Seed	8	8	2019-09-12 18:26:18.167314	2019-09-12 18:26:18.167314
37	13	Potato leaves	9	9	2019-11-13 13:57:41.014958	2019-11-13 13:57:41.014958
38	13	Tea leaves	9	9	2019-11-13 13:57:41.02152	2019-11-13 13:57:41.02152
39	13	Potato bug	9	9	2019-11-13 13:57:41.028828	2019-11-13 13:57:41.028828
40	14	Fodder	9	9	2019-11-13 13:57:41.033	2019-11-13 13:57:41.033
41	14	Nutrient	9	9	2019-11-13 13:57:41.036471	2019-11-13 13:57:41.036471
42	14	Seed	9	9	2019-11-13 13:57:41.041501	2019-11-13 13:57:41.041501
43	15	Potato leaves	4	4	2019-11-25 14:09:21.041385	2019-11-25 14:09:21.041385
44	15	Tea leaves	4	4	2019-11-25 14:09:21.049499	2019-11-25 14:09:21.049499
45	15	Potato bug	4	4	2019-11-25 14:09:21.054557	2019-11-25 14:09:21.054557
49	16	Conductivity	4	4	2019-11-25 17:44:01.140178	2019-11-25 17:44:01.140178
50	16	Residual chlorine	4	4	2019-11-25 17:44:01.14833	2019-11-25 17:44:01.14833
51	16	Autoclave Sporetol	4	4	2019-11-25 17:44:01.158897	2019-11-25 17:44:01.158897
52	16	Media QC	4	4	2019-11-25 17:44:01.170099	2019-11-25 17:44:01.170099
53	17	Potato leaves	10	10	2019-11-25 18:00:52.154854	2019-11-25 18:00:52.154854
54	17	Tea leaves	10	10	2019-11-25 18:00:52.158299	2019-11-25 18:00:52.158299
55	17	Potato bug	10	10	2019-11-25 18:00:52.161894	2019-11-25 18:00:52.161894
56	18	Fodder	10	10	2019-11-25 18:00:52.16589	2019-11-25 18:00:52.16589
57	18	Nutrient	10	10	2019-11-25 18:00:52.174028	2019-11-25 18:00:52.174028
58	18	Seed	10	10	2019-11-25 18:00:52.176963	2019-11-25 18:00:52.176963
59	19	Potato leaves	11	11	2020-05-12 16:13:00.919661	2020-05-12 16:13:00.919661
60	19	Tea leaves	11	11	2020-05-12 16:13:00.926126	2020-05-12 16:13:00.926126
61	19	Potato bug	11	11	2020-05-12 16:13:00.931106	2020-05-12 16:13:00.931106
62	20	Fodder	11	11	2020-05-12 16:13:01.024381	2020-05-12 16:13:01.024381
63	20	Nutrient	11	11	2020-05-12 16:13:01.028831	2020-05-12 16:13:01.028831
64	20	Seed	11	11	2020-05-12 16:13:01.034109	2020-05-12 16:13:01.034109
65	21	Potato leaves	12	12	2020-06-10 14:43:16.472919	2020-06-10 14:43:16.472919
66	21	Tea leaves	12	12	2020-06-10 14:43:16.482351	2020-06-10 14:43:16.482351
67	21	Potato bug	12	12	2020-06-10 14:43:16.485282	2020-06-10 14:43:16.485282
68	22	Fodder	12	12	2020-06-10 14:43:16.488169	2020-06-10 14:43:16.488169
69	22	Nutrient	12	12	2020-06-10 14:43:16.497901	2020-06-10 14:43:16.497901
70	22	Seed	12	12	2020-06-10 14:43:16.500714	2020-06-10 14:43:16.500714
71	23	Potato leaves	16	16	2020-10-07 19:20:24.035111	2020-10-07 19:20:24.035111
72	23	Tea leaves	16	16	2020-10-07 19:20:24.066842	2020-10-07 19:20:24.066842
73	23	Potato bug	16	16	2020-10-07 19:20:24.099171	2020-10-07 19:20:24.099171
74	24	Fodder	16	16	2020-10-07 19:20:24.124022	2020-10-07 19:20:24.124022
75	24	Nutrient	16	16	2020-10-07 19:20:24.132918	2020-10-07 19:20:24.132918
76	24	Seed	16	16	2020-10-07 19:20:24.144767	2020-10-07 19:20:24.144767
77	25	Potato leaves	1	1	2020-11-18 14:50:51.61349	2020-11-18 14:50:51.61349
78	25	Tea leaves	1	1	2020-11-18 14:50:51.623983	2020-11-18 14:50:51.623983
79	25	Potato bug	1	1	2020-11-18 14:50:51.632257	2020-11-18 14:50:51.632257
80	26	Fodder	1	1	2020-11-18 14:50:51.637686	2020-11-18 14:50:51.637686
81	26	Nutrient	1	1	2020-11-18 14:50:51.644068	2020-11-18 14:50:51.644068
82	26	Seed	1	1	2020-11-18 14:50:51.649826	2020-11-18 14:50:51.649826
83	27	Potato leaves	17	17	2020-11-18 14:53:24.38817	2020-11-18 14:53:24.38817
84	27	Tea leaves	17	17	2020-11-18 14:53:24.395101	2020-11-18 14:53:24.395101
85	27	Potato bug	17	17	2020-11-18 14:53:24.436702	2020-11-18 14:53:24.436702
86	28	Fodder	17	17	2020-11-18 14:53:24.44013	2020-11-18 14:53:24.44013
87	28	Nutrient	17	17	2020-11-18 14:53:24.444054	2020-11-18 14:53:24.444054
88	28	Seed	17	17	2020-11-18 14:53:24.448419	2020-11-18 14:53:24.448419
89	29	Potato leaves	20	20	2021-05-19 20:01:51.415935	2021-05-19 20:01:51.415935
90	29	Tea leaves	20	20	2021-05-19 20:01:51.421777	2021-05-19 20:01:51.421777
91	29	Potato bug	20	20	2021-05-19 20:01:51.425634	2021-05-19 20:01:51.425634
92	30	Fodder	20	20	2021-05-19 20:01:51.429141	2021-05-19 20:01:51.429141
93	30	Nutrient	20	20	2021-05-19 20:01:51.431432	2021-05-19 20:01:51.431432
94	30	Seed	20	20	2021-05-19 20:01:51.44531	2021-05-19 20:01:51.44531
95	31	Potato leaves	21	21	2021-05-20 14:12:30.274046	2021-05-20 14:12:30.274046
96	31	Tea leaves	21	21	2021-05-20 14:12:30.27637	2021-05-20 14:12:30.27637
97	31	Potato bug	21	21	2021-05-20 14:12:30.278851	2021-05-20 14:12:30.278851
98	32	Fodder	21	21	2021-05-20 14:12:30.280906	2021-05-20 14:12:30.280906
99	32	Nutrient	21	21	2021-05-20 14:12:30.283533	2021-05-20 14:12:30.283533
100	32	Seed	21	21	2021-05-20 14:12:30.285446	2021-05-20 14:12:30.285446
101	33	Potato leaves	22	22	2021-05-20 14:12:57.017945	2021-05-20 14:12:57.017945
102	33	Tea leaves	22	22	2021-05-20 14:12:57.020025	2021-05-20 14:12:57.020025
103	33	Potato bug	22	22	2021-05-20 14:12:57.022068	2021-05-20 14:12:57.022068
104	34	Fodder	22	22	2021-05-20 14:12:57.024493	2021-05-20 14:12:57.024493
105	34	Nutrient	22	22	2021-05-20 14:12:57.026438	2021-05-20 14:12:57.026438
106	34	Seed	22	22	2021-05-20 14:12:57.028336	2021-05-20 14:12:57.028336
107	35	Potato leaves	23	23	2021-05-20 14:13:33.864431	2021-05-20 14:13:33.864431
108	35	Tea leaves	23	23	2021-05-20 14:13:33.866898	2021-05-20 14:13:33.866898
109	35	Potato bug	23	23	2021-05-20 14:13:33.870649	2021-05-20 14:13:33.870649
110	36	Fodder	23	23	2021-05-20 14:13:33.87291	2021-05-20 14:13:33.87291
111	36	Nutrient	23	23	2021-05-20 14:13:33.874706	2021-05-20 14:13:33.874706
112	36	Seed	23	23	2021-05-20 14:13:33.876483	2021-05-20 14:13:33.876483
113	37	Potato leaves	24	24	2021-05-20 14:21:20.392182	2021-05-20 14:21:20.392182
114	37	Tea leaves	24	24	2021-05-20 14:21:20.395313	2021-05-20 14:21:20.395313
115	37	Potato bug	24	24	2021-05-20 14:21:20.398788	2021-05-20 14:21:20.398788
116	38	Fodder	24	24	2021-05-20 14:21:20.401348	2021-05-20 14:21:20.401348
117	38	Nutrient	24	24	2021-05-20 14:21:20.40496	2021-05-20 14:21:20.40496
118	38	Seed	24	24	2021-05-20 14:21:20.408069	2021-05-20 14:21:20.408069
119	39	Potato leaves	25	25	2021-05-20 14:30:30.186676	2021-05-20 14:30:30.186676
120	39	Tea leaves	25	25	2021-05-20 14:30:30.189589	2021-05-20 14:30:30.189589
121	39	Potato bug	25	25	2021-05-20 14:30:30.191497	2021-05-20 14:30:30.191497
122	40	Fodder	25	25	2021-05-20 14:30:30.193612	2021-05-20 14:30:30.193612
123	40	Nutrient	25	25	2021-05-20 14:30:30.195548	2021-05-20 14:30:30.195548
124	40	Seed	25	25	2021-05-20 14:30:30.197446	2021-05-20 14:30:30.197446
125	41	Potato leaves	26	26	2021-05-20 15:17:11.563658	2021-05-20 15:17:11.563658
126	41	Tea leaves	26	26	2021-05-20 15:17:11.565991	2021-05-20 15:17:11.565991
127	41	Potato bug	26	26	2021-05-20 15:17:11.568219	2021-05-20 15:17:11.568219
128	42	Fodder	26	26	2021-05-20 15:17:11.570419	2021-05-20 15:17:11.570419
129	42	Nutrient	26	26	2021-05-20 15:17:11.572669	2021-05-20 15:17:11.572669
130	42	Seed	26	26	2021-05-20 15:17:11.574942	2021-05-20 15:17:11.574942
131	43	Potato leaves	27	27	2021-05-28 15:07:41.58812	2021-05-28 15:07:41.58812
132	43	Tea leaves	27	27	2021-05-28 15:07:41.636631	2021-05-28 15:07:41.636631
133	43	Potato bug	27	27	2021-05-28 15:07:41.660309	2021-05-28 15:07:41.660309
134	44	Fodder	27	27	2021-05-28 15:07:41.683864	2021-05-28 15:07:41.683864
135	44	Nutrient	27	27	2021-05-28 15:07:41.700147	2021-05-28 15:07:41.700147
136	44	Seed	27	27	2021-05-28 15:07:41.713458	2021-05-28 15:07:41.713458
137	45	Potato leaves	28	28	2021-06-02 16:10:01.058336	2021-06-02 16:10:01.058336
138	45	Tea leaves	28	28	2021-06-02 16:10:01.103164	2021-06-02 16:10:01.103164
139	45	Potato bug	28	28	2021-06-02 16:10:01.110423	2021-06-02 16:10:01.110423
140	46	Fodder	28	28	2021-06-02 16:10:01.119535	2021-06-02 16:10:01.119535
141	46	Nutrient	28	28	2021-06-02 16:10:01.127593	2021-06-02 16:10:01.127593
142	46	Seed	28	28	2021-06-02 16:10:01.134098	2021-06-02 16:10:01.134098
143	47	Potato leaves	30	30	2021-08-23 14:58:27.264227	2021-08-23 14:58:27.264227
144	47	Tea leaves	30	30	2021-08-23 14:58:27.27323	2021-08-23 14:58:27.27323
145	47	Potato bug	30	30	2021-08-23 14:58:27.275727	2021-08-23 14:58:27.275727
146	48	Fodder	30	30	2021-08-23 14:58:27.279045	2021-08-23 14:58:27.279045
147	48	Nutrient	30	30	2021-08-23 14:58:27.280987	2021-08-23 14:58:27.280987
148	48	Seed	30	30	2021-08-23 14:58:27.283118	2021-08-23 14:58:27.283118
149	49	Potato leaves	31	31	2021-08-23 17:19:39.061	2021-08-23 17:19:39.061
150	49	Tea leaves	31	31	2021-08-23 17:19:39.063478	2021-08-23 17:19:39.063478
151	49	Potato bug	31	31	2021-08-23 17:19:39.065798	2021-08-23 17:19:39.065798
152	50	Fodder	31	31	2021-08-23 17:19:39.068	2021-08-23 17:19:39.068
153	50	Nutrient	31	31	2021-08-23 17:19:39.070191	2021-08-23 17:19:39.070191
154	50	Seed	31	31	2021-08-23 17:19:39.072824	2021-08-23 17:19:39.072824
\.


--
-- Data for Name: repository_list_values; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repository_list_values (id, repository_list_item_id, created_by_id, last_modified_by_id, created_at, updated_at) FROM stdin;
1	3	1	1	2019-05-30 12:40:53.026625	2019-05-30 12:40:53.026625
2	5	1	1	2019-05-30 12:40:53.032807	2019-05-30 12:40:53.032807
3	3	1	1	2019-05-30 12:40:53.039607	2019-05-30 12:40:53.039607
4	5	1	1	2019-05-30 12:40:53.044885	2019-05-30 12:40:53.044885
5	3	1	1	2019-05-30 12:40:53.051647	2019-05-30 12:40:53.051647
6	6	1	1	2019-05-30 12:40:53.056775	2019-05-30 12:40:53.056775
7	3	1	1	2019-05-30 12:40:53.063628	2019-05-30 12:40:53.063628
8	5	1	1	2019-05-30 12:40:53.068758	2019-05-30 12:40:53.068758
9	1	1	1	2019-05-30 12:40:53.075404	2019-05-30 12:40:53.075404
10	6	1	1	2019-05-30 12:40:53.080488	2019-05-30 12:40:53.080488
11	7	4	4	2019-08-27 15:14:58.771614	2019-08-27 15:14:58.771614
12	10	4	4	2019-08-27 15:14:58.792001	2019-08-27 15:14:58.792001
13	9	4	4	2019-08-27 15:14:58.818823	2019-08-27 15:14:58.818823
14	12	4	4	2019-08-27 15:14:58.83012	2019-08-27 15:14:58.83012
15	9	4	4	2019-08-27 15:14:58.846581	2019-08-27 15:14:58.846581
16	10	4	4	2019-08-27 15:14:58.859315	2019-08-27 15:14:58.859315
17	9	4	4	2019-08-27 15:14:58.874402	2019-08-27 15:14:58.874402
18	10	4	4	2019-08-27 15:14:58.888525	2019-08-27 15:14:58.888525
19	8	4	4	2019-08-27 15:14:58.944201	2019-08-27 15:14:58.944201
20	10	4	4	2019-08-27 15:14:58.967901	2019-08-27 15:14:58.967901
21	14	1	1	2019-08-28 07:30:58.208161	2019-08-28 07:30:58.208161
22	16	1	1	2019-08-28 07:30:58.217589	2019-08-28 07:30:58.217589
23	15	1	1	2019-08-28 07:30:58.22917	2019-08-28 07:30:58.22917
24	17	1	1	2019-08-28 07:30:58.237064	2019-08-28 07:30:58.237064
25	15	1	1	2019-08-28 07:30:58.249514	2019-08-28 07:30:58.249514
26	17	1	1	2019-08-28 07:30:58.257376	2019-08-28 07:30:58.257376
27	15	1	1	2019-08-28 07:30:58.274126	2019-08-28 07:30:58.274126
28	16	1	1	2019-08-28 07:30:58.282865	2019-08-28 07:30:58.282865
29	13	1	1	2019-08-28 07:30:58.294422	2019-08-28 07:30:58.294422
30	18	1	1	2019-08-28 07:30:58.302951	2019-08-28 07:30:58.302951
31	20	6	6	2019-08-28 08:34:51.68573	2019-08-28 08:34:51.68573
32	22	6	6	2019-08-28 08:34:51.698916	2019-08-28 08:34:51.698916
33	19	6	6	2019-08-28 08:34:51.709595	2019-08-28 08:34:51.709595
34	23	6	6	2019-08-28 08:34:51.716655	2019-08-28 08:34:51.716655
35	20	6	6	2019-08-28 08:34:51.735867	2019-08-28 08:34:51.735867
36	23	6	6	2019-08-28 08:34:51.743759	2019-08-28 08:34:51.743759
37	19	6	6	2019-08-28 08:34:51.756445	2019-08-28 08:34:51.756445
38	22	6	6	2019-08-28 08:34:52.260436	2019-08-28 08:34:52.260436
39	21	6	6	2019-08-28 08:34:52.388474	2019-08-28 08:34:52.388474
40	24	6	6	2019-08-28 08:34:52.468331	2019-08-28 08:34:52.468331
51	31	8	8	2019-09-12 18:26:18.198397	2019-09-12 18:26:18.198397
52	34	8	8	2019-09-12 18:26:18.207885	2019-09-12 18:26:18.207885
53	33	8	8	2019-09-12 18:26:18.216563	2019-09-12 18:26:18.216563
54	36	8	8	2019-09-12 18:26:18.223611	2019-09-12 18:26:18.223611
55	33	8	8	2019-09-12 18:26:18.232364	2019-09-12 18:26:18.232364
56	34	8	8	2019-09-12 18:26:18.239309	2019-09-12 18:26:18.239309
57	32	8	8	2019-09-12 18:26:18.248127	2019-09-12 18:26:18.248127
58	36	8	8	2019-09-12 18:26:18.254685	2019-09-12 18:26:18.254685
59	33	8	8	2019-09-12 18:26:18.262603	2019-09-12 18:26:18.262603
60	36	8	8	2019-09-12 18:26:18.268731	2019-09-12 18:26:18.268731
61	38	9	9	2019-11-13 13:57:41.106315	2019-11-13 13:57:41.106315
62	42	9	9	2019-11-13 13:57:41.121586	2019-11-13 13:57:41.121586
63	37	9	9	2019-11-13 13:57:41.134588	2019-11-13 13:57:41.134588
64	42	9	9	2019-11-13 13:57:41.143006	2019-11-13 13:57:41.143006
65	39	9	9	2019-11-13 13:57:41.155474	2019-11-13 13:57:41.155474
66	41	9	9	2019-11-13 13:57:41.167742	2019-11-13 13:57:41.167742
67	39	9	9	2019-11-13 13:57:41.181975	2019-11-13 13:57:41.181975
68	41	9	9	2019-11-13 13:57:41.192697	2019-11-13 13:57:41.192697
69	38	9	9	2019-11-13 13:57:41.205224	2019-11-13 13:57:41.205224
70	41	9	9	2019-11-13 13:57:41.213827	2019-11-13 13:57:41.213827
71	44	4	4	2019-11-25 14:09:21.303259	2019-11-25 14:09:21.303259
73	45	4	4	2019-11-25 14:09:21.347769	2019-11-25 14:09:21.347769
75	45	4	4	2019-11-25 14:09:21.381579	2019-11-25 14:09:21.381579
76	48	4	4	2019-11-25 14:09:21.395825	2019-11-25 14:09:21.395825
77	45	4	4	2019-11-25 14:09:21.426136	2019-11-25 14:09:21.426136
78	47	4	4	2019-11-25 14:09:21.437634	2019-11-25 14:09:21.437634
79	43	4	4	2019-11-25 14:09:21.458361	2019-11-25 14:09:21.458361
80	47	4	4	2019-11-25 14:09:21.47805	2019-11-25 14:09:21.47805
72	49	4	4	2019-11-25 14:09:21.324195	2019-11-25 17:45:32.283093
74	50	4	4	2019-11-25 14:09:21.362214	2019-11-25 17:45:39.404699
81	53	10	10	2019-11-25 18:00:52.188049	2019-11-25 18:00:52.188049
82	58	10	10	2019-11-25 18:00:52.198108	2019-11-25 18:00:52.198108
83	54	10	10	2019-11-25 18:00:52.209547	2019-11-25 18:00:52.209547
84	57	10	10	2019-11-25 18:00:52.217512	2019-11-25 18:00:52.217512
85	54	10	10	2019-11-25 18:00:52.227734	2019-11-25 18:00:52.227734
86	58	10	10	2019-11-25 18:00:52.235907	2019-11-25 18:00:52.235907
87	54	10	10	2019-11-25 18:00:52.245443	2019-11-25 18:00:52.245443
88	57	10	10	2019-11-25 18:00:52.253725	2019-11-25 18:00:52.253725
89	55	10	10	2019-11-25 18:00:52.265482	2019-11-25 18:00:52.265482
90	57	10	10	2019-11-25 18:00:52.276743	2019-11-25 18:00:52.276743
91	60	11	11	2020-05-12 16:13:01.130998	2020-05-12 16:13:01.130998
92	62	11	11	2020-05-12 16:13:01.148204	2020-05-12 16:13:01.148204
93	61	11	11	2020-05-12 16:13:01.161477	2020-05-12 16:13:01.161477
94	64	11	11	2020-05-12 16:13:01.171414	2020-05-12 16:13:01.171414
95	60	11	11	2020-05-12 16:13:01.184393	2020-05-12 16:13:01.184393
96	64	11	11	2020-05-12 16:13:01.197277	2020-05-12 16:13:01.197277
97	60	11	11	2020-05-12 16:13:01.215098	2020-05-12 16:13:01.215098
98	64	11	11	2020-05-12 16:13:01.225343	2020-05-12 16:13:01.225343
99	59	11	11	2020-05-12 16:13:01.240539	2020-05-12 16:13:01.240539
100	62	11	11	2020-05-12 16:13:01.258235	2020-05-12 16:13:01.258235
101	67	12	12	2020-06-10 14:43:16.548466	2020-06-10 14:43:16.548466
102	68	12	12	2020-06-10 14:43:16.565845	2020-06-10 14:43:16.565845
103	65	12	12	2020-06-10 14:43:16.575386	2020-06-10 14:43:16.575386
104	70	12	12	2020-06-10 14:43:16.582808	2020-06-10 14:43:16.582808
105	67	12	12	2020-06-10 14:43:16.59181	2020-06-10 14:43:16.59181
106	70	12	12	2020-06-10 14:43:16.598868	2020-06-10 14:43:16.598868
107	65	12	12	2020-06-10 14:43:16.607959	2020-06-10 14:43:16.607959
108	68	12	12	2020-06-10 14:43:16.614742	2020-06-10 14:43:16.614742
109	67	12	12	2020-06-10 14:43:16.623271	2020-06-10 14:43:16.623271
110	70	12	12	2020-06-10 14:43:16.630338	2020-06-10 14:43:16.630338
111	71	16	16	2020-10-07 19:20:24.365568	2020-10-07 19:20:24.365568
112	74	16	16	2020-10-07 19:20:24.393374	2020-10-07 19:20:24.393374
113	73	16	16	2020-10-07 19:20:24.410131	2020-10-07 19:20:24.410131
114	76	16	16	2020-10-07 19:20:24.424319	2020-10-07 19:20:24.424319
115	73	16	16	2020-10-07 19:20:24.443186	2020-10-07 19:20:24.443186
116	75	16	16	2020-10-07 19:20:24.455752	2020-10-07 19:20:24.455752
117	71	16	16	2020-10-07 19:20:24.470655	2020-10-07 19:20:24.470655
118	74	16	16	2020-10-07 19:20:24.484829	2020-10-07 19:20:24.484829
119	73	16	16	2020-10-07 19:20:24.502673	2020-10-07 19:20:24.502673
120	75	16	16	2020-10-07 19:20:24.515516	2020-10-07 19:20:24.515516
121	77	1	1	2020-11-18 14:50:51.737891	2020-11-18 14:50:51.737891
122	82	1	1	2020-11-18 14:50:51.755032	2020-11-18 14:50:51.755032
123	79	1	1	2020-11-18 14:50:51.769076	2020-11-18 14:50:51.769076
124	81	1	1	2020-11-18 14:50:51.779283	2020-11-18 14:50:51.779283
125	79	1	1	2020-11-18 14:50:51.794742	2020-11-18 14:50:51.794742
126	82	1	1	2020-11-18 14:50:51.805182	2020-11-18 14:50:51.805182
127	78	1	1	2020-11-18 14:50:51.820757	2020-11-18 14:50:51.820757
128	81	1	1	2020-11-18 14:50:51.83078	2020-11-18 14:50:51.83078
129	78	1	1	2020-11-18 14:50:51.845967	2020-11-18 14:50:51.845967
130	81	1	1	2020-11-18 14:50:51.856567	2020-11-18 14:50:51.856567
131	84	17	17	2020-11-18 14:53:24.492601	2020-11-18 14:53:24.492601
132	88	17	17	2020-11-18 14:53:24.500834	2020-11-18 14:53:24.500834
133	84	17	17	2020-11-18 14:53:24.511188	2020-11-18 14:53:24.511188
134	87	17	17	2020-11-18 14:53:24.519371	2020-11-18 14:53:24.519371
135	84	17	17	2020-11-18 14:53:24.529768	2020-11-18 14:53:24.529768
136	86	17	17	2020-11-18 14:53:24.538388	2020-11-18 14:53:24.538388
137	84	17	17	2020-11-18 14:53:24.548836	2020-11-18 14:53:24.548836
138	88	17	17	2020-11-18 14:53:24.556041	2020-11-18 14:53:24.556041
139	85	17	17	2020-11-18 14:53:24.566017	2020-11-18 14:53:24.566017
140	88	17	17	2020-11-18 14:53:24.573621	2020-11-18 14:53:24.573621
141	90	20	20	2021-05-19 20:01:51.455725	2021-05-19 20:01:51.455725
142	93	20	20	2021-05-19 20:01:51.467608	2021-05-19 20:01:51.467608
143	90	20	20	2021-05-19 20:01:51.475002	2021-05-19 20:01:51.475002
144	93	20	20	2021-05-19 20:01:51.480489	2021-05-19 20:01:51.480489
145	89	20	20	2021-05-19 20:01:51.487615	2021-05-19 20:01:51.487615
146	93	20	20	2021-05-19 20:01:51.492765	2021-05-19 20:01:51.492765
147	90	20	20	2021-05-19 20:01:51.500057	2021-05-19 20:01:51.500057
148	92	20	20	2021-05-19 20:01:51.505548	2021-05-19 20:01:51.505548
149	91	20	20	2021-05-19 20:01:51.512505	2021-05-19 20:01:51.512505
150	92	20	20	2021-05-19 20:01:51.542516	2021-05-19 20:01:51.542516
151	96	21	21	2021-05-20 14:12:30.293932	2021-05-20 14:12:30.293932
152	98	21	21	2021-05-20 14:12:30.299563	2021-05-20 14:12:30.299563
153	96	21	21	2021-05-20 14:12:30.305954	2021-05-20 14:12:30.305954
154	100	21	21	2021-05-20 14:12:30.31087	2021-05-20 14:12:30.31087
155	97	21	21	2021-05-20 14:12:30.317492	2021-05-20 14:12:30.317492
156	98	21	21	2021-05-20 14:12:30.322595	2021-05-20 14:12:30.322595
157	96	21	21	2021-05-20 14:12:30.329023	2021-05-20 14:12:30.329023
158	100	21	21	2021-05-20 14:12:30.334014	2021-05-20 14:12:30.334014
159	97	21	21	2021-05-20 14:12:30.340816	2021-05-20 14:12:30.340816
160	99	21	21	2021-05-20 14:12:30.345932	2021-05-20 14:12:30.345932
161	102	22	22	2021-05-20 14:12:57.034557	2021-05-20 14:12:57.034557
162	106	22	22	2021-05-20 14:12:57.039582	2021-05-20 14:12:57.039582
163	101	22	22	2021-05-20 14:12:57.047189	2021-05-20 14:12:57.047189
164	106	22	22	2021-05-20 14:12:57.05189	2021-05-20 14:12:57.05189
165	102	22	22	2021-05-20 14:12:57.057767	2021-05-20 14:12:57.057767
166	105	22	22	2021-05-20 14:12:57.062276	2021-05-20 14:12:57.062276
167	102	22	22	2021-05-20 14:12:57.069057	2021-05-20 14:12:57.069057
168	106	22	22	2021-05-20 14:12:57.074231	2021-05-20 14:12:57.074231
169	101	22	22	2021-05-20 14:12:57.080054	2021-05-20 14:12:57.080054
170	106	22	22	2021-05-20 14:12:57.084628	2021-05-20 14:12:57.084628
171	107	23	23	2021-05-20 14:13:33.881874	2021-05-20 14:13:33.881874
172	110	23	23	2021-05-20 14:13:33.886489	2021-05-20 14:13:33.886489
173	108	23	23	2021-05-20 14:13:33.892685	2021-05-20 14:13:33.892685
174	110	23	23	2021-05-20 14:13:33.900023	2021-05-20 14:13:33.900023
175	109	23	23	2021-05-20 14:13:33.906512	2021-05-20 14:13:33.906512
176	110	23	23	2021-05-20 14:13:33.910821	2021-05-20 14:13:33.910821
177	107	23	23	2021-05-20 14:13:33.916422	2021-05-20 14:13:33.916422
178	111	23	23	2021-05-20 14:13:33.920862	2021-05-20 14:13:33.920862
179	109	23	23	2021-05-20 14:13:33.92628	2021-05-20 14:13:33.92628
180	111	23	23	2021-05-20 14:13:33.930679	2021-05-20 14:13:33.930679
181	113	24	24	2021-05-20 14:21:20.417304	2021-05-20 14:21:20.417304
182	118	24	24	2021-05-20 14:21:20.425329	2021-05-20 14:21:20.425329
183	113	24	24	2021-05-20 14:21:20.432878	2021-05-20 14:21:20.432878
184	118	24	24	2021-05-20 14:21:20.439168	2021-05-20 14:21:20.439168
185	113	24	24	2021-05-20 14:21:20.446625	2021-05-20 14:21:20.446625
186	117	24	24	2021-05-20 14:21:20.45348	2021-05-20 14:21:20.45348
187	113	24	24	2021-05-20 14:21:20.46092	2021-05-20 14:21:20.46092
188	118	24	24	2021-05-20 14:21:20.465945	2021-05-20 14:21:20.465945
189	115	24	24	2021-05-20 14:21:20.472587	2021-05-20 14:21:20.472587
190	116	24	24	2021-05-20 14:21:20.479257	2021-05-20 14:21:20.479257
191	121	25	25	2021-05-20 14:30:30.204357	2021-05-20 14:30:30.204357
192	124	25	25	2021-05-20 14:30:30.209436	2021-05-20 14:30:30.209436
193	121	25	25	2021-05-20 14:30:30.215459	2021-05-20 14:30:30.215459
194	123	25	25	2021-05-20 14:30:30.220765	2021-05-20 14:30:30.220765
195	120	25	25	2021-05-20 14:30:30.227953	2021-05-20 14:30:30.227953
196	123	25	25	2021-05-20 14:30:30.23305	2021-05-20 14:30:30.23305
197	121	25	25	2021-05-20 14:30:30.238983	2021-05-20 14:30:30.238983
198	124	25	25	2021-05-20 14:30:30.243939	2021-05-20 14:30:30.243939
199	121	25	25	2021-05-20 14:30:30.250451	2021-05-20 14:30:30.250451
200	124	25	25	2021-05-20 14:30:30.25541	2021-05-20 14:30:30.25541
201	125	26	26	2021-05-20 15:17:11.583565	2021-05-20 15:17:11.583565
202	128	26	26	2021-05-20 15:17:11.589789	2021-05-20 15:17:11.589789
203	126	26	26	2021-05-20 15:17:11.599998	2021-05-20 15:17:11.599998
204	129	26	26	2021-05-20 15:17:11.604954	2021-05-20 15:17:11.604954
205	125	26	26	2021-05-20 15:17:11.611067	2021-05-20 15:17:11.611067
206	128	26	26	2021-05-20 15:17:11.616475	2021-05-20 15:17:11.616475
207	125	26	26	2021-05-20 15:17:11.623661	2021-05-20 15:17:11.623661
208	130	26	26	2021-05-20 15:17:11.62885	2021-05-20 15:17:11.62885
209	126	26	26	2021-05-20 15:17:11.635536	2021-05-20 15:17:11.635536
210	128	26	26	2021-05-20 15:17:11.640209	2021-05-20 15:17:11.640209
211	131	27	27	2021-05-28 15:07:42.303041	2021-05-28 15:07:42.303041
212	134	27	27	2021-05-28 15:07:42.359159	2021-05-28 15:07:42.359159
213	131	27	27	2021-05-28 15:07:42.393006	2021-05-28 15:07:42.393006
214	136	27	27	2021-05-28 15:07:42.40669	2021-05-28 15:07:42.40669
215	132	27	27	2021-05-28 15:07:42.466881	2021-05-28 15:07:42.466881
216	136	27	27	2021-05-28 15:07:42.496848	2021-05-28 15:07:42.496848
217	133	27	27	2021-05-28 15:07:42.932805	2021-05-28 15:07:42.932805
218	134	27	27	2021-05-28 15:07:42.973041	2021-05-28 15:07:42.973041
219	133	27	27	2021-05-28 15:07:43.048586	2021-05-28 15:07:43.048586
220	136	27	27	2021-05-28 15:07:43.087222	2021-05-28 15:07:43.087222
221	137	28	28	2021-06-02 16:10:01.232448	2021-06-02 16:10:01.232448
222	140	28	28	2021-06-02 16:10:01.265822	2021-06-02 16:10:01.265822
223	139	28	28	2021-06-02 16:10:01.290051	2021-06-02 16:10:01.290051
224	141	28	28	2021-06-02 16:10:01.306344	2021-06-02 16:10:01.306344
225	137	28	28	2021-06-02 16:10:01.331699	2021-06-02 16:10:01.331699
226	140	28	28	2021-06-02 16:10:01.345884	2021-06-02 16:10:01.345884
227	139	28	28	2021-06-02 16:10:01.364278	2021-06-02 16:10:01.364278
228	142	28	28	2021-06-02 16:10:01.381805	2021-06-02 16:10:01.381805
229	137	28	28	2021-06-02 16:10:01.391964	2021-06-02 16:10:01.391964
230	141	28	28	2021-06-02 16:10:01.40407	2021-06-02 16:10:01.40407
231	145	30	30	2021-08-23 14:58:27.324897	2021-08-23 14:58:27.324897
232	148	30	30	2021-08-23 14:58:27.343521	2021-08-23 14:58:27.343521
233	144	30	30	2021-08-23 14:58:27.354703	2021-08-23 14:58:27.354703
234	147	30	30	2021-08-23 14:58:27.360094	2021-08-23 14:58:27.360094
235	144	30	30	2021-08-23 14:58:27.367231	2021-08-23 14:58:27.367231
236	146	30	30	2021-08-23 14:58:27.372525	2021-08-23 14:58:27.372525
237	145	30	30	2021-08-23 14:58:27.378503	2021-08-23 14:58:27.378503
238	147	30	30	2021-08-23 14:58:27.383029	2021-08-23 14:58:27.383029
239	143	30	30	2021-08-23 14:58:27.389663	2021-08-23 14:58:27.389663
240	148	30	30	2021-08-23 14:58:27.394566	2021-08-23 14:58:27.394566
241	150	31	31	2021-08-23 17:19:39.079886	2021-08-23 17:19:39.079886
242	154	31	31	2021-08-23 17:19:39.085247	2021-08-23 17:19:39.085247
243	150	31	31	2021-08-23 17:19:39.091606	2021-08-23 17:19:39.091606
244	154	31	31	2021-08-23 17:19:39.097357	2021-08-23 17:19:39.097357
245	149	31	31	2021-08-23 17:19:39.104537	2021-08-23 17:19:39.104537
246	154	31	31	2021-08-23 17:19:39.110172	2021-08-23 17:19:39.110172
247	151	31	31	2021-08-23 17:19:39.117309	2021-08-23 17:19:39.117309
248	153	31	31	2021-08-23 17:19:39.123183	2021-08-23 17:19:39.123183
249	149	31	31	2021-08-23 17:19:39.129642	2021-08-23 17:19:39.129642
250	152	31	31	2021-08-23 17:19:39.134561	2021-08-23 17:19:39.134561
\.


--
-- Data for Name: repository_rows; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repository_rows (id, repository_id, created_by_id, last_modified_by_id, name, created_at, updated_at, parent_id, archived, archived_on, restored_on, archived_by_id, restored_by_id) FROM stdin;
1	1	1	1	FBT/1	2019-05-30 12:40:52.98973	2019-05-30 12:40:52.98973	\N	f	\N	\N	\N	\N
2	1	1	1	FBT/2	2019-05-30 12:40:53.035158	2019-05-30 12:40:53.035158	\N	f	\N	\N	\N	\N
3	1	1	1	FBT/3	2019-05-30 12:40:53.047182	2019-05-30 12:40:53.047182	\N	f	\N	\N	\N	\N
4	1	1	1	FBT/4	2019-05-30 12:40:53.059176	2019-05-30 12:40:53.059176	\N	f	\N	\N	\N	\N
5	1	1	1	FBT/5	2019-05-30 12:40:53.071024	2019-05-30 12:40:53.071024	\N	f	\N	\N	\N	\N
6	2	4	4	PCH/1	2019-08-27 15:14:58.626789	2019-08-27 15:14:58.626789	\N	f	\N	\N	\N	\N
7	2	4	4	PCH/2	2019-08-27 15:14:58.797932	2019-08-27 15:14:58.797932	\N	f	\N	\N	\N	\N
8	2	4	4	PCH/3	2019-08-27 15:14:58.836004	2019-08-27 15:14:58.836004	\N	f	\N	\N	\N	\N
9	2	4	4	PCH/4	2019-08-27 15:14:58.865047	2019-08-27 15:14:58.865047	\N	f	\N	\N	\N	\N
10	2	4	4	PCH/5	2019-08-27 15:14:58.91709	2019-08-27 15:14:58.91709	\N	f	\N	\N	\N	\N
11	3	1	1	EZP/1	2019-08-28 07:30:57.337133	2019-08-28 07:30:57.337133	\N	f	\N	\N	\N	\N
12	3	1	1	EZP/2	2019-08-28 07:30:58.222302	2019-08-28 07:30:58.222302	\N	f	\N	\N	\N	\N
13	3	1	1	EZP/3	2019-08-28 07:30:58.241739	2019-08-28 07:30:58.241739	\N	f	\N	\N	\N	\N
14	3	1	1	EZP/4	2019-08-28 07:30:58.267243	2019-08-28 07:30:58.267243	\N	f	\N	\N	\N	\N
15	3	1	1	EZP/5	2019-08-28 07:30:58.287175	2019-08-28 07:30:58.287175	\N	f	\N	\N	\N	\N
16	4	6	6	RLY/1	2019-08-28 08:34:51.663104	2019-08-28 08:34:51.663104	\N	f	\N	\N	\N	\N
17	4	6	6	RLY/2	2019-08-28 08:34:51.702138	2019-08-28 08:34:51.702138	\N	f	\N	\N	\N	\N
18	4	6	6	RLY/3	2019-08-28 08:34:51.720077	2019-08-28 08:34:51.720077	\N	f	\N	\N	\N	\N
19	4	6	6	RLY/4	2019-08-28 08:34:51.747629	2019-08-28 08:34:51.747629	\N	f	\N	\N	\N	\N
20	4	6	6	RLY/5	2019-08-28 08:34:52.297732	2019-08-28 08:34:52.297732	\N	f	\N	\N	\N	\N
26	6	7	7	ExTaq	2019-08-30 16:11:26.114768	2019-08-30 16:11:26.114768	\N	f	\N	\N	\N	\N
27	9	8	8	LJK/1	2019-09-12 18:26:18.169605	2019-09-12 18:26:18.169605	\N	f	\N	\N	\N	\N
28	9	8	8	LJK/2	2019-09-12 18:26:18.210852	2019-09-12 18:26:18.210852	\N	f	\N	\N	\N	\N
29	9	8	8	LJK/3	2019-09-12 18:26:18.226704	2019-09-12 18:26:18.226704	\N	f	\N	\N	\N	\N
30	9	8	8	LJK/4	2019-09-12 18:26:18.242381	2019-09-12 18:26:18.242381	\N	f	\N	\N	\N	\N
31	9	8	8	LJK/5	2019-09-12 18:26:18.257413	2019-09-12 18:26:18.257413	\N	f	\N	\N	\N	\N
32	10	9	9	MSZ/1	2019-11-13 13:57:41.058312	2019-11-13 13:57:41.058312	\N	f	\N	\N	\N	\N
33	10	9	9	MSZ/2	2019-11-13 13:57:41.126036	2019-11-13 13:57:41.126036	\N	f	\N	\N	\N	\N
34	10	9	9	MSZ/3	2019-11-13 13:57:41.147876	2019-11-13 13:57:41.147876	\N	f	\N	\N	\N	\N
35	10	9	9	MSZ/4	2019-11-13 13:57:41.172058	2019-11-13 13:57:41.172058	\N	f	\N	\N	\N	\N
36	10	9	9	MSZ/5	2019-11-13 13:57:41.197665	2019-11-13 13:57:41.197665	\N	f	\N	\N	\N	\N
37	11	4	4	KOK/1	2019-11-25 14:09:21.130391	2019-11-25 14:09:21.130391	\N	f	\N	\N	\N	\N
38	11	4	4	KOK/2	2019-11-25 14:09:21.332233	2019-11-25 14:09:21.332233	\N	f	\N	\N	\N	\N
39	11	4	4	KOK/3	2019-11-25 14:09:21.370103	2019-11-25 14:09:21.370103	\N	f	\N	\N	\N	\N
40	11	4	4	KOK/4	2019-11-25 14:09:21.414248	2019-11-25 14:09:21.414248	\N	f	\N	\N	\N	\N
41	11	4	4	KOK/5	2019-11-25 14:09:21.446338	2019-11-25 14:09:21.446338	\N	f	\N	\N	\N	\N
42	12	10	10	HOY/1	2019-11-25 18:00:52.179307	2019-11-25 18:00:52.179307	\N	f	\N	\N	\N	\N
43	12	10	10	HOY/2	2019-11-25 18:00:52.202508	2019-11-25 18:00:52.202508	\N	f	\N	\N	\N	\N
44	12	10	10	HOY/3	2019-11-25 18:00:52.220847	2019-11-25 18:00:52.220847	\N	f	\N	\N	\N	\N
45	12	10	10	HOY/4	2019-11-25 18:00:52.239258	2019-11-25 18:00:52.239258	\N	f	\N	\N	\N	\N
46	12	10	10	HOY/5	2019-11-25 18:00:52.257119	2019-11-25 18:00:52.257119	\N	f	\N	\N	\N	\N
47	13	11	11	DYB/1	2020-05-12 16:13:01.071942	2020-05-12 16:13:01.071942	\N	f	\N	\N	\N	\N
48	13	11	11	DYB/2	2020-05-12 16:13:01.153155	2020-05-12 16:13:01.153155	\N	f	\N	\N	\N	\N
49	13	11	11	DYB/3	2020-05-12 16:13:01.175884	2020-05-12 16:13:01.175884	\N	f	\N	\N	\N	\N
50	13	11	11	DYB/4	2020-05-12 16:13:01.20504	2020-05-12 16:13:01.20504	\N	f	\N	\N	\N	\N
51	13	11	11	DYB/5	2020-05-12 16:13:01.229014	2020-05-12 16:13:01.229014	\N	f	\N	\N	\N	\N
52	14	12	12	RTK/1	2020-06-10 14:43:16.515736	2020-06-10 14:43:16.515736	\N	f	\N	\N	\N	\N
53	14	12	12	RTK/2	2020-06-10 14:43:16.569293	2020-06-10 14:43:16.569293	\N	f	\N	\N	\N	\N
54	14	12	12	RTK/3	2020-06-10 14:43:16.586012	2020-06-10 14:43:16.586012	\N	f	\N	\N	\N	\N
55	14	12	12	RTK/4	2020-06-10 14:43:16.602214	2020-06-10 14:43:16.602214	\N	f	\N	\N	\N	\N
56	14	12	12	RTK/5	2020-06-10 14:43:16.617632	2020-06-10 14:43:16.617632	\N	f	\N	\N	\N	\N
57	15	16	16	DPC/1	2020-10-07 19:20:24.203993	2020-10-07 19:20:24.203993	\N	f	\N	\N	\N	\N
58	15	16	16	DPC/2	2020-10-07 19:20:24.399825	2020-10-07 19:20:24.399825	\N	f	\N	\N	\N	\N
59	15	16	16	DPC/3	2020-10-07 19:20:24.430972	2020-10-07 19:20:24.430972	\N	f	\N	\N	\N	\N
60	15	16	16	DPC/4	2020-10-07 19:20:24.461804	2020-10-07 19:20:24.461804	\N	f	\N	\N	\N	\N
61	15	16	16	DPC/5	2020-10-07 19:20:24.490242	2020-10-07 19:20:24.490242	\N	f	\N	\N	\N	\N
62	16	1	1	DYV/1	2020-11-18 14:50:51.678936	2020-11-18 14:50:51.678936	\N	f	\N	\N	\N	\N
63	16	1	1	DYV/2	2020-11-18 14:50:51.76076	2020-11-18 14:50:51.76076	\N	f	\N	\N	\N	\N
64	16	1	1	DYV/3	2020-11-18 14:50:51.784369	2020-11-18 14:50:51.784369	\N	f	\N	\N	\N	\N
65	16	1	1	DYV/4	2020-11-18 14:50:51.810586	2020-11-18 14:50:51.810586	\N	f	\N	\N	\N	\N
66	16	1	1	DYV/5	2020-11-18 14:50:51.836632	2020-11-18 14:50:51.836632	\N	f	\N	\N	\N	\N
67	17	17	17	ROW/1	2020-11-18 14:53:24.462593	2020-11-18 14:53:24.462593	\N	f	\N	\N	\N	\N
68	17	17	17	ROW/2	2020-11-18 14:53:24.504138	2020-11-18 14:53:24.504138	\N	f	\N	\N	\N	\N
69	17	17	17	ROW/3	2020-11-18 14:53:24.522817	2020-11-18 14:53:24.522817	\N	f	\N	\N	\N	\N
70	17	17	17	ROW/4	2020-11-18 14:53:24.541791	2020-11-18 14:53:24.541791	\N	f	\N	\N	\N	\N
71	17	17	17	ROW/5	2020-11-18 14:53:24.559859	2020-11-18 14:53:24.559859	\N	f	\N	\N	\N	\N
72	18	20	20	ROD/1	2021-05-19 20:01:51.447659	2021-05-19 20:01:51.447659	\N	f	\N	\N	\N	\N
73	18	20	20	ROD/2	2021-05-19 20:01:51.470527	2021-05-19 20:01:51.470527	\N	f	\N	\N	\N	\N
74	18	20	20	ROD/3	2021-05-19 20:01:51.482777	2021-05-19 20:01:51.482777	\N	f	\N	\N	\N	\N
75	18	20	20	ROD/4	2021-05-19 20:01:51.495331	2021-05-19 20:01:51.495331	\N	f	\N	\N	\N	\N
76	18	20	20	ROD/5	2021-05-19 20:01:51.507769	2021-05-19 20:01:51.507769	\N	f	\N	\N	\N	\N
77	19	21	21	FDL/1	2021-05-20 14:12:30.287115	2021-05-20 14:12:30.287115	\N	f	\N	\N	\N	\N
78	19	21	21	FDL/2	2021-05-20 14:12:30.301843	2021-05-20 14:12:30.301843	\N	f	\N	\N	\N	\N
79	19	21	21	FDL/3	2021-05-20 14:12:30.313101	2021-05-20 14:12:30.313101	\N	f	\N	\N	\N	\N
80	19	21	21	FDL/4	2021-05-20 14:12:30.324864	2021-05-20 14:12:30.324864	\N	f	\N	\N	\N	\N
81	19	21	21	FDL/5	2021-05-20 14:12:30.336556	2021-05-20 14:12:30.336556	\N	f	\N	\N	\N	\N
82	20	22	22	ZIO/1	2021-05-20 14:12:57.029757	2021-05-20 14:12:57.029757	\N	f	\N	\N	\N	\N
83	20	22	22	ZIO/2	2021-05-20 14:12:57.041826	2021-05-20 14:12:57.041826	\N	f	\N	\N	\N	\N
84	20	22	22	ZIO/3	2021-05-20 14:12:57.053947	2021-05-20 14:12:57.053947	\N	f	\N	\N	\N	\N
85	20	22	22	ZIO/4	2021-05-20 14:12:57.064328	2021-05-20 14:12:57.064328	\N	f	\N	\N	\N	\N
86	20	22	22	ZIO/5	2021-05-20 14:12:57.07627	2021-05-20 14:12:57.07627	\N	f	\N	\N	\N	\N
87	21	23	23	QEA/1	2021-05-20 14:13:33.877692	2021-05-20 14:13:33.877692	\N	f	\N	\N	\N	\N
88	21	23	23	QEA/2	2021-05-20 14:13:33.888712	2021-05-20 14:13:33.888712	\N	f	\N	\N	\N	\N
89	21	23	23	QEA/3	2021-05-20 14:13:33.902547	2021-05-20 14:13:33.902547	\N	f	\N	\N	\N	\N
90	21	23	23	QEA/4	2021-05-20 14:13:33.912749	2021-05-20 14:13:33.912749	\N	f	\N	\N	\N	\N
91	21	23	23	QEA/5	2021-05-20 14:13:33.922776	2021-05-20 14:13:33.922776	\N	f	\N	\N	\N	\N
92	22	24	24	VRK/1	2021-05-20 14:21:20.41047	2021-05-20 14:21:20.41047	\N	f	\N	\N	\N	\N
93	22	24	24	VRK/2	2021-05-20 14:21:20.428169	2021-05-20 14:21:20.428169	\N	f	\N	\N	\N	\N
94	22	24	24	VRK/3	2021-05-20 14:21:20.441655	2021-05-20 14:21:20.441655	\N	f	\N	\N	\N	\N
95	22	24	24	VRK/4	2021-05-20 14:21:20.456701	2021-05-20 14:21:20.456701	\N	f	\N	\N	\N	\N
96	22	24	24	VRK/5	2021-05-20 14:21:20.468356	2021-05-20 14:21:20.468356	\N	f	\N	\N	\N	\N
97	23	25	25	YKK/1	2021-05-20 14:30:30.19885	2021-05-20 14:30:30.19885	\N	f	\N	\N	\N	\N
98	23	25	25	YKK/2	2021-05-20 14:30:30.211577	2021-05-20 14:30:30.211577	\N	f	\N	\N	\N	\N
99	23	25	25	YKK/3	2021-05-20 14:30:30.22323	2021-05-20 14:30:30.22323	\N	f	\N	\N	\N	\N
100	23	25	25	YKK/4	2021-05-20 14:30:30.235179	2021-05-20 14:30:30.235179	\N	f	\N	\N	\N	\N
101	23	25	25	YKK/5	2021-05-20 14:30:30.245972	2021-05-20 14:30:30.245972	\N	f	\N	\N	\N	\N
102	24	26	26	MQU/1	2021-05-20 15:17:11.576656	2021-05-20 15:17:11.576656	\N	f	\N	\N	\N	\N
103	24	26	26	MQU/2	2021-05-20 15:17:11.592307	2021-05-20 15:17:11.592307	\N	f	\N	\N	\N	\N
104	24	26	26	MQU/3	2021-05-20 15:17:11.607139	2021-05-20 15:17:11.607139	\N	f	\N	\N	\N	\N
105	24	26	26	MQU/4	2021-05-20 15:17:11.61879	2021-05-20 15:17:11.61879	\N	f	\N	\N	\N	\N
106	24	26	26	MQU/5	2021-05-20 15:17:11.631504	2021-05-20 15:17:11.631504	\N	f	\N	\N	\N	\N
107	25	27	27	OJT/1	2021-05-28 15:07:41.892188	2021-05-28 15:07:41.892188	\N	f	\N	\N	\N	\N
108	25	27	27	OJT/2	2021-05-28 15:07:42.37283	2021-05-28 15:07:42.37283	\N	f	\N	\N	\N	\N
109	25	27	27	OJT/3	2021-05-28 15:07:42.417005	2021-05-28 15:07:42.417005	\N	f	\N	\N	\N	\N
110	25	27	27	OJT/4	2021-05-28 15:07:42.854972	2021-05-28 15:07:42.854972	\N	f	\N	\N	\N	\N
111	25	27	27	OJT/5	2021-05-28 15:07:42.985231	2021-05-28 15:07:42.985231	\N	f	\N	\N	\N	\N
112	26	28	28	AUG/1	2021-06-02 16:10:01.160469	2021-06-02 16:10:01.160469	\N	f	\N	\N	\N	\N
113	26	28	28	AUG/2	2021-06-02 16:10:01.276125	2021-06-02 16:10:01.276125	\N	f	\N	\N	\N	\N
114	26	28	28	AUG/3	2021-06-02 16:10:01.320758	2021-06-02 16:10:01.320758	\N	f	\N	\N	\N	\N
115	26	28	28	AUG/4	2021-06-02 16:10:01.351863	2021-06-02 16:10:01.351863	\N	f	\N	\N	\N	\N
116	26	28	28	AUG/5	2021-06-02 16:10:01.384795	2021-06-02 16:10:01.384795	\N	f	\N	\N	\N	\N
117	27	30	30	CKC/1	2021-08-23 14:58:27.293486	2021-08-23 14:58:27.293486	\N	f	\N	\N	\N	\N
118	27	30	30	CKC/2	2021-08-23 14:58:27.347584	2021-08-23 14:58:27.347584	\N	f	\N	\N	\N	\N
119	27	30	30	CKC/3	2021-08-23 14:58:27.363024	2021-08-23 14:58:27.363024	\N	f	\N	\N	\N	\N
120	27	30	30	CKC/4	2021-08-23 14:58:27.374637	2021-08-23 14:58:27.374637	\N	f	\N	\N	\N	\N
121	27	30	30	CKC/5	2021-08-23 14:58:27.384966	2021-08-23 14:58:27.384966	\N	f	\N	\N	\N	\N
122	28	31	31	NXT/1	2021-08-23 17:19:39.074761	2021-08-23 17:19:39.074761	\N	f	\N	\N	\N	\N
123	28	31	31	NXT/2	2021-08-23 17:19:39.087311	2021-08-23 17:19:39.087311	\N	f	\N	\N	\N	\N
124	28	31	31	NXT/3	2021-08-23 17:19:39.100046	2021-08-23 17:19:39.100046	\N	f	\N	\N	\N	\N
125	28	31	31	NXT/4	2021-08-23 17:19:39.112855	2021-08-23 17:19:39.112855	\N	f	\N	\N	\N	\N
126	28	31	31	NXT/5	2021-08-23 17:19:39.125441	2021-08-23 17:19:39.125441	\N	f	\N	\N	\N	\N
127	27	30	30	Surrogate Solution	2021-08-24 16:51:31.992595	2021-08-24 16:51:31.992595	\N	f	\N	\N	\N	\N
128	27	30	30	Clean Soil 1/1	2021-08-26 20:03:53.162191	2021-08-26 20:03:53.162191	\N	f	\N	\N	\N	\N
\.

--
-- Data for Name: repository_table_states; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.repository_table_states (id, state, user_id, repository_id, created_at, updated_at) FROM stdin;
1	{"time": "1621334766063", "order": [["5", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	1	1	2019-05-30 12:51:24.918637	2021-09-30 14:06:04.266953
2	{"time": "1566920298520", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	4	2	2019-08-27 15:35:51.075036	2021-09-30 14:06:04.269296
3	{"time": "1568298963417", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	7	5	2019-08-29 14:19:20.604173	2021-09-30 14:06:04.271266
4	{"time": "1567181520038", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}], "ColReorder": [0, 1, 2, 3, 4, 5, 6, 7]}	7	6	2019-08-30 16:08:40.672443	2021-09-30 14:06:04.272998
5	{"time": "1567181435147", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}], "ColReorder": [0, 1, 2, 3, 4, 5, 6, 7]}	7	7	2019-08-30 16:10:23.382828	2021-09-30 14:06:04.274642
6	{"time": "1567181436172", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}], "ColReorder": [0, 1, 2, 3, 4, 5, 6, 7]}	7	8	2019-08-30 16:10:30.465382	2021-09-30 14:06:04.276212
7	{"time": "1568298926773", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	1	3	2019-09-12 14:35:25.085985	2021-09-30 14:06:04.277838
8	{"time": "1574692303736", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	4	11	2019-11-25 14:31:43.161431	2021-09-30 14:06:04.279596
9	{"time": "1574704875924", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	10	12	2019-11-25 18:01:17.196908	2021-09-30 14:06:04.2817
10	{"time": "1589299993065", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	11	13	2020-05-12 16:13:12.574165	2021-09-30 14:06:04.283499
11	{"time": "1590691577451", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	10	11	2020-05-28 18:46:13.709352	2021-09-30 14:06:04.285301
12	{"time": "1591800226276", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	12	14	2020-06-10 14:43:46.027043	2021-09-30 14:06:04.28721
13	{"time": "1617887201958", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	6	4	2021-04-08 13:06:41.127241	2021-09-30 14:06:04.288975
14	{"time": "1621520033681", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	23	21	2021-05-20 14:13:52.934527	2021-09-30 14:06:04.290778
15	{"time": "1621520711958", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	24	22	2021-05-20 14:25:11.665179	2021-09-30 14:06:04.292649
16	{"time": "1621522852112", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	22	20	2021-05-20 15:00:51.643229	2021-09-30 14:06:04.294416
17	{"time": "1622040017164", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	21	19	2021-05-26 14:40:16.098895	2021-09-30 14:06:04.296128
18	{"time": "1623698656642", "order": [["1", "desc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	27	25	2021-06-01 13:44:00.477767	2021-09-30 14:06:04.297881
19	{"time": "1622652228315", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	28	26	2021-06-02 16:43:47.368226	2021-09-30 14:06:04.299634
20	{"time": "1629739257855", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	31	28	2021-08-23 17:20:57.601065	2021-09-30 14:06:04.301475
21	{"time": "1629752057567", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	30	27	2021-08-23 20:54:16.801528	2021-09-30 14:06:04.303248
22	{"time": "1629900088295", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	30	28	2021-08-25 14:01:27.698231	2021-09-30 14:06:04.304984
23	{"time": "1629905863605", "order": [["2", "asc"]], "start": "0", "length": "10", "search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "columns": [{"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}, {"search": {"regex": "false", "smart": "true", "search": "", "caseInsensitive": "true"}, "visible": "true"}], "ColReorder": [0, 1, 2, 3, 4, 5, 8, 9, 6, 7]}	31	27	2021-08-25 15:37:43.310157	2021-09-30 14:06:04.306779
24	{"time": 1644329830303, "order": [[2, "asc"]], "start": 0, "length": 10, "search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "columns": [{"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": false}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}, {"search": {"regex": false, "smart": true, "search": "", "caseInsensitive": true}, "visible": true}], "ColReorder": [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]}	33	1	2022-02-08 14:17:10.093454	2022-02-08 14:17:10.329134
\.

--
-- Data for Name: result_assets; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.result_assets (id, result_id, asset_id) FROM stdin;
1	10	2
2	11	3
3	12	4
4	13	6
5	14	12
6	15	13
7	16	14
8	17	15
9	18	16
10	19	22
11	20	23
12	21	24
13	31	26
14	32	27
15	33	28
16	34	30
17	35	36
18	36	37
19	37	38
20	38	39
21	39	40
22	40	46
23	41	47
24	42	48
25	52	50
26	53	51
27	54	52
28	55	54
29	56	60
30	57	61
31	58	62
32	59	63
33	60	64
34	61	70
35	62	71
36	63	72
37	73	74
38	74	75
39	75	76
40	76	78
41	77	84
42	78	85
43	79	86
44	80	87
45	81	88
46	82	94
47	83	95
48	84	96
49	94	98
50	95	99
51	96	100
52	97	102
53	98	108
54	99	109
55	100	110
56	101	111
57	102	112
58	103	118
59	104	119
60	105	120
61	115	122
62	116	123
63	117	124
64	118	126
65	119	132
66	120	133
67	121	134
68	122	135
69	123	136
70	124	142
71	125	143
72	126	144
73	136	146
74	137	147
75	138	148
76	139	150
77	140	156
78	141	157
79	142	158
80	143	159
81	144	160
82	145	166
83	146	167
84	147	168
85	157	170
86	158	171
87	159	172
88	160	174
89	161	180
90	162	181
91	163	182
92	164	183
93	165	184
94	166	190
95	167	191
96	168	192
97	178	194
98	179	195
99	180	196
100	181	198
101	182	204
102	183	205
103	184	206
104	185	207
105	186	208
106	187	214
107	188	215
108	189	216
109	199	218
110	200	219
111	201	220
112	202	222
113	203	228
114	204	229
115	205	230
116	206	231
117	207	232
118	208	238
119	209	239
120	210	240
121	220	242
122	221	243
123	222	244
124	223	246
125	224	252
126	225	253
127	226	254
128	227	255
129	228	256
130	229	262
131	230	263
132	231	264
133	241	266
134	242	267
135	243	268
136	244	270
137	245	276
138	246	277
139	247	278
140	248	279
141	249	280
142	250	286
143	251	287
144	252	288
145	262	290
146	263	291
147	264	292
148	265	294
149	266	300
150	267	301
151	268	302
152	269	303
153	270	304
154	271	310
155	272	311
156	273	312
157	283	314
158	284	315
159	285	316
160	286	318
161	287	324
162	288	325
163	289	326
164	290	327
165	291	328
166	292	334
167	293	335
168	294	336
169	295	338
170	307	340
171	308	341
172	309	342
173	310	344
174	311	350
175	312	351
176	313	352
177	314	353
178	315	354
179	316	360
180	317	361
181	318	362
182	337	364
183	338	365
184	339	366
185	340	368
186	341	374
187	342	375
188	343	376
189	344	377
190	345	378
191	346	384
192	347	385
193	348	386
194	349	388
195	351	389
196	354	390
197	361	392
198	362	398
199	363	399
200	364	400
201	365	401
202	366	402
203	367	408
204	368	409
205	369	410
206	370	412
207	371	413
208	372	414
209	373	416
210	374	422
211	375	423
212	376	424
213	377	425
214	378	426
215	379	432
216	380	433
217	381	434
218	391	436
219	392	437
220	393	438
221	394	440
222	395	446
223	396	447
224	397	448
225	398	449
226	399	450
227	400	456
228	401	457
229	402	458
230	412	460
231	413	461
232	414	462
233	415	464
234	416	470
235	417	471
236	418	472
237	419	473
238	420	474
239	421	480
240	422	481
241	423	482
242	433	485
243	434	486
244	435	487
245	436	489
246	437	495
247	438	496
248	439	497
249	440	498
250	441	499
251	442	505
252	443	506
253	444	507
254	454	509
255	455	510
256	456	511
257	457	513
258	458	519
259	459	520
260	460	521
261	461	522
262	462	523
263	463	529
264	464	530
265	465	531
266	475	533
267	476	534
268	477	535
269	478	537
270	479	543
271	480	544
272	481	545
273	482	546
274	483	547
275	484	553
276	485	554
277	486	555
278	497	557
279	498	558
280	499	559
281	500	561
282	501	567
283	502	568
284	503	569
285	504	570
286	505	571
287	506	577
288	507	578
289	508	579
290	518	581
291	519	582
292	520	583
293	521	585
294	522	591
295	523	592
296	524	593
297	525	594
298	526	595
299	527	601
300	528	602
301	529	603
\.




--
-- Data for Name: result_texts; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.result_texts (id, text, result_id) FROM stdin;
1	There are many biological replicates we harvested for each type of sample (code-names):\n\n* NSK/3\n\n* NSK/5\n\n* CQR/5\n\n* CQR/1	1
2	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	5
3	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	6
4	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	7
5	There are many biological replicates we harvested for each type of sample (code-names):\n\n* PBZ/5\n\n* PBZ/4\n\n* PBZ/1\n\n* PBZ/3	22
6	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	26
7	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	27
8	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	28
9	There are many biological replicates we harvested for each type of sample (code-names):\n\n* KGM/1\n\n* QXE/3\n\n* KGM/3\n\n* QXE/5	43
10	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	47
11	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	48
12	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	49
13	There are many biological replicates we harvested for each type of sample (code-names):\n\n* DDI/4\n\n* DDI/3\n\n* SDU/1\n\n* SDU/4	64
14	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	68
15	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	69
16	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	70
17	There are many biological replicates we harvested for each type of sample (code-names):\n\n* KOU/3\n\n* RUB/2\n\n* RUB/3\n\n* KOU/4	85
18	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	89
19	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	90
20	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	91
21	There are many biological replicates we harvested for each type of sample (code-names):\n\n* UNA/1\n\n* UNA/5\n\n* WZV/5\n\n* UNA/3	106
22	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	110
23	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	111
24	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	112
25	There are many biological replicates we harvested for each type of sample (code-names):\n\n* ZVE/3\n\n* IBU/3\n\n* ZVE/1\n\n* ZVE/2	127
26	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	131
27	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	132
28	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	133
29	There are many biological replicates we harvested for each type of sample (code-names):\n\n* MFW/2\n\n* MFW/1\n\n* MFW/4\n\n* DAB/5	148
30	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	152
31	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	153
32	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	154
33	There are many biological replicates we harvested for each type of sample (code-names):\n\n* VFH/2\n\n* VFH/3\n\n* VCH/3\n\n* VFH/1	169
34	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	173
35	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	174
36	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	175
37	There are many biological replicates we harvested for each type of sample (code-names):\n\n* ZHI/5\n\n* UMA/1\n\n* UMA/3\n\n* UMA/2	190
38	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	194
39	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	195
40	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	196
41	There are many biological replicates we harvested for each type of sample (code-names):\n\n* GJH/4\n\n* LIZ/4\n\n* GJH/1\n\n* LIZ/2	211
42	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	215
43	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	216
44	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	217
45	There are many biological replicates we harvested for each type of sample (code-names):\n\n* TYM/1\n\n* TYM/4\n\n* CQX/2\n\n* CQX/3	232
46	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	236
47	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	237
48	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	238
49	There are many biological replicates we harvested for each type of sample (code-names):\n\n* GQN/5\n\n* DKO/1\n\n* GQN/2\n\n* GQN/3	253
50	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	257
51	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	258
52	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	259
53	There are many biological replicates we harvested for each type of sample (code-names):\n\n* FTZ/2\n\n* FTZ/5\n\n* FTZ/1\n\n* DWB/2	274
54	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	278
55	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	279
56	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	280
57	<div>\r\n<ul>\r\n<li><span style="color: #ff6600;"> Use the MEDIA program: using a timer/stop watch, hit the start button on timer simultaneously as you start the cycle. </span></li>\r\n<li><span style="color: #ff6600;">Once the warm up cycle ends and the sterilization begins, Record time and start timing the sterilization cycle. </span></li>\r\n<li><span style="color: #ff6600;">Once the sterilization cycle ends, Record time and begin timing the exhaust to end cycle. </span></li>\r\n<li><span style="color: #ff6600;">Once the exhaust cycle ends, Record time. </span></li>\r\n<li><span style="color: #ff6600;">All three cycle times should add up to or less than 45 minutes. o If the sterilization time is more than the set 15 minutes adjust time and check again. </span>\r\n<ul>\r\n<li><span style="color: #ff6600;">Check the batteries of the timer and check again. </span></li>\r\n<li><span style="color: #ff6600;">Or if needed, call for service tech. to check the cycles.</span></li>\r\n</ul>\r\n</li>\r\n</ul>\r\n<table style="width: 100%; height: 226px; border-collapse: collapse;" border="1">\r\n<tbody>\r\n<tr style="height: 55px;">\r\n<td style="width: 14.28%; height: 55px; text-align: center;"><span style="color: #0000ff;"><strong>Date</strong></span></td>\r\n<td style="width: 14.28%; height: 55px; text-align: center;">\r\n<div><span style="color: #0000ff;"><strong>Time from Start to Cycle Sterilization </strong></span></div>\r\n<div><span style="color: #0000ff;"><strong>(minutes : seconds)</strong></span></div>\r\n</td>\r\n<td style="width: 14.28%; height: 55px; text-align: center;">\r\n<div><span style="color: #0000ff;"><strong>Time of Sterilization 15 min @ 15psi/121C</strong></span></div>\r\n<div><span style="color: #0000ff;"><strong>(minutes : seconds)</strong></span></div>\r\n</td>\r\n<td style="width: 14.28%; height: 55px; text-align: center;">\r\n<div><span style="color: #0000ff;"><strong>Time of Exhaust to End of Cycle</strong></span></div>\r\n<div><span style="color: #0000ff;"><strong>(minutes : seconds)</strong></span></div>\r\n</td>\r\n<td style="width: 14.28%; height: 55px; text-align: center;">\r\n<div><span style="color: #0000ff;"><strong>Total Cycle Time</strong></span></div>\r\n<div><span style="color: #0000ff;"><strong>&lt; 45 min</strong></span></div>\r\n<div><span style="color: #0000ff;"><strong>(minutes : seconds)</strong></span></div>\r\n</td>\r\n<td style="width: 14.28%; height: 55px; text-align: center;">\r\n<div><span style="color: #0000ff;"><strong>Acceptable</strong></span></div>\r\n<div><span style="color: #0000ff;"><strong>Yes or No</strong></span></div>\r\n</td>\r\n<td style="width: 14.28%; height: 55px; text-align: center;"><span style="color: #0000ff;"><strong>Initials</strong></span></td>\r\n</tr>\r\n<tr style="height: 19px;">\r\n<td style="width: 14.28%; height: 19px; text-align: center;">EXAMPLE</td>\r\n<td style="width: 14.28%; height: 19px; text-align: center;">7:05</td>\r\n<td style="width: 14.28%; height: 19px; text-align: center;">15:05</td>\r\n<td style="width: 14.28%; height: 19px; text-align: center;">20:05</td>\r\n<td style="width: 14.28%; height: 19px; text-align: center;">42:15</td>\r\n<td style="width: 14.28%; height: 19px; text-align: center;">Yes</td>\r\n<td style="width: 14.28%; height: 19px; text-align: center;">EXAMPLE</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n<td style="width: 14.28%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n<td style="width: 14.28%;">&nbsp;</td>\r\n</tr>\r\n<tr>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n<td style="width: 14.28%; text-align: center;">&nbsp;</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>	296
58	<div>\r\n<p><strong>Prior to use: Flush and Drain the Autoclave</strong></p>\r\n<ol>\r\n<li style="font-weight: 400;">Touch Screen to start process</li>\r\n<li style="font-weight: 400;">&nbsp;Enter Username : STERIS</li>\r\n<li style="font-weight: 400;">&nbsp;Enter Password: 1000</li>\r\n<li style="font-weight: 400;">&nbsp;Open front door and open the drain by turning the knob counter clockwise</li>\r\n<li style="font-weight: 400;">&nbsp;Close front door</li>\r\n<li style="font-weight: 400;">&nbsp;Touch Screen : Start Timer (5 minutes)</li>\r\n<li style="font-weight: 400;">&nbsp;Wait for timer to end</li>\r\n<li style="font-weight: 400;">&nbsp;Open front door and close drain by turning knob clock wise</li>\r\n<li style="font-weight: 400;">&nbsp;Close front door</li>\r\n<li style="font-weight: 400;">&nbsp;Wait 15minutes prior to use</li>\r\n</ol>\r\n<div>\r\n<table style="width: 100%; height: 437px; border-collapse: collapse;" border="1">\r\n<tbody>\r\n<tr style="height: 19px;">\r\n<td style="width: 10%; height: 19px;"><strong>Date of Flush</strong></td>\r\n<td style="width: 10%; height: 19px;"><strong>Initials</strong></td>\r\n</tr>\r\n<tr style="height: 19px;">\r\n<td style="width: 10%; height: 19px; text-align: left;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px; text-align: center;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px; text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="text-align: center;">\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px;">&nbsp;</td>\r\n</tr>\r\n<tr style="height: 19px;">\r\n<td style="width: 10%; height: 19px; text-align: center;">&nbsp;</td>\r\n<td style="width: 10%; height: 19px; text-align: center;">&nbsp;</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n</div>\r\n</div>	297
59	There are many biological replicates we harvested for each type of sample (code-names):\n\n* IYJ/2\n\n* LHD/5\n\n* LHD/3\n\n* LHD/4	298
60	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	302
61	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	303
62	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	304
63	There are many biological replicates we harvested for each type of sample (code-names):\n\n* DJX/4\n\n* JZF/3\n\n* JZF/4\n\n* DJX/1	319
64	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	323
65	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	324
66	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	325
67	There are many biological replicates we harvested for each type of sample (code-names):\n\n* NED/4\n\n* BSH/2\n\n* NED/1\n\n* NED/5	328
68	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	332
69	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	333
70	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	334
71	There are many biological replicates we harvested for each type of sample (code-names):\n\n* DOL/1\n\n* CUC/2\n\n* DOL/4\n\n* DOL/5	350
72	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	356
73	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	357
74	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	358
75	There are many biological replicates we harvested for each type of sample (code-names):\n\n* UKA/4\n\n* VOF/4\n\n* VOF/5\n\n* UKA/3	382
76	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	386
77	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	387
78	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	388
79	There are many biological replicates we harvested for each type of sample (code-names):\n\n* KOL/1\n\n* KOL/3\n\n* OFI/1\n\n* KOL/5	403
80	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	407
81	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	408
82	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	409
83	There are many biological replicates we harvested for each type of sample (code-names):\n\n* PLK/4\n\n* PLK/1\n\n* PLK/5\n\n* NWX/5	424
84	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	428
85	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	429
86	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	430
87	There are many biological replicates we harvested for each type of sample (code-names):\n\n* JAX/1\n\n* ASE/3\n\n* ASE/4\n\n* JAX/5	445
88	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	449
89	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	450
90	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	451
91	There are many biological replicates we harvested for each type of sample (code-names):\n\n* BGH/3\n\n* PNK/2\n\n* PNK/5\n\n* PNK/4	466
92	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	470
93	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	471
94	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	472
95	<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">\n<html><body><div>\r\n<p>PO4 adsorption test results below:</p>\r\n<p> </p>\r\n<table width="619">\r\n<tbody>\r\n<tr>\r\n<td width="474">\r\n<p><strong>Sample</strong></p>\r\n</td>\r\n<td width="145">\r\n<p>Capacity</p>\r\n<p>mg/g</p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td width="474">\r\n<p>5% Biochar 250 ⁰C</p>\r\n</td>\r\n<td width="145">\r\n<p>104.6</p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td width="474">\r\n<p>10% Biochar 350 ⁰C</p>\r\n</td>\r\n<td width="145">\r\n<p>98.7</p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td width="474">\r\n<p>Brown pellet (Reference)</p>\r\n</td>\r\n<td width="145">\r\n<p>28.7</p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td width="474">\r\n<p>“5% Biochar 250C”  + 30 min additional baking</p>\r\n</td>\r\n<td width="145">\r\n<p>85.1</p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td width="474">\r\n<p>“MgCO3 High surface area” + 30 min additional baking</p>\r\n</td>\r\n<td width="145">\r\n<p>41.8</p>\r\n</td>\r\n</tr>\r\n<tr>\r\n<td width="474">\r\n<p>MgCO3 High Surface Area, no heat treatment</p>\r\n</td>\r\n<td width="145">\r\n<p>39.6</p>\r\n</td>\r\n</tr>\r\n</tbody>\r\n</table>\r\n<div> </div>\r\n<div> </div>\r\n<div> </div>\r\n<div><img src="" alt="description-4" data-mce-token="4" class="img-responsive"></div>\r\n<div> </div>\r\n<div> </div>\r\n<div><img src="" alt="description-5" data-mce-token="5" class="img-responsive"></div>\r\n<div> </div>\r\n</div></body></html>\n	487
96	There are many biological replicates we harvested for each type of sample (code-names):\n\n* SSI/1\n\n* EJU/4\n\n* EJU/3\n\n* SSI/4	488
97	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	492
98	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	493
99	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	494
100	There are many biological replicates we harvested for each type of sample (code-names):\n\n* OBN/2\n\n* ONQ/3\n\n* ONQ/5\n\n* ONQ/1	509
101	<html><body><pre class= 'hljs  language-python '>\n<code># Read PCR data into a pandas DataFrame. You want a data file where each\n# row corresponds to a separate well, with columns for the sample name,\n# target name, and Cq value. NTC wells should have the sample name set to\n# a value like 'NTC '.\n&gt;&gt; df = pd.read_csv( 'my_data.csv ')\n\n# If your Sample, Target, and Cq columns are called other things, they\n# should be renamed to Sample, Target, and Cq.\n&gt;&gt; df = df.rename(columns={ 'Gene': 'Target ', 'Ct': 'Cq '})\n\n# Drop the wells that are too close to the NTC for that target.\n&gt;&gt; censored = eleven.censor_background(df)\n\n# Rank your candidate reference genes.\n&gt;&gt; ranked = eleven.rank_targets(censored, [ 'Gapdh ', 'Rn18s ', 'Hprt ',\n    'Ubc ', 'Actb '], 'Control ')\n\n# Normalize your data by your most stable genes and compute normalization\n# factors (NFs).\n&gt;&gt; nf = eleven.calculate_nf(censored, ranked.ix[ 'Target ', 0:3], 'Control ')\n\n# Now, normalize all of your expression data.\n&gt;&gt; censored[ 'RelExp '] = eleven.expression_nf(censored, nf, 'Control ')</code>\n</pre></body></html>	513
102	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. This technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. The specific region an antibody recognizes on an antigen is called an epitope. There have been.	514
103	Immunofluorescence is a technique used for light microscopy with a fluorescence microscope and is used primarily on microbiological samples. this technique uses the specificity of antibodies to their antigen to target fluorescent dyes to specific biomolecule targets within a cell, and therefore allows visualization of the distribution of the target molecule through the sample. the specific region an antibody recognizes on an antigen is called an epitope. There have been efforts in epitope mapping since many antibodies can bind the same epitope and levels of binding between antibodies that recognize the same epitope can vary. Additionally, the binding of the fluorophore to the antibody itself cannot interfere with the immunological specificity of the antibody or the binding capacity of its antigen. Immunofluorescence is a widely used example of immunostaining (using antibodies to stain proteins) and is a specific example of immunohistochemistry(the use of the antibody-antigen relationship in tissues). this technique primarily makes use of fluorophores to visualise the location of the antibodies.	515
\.


-- --
-- -- Data for Name: schema_migrations; Type: TABLE DATA; Schema: public; Owner: postgres
-- --

-- COPY public.schema_migrations (version) FROM stdin;
-- 20150713060702
-- 20150713061603
-- 20150713063224
-- 20150713070738
-- 20150713071921
-- 20150713072417
-- 20150714125221
-- 20150715122019
-- 20150715124934
-- 20150715131400
-- 20150715132459
-- 20150715132920
-- 20150715133511
-- 20150715133709
-- 20150715134133
-- 20150715135452
-- 20150715141810
-- 20150715142704
-- 20150715142929
-- 20150715143134
-- 20150716060140
-- 20150716061004
-- 20150716061555
-- 20150716061937
-- 20150716062013
-- 20150716062110
-- 20150716062801
-- 20150716064453
-- 20150716120130
-- 20150716120659
-- 20150717084645
-- 20150717085043
-- 20150717085133
-- 20150722095027
-- 20150722112911
-- 20150723134648
-- 20150730090021
-- 20150804055341
-- 20150820120553
-- 20150820123018
-- 20150820124022
-- 20150827130647
-- 20150827130822
-- 20150911125914
-- 20150915074650
-- 20150923065605
-- 20150923110208
-- 20150923154140
-- 20150924115001
-- 20150924181017
-- 20151005122041
-- 20151021082639
-- 20151021085335
-- 20151022123530
-- 20151028091615
-- 20151103155048
-- 20151111135802
-- 20151117083839
-- 20151119141714
-- 20151130160157
-- 20151203100514
-- 20151207151820
-- 20151214110800
-- 20151215103642
-- 20151215134147
-- 20151216095259
-- 20160114155705
-- 20160118114850
-- 20160119101947
-- 20160125200130
-- 20160125205500
-- 20160201085344
-- 20160205192344
-- 20160407063323
-- 20160421100400
-- 20160425133500
-- 20160429082100
-- 20160429102200
-- 20160513143117
-- 20160513165000
-- 20160527152000
-- 20160531094100
-- 20160704110900
-- 20160722082700
-- 20160803082801
-- 20160808083040
-- 20160809074757
-- 20160928114119
-- 20160928114915
-- 20161004074754
-- 20161006065203
-- 20161011074804
-- 20161012112900
-- 20161123161514
-- 20161125153600
-- 20161129111100
-- 20161129171012
-- 20161208145354
-- 20170105162500
-- 20170116143350
-- 20170124135736
-- 20170207100731
-- 20170217141402
-- 20170306121855
-- 20170321131116
-- 20170322095856
-- 20170330092353
-- 20170404150845
-- 20170407082423
-- 20170407104322
-- 20170418153541
-- 20170420075905
-- 20170515073041
-- 20170515141252
-- 20170619125051
-- 20170803153030
-- 20170809131000
-- 20171003082333
-- 20171005135350
-- 20171026090804
-- 20180109102914
-- 20180207095200
-- 20180308094354
-- 20180416114040
-- 20180416171923
-- 20180417062042
-- 20180418123509
-- 20180418123815
-- 20180418124021
-- 20180507160013
-- 20180524091143
-- 20180806115201
-- 20180813120338
-- 20180905142400
-- 20180930205254
-- 20181008130519
-- 20181212162649
-- 20190116101127
-- 20190117155006
-- 20190125122852
-- 20190125123107
-- 20190213064847
-- 20190227110801
-- 20190227125306
-- 20190227125352
-- 20190304153544
-- 20190410110605
-- 20190424113216
-- 20190307102521
-- 20190308092130
-- 20190520135317
-- 20190711125513
-- 20190715150326
-- 20190812065432
-- 20190812072649
-- 20190613094834
-- 20190613134100
-- 20190830141257
-- 20190910125740
-- 20191001133557
-- 20191009146101
-- 20190726102200
-- 20190903145834
-- 20191003091614
-- 20191007144622
-- 20191023162335
-- 20191105143702
-- 20191115143747
-- 20191204112549
-- 20191205133522
-- 20191206105058
-- 20191210103004
-- 20191218072619
-- 20200113143828
-- 20200204100934
-- 20200326114643
-- 20200331183640
-- 20200603125407
-- 20200604210943
-- 20200622140843
-- 20200622155632
-- 20200709142830
-- 20200713142353
-- 20200714082503
-- 20200826143431
-- 20200902093234
-- 20200909121441
-- 20201027133634
-- 20201028103608
-- 20201126203713
-- 20201209165626
-- 20201215161050
-- 20210128105457
-- 20210128105458
-- 20210217114042
-- 20210312185911
-- 20210325152257
-- 20210407143303
-- 20210410100006
-- 20210506125657
-- 20210531114633
-- 20210603152345
-- 20210616071836
-- 20210622101238
-- 20210715125349
-- 20210716124649
-- 20210720112050
-- 20210811103123
-- \.

--
-- Data for Name: step_assets; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.step_assets (id, step_id, asset_id) FROM stdin;
1	2	1
2	17	5
3	31	7
4	32	8
5	32	9
6	33	10
7	33	11
8	34	17
9	38	18
10	39	19
11	40	20
12	40	21
13	42	25
14	57	29
15	71	31
16	72	32
17	72	33
18	73	34
19	73	35
20	74	41
21	78	42
22	79	43
23	80	44
24	80	45
25	82	49
26	97	53
27	111	55
28	112	56
29	112	57
30	113	58
31	113	59
32	114	65
33	118	66
34	119	67
35	120	68
36	120	69
37	122	73
38	137	77
39	151	79
40	152	80
41	152	81
42	153	82
43	153	83
44	154	89
45	158	90
46	159	91
47	160	92
48	160	93
49	162	97
50	177	101
51	191	103
52	192	104
53	192	105
54	193	106
55	193	107
56	194	113
57	198	114
58	199	115
59	200	116
60	200	117
61	202	121
62	217	125
63	231	127
64	232	128
65	232	129
66	233	130
67	233	131
68	234	137
69	238	138
70	239	139
71	240	140
72	240	141
73	242	145
74	257	149
75	271	151
76	272	152
77	272	153
78	273	154
79	273	155
80	274	161
81	278	162
82	279	163
83	280	164
84	280	165
85	282	169
86	297	173
87	311	175
88	312	176
89	312	177
90	313	178
91	313	179
92	314	185
93	318	186
94	319	187
95	320	188
96	320	189
97	322	193
98	337	197
99	351	199
100	352	200
101	352	201
102	353	202
103	353	203
104	354	209
105	358	210
106	359	211
107	360	212
108	360	213
109	362	217
110	377	221
111	391	223
112	392	224
113	392	225
114	393	226
115	393	227
116	394	233
117	398	234
118	399	235
119	400	236
120	400	237
121	402	241
122	417	245
123	431	247
124	432	248
125	432	249
126	433	250
127	433	251
128	434	257
129	438	258
130	439	259
131	440	260
132	440	261
133	444	265
134	459	269
135	473	271
136	474	272
137	474	273
138	475	274
139	475	275
140	476	281
141	480	282
142	481	283
143	482	284
144	482	285
145	484	289
146	499	293
147	513	295
148	514	296
149	514	297
150	515	298
151	515	299
152	516	305
153	520	306
154	521	307
155	522	308
156	522	309
157	524	313
158	539	317
159	553	319
160	554	320
161	554	321
162	555	322
163	555	323
164	556	329
165	560	330
166	561	331
167	562	332
168	562	333
170	566	339
171	581	343
172	595	345
173	596	346
174	596	347
175	597	348
176	597	349
177	598	355
178	602	356
179	603	357
180	604	358
181	604	359
182	606	363
183	621	367
184	635	369
185	636	370
186	636	371
187	637	372
188	637	373
189	638	379
190	642	380
191	643	381
192	644	382
193	644	383
194	646	387
195	661	391
196	675	393
197	676	394
198	676	395
199	677	396
200	677	397
201	678	403
202	682	404
203	683	405
204	684	406
205	684	407
206	686	411
207	701	415
208	715	417
209	716	418
210	716	419
211	717	420
212	717	421
213	718	427
214	722	428
215	723	429
216	724	430
217	724	431
218	726	435
219	741	439
220	755	441
221	756	442
222	756	443
223	757	444
224	757	445
225	758	451
226	762	452
227	763	453
228	764	454
229	764	455
230	766	459
231	781	463
232	795	465
233	796	466
234	796	467
235	797	468
236	797	469
237	798	475
238	802	476
239	803	477
240	804	478
241	804	479
242	805	483
243	807	484
244	822	488
245	836	490
246	837	491
247	837	492
248	838	493
249	838	494
250	839	500
251	843	501
252	844	502
253	845	503
254	845	504
255	852	508
256	867	512
257	881	514
258	882	515
259	882	516
260	883	517
261	883	518
262	884	524
263	888	525
264	889	526
265	890	527
266	890	528
267	896	532
268	911	536
269	925	538
270	926	539
271	926	540
272	927	541
273	927	542
274	928	548
275	932	549
276	933	550
277	934	551
278	934	552
279	936	556
280	951	560
281	965	562
282	966	563
283	966	564
284	967	565
285	967	566
286	968	572
287	972	573
288	973	574
289	974	575
290	974	576
291	976	580
292	991	584
293	1005	586
294	1006	587
295	1006	588
296	1007	589
297	1007	590
298	1008	596
299	1012	597
300	1013	598
301	1014	599
302	1014	600
303	1018	604
304	1018	605
\.


--
-- Data for Name: step_tables; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.step_tables (id, step_id, table_id) FROM stdin;
1	1	1
2	2	2
3	20	4
4	31	6
5	41	10
6	42	11
7	60	13
8	71	15
9	81	19
10	82	20
11	100	22
12	111	24
13	121	28
14	122	29
15	140	31
16	151	33
17	161	37
18	162	38
19	180	40
20	191	42
21	201	46
22	202	47
23	220	49
24	231	51
25	241	55
26	242	56
27	260	58
28	271	60
29	281	64
30	282	65
31	300	67
32	311	69
33	321	73
34	322	74
35	340	76
36	351	78
37	361	82
38	362	83
39	380	85
40	391	87
41	401	91
42	402	92
43	420	94
44	431	96
45	443	100
46	444	101
47	462	103
48	473	105
49	483	109
50	484	110
51	502	112
52	513	114
53	523	118
54	524	119
55	542	121
56	553	123
57	565	127
58	566	128
59	584	130
60	595	132
61	605	136
62	606	137
63	624	139
64	635	141
65	645	145
66	646	146
67	664	148
68	675	150
69	685	154
70	686	155
71	704	157
72	715	159
73	725	163
74	726	164
75	744	166
76	755	168
77	765	172
78	766	173
79	784	175
80	795	177
81	806	181
82	807	182
83	825	184
84	836	186
85	851	190
86	852	191
87	870	193
88	881	195
89	895	199
90	896	200
91	914	202
92	925	204
93	935	208
94	936	209
95	954	211
96	965	213
97	975	217
98	976	218
99	994	220
100	1005	222
\.



--
-- Data for Name: tags; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.tags (id, name, created_at, updated_at, color, project_id, created_by_id, last_modified_by_id) FROM stdin;
1	Dry lab	2019-05-30 12:40:53.652374	2019-05-30 12:40:53.652374	#6C159E	1	1	1
2	Wet lab	2019-05-30 12:40:53.655014	2019-05-30 12:40:53.655014	#6495ED	1	1	1
3	Plant	2019-05-30 12:40:53.657413	2019-05-30 12:40:53.657413	#32CD32	1	1	1
4	Pathogenic virus	2019-05-30 12:40:53.659414	2019-05-30 12:40:53.659414	#DC143C	1	1	1
5	Infectious sample	2019-05-30 12:40:53.661293	2019-05-30 12:40:53.661293	#FF4500	1	1	1
6	Bacteria	2019-05-30 12:40:53.663281	2019-05-30 12:40:53.663281	#FF8C00	1	1	1
7	Results for patent	2019-05-30 12:40:53.665282	2019-05-30 12:40:53.665282	#008B8B	1	1	1
8	Assign unique identifires	2019-05-30 12:40:53.667473	2019-05-30 12:40:53.667473	#C71585	1	1	1
9	Plasmid A	2019-05-30 12:40:53.669302	2019-05-30 12:40:53.669302	#159B5E	1	1	1
10	Dry lab	2019-08-27 15:15:01.892586	2019-08-27 15:15:01.892586	#6C159E	2	4	4
11	Wet lab	2019-08-27 15:15:01.943711	2019-08-27 15:15:01.943711	#6495ED	2	4	4
12	Plant	2019-08-27 15:15:01.961804	2019-08-27 15:15:01.961804	#32CD32	2	4	4
13	Pathogenic virus	2019-08-27 15:15:01.979666	2019-08-27 15:15:01.979666	#DC143C	2	4	4
14	Infectious sample	2019-08-27 15:15:01.998742	2019-08-27 15:15:01.998742	#FF4500	2	4	4
15	Bacteria	2019-08-27 15:15:02.005238	2019-08-27 15:15:02.005238	#FF8C00	2	4	4
16	Results for patent	2019-08-27 15:15:02.010504	2019-08-27 15:15:02.010504	#008B8B	2	4	4
17	Assign unique identifires	2019-08-27 15:15:02.023714	2019-08-27 15:15:02.023714	#C71585	2	4	4
18	Plasmid A	2019-08-27 15:15:02.026116	2019-08-27 15:15:02.026116	#159B5E	2	4	4
19	Dry lab	2019-08-28 07:31:00.228943	2019-08-28 07:31:00.228943	#6C159E	3	1	1
20	Wet lab	2019-08-28 07:31:00.233304	2019-08-28 07:31:00.233304	#6495ED	3	1	1
21	Plant	2019-08-28 07:31:00.236649	2019-08-28 07:31:00.236649	#32CD32	3	1	1
22	Pathogenic virus	2019-08-28 07:31:00.23951	2019-08-28 07:31:00.23951	#DC143C	3	1	1
23	Infectious sample	2019-08-28 07:31:00.243334	2019-08-28 07:31:00.243334	#FF4500	3	1	1
24	Bacteria	2019-08-28 07:31:00.246503	2019-08-28 07:31:00.246503	#FF8C00	3	1	1
25	Results for patent	2019-08-28 07:31:00.249531	2019-08-28 07:31:00.249531	#008B8B	3	1	1
26	Assign unique identifires	2019-08-28 07:31:00.252525	2019-08-28 07:31:00.252525	#C71585	3	1	1
27	Plasmid A	2019-08-28 07:31:00.256425	2019-08-28 07:31:00.256425	#159B5E	3	1	1
28	Dry lab	2019-08-28 08:34:55.989961	2019-08-28 08:34:55.989961	#6C159E	4	6	6
29	Wet lab	2019-08-28 08:34:56.004227	2019-08-28 08:34:56.004227	#6495ED	4	6	6
30	Plant	2019-08-28 08:34:56.006103	2019-08-28 08:34:56.006103	#32CD32	4	6	6
31	Pathogenic virus	2019-08-28 08:34:56.007714	2019-08-28 08:34:56.007714	#DC143C	4	6	6
32	Infectious sample	2019-08-28 08:34:56.01256	2019-08-28 08:34:56.01256	#FF4500	4	6	6
33	Bacteria	2019-08-28 08:34:56.017161	2019-08-28 08:34:56.017161	#FF8C00	4	6	6
34	Results for patent	2019-08-28 08:34:56.037705	2019-08-28 08:34:56.037705	#008B8B	4	6	6
35	Assign unique identifires	2019-08-28 08:34:56.053531	2019-08-28 08:34:56.053531	#C71585	4	6	6
36	Plasmid A	2019-08-28 08:34:56.055409	2019-08-28 08:34:56.055409	#159B5E	4	6	6
37	Dry lab	2019-08-29 12:51:25.729905	2019-08-29 12:51:25.729905	#6C159E	5	7	7
38	Wet lab	2019-08-29 12:51:25.74062	2019-08-29 12:51:25.74062	#6495ED	5	7	7
39	Plant	2019-08-29 12:51:25.742572	2019-08-29 12:51:25.742572	#32CD32	5	7	7
40	Pathogenic virus	2019-08-29 12:51:25.744291	2019-08-29 12:51:25.744291	#DC143C	5	7	7
41	Infectious sample	2019-08-29 12:51:25.749812	2019-08-29 12:51:25.749812	#FF4500	5	7	7
42	Bacteria	2019-08-29 12:51:25.752973	2019-08-29 12:51:25.752973	#FF8C00	5	7	7
43	Results for patent	2019-08-29 12:51:25.75499	2019-08-29 12:51:25.75499	#008B8B	5	7	7
44	Assign unique identifires	2019-08-29 12:51:25.756573	2019-08-29 12:51:25.756573	#C71585	5	7	7
45	Plasmid A	2019-08-29 12:51:25.758104	2019-08-29 12:51:25.758104	#159B5E	5	7	7
46	Dry lab	2019-09-12 18:26:18.865375	2019-09-12 18:26:18.865375	#6C159E	6	8	8
47	Wet lab	2019-09-12 18:26:18.869539	2019-09-12 18:26:18.869539	#6495ED	6	8	8
48	Plant	2019-09-12 18:26:18.871247	2019-09-12 18:26:18.871247	#32CD32	6	8	8
49	Pathogenic virus	2019-09-12 18:26:18.872974	2019-09-12 18:26:18.872974	#DC143C	6	8	8
50	Infectious sample	2019-09-12 18:26:18.874706	2019-09-12 18:26:18.874706	#FF4500	6	8	8
51	Bacteria	2019-09-12 18:26:18.876401	2019-09-12 18:26:18.876401	#FF8C00	6	8	8
52	Results for patent	2019-09-12 18:26:18.878125	2019-09-12 18:26:18.878125	#008B8B	6	8	8
53	Assign unique identifires	2019-09-12 18:26:18.879671	2019-09-12 18:26:18.879671	#C71585	6	8	8
54	Plasmid A	2019-09-12 18:26:18.8812	2019-09-12 18:26:18.8812	#159B5E	6	8	8
55	Dry lab	2019-11-13 13:57:42.323418	2019-11-13 13:57:42.323418	#6C159E	22	9	9
56	Wet lab	2019-11-13 13:57:42.339617	2019-11-13 13:57:42.339617	#6495ED	22	9	9
57	Plant	2019-11-13 13:57:42.342771	2019-11-13 13:57:42.342771	#2CB72C	22	9	9
58	Pathogenic virus	2019-11-13 13:57:42.349271	2019-11-13 13:57:42.349271	#DC143C	22	9	9
59	Infectious sample	2019-11-13 13:57:42.352366	2019-11-13 13:57:42.352366	#FF4500	22	9	9
60	Bacteria	2019-11-13 13:57:42.35548	2019-11-13 13:57:42.35548	#FF8C00	22	9	9
61	Results for patent	2019-11-13 13:57:42.361379	2019-11-13 13:57:42.361379	#008B8B	22	9	9
62	Assign unique identifires	2019-11-13 13:57:42.363726	2019-11-13 13:57:42.363726	#C71585	22	9	9
63	Plasmid A	2019-11-13 13:57:42.366155	2019-11-13 13:57:42.366155	#159B5E	22	9	9
64	Dry lab	2019-11-25 14:09:25.848839	2019-11-25 14:09:25.848839	#6C159E	23	4	4
65	Wet lab	2019-11-25 14:09:25.858294	2019-11-25 14:09:25.858294	#6495ED	23	4	4
66	Plant	2019-11-25 14:09:25.864436	2019-11-25 14:09:25.864436	#2CB72C	23	4	4
67	Pathogenic virus	2019-11-25 14:09:25.871476	2019-11-25 14:09:25.871476	#DC143C	23	4	4
68	Infectious sample	2019-11-25 14:09:25.878127	2019-11-25 14:09:25.878127	#FF4500	23	4	4
69	Bacteria	2019-11-25 14:09:25.890316	2019-11-25 14:09:25.890316	#FF8C00	23	4	4
70	Results for patent	2019-11-25 14:09:25.938138	2019-11-25 14:09:25.938138	#008B8B	23	4	4
71	Assign unique identifires	2019-11-25 14:09:25.96513	2019-11-25 14:09:25.96513	#C71585	23	4	4
72	Plasmid A	2019-11-25 14:09:25.992041	2019-11-25 14:09:25.992041	#159B5E	23	4	4
74	New tag	2019-11-25 17:03:21.279895	2019-11-25 17:03:21.279895	#6C159E	25	4	4
73	COMPLETED - Drain and Flush Prep procedure (prior to use)	2019-11-25 17:02:26.931862	2019-11-25 17:04:02.031172	#6C159E	25	4	4
75	Dry lab	2019-11-25 18:00:52.990299	2019-11-25 18:00:52.990299	#6C159E	26	10	10
76	Wet lab	2019-11-25 18:00:52.99312	2019-11-25 18:00:52.99312	#6495ED	26	10	10
77	Plant	2019-11-25 18:00:52.997564	2019-11-25 18:00:52.997564	#2CB72C	26	10	10
78	Pathogenic virus	2019-11-25 18:00:53.001543	2019-11-25 18:00:53.001543	#DC143C	26	10	10
79	Infectious sample	2019-11-25 18:00:53.00346	2019-11-25 18:00:53.00346	#FF4500	26	10	10
80	Bacteria	2019-11-25 18:00:53.005638	2019-11-25 18:00:53.005638	#FF8C00	26	10	10
81	Results for patent	2019-11-25 18:00:53.007584	2019-11-25 18:00:53.007584	#008B8B	26	10	10
82	Assign unique identifires	2019-11-25 18:00:53.009716	2019-11-25 18:00:53.009716	#C71585	26	10	10
83	Plasmid A	2019-11-25 18:00:53.011684	2019-11-25 18:00:53.011684	#159B5E	26	10	10
84	Dry lab	2020-05-12 16:13:02.95963	2020-05-12 16:13:02.95963	#6C159E	27	11	11
85	Wet lab	2020-05-12 16:13:02.972171	2020-05-12 16:13:02.972171	#6495ED	27	11	11
86	Plant	2020-05-12 16:13:02.974629	2020-05-12 16:13:02.974629	#2CB72C	27	11	11
87	Pathogenic virus	2020-05-12 16:13:02.977087	2020-05-12 16:13:02.977087	#DC143C	27	11	11
88	Infectious sample	2020-05-12 16:13:02.979445	2020-05-12 16:13:02.979445	#FF4500	27	11	11
89	Bacteria	2020-05-12 16:13:02.98205	2020-05-12 16:13:02.98205	#FF8C00	27	11	11
90	Results for patent	2020-05-12 16:13:03.039181	2020-05-12 16:13:03.039181	#008B8B	27	11	11
91	Assign unique identifires	2020-05-12 16:13:03.059941	2020-05-12 16:13:03.059941	#C71585	27	11	11
92	Plasmid A	2020-05-12 16:13:03.185178	2020-05-12 16:13:03.185178	#159B5E	27	11	11
93	Dry lab	2020-06-10 14:43:17.753164	2020-06-10 14:43:17.753164	#6C159E	28	12	12
94	Wet lab	2020-06-10 14:43:17.759736	2020-06-10 14:43:17.759736	#6495ED	28	12	12
95	Plant	2020-06-10 14:43:17.762219	2020-06-10 14:43:17.762219	#2CB72C	28	12	12
96	Pathogenic virus	2020-06-10 14:43:17.764199	2020-06-10 14:43:17.764199	#DC143C	28	12	12
97	Infectious sample	2020-06-10 14:43:17.766	2020-06-10 14:43:17.766	#FF4500	28	12	12
98	Bacteria	2020-06-10 14:43:17.768062	2020-06-10 14:43:17.768062	#FF8C00	28	12	12
99	Results for patent	2020-06-10 14:43:17.769933	2020-06-10 14:43:17.769933	#008B8B	28	12	12
100	Assign unique identifires	2020-06-10 14:43:17.771672	2020-06-10 14:43:17.771672	#C71585	28	12	12
101	Plasmid A	2020-06-10 14:43:17.773304	2020-06-10 14:43:17.773304	#159B5E	28	12	12
102	Dry lab	2020-10-07 19:20:25.916119	2020-10-07 19:20:25.916119	#6C159E	33	16	16
103	Wet lab	2020-10-07 19:20:25.925396	2020-10-07 19:20:25.925396	#6495ED	33	16	16
104	Plant	2020-10-07 19:20:25.927713	2020-10-07 19:20:25.927713	#2CB72C	33	16	16
105	Pathogenic virus	2020-10-07 19:20:25.929672	2020-10-07 19:20:25.929672	#DC143C	33	16	16
106	Infectious sample	2020-10-07 19:20:25.932171	2020-10-07 19:20:25.932171	#FF4500	33	16	16
107	Bacteria	2020-10-07 19:20:25.934495	2020-10-07 19:20:25.934495	#FF8C00	33	16	16
108	Results for patent	2020-10-07 19:20:25.937862	2020-10-07 19:20:25.937862	#008B8B	33	16	16
109	Assign unique identifires	2020-10-07 19:20:25.940227	2020-10-07 19:20:25.940227	#C71585	33	16	16
110	Plasmid A	2020-10-07 19:20:25.942066	2020-10-07 19:20:25.942066	#159B5E	33	16	16
111	Dry lab	2020-11-18 14:50:53.476316	2020-11-18 14:50:53.476316	#6C159E	34	1	1
112	Wet lab	2020-11-18 14:50:53.489238	2020-11-18 14:50:53.489238	#6495ED	34	1	1
113	Plant	2020-11-18 14:50:53.495023	2020-11-18 14:50:53.495023	#2CB72C	34	1	1
114	Pathogenic virus	2020-11-18 14:50:53.498766	2020-11-18 14:50:53.498766	#DC143C	34	1	1
115	Infectious sample	2020-11-18 14:50:53.50334	2020-11-18 14:50:53.50334	#FF4500	34	1	1
116	Bacteria	2020-11-18 14:50:53.508214	2020-11-18 14:50:53.508214	#FF8C00	34	1	1
117	Results for patent	2020-11-18 14:50:53.512277	2020-11-18 14:50:53.512277	#008B8B	34	1	1
118	Assign unique identifires	2020-11-18 14:50:53.516701	2020-11-18 14:50:53.516701	#C71585	34	1	1
119	Plasmid A	2020-11-18 14:50:53.520847	2020-11-18 14:50:53.520847	#159B5E	34	1	1
120	Dry lab	2020-11-18 14:53:25.307845	2020-11-18 14:53:25.307845	#6C159E	35	17	17
121	Wet lab	2020-11-18 14:53:25.310622	2020-11-18 14:53:25.310622	#6495ED	35	17	17
122	Plant	2020-11-18 14:53:25.312816	2020-11-18 14:53:25.312816	#2CB72C	35	17	17
123	Pathogenic virus	2020-11-18 14:53:25.314824	2020-11-18 14:53:25.314824	#DC143C	35	17	17
124	Infectious sample	2020-11-18 14:53:25.317086	2020-11-18 14:53:25.317086	#FF4500	35	17	17
125	Bacteria	2020-11-18 14:53:25.319119	2020-11-18 14:53:25.319119	#FF8C00	35	17	17
126	Results for patent	2020-11-18 14:53:25.321065	2020-11-18 14:53:25.321065	#008B8B	35	17	17
127	Assign unique identifires	2020-11-18 14:53:25.322898	2020-11-18 14:53:25.322898	#C71585	35	17	17
128	Plasmid A	2020-11-18 14:53:25.324816	2020-11-18 14:53:25.324816	#159B5E	35	17	17
129	Dry lab	2021-05-19 20:01:52.214437	2021-05-19 20:01:52.214437	#6C159E	37	20	20
130	Wet lab	2021-05-19 20:01:52.220322	2021-05-19 20:01:52.220322	#6495ED	37	20	20
131	Plant	2021-05-19 20:01:52.22227	2021-05-19 20:01:52.22227	#2CB72C	37	20	20
132	Pathogenic virus	2021-05-19 20:01:52.223656	2021-05-19 20:01:52.223656	#DC143C	37	20	20
133	Infectious sample	2021-05-19 20:01:52.224919	2021-05-19 20:01:52.224919	#FF4500	37	20	20
134	Bacteria	2021-05-19 20:01:52.226184	2021-05-19 20:01:52.226184	#FF8C00	37	20	20
135	Results for patent	2021-05-19 20:01:52.227419	2021-05-19 20:01:52.227419	#008B8B	37	20	20
136	Assign unique identifires	2021-05-19 20:01:52.228533	2021-05-19 20:01:52.228533	#C71585	37	20	20
137	Plasmid A	2021-05-19 20:01:52.230133	2021-05-19 20:01:52.230133	#159B5E	37	20	20
138	Dry lab	2021-05-20 14:12:30.829082	2021-05-20 14:12:30.829082	#6C159E	38	21	21
139	Wet lab	2021-05-20 14:12:30.830737	2021-05-20 14:12:30.830737	#6495ED	38	21	21
140	Plant	2021-05-20 14:12:30.832082	2021-05-20 14:12:30.832082	#2CB72C	38	21	21
141	Pathogenic virus	2021-05-20 14:12:30.833266	2021-05-20 14:12:30.833266	#DC143C	38	21	21
142	Infectious sample	2021-05-20 14:12:30.834456	2021-05-20 14:12:30.834456	#FF4500	38	21	21
143	Bacteria	2021-05-20 14:12:30.835595	2021-05-20 14:12:30.835595	#FF8C00	38	21	21
144	Results for patent	2021-05-20 14:12:30.836748	2021-05-20 14:12:30.836748	#008B8B	38	21	21
145	Assign unique identifires	2021-05-20 14:12:30.837834	2021-05-20 14:12:30.837834	#C71585	38	21	21
146	Plasmid A	2021-05-20 14:12:30.838881	2021-05-20 14:12:30.838881	#159B5E	38	21	21
147	Dry lab	2021-05-20 14:12:57.4831	2021-05-20 14:12:57.4831	#6C159E	39	22	22
148	Wet lab	2021-05-20 14:12:57.484667	2021-05-20 14:12:57.484667	#6495ED	39	22	22
149	Plant	2021-05-20 14:12:57.485869	2021-05-20 14:12:57.485869	#2CB72C	39	22	22
150	Pathogenic virus	2021-05-20 14:12:57.487274	2021-05-20 14:12:57.487274	#DC143C	39	22	22
151	Infectious sample	2021-05-20 14:12:57.488875	2021-05-20 14:12:57.488875	#FF4500	39	22	22
152	Bacteria	2021-05-20 14:12:57.490067	2021-05-20 14:12:57.490067	#FF8C00	39	22	22
153	Results for patent	2021-05-20 14:12:57.491304	2021-05-20 14:12:57.491304	#008B8B	39	22	22
154	Assign unique identifires	2021-05-20 14:12:57.492462	2021-05-20 14:12:57.492462	#C71585	39	22	22
155	Plasmid A	2021-05-20 14:12:57.4936	2021-05-20 14:12:57.4936	#159B5E	39	22	22
156	Dry lab	2021-05-20 14:13:34.283674	2021-05-20 14:13:34.283674	#6C159E	40	23	23
157	Wet lab	2021-05-20 14:13:34.285134	2021-05-20 14:13:34.285134	#6495ED	40	23	23
158	Plant	2021-05-20 14:13:34.28619	2021-05-20 14:13:34.28619	#2CB72C	40	23	23
159	Pathogenic virus	2021-05-20 14:13:34.287256	2021-05-20 14:13:34.287256	#DC143C	40	23	23
160	Infectious sample	2021-05-20 14:13:34.288314	2021-05-20 14:13:34.288314	#FF4500	40	23	23
161	Bacteria	2021-05-20 14:13:34.289777	2021-05-20 14:13:34.289777	#FF8C00	40	23	23
162	Results for patent	2021-05-20 14:13:34.29087	2021-05-20 14:13:34.29087	#008B8B	40	23	23
163	Assign unique identifires	2021-05-20 14:13:34.291943	2021-05-20 14:13:34.291943	#C71585	40	23	23
164	Plasmid A	2021-05-20 14:13:34.292998	2021-05-20 14:13:34.292998	#159B5E	40	23	23
165	Dry lab	2021-05-20 14:21:20.847778	2021-05-20 14:21:20.847778	#6C159E	41	24	24
166	Wet lab	2021-05-20 14:21:20.849513	2021-05-20 14:21:20.849513	#6495ED	41	24	24
167	Plant	2021-05-20 14:21:20.850706	2021-05-20 14:21:20.850706	#2CB72C	41	24	24
168	Pathogenic virus	2021-05-20 14:21:20.851923	2021-05-20 14:21:20.851923	#DC143C	41	24	24
169	Infectious sample	2021-05-20 14:21:20.853381	2021-05-20 14:21:20.853381	#FF4500	41	24	24
170	Bacteria	2021-05-20 14:21:20.854727	2021-05-20 14:21:20.854727	#FF8C00	41	24	24
171	Results for patent	2021-05-20 14:21:20.85604	2021-05-20 14:21:20.85604	#008B8B	41	24	24
172	Assign unique identifires	2021-05-20 14:21:20.857109	2021-05-20 14:21:20.857109	#C71585	41	24	24
173	Plasmid A	2021-05-20 14:21:20.858272	2021-05-20 14:21:20.858272	#159B5E	41	24	24
174	Dry lab	2021-05-20 14:30:30.58949	2021-05-20 14:30:30.58949	#6C159E	43	25	25
175	Wet lab	2021-05-20 14:30:30.591145	2021-05-20 14:30:30.591145	#6495ED	43	25	25
176	Plant	2021-05-20 14:30:30.592324	2021-05-20 14:30:30.592324	#2CB72C	43	25	25
177	Pathogenic virus	2021-05-20 14:30:30.593514	2021-05-20 14:30:30.593514	#DC143C	43	25	25
178	Infectious sample	2021-05-20 14:30:30.594686	2021-05-20 14:30:30.594686	#FF4500	43	25	25
179	Bacteria	2021-05-20 14:30:30.595858	2021-05-20 14:30:30.595858	#FF8C00	43	25	25
180	Results for patent	2021-05-20 14:30:30.597048	2021-05-20 14:30:30.597048	#008B8B	43	25	25
181	Assign unique identifires	2021-05-20 14:30:30.598133	2021-05-20 14:30:30.598133	#C71585	43	25	25
182	Plasmid A	2021-05-20 14:30:30.599318	2021-05-20 14:30:30.599318	#159B5E	43	25	25
183	Dry lab	2021-05-20 15:17:12.010806	2021-05-20 15:17:12.010806	#6C159E	44	26	26
184	Wet lab	2021-05-20 15:17:12.012778	2021-05-20 15:17:12.012778	#6495ED	44	26	26
185	Plant	2021-05-20 15:17:12.014028	2021-05-20 15:17:12.014028	#2CB72C	44	26	26
186	Pathogenic virus	2021-05-20 15:17:12.01545	2021-05-20 15:17:12.01545	#DC143C	44	26	26
187	Infectious sample	2021-05-20 15:17:12.016812	2021-05-20 15:17:12.016812	#FF4500	44	26	26
188	Bacteria	2021-05-20 15:17:12.018132	2021-05-20 15:17:12.018132	#FF8C00	44	26	26
189	Results for patent	2021-05-20 15:17:12.01939	2021-05-20 15:17:12.01939	#008B8B	44	26	26
190	Assign unique identifires	2021-05-20 15:17:12.020608	2021-05-20 15:17:12.020608	#C71585	44	26	26
191	Plasmid A	2021-05-20 15:17:12.021697	2021-05-20 15:17:12.021697	#159B5E	44	26	26
192	Dry lab	2021-05-28 15:07:52.803621	2021-05-28 15:07:52.803621	#6C159E	46	27	27
193	Wet lab	2021-05-28 15:07:52.814173	2021-05-28 15:07:52.814173	#6495ED	46	27	27
194	Plant	2021-05-28 15:07:52.816554	2021-05-28 15:07:52.816554	#2CB72C	46	27	27
195	Pathogenic virus	2021-05-28 15:07:52.819072	2021-05-28 15:07:52.819072	#DC143C	46	27	27
196	Infectious sample	2021-05-28 15:07:52.82062	2021-05-28 15:07:52.82062	#FF4500	46	27	27
197	Bacteria	2021-05-28 15:07:52.823474	2021-05-28 15:07:52.823474	#FF8C00	46	27	27
198	Results for patent	2021-05-28 15:07:52.826399	2021-05-28 15:07:52.826399	#008B8B	46	27	27
199	Assign unique identifires	2021-05-28 15:07:52.828622	2021-05-28 15:07:52.828622	#C71585	46	27	27
200	Plasmid A	2021-05-28 15:07:52.831128	2021-05-28 15:07:52.831128	#159B5E	46	27	27
201	Dry lab	2021-06-02 16:10:03.478578	2021-06-02 16:10:03.478578	#6C159E	54	28	28
202	Wet lab	2021-06-02 16:10:03.509456	2021-06-02 16:10:03.509456	#6495ED	54	28	28
203	Plant	2021-06-02 16:10:03.515272	2021-06-02 16:10:03.515272	#2CB72C	54	28	28
204	Pathogenic virus	2021-06-02 16:10:03.521328	2021-06-02 16:10:03.521328	#DC143C	54	28	28
205	Infectious sample	2021-06-02 16:10:03.52543	2021-06-02 16:10:03.52543	#FF4500	54	28	28
206	Bacteria	2021-06-02 16:10:03.629437	2021-06-02 16:10:03.629437	#FF8C00	54	28	28
207	Results for patent	2021-06-02 16:10:03.634808	2021-06-02 16:10:03.634808	#008B8B	54	28	28
208	Assign unique identifires	2021-06-02 16:10:03.640242	2021-06-02 16:10:03.640242	#C71585	54	28	28
209	Plasmid A	2021-06-02 16:10:03.646281	2021-06-02 16:10:03.646281	#159B5E	54	28	28
210	Dry lab	2021-08-23 14:58:28.201415	2021-08-23 14:58:28.201415	#6C159E	55	30	30
211	Wet lab	2021-08-23 14:58:28.20797	2021-08-23 14:58:28.20797	#6495ED	55	30	30
212	Plant	2021-08-23 14:58:28.209547	2021-08-23 14:58:28.209547	#2CB72C	55	30	30
213	Pathogenic virus	2021-08-23 14:58:28.21089	2021-08-23 14:58:28.21089	#DC143C	55	30	30
214	Infectious sample	2021-08-23 14:58:28.21244	2021-08-23 14:58:28.21244	#FF4500	55	30	30
215	Bacteria	2021-08-23 14:58:28.214323	2021-08-23 14:58:28.214323	#FF8C00	55	30	30
216	Results for patent	2021-08-23 14:58:28.215642	2021-08-23 14:58:28.215642	#008B8B	55	30	30
217	Assign unique identifires	2021-08-23 14:58:28.216842	2021-08-23 14:58:28.216842	#C71585	55	30	30
218	Plasmid A	2021-08-23 14:58:28.217956	2021-08-23 14:58:28.217956	#159B5E	55	30	30
219	Dry lab	2021-08-23 17:19:39.512248	2021-08-23 17:19:39.512248	#6C159E	56	31	31
220	Wet lab	2021-08-23 17:19:39.513971	2021-08-23 17:19:39.513971	#6495ED	56	31	31
221	Plant	2021-08-23 17:19:39.515919	2021-08-23 17:19:39.515919	#2CB72C	56	31	31
222	Pathogenic virus	2021-08-23 17:19:39.517852	2021-08-23 17:19:39.517852	#DC143C	56	31	31
223	Infectious sample	2021-08-23 17:19:39.519259	2021-08-23 17:19:39.519259	#FF4500	56	31	31
224	Bacteria	2021-08-23 17:19:39.520652	2021-08-23 17:19:39.520652	#FF8C00	56	31	31
225	Results for patent	2021-08-23 17:19:39.522434	2021-08-23 17:19:39.522434	#008B8B	56	31	31
226	Assign unique identifires	2021-08-23 17:19:39.523807	2021-08-23 17:19:39.523807	#C71585	56	31	31
227	Plasmid A	2021-08-23 17:19:39.525173	2021-08-23 17:19:39.525173	#159B5E	56	31	31
\.


--
-- Data for Name: tiny_mce_assets; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.tiny_mce_assets (id, estimated_size, team_id, created_at, updated_at, saved, object_type, object_id) FROM stdin;
4	4373184	22	2021-06-23 15:37:30.293734	2021-06-23 15:38:02.311871	t	ResultText	95
5	131521	22	2021-06-23 15:37:58.848605	2021-06-23 15:38:02.317302	t	ResultText	95
\.

--
-- Data for Name: user_my_modules; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.user_my_modules (id, user_id, my_module_id, created_at, updated_at, assigned_by_id) FROM stdin;
1	1	1	2019-05-26 15:46:54.521081	2019-05-30 12:40:53.280795	1
2	1	2	2019-05-29 23:30:54.153678	2019-05-30 12:40:53.297265	1
3	1	3	2019-05-26 06:56:25.832038	2019-05-30 12:40:53.306666	1
4	1	4	2019-05-29 20:30:56.877952	2019-05-30 12:40:53.316166	1
5	1	5	2019-05-25 03:15:05.315079	2019-05-30 12:40:53.325439	1
6	1	6	2019-05-28 08:42:06.518459	2019-05-30 12:40:53.334645	1
7	1	7	2019-05-27 17:53:41.289764	2019-05-30 12:40:53.343622	1
8	1	8	2019-05-27 15:15:37.849757	2019-05-30 12:40:53.352061	1
9	1	9	2019-05-27 12:03:08.196623	2019-05-30 12:40:53.360405	1
10	4	10	2019-08-27 06:09:31.119233	2019-08-27 15:15:00.628575	4
11	4	11	2019-08-26 03:04:48.030353	2019-08-27 15:15:00.72581	4
12	4	12	2019-08-25 12:29:30.167494	2019-08-27 15:15:00.752075	4
13	4	13	2019-08-25 19:57:30.529952	2019-08-27 15:15:00.796622	4
14	4	14	2019-08-21 16:41:49.010936	2019-08-27 15:15:00.815386	4
15	4	15	2019-08-22 10:04:43.182075	2019-08-27 15:15:00.840715	4
16	4	16	2019-08-21 23:47:11.346128	2019-08-27 15:15:00.865427	4
17	4	17	2019-08-23 01:21:04.881718	2019-08-27 15:15:00.89003	4
18	4	18	2019-08-24 03:29:49.673679	2019-08-27 15:15:00.970644	4
19	1	19	2019-08-25 02:31:03.788952	2019-08-28 07:30:58.684862	1
20	1	20	2019-08-25 04:51:19.841458	2019-08-28 07:30:58.718084	1
21	1	21	2019-08-24 04:23:05.39954	2019-08-28 07:30:58.745594	1
22	1	22	2019-08-22 13:42:13.041819	2019-08-28 07:30:58.774871	1
23	1	23	2019-08-24 04:06:27.366804	2019-08-28 07:30:58.826322	1
24	1	24	2019-08-23 01:31:01.577014	2019-08-28 07:30:58.865912	1
25	1	25	2019-08-26 09:20:11.95345	2019-08-28 07:30:58.903374	1
26	1	26	2019-08-23 12:44:21.46798	2019-08-28 07:30:58.95376	1
27	1	27	2019-08-26 16:52:47.283523	2019-08-28 07:30:59.028176	1
28	6	28	2019-08-23 00:07:33.487219	2019-08-28 08:34:53.651606	6
29	6	29	2019-08-24 22:56:59.855466	2019-08-28 08:34:53.68691	6
30	6	30	2019-08-25 11:53:52.946837	2019-08-28 08:34:53.708353	6
31	6	31	2019-08-27 05:51:22.067601	2019-08-28 08:34:53.765682	6
32	6	32	2019-08-25 14:40:14.337003	2019-08-28 08:34:53.824173	6
33	6	33	2019-08-23 16:22:41.197284	2019-08-28 08:34:54.145318	6
34	6	34	2019-08-26 04:36:39.47893	2019-08-28 08:34:54.256611	6
35	6	35	2019-08-22 22:38:32.137213	2019-08-28 08:34:54.336129	6
36	6	36	2019-08-26 01:27:15.045909	2019-08-28 08:34:54.405397	6
37	7	37	2019-08-26 16:52:06.316651	2019-08-29 12:51:23.445588	7
38	7	38	2019-08-27 02:19:31.891539	2019-08-29 12:51:23.548355	7
39	7	39	2019-08-28 21:53:49.880272	2019-08-29 12:51:23.575126	7
40	7	40	2019-08-23 23:12:37.032084	2019-08-29 12:51:23.692805	7
41	7	41	2019-08-26 12:00:12.611138	2019-08-29 12:51:23.727997	7
42	7	42	2019-08-24 04:27:07.877391	2019-08-29 12:51:23.785881	7
43	7	43	2019-08-25 14:43:01.922671	2019-08-29 12:51:23.818167	7
44	7	44	2019-08-28 08:44:02.43594	2019-08-29 12:51:23.851196	7
45	7	45	2019-08-27 08:40:38.069748	2019-08-29 12:51:24.163708	7
46	8	46	2019-09-11 07:10:47.867869	2019-09-12 18:26:18.448169	8
47	8	47	2019-09-10 19:52:01.091879	2019-09-12 18:26:18.46512	8
48	8	48	2019-09-07 10:21:45.739039	2019-09-12 18:26:18.477325	8
49	8	49	2019-09-10 23:36:19.372442	2019-09-12 18:26:18.489099	8
50	8	50	2019-09-11 06:52:58.664474	2019-09-12 18:26:18.500302	8
51	8	51	2019-09-06 21:58:51.358719	2019-09-12 18:26:18.511765	8
52	8	52	2019-09-12 10:51:36.578116	2019-09-12 18:26:18.523831	8
53	8	53	2019-09-07 20:20:48.815476	2019-09-12 18:26:18.534763	8
54	8	54	2019-09-11 10:00:29.101735	2019-09-12 18:26:18.546519	8
55	9	55	2019-11-13 04:13:50.762415	2019-11-13 13:57:41.651167	9
56	9	56	2019-11-08 08:40:14.749678	2019-11-13 13:57:41.687253	9
57	9	57	2019-11-10 00:09:55.703336	2019-11-13 13:57:41.707043	9
58	9	58	2019-11-10 06:06:26.73847	2019-11-13 13:57:41.725615	9
59	9	59	2019-11-11 14:58:56.60084	2019-11-13 13:57:41.745941	9
60	9	60	2019-11-12 09:22:09.856796	2019-11-13 13:57:41.776874	9
61	9	61	2019-11-12 06:43:07.510138	2019-11-13 13:57:41.794104	9
62	9	62	2019-11-13 09:51:04.379519	2019-11-13 13:57:41.821633	9
63	9	63	2019-11-09 10:12:57.534552	2019-11-13 13:57:41.839032	9
64	4	64	2019-11-22 16:17:16.311174	2019-11-25 14:09:22.730491	4
65	4	65	2019-11-23 00:32:58.828912	2019-11-25 14:09:22.793554	4
66	4	66	2019-11-19 19:05:35.700069	2019-11-25 14:09:22.840843	4
67	4	67	2019-11-24 03:57:56.24218	2019-11-25 14:09:22.878681	4
68	4	68	2019-11-21 05:21:31.257469	2019-11-25 14:09:22.913078	4
69	4	69	2019-11-24 06:14:14.83221	2019-11-25 14:09:23.07933	4
70	4	70	2019-11-25 05:41:54.457257	2019-11-25 14:09:23.122167	4
71	4	71	2019-11-21 10:42:49.88468	2019-11-25 14:09:23.162404	4
72	4	72	2019-11-24 12:29:18.727723	2019-11-25 14:09:23.207921	4
73	10	78	2019-11-22 05:03:18.404442	2019-11-25 18:00:52.481819	10
74	10	79	2019-11-20 06:12:54.786391	2019-11-25 18:00:52.501692	10
75	10	80	2019-11-24 11:51:41.276538	2019-11-25 18:00:52.519961	10
76	10	81	2019-11-24 13:07:52.652651	2019-11-25 18:00:52.536992	10
77	10	82	2019-11-21 10:57:20.988967	2019-11-25 18:00:52.555697	10
78	10	83	2019-11-20 03:18:23.607519	2019-11-25 18:00:52.573202	10
79	10	84	2019-11-24 06:45:30.548289	2019-11-25 18:00:52.589661	10
80	10	85	2019-11-19 21:15:52.628557	2019-11-25 18:00:52.606609	10
81	10	86	2019-11-20 10:23:11.090582	2019-11-25 18:00:52.621594	10
82	4	75	2019-11-25 18:04:51.588928	2019-11-25 18:04:51.588928	4
83	4	76	2019-11-25 18:05:09.387591	2019-11-25 18:05:09.387591	4
84	11	87	2020-05-08 04:53:09.493221	2020-05-12 16:13:02.04905	11
85	11	88	2020-05-06 16:26:33.017701	2020-05-12 16:13:02.108476	11
86	11	89	2020-05-09 01:50:52.486096	2020-05-12 16:13:02.126291	11
87	11	90	2020-05-09 23:54:14.177015	2020-05-12 16:13:02.149515	11
88	11	91	2020-05-06 21:43:31.962521	2020-05-12 16:13:02.178769	11
89	11	92	2020-05-11 19:43:13.069009	2020-05-12 16:13:02.239585	11
90	11	93	2020-05-10 00:39:21.691041	2020-05-12 16:13:02.254459	11
91	11	94	2020-05-12 04:28:27.116598	2020-05-12 16:13:02.268829	11
92	11	95	2020-05-07 18:11:36.652283	2020-05-12 16:13:02.28991	11
93	12	96	2020-06-09 13:52:41.199665	2020-06-10 14:43:17.098954	12
94	12	97	2020-06-05 09:26:12.411929	2020-06-10 14:43:17.13857	12
95	12	98	2020-06-08 10:17:02.307281	2020-06-10 14:43:17.15505	12
96	12	99	2020-06-06 13:04:22.9785	2020-06-10 14:43:17.170036	12
97	12	100	2020-06-05 21:10:38.775386	2020-06-10 14:43:17.185918	12
98	12	101	2020-06-08 09:35:21.388292	2020-06-10 14:43:17.231878	12
99	12	102	2020-06-05 15:42:02.545396	2020-06-10 14:43:17.246102	12
100	12	103	2020-06-04 18:44:09.237865	2020-06-10 14:43:17.260301	12
101	12	104	2020-06-09 22:52:52.693793	2020-06-10 14:43:17.273332	12
102	16	106	2020-10-07 09:01:30.069224	2020-10-07 19:20:25.175749	16
103	16	107	2020-10-06 10:26:31.211417	2020-10-07 19:20:25.21584	16
104	16	108	2020-10-02 08:22:32.454127	2020-10-07 19:20:25.235926	16
105	16	109	2020-10-06 06:51:32.062549	2020-10-07 19:20:25.252094	16
106	16	110	2020-10-07 11:01:16.573343	2020-10-07 19:20:25.269557	16
107	16	111	2020-10-05 19:50:53.619894	2020-10-07 19:20:25.351368	16
108	16	112	2020-10-06 06:49:53.512856	2020-10-07 19:20:25.367739	16
109	16	113	2020-10-02 01:44:15.454365	2020-10-07 19:20:25.385431	16
110	16	114	2020-10-03 20:00:34.920619	2020-10-07 19:20:25.402858	16
111	1	115	2020-11-15 15:40:32.746524	2020-11-18 14:50:52.294347	1
112	1	116	2020-11-17 09:44:33.213729	2020-11-18 14:50:52.377971	1
113	1	117	2020-11-15 23:57:49.401267	2020-11-18 14:50:52.416304	1
114	1	118	2020-11-13 22:27:06.742639	2020-11-18 14:50:52.454428	1
115	1	119	2020-11-12 15:31:28.21654	2020-11-18 14:50:52.491566	1
116	1	120	2020-11-13 20:28:51.760936	2020-11-18 14:50:52.530058	1
117	1	121	2020-11-14 00:12:32.413844	2020-11-18 14:50:52.562141	1
118	1	122	2020-11-14 14:41:28.048021	2020-11-18 14:50:52.59878	1
119	1	123	2020-11-17 06:21:34.907601	2020-11-18 14:50:52.628691	1
120	17	124	2020-11-13 08:54:40.38794	2020-11-18 14:53:24.789049	17
121	17	125	2020-11-18 02:10:50.470017	2020-11-18 14:53:24.814443	17
122	17	126	2020-11-16 18:37:49.927248	2020-11-18 14:53:24.834279	17
123	17	127	2020-11-15 20:49:43.914172	2020-11-18 14:53:24.852402	17
124	17	128	2020-11-15 19:26:36.99136	2020-11-18 14:53:24.868498	17
125	17	129	2020-11-17 10:39:50.752557	2020-11-18 14:53:24.88916	17
126	17	130	2020-11-14 15:39:33.819476	2020-11-18 14:53:24.90418	17
127	17	131	2020-11-13 00:51:05.281777	2020-11-18 14:53:24.919917	17
128	17	132	2020-11-15 17:18:32.703134	2020-11-18 14:53:24.932066	17
129	4	133	2021-04-08 13:24:40.497005	2021-04-08 13:24:40.497005	4
130	10	133	2021-04-08 13:24:44.293693	2021-04-08 13:24:44.293693	4
131	4	135	2021-04-08 13:24:59.300125	2021-04-08 13:24:59.300125	4
132	10	135	2021-04-08 13:25:02.453417	2021-04-08 13:25:02.453417	4
133	4	134	2021-04-08 13:25:13.747011	2021-04-08 13:25:13.747011	4
134	10	134	2021-04-08 13:25:16.456362	2021-04-08 13:25:16.456362	4
135	4	136	2021-04-08 13:27:29.261682	2021-04-08 13:27:29.261682	4
136	10	136	2021-04-08 13:27:31.991564	2021-04-08 13:27:31.991564	4
137	20	137	2021-05-17 01:18:54.317131	2021-05-19 20:01:51.748849	20
138	20	138	2021-05-16 01:42:37.920682	2021-05-19 20:01:51.772781	20
139	20	139	2021-05-19 06:16:39.089365	2021-05-19 20:01:51.787087	20
140	20	140	2021-05-19 14:49:32.591132	2021-05-19 20:01:51.804218	20
141	20	141	2021-05-16 18:58:12.626965	2021-05-19 20:01:51.818836	20
142	20	142	2021-05-16 08:30:39.948035	2021-05-19 20:01:51.844581	20
143	20	143	2021-05-17 07:01:55.398782	2021-05-19 20:01:51.86247	20
144	20	144	2021-05-19 04:35:29.08274	2021-05-19 20:01:51.877338	20
145	20	145	2021-05-18 08:02:28.415329	2021-05-19 20:01:51.891353	20
146	21	155	2021-05-20 00:52:51.853141	2021-05-20 14:12:30.460728	21
147	21	156	2021-05-19 08:02:10.112519	2021-05-20 14:12:30.473237	21
148	21	157	2021-05-19 12:46:46.514485	2021-05-20 14:12:30.483839	21
149	21	158	2021-05-15 01:38:39.925148	2021-05-20 14:12:30.494043	21
150	21	159	2021-05-14 22:07:08.956237	2021-05-20 14:12:30.505674	21
151	21	160	2021-05-15 01:20:44.683441	2021-05-20 14:12:30.516785	21
152	21	161	2021-05-16 04:22:56.115374	2021-05-20 14:12:30.527205	21
153	21	162	2021-05-16 11:33:41.708162	2021-05-20 14:12:30.536685	21
154	21	163	2021-05-18 05:50:18.855147	2021-05-20 14:12:30.544957	21
155	22	164	2021-05-16 15:41:59.654615	2021-05-20 14:12:57.157631	22
156	22	165	2021-05-17 17:58:14.436208	2021-05-20 14:12:57.169582	22
157	22	166	2021-05-19 14:43:46.1421	2021-05-20 14:12:57.180231	22
158	22	167	2021-05-15 16:28:10.340321	2021-05-20 14:12:57.190793	22
159	22	168	2021-05-20 10:51:20.923456	2021-05-20 14:12:57.201744	22
160	22	169	2021-05-16 23:24:02.290486	2021-05-20 14:12:57.212117	22
161	22	170	2021-05-18 22:31:22.092551	2021-05-20 14:12:57.221725	22
162	22	171	2021-05-18 07:05:38.89336	2021-05-20 14:12:57.23408	22
163	22	172	2021-05-20 08:59:31.700389	2021-05-20 14:12:57.242073	22
164	23	173	2021-05-17 08:14:39.444597	2021-05-20 14:13:33.987015	23
165	23	174	2021-05-16 05:55:14.261692	2021-05-20 14:13:33.997029	23
166	23	175	2021-05-15 05:36:25.212598	2021-05-20 14:13:34.007203	23
167	23	176	2021-05-18 13:37:39.155038	2021-05-20 14:13:34.016725	23
168	23	177	2021-05-18 09:17:22.92793	2021-05-20 14:13:34.028706	23
169	23	178	2021-05-15 22:51:57.673595	2021-05-20 14:13:34.040694	23
170	23	179	2021-05-19 07:30:06.600834	2021-05-20 14:13:34.051067	23
171	23	180	2021-05-19 03:22:49.929495	2021-05-20 14:13:34.06045	23
172	23	181	2021-05-14 16:24:45.591805	2021-05-20 14:13:34.068052	23
173	24	182	2021-05-20 01:38:10.956568	2021-05-20 14:21:20.54707	24
174	24	183	2021-05-16 18:06:32.434372	2021-05-20 14:21:20.561235	24
175	24	184	2021-05-14 14:21:41.43115	2021-05-20 14:21:20.573595	24
176	24	185	2021-05-15 18:29:58.07602	2021-05-20 14:21:20.586175	24
177	24	186	2021-05-19 16:32:41.783998	2021-05-20 14:21:20.597503	24
178	24	187	2021-05-15 15:53:14.619615	2021-05-20 14:21:20.609449	24
179	24	188	2021-05-18 20:13:00.583508	2021-05-20 14:21:20.622492	24
180	24	189	2021-05-19 00:52:43.943393	2021-05-20 14:21:20.632909	24
181	24	190	2021-05-17 23:14:59.56194	2021-05-20 14:21:20.640197	24
182	25	191	2021-05-19 04:59:26.108712	2021-05-20 14:30:30.312668	25
183	25	192	2021-05-19 08:45:17.320093	2021-05-20 14:30:30.323208	25
184	25	193	2021-05-16 15:50:41.938415	2021-05-20 14:30:30.333387	25
185	25	194	2021-05-14 21:30:58.889603	2021-05-20 14:30:30.343962	25
186	25	195	2021-05-18 11:32:39.582264	2021-05-20 14:30:30.354654	25
187	25	196	2021-05-17 21:00:58.737835	2021-05-20 14:30:30.365079	25
188	25	197	2021-05-14 17:20:35.973054	2021-05-20 14:30:30.375741	25
189	25	198	2021-05-18 07:00:59.369335	2021-05-20 14:30:30.385621	25
190	25	199	2021-05-18 19:38:23.257848	2021-05-20 14:30:30.393626	25
191	26	200	2021-05-16 02:46:12.998456	2021-05-20 15:17:11.706838	26
192	26	201	2021-05-15 04:00:15.41741	2021-05-20 15:17:11.719677	26
193	26	202	2021-05-15 08:59:53.008953	2021-05-20 15:17:11.73306	26
194	26	203	2021-05-18 03:20:00.873855	2021-05-20 15:17:11.743719	26
195	26	204	2021-05-18 04:04:43.666303	2021-05-20 15:17:11.767716	26
196	26	205	2021-05-17 10:47:31.042357	2021-05-20 15:17:11.778918	26
197	26	206	2021-05-14 19:30:07.381563	2021-05-20 15:17:11.789655	26
198	26	207	2021-05-15 05:11:27.242295	2021-05-20 15:17:11.798525	26
199	26	208	2021-05-17 03:41:27.6443	2021-05-20 15:17:11.805795	26
200	27	212	2021-05-25 18:52:56.607816	2021-05-28 15:07:44.944916	27
201	27	213	2021-05-22 16:44:22.057115	2021-05-28 15:07:44.99192	27
202	27	214	2021-05-23 01:18:19.75162	2021-05-28 15:07:45.007311	27
203	27	215	2021-05-27 21:09:53.96886	2021-05-28 15:07:45.020404	27
204	27	216	2021-05-26 12:15:16.380849	2021-05-28 15:07:45.03299	27
205	27	217	2021-05-24 21:45:16.55452	2021-05-28 15:07:45.061898	27
206	27	218	2021-05-22 20:07:18.886597	2021-05-28 15:07:45.074348	27
207	27	219	2021-05-24 12:25:02.7748	2021-05-28 15:07:45.087029	27
208	27	220	2021-05-22 15:12:42.433883	2021-05-28 15:07:45.110763	27
209	28	221	2021-06-02 08:33:52.939704	2021-06-02 16:10:01.853401	28
210	28	222	2021-05-30 12:04:25.583278	2021-06-02 16:10:01.918138	28
211	28	223	2021-05-31 16:03:13.699694	2021-06-02 16:10:02.02534	28
212	28	224	2021-05-31 12:42:41.217116	2021-06-02 16:10:02.127719	28
213	28	225	2021-05-31 18:24:04.365814	2021-06-02 16:10:02.241749	28
214	28	226	2021-05-28 10:35:24.577402	2021-06-02 16:10:02.476874	28
215	28	227	2021-06-02 11:56:21.408211	2021-06-02 16:10:02.525612	28
216	28	228	2021-05-29 21:09:09.365182	2021-06-02 16:10:02.555656	28
217	28	229	2021-05-30 13:59:03.503468	2021-06-02 16:10:02.583544	28
218	27	230	2021-06-10 16:08:40.593251	2021-06-10 16:08:40.593251	27
219	27	235	2021-07-06 15:51:27.737161	2021-07-06 15:51:27.737161	27
220	30	241	2021-08-21 07:51:14.82213	2021-08-23 14:58:27.693403	30
221	30	242	2021-08-19 13:27:28.603031	2021-08-23 14:58:27.72202	30
222	30	243	2021-08-21 13:05:31.861303	2021-08-23 14:58:27.739211	30
223	30	244	2021-08-19 03:00:41.073251	2021-08-23 14:58:27.758218	30
224	30	245	2021-08-21 05:11:35.04865	2021-08-23 14:58:27.777065	30
225	30	246	2021-08-20 08:05:35.479338	2021-08-23 14:58:27.79457	30
226	30	247	2021-08-20 21:54:41.813292	2021-08-23 14:58:27.80574	30
227	30	248	2021-08-21 12:52:20.103144	2021-08-23 14:58:27.821836	30
228	30	249	2021-08-20 03:18:52.738682	2021-08-23 14:58:27.832727	30
229	31	250	2021-08-19 22:53:58.728512	2021-08-23 17:19:39.200401	31
230	31	251	2021-08-19 03:21:19.396371	2021-08-23 17:19:39.213667	31
231	31	252	2021-08-21 18:41:53.150834	2021-08-23 17:19:39.224729	31
232	31	253	2021-08-20 14:53:34.993347	2021-08-23 17:19:39.235443	31
233	31	254	2021-08-19 08:56:18.638399	2021-08-23 17:19:39.246765	31
234	31	255	2021-08-21 14:54:21.980421	2021-08-23 17:19:39.258523	31
235	31	256	2021-08-20 03:12:03.26344	2021-08-23 17:19:39.270297	31
236	31	257	2021-08-20 07:32:16.977562	2021-08-23 17:19:39.28104	31
237	31	258	2021-08-19 15:27:39.94221	2021-08-23 17:19:39.289526	31
\.


--
-- Data for Name: user_notifications; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.user_notifications (id, user_id, notification_id, checked, created_at, updated_at) FROM stdin;
1	2	1	f	2019-07-22 11:21:59.642146	2019-07-22 11:21:59.642146
2	3	2	t	2019-07-22 11:21:59.897401	2019-07-22 11:21:59.897401
4	3	4	f	2019-08-28 08:36:02.557673	2019-08-28 08:36:02.557673
5	6	5	t	2019-08-28 09:15:40.749862	2019-08-28 09:15:40.749862
7	8	7	f	2019-10-21 16:32:02.993022	2019-10-21 16:32:02.993022
8	10	9	t	2019-11-25 18:03:25.15582	2019-11-25 18:03:25.15582
6	7	6	t	2019-09-12 15:39:47.550811	2019-09-12 15:39:47.550811
9	7	10	f	2020-06-12 11:55:49.20305	2020-06-12 11:55:49.20305
10	2	11	f	2020-09-28 12:22:40.284784	2020-09-28 12:22:40.284784
11	3	12	f	2020-09-28 12:22:40.362756	2020-09-28 12:22:40.362756
12	13	13	f	2020-09-28 12:22:40.75306	2020-09-28 12:22:40.75306
13	14	14	f	2020-09-28 12:22:40.919542	2020-09-28 12:22:40.919542
14	2	15	f	2020-09-28 13:00:11.626161	2020-09-28 13:00:11.626161
15	15	16	f	2020-09-28 13:00:12.0257	2020-09-28 13:00:12.0257
16	13	17	f	2020-09-28 13:00:12.054623	2020-09-28 13:00:12.054623
17	18	18	f	2021-04-08 10:19:49.32155	2021-04-08 10:19:49.32155
18	19	19	f	2021-04-08 10:20:34.249602	2021-04-08 10:20:34.249602
19	10	20	f	2021-04-08 13:28:21.978357	2021-04-08 13:28:21.978357
20	7	23	f	2021-04-08 15:53:54.293498	2021-04-08 15:53:54.293498
21	7	24	f	2021-04-08 15:54:38.85975	2021-04-08 15:54:38.85975
22	29	25	f	2021-06-10 16:09:28.298819	2021-06-10 16:09:28.298819
23	30	34	t	2021-08-25 13:47:15.185997	2021-08-25 13:47:15.185997
24	31	35	f	2021-08-25 14:57:58.968278	2021-08-25 14:57:58.968278
25	33	36	f	2021-11-16 15:17:40.529231	2021-11-16 15:17:40.529231
26	34	37	f	2022-02-09 13:05:39.498214	2022-02-09 13:05:39.498214
27	33	38	f	2022-02-09 18:49:48.472068	2022-02-09 18:49:48.472068
\.


--
-- Data for Name: user_projects; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.user_projects (id, role, user_id, project_id, created_at, updated_at, assigned_by_id) FROM stdin;
1	0	1	1	2019-05-30 06:40:29.942488	2019-05-30 12:40:53.191988	\N
2	0	4	2	2019-08-26 01:10:15.495071	2019-08-27 15:14:59.765912	\N
3	0	1	3	2019-08-22 16:57:15.63032	2019-08-28 07:30:58.584882	\N
4	0	6	4	2019-08-25 00:28:51.243418	2019-08-28 08:34:53.545575	\N
5	0	7	5	2019-08-24 15:25:25.944939	2019-08-29 12:51:23.042922	\N
6	0	8	6	2019-09-12 06:51:05.662783	2019-09-12 18:26:18.391309	\N
7	0	1	21	2019-10-22 15:41:41.761689	2019-10-22 15:41:41.761689	\N
8	0	9	22	2019-11-12 03:55:08.168855	2019-11-13 13:57:41.44248	\N
9	0	4	23	2019-11-23 10:35:11.372357	2019-11-25 14:09:22.13923	\N
10	0	4	24	2019-11-25 14:09:53.321142	2019-11-25 14:09:53.321142	\N
11	0	4	25	2019-11-25 14:25:41.556954	2019-11-25 14:25:41.556954	\N
12	0	10	26	2019-11-21 09:28:05.477138	2019-11-25 18:00:52.424923	\N
13	1	10	25	2019-11-25 18:05:32.326059	2019-11-25 18:05:32.326059	4
14	0	11	27	2020-05-12 11:52:05.150216	2020-05-12 16:13:01.660189	\N
15	0	12	28	2020-06-10 10:51:09.753637	2020-06-10 14:43:16.855566	\N
16	0	7	29	2020-06-10 18:15:06.984498	2020-06-10 18:15:06.984498	\N
17	0	4	30	2020-06-12 11:47:19.264477	2020-06-12 11:47:19.264477	\N
18	0	4	31	2020-06-12 12:03:51.645406	2020-06-12 12:03:51.645406	\N
20	0	10	30	2020-06-12 12:04:45.749489	2020-06-12 12:04:45.749489	4
23	0	10	23	2020-06-12 12:05:08.905292	2020-06-12 12:05:08.905292	4
25	0	1	32	2020-09-28 12:07:47.049852	2020-09-28 12:07:47.049852	\N
26	0	16	33	2020-10-02 18:17:26.439173	2020-10-07 19:20:24.89863	\N
27	0	1	34	2020-11-15 16:02:40.403467	2020-11-18 14:50:52.127033	\N
28	0	17	35	2020-11-14 20:11:49.629974	2020-11-18 14:53:24.699025	\N
29	0	1	36	2021-05-19 14:31:33.227748	2021-05-19 14:31:33.227748	\N
30	0	20	37	2021-05-14 10:38:41.648751	2021-05-19 20:01:51.652105	\N
31	0	21	38	2021-05-13 15:41:16.117627	2021-05-20 14:12:30.429774	\N
32	0	22	39	2021-05-15 08:05:18.384643	2021-05-20 14:12:57.140847	\N
33	0	23	40	2021-05-14 17:29:25.318273	2021-05-20 14:13:33.975466	\N
34	0	24	41	2021-05-20 04:05:22.379763	2021-05-20 14:21:20.531177	\N
35	0	24	42	2021-05-20 14:22:42.022696	2021-05-20 14:22:42.022696	\N
36	0	25	43	2021-05-14 15:33:57.773674	2021-05-20 14:30:30.300353	\N
37	0	26	44	2021-05-20 03:50:08.321275	2021-05-20 15:17:11.689419	\N
38	0	21	45	2021-05-20 15:31:53.778007	2021-05-20 15:31:53.778007	\N
39	0	27	46	2021-05-22 22:27:14.95094	2021-05-28 15:07:44.626975	\N
40	0	27	47	2021-05-28 15:08:46.407373	2021-05-28 15:08:46.407373	\N
41	0	27	48	2021-05-28 15:09:19.348459	2021-05-28 15:09:19.348459	\N
42	0	27	49	2021-05-28 15:09:34.711014	2021-05-28 15:09:34.711014	\N
43	0	27	50	2021-05-28 15:09:46.644142	2021-05-28 15:09:46.644142	\N
44	0	27	51	2021-05-28 15:10:03.266536	2021-05-28 15:10:03.266536	\N
45	0	27	52	2021-05-28 15:10:11.199818	2021-05-28 15:10:11.199818	\N
46	0	27	53	2021-05-28 15:10:21.216393	2021-05-28 15:10:21.216393	\N
47	0	28	54	2021-05-28 12:20:32.47684	2021-06-02 16:10:01.65039	\N
48	0	30	55	2021-08-17 15:04:31.155477	2021-08-23 14:58:27.548324	\N
49	0	31	56	2021-08-19 09:20:53.753338	2021-08-23 17:19:39.184747	\N
50	0	30	57	2021-08-23 18:49:48.21403	2021-08-23 18:49:48.21403	\N
51	0	30	58	2021-08-25 16:06:30.307809	2021-08-25 16:06:30.307809	\N
52	0	30	59	2021-08-26 20:02:24.239451	2021-08-26 20:02:24.239451	\N
53	0	30	60	2021-08-26 20:53:28.256795	2021-08-26 20:53:28.256795	\N
54	0	33	32	2022-02-08 21:10:33.076235	2022-02-09 13:34:26.409117	1
55	0	34	32	2022-02-09 13:06:20.494481	2022-02-09 13:34:28.278961	1
\.



--
-- Data for Name: user_teams; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.user_teams (id, role, user_id, team_id, created_at, updated_at, assigned_by_id) FROM stdin;
1	2	1	1	2019-05-30 12:40:54.35633	2019-05-30 12:40:54.35633	\N
2	2	1	2	2019-07-22 11:19:36.274914	2019-07-22 11:19:36.274914	\N
3	1	2	2	2019-07-22 11:21:59.527914	2019-07-22 11:21:59.527914	\N
5	2	4	3	2019-08-27 15:15:03.966014	2019-08-27 15:15:03.966014	\N
4	2	3	2	2019-07-22 11:21:59.884779	2019-07-22 11:21:59.884779	\N
7	2	6	4	2019-08-28 08:34:57.447794	2019-08-28 08:34:57.447794	\N
8	2	3	4	2019-08-28 08:36:02.094007	2019-08-28 08:36:02.094007	\N
9	2	6	2	2019-08-28 09:15:40.189824	2019-08-28 09:15:40.189824	\N
10	2	7	5	2019-08-29 12:51:29.444975	2019-08-29 12:51:29.444975	\N
12	2	8	6	2019-09-12 18:26:19.69029	2019-09-12 18:26:19.69029	\N
13	0	8	3	2019-10-21 16:32:02.783915	2019-10-21 16:32:02.783915	\N
14	2	4	7	2019-10-21 16:34:59.048094	2019-10-21 16:34:59.048094	\N
15	2	9	8	2019-11-13 13:57:43.696705	2019-11-13 13:57:43.696705	\N
16	2	10	9	2019-11-25 18:00:54.116996	2019-11-25 18:00:54.116996	\N
18	2	11	10	2020-05-12 16:13:07.147671	2020-05-12 16:13:07.147671	\N
19	2	12	11	2020-06-10 14:43:18.857076	2020-06-10 14:43:18.857076	\N
21	2	1	12	2020-09-28 12:17:20.135925	2020-09-28 12:17:20.135925	\N
22	1	2	12	2020-09-28 12:22:40.144081	2020-09-28 12:22:40.144081	\N
24	1	13	12	2020-09-28 12:22:40.738356	2020-09-28 12:22:40.738356	\N
25	1	14	12	2020-09-28 12:22:40.873988	2020-09-28 12:22:40.873988	\N
26	1	2	1	2020-09-28 13:00:11.572994	2020-09-28 13:00:11.572994	\N
27	1	15	1	2020-09-28 13:00:12.011984	2020-09-28 13:00:12.011984	\N
28	1	13	1	2020-09-28 13:00:12.040601	2020-09-28 13:00:12.040601	\N
29	2	16	13	2020-10-07 19:20:27.777567	2020-10-07 19:20:27.777567	\N
30	2	17	14	2020-11-18 14:53:26.236344	2020-11-18 14:53:26.236344	\N
31	1	18	12	2021-04-08 10:19:49.232356	2021-04-08 10:19:49.232356	\N
32	1	19	12	2021-04-08 10:20:34.165002	2021-04-08 10:20:34.165002	\N
33	2	20	15	2021-05-19 20:01:52.942259	2021-05-19 20:01:52.942259	\N
23	2	3	12	2020-09-28 12:22:40.340631	2021-05-20 13:02:21.918981	\N
34	2	21	16	2021-05-20 14:12:31.431303	2021-05-20 14:12:31.431303	\N
35	2	22	17	2021-05-20 14:12:57.948188	2021-05-20 14:12:57.948188	\N
36	2	23	18	2021-05-20 14:13:34.828749	2021-05-20 14:13:34.828749	\N
37	2	24	19	2021-05-20 14:21:21.48987	2021-05-20 14:21:21.48987	\N
38	2	25	20	2021-05-20 14:30:31.148923	2021-05-20 14:30:31.148923	\N
39	2	26	21	2021-05-20 15:17:12.511153	2021-05-20 15:17:12.511153	\N
40	2	27	22	2021-05-28 15:07:58.665831	2021-05-28 15:07:58.665831	\N
41	2	28	23	2021-06-02 16:10:07.717891	2021-06-02 16:10:07.717891	\N
42	1	29	22	2021-06-10 16:09:28.154796	2021-06-10 16:09:28.154796	\N
43	2	30	24	2021-08-23 14:58:28.966447	2021-08-23 14:58:28.966447	\N
44	2	31	25	2021-08-23 17:19:40.140556	2021-08-23 17:19:40.140556	\N
45	1	30	25	2021-08-25 13:47:15.109274	2021-08-25 13:47:15.109274	\N
46	2	31	24	2021-08-25 14:57:58.939982	2021-08-25 14:58:54.29667	\N
47	2	32	26	2021-11-05 12:34:58.761523	2021-11-05 12:34:58.761523	\N
48	2	33	27	2021-11-15 13:50:58.165427	2021-11-15 13:50:58.165427	\N
49	2	33	1	2021-11-16 15:17:40.499654	2021-11-16 15:17:40.499654	\N
17	0	10	7	2019-11-25 18:03:25.106088	2021-12-08 18:50:02.409355	\N
50	2	4	28	2021-12-08 18:53:57.741503	2021-12-08 18:53:57.741503	\N
51	2	34	29	2022-02-09 13:04:39.923459	2022-02-09 13:04:39.923459	\N
52	2	34	1	2022-02-09 13:05:39.462006	2022-02-09 13:06:40.676037	\N
53	2	33	12	2022-02-09 18:49:48.440287	2022-02-09 18:49:48.440287	\N
54	2	35	30	2022-02-09 18:56:38.615373	2022-02-09 18:56:38.615373	\N
\.


--
-- Data for Name: view_states; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.view_states (id, state, user_id, viewable_type, viewable_id, created_at, updated_at) FROM stdin;
23	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1589299987, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	11	Team	10	2020-05-12 16:13:07.406871	2020-05-12 16:13:07.406871
24	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1591800199, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	12	Team	11	2020-06-10 14:43:19.130658	2020-06-10 14:43:19.130658
2	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1566918904, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	4	Team	3	2019-08-27 15:15:04.417664	2019-08-27 15:38:39.448055
25	{"assets": {"sort": "new"}}	12	Step	401	2020-06-10 14:44:25.050253	2020-06-10 14:44:25.050253
3	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1566977642, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "all"}}	1	Team	2	2019-08-28 07:34:02.978316	2019-08-28 08:20:50.162388
4	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1566981297, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	6	Team	4	2019-08-28 08:34:57.756893	2019-08-28 08:34:57.756893
6	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1568312779, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	8	Team	6	2019-09-12 18:26:19.87508	2019-09-12 18:26:19.87508
26	{"assets": {"sort": "new"}}	12	Step	402	2020-06-10 14:44:50.559228	2020-06-10 14:44:50.559228
27	{"assets": {"sort": "new"}}	12	Step	403	2020-06-10 14:44:50.731951	2020-06-10 14:44:50.731951
8	{"assets": {"sort": "new"}}	4	Step	41	2019-11-25 14:08:57.058079	2019-11-25 14:08:57.058079
28	{"assets": {"sort": "new"}}	12	Step	404	2020-06-10 14:44:50.776969	2020-06-10 14:44:50.776969
29	{"assets": {"sort": "new"}}	12	Step	434	2020-06-10 14:45:02.776131	2020-06-10 14:45:02.776131
30	{"assets": {"sort": "new"}}	12	Step	435	2020-06-10 14:45:02.872558	2020-06-10 14:45:02.872558
9	{"assets": {"sort": "new"}}	4	Step	285	2019-11-25 14:27:41.353663	2019-11-25 14:27:41.353663
10	{"assets": {"sort": "new"}}	4	Step	286	2019-11-25 14:27:41.388597	2019-11-25 14:27:41.388597
11	{"assets": {"sort": "new"}}	4	Step	287	2019-11-25 14:27:41.430132	2019-11-25 14:27:41.430132
12	{"assets": {"sort": "new"}}	4	Step	288	2019-11-25 14:27:41.463308	2019-11-25 14:27:41.463308
13	{"assets": {"sort": "new"}}	4	Step	289	2019-11-25 14:27:41.497714	2019-11-25 14:27:41.497714
14	{"assets": {"sort": "new"}}	4	Step	290	2019-11-25 14:27:41.532735	2019-11-25 14:27:41.532735
15	{"assets": {"sort": "new"}}	4	Step	291	2019-11-25 14:27:41.563107	2019-11-25 14:27:41.563107
16	{"assets": {"sort": "new"}}	4	Step	292	2019-11-25 14:27:41.595857	2019-11-25 14:27:41.595857
17	{"assets": {"sort": "new"}}	4	Step	293	2019-11-25 14:27:41.659432	2019-11-25 14:27:41.659432
18	{"assets": {"sort": "new"}}	4	Step	294	2019-11-25 14:27:41.69923	2019-11-25 14:27:41.69923
19	{"assets": {"sort": "new"}}	4	Step	295	2019-11-25 14:27:41.791379	2019-11-25 14:27:41.791379
20	{"assets": {"sort": "new"}}	4	Step	296	2019-11-25 14:27:41.827071	2019-11-25 14:27:41.827071
21	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1574704854, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	10	Team	9	2019-11-25 18:00:54.293973	2019-11-25 18:00:54.293973
22	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1574705184, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	10	Team	7	2019-11-25 18:06:24.094703	2019-11-25 18:06:24.094703
31	{"assets": {"sort": "new"}}	12	Step	436	2020-06-10 14:45:02.918413	2020-06-10 14:45:02.918413
32	{"assets": {"sort": "new"}}	12	Step	437	2020-06-10 14:45:02.945808	2020-06-10 14:45:02.945808
33	{"assets": {"sort": "new"}}	12	Step	438	2020-06-10 14:45:02.982284	2020-06-10 14:45:02.982284
34	{"assets": {"sort": "new"}}	12	Step	439	2020-06-10 14:45:03.018671	2020-06-10 14:45:03.018671
35	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1591812574, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	7	Team	3	2020-06-10 18:09:34.620687	2020-06-10 18:09:34.620687
36	{"assets": {"sort": "new"}}	7	Step	441	2020-06-10 18:14:51.084246	2020-06-10 18:14:51.084246
37	{"assets": {"sort": "new"}}	7	Step	442	2020-06-10 18:18:36.741032	2020-06-10 18:18:36.741032
38	{"assets": {"sort": "new"}}	7	Step	161	2020-06-10 18:29:25.255766	2020-06-10 18:29:25.255766
39	{"assets": {"sort": "new"}}	7	Step	191	2020-06-10 18:29:45.898402	2020-06-10 18:29:45.898402
40	{"assets": {"sort": "new"}}	7	Step	192	2020-06-10 18:29:45.98757	2020-06-10 18:29:45.98757
41	{"assets": {"sort": "new"}}	7	Step	193	2020-06-10 18:29:46.027926	2020-06-10 18:29:46.027926
42	{"assets": {"sort": "new"}}	7	Step	200	2020-06-10 18:38:17.72397	2020-06-10 18:38:17.72397
43	{"assets": {"sort": "new"}}	7	Step	165	2020-06-10 19:04:48.991826	2020-06-10 19:04:48.991826
44	{"assets": {"sort": "new"}}	7	Step	166	2020-06-10 19:04:49.02796	2020-06-10 19:04:49.02796
45	{"assets": {"sort": "new"}}	7	Step	167	2020-06-10 19:04:49.058259	2020-06-10 19:04:49.058259
46	{"assets": {"sort": "new"}}	7	Step	168	2020-06-10 19:04:49.085906	2020-06-10 19:04:49.085906
47	{"assets": {"sort": "new"}}	7	Step	169	2020-06-10 19:04:49.115834	2020-06-10 19:04:49.115834
48	{"assets": {"sort": "new"}}	7	Step	170	2020-06-10 19:04:49.167796	2020-06-10 19:04:49.167796
49	{"assets": {"sort": "new"}}	7	Step	171	2020-06-10 19:04:49.196636	2020-06-10 19:04:49.196636
50	{"assets": {"sort": "new"}}	7	Step	172	2020-06-10 19:04:49.22362	2020-06-10 19:04:49.22362
51	{"assets": {"sort": "new"}}	7	Step	173	2020-06-10 19:04:49.254989	2020-06-10 19:04:49.254989
52	{"assets": {"sort": "new"}}	7	Step	174	2020-06-10 19:04:49.288921	2020-06-10 19:04:49.288921
53	{"assets": {"sort": "new"}}	7	Step	175	2020-06-10 19:04:49.318242	2020-06-10 19:04:49.318242
54	{"assets": {"sort": "new"}}	7	Step	176	2020-06-10 19:04:49.351549	2020-06-10 19:04:49.351549
5	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1567083090, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "all"}}	7	Team	5	2019-08-29 12:51:30.580887	2020-06-10 19:05:50.144953
55	{"assets": {"sort": "new"}}	4	Step	282	2020-06-12 11:53:42.067634	2020-06-12 11:53:42.067634
56	{"assets": {"sort": "new"}}	4	Step	283	2020-06-12 11:53:42.120766	2020-06-12 11:53:42.120766
57	{"assets": {"sort": "new"}}	4	Step	284	2020-06-12 11:53:42.152839	2020-06-12 11:53:42.152839
58	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1599097806, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	9	Team	8	2020-09-03 01:50:06.50105	2020-09-03 01:50:06.50105
59	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1605711206, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	17	Team	14	2020-11-18 14:53:26.67177	2020-11-18 14:53:26.67177
62	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621454513, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	20	Team	15	2021-05-19 20:01:53.077591	2021-05-19 20:01:53.077591
63	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621519951, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	21	Team	16	2021-05-20 14:12:31.624853	2021-05-20 14:12:31.624853
64	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621519978, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	22	Team	17	2021-05-20 14:12:58.050067	2021-05-20 14:12:58.050067
65	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621520014, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	23	Team	18	2021-05-20 14:13:34.945423	2021-05-20 14:13:34.945423
66	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621520481, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "archived"}}	24	Team	19	2021-05-20 14:21:21.659416	2021-05-20 14:44:15.914909
85	{"assets": {"sort": "new"}}	24	Step	741	2021-05-20 14:45:00.724508	2021-05-20 14:45:00.724508
67	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621521031, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	25	Team	20	2021-05-20 14:30:31.304467	2021-05-20 14:30:31.304467
68	{"assets": {"sort": "new"}}	24	Step	725	2021-05-20 14:32:54.176996	2021-05-20 14:32:54.176996
69	{"assets": {"sort": "new"}}	24	Step	726	2021-05-20 14:33:09.598914	2021-05-20 14:33:09.598914
70	{"assets": {"sort": "new"}}	24	Step	727	2021-05-20 14:33:09.681919	2021-05-20 14:33:09.681919
71	{"assets": {"sort": "new"}}	24	Step	728	2021-05-20 14:33:09.711275	2021-05-20 14:33:09.711275
72	{"assets": {"sort": "new"}}	24	Step	729	2021-05-20 14:33:51.020021	2021-05-20 14:33:51.020021
73	{"assets": {"sort": "new"}}	24	Step	730	2021-05-20 14:33:51.042729	2021-05-20 14:33:51.042729
74	{"assets": {"sort": "new"}}	24	Step	731	2021-05-20 14:33:51.06429	2021-05-20 14:33:51.06429
75	{"assets": {"sort": "new"}}	24	Step	732	2021-05-20 14:33:51.083217	2021-05-20 14:33:51.083217
76	{"assets": {"sort": "new"}}	24	Step	733	2021-05-20 14:33:51.111359	2021-05-20 14:33:51.111359
77	{"assets": {"sort": "new"}}	24	Step	734	2021-05-20 14:33:51.134798	2021-05-20 14:33:51.134798
78	{"assets": {"sort": "new"}}	24	Step	735	2021-05-20 14:33:51.156384	2021-05-20 14:33:51.156384
79	{"assets": {"sort": "new"}}	24	Step	736	2021-05-20 14:33:51.175282	2021-05-20 14:33:51.175282
80	{"assets": {"sort": "new"}}	24	Step	737	2021-05-20 14:33:51.196717	2021-05-20 14:33:51.196717
81	{"assets": {"sort": "new"}}	24	Step	738	2021-05-20 14:33:51.216219	2021-05-20 14:33:51.216219
82	{"assets": {"sort": "new"}}	24	Step	739	2021-05-20 14:33:51.236836	2021-05-20 14:33:51.236836
83	{"assets": {"sort": "new"}}	24	Step	740	2021-05-20 14:33:51.257924	2021-05-20 14:33:51.257924
86	{"assets": {"sort": "new"}}	24	Step	742	2021-05-20 14:45:00.778883	2021-05-20 14:45:00.778883
87	{"assets": {"sort": "new"}}	24	Step	743	2021-05-20 14:45:00.802412	2021-05-20 14:45:00.802412
84	{"assets": {"sort": "new"}}	24	Step	805	2021-05-20 14:41:37.284416	2021-05-20 14:41:37.284416
88	{"assets": {"sort": "new"}}	24	Step	744	2021-05-20 14:45:00.825277	2021-05-20 14:45:00.825277
102	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1621523832, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	26	Team	21	2021-05-20 15:17:12.825325	2021-05-20 15:17:12.825325
89	{"assets": {"sort": "new"}}	24	Step	745	2021-05-20 14:45:00.851478	2021-05-20 14:45:00.851478
90	{"assets": {"sort": "new"}}	24	Step	746	2021-05-20 14:45:00.871474	2021-05-20 14:45:00.871474
91	{"assets": {"sort": "new"}}	24	Step	747	2021-05-20 14:45:00.898707	2021-05-20 14:45:00.898707
92	{"assets": {"sort": "new"}}	24	Step	748	2021-05-20 14:45:00.933785	2021-05-20 14:45:00.933785
93	{"assets": {"sort": "new"}}	24	Step	749	2021-05-20 14:45:00.958889	2021-05-20 14:45:00.958889
94	{"assets": {"sort": "new"}}	24	Step	750	2021-05-20 14:45:00.982451	2021-05-20 14:45:00.982451
95	{"assets": {"sort": "new"}}	24	Step	751	2021-05-20 14:45:01.00832	2021-05-20 14:45:01.00832
96	{"assets": {"sort": "new"}}	24	Step	752	2021-05-20 14:45:13.030765	2021-05-20 14:45:13.030765
97	{"assets": {"sort": "new"}}	24	Step	753	2021-05-20 14:45:13.049335	2021-05-20 14:45:13.049335
98	{"assets": {"sort": "new"}}	24	Step	754	2021-05-20 14:45:13.090177	2021-05-20 14:45:13.090177
99	{"assets": {"sort": "new"}}	22	Step	646	2021-05-20 14:57:29.145529	2021-05-20 14:57:29.145529
100	{"assets": {"sort": "new"}}	22	Step	647	2021-05-20 14:57:29.184292	2021-05-20 14:57:29.184292
101	{"assets": {"sort": "new"}}	22	Step	648	2021-05-20 14:57:29.207754	2021-05-20 14:57:29.207754
103	{"assets": {"sort": "new"}}	22	Step	678	2021-05-20 15:42:23.580346	2021-05-20 15:42:23.580346
104	{"assets": {"sort": "new"}}	22	Step	679	2021-05-20 15:42:23.623586	2021-05-20 15:42:23.623586
105	{"assets": {"sort": "new"}}	22	Step	680	2021-05-20 15:42:23.658782	2021-05-20 15:42:23.658782
106	{"assets": {"sort": "new"}}	22	Step	681	2021-05-20 15:42:23.682223	2021-05-20 15:42:23.682223
107	{"assets": {"sort": "new"}}	22	Step	682	2021-05-20 15:42:23.703857	2021-05-20 15:42:23.703857
108	{"assets": {"sort": "new"}}	22	Step	683	2021-05-20 15:42:23.726928	2021-05-20 15:42:23.726928
109	{"assets": {"sort": "new"}}	22	Step	684	2021-05-20 17:01:37.238335	2021-05-20 17:01:37.238335
110	{"assets": {"sort": "new"}}	22	Step	672	2021-05-20 17:01:40.323537	2021-05-20 17:01:40.323537
111	{"assets": {"sort": "new"}}	22	Step	673	2021-05-20 17:01:40.349528	2021-05-20 17:01:40.349528
112	{"assets": {"sort": "new"}}	22	Step	674	2021-05-20 17:01:40.39409	2021-05-20 17:01:40.39409
113	{"assets": {"sort": "new"}}	22	Step	649	2021-05-20 17:01:44.446	2021-05-20 17:01:44.446
114	{"assets": {"sort": "new"}}	22	Step	650	2021-05-20 17:01:44.467863	2021-05-20 17:01:44.467863
115	{"assets": {"sort": "new"}}	22	Step	651	2021-05-20 17:01:44.495093	2021-05-20 17:01:44.495093
116	{"assets": {"sort": "new"}}	22	Step	652	2021-05-20 17:01:44.515214	2021-05-20 17:01:44.515214
117	{"assets": {"sort": "new"}}	22	Step	653	2021-05-20 17:01:44.543433	2021-05-20 17:01:44.543433
118	{"assets": {"sort": "new"}}	22	Step	654	2021-05-20 17:01:44.567715	2021-05-20 17:01:44.567715
119	{"assets": {"sort": "new"}}	22	Step	655	2021-05-20 17:01:44.592268	2021-05-20 17:01:44.592268
120	{"assets": {"sort": "new"}}	22	Step	656	2021-05-20 17:01:44.613973	2021-05-20 17:01:44.613973
121	{"assets": {"sort": "new"}}	22	Step	657	2021-05-20 17:01:44.637712	2021-05-20 17:01:44.637712
122	{"assets": {"sort": "new"}}	22	Step	658	2021-05-20 17:01:44.663754	2021-05-20 17:01:44.663754
123	{"assets": {"sort": "new"}}	22	Step	659	2021-05-20 17:01:44.689873	2021-05-20 17:01:44.689873
124	{"assets": {"sort": "new"}}	22	Step	660	2021-05-20 17:01:44.711327	2021-05-20 17:01:44.711327
125	{"assets": {"sort": "new"}}	21	Step	846	2021-05-26 16:37:20.288291	2021-05-26 16:37:20.288291
126	{"assets": {"sort": "new"}}	21	Step	847	2021-05-27 14:50:15.453652	2021-05-27 14:50:15.453652
127	{"assets": {"sort": "new"}}	21	Step	848	2021-05-27 14:51:12.032458	2021-05-27 14:51:12.032458
128	{"assets": {"sort": "new"}}	21	Step	849	2021-05-27 17:00:13.537321	2021-05-27 17:00:13.537321
129	{"assets": {"sort": "new"}}	21	Step	850	2021-05-27 17:54:48.755089	2021-05-27 17:54:48.755089
130	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1622214478, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	27	Team	22	2021-05-28 15:07:58.941071	2021-05-28 15:07:58.941071
131	{"assets": {"sort": "new"}}	21	Step	891	2021-06-01 16:23:59.408463	2021-06-01 16:23:59.408463
132	{"assets": {"sort": "new"}}	21	Step	892	2021-06-01 16:51:11.919101	2021-06-01 16:51:11.919101
133	{"assets": {"sort": "new"}}	21	Step	893	2021-06-01 17:01:34.162397	2021-06-01 17:01:34.162397
134	{"assets": {"sort": "new"}}	21	Step	894	2021-06-01 17:17:54.306093	2021-06-01 17:17:54.306093
135	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1622650208, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	28	Team	23	2021-06-02 16:10:08.036326	2021-06-02 16:10:08.036326
136	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1629732050, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	30	Team	24	2021-08-23 15:20:50.950173	2021-08-23 15:20:50.950173
137	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1629739298, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	31	Team	25	2021-08-23 17:21:38.745361	2021-08-23 17:21:38.745361
138	{"assets": {"sort": "new"}}	30	Step	1015	2021-08-24 14:20:20.712207	2021-08-24 14:20:20.712207
139	{"assets": {"sort": "new"}}	30	Step	1016	2021-08-24 14:21:04.821268	2021-08-24 14:21:04.821268
140	{"assets": {"sort": "new"}}	30	Step	1017	2021-08-24 14:21:37.641614	2021-08-24 14:21:37.641614
141	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1629899841, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	30	Team	25	2021-08-25 13:57:21.435687	2021-08-25 13:57:21.435687
142	{"projects": {"cards": {"sort": "new"}, "table": {"time": 1629905770, "order": [[2, "asc"]], "start": 0, "length": 10}, "filter": "active"}}	31	Team	24	2021-08-25 15:36:10.605402	2021-08-25 15:36:10.605402
143	{"assets": {"sort": "new"}}	31	Step	1015	2021-08-25 15:38:27.226915	2021-08-25 15:38:27.226915
144	{"assets": {"sort": "new"}}	31	Step	1016	2021-08-25 15:38:27.247528	2021-08-25 15:38:27.247528
145	{"assets": {"sort": "new"}}	31	Step	1017	2021-08-25 15:38:27.287371	2021-08-25 15:38:27.287371
146	{"assets": {"sort": "new"}}	30	Step	1018	2021-08-25 20:48:16.047437	2021-08-25 20:48:16.047437
147	{"assets": {"sort": "new"}}	31	Step	975	2021-08-26 19:43:56.303198	2021-08-26 19:43:56.303198
148	{"assets": {"sort": "new"}}	31	Step	976	2021-08-26 19:44:11.804141	2021-08-26 19:44:11.804141
149	{"assets": {"sort": "new"}}	31	Step	977	2021-08-26 19:44:11.838037	2021-08-26 19:44:11.838037
150	{"assets": {"sort": "new"}}	31	Step	978	2021-08-26 19:44:11.859814	2021-08-26 19:44:11.859814
151	{"assets": {"sort": "new"}}	31	Step	979	2021-08-26 19:44:32.421657	2021-08-26 19:44:32.421657
152	{"assets": {"sort": "new"}}	31	Step	980	2021-08-26 19:44:32.450453	2021-08-26 19:44:32.450453
153	{"assets": {"sort": "new"}}	31	Step	981	2021-08-26 19:44:32.471677	2021-08-26 19:44:32.471677
154	{"assets": {"sort": "new"}}	31	Step	982	2021-08-26 19:44:32.503947	2021-08-26 19:44:32.503947
155	{"assets": {"sort": "new"}}	31	Step	983	2021-08-26 19:44:32.525834	2021-08-26 19:44:32.525834
156	{"assets": {"sort": "new"}}	31	Step	984	2021-08-26 19:44:32.562535	2021-08-26 19:44:32.562535
157	{"assets": {"sort": "new"}}	31	Step	985	2021-08-26 19:44:32.587353	2021-08-26 19:44:32.587353
158	{"assets": {"sort": "new"}}	31	Step	986	2021-08-26 19:44:32.610971	2021-08-26 19:44:32.610971
159	{"assets": {"sort": "new"}}	31	Step	987	2021-08-26 19:44:32.631479	2021-08-26 19:44:32.631479
160	{"assets": {"sort": "new"}}	31	Step	988	2021-08-26 19:44:32.650609	2021-08-26 19:44:32.650609
161	{"assets": {"sort": "new"}}	31	Step	989	2021-08-26 19:44:32.66773	2021-08-26 19:44:32.66773
162	{"assets": {"sort": "new"}}	31	Step	990	2021-08-26 19:44:32.687338	2021-08-26 19:44:32.687338
163	{"assets": {"sort": "new"}}	31	Step	991	2021-08-26 19:45:27.844781	2021-08-26 19:45:27.844781
164	{"assets": {"sort": "new"}}	31	Step	992	2021-08-26 19:45:27.89628	2021-08-26 19:45:27.89628
165	{"assets": {"sort": "new"}}	31	Step	993	2021-08-26 19:45:27.917214	2021-08-26 19:45:27.917214
166	{"assets": {"sort": "new"}}	31	Step	994	2021-08-26 19:45:27.941175	2021-08-26 19:45:27.941175
167	{"assets": {"sort": "new"}}	31	Step	995	2021-08-26 19:45:27.978162	2021-08-26 19:45:27.978162
168	{"assets": {"sort": "new"}}	31	Step	996	2021-08-26 19:45:28.008965	2021-08-26 19:45:28.008965
169	{"assets": {"sort": "new"}}	31	Step	997	2021-08-26 19:45:28.036625	2021-08-26 19:45:28.036625
170	{"assets": {"sort": "new"}}	31	Step	998	2021-08-26 19:45:28.061308	2021-08-26 19:45:28.061308
171	{"assets": {"sort": "new"}}	31	Step	999	2021-08-26 19:45:28.0976	2021-08-26 19:45:28.0976
172	{"assets": {"sort": "new"}}	31	Step	1000	2021-08-26 19:45:28.121383	2021-08-26 19:45:28.121383
173	{"assets": {"sort": "new"}}	31	Step	1001	2021-08-26 19:45:28.146431	2021-08-26 19:45:28.146431
174	{"assets": {"sort": "new"}}	31	Step	1002	2021-08-26 19:45:41.05556	2021-08-26 19:45:41.05556
175	{"assets": {"sort": "new"}}	31	Step	1003	2021-08-26 19:45:41.077286	2021-08-26 19:45:41.077286
176	{"assets": {"sort": "new"}}	31	Step	1004	2021-08-26 19:45:41.119616	2021-08-26 19:45:41.119616
177	{"assets": {"sort": "new"}}	31	Step	1005	2021-08-26 19:45:46.327013	2021-08-26 19:45:46.327013
178	{"assets": {"sort": "new"}}	31	Step	1006	2021-08-26 19:45:46.369047	2021-08-26 19:45:46.369047
179	{"assets": {"sort": "new"}}	31	Step	1007	2021-08-26 19:45:46.392303	2021-08-26 19:45:46.392303
180	{"assets": {"sort": "new"}}	31	Step	1008	2021-08-26 19:45:57.907695	2021-08-26 19:45:57.907695
181	{"assets": {"sort": "new"}}	31	Step	1009	2021-08-26 19:45:57.934811	2021-08-26 19:45:57.934811
182	{"assets": {"sort": "new"}}	31	Step	1010	2021-08-26 19:45:57.971228	2021-08-26 19:45:57.971228
183	{"assets": {"sort": "new"}}	31	Step	1011	2021-08-26 19:45:57.994249	2021-08-26 19:45:57.994249
184	{"assets": {"sort": "new"}}	31	Step	1012	2021-08-26 19:45:58.016048	2021-08-26 19:45:58.016048
185	{"assets": {"sort": "new"}}	31	Step	1013	2021-08-26 19:45:58.037518	2021-08-26 19:45:58.037518
186	{"assets": {"sort": "new"}}	31	Step	1014	2021-08-26 19:46:43.809383	2021-08-26 19:46:43.809383
187	{"assets": {"sort": "new"}}	30	Step	1019	2021-08-26 20:26:39.488307	2021-08-26 20:26:39.488307
188	{"assets": {"sort": "new"}}	30	Step	1020	2021-08-26 20:55:14.816587	2021-08-26 20:55:14.816587
189	{"assets": {"sort": "new"}}	30	Step	1021	2021-09-17 14:44:02.44634	2021-09-17 14:44:02.44634
190	{"assets": {"sort": "new"}}	30	Step	1022	2021-09-17 14:45:10.5546	2021-09-17 14:45:10.5546
191	{"assets": {"sort": "new"}}	30	Step	1023	2021-09-17 14:45:35.995189	2021-09-17 14:45:35.995189
192	{"projects": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	1	Team	1	2021-11-16 15:16:13.146956	2021-11-16 15:16:13.146956
193	{"experiments": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	1	Project	32	2021-11-16 15:16:19.389129	2021-11-16 15:16:19.389129
194	{"projects": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	33	Team	27	2021-11-16 15:28:37.18413	2021-11-16 15:28:37.18413
196	{"projects": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	33	Team	1	2021-11-16 15:31:34.995093	2021-11-16 15:31:34.995093
197	{"experiments": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	33	Project	1	2021-11-17 12:23:43.553103	2021-11-17 12:23:43.553103
198	{"experiments": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	4	Project	31	2021-12-08 18:45:07.526914	2021-12-08 18:45:07.526914
199	{"assets": {"sort": "new"}}	4	Step	281	2021-12-08 18:45:17.262868	2021-12-08 18:45:17.262868
200	{"projects": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	4	Team	7	2021-12-08 18:45:33.458468	2021-12-08 18:45:33.458468
201	{"experiments": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	4	Project	25	2021-12-08 18:45:38.296734	2021-12-08 18:45:38.296734
195	{"experiments": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	33	Project	32	2021-11-16 15:30:44.928014	2022-02-08 14:52:43.750312
202	{"projects": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	34	Team	29	2022-02-09 13:04:47.612824	2022-02-09 13:04:47.612824
203	{"projects": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	34	Team	1	2022-02-09 13:07:00.087813	2022-02-09 13:07:00.087813
204	{"experiments": {"active": {"sort": "new"}, "archived": {"sort": "new"}, "view_type": "cards"}}	34	Project	32	2022-02-09 13:07:02.315996	2022-02-09 13:07:02.315996
\.


--
-- Data for Name: result_tables; Type: TABLE DATA; Schema: public; Owner: postgres
--

COPY public.result_tables (id, result_id, table_id) FROM stdin;
1	2	3
2	3	5
3	4	7
4	8	8
5	9	9
6	23	12
7	24	14
8	25	16
9	29	17
10	30	18
11	44	21
12	45	23
13	46	25
14	50	26
15	51	27
16	65	30
17	66	32
18	67	34
19	71	35
20	72	36
21	86	39
22	87	41
23	88	43
24	92	44
25	93	45
26	107	48
27	108	50
28	109	52
29	113	53
30	114	54
31	128	57
32	129	59
33	130	61
34	134	62
35	135	63
36	149	66
37	150	68
38	151	70
39	155	71
40	156	72
41	170	75
42	171	77
43	172	79
44	176	80
45	177	81
46	191	84
47	192	86
48	193	88
49	197	89
50	198	90
51	212	93
52	213	95
53	214	97
54	218	98
55	219	99
56	233	102
57	234	104
58	235	106
59	239	107
60	240	108
61	254	111
62	255	113
63	256	115
64	260	116
65	261	117
66	275	120
67	276	122
68	277	124
69	281	125
70	282	126
71	299	129
72	300	131
73	301	133
74	305	134
75	306	135
76	320	138
77	321	140
78	322	142
79	326	143
80	327	144
81	329	147
82	330	149
83	331	151
84	335	152
85	336	153
86	352	156
87	353	158
88	355	160
89	359	161
90	360	162
91	383	165
92	384	167
93	385	169
94	389	170
95	390	171
96	404	174
97	405	176
98	406	178
99	410	179
100	411	180
101	425	183
102	426	185
103	427	187
104	431	188
105	432	189
106	446	192
107	447	194
108	448	196
109	452	197
110	453	198
111	467	201
112	468	203
113	469	205
114	473	206
115	474	207
116	489	210
117	490	212
118	491	214
119	495	215
120	496	216
121	510	219
122	511	221
123	512	223
124	516	224
125	517	225
\.


--
-- Name: active_storage_attachments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.active_storage_attachments_id_seq ', 150, true);


--
-- Name: active_storage_blobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.active_storage_blobs_id_seq ', 150, true);


--
-- Name: active_storage_variant_records_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.active_storage_variant_records_id_seq ', 1, false);


--
-- Name: activities_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.activities_id_seq ', 317, true);


--
-- Name: activity_filters_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.activity_filters_id_seq ', 1, false);


--
-- Name: asset_text_data_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.asset_text_data_id_seq ', 19, true);


--
-- Name: assets_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.assets_id_seq ', 605, true);


--
-- Name: checklist_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.checklist_items_id_seq ', 857, true);


--
-- Name: checklists_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.checklists_id_seq ', 167, true);


--
-- Name: comments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.comments_id_seq ', 604, true);


--
-- Name: connections_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.connections_id_seq ', 205, true);


--
-- Name: delayed_jobs_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.delayed_jobs_id_seq ', 2007, true);


--
-- Name: experiments_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.experiments_id_seq ', 59, true);


--
-- Name: label_printers_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.label_printers_id_seq ', 1, false);


--
-- Name: label_templates_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.label_templates_id_seq ', 1, true);


--
-- Name: my_module_groups_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_groups_id_seq ', 32, true);


--
-- Name: my_module_repository_rows_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_repository_rows_id_seq ', 875, true);


--
-- Name: my_module_status_conditions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_status_conditions_id_seq ', 1, false);


--
-- Name: my_module_status_consequences_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_status_consequences_id_seq ', 2, true);


--
-- Name: my_module_status_flows_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_status_flows_id_seq ', 1, true);


--
-- Name: my_module_status_implications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_status_implications_id_seq ', 1, false);


--
-- Name: my_module_statuses_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_statuses_id_seq ', 3, true);


--
-- Name: my_module_tags_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_module_tags_id_seq ', 456, true);


--
-- Name: my_modules_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.my_modules_id_seq ', 284, true);


--
-- Name: notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.notifications_id_seq ', 38, true);


--
-- Name: oauth_access_grants_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.oauth_access_grants_id_seq ', 1, false);


--
-- Name: oauth_access_tokens_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.oauth_access_tokens_id_seq ', 1, false);


--
-- Name: oauth_applications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.oauth_applications_id_seq ', 1, false);


--
-- Name: project_folders_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.project_folders_id_seq ', 1, false);


--
-- Name: projects_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.projects_id_seq ', 753, true);


--
-- Name: protocol_keywords_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.protocol_keywords_id_seq ', 1, false);


--
-- Name: protocol_protocol_keywords_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.protocol_protocol_keywords_id_seq ', 1, false);


--
-- Name: protocols_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.protocols_id_seq ', 303, true);



--
-- Name: report_elements_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.report_elements_id_seq ', 1, false);


--
-- Name: report_template_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.report_template_values_id_seq ', 1, false);


--
-- Name: reports_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.reports_id_seq ', 1, false);


--
-- Name: repositories_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repositories_id_seq ', 28, true);


--
-- Name: repository_asset_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_asset_values_id_seq ', 1, false);


--
-- Name: repository_cells_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_cells_id_seq ', 250, true);


--
-- Name: repository_checklist_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_checklist_items_id_seq ', 1, false);


--
-- Name: repository_checklist_items_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_checklist_items_values_id_seq ', 1, false);


--
-- Name: repository_checklist_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_checklist_values_id_seq ', 1, false);


--
-- Name: repository_columns_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_columns_id_seq ', 50, true);


--
-- Name: repository_date_time_range_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_date_time_range_values_id_seq ', 1, false);


--
-- Name: repository_date_time_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_date_time_values_id_seq ', 1, false);


--
-- Name: repository_list_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_list_items_id_seq ', 154, true);


--
-- Name: repository_list_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_list_values_id_seq ', 250, true);


--
-- Name: repository_number_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_number_values_id_seq ', 1, false);


--
-- Name: repository_rows_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_rows_id_seq ', 128, true);


--
-- Name: repository_status_items_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_status_items_id_seq ', 1, false);


--
-- Name: repository_status_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_status_values_id_seq ', 1, false);


--
-- Name: repository_table_states_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_table_states_id_seq ', 24, true);


--
-- Name: repository_text_values_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.repository_text_values_id_seq ', 1, false);


--
-- Name: result_assets_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.result_assets_id_seq ', 301, true);


--
-- Name: result_tables_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.result_tables_id_seq ', 125, true);


--
-- Name: result_texts_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.result_texts_id_seq ', 103, true);


--
-- Name: results_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.results_id_seq ', 529, true);


--
-- Name: settings_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.settings_id_seq ', 1, false);


--
-- Name: step_assets_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.step_assets_id_seq ', 304, true);


--
-- Name: step_tables_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.step_tables_id_seq ', 100, true);


--
-- Name: steps_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.steps_id_seq ', 1023, true);


--
-- Name: system_notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.system_notifications_id_seq ', 1, false);


--
-- Name: tables_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.tables_id_seq ', 225, true);


--
-- Name: tags_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.tags_id_seq ', 227, true);


--
-- Name: team_repositories_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.team_repositories_id_seq ', 1, false);


--
-- Name: teams_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.teams_id_seq ', 30, true);


--
-- Name: temp_files_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.temp_files_id_seq ', 1, false);


--
-- Name: tiny_mce_assets_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.tiny_mce_assets_id_seq ', 5, true);


--
-- Name: tokens_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.tokens_id_seq ', 1, false);


--
-- Name: user_identities_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.user_identities_id_seq ', 1, false);


--
-- Name: user_my_modules_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.user_my_modules_id_seq ', 245, true);


--
-- Name: user_notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.user_notifications_id_seq ', 27, true);


--
-- Name: user_projects_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.user_projects_id_seq ', 55, true);


--
-- Name: user_system_notifications_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.user_system_notifications_id_seq ', 1, false);


--
-- Name: user_teams_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.user_teams_id_seq ', 54, true);


--
-- Name: users_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.users_id_seq ', 35, true);


--
-- Name: view_states_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.view_states_id_seq ', 204, true);


--
-- Name: webhooks_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.webhooks_id_seq ', 1, false);


--
-- Name: wopi_actions_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.wopi_actions_id_seq ', 1, false);


--
-- Name: wopi_apps_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.wopi_apps_id_seq ', 1, false);


--
-- Name: wopi_discoveries_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.wopi_discoveries_id_seq ', 1, false);


--
-- Name: zip_exports_id_seq; Type: SEQUENCE SET; Schema: public; Owner: postgres
--

SELECT pg_catalog.setval( 'public.zip_exports_id_seq ', 1, false);


--
-- PostgreSQL database dump complete
--
update public.teams set created_by_id = 1 where id = 1;
update public.teams set created_by_id = 11 where id = 10;
update public.teams set created_by_id = 8 where id = 6;
update public.teams set created_by_id = 30 where id = 24;
update public.teams set created_by_id = 12 where id = 11;
update public.teams set created_by_id = 6 where id = 4;
update public.teams set created_by_id = 16 where id = 13;
update public.teams set created_by_id = 28 where id = 23;
update public.teams set created_by_id = 7 where id = 5;
update public.teams set created_by_id = 23 where id = 18;
update public.teams set created_by_id = 10 where id = 9;
update public.teams set created_by_id = 24 where id = 19;
update public.teams set created_by_id = 26 where id = 21;
update public.teams set created_by_id = 21 where id = 16;
update public.teams set created_by_id = 9 where id = 8;
update public.teams set created_by_id = 25 where id = 20;
update public.teams set created_by_id = 22 where id = 17;
update public.teams set created_by_id = 31 where id = 25;
update public.teams set created_by_id = 17 where id = 14;
update public.teams set created_by_id = 27 where id = 22;
update public.teams set created_by_id = 20 where id = 15;
update public.teams set created_by_id = 32 where id = 26;
update public.teams set created_by_id = 33 where id = 27;
update public.teams set created_by_id = 4 where id = 7;
update public.teams set created_by_id = 4 where id = 3;
update public.teams set created_by_id = 4 where id = 28;
update public.teams set created_by_id = 34 where id = 29;
update public.teams set created_by_id = 35 where id = 30;

update public.teams set last_modified_by_id = 1 where id = 1;
update public.teams set last_modified_by_id = 11 where id = 10;
update public.teams set last_modified_by_id = 8 where id = 6;
update public.teams set last_modified_by_id = 30 where id = 24;
update public.teams set last_modified_by_id = 12 where id = 11;
update public.teams set last_modified_by_id = 6 where id = 4;
update public.teams set last_modified_by_id = 16 where id = 13;
update public.teams set last_modified_by_id = 28 where id = 23;
update public.teams set last_modified_by_id = 7 where id = 5;
update public.teams set last_modified_by_id = 23 where id = 18;
update public.teams set last_modified_by_id = 10 where id = 9;
update public.teams set last_modified_by_id = 24 where id = 19;
update public.teams set last_modified_by_id = 26 where id = 21;
update public.teams set last_modified_by_id = 21 where id = 16;
update public.teams set last_modified_by_id = 9 where id = 8;
update public.teams set last_modified_by_id = 25 where id = 20;
update public.teams set last_modified_by_id = 22 where id = 17;
update public.teams set last_modified_by_id = 31 where id = 25;
update public.teams set last_modified_by_id = 17 where id = 14;
update public.teams set last_modified_by_id = 27 where id = 22;
update public.teams set last_modified_by_id = 20 where id = 15;
update public.teams set last_modified_by_id = 32 where id = 26;
update public.teams set last_modified_by_id = 33 where id = 27;
update public.teams set last_modified_by_id = 4 where id = 7;
update public.teams set last_modified_by_id = 4 where id = 3;
update public.teams set last_modified_by_id = 4 where id = 28;
update public.teams set last_modified_by_id = 34 where id = 29;
update public.teams set last_modified_by_id = 35 where id = 30;
